{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### Kindly cite this notebook in your publications if you use it for calculations.\n",
    "\n",
    "##### Cite as : NyroModellerNotebook v1, Girinath G. Pillai : https://github.com/giribio/ModellerNotebooks \n",
    "##### DOI: https://doi.org/10.5281/zenodo.4294751"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Used the tutorial from Salilab : https://salilab.org/modeller/tutorial/advanced.html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Install anaconda from the web\n",
    "# https://www.anaconda.com/products/individual\n",
    "\n",
    "###### nglview, numpy and matplotlib required for result analysis and ploting"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Install MODELLER using the following commands \n",
    "Ensure Anaconda is installed, therefore python3, conda, pip3, jupyter notebook and others are installed\n",
    "\n",
    "TESTED in LINUX and MACOSX, In Windows this notebook should work, but change commands for curl/wget/cat/etc as per the OS requirements. Create a folder and run the notebook within folder and download all supporting files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Warning: 'salilab' already in 'channels' list, moving to the top\n",
      "\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "conda config --add channels salilab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting package metadata (current_repodata.json): done\n",
      "Solving environment: done\n",
      "\n",
      "# All requested packages already installed.\n",
      "\n",
      "\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "conda install modeller"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### Go to https://salilab.org/modeller/ to get a license key, and then set the 'license' variable to it in the file ~//lib/modeller-9.25/modlib/modeller/config.py"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Advanced Homology Modeling\n",
    "#### Multiple template modelling"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Download the latest PDB sequence database from https://salilab.org/modeller/downloads/pdb_95.pir.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100 15.5M  100 15.5M    0     0  2711k      0  0:00:05  0:00:05 --:--:-- 3511k-:--:--     0\n"
     ]
    }
   ],
   "source": [
    "#To download latest PDB sequence database in PIR file\n",
    "!curl https://salilab.org/modeller/downloads/pdb_95.pir.gz -o pdb_95.pir.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Unzip the file\n",
    "!gunzip pdb_95.pir.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Searching template structures related to target sequence"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Which ever Target sequence, you would like to model, get the FASTA format of the sequence\n",
    "And make generate PIR format which is readable by MODELLER with file extension .ali\n",
    "\n",
    "SAMPLE\n",
    ">P1;CODE ##CODE (used in next line too) - some letters to represent target sequence##\n",
    "sequence:CODE:::::0.00:0.00 ##\n",
    "\n",
    "##FASTA FORMAT SEQUENCE##\n",
    "and at the end * (symbol without gap or space represents end of sequence)\n",
    "Then save the file with .ali as file extension, in this example we used the following \n",
    "sequence and filename with TvLDH.ali (copy paste the following to editor and save as TvLDH.ali)\n",
    "\n",
    "\n",
    ">P1;TvLDH\n",
    "sequence:TvLDH:::::::0.00: 0.00\n",
    "MSEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKA\n",
    "AFKDIDCAFLVASMPLKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPEN\n",
    "FSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKI\n",
    "GHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVV\n",
    "EGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQGG*"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "                         MODELLER 9.25, 2020/09/03, r11894\n",
      "\n",
      "     PROTEIN STRUCTURE MODELLING BY SATISFACTION OF SPATIAL RESTRAINTS\n",
      "\n",
      "\n",
      "                     Copyright(c) 1989-2020 Andrej Sali\n",
      "                            All Rights Reserved\n",
      "\n",
      "                             Written by A. Sali\n",
      "                               with help from\n",
      "              B. Webb, M.S. Madhusudhan, M-Y. Shen, G.Q. Dong,\n",
      "          M.A. Marti-Renom, N. Eswar, F. Alber, M. Topf, B. Oliva,\n",
      "             A. Fiser, R. Sanchez, B. Yerkovich, A. Badretdinov,\n",
      "                     F. Melo, J.P. Overington, E. Feyfant\n",
      "                 University of California, San Francisco, USA\n",
      "                    Rockefeller University, New York, USA\n",
      "                      Harvard University, Cambridge, USA\n",
      "                   Imperial Cancer Research Fund, London, UK\n",
      "              Birkbeck College, University of London, London, UK\n",
      "\n",
      "\n",
      "Kind, OS, HostName, Kernel, Processor: 4, Darwin Girinaths-MacBook-Pro.local 19.6.0 x86_64\n",
      "Date and time of compilation         : 2020/09/03 08:25:47\n",
      "MODELLER executable type             : mac10v4-intel64\n",
      "Job starting time (YY/MM/DD HH:MM:SS): 2020/12/03 23:38:34\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       191566     187.076     0.183\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       192094     187.592     0.183\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       240694     235.053     0.230\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       253994     248.041     0.242\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       523244     510.980     0.499\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      4084367    3988.640     3.895\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4086475    3990.698     3.897\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4087925    3992.114     3.899\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4090825    3994.946     3.901\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4096625    4000.610     3.907\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     17535024   17124.047    16.723\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17546624   17135.375    16.734\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17569824   17158.031    16.756\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17616224   17203.344    16.800\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17709024   17293.969    16.889\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     24428224   23855.688    23.297\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           TvLDH.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       525430     513.115     0.501\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       526880     514.531     0.502\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       529780     517.363     0.505\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       535580     523.027     0.511\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       536916     524.332     0.512\n",
      "\n",
      "Read the alignment from file       : TvLDH.ali\n",
      "\n",
      "Total number of alignment positions:   335\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      TvLDH     335      1       TvLDH\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:       538581     525.958     0.514\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       863756     843.512     0.824\n",
      "profile_iteration_> processing sequence:       1    335      1     0.0100000     0.0100000     0.0100000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       864834     844.564     0.825\n",
      "profile_iteration_> processing sequence:    6720    335      1     3.3699999     0.0005015     0.0000019  1994\n",
      "profile_iteration_> processing sequence:   13439    335      1     6.9099998     0.0005142     0.0000020  1945\n",
      "profile_iteration_> processing sequence:   20158    335      1    11.0299997     0.0005472     0.0000021  1828\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       865209     844.931     0.825\n",
      "profile_iteration_> processing sequence:   26877    335      1    15.0699997     0.0005607     0.0000022  1783\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       865445     845.161     0.825\n",
      "profile_iteration_> processing sequence:   33596    335      1    18.8099995     0.0005599     0.0000022  1786\n",
      "profile_iteration_> processing sequence:   40315    335      1    22.2900009     0.0005529     0.0000022  1809\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       868395     848.042     0.828\n",
      "profile_iteration_> processing sequence:   47034    335      1    25.6599998     0.0005456     0.0000021  1833\n",
      "profile_iteration_> processing sequence:   53753    335      1    29.2199993     0.0005436     0.0000021  1840\n",
      "profile_iteration_> processing sequence:   60472    335      1    33.2000008     0.0005490     0.0000022  1821\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       871905     851.470     0.832\n",
      "profile_iteration_> processing sequence:   67191    335      1    37.4599991     0.0005575     0.0000022  1794\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    26.38443\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    247.6263\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0182,      121,        0.0067,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00045   0.00004\n",
      "Z:         1  -2.35000   0.00089   0.00016\n",
      "Z:         1  -2.25000   0.00089   0.00055\n",
      "Z:         1  -2.15000   0.00610   0.00163\n",
      "Z:         1  -2.05000   0.00551   0.00416\n",
      "Z:         1  -1.95000   0.01831   0.00933\n",
      "Z:         1  -1.85000   0.02262   0.01872\n",
      "Z:         1  -1.75000   0.04167   0.03400\n",
      "Z:         1  -1.65000   0.05596   0.05658\n",
      "Z:         1  -1.55000   0.09391   0.08721\n",
      "Z:         1  -1.45000   0.12516   0.12565\n",
      "Z:         1  -1.35000   0.15746   0.17059\n",
      "Z:         1  -1.25000   0.21461   0.21981\n",
      "Z:         1  -1.15000   0.25569   0.27052\n",
      "Z:         1  -1.05000   0.31254   0.31973\n",
      "Z:         1  -0.95000   0.34855   0.36469\n",
      "Z:         1  -0.85000   0.40645   0.40316\n",
      "Z:         1  -0.75000   0.44202   0.43360\n",
      "Z:         1  -0.65000   0.46047   0.45519\n",
      "Z:         1  -0.55000   0.44306   0.46779\n",
      "Z:         1  -0.45000   0.47193   0.47182\n",
      "Z:         1  -0.35000   0.44976   0.46811\n",
      "Z:         1  -0.25000   0.45616   0.45775\n",
      "Z:         1  -0.15000   0.44797   0.44196\n",
      "Z:         1  -0.05000   0.43353   0.42194\n",
      "Z:         1   0.05000   0.38650   0.39889\n",
      "Z:         1   0.15000   0.38487   0.37383\n",
      "Z:         1   0.25000   0.33665   0.34769\n",
      "Z:         1   0.35000   0.32355   0.32121\n",
      "Z:         1   0.45000   0.28366   0.29500\n",
      "Z:         1   0.55000   0.27518   0.26953\n",
      "Z:         1   0.65000   0.23708   0.24513\n",
      "Z:         1   0.75000   0.22294   0.22205\n",
      "Z:         1   0.85000   0.19467   0.20043\n",
      "Z:         1   0.95000   0.18246   0.18036\n",
      "Z:         1   1.05000   0.16862   0.16185\n",
      "Z:         1   1.15000   0.15255   0.14489\n",
      "Z:         1   1.25000   0.13067   0.12944\n",
      "Z:         1   1.35000   0.12100   0.11542\n",
      "Z:         1   1.45000   0.09986   0.10275\n",
      "Z:         1   1.55000   0.09153   0.09133\n",
      "Z:         1   1.65000   0.08037   0.08109\n",
      "Z:         1   1.75000   0.07635   0.07191\n",
      "Z:         1   1.85000   0.06102   0.06371\n",
      "Z:         1   1.95000   0.05670   0.05639\n",
      "Z:         1   2.05000   0.05358   0.04988\n",
      "Z:         1   2.15000   0.05239   0.04409\n",
      "Z:         1   2.25000   0.04182   0.03895\n",
      "Z:         1   2.35000   0.03453   0.03439\n",
      "Z:         1   2.45000   0.02932   0.03035\n",
      "Z:         1   2.55000   0.02932   0.02678\n",
      "Z:         1   2.65000   0.02247   0.02361\n",
      "Z:         1   2.75000   0.02188   0.02082\n",
      "Z:         1   2.85000   0.02054   0.01835\n",
      "Z:         1   2.95000   0.01682   0.01617\n",
      "Z:         1   3.05000   0.01459   0.01424\n",
      "Z:         1   3.15000   0.01176   0.01255\n",
      "Z:         1   3.25000   0.01176   0.01105\n",
      "Z:         1   3.35000   0.00878   0.00973\n",
      "Z:         1   3.45000   0.00893   0.00857\n",
      "Z:         1   3.55000   0.00655   0.00754\n",
      "Z:         1   3.65000   0.00640   0.00664\n",
      "Z:         1   3.75000   0.00506   0.00584\n",
      "Z:         1   3.85000   0.00566   0.00514\n",
      "Z:         1   3.95000   0.00521   0.00453\n",
      "Z:         1   4.05000   0.00536   0.00398\n",
      "Z:         1   4.15000   0.00327   0.00350\n",
      "Z:         1   4.25000   0.00313   0.00308\n",
      "Z:         1   4.35000   0.00283   0.00271\n",
      "Z:         1   4.45000   0.00253   0.00239\n",
      "Z:         1   4.55000   0.00268   0.00210\n",
      "Z:         1   4.65000   0.00179   0.00185\n",
      "Z:         1   4.75000   0.00208   0.00163\n",
      "Z:         1   4.85000   0.00253   0.00143\n",
      "Z:         1   4.95000   0.00179   0.00126\n",
      "Z:         1   5.05000   0.00134   0.00111\n",
      "Z:         1   5.15000   0.00179   0.00097\n",
      "Z:         1   5.25000   0.00164   0.00086\n",
      "Z:         1   5.35000   0.00030   0.00075\n",
      "Z:         1   5.45000   0.00134   0.00066\n",
      "Z:         1   5.55000   0.00060   0.00058\n",
      "Z:         1   5.65000   0.00119   0.00051\n",
      "Z:         1   5.75000   0.00060   0.00045\n",
      "Z:         1   5.85000   0.00000   0.00040\n",
      "Z:         1   5.95000   0.00045   0.00035\n",
      "Z:         1   6.05000   0.00045   0.00031\n",
      "Z:         1   6.15000   0.00015   0.00027\n",
      "Z:         1   6.25000   0.00074   0.00024\n",
      "Z:         1   6.35000   0.00045   0.00021\n",
      "Z:         1   6.45000   0.00000   0.00018\n",
      "Z:         1   6.55000   0.00015   0.00016\n",
      "Z:         1   6.65000   0.00015   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00030   0.00011\n",
      "Z:         1   6.95000   0.00000   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00000   0.00005\n",
      "Z:         1   7.55000   0.00015   0.00004\n",
      "Z:         1   7.65000   0.00030   0.00004\n",
      "Z:         1   7.75000   0.00015   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00015   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:      1156905    1129.790     1.103\n",
      "> 3d5tA                      1    3132   29250     321     335   44.27     0.0           2   312     4   325     3   316\n",
      "> 1sovA                      1    3257    4700     316     335   26.59    0.74E-02       3   160    81   256    76   248\n",
      "> 6k12A                      1    3397    4650     297     335   27.89    0.88E-02       4   181    94   304    81   270\n",
      "> 6k13A                      1    3424    5100     332     335   28.08    0.99E-03       5   189    94   304   103   305\n",
      "> 6or9A                      1    3835    5300     306     335   25.00    0.32E-03       6   252    47   323    37   292\n",
      "> 1guzA                      1    4067    7100     306     335   25.27    0.28E-07       7   265    13   301     8   280\n",
      "> 3gviA                      1    5566    5350     314     335   29.44    0.25E-03       8   175    13   209    10   206\n",
      "> 1gv1B                      1    5627    4900     290     335   23.44    0.23E-02       9   191    99   301    80   271\n",
      "> 1gv0A                      1    5647    5350     301     335   25.53    0.24E-03      10   274    13   323     8   289\n",
      "> 4bguA                      1    7704    6500     303     335   26.48    0.62E-06      11   281     7   314     3   289\n",
      "> 4bgvA                      1    7705    7650     323     335   31.82    0.18E-08      12   168    13   191    10   185\n",
      "> 1a5zA                      1    8650    6900     312     335   28.14    0.82E-07      13   164    75   242    63   229\n",
      "> 4e0bA                      1   10200    5700     305     335   27.53    0.40E-04      14   282     4   314     2   288\n",
      "> 4ln1A                      1   10437    4850     317     335   24.50    0.34E-02      15   183    78   270    79   278\n",
      "> 3wswA                      1   10921    5800     316     335   24.51    0.25E-04      16   187    78   270    70   273\n",
      "> 6itlA                      1   11933    6350     312     335   26.63    0.14E-05      17   196     7   209     3   201\n",
      "> 6itkA                      1   11962   26850     318     335   40.52     0.0          18   305     7   325     5   310\n",
      "> 4cl3A                      1   12409    6700     309     335   32.16    0.23E-06      19   166    13   191    10   180\n",
      "> 3fi9A                      1   12918    8500     321     335   27.85    0.22E-10      20   231     9   260    12   248\n",
      "> 5mdhA                      1   14863   32900     333     335   44.41     0.0          21   328     2   332     1   331\n",
      "> 7mdhA                      1   14882   22600     351     335   33.74     0.0          22   325     6   334    14   339\n",
      "> 5kkcB                      1   15279    4850     320     335   26.02    0.34E-02      23   191   101   301    98   293\n",
      "> 5zi2A                      1   16147    5400     339     335   31.97    0.21E-03      24   108    87   195    76   197\n",
      "> 1bdmA                      1   17070   30700     318     335   44.52     0.0          25   309     1   325     1   310\n",
      "> 4h7pA                      1   19011   26600     313     335   40.85     0.0          26   304     4   324     3   308\n",
      "> 4h7pB                      1   19012   26350     306     335   42.33     0.0          27   298     4   324     2   301\n",
      "> 3hhpA                      1   19442    5950     312     335   27.24    0.11E-04      28   289     7   320     3   303\n",
      "> 4tvoA                      1   20363   26750     328     335   39.94     0.0          29   312     7   324     6   318\n",
      "> 4tvoB                      1   20364   25250     317     335   40.13     0.0          30   303     7   324     5   308\n",
      "> 2hjrA                      1   21755    6200     313     335   26.00    0.31E-05      31   288    13   333    10   309\n",
      "> 1t2dA                      1   21927    5200     315     335   24.70    0.55E-03      32   238     5   256     4   250\n",
      "> 4plgA                      1   22444    6950     324     335   27.67    0.66E-07      33   236     1   256     1   253\n",
      "> 4pltB                      1   22452    8400     319     335   31.48    0.37E-10      34   240    13   271    11   280\n",
      "> 4plvA                      1   22453    6950     305     335   31.25    0.61E-07      35   231    13   271    10   265\n",
      "> 4plzA                      1   22454    4850     302     335   26.11    0.32E-02      36   155    97   256    80   236\n",
      "> 3d0oA                      1   22775    4850     309     335   24.00    0.33E-02      37   212    79   296    71   295\n",
      "> 1y6jA                      1   23859    5750     289     335   32.73    0.29E-04      38   109    77   191    58   167\n",
      "> 1ez4B                      1   24804    6200     318     335   30.68    0.31E-05      39   159    69   239    55   230\n",
      "> 5hs4A                      1   28807    4950     301     335   26.63    0.19E-02      40   167     8   191     4   172\n",
      "> 5nueA                      1   29100   28000     331     335   38.30     0.0          41   327     3   332     3   331\n",
      "> 1mldA                      1   31072    6100     313     335   25.93    0.52E-05      42   183     5   198     1   189\n",
      "> 6j9sB                      1   31764    5450     315     335   26.86    0.15E-03      43   157    71   239    59   233\n",
      "> 6iheA                      1   32500    5000     303     335   29.93    0.15E-02      44   137    80   225    70   206\n",
      "> 6r8gA                      1   34789    6350     311     335   25.74    0.14E-05      45   232    13   257     8   244\n",
      "> 2v65A                      1   35640    4950     326     335   23.76    0.21E-02      46   197    94   300    97   298\n",
      "> 1v6aA                      1   35647    5100     332     335   25.25    0.99E-03      47   197    94   300   103   304\n",
      "> 6balA                      1   36021    6350     312     335   25.55    0.14E-05      48   269     4   295     1   274\n",
      "> 4q3nA                      1   38966    5850     314     335   23.33    0.19E-04      49   232    78   320    70   309\n",
      "> 4uulA                      1   40092   87350     334     335   99.70     0.0          50   332     2   333     1   332\n",
      "> 4uupA                      1   40095   53150     335     335   60.42     0.0          51   331     3   333     1   331\n",
      "> 4uuoA                      1   40100   39200     328     335   49.69     0.0          52   318     3   327     1   318\n",
      "> 2a92A                      1   40298    5450     316     335   25.97    0.15E-03      53   174     8   191     6   186\n",
      "> 6bb1E                      1   40487    5150     310     335   26.63    0.70E-03      54   182    97   295    94   277\n",
      "> 1oc4A                      1   40691    5450     315     335   27.87    0.15E-03      55   174     5   191     4   186\n",
      "> 1smkA                      1   40965    8750     313     335   33.85    0.75E-11      56   188     7   202     4   198\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "> 6vdhA                      1   42414    5900     308     335   32.73    0.14E-04      57   159    13   191    12   176\n",
      "> 4jcoA                      1   43467    5700     303     335   25.96    0.39E-04      58   274     7   320     3   287\n",
      "> 1lldA                      1   44348    6650     313     335   30.67    0.30E-06      59   216    13   242     9   233\n",
      "> 1hyeA                      1   45128    6800     307     335   29.28    0.13E-06      60   173     7   191     3   183\n",
      "> 4wleA                      1   45463    6250     314     335   25.93    0.24E-05      61   186     5   198     2   190\n",
      "> 3tl2A                      1   45519    6550     312     335   28.29    0.50E-06      62   149    39   191    34   185\n",
      "> 3p7mA                      1   46506    6900     318     335   30.77    0.84E-07      63   185    13   207    11   205\n",
      "> 4nd2A                      1   46802    6850     317     335   26.04    0.11E-06      64   270     1   301     1   288\n",
      "> 4nd5A                      1   46815    4850     302     335   25.72    0.32E-02      65   258     1   301     1   276\n",
      "> 5w8jA                      1   47976    5600     329     335   26.21    0.73E-04      66   196    85   295    91   296\n",
      "> 1b8pA                      1   48421   29400     327     335   42.50     0.0          67   316     7   331     6   325\n",
      "> 2ldxA                      1   48665    5400     331     335   25.83    0.21E-03      68   227    66   306    67   306\n",
      "> 9ldtA                      1   48689    6000     331     335   25.94    0.93E-05      69   207    85   301    93   304\n",
      "> 1ldnA                      1   48709    5000     316     335   25.76    0.16E-02      70   214    78   298    73   301\n",
      "> 5ldhA                      1   48727    5800     333     335   25.59    0.26E-04      71   207    85   300    94   304\n",
      "> 6cepA                      1   49739    5750     331     335   25.12    0.34E-04      72   207    85   300    94   304\n",
      "> 6cepD                      1   49740    5250     317     335   25.50    0.43E-03      73   196    96   300    90   289\n",
      "> 1civA                      1   51585   23950     374     335   34.66     0.0          74   325     6   334    33   358\n",
      "> 5nqbA                      1   52324    5650     331     335   25.94    0.57E-04      75   207    85   301    93   304\n",
      "> 6um4A                      1   52654   25750     326     335   37.93     0.0          76   317     7   331     8   326\n",
      "> 4m1qA                      1   57644    5300     312     335   23.64    0.32E-03      77   162    78   242    69   233\n",
      "> 1pzgA                      1   58602    6350     327     335   30.00    0.15E-05      78   114    74   191    71   190\n",
      "> 2x0jA                      1   58672    5700     294     335   27.98    0.38E-04      79   218    78   320    68   285\n",
      "> 5ujkA                      1   59017    6150     315     335   31.58    0.40E-05      80   169    13   191    10   180\n",
      "> 4ajjA                      1   59997    5750     330     335   25.47    0.34E-04      81   207    85   301    92   303\n",
      "> 3nepX                      1   65237    4650     305     335   24.61    0.91E-02      82   244    13   274     8   263\n",
      "> 6pblA                      1   65962   26900     327     335   39.17     0.0          83   313     7   325     5   318\n",
      "> 1y7tA                      1   66077   33200     327     335   45.14     0.0          84   318     1   325     1   319\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.010000\n",
      "profile_iteration__> Score Collection        [sec]:    37.459999\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000558\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  1808.180786\n",
      "profile_iteration__> Statistics              [sec]:     0.090000\n",
      "profile_iteration__> Add to profile          [sec]:     0.280000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000565\n",
      "profile_iteration__> Overall performance [seq/sec]:  1771.006958\n",
      "openf___224_> Open           build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1121045    1094.771     1.069\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1122381    1096.075     1.070\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1123625    1097.290     1.072\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1124261    1097.911     1.072\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1124981    1098.614     1.073\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1125733    1099.349     1.074\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1126737    1100.329     1.075\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1127793    1101.360     1.076\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1128489    1102.040     1.076\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1129249    1102.782     1.077\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1130341    1103.849     1.078\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1131461    1104.942     1.079\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1132129    1105.595     1.080\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1132781    1106.231     1.080\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1133905    1107.329     1.081\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1134633    1108.040     1.082\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1135377    1108.767     1.083\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1136157    1109.528     1.084\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1137373    1110.716     1.085\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1138033    1111.360     1.085\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1138953    1112.259     1.086\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1140261    1113.536     1.087\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1141557    1114.802     1.089\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1142317    1115.544     1.089\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1142745    1115.962     1.090\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1143977    1117.165     1.091\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1145189    1118.349     1.092\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1146377    1119.509     1.093\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1147529    1120.634     1.094\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1148773    1121.849     1.096\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1149981    1123.028     1.097\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1151129    1124.149     1.098\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1152077    1125.075     1.099\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1153017    1125.993     1.100\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1153973    1126.927     1.101\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1154893    1127.825     1.101\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1155509    1128.427     1.102\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1156353    1129.251     1.103\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1156785    1129.673     1.103\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1157417    1130.290     1.104\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1158081    1130.938     1.104\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1159385    1132.212     1.106\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1160113    1132.923     1.106\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1160737    1133.532     1.107\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1161281    1134.063     1.107\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1162205    1134.966     1.108\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1162989    1135.731     1.109\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1163773    1136.497     1.110\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1164845    1137.544     1.111\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1165769    1138.446     1.112\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1167093    1139.739     1.113\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1168413    1141.028     1.114\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1169681    1142.267     1.115\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1170373    1142.942     1.116\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1171097    1143.649     1.117\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1171789    1144.325     1.118\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1172537    1145.056     1.118\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1173169    1145.673     1.119\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1174261    1146.739     1.120\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1175121    1147.579     1.121\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1175809    1148.251     1.121\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1176549    1148.974     1.122\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1177141    1149.552     1.123\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1177877    1150.271     1.123\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1178953    1151.321     1.124\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1179981    1152.325     1.125\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1180761    1153.087     1.126\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1182021    1154.317     1.127\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1182925    1155.200     1.128\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1183749    1156.005     1.129\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1184601    1156.837     1.130\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1185425    1157.642     1.131\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1186249    1158.446     1.131\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1187029    1159.208     1.132\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1188325    1160.474     1.133\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1189149    1161.278     1.134\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1190413    1162.513     1.135\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1191057    1163.142     1.136\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1191509    1163.583     1.136\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1192377    1164.431     1.137\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1193049    1165.087     1.138\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1193873    1165.892     1.139\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1194845    1166.841     1.139\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1196093    1168.060     1.141\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1197361    1169.298     1.142\n",
      "openf___224_> Open           build_profile.ali\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "\n",
    "log.verbose()\n",
    "env = environ()\n",
    "\n",
    "#-- Prepare the input files\n",
    "\n",
    "#-- Read in the sequence database\n",
    "sdb = sequence_db(env)\n",
    "sdb.read(seq_database_file='pdb_95.pir', seq_database_format='PIR',\n",
    "         chains_list='ALL', minmax_db_seq_len=(30, 4000), clean_sequences=True)\n",
    "\n",
    "#-- Write the sequence database in binary form\n",
    "sdb.write(seq_database_file='pdb_95.bin', seq_database_format='BINARY',\n",
    "          chains_list='ALL')\n",
    "\n",
    "#-- Now, read in the binary database\n",
    "sdb.read(seq_database_file='pdb_95.bin', seq_database_format='BINARY',\n",
    "         chains_list='ALL')\n",
    "\n",
    "#-- Read the target sequence/alignment in .ali format\n",
    "aln = alignment(env)\n",
    "aln.append(file='TvLDH.ali', alignment_format='PIR', align_codes='ALL')\n",
    "\n",
    "#-- Convert the input sequence/alignment into\n",
    "#   profile format\n",
    "prf = aln.to_profile()\n",
    "\n",
    "#-- Scan sequence database to pick up homologous sequences\n",
    "# *** Feel free to make changes in gap penalties, aln evalue and others if you are aware of the significance\n",
    "\n",
    "prf.build(sdb, matrix_offset=-450, rr_file='${LIB}/blosum62.sim.mat',\n",
    "          gap_penalties_1d=(-500, -50), n_prof_iterations=1,\n",
    "          check_profile=False, max_aln_evalue=0.01)\n",
    "\n",
    "#-- Write out the profile in text format\n",
    "prf.write(file='build_profile.prf', profile_format='TEXT')\n",
    "\n",
    "#-- Convert the profile back to alignment format\n",
    "aln = prf.to_alignment()\n",
    "\n",
    "#-- Write out the alignment file\n",
    "aln.write(file='build_profile.ali', alignment_format='PIR')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Number of sequences:     84\r\n",
      "# Length of profile  :    335\r\n",
      "# N_PROF_ITERATIONS  :      1\r\n",
      "# GAP_PENALTIES_1D   :   -500.0   -50.0\r\n",
      "# MATRIX_OFFSET      : -450.0\r\n",
      "# RR_FILE            : ${LIB}/blosum62.sim.mat\r\n",
      "    1 TvLDH                                    S     0   335     1   335     0     0     0    0.    0.0     MSEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKAAFKDIDCAFLVASMPLKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPENFSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKIGHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVVEGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQGG\r\n",
      "    2 3d5tA                                    X     1   321     4   325     3   316   312   44.    0.0     ---AKRVAVTGAAGQIAYSLLFRIANGDLLGDQPVILQLLDLPQAQAAVKGVVMELDDCAFPLLAGVVITDDPKVAFKDADVALLVGARP--RSMERKDLLSANAEIFTVQGAALNEVASRDVKVLVVGNPANTNAYIAMKSAPDLPKKNFTAMLRLDHNRALSQLAAKSGKPVASIEKLAVWGNHSPTMYPDFRFA--TAEGESLKLIN--DDVWNRDTFIPTVGKRGAAIIEARGLSSAASAANAAIDHVRDWVLGT-NGKWVTMGI--PSDGSYGIPEDIIYGVPV-ICENGEYKRVEGLEIDAFSREKMDGTLAELLEERD----------\r\n",
      "    3 1sovA                                    X     1   316    81   256    76   248   160   27.   0.74E-02 --------------------------------------------------------------------------------DVVIITAGLTKVPS--RNDLLPFNAKIIREVAQGVKKYC-PLAFVIVVTNP--LDCMVKCFHEASGLPKNMGMANVLDSARFRRFIADQLEISPRDIQATVI-GTHGDHMLPLARYVTFIKKGKEAKLAEIVERT-------KKAG---GEIVRLLGQGSAYAPALSAITMAQAFL-------------------------------------------------------------------------------\r\n",
      "    4 6k12A                                    X     1   297    94   304    81   270   181   28.   0.88E-02 ---------------------------------------------------------------------------------------------GESRLNLVQRNVNIFKFIIPNVVKYS-PDCILLIVSNPVDILTYVAWKLSGFPLNRVIGSGCNLDSARFRYLVSEMIGIHPSNFHGCIL-GEHGDSSVPILSGLNIAGM-SIKNCKDV----------HKKVTESAYEIIKLKGYTS--------------WAIGLSVGDLLRKVHPVSTKGQFGIDNEVFLSVPCVLGRNG---ISEVFK-------------------------------\r\n",
      "    5 6k13A                                    X     1   332    94   304   103   305   189   28.   0.99E-03 ---------------------------------------------------------------------------------------------GESRLNLVQRNVNIFKFIIPNVVKYS-PDCILLIVSNPVDILTYVAWKLSGFPLNRVIGSGCNLDSARFRYLVSEMIGIHPSNFHGCIL-GEHGDSSVPILSGLNIA--GMSIKNLHT-DIDTVFKDVHKKVTESAYEIIKLKGYTS--------------WAIGLSVGDLLRKVHPVSTKGQFGIDNEVFLSVPCVLGRNG---ISEVFK-------------------------------\r\n",
      "    6 6or9A                                    X     1   306    47   323    37   292   252   25.   0.32E-03 ----------------------------------------------DKLKGELMDLQGSAFMRNVKIQASTDYSISAGSKICV-VTAGVRQREGESRLDLVQRNTDVLKIIIPQLVKHS-PDTILIIASNPVDILTYVSWKLSGLPKHRVIGSGTNLDSARFRYLLSEKLGIATTSCHGYII-GEHGDSSVSGVNIAGVRLSDLNQKINWKETHTMVVKS--------AYEVIKLKGYTSWAIGLSLS-QLARAILSNANSVHAVSTYLK----GEHDINDEVFLSLPCVLGRSG---------VCDVIRQPLTQTERSQLHQ------------\r\n",
      "    7 1guzA                                    X     1   306    13   301     8   280   265   25.   0.28E-07 ------------AGNVGATTAFRLAEKQLARE----LVLLDVVEGIPQGKALDM--------YESGPVGLFDTKADTANSDIVIITAGLPRKPGMTREDLLMKNAGIVKEVTDNIMKHSKNPI-IIVVSNPLDIMTHVAWVRSGLPKERVIGMAGVLDAARFRSFIAMELGVSMQDINACVL-GGHGDAMVPVVKYTTVA----GIPISDLLPAETI-DKLVERTRNGGAEIVEHLKQGSAYAPASSVVEMVESIVLDRKRVLPCAVGL---EGQ-YGIDKTFV-GVPVKLGRNGVEQIYE----------------------------------\r\n",
      "    8 3gviA                                    X     1   314    13   209    10   206   175   29.   0.25E-03 ------------SGMIGGTLAHLAGLKEL-GD----VVLFDIAEGTPQGKGLDIAESSPVDGFDAKFTGAND-YAAIEGADVVIVTAGVPRKD-----DLLGINLKVMEQVGAGIKKYA-PEAFVICITNP---------LDAMVWALQKFSGLGVLDSARFRYFLSEEFNVSVEDV-TVFVLGGHGDSMVPDLVKMGWTSQDKLDKII------------------------------------------------------------------------------------------------------------------------------\r\n",
      "    9 1gv1B                                    X     1   290    99   301    80   271   191   23.   0.23E-02 --------------------------------------------------------------------------------------------------DLLMKNAGIVKEVTDNIMKHSKNPI-IIVVSNPLDIMTHVAWVRSGLPKERVIGMAGVLDAARFRSFIAMELGVSMQDINACVL-GGHGDAMVPVVKYTTVA----GIPISDLLPAETI-DKLVERTRNGGAEIVEHLKQGSAYAPASSVVEMVESIVLDRKRVLPCAVGL---EGQ-YGIDKTFV-GVPVKLGRNGVEQIYE----------------------------------\r\n",
      "   10 1gv0A                                    X     1   301    13   323     8   289   274   26.   0.24E-03 ------------AGNVGATTAFRLAEKQLARE----LVLLDVVEGIPQGKALDM--------YESGPVGLFDTKADTANSDIVVITAGLPRKPGMT----LSMNAGIVREVTGRIMEHSKNPI-IVVVSNPLDIMTHVAWQKSGLPKERVIGMAGVLDSARFRSFIAMELGVSMQDV-TACVLGGHGDAMVPVVKYTTVA----GIPVADLISAERIAE-LVERTRTGGAEIVNHLKQGSAYSPATSVVEMVESIVLDRK--RVLTCAVSL--DGQYGI-DGTFVGVPVKLGKNGVEHIYE---------IKLDQSDLDLLQK------------\r\n",
      "   11 4bguA                                    X     1   303     7   314     3   289   281   26.   0.62E-06 ------VSVIGAAGTVGAAAGYNLALRDVCDE----LVFVDIPKMEDKTVGQAADTNH-GIAYDSNTVVTQGGYEDTAGSDVVVITAGIPRQPGQTRIDLAGDNAPIMDDIGSSLAEYNDDFVSITTSNPVDLLNRHLYETGDRD-RHKVIGFGGRLDSARFRYVLSQRFDVPVKNV-DATILGEHGDAQV-----PVFSK-------VRVDGNDPAFSADEKELQESAMDVIERKGATQWGPAT--GVAHMEAVLHDT--GEVLP-GSLVLDGE-FGYE-DTAFGVPVKLGSNGIEEVVE-WDLDDYEADLMD---------------------\r\n",
      "   12 4bgvA                                    X     1   323    13   191    10   185   168   32.   0.18E-08 ------------AGAVGATVAQTLAIRQT-GD----IYIFDIVDGLAEGKALDI-LEGA--PHIKGFCTADESKAEMKGSDVIVVTAGLARKPGMSRDDLLLKNIGIMKSVGEAIKKYS-PESKIVVVTNPADI-MAYAIYKASGISPERIIGLGSLDSTRFRTFLAQELNVSFEDVNAFVI-GGHGDDMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   13 1a5zA                                    X     1   312    75   242    63   229   164   28.   0.82E-07 --------------------------------------------------------------------------ADLKGSDVVIVAAGVPQKPGETRLQLLGRNARVMKEIARNVSKYAPDSI-VIVVTNPVDV-LTYFFLKESGMDPRKFGSGTVLDTARLRTLIAQHCGFSPRSVH-VYVIGEHGDSEV-PVWSGAMIGGIPLQNMCQVCQDSKILENFAEKTKRAAYEIIERKGATHYA---------------------------------------------------------------------------------------------\r\n",
      "   14 4e0bA                                    X     1   305     4   314     2   288   282   28.   0.40E-04 ---AMKVAVIGAAGGIGQALALLLKNRLPAGSD---LALYDIAPVTPGVAADLSHIP--THVSIKGY-AGEDPTPALEGADVVLISA------GVARADLFNVNAGIVKSLAERIAV-VCPNACIGIITNPVNTTVPIALKKAGVYDKRKLFGVTTLDVIRSETFVAELKGQDPGEVRVPVIGGHSGVTILPLLSQV----EG-----VEFSDEEIAALT--KRIQNAGTEVVEAKGGSATLSMGQAACRFGLALVKALQGEEVIEYA--YVEGNG---EHASFFAQPVKLGKEGVEEILPYGELSDFEKAALD---------------------\r\n",
      "   15 4ln1A                                    X     1   317    78   270    79   278   183   24.   0.34E-02 -----------------------------------------------------------------------------KDADLVVITAGLP----QTRLDLVEKNAKIFKQIVRSIMDSGFDGI-FLIATNPVDILTYVTWKESGLPKERVIGSGTTLDSARFRYMLGEYFDIGPHNIHAYII-GEHGDT---ELPVWSHVSVG-IQKLQTLLEKDNTYDKIFINVRDAAYHIIERKGATSLLRVTKASVLTVSAYLEGQYGQKDVYIGVP-----------------------------------------------------------------\r\n",
      "   16 3wswA                                    X     1   316    78   270    70   273   187   25.   0.25E-04 -----------------------------------------------------------------------------QDANLVILTAGVNQKPGQTRLDLVKTNATITRQIVKEVMASGFDGIFV-VASNPVDILTYLTWQESGLPASRVVGTGTTLDTTRFRKEIALKLAVDPRSVHGYIL-GEHGDSEVAAWSHTTV--GGK--PIMEYVEKDHRLEVLADKVKNAAYEIIDRKKATKAILNNEQAVLPVSAYLNGEYGEEDIFTGVP-----------------------------------------------------------------\r\n",
      "   17 6itlA                                    X     1   312     7   209     3   201   196   27.   0.14E-05 ------VAVLGAAGGIGQALALLL---KLQLPAGSSLSLYDVAPV---TPGVAKDLSHIPTDVVVEGFAGTDPSEALKGADIVLISAGVARKPGMTRADLFGVNAGIIRSLTEKVAEQC-PKACVGIITNPVNAMVAIALKKAGVYDKRKLFGITTLDILRAETFIAELKGLDPTRVTIPVIGGHSGVTILPLLSQVQNVEWSSEEEII------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   18 6itkA                                    X     1   318     7   325     5   310   305   41.    0.0     ------VTVTGAAGQISYSLLWRIANGEVFGDTPVELKLLEIPQALGGAEGVAMELLDSAFPLLRNITITADANEAFDGANAAFLVGAKP-----ERADLLANNGKIFGPQGKAINDNAADDIRVLVVGNPANTNALIASAAAPDVPASRFNAMMRLDHNRAISQLATKLGRGSAEFNNIVVWGNHSATQFPDITYATVGGE----KVTDLVDHDWYVEEFIPRVANRGAEIIEVRGKSSAASAASSAIDHMRDWVQGT---EAWS-SAAIPSTGAYGIPEGIFVGLP-TVSRNGEWEIVEGLEISDFQRARIDANAQELQAERE----------\r\n",
      "   19 4cl3A                                    X     1   309    13   191    10   180   166   32.   0.23E-06 ------------AGFVGSTTAHWLAAKEL-GD----IVLLDIVEGVPQGKAL--DLYEAS--PIEGFVTGTNNYADTANSDVIVVTSGAPRKPGMSREDLIKVNADI---TRACISQAASPNAVIIMVNNPLDAMTYLAAEVSGFPKERVIGQAGVLDAARYRTFIAMEAGVSVEDVQAMLM-GGHGDEMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   20 3fi9A                                    X     1   321     9   260    12   248   231   28.   0.22E-10 --------IVGAAGMIGSNMAQTAAMMRLTPNLCLY------DPFAVGLEGVAEEIRHCGFEGL-NLTFTSDIKEAL--TDAKYIVSS-----GGTREDLLKGNAEIAAQLGKDIKSYCPDCKHVIIIFNPADITGLVTLIYS-GLKPSQVTTLAGLDSTRLQSELAKHFGIKQSLVTNTRTYGGHGEQM------AVFASTAKVNPLTDLIGTDKLTNEQWAELKQRGANIIKLRGRSSFQSPSYVSIEMIRAAMGGEA---------------------------------------------------------------------------\r\n",
      "   21 5mdhA                                    X     1   333     2   332     1   331   328   44.    0.0     -SEPIRVLVTGAAGQIAYSLLYSIGNGSVFGDQPIILVLLDITPMMGVLDGVLMELQDCALPLLKDVIATDKEEIAFKDLDVAILVGSMPRRDGMERKDLLKANVKIFKCQGAALDKYAKKSVKVIVVGNPANTNCLTASKSAPSIPKENFSCLTRLDHNRAKAQIALKLGVTSDDVKNVIIWGNHSSTQYPDVNHAKVKLQAKEVGVYEVKDDSWLKGEFITTVQQRGAAVIKARKLSSAMSAAKAICDHVRDIWFGTPEGEFVSMGI-ISDGNSYGVPDDLLYSFPVTIDKTWKI--VEGLPINDFSREKMDLTAKELAEEKETAFEFLS---\r\n",
      "   22 7mdhA                                    X     1   351     6   334    14   339   325   34.    0.0     -----NIAVSGAAGMISNHLLFKLASGEVFGDQPIALKLLGSERSFQALEGVAMELEDSLYPLLREVSIGIDPYEVFEDVDWALLIGAKPRGPGMERAALLDINGQIFADQGKALNAVASKNVKVLVVGNPCNTNALICLKNAPDIPAKNFHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDFLNAKI--DGRPVKEV-IKRTKWLEEEFTITVQKRGGALIQKWGRSSAASTAVSIADAIKSLVTPTPEGDWFSTGV-YTTGNPYGIAEDIVFSMPCRSKGDGDYELATDVSNDDFLWERIKKSEAELLAEKKCVAHLTGEG-\r\n",
      "   23 5kkcB                                    X     1   320   101   301    98   293   191   26.   0.34E-02 ----------------------------------------------------------------------------------------------------VQRNVNIFKFIIPNVVKYS-PHCKLLVVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHALSCHGWIL-GEHGDSSVSGMNVAGVSLKTLHPELGTDADKEQ-WKQVHKQVVDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKEDVFLSVPCVLGQNGISDVVK----------------------------------\r\n",
      "   24 5zi2A                                    X     1   339    87   195    76   197   108   32.   0.21E-03 --------------------------------------------------------------------------------------AGVPRKPGLTRDDLFKMNAGIVKSLVTAVGKFA-PNARILVISNPVNSLVPIAVETLKKMKPGNVMGVTNLDLVRAYLMLKNKIGQDKTTMHRTVIGGHSGETIIPIIT--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   25 1bdmA                                    X     1   318     1   325     1   310   309   45.    0.0     MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGDQPVILQLLEIPQAMKALEGVVMELEDCAFPLLAGLEATDDPDVAFKDADYALLVGAAP---------RLQVNGKIFTEQGRALAEVAKKDVKVLVVGNPANTNALIAYKNAPGLNPRNFTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSIMFPDL----FHAEVDGRPALELVDMEWYEKVFIPTVAQRGAAIIQARGASSAASAANAAIEHIRDWALGTPEGDWVSMA--VPSQGEYGIPEGIVYSFPVTA-KDGAYRVVEGLEINEFARKRMEITAQELLDEME----------\r\n",
      "   26 4h7pA                                    X     1   313     4   324     3   308   304   41.    0.0     ---AVKVAVTGAAGQIGYALVPLIARGALLGTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKVVVTADPRVAFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLKSAQKLNPRHVTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVIGTTPAREAIKD----------FVQVVRGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDE-NPYGVPSGLIFSFPCT------CHAGEWTVVSGKLKQRLASTIAELQEER-----------\r\n",
      "   27 4h7pB                                    X     1   306     4   324     2   301   298   42.    0.0     ---AVKVAVTGAAGQIGYALVPLIARGALLGTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKVVVTADPRVAFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLKSAQKLNPRHVTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVI---GTTPDFVQV-------------VRGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDE-NPYGVPSGLIFSFPCT------CHAGEWTVVSGKLKQRLASTIAELQEER-----------\r\n",
      "   28 3hhpA                                    X     1   312     7   320     3   303   289   27.   0.11E-04 ------VAVLGAAGGIGQALALLLKTQLPSGSE---LSLYDIAPVTPGVAVDLSHIPTAV--KIKGF-SGEDATPALEGADVVLISAGVARKPGMDRSDLFNVNAGIVKNLVQQVAKTC-PKACIGIITNPVNTTVAIAVLKKAGVYDKNLFGVTTLDIIRSNTFVAELKGKQPGEVEVPVIGGHSGVTILPLLSQVSFTE----QEVADLT----------KRIQNAGTEVVEAKGGGSATLSMGQAAARFGLSLVRALQGEQGVVECAYVEGDGQYAR---FFSQPLLLGKNGSIGTLSAFEQNA-LEGMLDTLKKDI---------------\r\n",
      "   29 4tvoA                                    X     1   328     7   324     6   318   312   40.    0.0     ------VAVTGAAGQIGYSLLFRLASGSLLGDRPIELRLLEIEPALQALEGVVMELDDCAFPLLSGVEIGSDPQKIFDGVSLALLVGARPRGAGMERSDLLEANGAIFTAQGKALNAVAADDVRVGVTGNPANTNALIAMTNAPDIPRERFSALTRLDHNRAISQLAAKTGAAVTDIKKMTIWGNHSATQYPDLFHAEVA--GKNAAEV-VNDQAWIEDEFIPTVAKRGAAIIDARGASSAASAASATIDAARDWLLGTPADDWVSMAV-VSDGS-YGVPEGLISSFPVTT-KGGNWTIVSGLEIDEFSRGRIDKSTAELADER-----------\r\n",
      "   30 4tvoB                                    X     1   317     7   324     5   308   303   40.    0.0     ------VAVTGAAGQIGYSLLFRLASGSLLGDRPIELRLLEIEPALQALEGVVMELDDCAFPLLSGVEIGSDPQKIFDGVSLALLVGARP---------LLEANGAIFTAQGKALNAVAADDVRVGVTGNPANTNALIAMTNAPDIPRERFSALTRLDHNRAISQLAAKTGAAVTDIKKMTIWGNHSATQYPDLFHAEVA--GKNAAEV-VNDQAWIEDEFIPTVAKRGAAIIDARGASSAASAASATIDAARDWLLGTPADDWVSMAV-VSDGS-YGVPEGLISSFPVTT-KGGNWTIVSGLEIDEFSRGRIDKSTAELADER-----------\r\n",
      "   31 2hjrA                                    X     1   313    13   333    10   309   288   26.   0.31E-05 ------------AGQIGSTIALLLGQKDL-GD----VYMFDIIEGVPQGKAL--DLNHCALIGSPAKIFGENNYEYLQNSDVVIITAGVPRKPNMTRSDLLTVNAKIVGSVAENVGKYC-PNAFVICITNP--LDAMVYYFKEKSGIPANCGMSGVLDSARFRCNLSRALGVKPSDVSAIVV-GGHGDEMIPLTSSVTFVEQGK-------ITHSQI-NEIIKKTAFGGGEIVELLKTGSAYAPAASAVAMAQAYLKDSKSVLVCSTYLT----GQYNVN-NLFVGVPVVIGKNG---------IEDVVIVNLSDDEKSLFSKSVESIQNLVQ--\r\n",
      "   32 1t2dA                                    X     1   315     5   256     4   250   238   25.   0.55E-03 ----AKIVLVG-SGMIGGVMATLIVQKNL-GDVVLFDIVKNMPKALDTSHTNVMAYSNC---KVSGSNTYDDLAGA----DVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHIKKNC-PNAFIIVVTNP--VDVMVQLLHQHSGVPKNIGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMVLLKRYITVGGIPLQEFINNKLISDAELEAIFDRTVNTALEIVNLHA-SPYVAPAAAIIEMAESYL-------------------------------------------------------------------------------\r\n",
      "   33 4plgA                                    X     1   324     1   256     1   253   236   28.   0.66E-07 MTKRKKISLIG-SGMIGGTMAYLCAQKEL-GD----VVLFDVVKNMPQGKALDLS-HSSSIADTNVKVTGTNSYEDIKGSDVVIITAGLTKAPGKSRDDLLPFNAKIMREVGENIKKYC-PNAFVIVITNP--LDVMVKVLHEHSGLPKNCGMAGVLDSSRFRHFIAEKLNVSPRDVQAMVI-GAHGDKMVPLTRYVTFIKKGRTQEEIDEI---------VERTKNAGGEIVNLLGQGSAFAPAASAIEMAEAYL-------------------------------------------------------------------------------\r\n",
      "   34 4pltB                                    X     1   319    13   271    11   280   240   31.   0.37E-10 ------------AGNIGGTLAHLIAQKEL-GD----VVLFDIVEGMPQGKALDISHSS---PIMGSNVKITGTN-NYEDIDVVIITAGIPRKPGKSRDDLLSVNAKIMKDVAENIKKYC-PNAFVIVVTNP--LDVMVYVLHKYSGLPHNCGMAGVLDSSRFRYFLAEKLNVSPNDVQAMVI-GGHGDTMVGGIPLTEFIKQGITQEEID----EIVERT--RNAGGEIVNLLKTGSFAPAASAIEMAESYLKAYLEGQYGVKDLFVGVPV----------------------------------------------------------------\r\n",
      "   35 4plvA                                    X     1   305    13   271    10   265   231   31.   0.61E-07 ------------AGNIGGTLAHLIAQKEL-GD----VVLFDIVEGMPQGKALDISHSS---PIMGSNVKITGTN-NYEDIDVVIITAGI---------DLLSVNAKIMKDVAENIKKYC-PNAFVIVVTNP--LDVMVYVLHKYSGLPHNCGMAGVLDSSRFRYFLAEKLNVSPNDVQAMVI-GGHGDTMVGGIPLTEFIKQGITQEEID----EIVERT--RNAGGEIVNLLKTGSFAPAASAIEMAESYLKAYLEGQYGVKDLFVGVPV----------------------------------------------------------------\r\n",
      "   36 4plzA                                    X     1   302    97   256    80   236   155   26.   0.32E-02 ------------------------------------------------------------------------------------------------RDDLLPLNNKIMIEIGGHIKKNC-PNAFIIVVTNP--VDVMVQLLHQHSGVPKNIGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMVLLKRYITVGGIPLQEFINNKLISDAELEAIFDRTVNTALEIVNLHA-SPYVAPAAAIIEMAESYL-------------------------------------------------------------------------------\r\n",
      "   37 3d0oA                                    X     1   309    79   296    71   295   212   24.   0.33E-02 ------------------------------------------------------------------------------DADLVVICAGAAQKPGETRLDLVSKNLKIFKSVGEVMA--SKFDGIFLVATNPVDILAYATWKFSGLPKERVIGSGTILDSARFRLLLSEAFDVAPRSV-DAQIIGEHGDTELPVWSHANIAGQPLKTLLEQRPEGKAQIEQIFVQTRDAAYDIIQAKGITEAIFRNEDAVLTVSALLEGEYEEEDVYIGVPAVI-NRNGIRN--VVEIPLNDEEQSK---------------------------------------\r\n",
      "   38 1y6jA                                    X     1   289    77   191    58   167   109   33.   0.29E-04 ----------------------------------------------------------------------------YSDVDCDVIVVT----AGATRLDLAKKNVMIAKEVTQNIMKYYNHGV-ILVVSNPVDIITYMIQKWSGLPVGKVIGSGTVLDSIRFRYLLSEKLGVDVKNVHGYII-GEHGDSQL------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   39 1ez4B                                    X     1   318    69   239    55   230   159   31.   0.31E-05 --------------------------------------------------------------------AFTAPKKIYKDADLVVITAGAPQKPGESRLDLVNKNLNI-------LSSIVKPVVDFLVAANPVDILTYATWKFSGFPKERVIGSGTSLDSSRLRVALGKQFNVDPRSV-DAYIMGEHGDSEFAAYSTATI----GTRPVRDVAKEQGVSDDDLAKVRNKAYDIINLKGAT------------------------------------------------------------------------------------------------\r\n",
      "   40 5hs4A                                    X     1   301     8   191     4   172   167   27.   0.19E-02 -------IVLVGSGMIGGVMATLIVQKNL-GDVVMFDVVKNMPKALDTSHSNVMAYSNC---KVTGSNSYDDLKGA----DVVIVTAGF-------RDDLLPLNNKIMIEIGGHIKNLC-PNAFIIVVTNPVDVMVQLLFEHSGVPKNKIIGLGGVLDTSRLKYYISQKLNVCPRDVNALIV-GAHGNKMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   41 5nueA                                    X     1   331     3   332     3   331   327   38.    0.0     --EPVRVLVTGAAGQIGYALVPMIARGIMLGDQPVILHMLDIPPAAEALNGVKMELIDAAFPLLKGVVATTDAVEGCTGVNVAVMVGGFPRKEGMERKDVMSKNVSIYKSQAAALEKHAAPNCKVLVVANPANTNALILKEFAPSIPEKNISCLTRLDHNRALGQISERLSVPVSDVKNVIIWGNHSSSQYPDVNHAKVQTSSGEKPVRELVKDDAWLDEFISTVQQRGAAIIKARKLSSALSAASSACDHIRDWVLGTPEGTFVSMGV-YSDGS-YSVPSGLIYSFPVTC-RNGDWSIVQGLPIDEVSRKKMDLTAEELKEEKDLAYSCLS---\r\n",
      "   42 1mldA                                    X     1   313     5   198     1   189   183   26.   0.52E-05 ----AKVAVLGASGGIGQPLSLLLKNSPLVSRLTLYAHTPGVAADLSHITRATVK----------GYLGPEQLPDCLKGCDVVVIPAGVPRKPGMTRDDLFNTNATIVATLTAACAQHC-PDAMICIISNPVNSTIPIAEVFKKHYNPNKIFGVTTLDIVRANAFVAELKGLDPARVSVPVIGGHAGKTIIPLISQCT-----------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   43 6j9sB                                    X     1   315    71   239    59   233   157   27.   0.15E-03 ----------------------------------------------------------------------TSPKKIYKDADLVVITAGAPQKPGETRLDLVNKNLKILKSIVDPIVDSGFNGI-FLVAANPVDILTYATWKLSGFPKNRVVGSGTSLDTARFRQSIAKMVNVDARSVHAYIM-GEHGDANIGGVTIAEWVKEIKEDKLVKM----------FEDVRNKAYEIIKLKGAT------------------------------------------------------------------------------------------------\r\n",
      "   44 6iheA                                    X     1   303    80   225    70   206   137   30.   0.15E-02 -------------------------------------------------------------------------------MDIVVISAGKPRKPGMSRRDLFVDNAKIMIDLAQKLPS-KNPGAIYLMVANPVDMMASVFMKYSKQF---TISAGDQVETMRMRSFIAKKLKIPVTSV-DGFVGGEHGEDAV--VLWSTVKIKGKPVDEFNI-NKDEVSD-YVKKI--------------------------------------------------------------------------------------------------------------\r\n",
      "   45 6r8gA                                    X     1   311    13   257     8   244   232   26.   0.14E-05 ------------SGQIGAIVGE-LCLLENLGD----LILYDVVPGIPQGKALDLK----HFSTILGVILGTNQIEDIKDADIIVITAGVQRKEGMTREDLIGVNGKIMKSVAESVKLHCSKAF-VICVSNP--LDIMVNVFHKSNLPHEKICGMAILDTSRYCSLIADKLKVSAEDVNAVIL-GGHGDLMVPLQRYTSVNGVPLSEFVKKNMISQNEIQEIIQKTRNMGAEIIKLAKASAAFAPAAAITKMIKSYLY------------------------------------------------------------------------------\r\n",
      "   46 2v65A                                    X     1   326    94   300    97   298   197   24.   0.21E-02 ---------------------------------------------------------------------------------------------GESRLNLVQRNVNIFKFIIPNIVKYS-PNCILMVVSNPVDILTYVAWKLSGFPRHRVIGSGTNLDSARFRHLIGEKLHLHPSSCHAWIV-GEHGDSSVSGVNVAGVSLQGLNPQMGTEGDGEN-WKAIHKEVVDGAYEVIKLKGYTSWAIGMSVAVESIIKNMHKVHPVSTLVQGM-------HGVKDEVFLSVPCVLGNSGLTDVI-----------------------------------\r\n",
      "   47 1v6aA                                    X     1   332    94   300   103   304   197   25.   0.99E-03 ---------------------------------------------------------------------------------------------GESRLNLVQRNVNIFKFIIPNIIKYS-PNCILLVVSNPVDILTYVAWKLSGLPRNRVIGSGTNLDSARFRHLMGEKLGIHPSNCHGWVI-GEHGDSSVSGVNVAGVFLQGLNPDMGTDKDKE-DWKSVHKMVVDSAYEVIKLKGYTSWAIGMSAACQSILKNLRKCHPVSTLVKGM-------HGVNEEVFLSVPCILGNSGLTDVV-----------------------------------\r\n",
      "   48 6balA                                    X     1   312     4   295     1   274   269   26.   0.14E-05 ---AMKVAVLGAAGGIGQALALLL---KLQLPAGTDLSLYDIAPVTPGVAVDVSHIPTAV--NVKGF-SGEDPTPALEGADVVLISAGVARKPGMDRSDLFNINAGIVRGLIEKVAVTC-PKACVGIITNPVNTTVAIALKKAGVYDKRKLFGVTTLDVLRSETFVAELKGLNVSRTSVPVIGGHSGVTILPLLSQVQYAKWNEDE-----------IEPLTKRIQNAGTEVLNAKGGSATLSMAQAAARFARSLVKGLSGETVVECTYVEGDG-----KYARFFSQPVRLGKEG----------------------------------------\r\n",
      "   49 4q3nA                                    X     1   314    78   320    70   309   232   23.   0.19E-04 -----------------------------------------------------------------------------KDAGLVVITAGVNQKEGETRIHLLKRNAEIIKTVVKNIMSSGFNGI-LLVASNPVDVLAYVAWKESGLHHSRVIGSGTSLDTARLRYEISQYIHIDSRNVHAYIL-GEHGDSEFVCWSNANV----GAKPIADVIDSDEILEHIYLKVRDAAYEIISRKKATYYG--IGMALVRITAAIFN---NENRILPISVLNNGVYDCEDQVYIGLPAVLNKDGVHHIVKNEKENSQLNNSANILRKNL---------------\r\n",
      "   50 4uulA                                    X     1   334     2   333     1   332   332  100.    0.0     -SEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKAAFKDIDCAFLVASMPRKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPENFSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKIGHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVVEGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQ--\r\n",
      "   51 4uupA                                    X     1   335     3   333     1   331   331   60.    0.0     --EPIHVLITGAAGQIGYALAFRIAKGDLFGDRKVVLHLLEIPPAMKALEGVCMELQDCAFPTLAGVVATDDPEEAFKDVDVAFLVGSFPRKPGMERADLLEKNAGIFKVQGKALSEYAKPTVKVLVVGNPANTNCLIAMANAPKLGPENFSAMTRLDHNRAIGEIAAKLGVPVDKVHNVVVWGNHSNTQVPDVSHATVDKEGGTKKVSDALPKEYLEGEFVQKIAQRGGAVIEARGASSAASAANAALCHMRDWLFGTKPGDWVSMGIPVPEGNPYGIKPGVIYSFPCTVDKDGKVHIVEGLEINDWVREKMEATEKELIEERETAFKVLAQ--\r\n",
      "   52 4uuoA                                    X     1   328     3   327     1   318   318   50.    0.0     --EPIHILITGAAGQIGYALTFRIAKGDLCGDRKVVLHLLEIPFGMKALEGCVMELQDCAFPNVAGIVWTDKVEEAFKGVDVAFLVGSFPRKDGMDRSDLLAKNGGIFTVQGKALNDYAKPTVKVLVVGNPANTNCLIAQASAPKLQNKNWCAMTRLDHNRMVGALAAKFGVTPEKIHKVCIWGNHSNTQVPDTTHATV----DTVKVADKLPKEYLEGEFAQMIATRGGAVIKMRGASSAASAANAALTCVKDWLYGT---DFVSMAIPVPDNEPYGIKQGTIFSFPVTVSKDGEVHVVEGLELNDWVKGRLEATEKELIGEKETA--------\r\n",
      "   53 2a92A                                    X     1   316     8   191     6   186   174   26.   0.15E-03 -------IVLVGSGMIGGVMATLIVQKNL-GDVVMFDVVKNMPKALDTSHSNVMAYSNC---KVTGSNSYDDLKGA----DVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHIKNLC-PNAFIIVVTNPVDVMVQLLFEHSGVPKNKIIGLGGVLDTSRLKYYISQKLNVCPRDVNALIV-GAHGNKMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   54 6bb1E                                    X     1   310    97   295    94   277   182   27.   0.70E-03 ------------------------------------------------------------------------------------------------RLNLVQRNVNIFKFIIPNVVKYS-PNCKLLIVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVP--VWSGMNVAGVSLKTL------HPDKEVHKQVVESAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPVSTMIKGL-------YGIKDDVFLSVPCILGQNG----------------------------------------\r\n",
      "   55 1oc4A                                    X     1   315     5   191     4   186   174   28.   0.15E-03 ----AKIVLVG-SGMIGGVMATLIVQKNL-GDVVMFDIVKNMPKALDTSHTNVMAYSNCK-------VSGSNTYDDLKDADVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHI-KNNCPNAFIIVVTNP--VDVMVQLLHQHSGVPKNVGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   56 1smkA                                    X     1   313     7   202     4   198   188   34.   0.75E-11 ------VAILGAAGGIGQPLAMLMKMNPLVS----VLHLYDVVNAPG-VTADISHMDTGAV--VRGFLGQQQLEAALTGMDLIIVPAGVPRKPGMTRDDLFKINAGIVKTLCEGIAKCC-PRAIVNLISNPVNSTVPIAFKKAGTYDPKRLLGVTMLDVVRANTFVAEVLGLDPRDVDVPVVGGHAGVTILPLLSQSSFTQE-------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   57 6vdhA                                    X     1   308    13   191    12   176   159   33.   0.14E-04 ------------SGMIGGTMAYLCALKEL-GD----VVLFDVVKNMPQGKALDLS-------HSTS-VADTNVKVSYEDIKGSDVVI---ITAGLSRDDLLPINAKIMKEVGENIKKYC-PNAFVICITNP--LDVMVKVLQEASGLPHNCGMAGVLDSSRFRYFIAEKLNVSPRDVQAMVI-GAHGDNMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   58 4jcoA                                    X     1   303     7   320     3   287   274   26.   0.39E-04 ------VSVVGAAGTVGAAAGYNIALRDI-ADEVVF---VDIPDKEDDTVGQAADTN-----HGIAYDSNTRVRGGYEDTDVVVITAGIPRQPGQTRIDLAGDNAPIMEDIQSSLDEHNDDYISLTTSNPVDLLNRHLYEAGDRS-REQVIGFGGRLDSARFRYVLSEEFDAPVQNVEGTIL-GEHGDAQVVDGTDPEFSGDEKEQLLGDLQE--------------SAMDVIERKGATE-WGPARGVAHMVEAILHDT--GEVLPASVKL-EGE-FG-HEDTAFGVPVRLGSNGVEEIVE------W---DLDDYEQDL---------------\r\n",
      "   59 1lldA                                    X     1   313    13   242     9   233   216   31.   0.30E-06 ------------AGAVGSTLA--FAAAQRGIAREIVLE--DI--AKERVEAEVLDMQGSSFYPTVSIDGSDDPEIC-RDADMVVITAGPRQKPGQSRLELVGATVNILKAIMPNLVKVA-PNAIYMLITNP----VDIATHVAQKLTPENFGSGTNLDSARLRFLIAQQTGVNVKNVHAYIA-GEHGDSEVPLWESATIMSDWTPLPGHDPLDAD-KREEIHQEVKNAAYKIINGKGATNYA---------------------------------------------------------------------------------------------\r\n",
      "   60 1hyeA                                    X     1   307     7   191     3   183   173   29.   0.13E-06 ------VTIIGASGRVGSATALLLAKEPFMKD----LVLIGREHSINKLEGLREDIYDA----LAGYVESDENLRIIDESDVVIITSGVPRKEGMSRMDLAKTNA---KIVGKYAKKIAECDTKIFVITNPVDVMTYKALVDSKFERNQVFGLGTHLDSLRFKVAIAKFFGVHIDEVRTRII-GEHGDSMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   61 4wleA                                    X     1   314     5   198     2   190   186   26.   0.24E-05 ----AKVAVLGASGGIGQPLSLLLKNSPLVSRLTLY-DIAHTPGVAADLSHIETKAA------VKGYLGPEQLPDCLKGCDVVVIPAGVPRKPGMTRDDLFNTNATIVATLTAACAQHC-PEAMICVIANPVNSTIPIAEVFKKHYNPNKIFGVTTLDIVRANTFVAELKGLDPARVNVPVIGGHAGKTIIPLISQCT-----------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   62 3tl2A                                    X     1   312    39   191    34   185   149   28.   0.50E-06 --------------------------------------LVDIPQLENPTKGKALDMLEAS--PVQGFDANTSDYADTADSDVVVITAGIARKPGMSRDDLVATNSKIMKSITRDIAKHS-PNAIIVVLTNPVDAMTYSVFKEAGFPKERVIGQSGVLDTARFRTFIAQELNLSVKDITGFVL-GGHGDDMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   63 3p7mA                                    X     1   318    13   207    11   205   185   31.   0.84E-07 ------------AGNIGGTLAH-LALIKQLGD----VVLFDIAQGMPNGKALDL-LQTCPIEGVDFKVRGTNDYKDLENSDVVIVTAGVPRKPGMSRDDLLGINIKVMQTVGEGI-KHNCPNAFVICITNP--LDIMVNMLQKFSGVPDNVGMAGVLDSARFRTFLADELNVSVQQV-QAYVMGGHGDTMVAGVSLEQLVKEGKLKQ--------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   64 4nd2A                                    X     1   317     1   301     1   288   270   26.   0.11E-06 MIERRKIAVIG-SGQIGGNIAYIVGK-----DNLADVVLFDIAEGIPQGKALDITHSMVMFGSTSKVIGTND-YADISGSDVVIITASIPGRPKDDRSELLFGNARILDSVAEGVKKYC-PNAFVICITNP----LDVMVSHFQKVSPHNCGMAGVLDSSRFRTFIAQHFGVNASDVSANVI-GGHGDGMVGGVPLSSFIKQGLTQEQIDIVCHTRI-----------AWKEVADNLKTGTAAPAAAAVKMAEAYL------KDKKAVVPCSASNHYGVK-GIYMGVPTIIGKNGVEDILE----------------------------------\r\n",
      "   65 4nd5A                                    X     1   302     1   301     1   276   258   26.   0.32E-02 MIERRKIAVIG-SGQIGGNIAYIVGK-----DNLADVVLFDIAEGIPQGKALDITHSMVMFGSTSKVIGTND----YADISGSDVV---------IITALLFGNARILDSVAEGVKKYC-PNAFVICITNP----LDVMVSHFQKVSPHNCGMAGVLDSSRFRTFIAQHFGVNASDVSANVI-GGHGDGMVGGVPLSSFIKQGLTQEQIDIVCHTRI-----------AWKEVADNLKTGTAAPAAAAVKMAEAYL------KDKKAVVPCSASNHYGVK-GIYMGVPTIIGKNGVEDILE----------------------------------\r\n",
      "   66 5w8jA                                    X     1   329    85   295    91   296   196   26.   0.73E-04 ------------------------------------------------------------------------------------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PNCKLLIVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVAGVSLKTLHPDLGTDK-----DKEQ-WKEVHKQVVESAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPVSTMIKGL-------YGIKDDVFLSVPCILGQNG----------------------------------------\r\n",
      "   67 1b8pA                                    X     1   327     7   331     6   325   316   42.    0.0     ------VAVTGAAGQICYSLLFRIANGDMLGDQPVILQLLEIPKAQKALQGVMMEIDDCAFPLLAGMTAHADPMTAFKDADVALLVGARPRGPGMERKDLLEANAQIFTVQGKAIDAVASRNIKVLVVGNPANTNAYIAMKSAPSLPAKNFTAMLRLDHNRALSQIAAKTGKPVSSIEKLFVWGNHSPTMYADYRYAQI--DGASVK--DMINDDWNRDTFLPTVGKRGAAIIDARGVSSAASAANAAIDHIHDWVLGTA-GKWTTMGI--PSDGSYGIPEGVIFGFPVTTE-NGEYKIVQGLSIDAFSQERINVTLNELLEEQN-GVQHL----\r\n",
      "   68 2ldxA                                    X     1   331    66   306    67   306   227   26.   0.21E-03 -----------------------------------------------------------------GSLFLSTPKIVFKDYNVSIITAGARMVSGQTRLDLLQRNVAIMKAIVPGVIQ-NSPDCKIIVVTNPVDILTYVVWKISGFPVGRVIGSGCNLDSARFRYLIGEKLGVNPTSCHGWVL-GEHGDSSVSGVNVAGVTLKSLNPAIGTDKNKQH-WKNVHKQVVEGGYEVLDMKGYTSWASVTDLARSILKN-LKRVHPVTTLVKGF-------HGIKEEVFLSIPCVLGESG---ITDFVKVN-----------------------------\r\n",
      "   69 9ldtA                                    X     1   331    85   301    93   304   207   26.   0.93E-05 ------------------------------------------------------------------------------------ITAGARQQEGESRLNLVQRNVNIFKFIIPNIVKYS-PNCKLLVVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHPLSCHGWIL-GEHGDSSVSGVNVAGVSLKNLHPELGTDADKEH-WKAVHKEVVDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKENVFLSVPCILGQNGISDVVK----------------------------------\r\n",
      "   70 1ldnA                                    X     1   316    78   298    73   301   214   26.   0.16E-02 -----------------------------------------------------------------------------RDADLVVICAGANQKPGETRLDLVDKNIAIFRSIVESVMASGFQGL-FLVATNPVDI-LTYATWKFSGLPHERIGSGTILDTARFRFLLGEYFSVAPQNVHAYII-GEHGDTELPVWSQA-YIGVMPIRKLVESKGEEAQKDRIFVNVRDAAYQIIEKKGATGLARVTRAAILTVSAYLDGLYGERDVYIGVPAVI-NRNGIRE--VIEIELNDDEKNRFH-------------------------------------\r\n",
      "   71 5ldhA                                    X     1   333    85   300    94   304   207   26.   0.26E-04 ------------------------------------------------------------------------------------VTAGVRQQEGESRLNLVQRNVNVFKFIIPQIVKYS-PNCIIIVVSNPVDILTYVAWKLSGLPKHRVIGSGCNLDSARFRYLMGEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVVLQQLNPEMTDNDSEWKEVHKMVVESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI-----------------------------------\r\n",
      "   72 6cepA                                    X     1   331    85   300    94   304   207   25.   0.34E-04 ------------------------------------------------------------------------------------VTAGVRQQEGESRLNLVQRNVNVFKFIIPQIVKYS-PDCIIIVVSNPVDILTYVTWKLSGLPKHRVIGSGCNLDSARFRYLMAEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVSLQELNPEMTDNDSEWKEVHKMVVESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI-----------------------------------\r\n",
      "   73 6cepD                                    X     1   317    96   300    90   289   196   26.   0.43E-03 -----------------------------------------------------------------------------------------------VTAGVVQRNVNVFKFIIPQIVKYS-PDCIIIVVSNPVDILTYVTWKLSGLPKHRVIGSGCNLDSARFRYLMAEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVSLQELNPEMTDNDSEWKEVHKMVVESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI-----------------------------------\r\n",
      "   74 1civA                                    X     1   374     6   334    33   358   325   35.    0.0     -----NVAVSGAAGMISNHLLFKLASGEVFGDQPISLKLLGSERSFAALEGVAMELEDSLYPLLRQVSIGIDPYEIFQDAEWALLIGAKPRGPGMERADLLDINGQIFAEQGKALNAVASPNVKVMVVGNPCNTNALICLKNAPNIPPKNFHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDFLNAKIHGIPVTEV---IRDRKWLEDEFTNMVQTRGGVLIKKWGRSSAASTAVSIVDAIRSLVTPTPEGDWFSTGV-YTNGNPYGIAEDIVFSMPCRSKGDGDYEFVKDVIFDDYLSKKIKKSEDELLAEKKCVAHLTGEG-\r\n",
      "   75 5nqbA                                    X     1   331    85   301    93   304   207   26.   0.57E-04 ------------------------------------------------------------------------------------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PHCKLLVVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHALSCHGWIL-GEHGDSSVSGMNVAGVSLKTLHPELGTDADKEQ-WKQVHKQVVDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMLKGL-------YGIKEDVFLSVPCVLGQNGISDVVK----------------------------------\r\n",
      "   76 6um4A                                    X     1   326     7   331     8   326   317   38.    0.0     ------ITVSGACGQIANSLLFRIANGEMLGENQVKLRLLERPEAMKALEGVKMELDDGAFPLLEEIYLTDSENDAFRGADYAILLGGKPRGPGMERADVMKDNAAIFKAQGEALNKQANGDVLVLIVANPANTNAMITSANAPDIPPENITAMTRLDHDRGLAQVAAKVGCNITDISRFAIWGNHSATQYPDLSFTTI----KGQWGLNVIDEQWITNEFIPNVQQRGAAIIKARGKSSAASAADAAIKHMHDWVLGNS--EWVSMAI--PSRGQYGIPRGIWCSMPVQCFGAGKYGVIEGLPINSFSADRINASVKELIEEKKIVENLL----\r\n",
      "   77 4m1qA                                    X     1   312    78   242    69   233   162   24.   0.32E-03 -----------------------------------------------------------------------------KSADIVVITAGAAQKPGETRLDLVEKNANIFKGIVDQVMGSGFNGI-FIIATNPVDVLAYATWKFSGLPKERVIGSGTILDTARFRFLLSEYFDIDVRNIHGYIM-GEHGDTELPVWSQTRIGSE-PISRYMDKYKPDKDLDEIFVNVRDAAYHIIERKGATHYA---------------------------------------------------------------------------------------------\r\n",
      "   78 1pzgA                                    X     1   327    74   191    71   190   114   30.   0.15E-05 -------------------------------------------------------------------------EAALTGADCVIVTAGLTKVPGKSRNDLLPFNSKIIREIGQNIKKYC-PKTFIIVVTNP--LDCMVKVMEASGVPTNMICGMAMLDSGRFRRYVADALSVSPRDVQATVI-GTHGDCMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   79 2x0jA                                    X     1   294    78   320    68   285   218   28.   0.38E-04 -----------------------------------------------------------------------------KGSEIIVVTAGLARKPGMTRLDLAHKNAGIIKDIAKKIVENA-PESKILVVTNPMDVMTYIMWKESGKPRNEVFGMGNQLDSQRLKERL---YNAGARNIRRAWIIGEHGDSMFVAKSLADFDGEVDWEAV----ENDVRF---------VAAEVIKRKGAT-IFGPAVAIYRMVKAVVEDT--GEIIPTSM-ILQGE-YGIE-NVAVGVPAKLGKNGA--EVADIKLSDEEIEKLRNSAKIL---------------\r\n",
      "   80 5ujkA                                    X     1   315    13   191    10   180   169   32.   0.40E-05 ------------AGQIGGTLAHLAGLKEL-GD----VVLFDIVDGVPQGKALDI-AESAPVDGFDAKYSGASDYSAIAGADVVIVTAGVPRKPGMSRDDLIGINLKVMEAVGAGIKEHA-PDAFVICITNP--LDAMVWALQKFSGLPTNVGMAGVLDSARFRHFLAEEFGVSVEDV-TAFVLGGHGDDMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   81 4ajjA                                    X     1   330    85   301    92   303   207   25.   0.34E-04 ------------------------------------------------------------------------------------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PQCKLLIVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVSGVNVAGVSLKSLNPQLGTDADKEQWKDV-HKQVVDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKEDVFLSVPCILGQNGISDVVK----------------------------------\r\n",
      "   82 3nepX                                    X     1   305    13   274     8   263   244   25.   0.91E-02 ------------AGNVGATVAECVARQDVAKE----VVMVDIKDGMPQGKALDMR-ESSPIHGFDTRVTGTNDYGPTEDSDVCIITAGL-------RDDLLAKNTEIVGGVTEQFVE-GSPDSTIIVVANPLDVMTYVA--YEASGFPTNMGMAGVLDTGRFRSFIAEELDVSVRDVQALLM-GGHGDTMVPLPRYTTVGGIPVPQLIDDARIEEIVERT--KGAGGEIVDLMGTSAWAAAAEMTEAILKDNKAYCDGEYGLDDLFIGVPVKLG-------------------------------------------------------------\r\n",
      "   83 6pblA                                    X     1   327     7   325     5   318   313   39.    0.0     ------VAVTGAAGQIGYALVFRIASGQMFGNTEVELNLLELEPALPSLEGVAMELDDCAFPLLKRIVCTADLNKAMDGVNWALLVGSVPRKQGMERSDLLQINGGIFTKQGQAINDYASDDVRVFVVGNPCNTNCLIAMNHAKDVPSDRFYAMTTLDELRARTQLAKKAGVDITAVTQMTIWGNHSATQYPDFYNA---KINGTSAAQVINDETWLKETFVSTVQQRGAAVIKARGSSSAASAANAIITGVNHLVTDTPAGESFSM-CRRSKGE-YGVDEGLIFSFPCR-REHGELKVVENLEFNDFGRERFNTTLNELRSERD----------\r\n",
      "   84 1y7tA                                    X     1   327     1   325     1   319   318   45.    0.0     MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGDQPVILQLLEIPQAMKALEGVVMELEDCAFPLLAGLEATDDPKVAFKDADYALLVGAAPRKAGMERRDLLQVNGKIFTEQGRALAEVAKKDVKVLVVGNPANTNALIAYKNAPGLNPRNFTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSTMFPDL----FHAEVDGRPALELVDMEWYEKVFIPTVAQRGAAIIQARGASSAASAANAAIEHIRDWALGTPEGDWVSMA--VPSQGEYGIPEGIVYSFPVTA-KDGAYRVVEGLEINEFARKRMEITAQELLDEME----------\r\n"
     ]
    }
   ],
   "source": [
    "#Six PDB sequences show very significant similarities to the \n",
    "#query sequence with e-values equal to 0\n",
    "!cat build_profile.prf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      ">P1;TvLDH\r\n",
      "sequence:TvLDH:    0: :    0: :::-1.00:-1.00\r\n",
      "MSEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKA\r\n",
      "AFKDIDCAFLVASMPLKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPEN\r\n",
      "FSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKI\r\n",
      "GHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVV\r\n",
      "EGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQGG*\r\n",
      "\r\n",
      ">P1;3d5tA\r\n",
      "structure:3d5tA:    3: :  316: :::-1.00:-1.00\r\n",
      "---AKRVAVTGAAGQIAYSLLFRIANGDLLGDQPVILQLLDLPQAQAAVKGVVMELDDCAFPLLAGVVITDDPKV\r\n",
      "AFKDADVALLVGARP--RSMERKDLLSANAEIFTVQGAALNEVASRDVKVLVVGNPANTNAYIAMKSAPDLPKKN\r\n",
      "FTAMLRLDHNRALSQLAAKSGKPVASIEKLAVWGNHSPTMYPDFRFA--TAEGESLKLIN--DDVWNRDTFIPTV\r\n",
      "GKRGAAIIEARGLSSAASAANAAIDHVRDWVLGT-NGKWVTMGI--PSDGSYGIPEDIIYGVPV-ICENGEYKRV\r\n",
      "EGLEIDAFSREKMDGTLAELLEERD----------*\r\n",
      "\r\n",
      ">P1;1sovA\r\n",
      "structure:1sovA:   76: :  248: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----DVVIITAGLTKVPS--RNDLLPFNAKIIREVAQGVKKYC-PLAFVIVVTNP--LDCMVKCFHEASGLPKN\r",
      "\r\n",
      "MGMANVLDSARFRRFIADQLEISPRDIQATVI-GTHGDHMLPLARYVTFIKKGKEAKLAEIVERT-------KKA\r\n",
      "G---GEIVRLLGQGSAYAPALSAITMAQAFL--------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6k12A\r\n",
      "structure:6k12A:   81: :  270: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "------------------GESRLNLVQRNVNIFKFIIPNVVKYS-PDCILLIVSNPVDILTYVAWKLSGFPLNRV\r\n",
      "IGSGCNLDSARFRYLVSEMIGIHPSNFHGCIL-GEHGDSSVPILSGLNIAGM-SIKNCKDV----------HKKV\r\n",
      "TESAYEIIKLKGYTS--------------WAIGLSVGDLLRKVHPVSTKGQFGIDNEVFLSVPCVLGRNG---IS\r\n",
      "EVFK-------------------------------*\r\n",
      "\r\n",
      ">P1;6k13A\r\n",
      "structure:6k13A:  103: :  305: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "------------------GESRLNLVQRNVNIFKFIIPNVVKYS-PDCILLIVSNPVDILTYVAWKLSGFPLNRV\r\n",
      "IGSGCNLDSARFRYLVSEMIGIHPSNFHGCIL-GEHGDSSVPILSGLNIA--GMSIKNLHT-DIDTVFKDVHKKV\r\n",
      "TESAYEIIKLKGYTS--------------WAIGLSVGDLLRKVHPVSTKGQFGIDNEVFLSVPCVLGRNG---IS\r\n",
      "EVFK-------------------------------*\r\n",
      "\r\n",
      ">P1;6or9A\r\n",
      "structure:6or9A:   37: :  292: :::-1.00:-1.00\r\n",
      "----------------------------------------------DKLKGELMDLQGSAFMRNVKIQASTDYSI\r\n",
      "SAGSKICV-VTAGVRQREGESRLDLVQRNTDVLKIIIPQLVKHS-PDTILIIASNPVDILTYVSWKLSGLPKHRV\r\n",
      "IGSGTNLDSARFRYLLSEKLGIATTSCHGYII-GEHGDSSVSGVNIAGVRLSDLNQKINWKETHTMVVKS-----\r\n",
      "---AYEVIKLKGYTSWAIGLSLS-QLARAILSNANSVHAVSTYLK----GEHDINDEVFLSLPCVLGRSG-----\r\n",
      "----VCDVIRQPLTQTERSQLHQ------------*\r\n",
      "\r\n",
      ">P1;1guzA\r\n",
      "structure:1guzA:    8: :  280: :::-1.00:-1.00\r\n",
      "------------AGNVGATTAFRLAEKQLARE----LVLLDVVEGIPQGKALDM--------YESGPVGLFDTKA\r\n",
      "DTANSDIVIITAGLPRKPGMTREDLLMKNAGIVKEVTDNIMKHSKNPI-IIVVSNPLDIMTHVAWVRSGLPKERV\r\n",
      "IGMAGVLDAARFRSFIAMELGVSMQDINACVL-GGHGDAMVPVVKYTTVA----GIPISDLLPAETI-DKLVERT\r\n",
      "RNGGAEIVEHLKQGSAYAPASSVVEMVESIVLDRKRVLPCAVGL---EGQ-YGIDKTFV-GVPVKLGRNGVEQIY\r\n",
      "E----------------------------------*\r\n",
      "\r\n",
      ">P1;3gviA\r\n",
      "structure:3gviA:   10: :  206: :::-1.00:-1.00\r\n",
      "------------SGMIGGTLAHLAGLKEL-GD----VVLFDIAEGTPQGKGLDIAESSPVDGFDAKFTGAND-YA\r\n",
      "AIEGADVVIVTAGVPRKD-----DLLGINLKVMEQVGAGIKKYA-PEAFVICITNP---------LDAMVWALQK\r\n",
      "FSGLGVLDSARFRYFLSEEFNVSVEDV-TVFVLGGHGDSMVPDLVKMGWTSQDKLDKII----------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1gv1B\r\n",
      "structure:1gv1B:   80: :  271: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------DLLMKNAGIVKEVTDNIMKHSKNPI-IIVVSNPLDIMTHVAWVRSGLPKERV\r\n",
      "IGMAGVLDAARFRSFIAMELGVSMQDINACVL-GGHGDAMVPVVKYTTVA----GIPISDLLPAETI-DKLVERT\r\n",
      "RNGGAEIVEHLKQGSAYAPASSVVEMVESIVLDRKRVLPCAVGL---EGQ-YGIDKTFV-GVPVKLGRNGVEQIY\r\n",
      "E----------------------------------*\r\n",
      "\r\n",
      ">P1;1gv0A\r\n",
      "structure:1gv0A:    8: :  289: :::-1.00:-1.00\r\n",
      "------------AGNVGATTAFRLAEKQLARE----LVLLDVVEGIPQGKALDM--------YESGPVGLFDTKA\r\n",
      "DTANSDIVVITAGLPRKPGMT----LSMNAGIVREVTGRIMEHSKNPI-IVVVSNPLDIMTHVAWQKSGLPKERV\r\n",
      "IGMAGVLDSARFRSFIAMELGVSMQDV-TACVLGGHGDAMVPVVKYTTVA----GIPVADLISAERIAE-LVERT\r\n",
      "RTGGAEIVNHLKQGSAYSPATSVVEMVESIVLDRK--RVLTCAVSL--DGQYGI-DGTFVGVPVKLGKNGVEHIY\r\n",
      "E---------IKLDQSDLDLLQK------------*\r\n",
      "\r\n",
      ">P1;4bguA\r\n",
      "structure:4bguA:    3: :  289: :::-1.00:-1.00\r\n",
      "------VSVIGAAGTVGAAAGYNLALRDVCDE----LVFVDIPKMEDKTVGQAADTNH-GIAYDSNTVVTQGGYE\r\n",
      "DTAGSDVVVITAGIPRQPGQTRIDLAGDNAPIMDDIGSSLAEYNDDFVSITTSNPVDLLNRHLYETGDRD-RHKV\r\n",
      "IGFGGRLDSARFRYVLSQRFDVPVKNV-DATILGEHGDAQV-----PVFSK-------VRVDGNDPAFSADEKEL\r\n",
      "QESAMDVIERKGATQWGPAT--GVAHMEAVLHDT--GEVLP-GSLVLDGE-FGYE-DTAFGVPVKLGSNGIEEVV\r\n",
      "E-WDLDDYEADLMD---------------------*\r\n",
      "\r\n",
      ">P1;4bgvA\r\n",
      "structure:4bgvA:   10: :  185: :::-1.00:-1.00\r\n",
      "------------AGAVGATVAQTLAIRQT-GD----IYIFDIVDGLAEGKALDI-LEGA--PHIKGFCTADESKA\r\n",
      "EMKGSDVIVVTAGLARKPGMSRDDLLLKNIGIMKSVGEAIKKYS-PESKIVVVTNPADI-MAYAIYKASGISPER\r\n",
      "IIGLGSLDSTRFRTFLAQELNVSFEDVNAFVI-GGHGDDMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1a5zA\r\n",
      "structure:1a5zA:   63: :  229: :::-1.00:-1.00\r\n",
      "--------------------------------------------------------------------------A\r\n",
      "DLKGSDVVIVAAGVPQKPGETRLQLLGRNARVMKEIARNVSKYAPDSI-VIVVTNPVDV-LTYFFLKESGMDPRK\r\n",
      "FGSGTVLDTARLRTLIAQHCGFSPRSVH-VYVIGEHGDSEV-PVWSGAMIGGIPLQNMCQVCQDSKILENFAEKT\r\n",
      "KRAAYEIIERKGATHYA----------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4e0bA\r\n",
      "structure:4e0bA:    2: :  288: :::-1.00:-1.00\r\n",
      "---AMKVAVIGAAGGIGQALALLLKNRLPAGSD---LALYDIAPVTPGVAADLSHIP--THVSIKGY-AGEDPTP\r\n",
      "ALEGADVVLISA------GVARADLFNVNAGIVKSLAERIAV-VCPNACIGIITNPVNTTVPIALKKAGVYDKRK\r\n",
      "LFGVTTLDVIRSETFVAELKGQDPGEVRVPVIGGHSGVTILPLLSQV----EG-----VEFSDEEIAALT--KRI\r\n",
      "QNAGTEVVEAKGGSATLSMGQAACRFGLALVKALQGEEVIEYA--YVEGNG---EHASFFAQPVKLGKEGVEEIL\r\n",
      "PYGELSDFEKAALD---------------------*\r\n",
      "\r\n",
      ">P1;4ln1A\r\n",
      "structure:4ln1A:   79: :  278: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--KDADLVVITAGLP----QTRLDLVEKNAKIFKQIVRSIMDSGFDGI-FLIATNPVDILTYVTWKESGLPKERV\r\n",
      "IGSGTTLDSARFRYMLGEYFDIGPHNIHAYII-GEHGDT---ELPVWSHVSVG-IQKLQTLLEKDNTYDKIFINV\r\n",
      "RDAAYHIIERKGATSLLRVTKASVLTVSAYLEGQYGQKDVYIGVP------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3wswA\r\n",
      "structure:3wswA:   70: :  273: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--QDANLVILTAGVNQKPGQTRLDLVKTNATITRQIVKEVMASGFDGIFV-VASNPVDILTYLTWQESGLPASRV\r\n",
      "VGTGTTLDTTRFRKEIALKLAVDPRSVHGYIL-GEHGDSEVAAWSHTTV--GGK--PIMEYVEKDHRLEVLADKV\r\n",
      "KNAAYEIIDRKKATKAILNNEQAVLPVSAYLNGEYGEEDIFTGVP------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6itlA\r\n",
      "structure:6itlA:    3: :  201: :::-1.00:-1.00\r\n",
      "------VAVLGAAGGIGQALALLL---KLQLPAGSSLSLYDVAPV---TPGVAKDLSHIPTDVVVEGFAGTDPSE\r\n",
      "ALKGADIVLISAGVARKPGMTRADLFGVNAGIIRSLTEKVAEQC-PKACVGIITNPVNAMVAIALKKAGVYDKRK\r\n",
      "LFGITTLDILRAETFIAELKGLDPTRVTIPVIGGHSGVTILPLLSQVQNVEWSSEEEII----------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6itkA\r\n",
      "structure:6itkA:    5: :  310: :::-1.00:-1.00\r\n",
      "------VTVTGAAGQISYSLLWRIANGEVFGDTPVELKLLEIPQALGGAEGVAMELLDSAFPLLRNITITADANE\r\n",
      "AFDGANAAFLVGAKP-----ERADLLANNGKIFGPQGKAINDNAADDIRVLVVGNPANTNALIASAAAPDVPASR\r\n",
      "FNAMMRLDHNRAISQLATKLGRGSAEFNNIVVWGNHSATQFPDITYATVGGE----KVTDLVDHDWYVEEFIPRV\r\n",
      "ANRGAEIIEVRGKSSAASAASSAIDHMRDWVQGT---EAWS-SAAIPSTGAYGIPEGIFVGLP-TVSRNGEWEIV\r\n",
      "EGLEISDFQRARIDANAQELQAERE----------*\r\n",
      "\r\n",
      ">P1;4cl3A\r\n",
      "structure:4cl3A:   10: :  180: :::-1.00:-1.00\r\n",
      "------------AGFVGSTTAHWLAAKEL-GD----IVLLDIVEGVPQGKAL--DLYEAS--PIEGFVTGTNNYA\r\n",
      "DTANSDVIVVTSGAPRKPGMSREDLIKVNADI---TRACISQAASPNAVIIMVNNPLDAMTYLAAEVSGFPKERV\r\n",
      "IGQAGVLDAARYRTFIAMEAGVSVEDVQAMLM-GGHGDEMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3fi9A\r\n",
      "structure:3fi9A:   12: :  248: :::-1.00:-1.00\r\n",
      "--------IVGAAGMIGSNMAQTAAMMRLTPNLCLY------DPFAVGLEGVAEEIRHCGFEGL-NLTFTSDIKE\r\n",
      "AL--TDAKYIVSS-----GGTREDLLKGNAEIAAQLGKDIKSYCPDCKHVIIIFNPADITGLVTLIYS-GLKPSQ\r\n",
      "VTTLAGLDSTRLQSELAKHFGIKQSLVTNTRTYGGHGEQM------AVFASTAKVNPLTDLIGTDKLTNEQWAEL\r\n",
      "KQRGANIIKLRGRSSFQSPSYVSIEMIRAAMGGEA----------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;5mdhA\r\n",
      "structure:5mdhA:    1: :  331: :::-1.00:-1.00\r\n",
      "-SEPIRVLVTGAAGQIAYSLLYSIGNGSVFGDQPIILVLLDITPMMGVLDGVLMELQDCALPLLKDVIATDKEEI\r\n",
      "AFKDLDVAILVGSMPRRDGMERKDLLKANVKIFKCQGAALDKYAKKSVKVIVVGNPANTNCLTASKSAPSIPKEN\r\n",
      "FSCLTRLDHNRAKAQIALKLGVTSDDVKNVIIWGNHSSTQYPDVNHAKVKLQAKEVGVYEVKDDSWLKGEFITTV\r\n",
      "QQRGAAVIKARKLSSAMSAAKAICDHVRDIWFGTPEGEFVSMGI-ISDGNSYGVPDDLLYSFPVTIDKTWKI--V\r\n",
      "EGLPINDFSREKMDLTAKELAEEKETAFEFLS---*\r\n",
      "\r\n",
      ">P1;7mdhA\r\n",
      "structure:7mdhA:   14: :  339: :::-1.00:-1.00\r\n",
      "-----NIAVSGAAGMISNHLLFKLASGEVFGDQPIALKLLGSERSFQALEGVAMELEDSLYPLLREVSIGIDPYE\r\n",
      "VFEDVDWALLIGAKPRGPGMERAALLDINGQIFADQGKALNAVASKNVKVLVVGNPCNTNALICLKNAPDIPAKN\r\n",
      "FHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDFLNAKI--DGRPVKEV-IKRTKWLEEEFTITV\r\n",
      "QKRGGALIQKWGRSSAASTAVSIADAIKSLVTPTPEGDWFSTGV-YTTGNPYGIAEDIVFSMPCRSKGDGDYELA\r\n",
      "TDVSNDDFLWERIKKSEAELLAEKKCVAHLTGEG-*\r\n",
      "\r\n",
      ">P1;5kkcB\r\n",
      "structure:5kkcB:   98: :  293: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-------------------------VQRNVNIFKFIIPNVVKYS-PHCKLLVVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHALSCHGWIL-GEHGDSSVSGMNVAGVSLKTLHPELGTDADKEQ-WKQVHKQV\r\n",
      "VDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKEDVFLSVPCVLGQNGISDVV\r\n",
      "K----------------------------------*\r\n",
      "\r\n",
      ">P1;5zi2A\r\n",
      "structure:5zi2A:   76: :  197: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------AGVPRKPGLTRDDLFKMNAGIVKSLVTAVGKFA-PNARILVISNPVNSLVPIAVETLKKMKPGN\r\n",
      "VMGVTNLDLVRAYLMLKNKIGQDKTTMHRTVIGGHSGETIIPIIT------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1bdmA\r\n",
      "structure:1bdmA:    1: :  310: :::-1.00:-1.00\r\n",
      "MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGDQPVILQLLEIPQAMKALEGVVMELEDCAFPLLAGLEATDDPDV\r\n",
      "AFKDADYALLVGAAP---------RLQVNGKIFTEQGRALAEVAKKDVKVLVVGNPANTNALIAYKNAPGLNPRN\r\n",
      "FTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSIMFPDL----FHAEVDGRPALELVDMEWYEKVFIPTV\r\n",
      "AQRGAAIIQARGASSAASAANAAIEHIRDWALGTPEGDWVSMA--VPSQGEYGIPEGIVYSFPVTA-KDGAYRVV\r\n",
      "EGLEINEFARKRMEITAQELLDEME----------*\r\n",
      "\r\n",
      ">P1;4h7pA\r\n",
      "structure:4h7pA:    3: :  308: :::-1.00:-1.00\r\n",
      "---AVKVAVTGAAGQIGYALVPLIARGALLGTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKVVVTADPRV\r\n",
      "AFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLKSAQKLNPRH\r\n",
      "VTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVIGTTPAREAIKD----------FVQVV\r\n",
      "RGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDE-NPYGVPSGLIFSFPCT------CHAG\r\n",
      "EWTVVSGKLKQRLASTIAELQEER-----------*\r\n",
      "\r\n",
      ">P1;4h7pB\r\n",
      "structure:4h7pB:    2: :  301: :::-1.00:-1.00\r\n",
      "---AVKVAVTGAAGQIGYALVPLIARGALLGTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKVVVTADPRV\r\n",
      "AFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLKSAQKLNPRH\r\n",
      "VTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVI---GTTPDFVQV-------------V\r\n",
      "RGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDE-NPYGVPSGLIFSFPCT------CHAG\r\n",
      "EWTVVSGKLKQRLASTIAELQEER-----------*\r\n",
      "\r\n",
      ">P1;3hhpA\r\n",
      "structure:3hhpA:    3: :  303: :::-1.00:-1.00\r\n",
      "------VAVLGAAGGIGQALALLLKTQLPSGSE---LSLYDIAPVTPGVAVDLSHIPTAV--KIKGF-SGEDATP\r\n",
      "ALEGADVVLISAGVARKPGMDRSDLFNVNAGIVKNLVQQVAKTC-PKACIGIITNPVNTTVAIAVLKKAGVYDKN\r\n",
      "LFGVTTLDIIRSNTFVAELKGKQPGEVEVPVIGGHSGVTILPLLSQVSFTE----QEVADLT----------KRI\r\n",
      "QNAGTEVVEAKGGGSATLSMGQAAARFGLSLVRALQGEQGVVECAYVEGDGQYAR---FFSQPLLLGKNGSIGTL\r\n",
      "SAFEQNA-LEGMLDTLKKDI---------------*\r\n",
      "\r\n",
      ">P1;4tvoA\r\n",
      "structure:4tvoA:    6: :  318: :::-1.00:-1.00\r\n",
      "------VAVTGAAGQIGYSLLFRLASGSLLGDRPIELRLLEIEPALQALEGVVMELDDCAFPLLSGVEIGSDPQK\r\n",
      "IFDGVSLALLVGARPRGAGMERSDLLEANGAIFTAQGKALNAVAADDVRVGVTGNPANTNALIAMTNAPDIPRER\r\n",
      "FSALTRLDHNRAISQLAAKTGAAVTDIKKMTIWGNHSATQYPDLFHAEVA--GKNAAEV-VNDQAWIEDEFIPTV\r\n",
      "AKRGAAIIDARGASSAASAASATIDAARDWLLGTPADDWVSMAV-VSDGS-YGVPEGLISSFPVTT-KGGNWTIV\r\n",
      "SGLEIDEFSRGRIDKSTAELADER-----------*\r\n",
      "\r\n",
      ">P1;4tvoB\r\n",
      "structure:4tvoB:    5: :  308: :::-1.00:-1.00\r\n",
      "------VAVTGAAGQIGYSLLFRLASGSLLGDRPIELRLLEIEPALQALEGVVMELDDCAFPLLSGVEIGSDPQK\r\n",
      "IFDGVSLALLVGARP---------LLEANGAIFTAQGKALNAVAADDVRVGVTGNPANTNALIAMTNAPDIPRER\r\n",
      "FSALTRLDHNRAISQLAAKTGAAVTDIKKMTIWGNHSATQYPDLFHAEVA--GKNAAEV-VNDQAWIEDEFIPTV\r\n",
      "AKRGAAIIDARGASSAASAASATIDAARDWLLGTPADDWVSMAV-VSDGS-YGVPEGLISSFPVTT-KGGNWTIV\r\n",
      "SGLEIDEFSRGRIDKSTAELADER-----------*\r\n",
      "\r\n",
      ">P1;2hjrA\r",
      "\r\n",
      "structure:2hjrA:   10: :  309: :::-1.00:-1.00\r\n",
      "------------AGQIGSTIALLLGQKDL-GD----VYMFDIIEGVPQGKAL--DLNHCALIGSPAKIFGENNYE\r\n",
      "YLQNSDVVIITAGVPRKPNMTRSDLLTVNAKIVGSVAENVGKYC-PNAFVICITNP--LDAMVYYFKEKSGIPAN\r\n",
      "CGMSGVLDSARFRCNLSRALGVKPSDVSAIVV-GGHGDEMIPLTSSVTFVEQGK-------ITHSQI-NEIIKKT\r\n",
      "AFGGGEIVELLKTGSAYAPAASAVAMAQAYLKDSKSVLVCSTYLT----GQYNVN-NLFVGVPVVIGKNG-----\r\n",
      "----IEDVVIVNLSDDEKSLFSKSVESIQNLVQ--*\r\n",
      "\r\n",
      ">P1;1t2dA\r\n",
      "structure:1t2dA:    4: :  250: :::-1.00:-1.00\r\n",
      "----AKIVLVG-SGMIGGVMATLIVQKNL-GDVVLFDIVKNMPKALDTSHTNVMAYSNC---KVSGSNTYDDLAG\r\n",
      "A----DVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHIKKNC-PNAFIIVVTNP--VDVMVQLLHQHSGVPKN\r\n",
      "IGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMVLLKRYITVGGIPLQEFINNKLISDAELEAIFDRT\r\n",
      "VNTALEIVNLHA-SPYVAPAAAIIEMAESYL--------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4plgA\r\n",
      "structure:4plgA:    1: :  253: :::-1.00:-1.00\r\n",
      "MTKRKKISLIG-SGMIGGTMAYLCAQKEL-GD----VVLFDVVKNMPQGKALDLS-HSSSIADTNVKVTGTNSYE\r\n",
      "DIKGSDVVIITAGLTKAPGKSRDDLLPFNAKIMREVGENIKKYC-PNAFVIVITNP--LDVMVKVLHEHSGLPKN\r\n",
      "CGMAGVLDSSRFRHFIAEKLNVSPRDVQAMVI-GAHGDKMVPLTRYVTFIKKGRTQEEIDEI---------VERT\r\n",
      "KNAGGEIVNLLGQGSAFAPAASAIEMAEAYL--------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4pltB\r\n",
      "structure:4pltB:   11: :  280: :::-1.00:-1.00\r\n",
      "------------AGNIGGTLAHLIAQKEL-GD----VVLFDIVEGMPQGKALDISHSS---PIMGSNVKITGTN-\r\n",
      "NYEDIDVVIITAGIPRKPGKSRDDLLSVNAKIMKDVAENIKKYC-PNAFVIVVTNP--LDVMVYVLHKYSGLPHN\r\n",
      "CGMAGVLDSSRFRYFLAEKLNVSPNDVQAMVI-GGHGDTMVGGIPLTEFIKQGITQEEID----EIVERT--RNA\r\n",
      "GGEIVNLLKTGSFAPAASAIEMAESYLKAYLEGQYGVKDLFVGVPV-----------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4plvA\r\n",
      "structure:4plvA:   10: :  265: :::-1.00:-1.00\r\n",
      "------------AGNIGGTLAHLIAQKEL-GD----VVLFDIVEGMPQGKALDISHSS---PIMGSNVKITGTN-\r\n",
      "NYEDIDVVIITAGI---------DLLSVNAKIMKDVAENIKKYC-PNAFVIVVTNP--LDVMVYVLHKYSGLPHN\r\n",
      "CGMAGVLDSSRFRYFLAEKLNVSPNDVQAMVI-GGHGDTMVGGIPLTEFIKQGITQEEID----EIVERT--RNA\r\n",
      "GGEIVNLLKTGSFAPAASAIEMAESYLKAYLEGQYGVKDLFVGVPV-----------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4plzA\r\n",
      "structure:4plzA:   80: :  236: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------------------RDDLLPLNNKIMIEIGGHIKKNC-PNAFIIVVTNP--VDVMVQLLHQHSGVPKN\r\n",
      "IGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMVLLKRYITVGGIPLQEFINNKLISDAELEAIFDRT\r\n",
      "VNTALEIVNLHA-SPYVAPAAAIIEMAESYL--------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3d0oA\r\n",
      "structure:3d0oA:   71: :  295: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---DADLVVICAGAAQKPGETRLDLVSKNLKIFKSVGEVMA--SKFDGIFLVATNPVDILAYATWKFSGLPKERV\r\n",
      "IGSGTILDSARFRLLLSEAFDVAPRSV-DAQIIGEHGDTELPVWSHANIAGQPLKTLLEQRPEGKAQIEQIFVQT\r\n",
      "RDAAYDIIQAKGITEAIFRNEDAVLTVSALLEGEYEEEDVYIGVPAVI-NRNGIRN--VVEIPLNDEEQSK----\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1y6jA\r\n",
      "structure:1y6jA:   58: :  167: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-YSDVDCDVIVVT----AGATRLDLAKKNVMIAKEVTQNIMKYYNHGV-ILVVSNPVDIITYMIQKWSGLPVGKV\r\n",
      "IGSGTVLDSIRFRYLLSEKLGVDVKNVHGYII-GEHGDSQL----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1ez4B\r\n",
      "structure:1ez4B:   55: :  230: :::-1.00:-1.00\r\n",
      "--------------------------------------------------------------------AFTAPKK\r\n",
      "IYKDADLVVITAGAPQKPGESRLDLVNKNLNI-------LSSIVKPVVDFLVAANPVDILTYATWKFSGFPKERV\r\n",
      "IGSGTSLDSSRLRVALGKQFNVDPRSV-DAYIMGEHGDSEFAAYSTATI----GTRPVRDVAKEQGVSDDDLAKV\r\n",
      "RNKAYDIINLKGAT-------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;5hs4A\r\n",
      "structure:5hs4A:    4: :  172: :::-1.00:-1.00\r\n",
      "-------IVLVGSGMIGGVMATLIVQKNL-GDVVMFDVVKNMPKALDTSHSNVMAYSNC---KVTGSNSYDDLKG\r\n",
      "A----DVVIVTAGF-------RDDLLPLNNKIMIEIGGHIKNLC-PNAFIIVVTNPVDVMVQLLFEHSGVPKNKI\r\n",
      "IGLGGVLDTSRLKYYISQKLNVCPRDVNALIV-GAHGNKMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r",
      "\r\n",
      "\r\n",
      ">P1;5nueA\r\n",
      "structure:5nueA:    3: :  331: :::-1.00:-1.00\r\n",
      "--EPVRVLVTGAAGQIGYALVPMIARGIMLGDQPVILHMLDIPPAAEALNGVKMELIDAAFPLLKGVVATTDAVE\r\n",
      "GCTGVNVAVMVGGFPRKEGMERKDVMSKNVSIYKSQAAALEKHAAPNCKVLVVANPANTNALILKEFAPSIPEKN\r\n",
      "ISCLTRLDHNRALGQISERLSVPVSDVKNVIIWGNHSSSQYPDVNHAKVQTSSGEKPVRELVKDDAWLDEFISTV\r\n",
      "QQRGAAIIKARKLSSALSAASSACDHIRDWVLGTPEGTFVSMGV-YSDGS-YSVPSGLIYSFPVTC-RNGDWSIV\r\n",
      "QGLPIDEVSRKKMDLTAEELKEEKDLAYSCLS---*\r\n",
      "\r\n",
      ">P1;1mldA\r\n",
      "structure:1mldA:    1: :  189: :::-1.00:-1.00\r\n",
      "----AKVAVLGASGGIGQPLSLLLKNSPLVSRLTLYAHTPGVAADLSHITRATVK----------GYLGPEQLPD\r\n",
      "CLKGCDVVVIPAGVPRKPGMTRDDLFNTNATIVATLTAACAQHC-PDAMICIISNPVNSTIPIAEVFKKHYNPNK\r\n",
      "IFGVTTLDIVRANAFVAELKGLDPARVSVPVIGGHAGKTIIPLISQCT---------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6j9sB\r\n",
      "structure:6j9sB:   59: :  233: :::-1.00:-1.00\r\n",
      "----------------------------------------------------------------------TSPKK\r\n",
      "IYKDADLVVITAGAPQKPGETRLDLVNKNLKILKSIVDPIVDSGFNGI-FLVAANPVDILTYATWKLSGFPKNRV\r\n",
      "VGSGTSLDTARFRQSIAKMVNVDARSVHAYIM-GEHGDANIGGVTIAEWVKEIKEDKLVKM----------FEDV\r\n",
      "RNKAYEIIKLKGAT-------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6iheA\r\n",
      "structure:6iheA:   70: :  206: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "----MDIVVISAGKPRKPGMSRRDLFVDNAKIMIDLAQKLPS-KNPGAIYLMVANPVDMMASVFMKYSKQF---T\r\n",
      "ISAGDQVETMRMRSFIAKKLKIPVTSV-DGFVGGEHGEDAV--VLWSTVKIKGKPVDEFNI-NKDEVSD-YVKKI\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6r8gA\r\n",
      "structure:6r8gA:    8: :  244: :::-1.00:-1.00\r\n",
      "------------SGQIGAIVGE-LCLLENLGD----LILYDVVPGIPQGKALDLK----HFSTILGVILGTNQIE\r\n",
      "DIKDADIIVITAGVQRKEGMTREDLIGVNGKIMKSVAESVKLHCSKAF-VICVSNP--LDIMVNVFHKSNLPHEK\r\n",
      "ICGMAILDTSRYCSLIADKLKVSAEDVNAVIL-GGHGDLMVPLQRYTSVNGVPLSEFVKKNMISQNEIQEIIQKT\r\n",
      "RNMGAEIIKLAKASAAFAPAAAITKMIKSYLY-------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;2v65A\r\n",
      "structure:2v65A:   97: :  298: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "------------------GESRLNLVQRNVNIFKFIIPNIVKYS-PNCILMVVSNPVDILTYVAWKLSGFPRHRV\r\n",
      "IGSGTNLDSARFRHLIGEKLHLHPSSCHAWIV-GEHGDSSVSGVNVAGVSLQGLNPQMGTEGDGEN-WKAIHKEV\r\n",
      "VDGAYEVIKLKGYTSWAIGMSVAVESIIKNMHKVHPVSTLVQGM-------HGVKDEVFLSVPCVLGNSGLTDVI\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1v6aA\r\n",
      "structure:1v6aA:  103: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "------------------GESRLNLVQRNVNIFKFIIPNIIKYS-PNCILLVVSNPVDILTYVAWKLSGLPRNRV\r\n",
      "IGSGTNLDSARFRHLMGEKLGIHPSNCHGWVI-GEHGDSSVSGVNVAGVFLQGLNPDMGTDKDKE-DWKSVHKMV\r\n",
      "VDSAYEVIKLKGYTSWAIGMSAACQSILKNLRKCHPVSTLVKGM-------HGVNEEVFLSVPCILGNSGLTDVV\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6balA\r\n",
      "structure:6balA:    1: :  274: :::-1.00:-1.00\r\n",
      "---AMKVAVLGAAGGIGQALALLL---KLQLPAGTDLSLYDIAPVTPGVAVDVSHIPTAV--NVKGF-SGEDPTP\r\n",
      "ALEGADVVLISAGVARKPGMDRSDLFNINAGIVRGLIEKVAVTC-PKACVGIITNPVNTTVAIALKKAGVYDKRK\r\n",
      "LFGVTTLDVLRSETFVAELKGLNVSRTSVPVIGGHSGVTILPLLSQVQYAKWNEDE-----------IEPLTKRI\r\n",
      "QNAGTEVLNAKGGSATLSMAQAAARFARSLVKGLSGETVVECTYVEGDG-----KYARFFSQPVRLGKEG-----\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4q3nA\r\n",
      "structure:4q3nA:   70: :  309: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--KDAGLVVITAGVNQKEGETRIHLLKRNAEIIKTVVKNIMSSGFNGI-LLVASNPVDVLAYVAWKESGLHHSRV\r\n",
      "IGSGTSLDTARLRYEISQYIHIDSRNVHAYIL-GEHGDSEFVCWSNANV----GAKPIADVIDSDEILEHIYLKV\r\n",
      "RDAAYEIISRKKATYYG--IGMALVRITAAIFN---NENRILPISVLNNGVYDCEDQVYIGLPAVLNKDGVHHIV\r\n",
      "KNEKENSQLNNSANILRKNL---------------*\r\n",
      "\r\n",
      ">P1;4uulA\r\n",
      "structure:4uulA:    1: :  332: :::-1.00:-1.00\r\n",
      "-SEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKA\r\n",
      "AFKDIDCAFLVASMPRKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPEN\r\n",
      "FSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKI\r\n",
      "GHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVV\r\n",
      "EGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQ--*\r\n",
      "\r\n",
      ">P1;4uupA\r\n",
      "structure:4uupA:    1: :  331: :::-1.00:-1.00\r\n",
      "--EPIHVLITGAAGQIGYALAFRIAKGDLFGDRKVVLHLLEIPPAMKALEGVCMELQDCAFPTLAGVVATDDPEE\r\n",
      "AFKDVDVAFLVGSFPRKPGMERADLLEKNAGIFKVQGKALSEYAKPTVKVLVVGNPANTNCLIAMANAPKLGPEN\r\n",
      "FSAMTRLDHNRAIGEIAAKLGVPVDKVHNVVVWGNHSNTQVPDVSHATVDKEGGTKKVSDALPKEYLEGEFVQKI\r\n",
      "AQRGGAVIEARGASSAASAANAALCHMRDWLFGTKPGDWVSMGIPVPEGNPYGIKPGVIYSFPCTVDKDGKVHIV\r\n",
      "EGLEINDWVREKMEATEKELIEERETAFKVLAQ--*\r\n",
      "\r\n",
      ">P1;4uuoA\r\n",
      "structure:4uuoA:    1: :  318: :::-1.00:-1.00\r\n",
      "--EPIHILITGAAGQIGYALTFRIAKGDLCGDRKVVLHLLEIPFGMKALEGCVMELQDCAFPNVAGIVWTDKVEE\r\n",
      "AFKGVDVAFLVGSFPRKDGMDRSDLLAKNGGIFTVQGKALNDYAKPTVKVLVVGNPANTNCLIAQASAPKLQNKN\r\n",
      "WCAMTRLDHNRMVGALAAKFGVTPEKIHKVCIWGNHSNTQVPDTTHATV----DTVKVADKLPKEYLEGEFAQMI\r\n",
      "ATRGGAVIKMRGASSAASAANAALTCVKDWLYGT---DFVSMAIPVPDNEPYGIKQGTIFSFPVTVSKDGEVHVV\r\n",
      "EGLELNDWVKGRLEATEKELIGEKETA--------*\r\n",
      "\r\n",
      ">P1;2a92A\r\n",
      "structure:2a92A:    6: :  186: :::-1.00:-1.00\r\n",
      "-------IVLVGSGMIGGVMATLIVQKNL-GDVVMFDVVKNMPKALDTSHSNVMAYSNC---KVTGSNSYDDLKG\r\n",
      "A----DVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHIKNLC-PNAFIIVVTNPVDVMVQLLFEHSGVPKNKI\r\n",
      "IGLGGVLDTSRLKYYISQKLNVCPRDVNALIV-GAHGNKMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6bb1E\r\n",
      "structure:6bb1E:   94: :  277: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------------------RLNLVQRNVNIFKFIIPNVVKYS-PNCKLLIVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVP--VWSGMNVAGVSLKTL------HPDKEVHKQV\r\n",
      "VESAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPVSTMIKGL-------YGIKDDVFLSVPCILGQNG-----\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1oc4A\r\n",
      "structure:1oc4A:    4: :  186: :::-1.00:-1.00\r\n",
      "----AKIVLVG-SGMIGGVMATLIVQKNL-GDVVMFDIVKNMPKALDTSHTNVMAYSNCK-------VSGSNTYD\r\n",
      "DLKDADVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHI-KNNCPNAFIIVVTNP--VDVMVQLLHQHSGVPKN\r\n",
      "VGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1smkA\r\n",
      "structure:1smkA:    4: :  198: :::-1.00:-1.00\r\n",
      "------VAILGAAGGIGQPLAMLMKMNPLVS----VLHLYDVVNAPG-VTADISHMDTGAV--VRGFLGQQQLEA\r\n",
      "ALTGMDLIIVPAGVPRKPGMTRDDLFKINAGIVKTLCEGIAKCC-PRAIVNLISNPVNSTVPIAFKKAGTYDPKR\r\n",
      "LLGVTMLDVVRANTFVAEVLGLDPRDVDVPVVGGHAGVTILPLLSQSSFTQE-----------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6vdhA\r\n",
      "structure:6vdhA:   12: :  176: :::-1.00:-1.00\r\n",
      "------------SGMIGGTMAYLCALKEL-GD----VVLFDVVKNMPQGKALDLS-------HSTS-VADTNVKV\r\n",
      "SYEDIKGSDVVI---ITAGLSRDDLLPINAKIMKEVGENIKKYC-PNAFVICITNP--LDVMVKVLQEASGLPHN\r\n",
      "CGMAGVLDSSRFRYFIAEKLNVSPRDVQAMVI-GAHGDNMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4jcoA\r\n",
      "structure:4jcoA:    3: :  287: :::-1.00:-1.00\r\n",
      "------VSVVGAAGTVGAAAGYNIALRDI-ADEVVF---VDIPDKEDDTVGQAADTN-----HGIAYDSNTRVRG\r\n",
      "GYEDTDVVVITAGIPRQPGQTRIDLAGDNAPIMEDIQSSLDEHNDDYISLTTSNPVDLLNRHLYEAGDRS-REQV\r\n",
      "IGFGGRLDSARFRYVLSEEFDAPVQNVEGTIL-GEHGDAQVVDGTDPEFSGDEKEQLLGDLQE------------\r\n",
      "--SAMDVIERKGATE-WGPARGVAHMVEAILHDT--GEVLPASVKL-EGE-FG-HEDTAFGVPVRLGSNGVEEIV\r\n",
      "E------W---DLDDYEQDL---------------*\r\n",
      "\r\n",
      ">P1;1lldA\r\n",
      "structure:1lldA:    9: :  233: :::-1.00:-1.00\r\n",
      "------------AGAVGSTLA--FAAAQRGIAREIVLE--DI--AKERVEAEVLDMQGSSFYPTVSIDGSDDPEI\r\n",
      "C-RDADMVVITAGPRQKPGQSRLELVGATVNILKAIMPNLVKVA-PNAIYMLITNP----VDIATHVAQKLTPEN\r\n",
      "FGSGTNLDSARLRFLIAQQTGVNVKNVHAYIA-GEHGDSEVPLWESATIMSDWTPLPGHDPLDAD-KREEIHQEV\r\n",
      "KNAAYKIINGKGATNYA----------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1hyeA\r\n",
      "structure:1hyeA:    3: :  183: :::-1.00:-1.00\r\n",
      "------VTIIGASGRVGSATALLLAKEPFMKD----LVLIGREHSINKLEGLREDIYDA----LAGYVESDENLR\r\n",
      "IIDESDVVIITSGVPRKEGMSRMDLAKTNA---KIVGKYAKKIAECDTKIFVITNPVDVMTYKALVDSKFERNQV\r\n",
      "FGLGTHLDSLRFKVAIAKFFGVHIDEVRTRII-GEHGDSMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4wleA\r\n",
      "structure:4wleA:    2: :  190: :::-1.00:-1.00\r\n",
      "----AKVAVLGASGGIGQPLSLLLKNSPLVSRLTLY-DIAHTPGVAADLSHIETKAA------VKGYLGPEQLPD\r\n",
      "CLKGCDVVVIPAGVPRKPGMTRDDLFNTNATIVATLTAACAQHC-PEAMICVIANPVNSTIPIAEVFKKHYNPNK\r\n",
      "IFGVTTLDIVRANTFVAELKGLDPARVNVPVIGGHAGKTIIPLISQCT---------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3tl2A\r\n",
      "structure:3tl2A:   34: :  185: :::-1.00:-1.00\r\n",
      "--------------------------------------LVDIPQLENPTKGKALDMLEAS--PVQGFDANTSDYA\r\n",
      "DTADSDVVVITAGIARKPGMSRDDLVATNSKIMKSITRDIAKHS-PNAIIVVLTNPVDAMTYSVFKEAGFPKERV\r\n",
      "IGQSGVLDTARFRTFIAQELNLSVKDITGFVL-GGHGDDMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3p7mA\r\n",
      "structure:3p7mA:   11: :  205: :::-1.00:-1.00\r\n",
      "------------AGNIGGTLAH-LALIKQLGD----VVLFDIAQGMPNGKALDL-LQTCPIEGVDFKVRGTNDYK\r\n",
      "DLENSDVVIVTAGVPRKPGMSRDDLLGINIKVMQTVGEGI-KHNCPNAFVICITNP--LDIMVNMLQKFSGVPDN\r\n",
      "VGMAGVLDSARFRTFLADELNVSVQQV-QAYVMGGHGDTMVAGVSLEQLVKEGKLKQ------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4nd2A\r\n",
      "structure:4nd2A:    1: :  288: :::-1.00:-1.00\r\n",
      "MIERRKIAVIG-SGQIGGNIAYIVGK-----DNLADVVLFDIAEGIPQGKALDITHSMVMFGSTSKVIGTND-YA\r\n",
      "DISGSDVVIITASIPGRPKDDRSELLFGNARILDSVAEGVKKYC-PNAFVICITNP----LDVMVSHFQKVSPHN\r\n",
      "CGMAGVLDSSRFRTFIAQHFGVNASDVSANVI-GGHGDGMVGGVPLSSFIKQGLTQEQIDIVCHTRI--------\r\n",
      "---AWKEVADNLKTGTAAPAAAAVKMAEAYL------KDKKAVVPCSASNHYGVK-GIYMGVPTIIGKNGVEDIL\r\n",
      "E----------------------------------*\r\n",
      "\r\n",
      ">P1;4nd5A\r\n",
      "structure:4nd5A:    1: :  276: :::-1.00:-1.00\r\n",
      "MIERRKIAVIG-SGQIGGNIAYIVGK-----DNLADVVLFDIAEGIPQGKALDITHSMVMFGSTSKVIGTND---\r\n",
      "-YADISGSDVV---------IITALLFGNARILDSVAEGVKKYC-PNAFVICITNP----LDVMVSHFQKVSPHN\r\n",
      "CGMAGVLDSSRFRTFIAQHFGVNASDVSANVI-GGHGDGMVGGVPLSSFIKQGLTQEQIDIVCHTRI--------\r\n",
      "---AWKEVADNLKTGTAAPAAAAVKMAEAYL------KDKKAVVPCSASNHYGVK-GIYMGVPTIIGKNGVEDIL\r\n",
      "E----------------------------------*\r\n",
      "\r\n",
      ">P1;5w8jA\r\n",
      "structure:5w8jA:   91: :  296: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PNCKLLIVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVAGVSLKTLHPDLGTDK-----DKEQ-WKEVHKQV\r\n",
      "VESAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPVSTMIKGL-------YGIKDDVFLSVPCILGQNG-----\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1b8pA\r\n",
      "structure:1b8pA:    6: :  325: :::-1.00:-1.00\r\n",
      "------VAVTGAAGQICYSLLFRIANGDMLGDQPVILQLLEIPKAQKALQGVMMEIDDCAFPLLAGMTAHADPMT\r\n",
      "AFKDADVALLVGARPRGPGMERKDLLEANAQIFTVQGKAIDAVASRNIKVLVVGNPANTNAYIAMKSAPSLPAKN\r\n",
      "FTAMLRLDHNRALSQIAAKTGKPVSSIEKLFVWGNHSPTMYADYRYAQI--DGASVK--DMINDDWNRDTFLPTV\r\n",
      "GKRGAAIIDARGVSSAASAANAAIDHIHDWVLGTA-GKWTTMGI--PSDGSYGIPEGVIFGFPVTTE-NGEYKIV\r\n",
      "QGLSIDAFSQERINVTLNELLEEQN-GVQHL----*\r\n",
      "\r\n",
      ">P1;2ldxA\r\n",
      "structure:2ldxA:   67: :  306: :::-1.00:-1.00\r\n",
      "-----------------------------------------------------------------GSLFLSTPKI\r\n",
      "VFKDYNVSIITAGARMVSGQTRLDLLQRNVAIMKAIVPGVIQ-NSPDCKIIVVTNPVDILTYVVWKISGFPVGRV\r\n",
      "IGSGCNLDSARFRYLIGEKLGVNPTSCHGWVL-GEHGDSSVSGVNVAGVTLKSLNPAIGTDKNKQH-WKNVHKQV\r\n",
      "VEGGYEVLDMKGYTSWASVTDLARSILKN-LKRVHPVTTLVKGF-------HGIKEEVFLSIPCVLGESG---IT\r\n",
      "DFVKVN-----------------------------*\r\n",
      "\r\n",
      ">P1;9ldtA\r\n",
      "structure:9ldtA:   93: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------ITAGARQQEGESRLNLVQRNVNIFKFIIPNIVKYS-PNCKLLVVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHPLSCHGWIL-GEHGDSSVSGVNVAGVSLKNLHPELGTDADKEH-WKAVHKEV\r\n",
      "VDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKENVFLSVPCILGQNGISDVV\r\n",
      "K----------------------------------*\r\n",
      "\r\n",
      ">P1;1ldnA\r\n",
      "structure:1ldnA:   73: :  301: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--RDADLVVICAGANQKPGETRLDLVDKNIAIFRSIVESVMASGFQGL-FLVATNPVDI-LTYATWKFSGLPHER\r\n",
      "IGSGTILDTARFRFLLGEYFSVAPQNVHAYII-GEHGDTELPVWSQA-YIGVMPIRKLVESKGEEAQKDRIFVNV\r\n",
      "RDAAYQIIEKKGATGLARVTRAAILTVSAYLDGLYGERDVYIGVPAVI-NRNGIRE--VIEIELNDDEKNRFH--\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;5ldhA\r\n",
      "structure:5ldhA:   94: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------VTAGVRQQEGESRLNLVQRNVNVFKFIIPQIVKYS-PNCIIIVVSNPVDILTYVAWKLSGLPKHRV\r\n",
      "IGSGCNLDSARFRYLMGEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVVLQQLNPEMTDNDSEWKEVHKMV\r\n",
      "VESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6cepA\r\n",
      "structure:6cepA:   94: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------VTAGVRQQEGESRLNLVQRNVNVFKFIIPQIVKYS-PDCIIIVVSNPVDILTYVTWKLSGLPKHRV\r\n",
      "IGSGCNLDSARFRYLMAEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVSLQELNPEMTDNDSEWKEVHKMV\r\n",
      "VESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6cepD\r\n",
      "structure:6cepD:   90: :  289: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--------------------VTAGVVQRNVNVFKFIIPQIVKYS-PDCIIIVVSNPVDILTYVTWKLSGLPKHRV\r\n",
      "IGSGCNLDSARFRYLMAEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVSLQELNPEMTDNDSEWKEVHKMV\r\n",
      "VESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1civA\r\n",
      "structure:1civA:   33: :  358: :::-1.00:-1.00\r\n",
      "-----NVAVSGAAGMISNHLLFKLASGEVFGDQPISLKLLGSERSFAALEGVAMELEDSLYPLLRQVSIGIDPYE\r\n",
      "IFQDAEWALLIGAKPRGPGMERADLLDINGQIFAEQGKALNAVASPNVKVMVVGNPCNTNALICLKNAPNIPPKN\r\n",
      "FHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDFLNAKIHGIPVTEV---IRDRKWLEDEFTNMV\r\n",
      "QTRGGVLIKKWGRSSAASTAVSIVDAIRSLVTPTPEGDWFSTGV-YTNGNPYGIAEDIVFSMPCRSKGDGDYEFV\r\n",
      "KDVIFDDYLSKKIKKSEDELLAEKKCVAHLTGEG-*\r\n",
      "\r\n",
      ">P1;5nqbA\r\n",
      "structure:5nqbA:   93: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PHCKLLVVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHALSCHGWIL-GEHGDSSVSGMNVAGVSLKTLHPELGTDADKEQ-WKQVHKQV\r\n",
      "VDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMLKGL-------YGIKEDVFLSVPCVLGQNGISDVV\r\n",
      "K----------------------------------*\r\n",
      "\r\n",
      ">P1;6um4A\r\n",
      "structure:6um4A:    8: :  326: :::-1.00:-1.00\r\n",
      "------ITVSGACGQIANSLLFRIANGEMLGENQVKLRLLERPEAMKALEGVKMELDDGAFPLLEEIYLTDSEND\r\n",
      "AFRGADYAILLGGKPRGPGMERADVMKDNAAIFKAQGEALNKQANGDVLVLIVANPANTNAMITSANAPDIPPEN\r\n",
      "ITAMTRLDHDRGLAQVAAKVGCNITDISRFAIWGNHSATQYPDLSFTTI----KGQWGLNVIDEQWITNEFIPNV\r\n",
      "QQRGAAIIKARGKSSAASAADAAIKHMHDWVLGNS--EWVSMAI--PSRGQYGIPRGIWCSMPVQCFGAGKYGVI\r\n",
      "EGLPINSFSADRINASVKELIEEKKIVENLL----*\r\n",
      "\r\n",
      ">P1;4m1qA\r\n",
      "structure:4m1qA:   69: :  233: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--KSADIVVITAGAAQKPGETRLDLVEKNANIFKGIVDQVMGSGFNGI-FIIATNPVDVLAYATWKFSGLPKERV\r\n",
      "IGSGTILDTARFRFLLSEYFDIDVRNIHGYIM-GEHGDTELPVWSQTRIGSE-PISRYMDKYKPDKDLDEIFVNV\r\n",
      "RDAAYHIIERKGATHYA----------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1pzgA\r\n",
      "structure:1pzgA:   71: :  190: :::-1.00:-1.00\r\n",
      "-------------------------------------------------------------------------EA\r\n",
      "ALTGADCVIVTAGLTKVPGKSRNDLLPFNSKIIREIGQNIKKYC-PKTFIIVVTNP--LDCMVKVMEASGVPTNM\r\n",
      "ICGMAMLDSGRFRRYVADALSVSPRDVQATVI-GTHGDCMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;2x0jA\r\n",
      "structure:2x0jA:   68: :  285: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--KGSEIIVVTAGLARKPGMTRLDLAHKNAGIIKDIAKKIVENA-PESKILVVTNPMDVMTYIMWKESGKPRNEV\r\n",
      "FGMGNQLDSQRLKERL---YNAGARNIRRAWIIGEHGDSMFVAKSLADFDGEVDWEAV----ENDVRF-------\r\n",
      "--VAAEVIKRKGAT-IFGPAVAIYRMVKAVVEDT--GEIIPTSM-ILQGE-YGIE-NVAVGVPAKLGKNGA--EV\r\n",
      "ADIKLSDEEIEKLRNSAKIL---------------*\r\n",
      "\r\n",
      ">P1;5ujkA\r\n",
      "structure:5ujkA:   10: :  180: :::-1.00:-1.00\r\n",
      "------------AGQIGGTLAHLAGLKEL-GD----VVLFDIVDGVPQGKALDI-AESAPVDGFDAKYSGASDYS\r\n",
      "AIAGADVVIVTAGVPRKPGMSRDDLIGINLKVMEAVGAGIKEHA-PDAFVICITNP--LDAMVWALQKFSGLPTN\r\n",
      "VGMAGVLDSARFRHFLAEEFGVSVEDV-TAFVLGGHGDDMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4ajjA\r\n",
      "structure:4ajjA:   92: :  303: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PQCKLLIVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVSGVNVAGVSLKSLNPQLGTDADKEQWKDV-HKQV\r\n",
      "VDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKEDVFLSVPCILGQNGISDVV\r\n",
      "K----------------------------------*\r\n",
      "\r\n",
      ">P1;3nepX\r\n",
      "structure:3nepX:    8: :  263: :::-1.00:-1.00\r\n",
      "------------AGNVGATVAECVARQDVAKE----VVMVDIKDGMPQGKALDMR-ESSPIHGFDTRVTGTNDYG\r\n",
      "PTEDSDVCIITAGL-------RDDLLAKNTEIVGGVTEQFVE-GSPDSTIIVVANPLDVMTYVA--YEASGFPTN\r\n",
      "MGMAGVLDTGRFRSFIAEELDVSVRDVQALLM-GGHGDTMVPLPRYTTVGGIPVPQLIDDARIEEIVERT--KGA\r\n",
      "GGEIVDLMGTSAWAAAAEMTEAILKDNKAYCDGEYGLDDLFIGVPVKLG--------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6pblA\r\n",
      "structure:6pblA:    5: :  318: :::-1.00:-1.00\r\n",
      "------VAVTGAAGQIGYALVFRIASGQMFGNTEVELNLLELEPALPSLEGVAMELDDCAFPLLKRIVCTADLNK\r\n",
      "AMDGVNWALLVGSVPRKQGMERSDLLQINGGIFTKQGQAINDYASDDVRVFVVGNPCNTNCLIAMNHAKDVPSDR\r\n",
      "FYAMTTLDELRARTQLAKKAGVDITAVTQMTIWGNHSATQYPDFYNA---KINGTSAAQVINDETWLKETFVSTV\r\n",
      "QQRGAAVIKARGSSSAASAANAIITGVNHLVTDTPAGESFSM-CRRSKGE-YGVDEGLIFSFPCR-REHGELKVV\r\n",
      "ENLEFNDFGRERFNTTLNELRSERD----------*\r\n",
      "\r\n",
      ">P1;1y7tA\r\n",
      "structure:1y7tA:    1: :  319: :::-1.00:-1.00\r\n",
      "MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGDQPVILQLLEIPQAMKALEGVVMELEDCAFPLLAGLEATDDPKV\r\n",
      "AFKDADYALLVGAAPRKAGMERRDLLQVNGKIFTEQGRALAEVAKKDVKVLVVGNPANTNALIAYKNAPGLNPRN\r\n",
      "FTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSTMFPDL----FHAEVDGRPALELVDMEWYEKVFIPTV\r\n",
      "AQRGAAIIQARGASSAASAANAAIEHIRDWALGTPEGDWVSMA--VPSQGEYGIPEGIVYSFPVTA-KDGAYRVV\r\n",
      "EGLEINEFARKRMEITAQELLDEME----------*\r\n"
     ]
    }
   ],
   "source": [
    "!cat build_profile.ali"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Select the most appropriate templates for our query sequence over the 2/3/4/5/6/less/more similar PDB structures\n",
    "#Download the pdb files either manually from RCSB or using following command in Linux/MacOSX (wget/curl)\n",
    "#In this example we are using 6 PDBs just to get you an idea but may not be best models!\n",
    "\n",
    "#Change PDBIDs in the following link and after -o option (all in small letters) for eg.\n",
    "!curl https://files.rcsb.org/download/4h7p.pdb -o 4h7p.pdb"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Multiple Templates"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Identify the protein family and pick the PDBs to cluster them within single family very close to target sequence."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1375173    1342.942     1.311\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1375701    1343.458     1.312\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1424301    1390.919     1.358\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1437601    1403.907     1.371\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "io_data____W> Setting io.atom_files_directory to a colon-delimited string is\n",
      "              deprecated, as it is not robust on Windows systems. Set it to\n",
      "              a list of directories instead. For example:\n",
      "              env.io.atom_files_directory = ['./', '../atom_files/']\n",
      "openf___224_> Open           ./3d5t.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1075679    1050.468     1.026\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1075755    1050.542     1.026\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1075926    1050.709     1.026\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1076776    1051.539     1.027\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1078034    1052.768     1.028\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1079938    1054.627     1.030\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1079978    1054.666     1.030\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1080068    1054.754     1.030\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1082924    1057.543     1.033\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1082984    1057.602     1.033\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1083119    1057.733     1.033\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1087403    1061.917     1.037\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1087491    1062.003     1.037\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1087689    1062.196     1.037\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1094115    1068.472     1.043\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1094247    1068.601     1.044\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1094544    1068.891     1.044\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1104166    1078.287     1.053\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1104366    1078.482     1.053\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1104816    1078.922     1.054\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1119266    1093.033     1.067\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1119566    1093.326     1.068\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1120241    1093.985     1.068\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1141899    1115.136     1.089\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1142347    1115.573     1.089\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1143355    1116.558     1.090\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1175859    1148.300     1.121\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1407731    1374.737     1.343\n",
      "read_mo_297_> Segments, residues, atoms:        1      321     2376\n",
      "read_mo_298_> Segment:        1     3 A   325 A     2376\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1417011    1383.800     1.351\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1598280    1560.820     1.524\n",
      "openf___224_> Open           ./6itk.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1609291    1571.573     1.535\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1609367    1571.647     1.535\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1609538    1571.814     1.535\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1610388    1572.645     1.536\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1611646    1573.873     1.537\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1613550    1575.732     1.539\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1613590    1575.771     1.539\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1613680    1575.859     1.539\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1616536    1578.648     1.542\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1616596    1578.707     1.542\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1616731    1578.839     1.542\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1621015    1583.022     1.546\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1621103    1583.108     1.546\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1621301    1583.302     1.546\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1627727    1589.577     1.552\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1627859    1589.706     1.552\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1628156    1589.996     1.553\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1637778    1599.393     1.562\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1637978    1599.588     1.562\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1638428    1600.027     1.563\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1652878    1614.139     1.576\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1653178    1614.432     1.577\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1653853    1615.091     1.577\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1675511    1636.241     1.598\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1675959    1636.679     1.598\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1676967    1637.663     1.599\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1709471    1669.405     1.630\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1941343    1895.843     1.851\n",
      "read_mo_297_> Segments, residues, atoms:        1      318     2387\n",
      "read_mo_298_> Segment:        1     8 A   330 A     2387\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1601611    1564.073     1.527\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1782315    1740.542     1.700\n",
      "openf___224_> Open           ./5mdh.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1793326    1751.295     1.710\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1793402    1751.369     1.710\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1793573    1751.536     1.710\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1794423    1752.366     1.711\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1795681    1753.595     1.712\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1797585    1755.454     1.714\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1797625    1755.493     1.714\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1797715    1755.581     1.714\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1800571    1758.370     1.717\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1800631    1758.429     1.717\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1800766    1758.561     1.717\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1805050    1762.744     1.721\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1805138    1762.830     1.722\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1805336    1763.023     1.722\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1811762    1769.299     1.728\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1811894    1769.428     1.728\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1812191    1769.718     1.728\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1821813    1779.114     1.737\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1822013    1779.310     1.738\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1822463    1779.749     1.738\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1836913    1793.860     1.752\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1837213    1794.153     1.752\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1837888    1794.812     1.753\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1859546    1815.963     1.773\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1859994    1816.400     1.774\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1861002    1817.385     1.775\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1893506    1849.127     1.806\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2125378    2075.564     2.027\n",
      "read_mo_297_> Segments, residues, atoms:        1      333     2550\n",
      "read_mo_298_> Segment:        1     1 A   333 A     2550\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1786210    1744.346     1.703\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1977151    1930.812     1.886\n",
      "openf___224_> Open           ./1bdm.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1988162    1941.564     1.896\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1988238    1941.639     1.896\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1988409    1941.806     1.896\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1989259    1942.636     1.897\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1990517    1943.864     1.898\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1992421    1945.724     1.900\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1992461    1945.763     1.900\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1992551    1945.851     1.900\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1995407    1948.640     1.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1995467    1948.698     1.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1995602    1948.830     1.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1999886    1953.014     1.907\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1999974    1953.100     1.907\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2000172    1953.293     1.908\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2006598    1959.568     1.914\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2006730    1959.697     1.914\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2007027    1959.987     1.914\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2016649    1969.384     1.923\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2016849    1969.579     1.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2017299    1970.019     1.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2031749    1984.130     1.938\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2032049    1984.423     1.938\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2032724    1985.082     1.939\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2054382    2006.232     1.959\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2054830    2006.670     1.960\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2055838    2007.654     1.961\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2088342    2039.396     1.992\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2320214    2265.834     2.213\n",
      "read_mo_297_> Segments, residues, atoms:        1      318     2406\n",
      "read_mo_298_> Segment:        1     0 A   332 A     2406\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1980590    1934.170     1.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2161940    2111.270     2.062\n",
      "openf___224_> Open           ./7mdh.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2172951    2122.022     2.072\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2173027    2122.097     2.072\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2173198    2122.264     2.073\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2174048    2123.094     2.073\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2175306    2124.322     2.075\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2177210    2126.182     2.076\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2177250    2126.221     2.076\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2177340    2126.309     2.076\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2180196    2129.098     2.079\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2180256    2129.156     2.079\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2180391    2129.288     2.079\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2184675    2133.472     2.083\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2184763    2133.558     2.084\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2184961    2133.751     2.084\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2191387    2140.026     2.090\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2191519    2140.155     2.090\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2191816    2140.445     2.090\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2201438    2149.842     2.099\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2201638    2150.037     2.100\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2202088    2150.477     2.100\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2216538    2164.588     2.114\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2216838    2164.881     2.114\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2217513    2165.540     2.115\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2239171    2186.690     2.135\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2239619    2187.128     2.136\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2240627    2188.112     2.137\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2273131    2219.854     2.168\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2273803    2220.511     2.168\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2275315    2221.987     2.170\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2531379    2472.050     2.414\n",
      "read_mo_297_> Segments, residues, atoms:        1      351     2688\n",
      "read_mo_298_> Segment:        1    23 A   385 A     2688\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2192925    2141.528     2.091\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2394192    2338.078     2.283\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ./4h7p.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2405203    2348.831     2.294\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2405279    2348.905     2.294\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2405450    2349.072     2.294\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2406300    2349.902     2.295\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2407558    2351.131     2.296\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2409462    2352.990     2.298\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2409502    2353.029     2.298\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2409592    2353.117     2.298\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2412448    2355.906     2.301\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2412508    2355.965     2.301\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2412643    2356.097     2.301\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2416927    2360.280     2.305\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2417015    2360.366     2.305\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2417213    2360.560     2.305\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2423639    2366.835     2.311\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2423771    2366.964     2.311\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2424068    2367.254     2.312\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2433690    2376.650     2.321\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2433890    2376.846     2.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2434340    2377.285     2.322\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2448790    2391.396     2.335\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2449090    2391.689     2.336\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2449765    2392.349     2.336\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2471423    2413.499     2.357\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2471871    2413.937     2.357\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2472879    2414.921     2.358\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2505383    2446.663     2.389\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2737255    2673.101     2.610\n",
      "read_mo_297_> Segments, residues, atoms:        1      313     2244\n",
      "read_mo_298_> Segment:        1     1 A   324 A     2244\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2371417    2315.837     2.262\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2545694    2486.029     2.428\n",
      "  ALIGN_CODES                                   : 3d5tA 6itkA 5mdhA 1bdmA 7mdhA 4h7pA \n",
      "  Atom type for alignment (FIT_ATOMS)           : CA\n",
      "  FIT                                           :         T\n",
      "  IMPROVE_ALIGNMENT                             :         T\n",
      "  WATER_IO, HETATM_IO, HYDROGEN_IO              :         F        F        F\n",
      "  NO_TER                                        :         F\n",
      "  OUTPUT                                        : ALIGNMENT QUALITY\n",
      "  FIT_ON_FIRST                                  :         F\n",
      "  WRITE_FIT                                     :         F\n",
      "  FIT_PDBNAM                                    :         T\n",
      "  WRITE_WHOLE_PDB                               :         T\n",
      "  CURRENT_DIRECTORY                             :         T\n",
      "  Feature weights (FEATURE_WEIGHTS)             :    1.0000   0.0000   0.0000   0.0000   1.0000   0.0000\n",
      "  NORMALIZE_PP_SCORES                           :         F\n",
      "  Gap initiation penalty (GAP_PENALTIES[1])     :  -450.0000\n",
      "  Gap extension penalty (GAP_PENALTIES[2])      :   -50.0000\n",
      "  Break-break bonus                             : 10000.0000\n",
      "  Gap_function (align2d flag)                    :         F\n",
      "  2-D Gap penalties                             :  3.50  3.50  3.50  0.20  4.00  6.50  2.00  0.00\n",
      "  GAP-GAP_SCORE                                 :    0.0000\n",
      "  GAP-RESIDUE_SCORE                             :    0.0000\n",
      "  Perform gap-residue and gap-gap correction    :         T\n",
      "  Residue type - residue type file (RR_FILE)    : $(LIB)/as1.sim.mat\n",
      "  OFF_DIAGONAL                                  :       100\n",
      "  OVERHANG                                      :        30\n",
      "  LOCAL_ALIGNMENT                               :         F\n",
      "  MATRIX_OFFSET                                 :    0.0000\n",
      "  N_SUBOPT                                      :         0\n",
      "  SUBOPT_OFFSET                                 :    0.0000\n",
      "  Gap introduction penalty (GAP_PENALTIES_3D_1) :    0.0000\n",
      "  Gap extension penalty (GAP_PENALTIES_3D_2)    :    3.0000\n",
      "  Max dist for equiv (2 * GAP_PENALTIES_3D_2)   :    6.0000\n",
      "  ALIGN3D_TRF                                   :         F\n",
      "  RMS_CUTOFF                                    :    3.5\n",
      "\n",
      "\n",
      "\n",
      "salign__276_> 'align_block' changed to 1.\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2545990    2486.318     2.428\n",
      "\n",
      "SALIGN______> Matrix of pairwise protein distances for the initial tree:\n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA       0.000   0.207   0.231   0.211   0.274   0.255\n",
      "6itkA       0.207   0.000   0.249   0.199   0.287   0.284\n",
      "5mdhA       0.231   0.249   0.000   0.247   0.263   0.258\n",
      "1bdmA       0.211   0.199   0.247   0.000   0.276   0.272\n",
      "7mdhA       0.274   0.287   0.263   0.276   0.000   0.303\n",
      "4h7pA       0.255   0.284   0.258   0.272   0.303   0.000\n",
      "\n",
      "SALIGN____> Matrix of pairwise equivalences: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     312     306     302     316     289\n",
      "6itkA           0       0     292     307     304     261\n",
      "5mdhA           0       0       0     300     301     272\n",
      "1bdmA           0       0       0       0     310     274\n",
      "7mdhA           0       0       0       0       0     296\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "openf___224_> Open           ldh.tree\n",
      "                                                         .--------- 3d5tA          0.2086\n",
      "                                                         |\n",
      "                                                         |     .--- 6itkA          0.1986\n",
      "                                                         |     |\n",
      "                                     .----------------------------- 1bdmA          0.2397\n",
      "                                     |\n",
      "                       .------------------------------------------- 5mdhA          0.2620\n",
      "                       |\n",
      "        .---------------------------------------------------------- 4h7pA          0.2864\n",
      "        |\n",
      "      .------------------------------------------------------------ 7mdhA\n",
      "\n",
      "      +----+----+----+----+----+----+----+----+----+----+----+----+\n",
      "     0.2899    0.2741    0.2583    0.2425    0.2267    0.2109    0.1951\n",
      "          0.2820    0.2662    0.2504    0.2346    0.2188    0.2030\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    1\n",
      "\n",
      "Current alignment total score:        63.95\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00           -               M   0 A                 \n",
      "   2    0.00    0.00    0.00    0.00           -               K   1 A                 \n",
      "   3    0.20    1.07    0.54    0.00           S   8 A         A   2 A                 \n",
      "   4    0.29    0.83    0.42    0.00           T   9 A         P   3 A                 \n",
      "   5    0.30    0.17    0.09    0.00           K  10 A         V   4 A                 \n",
      "   6    0.20    0.19    0.10    0.00           K  11 A         R   5 A                 \n",
      "   7    0.07    0.26    0.13    0.00 *         V  12 A         V   6 A                 \n",
      "   8    0.24    0.36    0.18    0.00           T  13 A         A   7 A                 \n",
      "   9    0.06    0.50    0.25    0.00 *         V  14 A         V   8 A                 \n",
      "  10    0.07    0.26    0.13    0.00 *         T  15 A         T   9 A                 \n",
      "  11    0.02    0.38    0.19    0.00 *         G  16 A         G  10 A                 \n",
      "  12    0.08    0.58    0.29    0.00 *         A  17 A         A  11 A                 \n",
      "  13    0.08    0.59    0.29    0.00 *         A  18 A         A  12 A                 \n",
      "  14    0.04    0.16    0.08    0.00 *         G  19 A         G  13 A                 \n",
      "  15    0.10    0.38    0.19    0.00 *         Q  20 A         Q  14 A                 \n",
      "  16    0.08    0.13    0.07    0.00 *         I  21 A         I  15 A                 \n",
      "  17    0.25    0.54    0.27    0.00           S  22 A         G  16 A                 \n",
      "  18    0.05    0.34    0.17    0.00 *         Y  23 A         Y  17 A                 \n",
      "  19    0.07    0.46    0.23    0.00 *         S  24 A         S  18 A                 \n",
      "  20    0.05    0.39    0.19    0.00 *         L  25 A         L  19 A                 \n",
      "  21    0.05    0.41    0.20    0.00 *         L  26 A         L  20 A                 \n",
      "  22    0.29    0.83    0.42    0.00           W  27 A         F  21 A                 \n",
      "  23    0.07    1.16    0.58    0.00 *         R  28 A         R  22 A                 \n",
      "  24    0.08    0.88    0.44    0.00 *         I  29 A         I  23 A                 \n",
      "  25    0.08    0.54    0.27    0.00 *         A  30 A         A  24 A                 \n",
      "  26    0.28    0.92    0.46    0.00           N  31 A         A  25 A                 \n",
      "  27    0.03    1.03    0.51    0.00 *         G  32 A         G  26 A                 \n",
      "  28    0.08    1.26    0.63    0.00 *         E  33 A         E  27 A                 \n",
      "  29    0.27    1.42    0.71    0.00           V  34 A         M  28 A                 \n",
      "  30    0.21    1.52    0.76    0.00           F  35 A         L  29 A                 \n",
      "  31    0.02    1.41    0.71    0.00 *         G  36 A         G  30 A                 \n",
      "  32    0.24    0.93    0.47    0.00           T  37 A         K  31 A                 \n",
      "  33    0.07    1.06    0.53    0.00 *         D  38 A         D  32 A                 \n",
      "  34    0.29    1.04    0.52    0.00           T  39 A         Q  33 A                 \n",
      "  35    0.04    1.07    0.53    0.00 *         P  40 A         P  34 A                 \n",
      "  36    0.07    0.56    0.28    0.00 *         V  41 A         V  35 A                 \n",
      "  37    0.36    0.50    0.25    0.00           E  42 A         I  36 A                 \n",
      "  38    0.06    0.31    0.15    0.00 *         L  43 A         L  37 A                 \n",
      "  39    0.23    0.41    0.20    0.00           K  44 A         Q  38 A                 \n",
      "  40    0.05    0.55    0.27    0.00 *         L  45 A         L  39 A                 \n",
      "  41    0.05    0.23    0.12    0.00 *         L  46 A         L  40 A                 \n",
      "  42    0.08    0.35    0.17    0.00 *         E  47 A         E  41 A                 \n",
      "  43    0.08    0.39    0.20    0.00 *         I  48 A         I  42 A                 \n",
      "  44    0.04    0.44    0.22    0.00 *         P  49 A         P  43 A                 \n",
      "  45    0.08    0.29    0.15    0.00 *         Q  50 A         Q  44 A                 \n",
      "  46    0.08    0.20    0.10    0.00 *         A  51 A         A  45 A                 \n",
      "  47    0.19    0.44    0.22    0.00           L  52 A         M  46 A                 \n",
      "  48    0.29    0.41    0.20    0.00           G  53 A         K  47 A                 \n",
      "  49    0.22    0.70    0.35    0.00           G  54 A         A  48 A                 \n",
      "  50    0.28    0.76    0.38    0.00           A  55 A         L  49 A                 \n",
      "  51    0.08    0.50    0.25    0.00 *         E  56 A         E  50 A                 \n",
      "  52    0.03    0.44    0.22    0.00 *         G  57 A         G  51 A                 \n",
      "  53    0.07    0.47    0.23    0.00 *         V  58 A         V  52 A                 \n",
      "  54    0.23    0.66    0.33    0.00           A  59 A         V  53 A                 \n",
      "  55    0.12    0.65    0.32    0.00 *         M  60 A         M  54 A                 \n",
      "  56    0.08    0.53    0.27    0.00 *         E  61 A         E  55 A                 \n",
      "  57    0.05    0.41    0.21    0.00 *         L  62 A         L  56 A                 \n",
      "  58    0.35    0.79    0.40    0.00           L  63 A         E  57 A                 \n",
      "  59    0.06    0.85    0.43    0.00 *         D  64 A         D  58 A                 \n",
      "  60    0.40    0.60    0.30    0.00           S  65 A         C  59 A                 \n",
      "  61    0.08    0.77    0.39    0.00 *         A  66 A         A  60 A                 \n",
      "  62    0.06    0.61    0.30    0.00 *         F  67 A         F  61 A                 \n",
      "  63    0.05    0.51    0.26    0.00 *         P  68 A         P  62 A                 \n",
      "  64    0.06    0.67    0.33    0.00 *         L  69 A         L  63 A                 \n",
      "  65    0.06    0.34    0.17    0.00 *         L  70 A         L  64 A                 \n",
      "  66    0.32    1.02    0.51    0.00           R  71 A         A  65 A                 \n",
      "  67    0.28    0.81    0.40    0.00           N  72 A         G  66 A                 \n",
      "  68    0.18    0.66    0.33    0.00           I  73 A         L  67 A                 \n",
      "  69    0.27    0.60    0.30    0.00           T  74 A         E  68 A                 \n",
      "  70    0.29    0.65    0.33    0.00           I  75 A         A  69 A                 \n",
      "  71    0.07    0.63    0.31    0.00 *         T  76 A         T  70 A                 \n",
      "  72    0.26    0.55    0.27    0.00           A  77 A         D  71 A                 \n",
      "  73    0.06    0.47    0.23    0.00 *         D  78 A         D  72 A                 \n",
      "  74    0.25    0.41    0.20    0.00           A  79 A         P  73 A                 \n",
      "  75    0.21    0.36    0.18    0.00           N  80 A         D  74 A                 \n",
      "  76    0.32    0.33    0.16    0.00           E  81 A         V  75 A                 \n",
      "  77    0.08    0.53    0.26    0.00 *         A  82 A         A  76 A                 \n",
      "  78    0.06    0.26    0.13    0.00 *         F  83 A         F  77 A                 \n",
      "  79    0.27    0.43    0.22    0.00           D  84 A         K  78 A                 \n",
      "  80    0.27    0.22    0.11    0.00           G  85 A         D  79 A                 \n",
      "  81    0.08    0.37    0.19    0.00 *         A  86 A         A  80 A                 \n",
      "  82    0.21    0.17    0.08    0.00           N  87 A         D  81 A                 \n",
      "  83    0.34    0.15    0.07    0.00           A  88 A         Y  82 A                 \n",
      "  84    0.09    0.65    0.32    0.00 *         A  89 A         A  83 A                 \n",
      "  85    0.22    0.63    0.31    0.00           F  90 A         L  84 A                 \n",
      "  86    0.06    0.43    0.22    0.00 *         L  91 A         L  85 A                 \n",
      "  87    0.07    0.56    0.28    0.00 *         V  92 A         V  86 A                 \n",
      "  88    0.03    0.54    0.27    0.00 *         G  93 A         G  87 A                 \n",
      "  89    0.11    1.35    0.68    0.00 *         A  94 A         A  88 A                 \n",
      "  90    0.29    2.87    1.44    0.00           K  95 A         A  89 A                 \n",
      "  91    0.20    3.36    1.68    0.00 *         P  96 A         P  90 A                 \n",
      "  92    0.47    0.00    0.00    0.00 *         E 102 A                                 \n",
      "  93    0.00    0.00    0.00    0.00           R 103 A         -                       \n",
      "  94    0.00    0.00    0.00    0.00           A 104 A         -                       \n",
      "  95    0.00    0.00    0.00    0.00           D 105 A         -                       \n",
      "  96    0.00    0.00    0.00    0.00           L 106 A         -                       \n",
      "  97    0.34    3.01    1.51    0.00 *         L 107 A         L 101 A                 \n",
      "  98    0.28    2.99    1.49    0.00           A 108 A         Q 102 A                 \n",
      "  99    0.35    2.81    1.41    0.00           N 109 A         V 103 A                 \n",
      " 100    0.11    2.12    1.06    0.00 *         N 110 A         N 104 A                 \n",
      " 101    0.05    2.67    1.33    0.00 *         G 111 A         G 105 A                 \n",
      " 102    0.10    2.16    1.08    0.00 *         K 112 A         K 106 A                 \n",
      " 103    0.09    0.96    0.48    0.00 *         I 113 A         I 107 A                 \n",
      " 104    0.08    0.94    0.47    0.00 *         F 114 A         F 108 A                 \n",
      " 105    0.32    1.14    0.57    0.00           G 115 A         T 109 A                 \n",
      " 106    0.30    0.75    0.37    0.00           P 116 A         E 110 A                 \n",
      " 107    0.08    0.54    0.27    0.00 *         Q 117 A         Q 111 A                 \n",
      " 108    0.02    0.63    0.31    0.00 *         G 118 A         G 112 A                 \n",
      " 109    0.19    0.54    0.27    0.00           K 119 A         R 113 A                 \n",
      " 110    0.08    0.61    0.30    0.00 *         A 120 A         A 114 A                 \n",
      " 111    0.19    0.60    0.30    0.00           I 121 A         L 115 A                 \n",
      " 112    0.27    0.66    0.33    0.00           N 122 A         A 116 A                 \n",
      " 113    0.21    0.68    0.34    0.00           D 123 A         E 117 A                 \n",
      " 114    0.36    0.52    0.26    0.00           N 124 A         V 118 A                 \n",
      " 115    0.11    0.39    0.20    0.00 *         A 125 A         A 119 A                 \n",
      " 116    0.26    0.78    0.39    0.00           A 126 A         K 120 A                 \n",
      " 117    0.28    1.81    0.90    0.00           D 127 A         K 121 A                 \n",
      " 118    0.08    2.04    1.02    0.00 *         D 128 A         D 122 A                 \n",
      " 119    0.18    0.26    0.13    0.00           I 129 A         V 123 A                 \n",
      " 120    0.21    0.26    0.13    0.00           R 130 A         K 124 A                 \n",
      " 121    0.06    0.38    0.19    0.00 *         V 131 A         V 125 A                 \n",
      " 122    0.05    0.34    0.17    0.00 *         L 132 A         L 126 A                 \n",
      " 123    0.06    0.31    0.16    0.00 *         V 133 A         V 127 A                 \n",
      " 124    0.06    0.39    0.20    0.00 *         V 134 A         V 128 A                 \n",
      " 125    0.03    0.27    0.14    0.00 *         G 135 A         G 129 A                 \n",
      " 126    0.10    0.42    0.21    0.00 *         N 136 A         N 130 A                 \n",
      " 127    0.04    0.41    0.20    0.00 *         P 137 A         P 131 A                 \n",
      " 128    0.07    0.65    0.33    0.00 *         A 138 A         A 132 A                 \n",
      " 129    0.10    0.53    0.27    0.00 *         N 139 A         N 133 A                 \n",
      " 130    0.07    0.39    0.20    0.00 *         T 140 A         T 134 A                 \n",
      " 131    0.10    0.57    0.28    0.00 *         N 141 A         N 135 A                 \n",
      " 132    0.08    0.54    0.27    0.00 *         A 142 A         A 136 A                 \n",
      " 133    0.05    0.58    0.29    0.00 *         L 143 A         L 137 A                 \n",
      " 134    0.07    0.62    0.31    0.00 *         I 144 A         I 138 A                 \n",
      " 135    0.08    0.71    0.36    0.00 *         A 145 A         A 139 A                 \n",
      " 136    0.32    0.55    0.27    0.00           S 146 A         Y 140 A                 \n",
      " 137    0.26    0.57    0.29    0.00           A 147 A         K 141 A                 \n",
      " 138    0.29    0.59    0.30    0.00           A 148 A         N 142 A                 \n",
      " 139    0.09    0.60    0.30    0.00 *         A 149 A         A 143 A                 \n",
      " 140    0.06    1.76    0.88    0.00 *         P 150 A         P 144 A                 \n",
      " 141    0.29    0.49    0.25    0.00           D 151 A         G 145 A                 \n",
      " 142    0.21    0.67    0.34    0.00           V 152 A         L 146 A                 \n",
      " 143    0.31    0.91    0.45    0.00           P 153 A         N 147 A                 \n",
      " 144    0.27    0.92    0.46    0.00           A 154 A         P 148 A                 \n",
      " 145    0.28    0.68    0.34    0.00           S 155 A         R 149 A                 \n",
      " 146    0.29    0.57    0.29    0.00           R 156 A         N 150 A                 \n",
      " 147    0.06    0.56    0.28    0.00 *         F 157 A         F 151 A                 \n",
      " 148    0.26    0.47    0.24    0.00           N 158 A         T 152 A                 \n",
      " 149    0.09    0.36    0.18    0.00 *         A 159 A         A 153 A                 \n",
      " 150    0.13    0.33    0.16    0.00 *         M 160 A         M 154 A                 \n",
      " 151    0.33    0.13    0.06    0.00           M 161 A         T 155 A                 \n",
      " 152    0.07    0.17    0.09    0.00 *         R 162 A         R 156 A                 \n",
      " 153    0.05    0.17    0.08    0.00 *         L 163 A         L 157 A                 \n",
      " 154    0.06    0.29    0.14    0.00 *         D 164 A         D 158 A                 \n",
      " 155    0.07    0.08    0.04    0.00 *         H 165 A         H 159 A                 \n",
      " 156    0.10    0.23    0.11    0.00 *         N 166 A         N 160 A                 \n",
      " 157    0.07    0.26    0.13    0.00 *         R 167 A         R 161 A                 \n",
      " 158    0.08    0.28    0.14    0.00 *         A 168 A         A 162 A                 \n",
      " 159    0.35    0.16    0.08    0.00           I 169 A         K 163 A                 \n",
      " 160    0.21    0.25    0.12    0.00           S 170 A         A 164 A                 \n",
      " 161    0.08    0.46    0.23    0.00 *         Q 171 A         Q 165 A                 \n",
      " 162    0.06    0.41    0.20    0.00 *         L 172 A         L 166 A                 \n",
      " 163    0.08    0.74    0.37    0.00 *         A 173 A         A 167 A                 \n",
      " 164    0.24    0.79    0.39    0.00           T 174 A         K 168 A                 \n",
      " 165    0.08    0.63    0.32    0.00 *         K 175 A         K 169 A                 \n",
      " 166    0.31    0.83    0.41    0.00           L 176 A         T 170 A                 \n",
      " 167    0.04    0.95    0.47    0.00 *         G 177 A         G 171 A                 \n",
      " 168    0.29    0.78    0.39    0.00           R 178 A         T 172 A                 \n",
      " 169    0.03    0.90    0.45    0.00 *         G 179 A         G 173 A                 \n",
      " 170    0.31    1.04    0.52    0.00           S 180 A         V 174 A                 \n",
      " 171    0.27    1.15    0.58    0.00           A 181 A         D 175 A                 \n",
      " 172    0.31    0.34    0.17    0.00           E 182 A         R 176 A                 \n",
      " 173    0.26    0.86    0.43    0.00           F 183 A         I 177 A                 \n",
      " 174    0.30    1.25    0.63    0.00           N 184 A         R 178 A                 \n",
      " 175    0.29    1.15    0.57    0.00           N 185 A         R 179 A                 \n",
      " 176    0.25    0.79    0.39    0.00           I 186 A         M 180 A                 \n",
      " 177    0.26    0.41    0.20    0.00           V 187 A         T 181 A                 \n",
      " 178    0.07    0.23    0.12    0.00 *         V 188 A         V 182 A                 \n",
      " 179    0.06    0.24    0.12    0.00 *         W 189 A         W 183 A                 \n",
      " 180    0.05    0.41    0.20    0.00 *         G 190 A         G 184 A                 \n",
      " 181    0.11    1.65    0.83    0.00 *         N 191 A         N 185 A                 \n",
      " 182    0.08    1.01    0.51    0.00 *         H 192 A         H 186 A                 \n",
      " 183    0.09    0.39    0.20    0.00 *         S 193 A         S 187 A                 \n",
      " 184    0.21    0.23    0.11    0.00           A 194 A         S 188 A                 \n",
      " 185    0.30    0.62    0.31    0.00           T 195 A         I 189 A                 \n",
      " 186    0.35    0.14    0.07    0.00           Q 196 A         M 190 A                 \n",
      " 187    0.06    0.17    0.09    0.00 *         F 197 A         F 191 A                 \n",
      " 188    0.05    0.19    0.10    0.00 *         P 198 A         P 192 A                 \n",
      " 189    0.07    0.35    0.18    0.00 *         D 199 A         D 193 A                 \n",
      " 190    0.21    0.59    0.29    0.00           I 200 A         L 194 A                 \n",
      " 191    0.38    0.56    0.28    0.00           T 201 A         F 195 A                 \n",
      " 192    0.30    0.69    0.35    0.00           Y 202 A         H 196 A                 \n",
      " 193    0.10    0.68    0.34    0.00 *         A 203 A         A 197 A                 \n",
      " 194    0.30    0.85    0.42    0.00           T 204 A         E 198 A                 \n",
      " 195    0.10    1.20    0.60    0.00 *         V 205 A         V 199 A                 \n",
      " 196    0.28    1.81    0.91    0.00           G 206 A         D 200 A                 \n",
      " 197    0.03    2.07    1.03    0.00 *         G 207 A         G 205 A                 \n",
      " 198    0.31    1.15    0.58    0.00           E 208 A         R 206 A                 \n",
      " 199    0.31    0.76    0.38    0.00           K 209 A         P 207 A                 \n",
      " 200    0.23    0.47    0.23    0.00           V 210 A         A 208 A                 \n",
      " 201    0.31    0.59    0.30    0.00           T 211 A         L 209 A                 \n",
      " 202    0.23    0.89    0.45    0.00           D 212 A         E 210 A                 \n",
      " 203    0.10    0.75    0.38    0.00 *         L 213 A         L 211 A                 \n",
      " 204    0.09    0.88    0.44    0.00 *         V 214 A         V 212 A                 \n",
      " 205    0.08    1.50    0.75    0.00 *         D 215 A         D 214 A                 \n",
      " 206    0.40    0.27    0.14    0.00           H 216 A         M 215 A                 \n",
      " 207    0.21    1.18    0.59    0.00           D 217 A         E 216 A                 \n",
      " 208    0.03    1.29    0.64    0.00 *         W 218 A         W 217 A                 \n",
      " 209    0.05    0.60    0.30    0.00 *         Y 219 A         Y 218 A                 \n",
      " 210    0.32    0.74    0.37    0.00           V 220 A         E 219 A                 \n",
      " 211    0.23    1.44    0.72    0.00           E 221 A         K 220 A                 \n",
      " 212    0.32    1.24    0.62    0.00           E 222 A         V 221 A                 \n",
      " 213    0.06    0.63    0.32    0.00 *         F 223 A         F 222 A                 \n",
      " 214    0.07    0.62    0.31    0.00 *         I 224 A         I 223 A                 \n",
      " 215    0.04    1.26    0.63    0.00 *         P 225 A         P 224 A                 \n",
      " 216    0.29    0.93    0.46    0.00           R 226 A         T 225 A                 \n",
      " 217    0.07    0.71    0.36    0.00 *         V 227 A         V 226 A                 \n",
      " 218    0.08    1.38    0.69    0.00 *         A 228 A         A 227 A                 \n",
      " 219    0.29    1.80    0.90    0.00           N 229 A         Q 228 A                 \n",
      " 220    0.08    1.06    0.53    0.00 *         R 230 A         R 229 A                 \n",
      " 221    0.03    1.13    0.56    0.00 *         G 231 A         G 230 A                 \n",
      " 222    0.08    1.38    0.69    0.00 *         A 232 A         A 231 A                 \n",
      " 223    0.25    1.37    0.69    0.00           E 233 A         A 232 A                 \n",
      " 224    0.07    1.22    0.61    0.00 *         I 234 A         I 233 A                 \n",
      " 225    0.09    1.24    0.62    0.00 *         I 235 A         I 234 A                 \n",
      " 226    0.24    1.00    0.50    0.00           E 236 A         Q 235 A                 \n",
      " 227    0.24    0.89    0.44    0.00           V 237 A         A 236 A                 \n",
      " 228    0.08    1.18    0.59    0.00 *         R 238 A         R 237 A                 \n",
      " 229    0.04    1.20    0.60    0.00 *         G 239 A         G 238 A                 \n",
      " 230    0.25    0.84    0.42    0.00           K 240 A         A 239 A                 \n",
      " 231    0.08    1.08    0.54    0.00 *         S 241 A         S 240 A                 \n",
      " 232    0.09    0.81    0.41    0.00 *         S 242 A         S 241 A                 \n",
      " 233    0.10    0.24    0.12    0.00 *         A 243 A         A 242 A                 \n",
      " 234    0.08    0.20    0.10    0.00 *         A 244 A         A 243 A                 \n",
      " 235    0.06    0.20    0.10    0.00 *         S 245 A         S 244 A                 \n",
      " 236    0.08    0.34    0.17    0.00 *         A 246 A         A 245 A                 \n",
      " 237    0.08    0.15    0.07    0.00 *         A 247 A         A 246 A                 \n",
      " 238    0.22    0.30    0.15    0.00           S 248 A         N 247 A                 \n",
      " 239    0.21    0.24    0.12    0.00           S 249 A         A 248 A                 \n",
      " 240    0.08    0.21    0.11    0.00 *         A 250 A         A 249 A                 \n",
      " 241    0.07    0.42    0.21    0.00 *         I 251 A         I 250 A                 \n",
      " 242    0.21    0.47    0.24    0.00           D 252 A         E 251 A                 \n",
      " 243    0.08    0.30    0.15    0.00 *         H 253 A         H 252 A                 \n",
      " 244    0.25    0.31    0.16    0.00           M 254 A         I 253 A                 \n",
      " 245    0.08    0.42    0.21    0.00 *         R 255 A         R 254 A                 \n",
      " 246    0.07    0.38    0.19    0.00 *         D 256 A         D 255 A                 \n",
      " 247    0.03    0.70    0.35    0.00 *         W 257 A         W 256 A                 \n",
      " 248    0.23    0.29    0.15    0.00           V 258 A         A 257 A                 \n",
      " 249    0.33    0.32    0.16    0.00           Q 259 A         L 258 A                 \n",
      " 250    0.05    0.38    0.19    0.00 *         G 260 A         G 259 A                 \n",
      " 251    0.10    0.57    0.29    0.00 *         T 261 A         T 260 A                 \n",
      " 252    0.33    3.03    1.51    0.00           E 262 A         P 261 A                 \n",
      " 253    0.35    6.28    3.14    0.00           A 263 A         E 262 A                 \n",
      " 254    0.00    0.00    0.00    0.00           -               G 263 A                 \n",
      " 255    0.00    0.00    0.00    0.00           -               D 264 A                 \n",
      " 256    0.11    1.39    0.70    0.00 *         W 264 A         W 265 A                 \n",
      " 257    0.37    0.87    0.43    0.00           S 265 A         V 266 A                 \n",
      " 258    0.11    0.42    0.21    0.00 *         S 266 A         S 267 A                 \n",
      " 259    0.34    0.62    0.31    0.00           A 267 A         M 268 A                 \n",
      " 260    0.10    0.45    0.23    0.00 *         A 268 A         A 269 A                 \n",
      " 261    0.16    0.54    0.27    0.00           I 269 A         V 270 A                 \n",
      " 262    0.04    0.58    0.29    0.00 *         P 270 A         P 271 A                 \n",
      " 263    0.07    0.61    0.31    0.00 *         S 271 A         S 272 A                 \n",
      " 264    0.30    0.90    0.45    0.00           T 272 A         Q 273 A                 \n",
      " 265    0.03    0.51    0.26    0.00 *         G 273 A         G 274 A                 \n",
      " 266    0.26    0.82    0.41    0.00           A 274 A         E 275 A                 \n",
      " 267    0.06    0.61    0.30    0.00 *         Y 275 A         Y 277 A                 \n",
      " 268    0.03    0.94    0.47    0.00 *         G 276 A         G 278 A                 \n",
      " 269    0.10    0.49    0.24    0.00 *         I 277 A         I 279 A                 \n",
      " 270    0.07    0.44    0.22    0.00 *         P 278 A         P 280 A                 \n",
      " 271    0.10    0.53    0.26    0.00 *         E 279 A         E 281 A                 \n",
      " 272    0.05    1.35    0.67    0.00 *         G 280 A         G 282 A                 \n",
      " 273    0.10    0.50    0.25    0.00 *         I 281 A         I 283 A                 \n",
      " 274    0.31    0.62    0.31    0.00           F 282 A         V 284 A                 \n",
      " 275    0.33    0.73    0.36    0.00           V 283 A         Y 285 A                 \n",
      " 276    0.25    0.76    0.38    0.00           G 284 A         S 286 A                 \n",
      " 277    0.23    0.35    0.18    0.00           L 285 A         F 287 A                 \n",
      " 278    0.06    0.95    0.48    0.00 *         P 286 A         P 288 A                 \n",
      " 279    0.27    0.70    0.35    0.00           T 287 A         V 289 A                 \n",
      " 280    0.27    0.93    0.46    0.00           V 288 A         T 290 A                 \n",
      " 281    0.21    0.69    0.35    0.00           S 289 A         A 291 A                 \n",
      " 282    0.20    1.52    0.76    0.00           R 290 A         K 292 A                 \n",
      " 283    0.21    2.56    1.28    0.00           N 291 A         D 293 A                 \n",
      " 284    0.03    0.71    0.35    0.00 *         G 292 A         G 294 A                 \n",
      " 285    0.29    1.20    0.60    0.00           E 293 A         A 295 A                 \n",
      " 286    0.29    1.54    0.77    0.00           W 294 A         Y 296 A                 \n",
      " 287    0.31    0.84    0.42    0.00           E 295 A         R 297 A                 \n",
      " 288    0.16    0.56    0.28    0.00           I 296 A         V 298 A                 \n",
      " 289    0.07    0.73    0.37    0.00 *         V 297 A         V 299 A                 \n",
      " 290    0.08    1.44    0.72    0.00 *         E 298 A         E 300 A                 \n",
      " 291    0.03    1.17    0.58    0.00 *         G 299 A         G 301 A                 \n",
      " 292    0.08    0.93    0.47    0.00 *         L 300 A         L 302 A                 \n",
      " 293    0.10    1.25    0.62    0.00 *         E 301 A         E 303 A                 \n",
      " 294    0.09    0.93    0.47    0.00 *         I 302 A         I 304 A                 \n",
      " 295    0.22    0.52    0.26    0.00           S 303 A         N 305 A                 \n",
      " 296    0.22    0.90    0.45    0.00           D 304 A         E 306 A                 \n",
      " 297    0.06    0.73    0.36    0.00 *         F 305 A         F 307 A                 \n",
      " 298    0.27    0.73    0.37    0.00           Q 306 A         A 308 A                 \n",
      " 299    0.07    0.64    0.32    0.00 *         R 307 A         R 309 A                 \n",
      " 300    0.25    0.26    0.13    0.00           A 308 A         K 310 A                 \n",
      " 301    0.08    0.32    0.16    0.00 *         R 309 A         R 311 A                 \n",
      " 302    0.25    0.30    0.15    0.00           I 310 A         M 312 A                 \n",
      " 303    0.22    0.39    0.20    0.00           D 311 A         E 313 A                 \n",
      " 304    0.30    0.46    0.23    0.00           A 312 A         I 314 A                 \n",
      " 305    0.25    0.77    0.38    0.00           N 313 A         T 315 A                 \n",
      " 306    0.08    0.42    0.21    0.00 *         A 314 A         A 316 A                 \n",
      " 307    0.07    0.18    0.09    0.00 *         Q 315 A         Q 317 A                 \n",
      " 308    0.08    0.76    0.38    0.00 *         E 316 A         E 318 A                 \n",
      " 309    0.05    0.43    0.22    0.00 *         L 317 A         L 319 A                 \n",
      " 310    0.32    0.45    0.22    0.00           Q 318 A         L 320 A                 \n",
      " 311    0.26    1.18    0.59    0.00           A 319 A         D 321 A                 \n",
      " 312    0.08    1.46    0.73    0.00 *         E 320 A         E 322 A                 \n",
      " 313    0.41    1.11    0.56    0.00           R 321 A         M 323 A                 \n",
      " 314    0.09    1.52    0.76    0.00 *         E 322 A         E 324 A                 \n",
      " 315    0.28    1.92    0.96    0.00           A 323 A         Q 325 A                 \n",
      " 316    0.16    1.68    0.84    0.00 *         V 324 A         V 326 A                 \n",
      " 317    0.29    1.66    0.83    0.00           R 325 A         K 327 A                 \n",
      " 318    0.28    4.53    2.27    0.00           D 326 A         A 328 A                 \n",
      " 319    0.14    5.37    2.69    0.00 *         L 327 A         L 329 A                 \n",
      " 320    0.39    5.79    2.90    0.00           L 328 A         G 330 A                 \n",
      " 321    0.09    5.01    2.51    0.00 *         L 329 A         L 331 A                 \n",
      " 322    0.36    5.18    2.59    0.00           E 330 A         I 332 A                 \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    2\n",
      "\n",
      "Current alignment total score:        68.03\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       2       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -               M   0 A                 \n",
      "   2    0.31    0.92    0.46    0.00   K   3 A -               K   1 A                 \n",
      "   3    0.26    1.28    0.69    0.27   P   4 A S   8 A         A   2 A                 \n",
      "   4    0.25    0.63    0.44    0.09   A   5 A T   9 A         P   3 A                 \n",
      "   5    0.20    0.17    0.11    0.02   K   6 A K  10 A         V   4 A                 \n",
      "   6    0.14    0.29    0.15    0.04   R   7 A K  11 A         R   5 A                 \n",
      "   7    0.08    0.29    0.17    0.03 * V   8 A V  12 A         V   6 A                 \n",
      "   8    0.16    0.30    0.20    0.04   A   9 A T  13 A         A   7 A                 \n",
      "   9    0.06    0.33    0.25    0.04 * V  10 A V  14 A         V   8 A                 \n",
      "  10    0.07    0.28    0.17    0.03 * T  11 A T  15 A         T   9 A                 \n",
      "  11    0.03    0.20    0.18    0.05 * G  12 A G  16 A         G  10 A                 \n",
      "  12    0.08    0.49    0.31    0.10 * A  13 A A  17 A         A  11 A                 \n",
      "  13    0.08    0.47    0.32    0.08 * A  14 A A  18 A         A  12 A                 \n",
      "  14    0.03    0.29    0.14    0.06 * G  15 A G  19 A         G  13 A                 \n",
      "  15    0.09    0.15    0.16    0.05 * Q  16 A Q  20 A         Q  14 A                 \n",
      "  16    0.08    0.30    0.14    0.06 * I  17 A I  21 A         I  15 A                 \n",
      "  17    0.22    0.15    0.22    0.08   A  18 A S  22 A         G  16 A                 \n",
      "  18    0.05    0.16    0.15    0.04 * Y  19 A Y  23 A         Y  17 A                 \n",
      "  19    0.07    0.22    0.21    0.06 * S  20 A S  24 A         S  18 A                 \n",
      "  20    0.06    0.20    0.18    0.04 * L  21 A L  25 A         L  19 A                 \n",
      "  21    0.06    0.29    0.21    0.06 * L  22 A L  26 A         L  20 A                 \n",
      "  22    0.18    0.42    0.37    0.12   F  23 A W  27 A         F  21 A                 \n",
      "  23    0.08    0.48    0.49    0.18 * R  24 A R  28 A         R  22 A                 \n",
      "  24    0.08    0.53    0.42    0.12 * I  25 A I  29 A         I  23 A                 \n",
      "  25    0.08    0.40    0.27    0.10 * A  26 A A  30 A         A  24 A                 \n",
      "  26    0.19    0.35    0.39    0.14   N  27 A N  31 A         A  25 A                 \n",
      "  27    0.03    0.46    0.45    0.14 * G  28 A G  32 A         G  26 A                 \n",
      "  28    0.21    0.68    0.59    0.14   D  29 A E  33 A         E  27 A                 \n",
      "  29    0.19    0.69    0.65    0.14   L  30 A V  34 A         M  28 A                 \n",
      "  30    0.14    0.42    0.61    0.23   L  31 A F  35 A         L  29 A                 \n",
      "  31    0.03    0.62    0.62    0.18 * G  32 A G  36 A         G  30 A                 \n",
      "  32    0.17    0.78    0.50    0.18   K  33 A T  37 A         K  31 A                 \n",
      "  33    0.08    0.79    0.54    0.19 * D  34 A D  38 A         D  32 A                 \n",
      "  34    0.20    0.27    0.41    0.17   Q  35 A T  39 A         Q  33 A                 \n",
      "  35    0.04    0.31    0.43    0.17 * P  36 A P  40 A         P  34 A                 \n",
      "  36    0.07    0.12    0.22    0.10 * V  37 A V  41 A         V  35 A                 \n",
      "  37    0.22    0.29    0.24    0.05   I  38 A E  42 A         I  36 A                 \n",
      "  38    0.06    0.20    0.15    0.04 * L  39 A L  43 A         L  37 A                 \n",
      "  39    0.16    0.11    0.16    0.06   Q  40 A K  44 A         Q  38 A                 \n",
      "  40    0.06    0.47    0.31    0.01 * L  41 A L  45 A         L  39 A                 \n",
      "  41    0.06    0.35    0.16    0.10 * L  42 A L  46 A         L  40 A                 \n",
      "  42    0.22    0.57    0.29    0.07   D  43 A E  47 A         E  41 A                 \n",
      "  43    0.20    0.57    0.31    0.05   L  44 A I  48 A         I  42 A                 \n",
      "  44    0.06    0.37    0.25    0.04 * P  45 A P  49 A         P  43 A                 \n",
      "  45    0.09    0.92    0.43    0.13 * Q  46 A Q  50 A         Q  44 A                 \n",
      "  46    0.09    0.53    0.25    0.09 * A  47 A A  51 A         A  45 A                 \n",
      "  47    0.33    0.46    0.28    0.05   Q  48 A L  52 A         M  46 A                 \n",
      "  48    0.24    0.61    0.32    0.08   A  49 A G  53 A         K  47 A                 \n",
      "  49    0.15    0.40    0.34    0.05   A  50 A G  54 A         A  48 A                 \n",
      "  50    0.21    0.27    0.32    0.10   V  51 A A  55 A         L  49 A                 \n",
      "  51    0.24    0.47    0.30    0.02   K  52 A E  56 A         E  50 A                 \n",
      "  52    0.03    0.29    0.21    0.07 * G  53 A G  57 A         G  51 A                 \n",
      "  53    0.07    0.46    0.26    0.12 * V  54 A V  58 A         V  52 A                 \n",
      "  54    0.15    0.69    0.41    0.11   V  55 A A  59 A         V  53 A                 \n",
      "  55    0.12    0.57    0.35    0.13 * M  56 A M  60 A         M  54 A                 \n",
      "  56    0.09    0.43    0.28    0.10 * E  57 A E  61 A         E  55 A                 \n",
      "  57    0.05    0.51    0.26    0.14 * L  58 A L  62 A         L  56 A                 \n",
      "  58    0.30    0.68    0.43    0.16   D  59 A L  63 A         E  57 A                 \n",
      "  59    0.07    0.73    0.45    0.19 * D  60 A D  64 A         D  58 A                 \n",
      "  60    0.21    0.53    0.32    0.14   C  61 A S  65 A         C  59 A                 \n",
      "  61    0.09    0.65    0.42    0.15 * A  62 A A  66 A         A  60 A                 \n",
      "  62    0.06    0.37    0.30    0.07 * F  63 A F  67 A         F  61 A                 \n",
      "  63    0.05    0.35    0.26    0.04 * P  64 A P  68 A         P  62 A                 \n",
      "  64    0.06    0.35    0.31    0.09 * L  65 A L  69 A         L  63 A                 \n",
      "  65    0.06    0.21    0.16    0.05 * L  66 A L  70 A         L  64 A                 \n",
      "  66    0.20    0.28    0.41    0.16   A  67 A R  71 A         A  65 A                 \n",
      "  67    0.15    0.42    0.37    0.10   G  68 A N  72 A         G  66 A                 \n",
      "  68    0.18    0.43    0.33    0.05   V  69 A I  73 A         L  67 A                 \n",
      "  69    0.29    0.71    0.40    0.10   V  70 A T  74 A         E  68 A                 \n",
      "  70    0.19    0.38    0.31    0.06   I  71 A I  75 A         A  69 A                 \n",
      "  71    0.07    0.27    0.27    0.09 * T  72 A T  76 A         T  70 A                 \n",
      "  72    0.16    0.29    0.25    0.06   D  73 A A  77 A         D  71 A                 \n",
      "  73    0.07    0.49    0.30    0.03 * D  74 A D  78 A         D  72 A                 \n",
      "  74    0.15    0.48    0.28    0.05   P  75 A A  79 A         P  73 A                 \n",
      "  75    0.27    0.55    0.28    0.11   K  76 A N  80 A         D  74 A                 \n",
      "  76    0.20    0.50    0.26    0.08   V  77 A E  81 A         V  75 A                 \n",
      "  77    0.09    0.56    0.31    0.14 * A  78 A A  82 A         A  76 A                 \n",
      "  78    0.07    0.32    0.17    0.06 * F  79 A F  83 A         F  77 A                 \n",
      "  79    0.18    0.30    0.22    0.03   K  80 A D  84 A         K  78 A                 \n",
      "  80    0.17    0.41    0.21    0.05   D  81 A G  85 A         D  79 A                 \n",
      "  81    0.08    0.40    0.23    0.07 * A  82 A A  86 A         A  80 A                 \n",
      "  82    0.15    0.32    0.16    0.04   D  83 A N  87 A         D  81 A                 \n",
      "  83    0.29    0.22    0.12    0.03   V  84 A A  88 A         Y  82 A                 \n",
      "  84    0.09    0.34    0.29    0.10 * A  85 A A  89 A         A  83 A                 \n",
      "  85    0.14    0.33    0.29    0.08   L  86 A F  90 A         L  84 A                 \n",
      "  86    0.06    0.15    0.18    0.06 * L  87 A L  91 A         L  85 A                 \n",
      "  87    0.07    0.25    0.25    0.07 * V  88 A V  92 A         V  86 A                 \n",
      "  88    0.03    0.32    0.26    0.07 * G  89 A G  93 A         G  87 A                 \n",
      "  89    0.10    0.24    0.51    0.25 * A  90 A A  94 A         A  88 A                 \n",
      "  90    0.44    0.60    1.10    0.50   R  91 A K  95 A         A  89 A                 \n",
      "  91    0.32    1.25    1.42    0.45 * P  92 A P  96 A         P  90 A                 \n",
      "  92    0.46    6.88    3.44    0.00   R  93 A E 102 A                                 \n",
      "  93    0.00    0.00    0.00    0.00   S  94 A -               -                       \n",
      "  94    0.00    0.00    0.00    0.00   M  97 A -               -                       \n",
      "  95    0.00    0.00    0.00    0.00   E  98 A -               -                       \n",
      "  96    0.56    1.91    0.95    0.00   R  99 A R 103 A         -                       \n",
      "  97    0.42    2.27    1.14    0.00   K 100 A A 104 A         -                       \n",
      "  98    0.32    1.96    0.98    0.00   D 101 A D 105 A         -                       \n",
      "  99    0.32    1.13    0.57    0.00   L 102 A L 106 A         -                       \n",
      " 100    0.29    0.71    1.17    0.51 * L 103 A L 107 A         L 101 A                 \n",
      " 101    0.26    0.70    1.15    0.51   S 104 A A 108 A         Q 102 A                 \n",
      " 102    0.28    0.45    1.04    0.53   A 105 A N 109 A         V 103 A                 \n",
      " 103    0.13    0.64    0.85    0.34 * N 106 A N 110 A         N 104 A                 \n",
      " 104    0.24    0.62    1.03    0.46   A 107 A G 111 A         G 105 A                 \n",
      " 105    0.25    0.35    0.80    0.41   E 108 A K 112 A         K 106 A                 \n",
      " 106    0.09    0.19    0.36    0.17 * I 109 A I 113 A         I 107 A                 \n",
      " 107    0.08    0.32    0.39    0.13 * F 110 A F 114 A         F 108 A                 \n",
      " 108    0.20    0.26    0.44    0.19   T 111 A G 115 A         T 109 A                 \n",
      " 109    0.33    0.50    0.38    0.06   V 112 A P 116 A         E 110 A                 \n",
      " 110    0.09    0.29    0.25    0.06 * Q 113 A Q 117 A         Q 111 A                 \n",
      " 111    0.04    0.07    0.23    0.13 * G 114 A G 118 A         G 112 A                 \n",
      " 112    0.29    0.61    0.36    0.04   A 115 A K 119 A         R 113 A                 \n",
      " 113    0.09    0.48    0.33    0.01 * A 116 A A 120 A         A 114 A                 \n",
      " 114    0.13    0.36    0.30    0.04   L 117 A I 121 A         L 115 A                 \n",
      " 115    0.19    0.35    0.30    0.10   N 118 A N 122 A         A 116 A                 \n",
      " 116    0.15    0.53    0.36    0.12   E 119 A D 123 A         E 117 A                 \n",
      " 117    0.22    0.45    0.29    0.05   V 120 A N 124 A         V 118 A                 \n",
      " 118    0.11    0.49    0.27    0.06 * A 121 A A 125 A         A 119 A                 \n",
      " 119    0.26    0.49    0.39    0.08   S 122 A A 126 A         K 120 A                 \n",
      " 120    0.28    0.89    0.80    0.27   R 123 A D 127 A         K 121 A                 \n",
      " 121    0.08    0.48    0.79    0.35 * D 124 A D 128 A         D 122 A                 \n",
      " 122    0.13    0.36    0.20    0.04   V 125 A I 129 A         V 123 A                 \n",
      " 123    0.15    0.34    0.18    0.06   K 126 A R 130 A         K 124 A                 \n",
      " 124    0.07    0.37    0.22    0.07 * V 127 A V 131 A         V 125 A                 \n",
      " 125    0.05    0.29    0.19    0.06 * L 128 A L 132 A         L 126 A                 \n",
      " 126    0.07    0.06    0.12    0.06 * V 129 A V 133 A         V 127 A                 \n",
      " 127    0.07    0.11    0.16    0.06 * V 130 A V 134 A         V 128 A                 \n",
      " 128    0.03    0.34    0.19    0.06 * G 131 A G 135 A         G 129 A                 \n",
      " 129    0.11    0.69    0.31    0.19 * N 132 A N 136 A         N 130 A                 \n",
      " 130    0.05    0.34    0.22    0.07 * P 133 A P 137 A         P 131 A                 \n",
      " 131    0.09    0.32    0.29    0.10 * A 134 A A 138 A         A 132 A                 \n",
      " 132    0.10    0.18    0.22    0.07 * N 135 A N 139 A         N 133 A                 \n",
      " 133    0.08    0.11    0.16    0.06 * T 136 A T 140 A         T 134 A                 \n",
      " 134    0.11    0.23    0.25    0.07 * N 137 A N 141 A         N 135 A                 \n",
      " 135    0.08    0.40    0.29    0.05 * A 138 A A 142 A         A 136 A                 \n",
      " 136    0.30    0.39    0.28    0.10   Y 139 A L 143 A         L 137 A                 \n",
      " 137    0.08    0.32    0.28    0.08 * I 140 A I 144 A         I 138 A                 \n",
      " 138    0.08    0.35    0.33    0.07 * A 141 A A 145 A         A 139 A                 \n",
      " 139    0.36    0.51    0.32    0.08   M 142 A S 146 A         Y 140 A                 \n",
      " 140    0.18    0.44    0.31    0.02   K 143 A A 147 A         K 141 A                 \n",
      " 141    0.23    0.53    0.35    0.01   S 144 A A 148 A         N 142 A                 \n",
      " 142    0.09    0.50    0.32    0.10 * A 145 A A 149 A         A 143 A                 \n",
      " 143    0.06    1.32    0.88    0.35 * P 146 A P 150 A         P 144 A                 \n",
      " 144    0.19    0.37    0.25    0.09   D 147 A D 151 A         G 145 A                 \n",
      " 145    0.16    0.51    0.37    0.02   L 148 A V 152 A         L 146 A                 \n",
      " 146    0.20    0.66    0.48    0.08   P 149 A P 153 A         N 147 A                 \n",
      " 147    0.29    0.74    0.50    0.10   K 150 A A 154 A         P 148 A                 \n",
      " 148    0.24    0.62    0.39    0.09   K 151 A S 155 A         R 149 A                 \n",
      " 149    0.22    0.40    0.29    0.05   N 152 A R 156 A         N 150 A                 \n",
      " 150    0.09    0.27    0.25    0.06 * F 153 A F 157 A         F 151 A                 \n",
      " 151    0.17    0.20    0.21    0.05   T 154 A N 158 A         T 152 A                 \n",
      " 152    0.09    0.10    0.14    0.06 * A 155 A A 159 A         A 153 A                 \n",
      " 153    0.12    0.18    0.16    0.03 * M 156 A M 160 A         M 154 A                 \n",
      " 154    0.26    0.54    0.24    0.08   L 157 A M 161 A         T 155 A                 \n",
      " 155    0.08    0.40    0.18    0.08 * R 158 A R 162 A         R 156 A                 \n",
      " 156    0.05    0.54    0.24    0.10 * L 159 A L 163 A         L 157 A                 \n",
      " 157    0.06    0.43    0.22    0.08 * D 160 A D 164 A         D 158 A                 \n",
      " 158    0.07    0.39    0.17    0.06 * H 161 A H 165 A         H 159 A                 \n",
      " 159    0.10    0.40    0.20    0.06 * N 162 A N 166 A         N 160 A                 \n",
      " 160    0.07    0.39    0.20    0.06 * R 163 A R 167 A         R 161 A                 \n",
      " 161    0.09    0.14    0.13    0.04 * A 164 A A 168 A         A 162 A                 \n",
      " 162    0.25    0.13    0.08    0.02   L 165 A I 169 A         K 163 A                 \n",
      " 163    0.14    0.50    0.23    0.11   S 166 A S 170 A         A 164 A                 \n",
      " 164    0.08    0.68    0.36    0.09 * Q 167 A Q 171 A         Q 165 A                 \n",
      " 165    0.07    0.52    0.29    0.08 * L 168 A L 172 A         L 166 A                 \n",
      " 166    0.08    0.65    0.41    0.12 * A 169 A A 173 A         A 167 A                 \n",
      " 167    0.24    0.92    0.50    0.21   A 170 A T 174 A         K 168 A                 \n",
      " 168    0.08    0.96    0.48    0.19 * K 171 A K 175 A         K 169 A                 \n",
      " 169    0.26    1.16    0.57    0.30   S 172 A L 176 A         T 170 A                 \n",
      " 170    0.04    1.06    0.57    0.27 * G 173 A G 177 A         G 171 A                 \n",
      " 171    0.24    1.28    0.64    0.22   K 174 A R 178 A         T 172 A                 \n",
      " 172    0.32    0.98    0.58    0.11   P 175 A G 179 A         G 173 A                 \n",
      " 173    0.20    0.54    0.48    0.10   V 176 A S 180 A         V 174 A                 \n",
      " 174    0.18    0.45    0.49    0.14   A 177 A A 181 A         D 175 A                 \n",
      " 175    0.29    0.92    0.44    0.12   S 178 A E 182 A         R 176 A                 \n",
      " 176    0.18    0.66    0.47    0.06   I 179 A F 183 A         I 177 A                 \n",
      " 177    0.29    0.83    0.63    0.14   E 180 A N 184 A         R 178 A                 \n",
      " 178    0.22    0.91    0.62    0.14   K 181 A N 185 A         R 179 A                 \n",
      " 179    0.19    1.05    0.55    0.22   L 182 A I 186 A         M 180 A                 \n",
      " 180    0.26    0.87    0.43    0.11   A 183 A V 187 A         T 181 A                 \n",
      " 181    0.09    0.88    0.41    0.13 * V 184 A V 188 A         V 182 A                 \n",
      " 182    0.08    0.23    0.13    0.05 * W 185 A W 189 A         W 183 A                 \n",
      " 183    0.06    0.08    0.16    0.07 * G 186 A G 190 A         G 184 A                 \n",
      " 184    0.12    0.57    0.68    0.26 * N 187 A N 191 A         N 185 A                 \n",
      " 185    0.09    0.61    0.50    0.09 * H 188 A H 192 A         H 186 A                 \n",
      " 186    0.10    0.79    0.37    0.16 * S 189 A S 193 A         S 187 A                 \n",
      " 187    0.27    1.00    0.46    0.15   P 190 A A 194 A         S 188 A                 \n",
      " 188    0.19    0.86    0.47    0.07   T 191 A T 195 A         I 189 A                 \n",
      " 189    0.25    0.63    0.28    0.10   M 192 A Q 196 A         M 190 A                 \n",
      " 190    0.20    0.34    0.17    0.04   Y 193 A F 197 A         F 191 A                 \n",
      " 191    0.05    0.12    0.09    0.02 * P 194 A P 198 A         P 192 A                 \n",
      " 192    0.08    0.27    0.18    0.06 * D 195 A D 199 A         D 193 A                 \n",
      " 193    0.26    0.95    0.47    0.20   F 196 A I 200 A         L 194 A                 \n",
      " 194    0.36    0.76    0.40    0.15   R 197 A T 201 A         F 195 A                 \n",
      " 195    0.28    0.79    0.46    0.10   F 198 A Y 202 A         H 196 A                 \n",
      " 196    0.09    0.64    0.40    0.08 * A 199 A A 203 A         A 197 A                 \n",
      " 197    0.19    0.74    0.48    0.08   T 200 A T 204 A         E 198 A                 \n",
      " 198    0.26    1.24    0.72    0.25   A 201 A V 205 A         V 199 A                 \n",
      " 199    0.27    1.92    1.15    0.22   E 202 A G 206 A         D 200 A                 \n",
      " 200    0.05    1.66    1.14    0.15 * G 203 A G 207 A         G 205 A                 \n",
      " 201    0.22    1.18    0.72    0.06   E 204 A E 208 A         R 206 A                 \n",
      " 202    0.28    1.15    0.62    0.11   S 205 A K 209 A         P 207 A                 \n",
      " 203    0.24    0.75    0.39    0.08   L 206 A V 210 A         A 208 A                 \n",
      " 204    0.20    0.79    0.41    0.16   L 207 A T 211 A         L 209 A                 \n",
      " 205    0.27    0.89    0.52    0.19   K 208 A D 212 A         E 210 A                 \n",
      " 206    0.17    1.65    0.76    0.36 * L 209 A L 213 A         L 211 A                 \n",
      " 207    0.25    1.73    0.81    0.37   I 210 A V 214 A         V 212 A                 \n",
      " 208    0.00    0.00    0.00    0.00   N 211 A -               -                       \n",
      " 209    0.16    0.70    0.67    0.20 * D 212 A D 215 A         D 214 A                 \n",
      " 210    0.42    0.41    0.20    0.11   D 213 A H 216 A         M 215 A                 \n",
      " 211    0.35    0.54    0.53    0.12   V 214 A D 217 A         E 216 A                 \n",
      " 212    0.05    0.48    0.55    0.16 * W 215 A W 218 A         W 217 A                 \n",
      " 213    0.33    0.84    0.46    0.07   N 216 A Y 219 A         Y 218 A                 \n",
      " 214    0.33    0.96    0.54    0.09   R 217 A V 220 A         E 219 A                 \n",
      " 215    0.25    0.46    0.59    0.20   D 218 A E 221 A         K 220 A                 \n",
      " 216    0.28    0.51    0.54    0.16   T 219 A E 222 A         V 221 A                 \n",
      " 217    0.06    0.32    0.29    0.08 * F 220 A F 223 A         F 222 A                 \n",
      " 218    0.08    0.42    0.30    0.10 * I 221 A I 224 A         I 223 A                 \n",
      " 219    0.05    0.60    0.56    0.19 * P 222 A P 225 A         P 224 A                 \n",
      " 220    0.19    0.53    0.43    0.15   T 223 A R 226 A         T 225 A                 \n",
      " 221    0.07    0.51    0.35    0.13 * V 224 A V 227 A         V 226 A                 \n",
      " 222    0.23    0.50    0.58    0.18   G 225 A A 228 A         A 227 A                 \n",
      " 223    0.25    0.39    0.69    0.31   K 226 A N 229 A         Q 228 A                 \n",
      " 224    0.08    0.41    0.44    0.16 * R 227 A R 230 A         R 229 A                 \n",
      " 225    0.03    0.38    0.47    0.16 * G 228 A G 231 A         G 230 A                 \n",
      " 226    0.08    0.49    0.58    0.18 * A 229 A A 232 A         A 231 A                 \n",
      " 227    0.17    0.35    0.54    0.23   A 230 A E 233 A         A 232 A                 \n",
      " 228    0.08    0.12    0.43    0.25 * I 231 A I 234 A         I 233 A                 \n",
      " 229    0.09    0.27    0.47    0.22 * I 232 A I 235 A         I 234 A                 \n",
      " 230    0.17    0.25    0.39    0.16   E 233 A E 236 A         Q 235 A                 \n",
      " 231    0.16    0.20    0.34    0.15   A 234 A V 237 A         A 236 A                 \n",
      " 232    0.08    0.33    0.47    0.18 * R 235 A R 238 A         R 237 A                 \n",
      " 233    0.04    0.19    0.44    0.23 * G 236 A G 239 A         G 238 A                 \n",
      " 234    0.30    0.22    0.33    0.13   L 237 A K 240 A         A 239 A                 \n",
      " 235    0.07    0.45    0.47    0.14 * S 238 A S 241 A         S 240 A                 \n",
      " 236    0.08    0.28    0.34    0.12 * S 239 A S 242 A         S 241 A                 \n",
      " 237    0.10    0.45    0.22    0.06 * A 240 A A 243 A         A 242 A                 \n",
      " 238    0.09    0.44    0.20    0.09 * A 241 A A 244 A         A 243 A                 \n",
      " 239    0.07    0.47    0.23    0.06 * S 242 A S 245 A         S 244 A                 \n",
      " 240    0.08    0.30    0.20    0.03 * A 243 A A 246 A         A 245 A                 \n",
      " 241    0.08    0.29    0.14    0.06 * A 244 A A 247 A         A 246 A                 \n",
      " 242    0.16    0.37    0.19    0.10   N 245 A S 248 A         N 247 A                 \n",
      " 243    0.15    0.41    0.20    0.08   A 246 A S 249 A         A 248 A                 \n",
      " 244    0.08    0.32    0.16    0.08 * A 247 A A 250 A         A 249 A                 \n",
      " 245    0.08    0.40    0.25    0.05 * I 248 A I 251 A         I 250 A                 \n",
      " 246    0.14    0.37    0.26    0.01   D 249 A D 252 A         E 251 A                 \n",
      " 247    0.08    0.36    0.21    0.03 * H 250 A H 253 A         H 252 A                 \n",
      " 248    0.22    0.41    0.22    0.07   V 251 A M 254 A         I 253 A                 \n",
      " 249    0.08    0.21    0.19    0.04 * R 252 A R 255 A         R 254 A                 \n",
      " 250    0.07    0.14    0.16    0.05 * D 253 A D 256 A         D 255 A                 \n",
      " 251    0.03    0.17    0.27    0.12 * W 254 A W 257 A         W 256 A                 \n",
      " 252    0.16    0.34    0.18    0.09   V 255 A V 258 A         A 257 A                 \n",
      " 253    0.20    0.25    0.17    0.01   L 256 A Q 259 A         L 258 A                 \n",
      " 254    0.08    0.67    0.34    0.07 * G 257 A G 260 A         G 259 A                 \n",
      " 255    0.12    0.20    0.24    0.09 * T 258 A T 261 A         T 260 A                 \n",
      " 256    0.00    0.00    1.51    0.00   -       E 262 A         P 261 A                 \n",
      " 257    0.38    3.83    3.07    0.66   N 259 A A 263 A         E 262 A                 \n",
      " 258    0.30    3.98    1.99    0.00   G 260 A -               G 263 A                 \n",
      " 259    0.46    1.44    0.72    0.00   K 261 A -               D 264 A                 \n",
      " 260    0.09    0.62    0.62    0.17 * W 262 A W 264 A         W 265 A                 \n",
      " 261    0.23    0.66    0.44    0.18   V 263 A S 265 A         V 266 A                 \n",
      " 262    0.20    0.36    0.23    0.06   T 264 A S 266 A         S 267 A                 \n",
      " 263    0.23    0.18    0.25    0.09   M 265 A A 267 A         M 268 A                 \n",
      " 264    0.23    0.31    0.23    0.02   G 266 A A 268 A         A 269 A                 \n",
      " 265    0.12    0.12    0.21    0.09   I 267 A I 269 A         V 270 A                 \n",
      " 266    0.05    0.15    0.23    0.10 * P 268 A P 270 A         P 271 A                 \n",
      " 267    0.07    0.31    0.28    0.06 * S 269 A S 271 A         S 272 A                 \n",
      " 268    0.31    0.42    0.40    0.12   D 270 A T 272 A         Q 273 A                 \n",
      " 269    0.05    0.12    0.20    0.09 * G 271 A G 273 A         G 274 A                 \n",
      " 270    0.29    1.35    0.66    0.30   S 272 A A 274 A         E 275 A                 \n",
      " 271    0.09    1.06    0.54    0.16 * Y 273 A Y 275 A         Y 277 A                 \n",
      " 272    0.04    1.80    0.87    0.34 * G 274 A G 276 A         G 278 A                 \n",
      " 273    0.10    0.77    0.41    0.08 * I 275 A I 277 A         I 279 A                 \n",
      " 274    0.07    0.65    0.34    0.10 * P 276 A P 278 A         P 280 A                 \n",
      " 275    0.13    0.64    0.35    0.12 * E 277 A E 279 A         E 281 A                 \n",
      " 276    0.30    1.80    0.86    0.53   D 278 A G 280 A         G 282 A                 \n",
      " 277    0.10    1.17    0.57    0.15 * I 279 A I 281 A         I 283 A                 \n",
      " 278    0.23    0.21    0.26    0.08   I 280 A F 282 A         V 284 A                 \n",
      " 279    0.21    0.35    0.33    0.07   Y 281 A V 283 A         Y 285 A                 \n",
      " 280    0.15    0.28    0.32    0.10   G 282 A G 284 A         S 286 A                 \n",
      " 281    0.27    0.27    0.19    0.03   V 283 A L 285 A         F 287 A                 \n",
      " 282    0.08    0.65    0.46    0.17 * P 284 A P 286 A         P 288 A                 \n",
      " 283    0.19    0.60    0.40    0.05   V 285 A T 287 A         V 289 A                 \n",
      " 284    0.23    0.82    0.52    0.16   I 286 A V 288 A         T 290 A                 \n",
      " 285    0.38    0.53    0.37    0.03   C 287 A S 289 A         A 291 A                 \n",
      " 286    0.28    0.71    0.68    0.19   E 288 A R 290 A         K 292 A                 \n",
      " 287    0.17    0.97    1.08    0.36   N 289 A N 291 A         D 293 A                 \n",
      " 288    0.04    0.87    0.49    0.08 * G 290 A G 292 A         G 294 A                 \n",
      " 289    0.20    0.77    0.60    0.11   E 291 A E 293 A         A 295 A                 \n",
      " 290    0.17    0.46    0.62    0.24   Y 292 A W 294 A         Y 296 A                 \n",
      " 291    0.23    0.68    0.47    0.02   K 293 A E 295 A         R 297 A                 \n",
      " 292    0.37    0.56    0.34    0.09   R 294 A I 296 A         V 298 A                 \n",
      " 293    0.08    0.28    0.31    0.09 * V 295 A V 297 A         V 299 A                 \n",
      " 294    0.09    0.53    0.61    0.19 * E 296 A E 298 A         E 300 A                 \n",
      " 295    0.04    1.34    0.75    0.25 * G 297 A G 299 A         G 301 A                 \n",
      " 296    0.07    1.08    0.63    0.07 * L 298 A L 300 A         L 302 A                 \n",
      " 297    0.10    0.97    0.68    0.03 * E 299 A E 301 A         E 303 A                 \n",
      " 298    0.09    0.82    0.54    0.01 * I 300 A I 302 A         I 304 A                 \n",
      " 299    0.24    0.85    0.44    0.09   D 301 A S 303 A         N 305 A                 \n",
      " 300    0.27    1.07    0.62    0.07   A 302 A D 304 A         E 306 A                 \n",
      " 301    0.08    1.29    0.65    0.17 * F 303 A F 305 A         F 307 A                 \n",
      " 302    0.26    1.04    0.55    0.18   S 304 A Q 306 A         A 308 A                 \n",
      " 303    0.09    1.01    0.51    0.19 * R 305 A R 307 A         R 309 A                 \n",
      " 304    0.25    0.78    0.37    0.11   E 306 A A 308 A         K 310 A                 \n",
      " 305    0.20    0.81    0.38    0.13   K 307 A R 309 A         R 311 A                 \n",
      " 306    0.19    0.82    0.38    0.16   M 308 A I 310 A         M 312 A                 \n",
      " 307    0.14    1.03    0.46    0.23   D 309 A D 311 A         E 313 A                 \n",
      " 308    0.31    1.05    0.51    0.15   G 310 A A 312 A         I 314 A                 \n",
      " 309    0.17    1.24    0.63    0.20   T 311 A N 313 A         T 315 A                 \n",
      " 310    0.28    1.36    0.61    0.27   L 312 A A 314 A         A 316 A                 \n",
      " 311    0.28    1.54    0.69    0.24   A 313 A Q 315 A         Q 317 A                 \n",
      " 312    0.09    1.50    0.75    0.18 * E 314 A E 316 A         E 318 A                 \n",
      " 313    0.07    0.95    0.47    0.13 * L 315 A L 317 A         L 319 A                 \n",
      " 314    0.21    1.30    0.58    0.27   L 316 A Q 318 A         L 320 A                 \n",
      " 315    0.25    1.49    0.78    0.33   E 317 A A 319 A         D 321 A                 \n",
      " 316    0.10    1.06    0.73    0.26 * E 318 A E 320 A         E 322 A                 \n",
      " 317    0.25    0.96    0.59    0.25   R 319 A R 321 A         M 323 A                 \n",
      " 318    0.24    0.78    0.68    0.24   D 320 A E 322 A         E 324 A                 \n",
      " 319    0.29    0.73    0.81    0.29   G 321 A A 323 A         Q 325 A                 \n",
      " 320    0.13    0.85    0.77    0.22 * V 322 A V 324 A         V 326 A                 \n",
      " 321    0.38    0.77    0.73    0.25   A 323 A R 325 A         K 327 A                 \n",
      " 322    0.40    0.90    1.72    0.81   H 324 A D 326 A         A 328 A                 \n",
      " 323    0.06    3.50    2.72    0.39 * L 325 A L 327 A         L 329 A                 \n",
      " 324    0.00    0.00    2.90    0.00   -       L 328 A         G 330 A                 \n",
      " 325    0.00    0.00    2.51    0.00   -       L 329 A         L 331 A                 \n",
      " 326    0.00    0.00    2.59    0.00   -       E 330 A         I 332 A                 \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Current alignment total score:        79.52\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -       -       M   0 A                 \n",
      "   2    0.34    1.12    0.63    0.13   K   3 A -       S   1 A K   1 A                 \n",
      "   3    0.28    0.76    0.70    0.17   P   4 A S   8 A E   2 A A   2 A                 \n",
      "   4    0.20    0.32    0.40    0.10   A   5 A T   9 A P   3 A P   3 A                 \n",
      "   5    0.29    0.48    0.20    0.10   K   6 A K  10 A I   4 A V   4 A                 \n",
      "   6    0.12    0.47    0.23    0.08   R   7 A K  11 A R   5 A R   5 A                 \n",
      "   7    0.07    0.29    0.19    0.05 * V   8 A V  12 A V   6 A V   6 A                 \n",
      "   8    0.29    0.47    0.25    0.08   A   9 A T  13 A L   7 A A   7 A                 \n",
      "   9    0.07    0.49    0.30    0.04 * V  10 A V  14 A V   8 A V   8 A                 \n",
      "  10    0.08    0.27    0.19    0.04 * T  11 A T  15 A T   9 A T   9 A                 \n",
      "  11    0.03    0.43    0.22    0.09 * G  12 A G  16 A G  10 A G  10 A                 \n",
      "  12    0.08    0.21    0.28    0.12 * A  13 A A  17 A A  11 A A  11 A                 \n",
      "  13    0.08    0.34    0.31    0.08 * A  14 A A  18 A A  12 A A  12 A                 \n",
      "  14    0.04    0.32    0.18    0.07 * G  15 A G  19 A G  13 A G  13 A                 \n",
      "  15    0.09    0.32    0.19    0.08 * Q  16 A Q  20 A Q  14 A Q  14 A                 \n",
      "  16    0.09    0.23    0.16    0.05 * I  17 A I  21 A I  15 A I  15 A                 \n",
      "  17    0.18    0.46    0.27    0.08   A  18 A S  22 A A  16 A G  16 A                 \n",
      "  18    0.06    0.15    0.15    0.03 * Y  19 A Y  23 A Y  17 A Y  17 A                 \n",
      "  19    0.07    0.38    0.23    0.09 * S  20 A S  24 A S  18 A S  18 A                 \n",
      "  20    0.08    0.61    0.29    0.11 * L  21 A L  25 A L  19 A L  19 A                 \n",
      "  21    0.08    0.61    0.31    0.10 * L  22 A L  26 A L  20 A L  20 A                 \n",
      "  22    0.23    0.83    0.49    0.08   F  23 A W  27 A Y  21 A F  21 A                 \n",
      "  23    0.28    0.58    0.51    0.08   R  24 A R  28 A S  22 A R  22 A                 \n",
      "  24    0.09    0.37    0.40    0.10 * I  25 A I  29 A I  23 A I  23 A                 \n",
      "  25    0.23    0.42    0.30    0.07   A  26 A A  30 A G  24 A A  24 A                 \n",
      "  26    0.19    0.45    0.39    0.10   N  27 A N  31 A N  25 A A  25 A                 \n",
      "  27    0.04    0.57    0.47    0.09 * G  28 A G  32 A G  26 A G  26 A                 \n",
      "  28    0.28    1.28    0.75    0.16   D  29 A E  33 A S  27 A E  27 A                 \n",
      "  29    0.18    1.17    0.75    0.14   L  30 A V  34 A V  28 A M  28 A                 \n",
      "  30    0.18    0.57    0.57    0.23   L  31 A F  35 A F  29 A L  29 A                 \n",
      "  31    0.04    0.71    0.63    0.09 * G  32 A G  36 A G  30 A G  30 A                 \n",
      "  32    0.16    0.36    0.45    0.17   K  33 A T  37 A K  31 A K  31 A                 \n",
      "  33    0.09    0.78    0.59    0.11 * D  34 A D  38 A D  32 A D  32 A                 \n",
      "  34    0.16    0.33    0.38    0.16   Q  35 A T  39 A Q  33 A Q  33 A                 \n",
      "  35    0.05    0.46    0.43    0.11 * P  36 A P  40 A P  34 A P  34 A                 \n",
      "  36    0.20    0.36    0.25    0.07   V  37 A V  41 A I  35 A V  35 A                 \n",
      "  37    0.21    0.88    0.41    0.15   I  38 A E  42 A I  36 A I  36 A                 \n",
      "  38    0.08    0.63    0.27    0.14 * L  39 A L  43 A L  37 A L  37 A                 \n",
      "  39    0.32    0.48    0.25    0.07   Q  40 A K  44 A V  38 A Q  38 A                 \n",
      "  40    0.07    0.69    0.39    0.12 * L  41 A L  45 A L  39 A L  39 A                 \n",
      "  41    0.06    0.65    0.30    0.13 * L  42 A L  46 A L  40 A L  40 A                 \n",
      "  42    0.17    0.46    0.32    0.08   D  43 A E  47 A D  41 A E  41 A                 \n",
      "  43    0.13    1.09    0.50    0.20   L  44 A I  48 A I  42 A I  42 A                 \n",
      "  44    0.31    1.35    0.57    0.26   P  45 A P  49 A T  43 A P  43 A                 \n",
      "  45    0.35    1.73    0.78    0.32   Q  46 A Q  50 A P  44 A Q  44 A                 \n",
      "  46    0.32    1.30    0.55    0.25   A  47 A A  51 A M  45 A A  45 A                 \n",
      "  47    0.22    1.39    0.58    0.29   Q  48 A L  52 A M  46 A M  46 A                 \n",
      "  48    0.18    1.47    0.64    0.29   A  49 A G  53 A G  47 A K  47 A                 \n",
      "  49    0.27    1.19    0.55    0.22   A  50 A G  54 A V  48 A A  48 A                 \n",
      "  50    0.18    1.41    0.60    0.31   V  51 A A  55 A L  49 A L  49 A                 \n",
      "  51    0.23    1.72    0.68    0.40   K  52 A E  56 A D  50 A E  50 A                 \n",
      "  52    0.03    1.47    0.58    0.32 * G  53 A G  57 A G  51 A G  51 A                 \n",
      "  53    0.07    1.11    0.50    0.20 * V  54 A V  58 A V  52 A V  52 A                 \n",
      "  54    0.22    1.51    0.69    0.30   V  55 A A  59 A L  53 A V  53 A                 \n",
      "  55    0.12    1.81    0.74    0.41 * M  56 A M  60 A M  54 A M  54 A                 \n",
      "  56    0.08    1.21    0.53    0.24 * E  57 A E  61 A E  55 A E  55 A                 \n",
      "  57    0.05    1.08    0.49    0.21 * L  58 A L  62 A L  56 A L  56 A                 \n",
      "  58    0.28    1.89    0.78    0.46   D  59 A L  63 A Q  57 A E  57 A                 \n",
      "  59    0.08    2.06    0.85    0.51 * D  60 A D  64 A D  58 A D  58 A                 \n",
      "  60    0.17    1.69    0.67    0.43   C  61 A S  65 A C  59 A C  59 A                 \n",
      "  61    0.10    2.56    1.01    0.59 * A  62 A A  66 A A  60 A A  60 A                 \n",
      "  62    0.23    0.57    0.35    0.10   F  63 A F  67 A L  61 A F  61 A                 \n",
      "  63    0.09    1.15    0.50    0.23 * P  64 A P  68 A P  62 A P  62 A                 \n",
      "  64    0.09    0.22    0.27    0.10 * L  65 A L  69 A L  63 A L  63 A                 \n",
      "  65    0.10    0.31    0.20    0.03 * L  66 A L  70 A L  64 A L  64 A                 \n",
      "  66    0.26    0.16    0.34    0.18   A  67 A R  71 A K  65 A A  65 A                 \n",
      "  67    0.29    0.22    0.32    0.14   G  68 A N  72 A D  66 A G  66 A                 \n",
      "  68    0.17    0.61    0.39    0.05   V  69 A I  73 A V  67 A L  67 A                 \n",
      "  69    0.30    0.49    0.41    0.09   V  70 A T  74 A I  68 A E  68 A                 \n",
      "  70    0.24    1.00    0.49    0.17   I  71 A I  75 A A  69 A A  69 A                 \n",
      "  71    0.09    0.95    0.45    0.16 * T  72 A T  76 A T  70 A T  70 A                 \n",
      "  72    0.17    0.94    0.43    0.18   D  73 A A  77 A D  71 A D  71 A                 \n",
      "  73    0.29    1.16    0.51    0.23   D  74 A D  78 A K  72 A D  72 A                 \n",
      "  74    0.29    1.53    0.62    0.35   P  75 A A  79 A E  73 A P  73 A                 \n",
      "  75    0.25    2.03    0.80    0.44   K  76 A N  80 A E  74 A D  74 A                 \n",
      "  76    0.24    0.84    0.38    0.21   V  77 A E  81 A I  75 A V  75 A                 \n",
      "  77    0.10    0.37    0.29    0.16 * A  78 A A  82 A A  76 A A  76 A                 \n",
      "  78    0.07    0.96    0.39    0.21 * F  79 A F  83 A F  77 A F  77 A                 \n",
      "  79    0.18    0.64    0.32    0.11   K  80 A D  84 A K  78 A K  78 A                 \n",
      "  80    0.18    0.77    0.35    0.14   D  81 A G  85 A D  79 A D  79 A                 \n",
      "  81    0.30    0.51    0.30    0.08   A  82 A A  86 A L  80 A A  80 A                 \n",
      "  82    0.13    0.29    0.18    0.06   D  83 A N  87 A D  81 A D  81 A                 \n",
      "  83    0.22    0.33    0.17    0.06   V  84 A A  88 A V  82 A Y  82 A                 \n",
      "  84    0.09    0.54    0.29    0.20 * A  85 A A  89 A A  83 A A  83 A                 \n",
      "  85    0.22    0.36    0.28    0.12   L  86 A F  90 A I  84 A L  84 A                 \n",
      "  86    0.07    0.16    0.17    0.06 * L  87 A L  91 A L  85 A L  85 A                 \n",
      "  87    0.08    0.24    0.23    0.08 * V  88 A V  92 A V  86 A V  86 A                 \n",
      "  88    0.06    0.65    0.35    0.09 * G  89 A G  93 A G  87 A G  87 A                 \n",
      "  89    0.29    0.91    0.60    0.18   A  90 A A  94 A S  88 A A  88 A                 \n",
      "  90    0.51    1.99    1.30    0.38   R  91 A K  95 A M  89 A A  89 A                 \n",
      "  91    0.27    3.26    1.77    0.72 * P  92 A P  96 A P  90 A P  90 A                 \n",
      "  92    0.39    6.60    4.00    1.26   R  93 A E 102 A R  91 A                         \n",
      "  93    0.54    6.80    3.40    0.00   S  94 A -       R  92 A -                       \n",
      "  94    0.00    0.00    0.00    0.00   -       -       D  93 A -                       \n",
      "  95    0.00    0.00    0.00    0.00   -       -       G  94 A -                       \n",
      "  96    0.52    9.27    4.63    0.00   M  97 A -       M  95 A -                       \n",
      "  97    0.44    9.31    4.66    0.00   E  98 A -       E  96 A -                       \n",
      "  98    0.39    6.90    3.09    1.28   R  99 A R 103 A R  97 A -                       \n",
      "  99    0.37    6.74    3.05    1.29   K 100 A A 104 A K  98 A -                       \n",
      " 100    0.28    5.09    2.33    0.98   D 101 A D 105 A D  99 A -                       \n",
      " 101    0.33    1.86    0.95    0.27   L 102 A L 106 A L 100 A -                       \n",
      " 102    0.26    1.75    1.20    0.59 * L 103 A L 107 A L 101 A L 101 A                 \n",
      " 103    0.27    1.74    1.16    0.65   S 104 A A 108 A K 102 A Q 102 A                 \n",
      " 104    0.23    1.66    1.19    0.35   A 105 A N 109 A A 103 A V 103 A                 \n",
      " 105    0.14    1.00    0.89    0.18 * N 106 A N 110 A N 104 A N 104 A                 \n",
      " 106    0.34    1.62    1.17    0.27   A 107 A G 111 A V 105 A G 105 A                 \n",
      " 107    0.15    1.25    0.91    0.26   E 108 A K 112 A K 106 A K 106 A                 \n",
      " 108    0.09    0.48    0.39    0.11 * I 109 A I 113 A I 107 A I 107 A                 \n",
      " 109    0.08    0.64    0.44    0.10 * F 110 A F 114 A F 108 A F 108 A                 \n",
      " 110    0.27    0.79    0.50    0.21   T 111 A G 115 A K 109 A T 109 A                 \n",
      " 111    0.43    0.59    0.40    0.14   V 112 A P 116 A C 110 A E 110 A                 \n",
      " 112    0.08    0.31    0.26    0.03 * Q 113 A Q 117 A Q 111 A Q 111 A                 \n",
      " 113    0.03    0.29    0.24    0.09 * G 114 A G 118 A G 112 A G 112 A                 \n",
      " 114    0.22    0.43    0.36    0.06   A 115 A K 119 A A 113 A R 113 A                 \n",
      " 115    0.09    0.42    0.34    0.07 * A 116 A A 120 A A 114 A A 114 A                 \n",
      " 116    0.10    0.28    0.28    0.07   L 117 A I 121 A L 115 A L 115 A                 \n",
      " 117    0.23    0.62    0.37    0.09   N 118 A N 122 A D 116 A A 116 A                 \n",
      " 118    0.25    0.71    0.42    0.15   E 119 A D 123 A K 117 A E 117 A                 \n",
      " 119    0.33    0.77    0.40    0.12   V 120 A N 124 A Y 118 A V 118 A                 \n",
      " 120    0.11    0.62    0.36    0.07 * A 121 A A 125 A A 119 A A 119 A                 \n",
      " 121    0.21    1.19    0.56    0.26   S 122 A A 126 A K 120 A K 120 A                 \n",
      " 122    0.19    1.41    0.80    0.53   R 123 A D 127 A K 121 A K 121 A                 \n",
      " 123    0.26    0.69    0.69    0.42   D 124 A D 128 A S 122 A D 122 A                 \n",
      " 124    0.13    0.30    0.21    0.07   V 125 A I 129 A V 123 A V 123 A                 \n",
      " 125    0.14    0.19    0.18    0.06   K 126 A R 130 A K 124 A K 124 A                 \n",
      " 126    0.08    0.45    0.27    0.09 * V 127 A V 131 A V 125 A V 125 A                 \n",
      " 127    0.19    0.44    0.23    0.12   L 128 A L 132 A I 126 A L 126 A                 \n",
      " 128    0.07    0.34    0.18    0.07 * V 129 A V 133 A V 127 A V 127 A                 \n",
      " 129    0.08    0.43    0.22    0.09 * V 130 A V 134 A V 128 A V 128 A                 \n",
      " 130    0.04    0.34    0.22    0.03 * G 131 A G 135 A G 129 A G 129 A                 \n",
      " 131    0.11    0.52    0.35    0.16 * N 132 A N 136 A N 130 A N 130 A                 \n",
      " 132    0.05    0.72    0.35    0.12 * P 133 A P 137 A P 131 A P 131 A                 \n",
      " 133    0.09    0.49    0.34    0.04 * A 134 A A 138 A A 132 A A 132 A                 \n",
      " 134    0.11    0.46    0.28    0.06 * N 135 A N 139 A N 133 A N 133 A                 \n",
      " 135    0.09    0.44    0.23    0.06 * T 136 A T 140 A T 134 A T 134 A                 \n",
      " 136    0.12    0.31    0.26    0.04 * N 137 A N 141 A N 135 A N 135 A                 \n",
      " 137    0.35    0.34    0.28    0.09   A 138 A A 142 A C 136 A A 136 A                 \n",
      " 138    0.14    0.47    0.31    0.10   Y 139 A L 143 A L 137 A L 137 A                 \n",
      " 139    0.30    0.60    0.35    0.10   I 140 A I 144 A T 138 A I 138 A                 \n",
      " 140    0.08    0.18    0.28    0.11 * A 141 A A 145 A A 139 A A 139 A                 \n",
      " 141    0.26    0.16    0.27    0.12   M 142 A S 146 A S 140 A Y 140 A                 \n",
      " 142    0.15    0.51    0.34    0.07   K 143 A A 147 A K 141 A K 141 A                 \n",
      " 143    0.19    0.31    0.32    0.06   S 144 A A 148 A S 142 A N 142 A                 \n",
      " 144    0.11    0.29    0.29    0.13 * A 145 A A 149 A A 143 A A 143 A                 \n",
      " 145    0.06    0.78    0.77    0.44 * P 146 A P 150 A P 144 A P 144 A                 \n",
      " 146    0.27    0.23    0.24    0.08   D 147 A D 151 A S 145 A G 145 A                 \n",
      " 147    0.21    0.23    0.32    0.09   L 148 A V 152 A I 146 A L 146 A                 \n",
      " 148    0.15    0.44    0.44    0.13   P 149 A P 153 A P 147 A N 147 A                 \n",
      " 149    0.22    0.82    0.54    0.16   K 150 A A 154 A K 148 A P 148 A                 \n",
      " 150    0.28    0.89    0.49    0.16   K 151 A S 155 A E 149 A R 149 A                 \n",
      " 151    0.18    0.56    0.35    0.07   N 152 A R 156 A N 150 A N 150 A                 \n",
      " 152    0.08    0.59    0.33    0.10 * F 153 A F 157 A F 151 A F 151 A                 \n",
      " 153    0.20    0.56    0.29    0.08   T 154 A N 158 A S 152 A T 152 A                 \n",
      " 154    0.36    0.61    0.24    0.17   A 155 A A 159 A C 153 A A 153 A                 \n",
      " 155    0.20    0.19    0.15    0.05   M 156 A M 160 A L 154 A M 154 A                 \n",
      " 156    0.25    0.18    0.23    0.07   L 157 A M 161 A T 155 A T 155 A                 \n",
      " 157    0.08    0.30    0.21    0.05 * R 158 A R 162 A R 156 A R 156 A                 \n",
      " 158    0.06    0.38    0.28    0.05 * L 159 A L 163 A L 157 A L 157 A                 \n",
      " 159    0.06    0.39    0.25    0.07 * D 160 A D 164 A D 158 A D 158 A                 \n",
      " 160    0.08    0.32    0.21    0.02 * H 161 A H 165 A H 159 A H 159 A                 \n",
      " 161    0.10    0.17    0.19    0.05 * N 162 A N 166 A N 160 A N 160 A                 \n",
      " 162    0.07    0.51    0.27    0.10 * R 163 A R 167 A R 161 A R 161 A                 \n",
      " 163    0.08    0.48    0.19    0.14 * A 164 A A 168 A A 162 A A 162 A                 \n",
      " 164    0.25    0.25    0.12    0.06   L 165 A I 169 A K 163 A K 163 A                 \n",
      " 165    0.17    0.43    0.28    0.07   S 166 A S 170 A A 164 A A 164 A                 \n",
      " 166    0.08    0.51    0.39    0.03 * Q 167 A Q 171 A Q 165 A Q 165 A                 \n",
      " 167    0.19    0.17    0.25    0.09   L 168 A L 172 A I 166 A L 166 A                 \n",
      " 168    0.08    0.43    0.41    0.08 * A 169 A A 173 A A 167 A A 167 A                 \n",
      " 169    0.30    0.78    0.57    0.12   A 170 A T 174 A L 168 A K 168 A                 \n",
      " 170    0.09    0.59    0.50    0.14 * K 171 A K 175 A K 169 A K 169 A                 \n",
      " 171    0.24    0.97    0.65    0.25   S 172 A L 176 A L 170 A T 170 A                 \n",
      " 172    0.04    1.46    0.79    0.27 * G 173 A G 177 A G 171 A G 171 A                 \n",
      " 173    0.32    1.52    0.80    0.37   K 174 A R 178 A V 172 A T 172 A                 \n",
      " 174    0.32    1.45    0.76    0.28   P 175 A G 179 A T 173 A G 173 A                 \n",
      " 175    0.25    1.15    0.64    0.15   V 176 A S 180 A S 174 A V 174 A                 \n",
      " 176    0.21    0.78    0.52    0.19   A 177 A A 181 A D 175 A D 175 A                 \n",
      " 177    0.32    1.33    0.61    0.34   S 178 A E 182 A D 176 A R 176 A                 \n",
      " 178    0.26    1.43    0.68    0.29   I 179 A F 183 A V 177 A I 177 A                 \n",
      " 179    0.24    1.85    0.91    0.35   E 180 A N 184 A K 178 A R 178 A                 \n",
      " 180    0.23    1.30    0.76    0.19   K 181 A N 185 A N 179 A R 179 A                 \n",
      " 181    0.24    1.12    0.66    0.24   L 182 A I 186 A V 180 A M 180 A                 \n",
      " 182    0.27    0.48    0.41    0.15   A 183 A V 187 A I 181 A T 181 A                 \n",
      " 183    0.17    0.63    0.43    0.15   V 184 A V 188 A I 182 A V 182 A                 \n",
      " 184    0.06    0.34    0.18    0.06 * W 185 A W 189 A W 183 A W 183 A                 \n",
      " 185    0.06    0.88    0.37    0.17 * G 186 A G 190 A G 184 A G 184 A                 \n",
      " 186    0.12    0.94    0.67    0.34 * N 187 A N 191 A N 185 A N 185 A                 \n",
      " 187    0.09    0.94    0.55    0.23 * H 188 A H 192 A H 186 A H 186 A                 \n",
      " 188    0.10    0.89    0.50    0.14 * S 189 A S 193 A S 187 A S 187 A                 \n",
      " 189    0.19    1.06    0.57    0.23   P 190 A A 194 A S 188 A S 188 A                 \n",
      " 190    0.16    0.85    0.54    0.14   T 191 A T 195 A T 189 A I 189 A                 \n",
      " 191    0.27    0.53    0.34    0.06   M 192 A Q 196 A Q 190 A M 190 A                 \n",
      " 192    0.16    0.65    0.30    0.13   Y 193 A F 197 A Y 191 A F 191 A                 \n",
      " 193    0.06    0.50    0.20    0.11 * P 194 A P 198 A P 192 A P 192 A                 \n",
      " 194    0.08    0.49    0.26    0.06 * D 195 A D 199 A D 193 A D 193 A                 \n",
      " 195    0.23    0.53    0.47    0.15   F 196 A I 200 A V 194 A L 194 A                 \n",
      " 196    0.32    0.39    0.37    0.16   R 197 A T 201 A N 195 A F 195 A                 \n",
      " 197    0.25    0.61    0.47    0.12   F 198 A Y 202 A H 196 A H 196 A                 \n",
      " 198    0.10    0.54    0.42    0.08 * A 199 A A 203 A A 197 A A 197 A                 \n",
      " 199    0.26    0.79    0.54    0.08   T 200 A T 204 A K 198 A E 198 A                 \n",
      " 200    0.31    0.85    0.73    0.18   A 201 A V 205 A V 199 A V 199 A                 \n",
      " 201    0.00    0.00    0.00    0.00   -       -       K 200 A -                       \n",
      " 202    0.00    0.00    0.00    0.00   -       -       L 201 A -                       \n",
      " 203    0.00    0.00    0.00    0.00   -       -       Q 202 A -                       \n",
      " 204    0.00    0.00    0.00    0.00   -       -       A 203 A -                       \n",
      " 205    0.53    5.42    2.24    1.22   E 202 A G 206 A K 204 A D 200 A                 \n",
      " 206    0.59    1.19    1.10    0.23   G 203 A G 207 A E 205 A G 205 A                 \n",
      " 207    0.45    0.67    0.68    0.16   E 204 A E 208 A V 206 A R 206 A                 \n",
      " 208    0.30    0.92    0.65    0.20   S 205 A K 209 A G 207 A P 207 A                 \n",
      " 209    0.19    0.86    0.47    0.20   L 206 A V 210 A V 208 A A 208 A                 \n",
      " 210    0.33    0.73    0.45    0.21   L 207 A T 211 A Y 209 A L 209 A                 \n",
      " 211    0.19    0.57    0.49    0.22   K 208 A D 212 A E 210 A E 210 A                 \n",
      " 212    0.36    1.10    0.82    0.27   L 209 A L 213 A A 211 A L 211 A                 \n",
      " 213    0.18    1.23    0.91    0.23   I 210 A V 214 A V 212 A V 212 A                 \n",
      " 214    0.45    2.15    1.08    0.00   N 211 A -       K 213 A -                       \n",
      " 215    0.15    0.82    0.64    0.27 * D 212 A D 215 A D 214 A D 214 A                 \n",
      " 216    0.32    0.43    0.25    0.09   D 213 A H 216 A D 215 A M 215 A                 \n",
      " 217    0.29    0.65    0.54    0.12   V 214 A D 217 A S 216 A E 216 A                 \n",
      " 218    0.05    1.11    0.66    0.19 * W 215 A W 218 A W 217 A W 217 A                 \n",
      " 219    0.34    1.18    0.61    0.23   N 216 A Y 219 A L 218 A Y 218 A                 \n",
      " 220    0.29    1.91    0.85    0.44   R 217 A V 220 A K 219 A E 219 A                 \n",
      " 221    0.33    2.06    0.97    0.39   D 218 A E 221 A G 220 A K 220 A                 \n",
      " 222    0.29    1.40    0.68    0.37   T 219 A E 222 A E 221 A V 221 A                 \n",
      " 223    0.12    0.79    0.37    0.22 * F 220 A F 223 A F 222 A F 222 A                 \n",
      " 224    0.10    0.82    0.38    0.24 * I 221 A I 224 A I 223 A I 223 A                 \n",
      " 225    0.30    0.76    0.54    0.28   P 222 A P 225 A T 224 A P 224 A                 \n",
      " 226    0.15    1.05    0.49    0.35   T 223 A R 226 A T 225 A T 225 A                 \n",
      " 227    0.07    1.20    0.50    0.36 * V 224 A V 227 A V 226 A V 226 A                 \n",
      " 228    0.30    1.38    0.72    0.34   G 225 A A 228 A Q 227 A A 227 A                 \n",
      " 229    0.21    1.63    0.83    0.49   K 226 A N 229 A Q 228 A Q 228 A                 \n",
      " 230    0.09    1.61    0.65    0.48 * R 227 A R 230 A R 229 A R 229 A                 \n",
      " 231    0.04    2.01    0.83    0.50 * G 228 A G 231 A G 230 A G 230 A                 \n",
      " 232    0.08    2.76    1.13    0.65 * A 229 A A 232 A A 231 A A 231 A                 \n",
      " 233    0.14    2.64    1.02    0.72   A 230 A E 233 A A 232 A A 232 A                 \n",
      " 234    0.17    2.22    0.86    0.61   I 231 A I 234 A V 233 A I 233 A                 \n",
      " 235    0.09    2.74    1.10    0.63 * I 232 A I 235 A I 234 A I 234 A                 \n",
      " 236    0.25    3.50    1.35    0.78   E 233 A E 236 A K 235 A Q 235 A                 \n",
      " 237    0.14    3.38    1.30    0.74   A 234 A V 237 A A 236 A A 236 A                 \n",
      " 238    0.08    3.13    1.24    0.71 * R 235 A R 238 A R 237 A R 237 A                 \n",
      " 239    0.31    3.29    1.28    0.76   G 236 A G 239 A K 238 A G 238 A                 \n",
      " 240    0.24    2.63    1.02    0.59   L 237 A K 240 A L 239 A A 239 A                 \n",
      " 241    0.08    2.30    0.98    0.46 * S 238 A S 241 A S 240 A S 240 A                 \n",
      " 242    0.09    1.49    0.65    0.30 * S 239 A S 242 A S 241 A S 241 A                 \n",
      " 243    0.10    0.63    0.32    0.12 * A 240 A A 243 A A 242 A A 242 A                 \n",
      " 244    0.33    0.50    0.28    0.08   A 241 A A 244 A M 243 A A 243 A                 \n",
      " 245    0.08    0.52    0.30    0.06 * S 242 A S 245 A S 244 A S 244 A                 \n",
      " 246    0.09    0.31    0.21    0.03 * A 243 A A 246 A A 245 A A 245 A                 \n",
      " 247    0.08    0.20    0.15    0.04 * A 244 A A 247 A A 246 A A 246 A                 \n",
      " 248    0.25    0.18    0.18    0.10   N 245 A S 248 A K 247 A N 247 A                 \n",
      " 249    0.12    0.21    0.20    0.05   A 246 A S 249 A A 248 A A 248 A                 \n",
      " 250    0.30    0.48    0.25    0.07   A 247 A A 250 A I 249 A A 249 A                 \n",
      " 251    0.43    0.57    0.32    0.07   I 248 A I 251 A C 250 A I 250 A                 \n",
      " 252    0.12    0.35    0.27    0.04   D 249 A D 252 A D 251 A E 251 A                 \n",
      " 253    0.08    0.30    0.22    0.04 * H 250 A H 253 A H 252 A H 252 A                 \n",
      " 254    0.17    0.42    0.26    0.09   V 251 A M 254 A V 253 A I 253 A                 \n",
      " 255    0.08    0.37    0.23    0.04 * R 252 A R 255 A R 254 A R 254 A                 \n",
      " 256    0.08    0.35    0.20    0.07 * D 253 A D 256 A D 255 A D 255 A                 \n",
      " 257    0.40    0.80    0.41    0.12   W 254 A W 257 A I 256 A W 256 A                 \n",
      " 258    0.41    0.94    0.38    0.23   V 255 A V 258 A W 257 A A 257 A                 \n",
      " 259    0.29    0.50    0.25    0.09   L 256 A Q 259 A F 258 A L 258 A                 \n",
      " 260    0.07    0.67    0.38    0.18 * G 257 A G 260 A G 259 A G 259 A                 \n",
      " 261    0.12    0.61    0.33    0.09 * T 258 A T 261 A T 260 A T 260 A                 \n",
      " 262    0.32    1.23    1.30    0.43   -       E 262 A P 261 A P 261 A                 \n",
      " 263    0.30    3.04    2.83    1.05   N 259 A A 263 A E 262 A E 262 A                 \n",
      " 264    0.22    1.80    1.73    0.60   G 260 A -       G 263 A G 263 A                 \n",
      " 265    0.37    0.80    0.68    0.17   K 261 A -       E 264 A D 264 A                 \n",
      " 266    0.36    0.98    0.66    0.23   W 262 A W 264 A F 265 A W 265 A                 \n",
      " 267    0.18    0.57    0.41    0.24   V 263 A S 265 A V 266 A V 266 A                 \n",
      " 268    0.13    0.11    0.20    0.08   T 264 A S 266 A S 267 A S 267 A                 \n",
      " 269    0.20    0.43    0.28    0.11   M 265 A A 267 A M 268 A M 268 A                 \n",
      " 270    0.17    0.37    0.25    0.07   G 266 A A 268 A G 269 A A 269 A                 \n",
      " 271    0.11    0.38    0.24    0.09   I 267 A I 269 A I 270 A V 270 A                 \n",
      " 272    0.38    0.44    0.27    0.10   P 268 A P 270 A I 271 A P 271 A                 \n",
      " 273    0.10    0.39    0.28    0.10 * S 269 A S 271 A S 272 A S 272 A                 \n",
      " 274    0.28    1.73    0.72    0.41   D 270 A T 272 A D 273 A Q 273 A                 \n",
      " 275    0.08    3.65    1.37    0.81 * G 271 A G 273 A G 274 A G 274 A                 \n",
      " 276    0.00    0.00    0.00    0.00   -       -       N 275 A -                       \n",
      " 277    0.26    2.93    1.28    0.60   S 272 A A 274 A S 276 A E 275 A                 \n",
      " 278    0.16    0.72    0.52    0.25 * Y 273 A Y 275 A Y 277 A Y 277 A                 \n",
      " 279    0.12    1.15    0.86    0.40 * G 274 A G 276 A G 278 A G 278 A                 \n",
      " 280    0.17    1.02    0.51    0.24   I 275 A I 277 A V 279 A I 279 A                 \n",
      " 281    0.06    1.01    0.47    0.25 * P 276 A P 278 A P 280 A P 280 A                 \n",
      " 282    0.24    1.02    0.49    0.24   E 277 A E 279 A D 281 A E 281 A                 \n",
      " 283    0.22    1.24    0.94    0.41   D 278 A G 280 A D 282 A G 282 A                 \n",
      " 284    0.21    0.83    0.62    0.09   I 279 A I 281 A L 283 A I 283 A                 \n",
      " 285    0.23    0.66    0.34    0.15   I 280 A F 282 A L 284 A V 284 A                 \n",
      " 286    0.16    0.65    0.38    0.16   Y 281 A V 283 A Y 285 A Y 285 A                 \n",
      " 287    0.20    0.69    0.37    0.19   G 282 A G 284 A S 286 A S 286 A                 \n",
      " 288    0.20    0.56    0.28    0.10   V 283 A L 285 A F 287 A F 287 A                 \n",
      " 289    0.07    0.34    0.40    0.22 * P 284 A P 286 A P 288 A P 288 A                 \n",
      " 290    0.15    0.62    0.43    0.09   V 285 A T 287 A V 289 A V 289 A                 \n",
      " 291    0.21    0.84    0.54    0.25   I 286 A V 288 A T 290 A T 290 A                 \n",
      " 292    0.35    0.88    0.48    0.16   C 287 A S 289 A I 291 A A 291 A                 \n",
      " 293    0.20    0.92    0.73    0.05   E 288 A R 290 A K 292 A K 292 A                 \n",
      " 294    0.19    0.93    0.99    0.40   N 289 A N 291 A D 293 A D 293 A                 \n",
      " 295    0.31    0.57    0.48    0.15   G 290 A G 292 A K 294 A G 294 A                 \n",
      " 296    0.29    0.78    0.62    0.11   E 291 A E 293 A T 295 A A 295 A                 \n",
      " 297    0.21    0.80    0.64    0.20   Y 292 A W 294 A W 296 A Y 296 A                 \n",
      " 298    0.18    0.77    0.51    0.11   K 293 A E 295 A K 297 A R 297 A                 \n",
      " 299    0.22    0.35    0.32    0.10   R 294 A I 296 A I 298 A V 298 A                 \n",
      " 300    0.08    0.28    0.29    0.09 * V 295 A V 297 A V 299 A V 299 A                 \n",
      " 301    0.10    0.80    0.62    0.20 * E 296 A E 298 A E 300 A E 300 A                 \n",
      " 302    0.04    1.54    0.92    0.27 * G 297 A G 299 A G 301 A G 301 A                 \n",
      " 303    0.07    0.93    0.66    0.18 * L 298 A L 300 A L 302 A L 302 A                 \n",
      " 304    0.31    1.05    0.73    0.14   E 299 A E 301 A P 303 A E 303 A                 \n",
      " 305    0.08    0.87    0.59    0.09 * I 300 A I 302 A I 304 A I 304 A                 \n",
      " 306    0.19    0.79    0.50    0.14   D 301 A S 303 A N 305 A N 305 A                 \n",
      " 307    0.19    0.54    0.57    0.13   A 302 A D 304 A D 306 A E 306 A                 \n",
      " 308    0.07    0.16    0.52    0.28 * F 303 A F 305 A F 307 A F 307 A                 \n",
      " 309    0.20    0.34    0.48    0.20   S 304 A Q 306 A S 308 A A 308 A                 \n",
      " 310    0.10    0.92    0.57    0.22 * R 305 A R 307 A R 309 A R 309 A                 \n",
      " 311    0.22    0.68    0.38    0.23   E 306 A A 308 A E 310 A K 310 A                 \n",
      " 312    0.19    0.45    0.37    0.16   K 307 A R 309 A K 311 A R 311 A                 \n",
      " 313    0.17    0.40    0.35    0.18   M 308 A I 310 A M 312 A M 312 A                 \n",
      " 314    0.12    0.63    0.44    0.27   D 309 A D 311 A D 313 A E 313 A                 \n",
      " 315    0.28    0.65    0.49    0.23   G 310 A A 312 A L 314 A I 314 A                 \n",
      " 316    0.14    0.50    0.56    0.25   T 311 A N 313 A T 315 A T 315 A                 \n",
      " 317    0.15    0.47    0.52    0.31   L 312 A A 314 A A 316 A A 316 A                 \n",
      " 318    0.25    0.61    0.60    0.32   A 313 A Q 315 A K 317 A Q 317 A                 \n",
      " 319    0.09    0.57    0.66    0.27 * E 314 A E 316 A E 318 A E 318 A                 \n",
      " 320    0.06    0.35    0.41    0.17 * L 315 A L 317 A L 319 A L 319 A                 \n",
      " 321    0.28    0.24    0.49    0.28   L 316 A Q 318 A A 320 A L 320 A                 \n",
      " 322    0.19    0.11    0.61    0.42   E 317 A A 319 A E 321 A D 321 A                 \n",
      " 323    0.09    0.27    0.59    0.35 * E 318 A E 320 A E 322 A E 322 A                 \n",
      " 324    0.26    0.28    0.49    0.28   R 319 A R 321 A K 323 A M 323 A                 \n",
      " 325    0.15    0.64    0.60    0.33   D 320 A E 322 A E 324 A E 324 A                 \n",
      " 326    0.00    0.00    0.00    0.00   -       -       T 325 A -                       \n",
      " 327    0.21    3.88    1.65    0.81   G 321 A A 323 A A 326 A Q 325 A                 \n",
      " 328    0.36    2.70    1.25    0.52   V 322 A V 324 A F 327 A V 326 A                 \n",
      " 329    0.39    3.99    1.69    0.76   A 323 A R 325 A E 328 A K 327 A                 \n",
      " 330    0.41    4.33    2.39    0.71   H 324 A D 326 A F 329 A A 328 A                 \n",
      " 331    0.15    1.77    2.40    0.69 * L 325 A L 327 A L 330 A L 329 A                 \n",
      " 332    0.52    4.08    2.86    1.04   -       L 328 A S 331 A G 330 A                 \n",
      " 333    0.51    9.23    4.52    1.66   -       L 329 A S 332 A L 331 A                 \n",
      " 334    0.35   12.67    6.05    1.88   -       E 330 A A 333 A I 332 A                 \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    4\n",
      "\n",
      "Current alignment total score:        90.69\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -       -       M   0 A         -       \n",
      "   2    0.39    2.71    1.16    0.59   K   3 A -       S   1 A K   1 A         M   1 A \n",
      "   3    0.21    2.32    1.03    0.47   P   4 A S   8 A E   2 A A   2 A         S   2 A \n",
      "   4    0.23    1.21    0.56    0.24   A   5 A T   9 A P   3 A P   3 A         A   3 A \n",
      "   5    0.23    0.65    0.28    0.17   K   6 A K  10 A I   4 A V   4 A         V   4 A \n",
      "   6    0.17    0.44    0.25    0.13   R   7 A K  11 A R   5 A R   5 A         K   5 A \n",
      "   7    0.07    0.26    0.20    0.06 * V   8 A V  12 A V   6 A V   6 A         V   6 A \n",
      "   8    0.16    0.33    0.27    0.05   A   9 A T  13 A L   7 A A   7 A         A   7 A \n",
      "   9    0.07    0.29    0.30    0.03 * V  10 A V  14 A V   8 A V   8 A         V   8 A \n",
      "  10    0.07    0.45    0.24    0.07 * T  11 A T  15 A T   9 A T   9 A         T   9 A \n",
      "  11    0.03    0.64    0.31    0.14 * G  12 A G  16 A G  10 A G  10 A         G  10 A \n",
      "  12    0.08    0.48    0.32    0.07 * A  13 A A  17 A A  11 A A  11 A         A  11 A \n",
      "  13    0.08    0.48    0.34    0.04 * A  14 A A  18 A A  12 A A  12 A         A  12 A \n",
      "  14    0.04    0.36    0.21    0.08 * G  15 A G  19 A G  13 A G  13 A         G  13 A \n",
      "  15    0.08    0.28    0.20    0.07 * Q  16 A Q  20 A Q  14 A Q  14 A         Q  14 A \n",
      "  16    0.08    0.16    0.16    0.04 * I  17 A I  21 A I  15 A I  15 A         I  15 A \n",
      "  17    0.18    0.29    0.26    0.08   A  18 A S  22 A A  16 A G  16 A         G  16 A \n",
      "  18    0.05    0.34    0.18    0.05 * Y  19 A Y  23 A Y  17 A Y  17 A         Y  17 A \n",
      "  19    0.21    0.29    0.23    0.09   S  20 A S  24 A S  18 A S  18 A         A  18 A \n",
      "  20    0.06    0.23    0.27    0.12 * L  21 A L  25 A L  19 A L  19 A         L  19 A \n",
      "  21    0.19    0.17    0.27    0.12   L  22 A L  26 A L  20 A L  20 A         V  20 A \n",
      "  22    0.40    0.09    0.40    0.18   F  23 A W  27 A Y  21 A F  21 A         P  21 A \n",
      "  23    0.31    0.36    0.47    0.11   R  24 A R  28 A S  22 A R  22 A         L  22 A \n",
      "  24    0.08    0.18    0.35    0.13 * I  25 A I  29 A I  23 A I  23 A         I  23 A \n",
      "  25    0.12    0.22    0.28    0.09   A  26 A A  30 A G  24 A A  24 A         A  24 A \n",
      "  26    0.30    0.23    0.35    0.14   N  27 A N  31 A N  25 A A  25 A         R  25 A \n",
      "  27    0.03    0.37    0.43    0.14 * G  28 A G  32 A G  26 A G  26 A         G  26 A \n",
      "  28    0.25    0.69    0.70    0.22   D  29 A E  33 A S  27 A E  27 A         A  27 A \n",
      "  29    0.15    0.71    0.72    0.17   L  30 A V  34 A V  28 A M  28 A         L  28 A \n",
      "  30    0.13    0.49    0.54    0.21   L  31 A F  35 A F  29 A L  29 A         L  29 A \n",
      "  31    0.03    0.44    0.58    0.15 * G  32 A G  36 A G  30 A G  30 A         G  30 A \n",
      "  32    0.30    0.67    0.47    0.21   K  33 A T  37 A K  31 A K  31 A         P  31 A \n",
      "  33    0.28    0.85    0.61    0.17   D  34 A D  38 A D  32 A D  32 A         T  32 A \n",
      "  34    0.24    0.77    0.44    0.19   Q  35 A T  39 A Q  33 A Q  33 A         T  33 A \n",
      "  35    0.04    0.51    0.44    0.10 * P  36 A P  40 A P  34 A P  34 A         P  34 A \n",
      "  36    0.09    0.49    0.29    0.08   V  37 A V  41 A I  35 A V  35 A         V  35 A \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  37    0.29    0.56    0.41    0.19   I  38 A E  42 A I  36 A I  36 A         E  36 A \n",
      "  38    0.06    0.29    0.26    0.13 * L  39 A L  43 A L  37 A L  37 A         L  37 A \n",
      "  39    0.27    0.28    0.25    0.03   Q  40 A K  44 A V  38 A Q  38 A         R  38 A \n",
      "  40    0.06    0.45    0.39    0.11 * L  41 A L  45 A L  39 A L  39 A         L  39 A \n",
      "  41    0.05    0.46    0.33    0.09 * L  42 A L  46 A L  40 A L  40 A         L  40 A \n",
      "  42    0.14    0.71    0.40    0.09   D  43 A E  47 A D  41 A E  41 A         D  41 A \n",
      "  43    0.11    0.62    0.50    0.22   L  44 A I  48 A I  42 A I  42 A         I  42 A \n",
      "  44    0.29    0.83    0.57    0.31   P  45 A P  49 A T  43 A P  43 A         E  43 A \n",
      "  45    0.27    0.58    0.71    0.35   Q  46 A Q  50 A P  44 A Q  44 A         P  44 A \n",
      "  46    0.14    0.57    0.53    0.25   A  47 A A  51 A M  45 A A  45 A         A  45 A \n",
      "  47    0.19    0.52    0.55    0.29   Q  48 A L  52 A M  46 A M  46 A         L  46 A \n",
      "  48    0.23    0.32    0.55    0.32   A  49 A G  53 A G  47 A K  47 A         K  47 A \n",
      "  49    0.15    0.44    0.50    0.25   A  50 A G  54 A V  48 A A  48 A         A  48 A \n",
      "  50    0.14    0.51    0.56    0.30   V  51 A A  55 A L  49 A L  49 A         L  49 A \n",
      "  51    0.25    0.37    0.60    0.41   K  52 A E  56 A D  50 A E  50 A         A  50 A \n",
      "  52    0.03    0.48    0.52    0.34 * G  53 A G  57 A G  51 A G  51 A         G  51 A \n",
      "  53    0.07    0.52    0.47    0.23 * V  54 A V  58 A V  52 A V  52 A         V  52 A \n",
      "  54    0.31    0.37    0.60    0.33   V  55 A A  59 A L  53 A V  53 A         E  53 A \n",
      "  55    0.31    0.41    0.64    0.44   M  56 A M  60 A M  54 A M  54 A         A  54 A \n",
      "  56    0.08    0.40    0.47    0.28 * E  57 A E  61 A E  55 A E  55 A         E  55 A \n",
      "  57    0.05    0.26    0.42    0.25 * L  58 A L  62 A L  56 A L  56 A         L  56 A \n",
      "  58    0.21    0.31    0.66    0.49   D  59 A L  63 A Q  57 A E  57 A         E  57 A \n",
      "  59    0.06    0.35    0.72    0.54 * D  60 A D  64 A D  58 A D  58 A         D  58 A \n",
      "  60    0.11    0.34    0.58    0.43   C  61 A S  65 A C  59 A C  59 A         C  59 A \n",
      "  61    0.08    0.53    0.87    0.62 * A  62 A A  66 A A  60 A A  60 A         A  60 A \n",
      "  62    0.10    0.44    0.34    0.13   F  63 A F  67 A L  61 A F  61 A         F  61 A \n",
      "  63    0.05    0.65    0.49    0.26 * P  64 A P  68 A P  62 A P  62 A         P  62 A \n",
      "  64    0.06    0.57    0.32    0.13 * L  65 A L  69 A L  63 A L  63 A         L  63 A \n",
      "  65    0.06    0.47    0.23    0.12 * L  66 A L  70 A L  64 A L  64 A         L  64 A \n",
      "  66    0.29    0.64    0.39    0.17   A  67 A R  71 A K  65 A A  65 A         D  65 A \n",
      "  67    0.28    0.51    0.33    0.17   G  68 A N  72 A D  66 A G  66 A         K  66 A \n",
      "  68    0.12    0.39    0.38    0.06   V  69 A I  73 A V  67 A L  67 A         V  67 A \n",
      "  69    0.20    0.30    0.37    0.13   V  70 A T  74 A I  68 A E  68 A         V  68 A \n",
      "  70    0.19    0.34    0.44    0.18   I  71 A I  75 A A  69 A A  69 A         V  69 A \n",
      "  71    0.07    0.41    0.44    0.15 * T  72 A T  76 A T  70 A T  70 A         T  70 A \n",
      "  72    0.22    0.51    0.43    0.18   D  73 A A  77 A D  71 A D  71 A         A  71 A \n",
      "  73    0.12    0.46    0.48    0.25   D  74 A D  78 A K  72 A D  72 A         D  72 A \n",
      "  74    0.16    0.77    0.59    0.38   P  75 A A  79 A E  73 A P  73 A         P  73 A \n",
      "  75    0.28    0.43    0.69    0.47   K  76 A N  80 A E  74 A D  74 A         R  74 A \n",
      "  76    0.15    0.35    0.37    0.19   V  77 A E  81 A I  75 A V  75 A         V  75 A \n",
      "  77    0.08    0.54    0.35    0.11 * A  78 A A  82 A A  76 A A  76 A         A  76 A \n",
      "  78    0.06    0.68    0.40    0.26 * F  79 A F  83 A F  77 A F  77 A         F  77 A \n",
      "  79    0.23    0.60    0.36    0.15   K  80 A D  84 A K  78 A K  78 A         D  78 A \n",
      "  80    0.22    0.60    0.36    0.20   D  81 A G  85 A D  79 A D  79 A         G  79 A \n",
      "  81    0.22    0.74    0.37    0.16   A  82 A A  86 A L  80 A A  80 A         V  80 A \n",
      "  82    0.27    0.21    0.18    0.05   D  83 A N  87 A D  81 A D  81 A         A  81 A \n",
      "  83    0.24    0.35    0.19    0.10   V  84 A A  88 A V  82 A Y  82 A         I  82 A \n",
      "  84    0.08    0.26    0.29    0.16 * A  85 A A  89 A A  83 A A  83 A         A  83 A \n",
      "  85    0.18    0.49    0.32    0.09   L  86 A F  90 A I  84 A L  84 A         I  84 A \n",
      "  86    0.19    0.52    0.25    0.09   L  87 A L  91 A L  85 A L  85 A         M  85 A \n",
      "  87    0.37    0.29    0.24    0.06   V  88 A V  92 A V  86 A V  86 A         C  86 A \n",
      "  88    0.03    0.40    0.36    0.07 * G  89 A G  93 A G  87 A G  87 A         G  87 A \n",
      "  89    0.13    0.68    0.60    0.18   A  90 A A  94 A S  88 A A  88 A         A  88 A \n",
      "  90    0.42    1.25    1.25    0.40   R  91 A K  95 A M  89 A A  89 A         F  89 A \n",
      "  91    0.15    0.82    1.56    0.77 * P  92 A P  96 A P  90 A P  90 A         P  90 A \n",
      "  92    0.24    2.62    3.42    1.67   R  93 A E 102 A R  91 A                 R  91 A \n",
      "  93    0.40    2.76    2.89    1.03   S  94 A -       R  92 A -               K  92 A \n",
      "  94    0.45    8.03    4.01    0.00   -       -       D  93 A -               A  93 A \n",
      "  95    0.40    8.92    4.46    0.00   -       -       G  94 A -               G  94 A \n",
      "  96    0.36    2.64    3.69    1.49   M  97 A -       M  95 A -               M  95 A \n",
      "  97    0.32    2.79    3.74    1.47   E  98 A -       E  96 A -               E  96 A \n",
      "  98    0.27    1.57    2.56    1.51   R  99 A R 103 A R  97 A -               R  97 A \n",
      "  99    0.26    1.66    2.56    1.48   K 100 A A 104 A K  98 A -               K  98 A \n",
      " 100    0.20    1.36    1.96    1.15   D 101 A D 105 A D  99 A -               D  99 A \n",
      " 101    0.20    0.90    0.89    0.31   L 102 A L 106 A L 100 A -               L 100 A \n",
      " 102    0.15    0.84    1.12    0.55 * L 103 A L 107 A L 101 A L 101 A         L 101 A \n",
      " 103    0.26    0.62    1.01    0.67   S 104 A A 108 A K 102 A Q 102 A         E 102 A \n",
      " 104    0.34    0.45    1.02    0.46   A 105 A N 109 A A 103 A V 103 A         M 103 A \n",
      " 105    0.13    0.77    0.83    0.23 * N 106 A N 110 A N 104 A N 104 A         N 104 A \n",
      " 106    0.20    0.96    1.09    0.33   A 107 A G 111 A V 105 A G 105 A         A 105 A \n",
      " 107    0.22    0.85    0.87    0.27   E 108 A K 112 A K 106 A K 106 A         R 106 A \n",
      " 108    0.08    0.39    0.38    0.10 * I 109 A I 113 A I 107 A I 107 A         I 107 A \n",
      " 109    0.06    0.46    0.44    0.08 * F 110 A F 114 A F 108 A F 108 A         F 108 A \n",
      " 110    0.21    0.67    0.53    0.17   T 111 A G 115 A K 109 A T 109 A         K 109 A \n",
      " 111    0.29    0.62    0.44    0.09   V 112 A P 116 A C 110 A E 110 A         E 110 A \n",
      " 112    0.08    0.44    0.29    0.03 * Q 113 A Q 117 A Q 111 A Q 111 A         Q 111 A \n",
      " 113    0.03    0.56    0.30    0.12 * G 114 A G 118 A G 112 A G 112 A         G 112 A \n",
      " 114    0.26    0.69    0.42    0.08   A 115 A K 119 A A 113 A R 113 A         E 113 A \n",
      " 115    0.08    0.60    0.38    0.06 * A 116 A A 120 A A 114 A A 114 A         A 114 A \n",
      " 116    0.16    0.56    0.33    0.06   L 117 A I 121 A L 115 A L 115 A         I 115 A \n",
      " 117    0.22    0.73    0.43    0.12   N 118 A N 122 A D 116 A A 116 A         A 116 A \n",
      " 118    0.25    0.69    0.47    0.13   E 119 A D 123 A K 117 A E 117 A         A 117 A \n",
      " 119    0.20    0.72    0.44    0.16   V 120 A N 124 A Y 118 A V 118 A         V 118 A \n",
      " 120    0.09    0.60    0.38    0.13 * A 121 A A 125 A A 119 A A 119 A         A 119 A \n",
      " 121    0.20    0.75    0.58    0.24   S 122 A A 126 A K 120 A K 120 A         A 120 A \n",
      " 122    0.26    0.93    0.84    0.40   R 123 A D 127 A K 121 A K 121 A         S 121 A \n",
      " 123    0.11    0.24    0.59    0.43   D 124 A D 128 A S 122 A D 122 A         D 122 A \n",
      " 124    0.39    0.28    0.21    0.08   V 125 A I 129 A V 123 A V 123 A         C 123 A \n",
      " 125    0.17    0.36    0.20    0.09   K 126 A R 130 A K 124 A K 124 A         R 124 A \n",
      " 126    0.07    0.21    0.25    0.10 * V 127 A V 131 A V 125 A V 125 A         V 125 A \n",
      " 127    0.18    0.41    0.25    0.12   L 128 A L 132 A I 126 A L 126 A         V 126 A \n",
      " 128    0.06    0.04    0.15    0.08 * V 129 A V 133 A V 127 A V 127 A         V 127 A \n",
      " 129    0.07    0.31    0.23    0.08 * V 130 A V 134 A V 128 A V 128 A         V 128 A \n",
      " 130    0.03    0.41    0.25    0.06 * G 131 A G 135 A G 129 A G 129 A         G 129 A \n",
      " 131    0.10    0.44    0.36    0.16 * N 132 A N 136 A N 130 A N 130 A         N 130 A \n",
      " 132    0.04    0.52    0.38    0.10 * P 133 A P 137 A P 131 A P 131 A         P 131 A \n",
      " 133    0.08    0.37    0.34    0.03 * A 134 A A 138 A A 132 A A 132 A         A 132 A \n",
      " 134    0.10    0.53    0.31    0.10 * N 135 A N 139 A N 133 A N 133 A         N 133 A \n",
      " 135    0.07    0.63    0.31    0.11 * T 136 A T 140 A T 134 A T 134 A         T 134 A \n",
      " 136    0.10    0.44    0.29    0.05 * N 137 A N 141 A N 135 A N 135 A         N 135 A \n",
      " 137    0.14    0.58    0.34    0.08   A 138 A A 142 A C 136 A A 136 A         A 136 A \n",
      " 138    0.12    0.65    0.38    0.07   Y 139 A L 143 A L 137 A L 137 A         L 137 A \n",
      " 139    0.13    0.49    0.37    0.06   I 140 A I 144 A T 138 A I 138 A         I 138 A \n",
      " 140    0.27    0.41    0.31    0.06   A 141 A A 145 A A 139 A A 139 A         L 139 A \n",
      " 141    0.29    0.61    0.34    0.11   M 142 A S 146 A S 140 A Y 140 A         L 140 A \n",
      " 142    0.13    0.53    0.37    0.09   K 143 A A 147 A K 141 A K 141 A         K 141 A \n",
      " 143    0.16    0.54    0.36    0.08   S 144 A A 148 A S 142 A N 142 A         S 142 A \n",
      " 144    0.14    1.16    0.50    0.21 * A 145 A A 149 A A 143 A A 143 A         A 143 A \n",
      " 145    0.36    2.46    1.18    0.47   P 146 A P 150 A P 144 A P 144 A         Q 144 A \n",
      " 146    0.26    3.58    1.17    0.85   D 147 A D 151 A S 145 A G 145 A         G 145 A \n",
      " 147    0.00    0.00    0.00    0.00   -       -       -       -               K 146 A \n",
      " 148    0.13    0.87    0.43    0.16   L 148 A V 152 A I 146 A L 146 A         L 147 A \n",
      " 149    0.26    0.86    0.49    0.21   P 149 A P 153 A P 147 A N 147 A         N 148 A \n",
      " 150    0.23    0.93    0.59    0.21   K 150 A A 154 A K 148 A P 148 A         P 149 A \n",
      " 151    0.23    1.02    0.59    0.19   K 151 A S 155 A E 149 A R 149 A         R 150 A \n",
      " 152    0.29    0.56    0.38    0.09   N 152 A R 156 A N 150 A N 150 A         H 151 A \n",
      " 153    0.29    0.53    0.36    0.10   F 153 A F 157 A F 151 A F 151 A         V 152 A \n",
      " 154    0.14    0.41    0.31    0.09   T 154 A N 158 A S 152 A T 152 A         T 153 A \n",
      " 155    0.15    0.37    0.26    0.15   A 155 A A 159 A C 153 A A 153 A         A 154 A \n",
      " 156    0.14    0.40    0.19    0.09   M 156 A M 160 A L 154 A M 154 A         M 155 A \n",
      " 157    0.19    0.23    0.22    0.06   L 157 A M 161 A T 155 A T 155 A         T 156 A \n",
      " 158    0.07    0.25    0.22    0.04 * R 158 A R 162 A R 156 A R 156 A         R 157 A \n",
      " 159    0.05    0.33    0.28    0.04 * L 159 A L 163 A L 157 A L 157 A         L 158 A \n",
      " 160    0.06    0.32    0.26    0.07 * D 160 A D 164 A D 158 A D 158 A         D 159 A \n",
      " 161    0.07    0.31    0.22    0.02 * H 161 A H 165 A H 159 A H 159 A         H 160 A \n",
      " 162    0.10    0.27    0.20    0.05 * N 162 A N 166 A N 160 A N 160 A         N 161 A \n",
      " 163    0.07    0.30    0.27    0.10 * R 163 A R 167 A R 161 A R 161 A         R 162 A \n",
      " 164    0.08    0.32    0.22    0.11 * A 164 A A 168 A A 162 A A 162 A         A 163 A \n",
      " 165    0.21    0.53    0.22    0.11   L 165 A I 169 A K 163 A K 163 A         L 164 A \n",
      " 166    0.14    0.49    0.31    0.09   S 166 A S 170 A A 164 A A 164 A         S 165 A \n",
      " 167    0.32    0.61    0.43    0.04   Q 167 A Q 171 A Q 165 A Q 165 A         L 166 A \n",
      " 168    0.09    0.56    0.31    0.12   L 168 A L 172 A I 166 A L 166 A         L 167 A \n",
      " 169    0.08    0.41    0.40    0.07 * A 169 A A 173 A A 167 A A 167 A         A 168 A \n",
      " 170    0.28    0.36    0.51    0.15   A 170 A T 174 A L 168 A K 168 A         R 169 A \n",
      " 171    0.08    0.71    0.53    0.14 * K 171 A K 175 A K 169 A K 169 A         K 170 A \n",
      " 172    0.25    0.62    0.64    0.21   S 172 A L 176 A L 170 A T 170 A         A 171 A \n",
      " 173    0.03    0.73    0.77    0.22 * G 173 A G 177 A G 171 A G 171 A         G 172 A \n",
      " 174    0.25    0.74    0.77    0.35   K 174 A R 178 A V 172 A T 172 A         V 173 A \n",
      " 175    0.24    0.88    0.77    0.25   P 175 A G 179 A T 173 A G 173 A         P 174 A \n",
      " 176    0.19    0.61    0.62    0.13   V 176 A S 180 A S 174 A V 174 A         V 175 A \n",
      " 177    0.24    1.14    0.63    0.24   A 177 A A 181 A D 175 A D 175 A         S 176 A \n",
      " 178    0.29    0.54    0.60    0.28   S 178 A E 182 A D 176 A R 176 A         Q 177 A \n",
      " 179    0.18    1.02    0.72    0.29   I 179 A F 183 A V 177 A I 177 A         V 178 A \n",
      " 180    0.23    1.84    1.05    0.45   E 180 A N 184 A K 178 A R 178 A         R 179 A \n",
      " 181    0.21    1.59    0.88    0.36   K 181 A N 185 A N 179 A R 179 A         N 180 A \n",
      " 182    0.18    1.22    0.76    0.24   L 182 A I 186 A V 180 A M 180 A         V 181 A \n",
      " 183    0.21    0.78    0.45    0.22   A 183 A V 187 A I 181 A T 181 A         I 182 A \n",
      " 184    0.14    0.74    0.47    0.19   V 184 A V 188 A I 182 A V 182 A         I 183 A \n",
      " 185    0.04    0.48    0.25    0.07 * W 185 A W 189 A W 183 A W 183 A         W 184 A \n",
      " 186    0.04    0.81    0.48    0.10 * G 186 A G 190 A G 184 A G 184 A         G 185 A \n",
      " 187    0.11    1.00    0.69    0.36 * N 187 A N 191 A N 185 A N 185 A         N 186 A \n",
      " 188    0.08    1.00    0.61    0.27 * H 188 A H 192 A H 186 A H 186 A         H 187 A \n",
      " 189    0.08    1.16    0.63    0.19 * S 189 A S 193 A S 187 A S 187 A         S 188 A \n",
      " 190    0.16    1.49    0.75    0.30   P 190 A A 194 A S 188 A S 188 A         S 189 A \n",
      " 191    0.13    1.60    0.74    0.34   T 191 A T 195 A T 189 A I 189 A         T 190 A \n",
      " 192    0.22    1.04    0.48    0.20   M 192 A Q 196 A Q 190 A M 190 A         Q 191 A \n",
      " 193    0.30    0.98    0.45    0.17   Y 193 A F 197 A Y 191 A F 191 A         V 192 A \n",
      " 194    0.05    1.12    0.43    0.24 * P 194 A P 198 A P 192 A P 192 A         P 193 A \n",
      " 195    0.07    0.78    0.37    0.14 * D 195 A D 199 A D 193 A D 193 A         D 194 A \n",
      " 196    0.31    1.00    0.58    0.17   F 196 A I 200 A V 194 A L 194 A         T 195 A \n",
      " 197    0.31    0.92    0.46    0.23   R 197 A T 201 A N 195 A F 195 A         D 196 A \n",
      " 198    0.33    0.94    0.54    0.20   F 198 A Y 202 A H 196 A H 196 A         S 197 A \n",
      " 199    0.08    0.97    0.50    0.20 * A 199 A A 203 A A 197 A A 197 A         A 198 A \n",
      " 200    0.31    0.95    0.58    0.20   T 200 A T 204 A K 198 A E 198 A         V 199 A \n",
      " 201    0.25    0.71    0.72    0.14   A 201 A V 205 A V 199 A V 199 A         I 200 A \n",
      " 202    0.00    0.00    0.00    0.00   -       -       K 200 A -               -       \n",
      " 203    0.00    0.00    0.00    0.00   -       -       L 201 A -               -       \n",
      " 204    0.00    0.00    0.00    0.00   -       -       Q 202 A -               -       \n",
      " 205    0.00    0.00    0.00    0.00   -       -       A 203 A -               -       \n",
      " 206    0.00    0.00    2.24    1.22   E 202 A G 206 A K 204 A D 200 A         -       \n",
      " 207    0.00    0.00    1.10    0.23   G 203 A G 207 A E 205 A G 205 A         -       \n",
      " 208    0.00    0.00    0.68    0.16   E 204 A E 208 A V 206 A R 206 A         -       \n",
      " 209    0.00    0.00    0.65    0.20   S 205 A K 209 A G 207 A P 207 A         -       \n",
      " 210    0.00    0.00    0.47    0.20   L 206 A V 210 A V 208 A A 208 A         -       \n",
      " 211    0.00    0.00    0.45    0.21   L 207 A T 211 A Y 209 A L 209 A         -       \n",
      " 212    0.00    0.00    0.49    0.22   K 208 A D 212 A E 210 A E 210 A         -       \n",
      " 213    0.37   11.93    3.87    2.89   L 209 A L 213 A A 211 A L 211 A         G 201 A \n",
      " 214    0.35   14.54    4.73    3.49   I 210 A V 214 A V 212 A V 212 A         T 202 A \n",
      " 215    0.43   12.33    5.57    1.89   N 211 A -       K 213 A -               T 203 A \n",
      " 216    0.32   12.55    4.05    3.04   D 212 A D 215 A D 214 A D 214 A         P 204 A \n",
      " 217    0.32   11.12    3.57    2.66   D 213 A H 216 A D 215 A M 215 A         A 205 A \n",
      " 218    0.33   11.18    3.62    2.67   V 214 A D 217 A S 216 A E 216 A         R 206 A \n",
      " 219    0.44   13.35    4.31    3.21   W 215 A W 218 A W 217 A W 217 A         E 207 A \n",
      " 220    0.40   12.59    4.07    3.02   N 216 A Y 219 A L 218 A Y 218 A         A 208 A \n",
      " 221    0.77   13.38    4.39    3.17   R 217 A V 220 A K 219 A E 219 A         I 209 A \n",
      " 222    0.70   14.89    4.88    3.54   D 218 A E 221 A G 220 A K 220 A         K 210 A \n",
      " 223    0.74    0.83    0.69    0.33   T 219 A E 222 A E 221 A V 221 A         D 217 A \n",
      " 224    0.52    0.79    0.46    0.19 * F 220 A F 223 A F 222 A F 222 A         F 218 A \n",
      " 225    0.16    0.80    0.46    0.23   I 221 A I 224 A I 223 A I 223 A         V 219 A \n",
      " 226    0.32    0.33    0.47    0.30   P 222 A P 225 A T 224 A P 224 A         Q 220 A \n",
      " 227    0.28    0.46    0.46    0.33   T 223 A R 226 A T 225 A T 225 A         V 221 A \n",
      " 228    0.07    0.69    0.54    0.30 * V 224 A V 227 A V 226 A V 226 A         V 222 A \n",
      " 229    0.30    0.64    0.69    0.31   G 225 A A 228 A Q 227 A A 227 A         R 223 A \n",
      " 230    0.30    0.19    0.69    0.52   K 226 A N 229 A Q 228 A Q 228 A         G 224 A \n",
      " 231    0.07    0.35    0.57    0.46 * R 227 A R 230 A R 229 A R 229 A         R 225 A \n",
      " 232    0.03    0.34    0.73    0.49 * G 228 A G 231 A G 230 A G 230 A         G 226 A \n",
      " 233    0.08    0.48    1.00    0.63 * A 229 A A 232 A A 231 A A 231 A         A 227 A \n",
      " 234    0.21    0.52    0.93    0.65   A 230 A E 233 A A 232 A A 232 A         E 228 A \n",
      " 235    0.10    0.58    0.81    0.53   I 231 A I 234 A V 233 A I 233 A         I 229 A \n",
      " 236    0.08    0.40    0.96    0.63 * I 232 A I 235 A I 234 A I 234 A         I 230 A \n",
      " 237    0.19    0.25    1.13    0.82   E 233 A E 236 A K 235 A Q 235 A         Q 231 A \n",
      " 238    0.25    0.56    1.16    0.70   A 234 A V 237 A A 236 A A 236 A         L 232 A \n",
      " 239    0.07    0.68    1.14    0.63 * R 235 A R 238 A R 237 A R 237 A         R 233 A \n",
      " 240    0.10    0.54    1.15    0.71   G 236 A G 239 A K 238 A G 238 A         G 234 A \n",
      " 241    0.17    0.53    0.94    0.53   L 237 A K 240 A L 239 A A 239 A         L 235 A \n",
      " 242    0.07    0.54    0.90    0.42 * S 238 A S 241 A S 240 A S 240 A         S 236 A \n",
      " 243    0.07    0.55    0.63    0.25 * S 239 A S 242 A S 241 A S 241 A         S 237 A \n",
      " 244    0.08    0.25    0.30    0.11 * A 240 A A 243 A A 242 A A 242 A         A 238 A \n",
      " 245    0.27    0.15    0.24    0.11   A 241 A A 244 A M 243 A A 243 A         M 239 A \n",
      " 246    0.07    0.38    0.31    0.05 * S 242 A S 245 A S 244 A S 244 A         S 240 A \n",
      " 247    0.08    0.44    0.25    0.07 * A 243 A A 246 A A 245 A A 245 A         A 241 A \n",
      " 248    0.08    0.33    0.19    0.05 * A 244 A A 247 A A 246 A A 246 A         A 242 A \n",
      " 249    0.21    0.26    0.19    0.08   N 245 A S 248 A K 247 A N 247 A         K 243 A \n",
      " 250    0.11    0.33    0.22    0.05   A 246 A S 249 A A 248 A A 248 A         A 244 A \n",
      " 251    0.13    0.33    0.26    0.05   A 247 A A 250 A I 249 A A 249 A         A 245 A \n",
      " 252    0.21    0.43    0.34    0.07   I 248 A I 251 A C 250 A I 250 A         V 246 A \n",
      " 253    0.10    0.27    0.26    0.04   D 249 A D 252 A D 251 A E 251 A         D 247 A \n",
      " 254    0.07    0.23    0.22    0.04 * H 250 A H 253 A H 252 A H 252 A         H 248 A \n",
      " 255    0.14    0.29    0.25    0.09   V 251 A M 254 A V 253 A I 253 A         V 249 A \n",
      " 256    0.31    0.24    0.22    0.06   R 252 A R 255 A R 254 A R 254 A         H 250 A \n",
      " 257    0.06    0.22    0.20    0.06 * D 253 A D 256 A D 255 A D 255 A         D 251 A \n",
      " 258    0.12    0.27    0.37    0.14   W 254 A W 257 A I 256 A W 256 A         W 252 A \n",
      " 259    0.25    0.42    0.37    0.22   V 255 A V 258 A W 257 A A 257 A         I 253 A \n",
      " 260    0.34    0.13    0.22    0.10   L 256 A Q 259 A F 258 A L 258 A         H 254 A \n",
      " 261    0.05    0.11    0.32    0.20 * G 257 A G 260 A G 259 A G 259 A         G 255 A \n",
      " 262    0.09    0.19    0.30    0.11 * T 258 A T 261 A T 260 A T 260 A         T 256 A \n",
      " 263    0.24    1.38    1.17    0.63   -       E 262 A P 261 A P 261 A         P 257 A \n",
      " 264    0.22    1.85    2.49    1.28   N 259 A A 263 A E 262 A E 262 A         E 258 A \n",
      " 265    0.17    1.28    1.49    0.78   G 260 A -       G 263 A G 263 A         G 259 A \n",
      " 266    0.39    0.67    0.66    0.13   K 261 A -       E 264 A D 264 A         V 260 A \n",
      " 267    0.28    0.30    0.58    0.26   W 262 A W 264 A F 265 A W 265 A         Y 261 A \n",
      " 268    0.14    0.34    0.40    0.21   V 263 A S 265 A V 266 A V 266 A         V 262 A \n",
      " 269    0.10    0.29    0.21    0.07   T 264 A S 266 A S 267 A S 267 A         S 263 A \n",
      " 270    0.17    0.32    0.28    0.10   M 265 A A 267 A M 268 A M 268 A         M 264 A \n",
      " 271    0.13    0.46    0.28    0.10   G 266 A A 268 A G 269 A A 269 A         G 265 A \n",
      " 272    0.14    0.30    0.24    0.09   I 267 A I 269 A I 270 A V 270 A         V 266 A \n",
      " 273    0.36    0.75    0.37    0.13   P 268 A P 270 A I 271 A P 271 A         Y 267 A \n",
      " 274    0.08    0.74    0.37    0.15 * S 269 A S 271 A S 272 A S 272 A         S 268 A \n",
      " 275    0.21    1.85    0.97    0.40   D 270 A T 272 A D 273 A Q 273 A         D 269 A \n",
      " 276    0.30    3.82    2.01    0.56   G 271 A G 273 A G 274 A G 274 A         E 270 A \n",
      " 277    0.41    1.41    0.70    0.00   -       -       N 275 A -               N 271 A \n",
      " 278    0.31    2.45    1.55    0.38   S 272 A A 274 A S 276 A E 275 A         P 272 A \n",
      " 279    0.10    1.24    0.64    0.32 * Y 273 A Y 275 A Y 277 A Y 277 A         Y 273 A \n",
      " 280    0.09    2.41    1.13    0.61 * G 274 A G 276 A G 278 A G 278 A         G 274 A \n",
      " 281    0.15    0.96    0.60    0.22   I 275 A I 277 A V 279 A I 279 A         V 275 A \n",
      " 282    0.06    0.59    0.50    0.20 * P 276 A P 278 A P 280 A P 280 A         P 276 A \n",
      " 283    0.28    0.61    0.50    0.22   E 277 A E 279 A D 281 A E 281 A         S 277 A \n",
      " 284    0.16    0.72    0.88    0.38   D 278 A G 280 A D 282 A G 282 A         G 278 A \n",
      " 285    0.16    0.82    0.64    0.07   I 279 A I 281 A L 283 A I 283 A         L 279 A \n",
      " 286    0.18    0.78    0.42    0.17   I 280 A F 282 A L 284 A V 284 A         I 280 A \n",
      " 287    0.22    0.73    0.42    0.19   Y 281 A V 283 A Y 285 A Y 285 A         F 281 A \n",
      " 288    0.16    0.51    0.38    0.20   G 282 A G 284 A S 286 A S 286 A         S 282 A \n",
      " 289    0.16    0.33    0.28    0.09   V 283 A L 285 A F 287 A F 287 A         F 283 A \n",
      " 290    0.05    0.36    0.36    0.24 * P 284 A P 286 A P 288 A P 288 A         P 284 A \n",
      " 291    0.38    0.42    0.41    0.11   V 285 A T 287 A V 289 A V 289 A         C 285 A \n",
      " 292    0.18    0.11    0.45    0.29   I 286 A V 288 A T 290 A T 290 A         T 286 A \n",
      " 293    0.30    0.16    0.41    0.20   C 287 A S 289 A I 291 A A 291 A         C 287 A \n",
      " 294    0.31    0.40    0.65    0.17   E 288 A R 290 A K 292 A K 292 A         H 288 A \n",
      " 295    0.27    0.72    0.88    0.47   N 289 A N 291 A D 293 A D 293 A         A 289 A \n",
      " 296    0.09    0.23    0.42    0.18   G 290 A G 292 A K 294 A G 294 A         G 290 A \n",
      " 297    0.18    0.21    0.53    0.20   E 291 A E 293 A T 295 A A 295 A         E 291 A \n",
      " 298    0.15    0.31    0.57    0.22   Y 292 A W 294 A W 296 A Y 296 A         W 292 A \n",
      " 299    0.27    0.28    0.46    0.16   K 293 A E 295 A K 297 A R 297 A         T 293 A \n",
      " 300    0.19    0.43    0.33    0.10   R 294 A I 296 A I 298 A V 298 A         V 294 A \n",
      " 301    0.07    0.35    0.30    0.07 * V 295 A V 297 A V 299 A V 299 A         V 295 A \n",
      " 302    0.28    1.26    0.70    0.32   E 296 A E 298 A E 300 A E 300 A         S 296 A \n",
      " 303    0.06    2.43    1.22    0.43 * G 297 A G 299 A G 301 A G 301 A         G 297 A \n",
      " 304    0.40    2.75    1.10    0.61   L 298 A L 300 A L 302 A L 302 A         K 298 A \n",
      " 305    0.50    4.13    1.52    0.92   E 299 A E 301 A P 303 A E 303 A         L 299 A \n",
      " 306    0.50    5.57    1.85    1.35   I 300 A I 302 A I 304 A I 304 A         K 305 A \n",
      " 307    0.40    8.16    2.64    1.97   D 301 A S 303 A N 305 A N 305 A         Q 306 A \n",
      " 308    0.40    6.58    2.20    1.54   A 302 A D 304 A D 306 A E 306 A         R 307 A \n",
      " 309    0.29    9.46    3.10    2.23   F 303 A F 305 A F 307 A F 307 A         L 308 A \n",
      " 310    0.27    9.69    3.16    2.30   S 304 A Q 306 A S 308 A A 308 A         A 309 A \n",
      " 311    0.29    9.46    3.07    2.27   R 305 A R 307 A R 309 A R 309 A         S 310 A \n",
      " 312    0.26    9.24    2.99    2.20   E 306 A A 308 A E 310 A K 310 A         T 311 A \n",
      " 313    0.37    9.29    3.00    2.23   K 307 A R 309 A K 311 A R 311 A         I 312 A \n",
      " 314    0.31    9.61    3.11    2.29   M 308 A I 310 A M 312 A M 312 A         A 313 A \n",
      " 315    0.19    9.11    2.96    2.18   D 309 A D 311 A D 313 A E 313 A         E 314 A \n",
      " 316    0.23    9.43    3.07    2.25   G 310 A A 312 A L 314 A I 314 A         L 315 A \n",
      " 317    0.29    9.51    3.10    2.28   T 311 A N 313 A T 315 A T 315 A         Q 316 A \n",
      " 318    0.28    9.19    3.02    2.17   L 312 A A 314 A A 316 A A 316 A         E 317 A \n",
      " 319    0.23    9.18    3.03    2.16   A 313 A Q 315 A K 317 A Q 317 A         E 318 A \n",
      " 320    0.30    9.60    3.15    2.28   E 314 A E 316 A E 318 A E 318 A         R 319 A \n",
      " 321    0.27    9.30    3.01    2.22   L 315 A L 317 A L 319 A L 319 A         A 320 A \n",
      " 322    0.25    8.90    2.90    2.12   L 316 A Q 318 A A 320 A L 320 A         Q 321 A \n",
      " 323    0.26    9.66    3.20    2.27   E 317 A A 319 A E 321 A D 321 A         A 322 A \n",
      " 324    0.36    9.76    3.17    2.36   E 318 A E 320 A E 322 A E 322 A         G 323 A \n",
      " 325    0.29    6.30    2.09    1.49   R 319 A R 321 A K 323 A M 323 A         L 324 A \n",
      " 326    0.00    0.00    0.60    0.33   D 320 A E 322 A E 324 A E 324 A         -       \n",
      " 327    0.00    0.00    0.00    0.00   -       -       T 325 A -               -       \n",
      " 328    0.00    0.00    1.65    0.81   G 321 A A 323 A A 326 A Q 325 A         -       \n",
      " 329    0.00    0.00    1.25    0.52   V 322 A V 324 A F 327 A V 326 A         -       \n",
      " 330    0.00    0.00    1.69    0.76   A 323 A R 325 A E 328 A K 327 A         -       \n",
      " 331    0.00    0.00    2.39    0.71   H 324 A D 326 A F 329 A A 328 A         -       \n",
      " 332    0.00    0.00    2.40    0.69   L 325 A L 327 A L 330 A L 329 A         -       \n",
      " 333    0.00    0.00    2.86    1.04   -       L 328 A S 331 A G 330 A         -       \n",
      " 334    0.00    0.00    4.52    1.66   -       L 329 A S 332 A L 331 A         -       \n",
      " 335    0.00    0.00    6.05    1.88   -       E 330 A A 333 A I 332 A         -       \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    5\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2546064    2486.391     2.428\n",
      "\n",
      "Current alignment total score:        97.66\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -       -       -       D  23 A -       \n",
      "   2    0.00    0.00    0.00    0.00   -       -       -       -       C  24 A -       \n",
      "   3    0.00    0.00    0.00    0.00   -       -       -       -       F  25 A -       \n",
      "   4    0.00    0.00    0.00    0.00   -       -       -       -       G  26 A -       \n",
      "   5    0.00    0.00    0.00    0.00   -       -       -       -       V  27 A -       \n",
      "   6    0.00    0.00    0.00    0.00   -       -       -       -       F  28 A -       \n",
      "   7    0.00    0.00    0.00    0.00   -       -       -       -       C  29 A -       \n",
      "   8    0.00    0.00    0.00    0.00   -       -       -       -       T  30 A -       \n",
      "   9    0.48    0.37    0.18    0.00   -       -       -       M   0 A W  43 A -       \n",
      "  10    0.25    1.88    1.19    0.70   K   3 A -       S   1 A K   1 A K  44 A M   1 A \n",
      "  11    0.26    2.05    1.13    0.61   P   4 A S   8 A E   2 A A   2 A K  45 A S   2 A \n",
      "  12    0.30    1.13    0.62    0.32   A   5 A T   9 A P   3 A P   3 A L  46 A A   3 A \n",
      "  13    0.18    0.48    0.30    0.18   K   6 A K  10 A I   4 A V   4 A V  47 A V   4 A \n",
      "  14    0.28    0.37    0.27    0.10   R   7 A K  11 A R   5 A R   5 A N  48 A K   5 A \n",
      "  15    0.16    0.34    0.22    0.06   V   8 A V  12 A V   6 A V   6 A I  49 A V   6 A \n",
      "  16    0.15    0.47    0.29    0.08   A   9 A T  13 A L   7 A A   7 A A  50 A A   7 A \n",
      "  17    0.06    0.59    0.34    0.09 * V  10 A V  14 A V   8 A V   8 A V  51 A V   8 A \n",
      "  18    0.18    0.46    0.27    0.09   T  11 A T  15 A T   9 A T   9 A S  52 A T   9 A \n",
      "  19    0.03    0.28    0.30    0.12 * G  12 A G  16 A G  10 A G  10 A G  53 A G  10 A \n",
      "  20    0.08    0.62    0.37    0.08 * A  13 A A  17 A A  11 A A  11 A A  54 A A  11 A \n",
      "  21    0.08    0.58    0.38    0.06 * A  14 A A  18 A A  12 A A  12 A A  55 A A  12 A \n",
      "  22    0.03    0.59    0.28    0.11 * G  15 A G  19 A G  13 A G  13 A G  56 A G  13 A \n",
      "  23    0.34    0.50    0.24    0.12   Q  16 A Q  20 A Q  14 A Q  14 A M  57 A Q  14 A \n",
      "  24    0.07    0.50    0.21    0.10 * I  17 A I  21 A I  15 A I  15 A I  58 A I  15 A \n",
      "  25    0.20    0.42    0.29    0.08   A  18 A S  22 A A  16 A G  16 A S  59 A G  16 A \n",
      "  26    0.31    0.46    0.23    0.07   Y  19 A Y  23 A Y  17 A Y  17 A N  60 A Y  17 A \n",
      "  27    0.33    1.01    0.38    0.23   S  20 A S  24 A S  18 A S  18 A H  61 A A  18 A \n",
      "  28    0.06    0.49    0.31    0.10 * L  21 A L  25 A L  19 A L  19 A L  62 A L  19 A \n",
      "  29    0.09    0.41    0.28    0.13   L  22 A L  26 A L  20 A L  20 A L  63 A V  20 A \n",
      "  30    0.20    0.42    0.39    0.19   F  23 A W  27 A Y  21 A F  21 A F  64 A P  21 A \n",
      "  31    0.22    0.23    0.43    0.14   R  24 A R  28 A S  22 A R  22 A K  65 A L  22 A \n",
      "  32    0.18    0.37    0.34    0.14   I  25 A I  29 A I  23 A I  23 A L  66 A I  23 A \n",
      "  33    0.10    0.35    0.28    0.10   A  26 A A  30 A G  24 A A  24 A A  67 A A  24 A \n",
      "  34    0.23    0.68    0.38    0.18   N  27 A N  31 A N  25 A A  25 A S  68 A R  25 A \n",
      "  35    0.03    0.66    0.45    0.17 * G  28 A G  32 A G  26 A G  26 A G  69 A G  26 A \n",
      "  36    0.18    1.50    0.84    0.24   D  29 A E  33 A S  27 A E  27 A E  70 A A  27 A \n",
      "  37    0.15    1.55    0.85    0.25   L  30 A V  34 A V  28 A M  28 A V  71 A L  28 A \n",
      "  38    0.15    1.20    0.64    0.28   L  31 A F  35 A F  29 A L  29 A F  72 A L  29 A \n",
      "  39    0.03    1.16    0.65    0.24 * G  32 A G  36 A G  30 A G  30 A G  73 A G  30 A \n",
      "  40    0.26    0.97    0.54    0.23   K  33 A T  37 A K  31 A K  31 A Q  74 A P  31 A \n",
      "  41    0.11    0.53    0.59    0.16   D  34 A D  38 A D  32 A D  32 A D  75 A T  32 A \n",
      "  42    0.16    0.35    0.41    0.19   Q  35 A T  39 A Q  33 A Q  33 A Q  76 A T  33 A \n",
      "  43    0.04    0.30    0.41    0.12 * P  36 A P  40 A P  34 A P  34 A P  77 A P  34 A \n",
      "  44    0.14    0.38    0.30    0.06   V  37 A V  41 A I  35 A V  35 A I  78 A V  35 A \n",
      "  45    0.28    0.23    0.37    0.20   I  38 A E  42 A I  36 A I  36 A A  79 A E  36 A \n",
      "  46    0.06    0.28    0.26    0.12 * L  39 A L  43 A L  37 A L  37 A L  80 A L  37 A \n",
      "  47    0.20    0.39    0.27    0.06   Q  40 A K  44 A V  38 A Q  38 A K  81 A R  38 A \n",
      "  48    0.06    0.25    0.36    0.12 * L  41 A L  45 A L  39 A L  39 A L  82 A L  39 A \n",
      "  49    0.05    0.34    0.32    0.11 * L  42 A L  46 A L  40 A L  40 A L  83 A L  40 A \n",
      "  50    0.27    0.31    0.38    0.10   D  43 A E  47 A D  41 A E  41 A G  84 A D  41 A \n",
      "  51    0.33    0.63    0.50    0.23   L  44 A I  48 A I  42 A I  42 A S  85 A I  42 A \n",
      "  52    0.25    0.80    0.57    0.34   P  45 A P  49 A T  43 A P  43 A E  86 A E  43 A \n",
      "  53    0.31    1.87    0.84    0.55   Q  46 A Q  50 A P  44 A Q  44 A R  87 A P  44 A \n",
      "  54    0.25    0.77    0.54    0.28   A  47 A A  51 A M  45 A A  45 A S  88 A A  45 A \n",
      "  55    0.29    0.95    0.58    0.32   Q  48 A L  52 A M  46 A M  46 A F  89 A L  46 A \n",
      "  56    0.27    0.87    0.58    0.32   A  49 A G  53 A G  47 A K  47 A Q  90 A K  47 A \n",
      "  57    0.13    0.84    0.53    0.28   A  50 A G  54 A V  48 A A  48 A A  91 A A  48 A \n",
      "  58    0.12    0.91    0.58    0.35   V  51 A A  55 A L  49 A L  49 A L  92 A L  49 A \n",
      "  59    0.17    0.83    0.59    0.43   K  52 A E  56 A D  50 A E  50 A E  93 A A  50 A \n",
      "  60    0.03    0.77    0.51    0.39 * G  53 A G  57 A G  51 A G  51 A G  94 A G  51 A \n",
      "  61    0.06    0.56    0.46    0.26 * V  54 A V  58 A V  52 A V  52 A V  95 A V  52 A \n",
      "  62    0.21    0.86    0.60    0.37   V  55 A A  59 A L  53 A V  53 A A  96 A E  53 A \n",
      "  63    0.15    1.04    0.63    0.50   M  56 A M  60 A M  54 A M  54 A M  97 A A  54 A \n",
      "  64    0.08    0.82    0.48    0.34 * E  57 A E  61 A E  55 A E  55 A E  98 A E  55 A \n",
      "  65    0.05    0.97    0.49    0.31 * L  58 A L  62 A L  56 A L  56 A L  99 A L  56 A \n",
      "  66    0.19    1.29    0.72    0.53   D  59 A L  63 A Q  57 A E  57 A E 100 A E  57 A \n",
      "  67    0.06    1.13    0.71    0.58 * D  60 A D  64 A D  58 A D  58 A D 101 A D  58 A \n",
      "  68    0.34    1.20    0.62    0.50   C  61 A S  65 A C  59 A C  59 A S 102 A C  59 A \n",
      "  69    0.27    1.14    0.83    0.66   A  62 A A  66 A A  60 A A  60 A L 103 A A  60 A \n",
      "  70    0.21    0.52    0.37    0.10   F  63 A F  67 A L  61 A F  61 A Y 104 A F  61 A \n",
      "  71    0.04    0.79    0.56    0.18 * P  64 A P  68 A P  62 A P  62 A P 105 A P  62 A \n",
      "  72    0.06    0.11    0.28    0.15 * L  65 A L  69 A L  63 A L  63 A L 106 A L  63 A \n",
      "  73    0.06    0.30    0.24    0.10 * L  66 A L  70 A L  64 A L  64 A L 107 A L  64 A \n",
      "  74    0.24    0.25    0.37    0.17   A  67 A R  71 A K  65 A A  65 A R 108 A D  65 A \n",
      "  75    0.26    0.71    0.41    0.14   G  68 A N  72 A D  66 A G  66 A E 109 A K  66 A \n",
      "  76    0.11    0.37    0.37    0.05   V  69 A I  73 A V  67 A L  67 A V 110 A V  67 A \n",
      "  77    0.28    0.46    0.37    0.14   V  70 A T  74 A I  68 A E  68 A S 111 A V  68 A \n",
      "  78    0.18    0.74    0.48    0.19   I  71 A I  75 A A  69 A A  69 A I 112 A V  69 A \n",
      "  79    0.32    0.57    0.44    0.16   T  72 A T  76 A T  70 A T  70 A G 113 A T  70 A \n",
      "  80    0.35    0.63    0.44    0.20   D  73 A A  77 A D  71 A D  71 A I 114 A A  71 A \n",
      "  81    0.11    0.94    0.52    0.31   D  74 A D  78 A K  72 A D  72 A D 115 A D  72 A \n",
      "  82    0.13    0.74    0.59    0.38   P  75 A A  79 A E  73 A P  73 A P 116 A P  73 A \n",
      "  83    0.34    1.15    0.74    0.47   K  76 A N  80 A E  74 A D  74 A Y 117 A R  74 A \n",
      "  84    0.28    1.35    0.53    0.33   V  77 A E  81 A I  75 A V  75 A E 118 A V  75 A \n",
      "  85    0.23    0.97    0.46    0.19   A  78 A A  82 A A  76 A A  76 A V 119 A A  76 A \n",
      "  86    0.06    0.41    0.41    0.22 * F  79 A F  83 A F  77 A F  77 A F 120 A F  77 A \n",
      "  87    0.23    0.44    0.36    0.14   K  80 A D  84 A K  78 A K  78 A E 121 A D  78 A \n",
      "  88    0.15    0.18    0.33    0.20   D  81 A G  85 A D  79 A D  79 A D 122 A G  79 A \n",
      "  89    0.18    0.19    0.34    0.16   A  82 A A  86 A L  80 A A  80 A V 123 A V  80 A \n",
      "  90    0.13    0.19    0.18    0.04   D  83 A N  87 A D  81 A D  81 A D 124 A A  81 A \n",
      "  91    0.37    0.32    0.21    0.09   V  84 A A  88 A V  82 A Y  82 A W 125 A I  82 A \n",
      "  92    0.08    0.54    0.31    0.19 * A  85 A A  89 A A  83 A A  83 A A 126 A A  83 A \n",
      "  93    0.14    0.35    0.31    0.12   L  86 A F  90 A I  84 A L  84 A L 127 A I  84 A \n",
      "  94    0.08    0.25    0.24    0.10   L  87 A L  91 A L  85 A L  85 A L 128 A M  85 A \n",
      "  95    0.21    0.28    0.24    0.07   V  88 A V  92 A V  86 A V  86 A I 129 A C  86 A \n",
      "  96    0.03    0.51    0.37    0.08 * G  89 A G  93 A G  87 A G  87 A G 130 A G  87 A \n",
      "  97    0.11    0.45    0.56    0.20   A  90 A A  94 A S  88 A A  88 A A 131 A A  88 A \n",
      "  98    0.28    0.66    1.13    0.47   R  91 A K  95 A M  89 A A  89 A K 132 A F  89 A \n",
      "  99    0.10    0.71    1.39    0.82 * P  92 A P  96 A P  90 A P  90 A P 133 A P  90 A \n",
      " 100    0.18    1.74    2.93    1.86   R  93 A E 102 A R  91 A         R 134 A R  91 A \n",
      " 101    0.39    4.00    2.70    1.66   S  94 A -       R  92 A -       G 135 A K  92 A \n",
      " 102    0.42    6.91    4.23    1.85   -       -       D  93 A -       P 136 A A  93 A \n",
      " 103    0.32    5.56    4.21    1.55   -       -       G  94 A -       G 137 A G  94 A \n",
      " 104    0.29    3.29    3.22    1.88   M  97 A -       M  95 A -       M 138 A M  95 A \n",
      " 105    0.26    4.12    3.39    1.96   E  98 A -       E  96 A -       E 139 A E  96 A \n",
      " 106    0.19    2.81    2.36    1.66   R  99 A R 103 A R  97 A -       R 140 A R  97 A \n",
      " 107    0.27    2.70    2.36    1.60   K 100 A A 104 A K  98 A -       A 141 A K  98 A \n",
      " 108    0.31    2.78    1.91    1.31   D 101 A D 105 A D  99 A -       A 142 A D  99 A \n",
      " 109    0.15    1.96    1.04    0.50   L 102 A L 106 A L 100 A -       L 143 A L 100 A \n",
      " 110    0.09    1.22    1.13    0.47 * L 103 A L 107 A L 101 A L 101 A L 144 A L 101 A \n",
      " 111    0.25    1.66    1.12    0.59   S 104 A A 108 A K 102 A Q 102 A D 145 A E 102 A \n",
      " 112    0.27    1.58    1.08    0.49   A 105 A N 109 A A 103 A V 103 A I 146 A M 103 A \n",
      " 113    0.11    0.78    0.79    0.29 * N 106 A N 110 A N 104 A N 104 A N 147 A N 104 A \n",
      " 114    0.18    1.02    1.06    0.31   A 107 A G 111 A V 105 A G 105 A G 148 A A 105 A \n",
      " 115    0.24    1.24    0.93    0.23   E 108 A K 112 A K 106 A K 106 A Q 149 A R 106 A \n",
      " 116    0.08    0.83    0.44    0.18 * I 109 A I 113 A I 107 A I 107 A I 150 A I 107 A \n",
      " 117    0.06    0.68    0.47    0.10 * F 110 A F 114 A F 108 A F 108 A F 151 A F 108 A \n",
      " 118    0.24    0.63    0.54    0.15   T 111 A G 115 A K 109 A T 109 A A 152 A K 109 A \n",
      " 119    0.30    0.71    0.48    0.11   V 112 A P 116 A C 110 A E 110 A D 153 A E 110 A \n",
      " 120    0.07    0.59    0.33    0.10 * Q 113 A Q 117 A Q 111 A Q 111 A Q 154 A Q 111 A \n",
      " 121    0.03    0.55    0.34    0.10 * G 114 A G 118 A G 112 A G 112 A G 155 A G 112 A \n",
      " 122    0.20    0.62    0.44    0.10   A 115 A K 119 A A 113 A R 113 A K 156 A E 113 A \n",
      " 123    0.07    0.57    0.40    0.08 * A 116 A A 120 A A 114 A A 114 A A 157 A A 114 A \n",
      " 124    0.10    0.54    0.36    0.09   L 117 A I 121 A L 115 A L 115 A L 158 A I 115 A \n",
      " 125    0.19    0.48    0.44    0.10   N 118 A N 122 A D 116 A A 116 A N 159 A A 116 A \n",
      " 126    0.21    0.49    0.47    0.11   E 119 A D 123 A K 117 A E 117 A A 160 A A 117 A \n",
      " 127    0.17    0.55    0.45    0.15   V 120 A N 124 A Y 118 A V 118 A V 161 A V 118 A \n",
      " 128    0.08    0.30    0.36    0.12 * A 121 A A 125 A A 119 A A 119 A A 162 A A 119 A \n",
      " 129    0.20    0.27    0.52    0.26   S 122 A A 126 A K 120 A K 120 A S 163 A A 120 A \n",
      " 130    0.17    0.62    0.77    0.42   R 123 A D 127 A K 121 A K 121 A K 164 A S 121 A \n",
      " 131    0.21    0.88    0.59    0.45   D 124 A D 128 A S 122 A D 122 A N 165 A D 122 A \n",
      " 132    0.15    0.34    0.23    0.08   V 125 A I 129 A V 123 A V 123 A V 166 A C 123 A \n",
      " 133    0.13    0.35    0.22    0.08   K 126 A R 130 A K 124 A K 124 A K 167 A R 124 A \n",
      " 134    0.06    0.36    0.26    0.09 * V 127 A V 131 A V 125 A V 125 A V 168 A V 125 A \n",
      " 135    0.11    0.17    0.24    0.12   L 128 A L 132 A I 126 A L 126 A L 169 A V 126 A \n",
      " 136    0.07    0.10    0.14    0.08 * V 129 A V 133 A V 127 A V 127 A V 170 A V 127 A \n",
      " 137    0.07    0.58    0.29    0.12 * V 130 A V 134 A V 128 A V 128 A V 171 A V 128 A \n",
      " 138    0.03    1.05    0.38    0.25 * G 131 A G 135 A G 129 A G 129 A G 172 A G 129 A \n",
      " 139    0.10    0.97    0.43    0.28 * N 132 A N 136 A N 130 A N 130 A N 173 A N 130 A \n",
      " 140    0.04    0.97    0.46    0.21 * P 133 A P 137 A P 131 A P 131 A P 174 A P 131 A \n",
      " 141    0.35    0.78    0.39    0.17   A 134 A A 138 A A 132 A A 132 A C 175 A A 132 A \n",
      " 142    0.10    0.67    0.36    0.16 * N 135 A N 139 A N 133 A N 133 A N 176 A N 133 A \n",
      " 143    0.07    0.68    0.37    0.13 * T 136 A T 140 A T 134 A T 134 A T 177 A T 134 A \n",
      " 144    0.10    0.75    0.35    0.16 * N 137 A N 141 A N 135 A N 135 A N 178 A N 135 A \n",
      " 145    0.13    0.58    0.38    0.08   A 138 A A 142 A C 136 A A 136 A A 179 A A 136 A \n",
      " 146    0.10    0.63    0.42    0.07   Y 139 A L 143 A L 137 A L 137 A L 180 A L 137 A \n",
      " 147    0.12    0.79    0.44    0.11   I 140 A I 144 A T 138 A I 138 A I 181 A I 138 A \n",
      " 148    0.36    0.56    0.34    0.09   A 141 A A 145 A A 139 A A 139 A C 182 A L 139 A \n",
      " 149    0.24    0.75    0.41    0.13   M 142 A S 146 A S 140 A Y 140 A L 183 A L 140 A \n",
      " 150    0.12    0.70    0.40    0.16   K 143 A A 147 A K 141 A K 141 A K 184 A K 141 A \n",
      " 151    0.21    0.79    0.41    0.17   S 144 A A 148 A S 142 A N 142 A N 185 A S 142 A \n",
      " 152    0.08    0.63    0.50    0.23 * A 145 A A 149 A A 143 A A 143 A A 186 A A 143 A \n",
      " 153    0.10    1.49    1.14    0.59   P 146 A P 150 A P 144 A P 144 A P 187 A Q 144 A \n",
      " 154    0.19    1.45    1.07    0.94   D 147 A D 151 A S 145 A G 145 A D 188 A G 145 A \n",
      " 155    0.00    0.00    0.00    0.00   -       -       -       -       -       K 146 A \n",
      " 156    0.17    0.55    0.43    0.17   L 148 A V 152 A I 146 A L 146 A I 189 A L 147 A \n",
      " 157    0.15    0.31    0.45    0.22   P 149 A P 153 A P 147 A N 147 A P 190 A N 148 A \n",
      " 158    0.22    0.22    0.52    0.26   K 150 A A 154 A K 148 A P 148 A A 191 A P 149 A \n",
      " 159    0.19    0.49    0.56    0.20   K 151 A S 155 A E 149 A R 149 A K 192 A R 150 A \n",
      " 160    0.18    0.40    0.37    0.10   N 152 A R 156 A N 150 A N 150 A N 193 A H 151 A \n",
      " 161    0.11    0.40    0.36    0.08   F 153 A F 157 A F 151 A F 151 A F 194 A V 152 A \n",
      " 162    0.33    0.47    0.33    0.08   T 154 A N 158 A S 152 A T 152 A H 195 A T 153 A \n",
      " 163    0.14    0.65    0.32    0.18   A 155 A A 159 A C 153 A A 153 A A 196 A A 154 A \n",
      " 164    0.17    0.25    0.21    0.07   M 156 A M 160 A L 154 A M 154 A L 197 A M 155 A \n",
      " 165    0.17    0.55    0.27    0.12   L 157 A M 161 A T 155 A T 155 A T 198 A T 156 A \n",
      " 166    0.07    0.22    0.21    0.05 * R 158 A R 162 A R 156 A R 156 A R 199 A R 157 A \n",
      " 167    0.05    0.36    0.29    0.04 * L 159 A L 163 A L 157 A L 157 A L 200 A L 158 A \n",
      " 168    0.06    0.48    0.29    0.09 * D 160 A D 164 A D 158 A D 158 A D 201 A D 159 A \n",
      " 169    0.32    0.37    0.24    0.06   H 161 A H 165 A H 159 A H 159 A E 202 A H 160 A \n",
      " 170    0.10    0.25    0.20    0.05 * N 162 A N 166 A N 160 A N 160 A N 203 A N 161 A \n",
      " 171    0.07    0.35    0.28    0.09 * R 163 A R 167 A R 161 A R 161 A R 204 A R 162 A \n",
      " 172    0.07    0.48    0.27    0.11 * A 164 A A 168 A A 162 A A 162 A A 205 A A 163 A \n",
      " 173    0.22    0.46    0.26    0.11   L 165 A I 169 A K 163 A K 163 A K 206 A L 164 A \n",
      " 174    0.38    0.46    0.33    0.08   S 166 A S 170 A A 164 A A 164 A C 207 A S 165 A \n",
      " 175    0.12    0.55    0.43    0.08   Q 167 A Q 171 A Q 165 A Q 165 A Q 208 A L 166 A \n",
      " 176    0.08    0.83    0.40    0.16   L 168 A L 172 A I 166 A L 166 A L 209 A L 167 A \n",
      " 177    0.07    0.92    0.48    0.17 * A 169 A A 173 A A 167 A A 167 A A 210 A A 168 A \n",
      " 178    0.25    1.00    0.59    0.17   A 170 A T 174 A L 168 A K 168 A L 211 A R 169 A \n",
      " 179    0.08    1.18    0.63    0.21 * K 171 A K 175 A K 169 A K 169 A K 212 A K 170 A \n",
      " 180    0.21    1.24    0.73    0.23   S 172 A L 176 A L 170 A T 170 A A 213 A A 171 A \n",
      " 181    0.03    1.54    0.89    0.28 * G 173 A G 177 A G 171 A G 171 A G 214 A G 172 A \n",
      " 182    0.21    1.12    0.83    0.29   K 174 A R 178 A V 172 A T 172 A V 215 A V 173 A \n",
      " 183    0.39    1.57    0.90    0.28   P 175 A G 179 A T 173 A G 173 A F 216 A P 174 A \n",
      " 184    0.32    0.99    0.68    0.14   V 176 A S 180 A S 174 A V 174 A Y 217 A V 175 A \n",
      " 185    0.18    1.05    0.71    0.19   A 177 A A 181 A D 175 A D 175 A D 218 A S 176 A \n",
      " 186    0.24    1.14    0.68    0.28   S 178 A E 182 A D 176 A R 176 A K 219 A Q 177 A \n",
      " 187    0.15    0.95    0.75    0.25   I 179 A F 183 A V 177 A I 177 A V 220 A V 178 A \n",
      " 188    0.26    1.20    1.06    0.40   E 180 A N 184 A K 178 A R 178 A S 221 A R 179 A \n",
      " 189    0.17    0.64    0.82    0.37   K 181 A N 185 A N 179 A R 179 A N 222 A N 180 A \n",
      " 190    0.15    0.93    0.77    0.25   L 182 A I 186 A V 180 A M 180 A V 223 A V 181 A \n",
      " 191    0.23    0.91    0.50    0.27   A 183 A V 187 A I 181 A T 181 A T 224 A I 182 A \n",
      " 192    0.12    0.83    0.51    0.23   V 184 A V 188 A I 182 A V 182 A I 225 A I 183 A \n",
      " 193    0.03    0.63    0.32    0.10 * W 185 A W 189 A W 183 A W 183 A W 226 A W 184 A \n",
      " 194    0.03    0.62    0.50    0.04 * G 186 A G 190 A G 184 A G 184 A G 227 A G 185 A \n",
      " 195    0.10    0.98    0.71    0.38 * N 187 A N 191 A N 185 A N 185 A N 228 A N 186 A \n",
      " 196    0.07    0.88    0.64    0.27 * H 188 A H 192 A H 186 A H 186 A H 229 A H 187 A \n",
      " 197    0.07    0.67    0.63    0.18 * S 189 A S 193 A S 187 A S 187 A S 230 A S 188 A \n",
      " 198    0.21    1.00    0.78    0.28   P 190 A A 194 A S 188 A S 188 A T 231 A S 189 A \n",
      " 199    0.11    1.04    0.79    0.28   T 191 A T 195 A T 189 A I 189 A T 232 A T 190 A \n",
      " 200    0.18    0.79    0.53    0.16   M 192 A Q 196 A Q 190 A M 190 A Q 233 A Q 191 A \n",
      " 201    0.25    0.75    0.51    0.13   Y 193 A F 197 A Y 191 A F 191 A V 234 A V 192 A \n",
      " 202    0.05    0.43    0.43    0.20 * P 194 A P 198 A P 192 A P 192 A P 235 A P 193 A \n",
      " 203    0.07    0.78    0.44    0.15 * D 195 A D 199 A D 193 A D 193 A D 236 A D 194 A \n",
      " 204    0.25    1.48    0.72    0.28   F 196 A I 200 A V 194 A L 194 A F 237 A T 195 A \n",
      " 205    0.33    1.25    0.60    0.28   R 197 A T 201 A N 195 A F 195 A L 238 A D 196 A \n",
      " 206    0.29    1.84    0.73    0.46   F 198 A Y 202 A H 196 A H 196 A N 239 A S 197 A \n",
      " 207    0.08    1.90    0.74    0.45 * A 199 A A 203 A A 197 A A 197 A A 240 A A 198 A \n",
      " 208    0.22    1.80    0.74    0.47   T 200 A T 204 A K 198 A E 198 A K 241 A V 199 A \n",
      " 209    0.19    1.57    0.83    0.31   A 201 A V 205 A V 199 A V 199 A I 242 A I 200 A \n",
      " 210    0.00    0.00    0.00    0.00   -       -       K 200 A -       -       -       \n",
      " 211    0.00    0.00    0.00    0.00   -       -       L 201 A -       -       -       \n",
      " 212    0.00    0.00    0.00    0.00   -       -       Q 202 A -       -       -       \n",
      " 213    0.00    0.00    0.00    0.00   -       -       A 203 A -       -       -       \n",
      " 214    0.29    3.03    2.21    1.34   E 202 A G 206 A K 204 A D 200 A D 243 A -       \n",
      " 215    0.20    2.89    1.46    0.46   G 203 A G 207 A E 205 A G 205 A G 244 A -       \n",
      " 216    0.32    2.20    0.94    0.53   E 204 A E 208 A V 206 A R 206 A R 245 A -       \n",
      " 217    0.28    1.64    0.78    0.42   S 205 A K 209 A G 207 A P 207 A P 246 A -       \n",
      " 218    0.21    0.83    0.51    0.24   L 206 A V 210 A V 208 A A 208 A V 247 A -       \n",
      " 219    0.34    0.56    0.46    0.18   L 207 A T 211 A Y 209 A L 209 A K 248 A -       \n",
      " 220    0.23    1.07    0.59    0.27   K 208 A D 212 A E 210 A E 210 A E 249 A -       \n",
      " 221    0.24    1.80    3.34    2.97   L 209 A L 213 A A 211 A L 211 A V 250 A G 201 A \n",
      " 222    0.20    2.42    4.08    3.59   I 210 A V 214 A V 212 A V 212 A I 251 A T 202 A \n",
      " 223    0.33    4.99    4.84    2.68   N 211 A -       K 213 A -       K 252 A T 203 A \n",
      " 224    0.36    3.74    3.62    3.19   D 212 A D 215 A D 214 A D 214 A R 253 A P 204 A \n",
      " 225    0.30    2.79    3.16    2.76   D 213 A H 216 A D 215 A M 215 A T 254 A A 205 A \n",
      " 226    0.25    2.75    3.19    2.79   V 214 A D 217 A S 216 A E 216 A K 255 A R 206 A \n",
      " 227    0.12    2.98    3.78    3.32   W 215 A W 218 A W 217 A W 217 A W 256 A E 207 A \n",
      " 228    0.27    2.44    3.55    3.11   N 216 A Y 219 A L 218 A Y 218 A L 257 A A 208 A \n",
      " 229    0.31    2.87    3.88    3.25   R 217 A V 220 A K 219 A E 219 A E 258 A I 209 A \n",
      " 230    0.27    4.29    4.39    3.69   D 218 A E 221 A G 220 A K 220 A E 259 A K 210 A \n",
      " 231    0.26    1.36    0.79    0.36   T 219 A E 222 A E 221 A V 221 A E 260 A D 217 A \n",
      " 232    0.12    0.89    0.53    0.19 * F 220 A F 223 A F 222 A F 222 A F 261 A F 218 A \n",
      " 233    0.29    0.84    0.52    0.22   I 221 A I 224 A I 223 A I 223 A T 262 A V 219 A \n",
      " 234    0.35    0.30    0.43    0.29   P 222 A P 225 A T 224 A P 224 A I 263 A Q 220 A \n",
      " 235    0.15    0.39    0.45    0.30   T 223 A R 226 A T 225 A T 225 A T 264 A V 221 A \n",
      " 236    0.07    0.50    0.52    0.27 * V 224 A V 227 A V 226 A V 226 A V 265 A V 222 A \n",
      " 237    0.24    1.03    0.73    0.30   G 225 A A 228 A Q 227 A A 227 A Q 266 A R 223 A \n",
      " 238    0.22    1.29    0.79    0.48   K 226 A N 229 A Q 228 A Q 228 A K 267 A G 224 A \n",
      " 239    0.07    1.21    0.66    0.46 * R 227 A R 230 A R 229 A R 229 A R 268 A R 225 A \n",
      " 240    0.03    1.42    0.81    0.52 * G 228 A G 231 A G 230 A G 230 A G 269 A G 226 A \n",
      " 241    0.22    1.71    1.07    0.66   A 229 A A 232 A A 231 A A 231 A G 270 A A 227 A \n",
      " 242    0.14    1.63    0.99    0.69   A 230 A E 233 A A 232 A A 232 A A 271 A E 228 A \n",
      " 243    0.19    1.45    0.89    0.54   I 231 A I 234 A V 233 A I 233 A L 272 A I 229 A \n",
      " 244    0.08    1.72    1.04    0.65 * I 232 A I 235 A I 234 A I 234 A I 273 A I 230 A \n",
      " 245    0.16    2.14    1.20    0.91   E 233 A E 236 A K 235 A Q 235 A Q 274 A Q 231 A \n",
      " 246    0.27    1.96    1.19    0.80   A 234 A V 237 A A 236 A A 236 A K 275 A L 232 A \n",
      " 247    0.38    1.97    1.22    0.68   R 235 A R 238 A R 237 A R 237 A W 276 A R 233 A \n",
      " 248    0.09    2.32    1.26    0.83   G 236 A G 239 A K 238 A G 238 A G 277 A G 234 A \n",
      " 249    0.29    1.20    0.93    0.55   L 237 A K 240 A L 239 A A 239 A R 278 A L 235 A \n",
      " 250    0.07    1.88    1.04    0.48 * S 238 A S 241 A S 240 A S 240 A S 279 A S 236 A \n",
      " 251    0.08    1.51    0.74    0.38 * S 239 A S 242 A S 241 A S 241 A S 280 A S 237 A \n",
      " 252    0.09    0.77    0.37    0.17 * A 240 A A 243 A A 242 A A 242 A A 281 A A 238 A \n",
      " 253    0.18    0.75    0.32    0.20   A 241 A A 244 A M 243 A A 243 A A 282 A M 239 A \n",
      " 254    0.07    0.91    0.39    0.21 * S 242 A S 245 A S 244 A S 244 A S 283 A S 240 A \n",
      " 255    0.24    0.76    0.34    0.15   A 243 A A 246 A A 245 A A 245 A T 284 A A 241 A \n",
      " 256    0.08    0.68    0.27    0.15 * A 244 A A 247 A A 246 A A 246 A A 285 A A 242 A \n",
      " 257    0.32    0.52    0.25    0.10   N 245 A S 248 A K 247 A N 247 A V 286 A K 243 A \n",
      " 258    0.18    0.58    0.28    0.11   A 246 A S 249 A A 248 A A 248 A S 287 A A 244 A \n",
      " 259    0.24    0.60    0.31    0.11   A 247 A A 250 A I 249 A A 249 A I 288 A A 245 A \n",
      " 260    0.29    0.51    0.36    0.07   I 248 A I 251 A C 250 A I 250 A A 289 A V 246 A \n",
      " 261    0.09    0.46    0.29    0.05   D 249 A D 252 A D 251 A E 251 A D 290 A D 247 A \n",
      " 262    0.33    0.53    0.27    0.10   H 250 A H 253 A H 252 A H 252 A A 291 A H 248 A \n",
      " 263    0.16    0.79    0.35    0.15   V 251 A M 254 A V 253 A I 253 A I 292 A V 249 A \n",
      " 264    0.22    1.16    0.40    0.27   R 252 A R 255 A R 254 A R 254 A K 293 A H 250 A \n",
      " 265    0.24    0.96    0.34    0.22   D 253 A D 256 A D 255 A D 255 A S 294 A D 251 A \n",
      " 266    0.30    0.75    0.44    0.13   W 254 A W 257 A I 256 A W 256 A L 295 A W 252 A \n",
      " 267    0.22    1.80    0.68    0.38   V 255 A V 258 A W 257 A A 257 A V 296 A I 253 A \n",
      " 268    0.35    2.12    0.63    0.53   L 256 A Q 259 A F 258 A L 258 A T 297 A H 254 A \n",
      " 269    0.32    2.62    0.83    0.61   G 257 A G 260 A G 259 A G 259 A P 298 A G 255 A \n",
      " 270    0.08    1.46    0.53    0.31 * T 258 A T 261 A T 260 A T 260 A T 299 A T 256 A \n",
      " 271    0.19    1.36    1.19    0.54   -       E 262 A P 261 A P 261 A P 300 A P 257 A \n",
      " 272    0.17    2.72    2.39    1.36   N 259 A A 263 A E 262 A E 262 A E 301 A E 258 A \n",
      " 273    0.12    2.28    1.54    0.85   G 260 A -       G 263 A G 263 A G 302 A G 259 A \n",
      " 274    0.28    1.32    0.75    0.28   K 261 A -       E 264 A D 264 A D 303 A V 260 A \n",
      " 275    0.14    0.55    0.56    0.27   W 262 A W 264 A F 265 A W 265 A W 304 A Y 261 A \n",
      " 276    0.30    0.53    0.42    0.17   V 263 A S 265 A V 266 A V 266 A F 305 A V 262 A \n",
      " 277    0.09    0.43    0.24    0.08   T 264 A S 266 A S 267 A S 267 A S 306 A S 263 A \n",
      " 278    0.31    0.83    0.36    0.20   M 265 A A 267 A M 268 A M 268 A T 307 A M 264 A \n",
      " 279    0.10    0.65    0.33    0.15   G 266 A A 268 A G 269 A A 269 A G 308 A G 265 A \n",
      " 280    0.12    0.66    0.31    0.15   I 267 A I 269 A I 270 A V 270 A V 309 A V 266 A \n",
      " 281    0.30    0.63    0.42    0.08   P 268 A P 270 A I 271 A P 271 A Y 310 A Y 267 A \n",
      " 282    0.18    0.58    0.40    0.13   S 269 A S 271 A S 272 A S 272 A T 311 A S 268 A \n",
      " 283    0.26    1.64    1.08    0.35   D 270 A T 272 A D 273 A Q 273 A T 312 A D 269 A \n",
      " 284    0.09    3.54    2.27    0.45   G 271 A G 273 A G 274 A G 274 A G 313 A E 270 A \n",
      " 285    0.35    0.78    0.65    0.20   -       -       N 275 A -       N 314 A N 271 A \n",
      " 286    0.24    1.79    1.59    0.21   S 272 A A 274 A S 276 A E 275 A P 315 A P 272 A \n",
      " 287    0.07    1.26    0.73    0.34 * Y 273 A Y 275 A Y 277 A Y 277 A Y 316 A Y 273 A \n",
      " 288    0.05    1.53    1.18    0.55 * G 274 A G 276 A G 278 A G 278 A G 317 A G 274 A \n",
      " 289    0.11    0.71    0.61    0.20   I 275 A I 277 A V 279 A I 279 A I 318 A V 275 A \n",
      " 290    0.26    0.57    0.49    0.20   P 276 A P 278 A P 280 A P 280 A A 319 A P 276 A \n",
      " 291    0.16    0.41    0.48    0.19   E 277 A E 279 A D 281 A E 281 A E 320 A S 277 A \n",
      " 292    0.19    1.02    0.89    0.36   D 278 A G 280 A D 282 A G 282 A D 321 A G 278 A \n",
      " 293    0.13    1.16    0.71    0.16   I 279 A I 281 A L 283 A I 283 A I 322 A L 279 A \n",
      " 294    0.18    0.50    0.43    0.15   I 280 A F 282 A L 284 A V 284 A V 323 A I 280 A \n",
      " 295    0.18    0.50    0.42    0.19   Y 281 A V 283 A Y 285 A Y 285 A F 324 A F 281 A \n",
      " 296    0.14    0.64    0.41    0.20   G 282 A G 284 A S 286 A S 286 A S 325 A S 282 A \n",
      " 297    0.28    0.62    0.33    0.12   V 283 A L 285 A F 287 A F 287 A M 326 A F 283 A \n",
      " 298    0.05    0.76    0.38    0.30 * P 284 A P 286 A P 288 A P 288 A P 327 A P 284 A \n",
      " 299    0.31    0.65    0.43    0.15   V 285 A T 287 A V 289 A V 289 A C 328 A C 285 A \n",
      " 300    0.32    0.46    0.43    0.28   I 286 A V 288 A T 290 A T 290 A R 329 A T 286 A \n",
      " 301    0.31    0.89    0.48    0.24   C 287 A S 289 A I 291 A A 291 A S 330 A C 287 A \n",
      " 302    0.20    1.21    0.74    0.18   E 288 A R 290 A K 292 A K 292 A K 331 A H 288 A \n",
      " 303    0.00    0.00    0.00    0.00   -       -       -       -       G 332 A -       \n",
      " 304    0.20    2.40    1.20    0.44   N 289 A N 291 A D 293 A D 293 A D 333 A A 289 A \n",
      " 305    0.10    2.10    0.74    0.49   G 290 A G 292 A K 294 A G 294 A G 334 A G 290 A \n",
      " 306    0.25    1.83    0.77    0.39   E 291 A E 293 A T 295 A A 295 A D 335 A E 291 A \n",
      " 307    0.19    1.04    0.63    0.25   Y 292 A W 294 A W 296 A Y 296 A Y 336 A W 292 A \n",
      " 308    0.23    0.93    0.52    0.21   K 293 A E 295 A K 297 A R 297 A E 337 A T 293 A \n",
      " 309    0.21    0.70    0.40    0.10   R 294 A I 296 A I 298 A V 298 A L 338 A V 294 A \n",
      " 310    0.24    0.82    0.39    0.13   V 295 A V 297 A V 299 A V 299 A A 339 A V 295 A \n",
      " 311    0.27    1.40    0.82    0.30   E 296 A E 298 A E 300 A E 300 A T 340 A S 296 A \n",
      " 312    0.28    4.75    1.88    1.00   G 297 A G 299 A G 301 A G 301 A D 341 A G 297 A \n",
      " 313    0.23    2.04    1.26    0.56   L 298 A L 300 A L 302 A L 302 A V 342 A K 298 A \n",
      " 314    0.32    2.17    1.61    0.84   E 299 A E 301 A P 303 A E 303 A S 343 A L 299 A \n",
      " 315    0.36    2.00    1.71    1.42   I 300 A I 302 A I 304 A I 304 A N 344 A K 305 A \n",
      " 316    0.22    1.38    2.30    2.01   D 301 A S 303 A N 305 A N 305 A D 345 A Q 306 A \n",
      " 317    0.20    0.71    1.90    1.57   A 302 A D 304 A D 306 A E 306 A D 346 A R 307 A \n",
      " 318    0.10    1.30    2.66    2.31   F 303 A F 305 A F 307 A F 307 A F 347 A L 308 A \n",
      " 319    0.31    1.68    2.74    2.36   S 304 A Q 306 A S 308 A A 308 A L 348 A A 309 A \n",
      " 320    0.39    2.00    2.72    2.31   R 305 A R 307 A R 309 A R 309 A W 349 A S 310 A \n",
      " 321    0.18    1.64    2.61    2.25   E 306 A A 308 A E 310 A K 310 A E 350 A T 311 A \n",
      " 322    0.18    1.24    2.57    2.29   K 307 A R 309 A K 311 A R 311 A R 351 A I 312 A \n",
      " 323    0.22    1.39    2.68    2.35   M 308 A I 310 A M 312 A M 312 A I 352 A A 313 A \n",
      " 324    0.26    1.37    2.56    2.23   D 309 A D 311 A D 313 A E 313 A K 353 A E 314 A \n",
      " 325    0.30    1.58    2.67    2.30   G 310 A A 312 A L 314 A I 314 A K 354 A L 315 A \n",
      " 326    0.20    1.25    2.66    2.34   T 311 A N 313 A T 315 A T 315 A S 355 A Q 316 A \n",
      " 327    0.23    1.31    2.60    2.23   L 312 A A 314 A A 316 A A 316 A E 356 A E 317 A \n",
      " 328    0.22    1.75    2.65    2.22   A 313 A Q 315 A K 317 A Q 317 A A 357 A E 318 A \n",
      " 329    0.12    1.31    2.72    2.34   E 314 A E 316 A E 318 A E 318 A E 358 A R 319 A \n",
      " 330    0.10    1.15    2.58    2.28   L 315 A L 317 A L 319 A L 319 A L 359 A A 320 A \n",
      " 331    0.20    1.76    2.55    2.17   L 316 A Q 318 A A 320 A L 320 A L 360 A Q 321 A \n",
      " 332    0.19    1.65    2.79    2.32   E 317 A A 319 A E 321 A D 321 A A 361 A A 322 A \n",
      " 333    0.13    1.17    2.71    2.42   E 318 A E 320 A E 322 A E 322 A E 362 A G 323 A \n",
      " 334    0.22    0.55    1.79    1.53   R 319 A R 321 A K 323 A M 323 A K 363 A L 324 A \n",
      " 335    0.30    1.22    0.76    0.19   D 320 A E 322 A E 324 A E 324 A K 364 A -       \n",
      " 336    0.49    1.52    0.76    0.00   -       -       T 325 A -       C 365 A -       \n",
      " 337    0.37    3.77    2.17    0.49   G 321 A A 323 A A 326 A Q 325 A V 366 A -       \n",
      " 338    0.35    4.16    1.86    0.89   V 322 A V 324 A F 327 A V 326 A A 367 A -       \n",
      " 339    0.38    4.32    2.24    0.84   A 323 A R 325 A E 328 A K 327 A H 368 A -       \n",
      " 340    0.36    6.08    2.94    1.49   H 324 A D 326 A F 329 A A 328 A L 369 A -       \n",
      " 341    0.36    6.55    3.18    1.31   L 325 A L 327 A L 330 A L 329 A T 370 A -       \n",
      " 342    0.35    5.51    3.50    0.63   -       L 328 A S 331 A G 330 A G 371 A -       \n",
      " 343    0.00    0.00    0.00    0.00   -       -       -       -       E 372 A -       \n",
      " 344    0.00    0.00    0.00    0.00   -       -       -       -       G 373 A -       \n",
      " 345    0.00    0.00    0.00    0.00   -       -       -       -       N 374 A -       \n",
      " 346    0.00    0.00    0.00    0.00   -       -       -       -       A 375 A -       \n",
      " 347    0.00    0.00    0.00    0.00   -       -       -       -       Y 376 A -       \n",
      " 348    0.00    0.00    0.00    0.00   -       -       -       -       C 377 A -       \n",
      " 349    0.00    0.00    0.00    0.00   -       -       -       -       D 378 A -       \n",
      " 350    0.00    0.00    0.00    0.00   -       -       -       -       V 379 A -       \n",
      " 351    0.00    0.00    0.00    0.00   -       -       -       -       P 380 A -       \n",
      " 352    0.00    0.00    0.00    0.00   -       -       -       -       E 381 A -       \n",
      " 353    0.00    0.00    0.00    0.00   -       -       -       -       D 382 A -       \n",
      " 354    0.00    0.00    0.00    0.00   -       -       -       -       T 383 A -       \n",
      " 355    0.37   22.80    9.62    4.72   -       L 329 A S 332 A L 331 A M 384 A -       \n",
      " 356    0.38   26.54   11.62    5.20   -       E 330 A A 333 A I 332 A L 385 A -       \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "SALIGN______> Raw QUALITY_SCORE of the multiple alignment:     295.1\n",
      "              QUALITY_SCORE (percentage)                 :      92.9\n",
      "              Number of unique protein pairs             :        15\n",
      "              QUALITY_SCORE_min                          :       266\n",
      "              RMS_CUTOFF                                 :    3.5000\n",
      "              QS is the average number of structurally equivalent residue pairs.\n",
      "              Two residues are equivalent when closer than RMS_CUTOFF upon\n",
      "              pairwise least-squares superposition given current alignment.\n",
      "\n",
      "\n",
      "SALIGN______> Matrix of pairwise equivalences from individual pairwise alignments:\n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     312     306     302     316     289\n",
      "6itkA           0       0     292     307     304     261\n",
      "5mdhA           0       0       0     300     301     272\n",
      "1bdmA           0       0       0       0     310     274\n",
      "7mdhA           0       0       0       0       0     296\n",
      "4h7pA           0       0       0       0       0       0\n",
      "SALIGN______> Matrix of pairwise equivalences: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     311     309     310     311     272\n",
      "6itkA           0       0     305     307     306     266\n",
      "5mdhA           0       0       0     303     302     272\n",
      "1bdmA           0       0       0       0     307     267\n",
      "7mdhA           0       0       0       0       0     279\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n",
      "SALIGN______> Matrix of differences in pairwise equivalences inferred from the tree and individually: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0      -1       3       8      -5     -17\n",
      "6itkA           0       0      13       0       2       5\n",
      "5mdhA           0       0       0       3       1       0\n",
      "1bdmA           0       0       0       0      -3      -7\n",
      "7mdhA           0       0       0       0       0     -17\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n",
      "SALIGN______> Matrix of %differences in pairwise equivalences inferred from the tree and individually: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0       0       1       3      -2      -5\n",
      "6itkA           0       0       4       0       1       2\n",
      "5mdhA           0       0       0       1       0       0\n",
      "1bdmA           0       0       0       0      -1      -2\n",
      "7mdhA           0       0       0       0       0      -5\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n",
      "  ALIGN_CODES                                   : 3d5tA 6itkA 5mdhA 1bdmA 7mdhA 4h7pA \n",
      "  Atom type for alignment (FIT_ATOMS)           : CA\n",
      "  FIT                                           :         T\n",
      "  IMPROVE_ALIGNMENT                             :         T\n",
      "  WATER_IO, HETATM_IO, HYDROGEN_IO              :         F        F        F\n",
      "  NO_TER                                        :         F\n",
      "  OUTPUT                                        : ALIGNMENT QUALITY\n",
      "  FIT_ON_FIRST                                  :         F\n",
      "  WRITE_FIT                                     :         F\n",
      "  FIT_PDBNAM                                    :         T\n",
      "  WRITE_WHOLE_PDB                               :         T\n",
      "  CURRENT_DIRECTORY                             :         T\n",
      "  Feature weights (FEATURE_WEIGHTS)             :    1.0000   0.5000   1.0000   1.0000   1.0000   0.0000\n",
      "  NORMALIZE_PP_SCORES                           :         F\n",
      "  Gap initiation penalty (GAP_PENALTIES[1])     :  -450.0000\n",
      "  Gap extension penalty (GAP_PENALTIES[2])      :   -50.0000\n",
      "  Break-break bonus                             : 10000.0000\n",
      "  Gap_function (align2d flag)                    :         F\n",
      "  2-D Gap penalties                             :  3.50  3.50  3.50  0.20  4.00  6.50  2.00  0.00\n",
      "  GAP-GAP_SCORE                                 :    0.0000\n",
      "  GAP-RESIDUE_SCORE                             :    0.0000\n",
      "  Perform gap-residue and gap-gap correction    :         T\n",
      "  Residue type - residue type file (RR_FILE)    : $(LIB)/as1.sim.mat\n",
      "  OFF_DIAGONAL                                  :       100\n",
      "  OVERHANG                                      :        30\n",
      "  LOCAL_ALIGNMENT                               :         F\n",
      "  MATRIX_OFFSET                                 :    0.0000\n",
      "  N_SUBOPT                                      :         0\n",
      "  SUBOPT_OFFSET                                 :    0.0000\n",
      "  Gap introduction penalty (GAP_PENALTIES_3D_1) :    0.0000\n",
      "  Gap extension penalty (GAP_PENALTIES_3D_2)    :    3.0000\n",
      "  Max dist for equiv (2 * GAP_PENALTIES_3D_2)   :    6.0000\n",
      "  ALIGN3D_TRF                                   :         F\n",
      "  RMS_CUTOFF                                    :    3.5\n",
      "\n",
      "\n",
      "\n",
      "salign__276_> 'align_block' changed to 1.\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      315      315        0.8822        0.8822        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      315      315        0.8822        0.8822        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      316      316        1.2115        1.2115        0.0002\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      316      316        1.2115        1.2115        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      312      312        0.8224        0.8224        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      321      321        1.2299        1.2299        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      307      307        1.1344        1.1344        0.0019\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      307      307        1.0979        1.0979        0.0001\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      313      313        1.1002        1.1002        0.0006\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      313      313        1.1002        1.1002        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      311      311        0.9316        0.9316        0.0001\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      311      311        0.9316        0.9316        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      317      317        1.2407        1.2407        0.0002\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      317      317        1.2407        1.2407        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      302      302        1.0866        1.0866        0.0021\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      302      302        1.0694        1.0694        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      313      313        1.2063        1.2063        0.0000\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2547100    2487.402     2.429\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      314      314        1.2549        1.2549        0.0001\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      314      314        1.2549        1.2549        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      306      306        1.1744        1.1744        0.0012\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      306      306        1.1744        1.1744        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      315      315        0.9869        0.9869        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      315      315        0.9869        0.9869        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      300      300        1.0186        1.0186        0.0015\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      300      300        1.0091        1.0091        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      312      312        1.1923        1.1923        0.0024\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      312      312        1.1880        1.1880        0.0000\n",
      "\n",
      "SALIGN______> Matrix of pairwise protein distances for the initial tree:\n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA       0.000   0.212   0.282   0.215   0.316   0.266\n",
      "6itkA       0.212   0.000   0.290   0.213   0.329   0.296\n",
      "5mdhA       0.282   0.290   0.000   0.281   0.342   0.300\n",
      "1bdmA       0.215   0.213   0.281   0.000   0.302   0.283\n",
      "7mdhA       0.316   0.329   0.342   0.302   0.000   0.337\n",
      "4h7pA       0.266   0.296   0.300   0.283   0.337   0.000\n",
      "\n",
      "SALIGN____> Matrix of pairwise equivalences: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     314     312     310     318     302\n",
      "6itkA           0       0     307     308     314     297\n",
      "5mdhA           0       0       0     309     302     300\n",
      "1bdmA           0       0       0       0     310     297\n",
      "7mdhA           0       0       0       0       0     306\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "openf___224_> Open           ldh.tree\n",
      "                                                               .--- 3d5tA          0.2116\n",
      "                                                               |\n",
      "                                                              .---- 6itkA          0.2141\n",
      "                                                              |\n",
      "                               .----------------------------------- 1bdmA          0.2822\n",
      "                               |\n",
      "                           .--------------------------------------- 4h7pA          0.2916\n",
      "                           |\n",
      "        .---------------------------------------------------------- 5mdhA          0.3333\n",
      "        |\n",
      "      .------------------------------------------------------------ 7mdhA\n",
      "\n",
      "      +----+----+----+----+----+----+----+----+----+----+----+----+\n",
      "     0.3381    0.3162    0.2943    0.2724    0.2505    0.2286    0.2067\n",
      "          0.3272    0.3053    0.2834    0.2615    0.2396    0.2177\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    1\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      315      315        0.8822        0.8822        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      315      315        0.8822        0.8822        0.0000\n",
      "\n",
      "Current alignment total score:        68.56\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   K   3 A -                                       \n",
      "   2    0.31    1.76    0.88    0.00   P   4 A S   8 A                                 \n",
      "   3    0.32    0.94    0.47    0.00   A   5 A T   9 A                                 \n",
      "   4    0.06    0.23    0.12    0.00 * K   6 A K  10 A                                 \n",
      "   5    0.13    0.35    0.17    0.00   R   7 A K  11 A                                 \n",
      "   6    0.06    0.26    0.13    0.00 * V   8 A V  12 A                                 \n",
      "   7    0.13    0.25    0.12    0.00   A   9 A T  13 A                                 \n",
      "   8    0.08    0.48    0.24    0.00 * V  10 A V  14 A                                 \n",
      "   9    0.06    0.25    0.12    0.00 * T  11 A T  15 A                                 \n",
      "  10    0.03    0.15    0.07    0.00 * G  12 A G  16 A                                 \n",
      "  11    0.07    0.29    0.15    0.00 * A  13 A A  17 A                                 \n",
      "  12    0.07    0.34    0.17    0.00 * A  14 A A  18 A                                 \n",
      "  13    0.05    0.34    0.17    0.00 * G  15 A G  19 A                                 \n",
      "  14    0.06    0.21    0.10    0.00 * Q  16 A Q  20 A                                 \n",
      "  15    0.07    0.35    0.17    0.00 * I  17 A I  21 A                                 \n",
      "  16    0.13    0.35    0.17    0.00   A  18 A S  22 A                                 \n",
      "  17    0.05    0.28    0.14    0.00 * Y  19 A Y  23 A                                 \n",
      "  18    0.07    0.39    0.20    0.00 * S  20 A S  24 A                                 \n",
      "  19    0.06    0.35    0.17    0.00 * L  21 A L  25 A                                 \n",
      "  20    0.07    0.45    0.23    0.00 * L  22 A L  26 A                                 \n",
      "  21    0.22    0.80    0.40    0.00   F  23 A W  27 A                                 \n",
      "  22    0.15    1.06    0.53    0.00 * R  24 A R  28 A                                 \n",
      "  23    0.14    0.92    0.46    0.00 * I  25 A I  29 A                                 \n",
      "  24    0.11    0.65    0.33    0.00 * A  26 A A  30 A                                 \n",
      "  25    0.13    0.78    0.39    0.00 * N  27 A N  31 A                                 \n",
      "  26    0.11    0.90    0.45    0.00 * G  28 A G  32 A                                 \n",
      "  27    0.22    1.16    0.58    0.00   D  29 A E  33 A                                 \n",
      "  28    0.20    1.07    0.54    0.00   L  30 A V  34 A                                 \n",
      "  29    0.20    0.96    0.48    0.00   L  31 A F  35 A                                 \n",
      "  30    0.15    1.19    0.60    0.00 * G  32 A G  36 A                                 \n",
      "  31    0.24    1.21    0.60    0.00   K  33 A T  37 A                                 \n",
      "  32    0.17    1.28    0.64    0.00 * D  34 A D  38 A                                 \n",
      "  33    0.55    0.75    0.38    0.00   Q  35 A T  39 A                                 \n",
      "  34    0.11    0.82    0.41    0.00 * P  36 A P  40 A                                 \n",
      "  35    0.07    0.37    0.18    0.00 * V  37 A V  41 A                                 \n",
      "  36    0.19    0.31    0.15    0.00   I  38 A E  42 A                                 \n",
      "  37    0.03    0.11    0.05    0.00 * L  39 A L  43 A                                 \n",
      "  38    0.13    0.20    0.10    0.00   Q  40 A K  44 A                                 \n",
      "  39    0.09    0.57    0.28    0.00 * L  41 A L  45 A                                 \n",
      "  40    0.05    0.22    0.11    0.00 * L  42 A L  46 A                                 \n",
      "  41    0.15    0.52    0.26    0.00   D  43 A E  47 A                                 \n",
      "  42    0.38    0.63    0.32    0.00   L  44 A I  48 A                                 \n",
      "  43    0.29    0.35    0.17    0.00 * P  45 A P  49 A                                 \n",
      "  44    0.14    0.89    0.45    0.00 * Q  46 A Q  50 A                                 \n",
      "  45    0.09    0.47    0.24    0.00 * A  47 A A  51 A                                 \n",
      "  46    0.19    0.44    0.22    0.00   Q  48 A L  52 A                                 \n",
      "  47    0.16    0.56    0.28    0.00   A  49 A G  53 A                                 \n",
      "  48    0.16    0.58    0.29    0.00   A  50 A G  54 A                                 \n",
      "  49    0.15    0.49    0.24    0.00   V  51 A A  55 A                                 \n",
      "  50    0.17    0.59    0.29    0.00   K  52 A E  56 A                                 \n",
      "  51    0.07    0.51    0.25    0.00 * G  53 A G  57 A                                 \n",
      "  52    0.11    0.69    0.34    0.00 * V  54 A V  58 A                                 \n",
      "  53    0.21    0.95    0.47    0.00   V  55 A A  59 A                                 \n",
      "  54    0.15    0.87    0.44    0.00 * M  56 A M  60 A                                 \n",
      "  55    0.11    0.67    0.34    0.00 * E  57 A E  61 A                                 \n",
      "  56    0.10    0.72    0.36    0.00 * L  58 A L  62 A                                 \n",
      "  57    0.29    1.05    0.53    0.00   D  59 A L  63 A                                 \n",
      "  58    0.16    1.15    0.58    0.00 * D  60 A D  64 A                                 \n",
      "  59    0.27    0.83    0.41    0.00   C  61 A S  65 A                                 \n",
      "  60    0.15    1.01    0.51    0.00 * A  62 A A  66 A                                 \n",
      "  61    0.09    0.61    0.31    0.00 * F  63 A F  67 A                                 \n",
      "  62    0.08    0.52    0.26    0.00 * P  64 A P  68 A                                 \n",
      "  63    0.10    0.64    0.32    0.00 * L  65 A L  69 A                                 \n",
      "  64    0.06    0.36    0.18    0.00 * L  66 A L  70 A                                 \n",
      "  65    0.19    0.46    0.23    0.00   A  67 A R  71 A                                 \n",
      "  66    0.16    0.37    0.18    0.00   G  68 A N  72 A                                 \n",
      "  67    0.12    0.43    0.21    0.00   V  69 A I  73 A                                 \n",
      "  68    0.18    0.60    0.30    0.00   V  70 A T  74 A                                 \n",
      "  69    0.08    0.38    0.19    0.00 * I  71 A I  75 A                                 \n",
      "  70    0.05    0.15    0.08    0.00 * T  72 A T  76 A                                 \n",
      "  71    0.48    0.28    0.14    0.00   D  73 A A  77 A                                 \n",
      "  72    0.09    0.58    0.29    0.00 * D  74 A D  78 A                                 \n",
      "  73    0.16    0.44    0.22    0.00   P  75 A A  79 A                                 \n",
      "  74    0.16    0.42    0.21    0.00   K  76 A N  80 A                                 \n",
      "  75    0.19    0.42    0.21    0.00   V  77 A E  81 A                                 \n",
      "  76    0.07    0.32    0.16    0.00 * A  78 A A  82 A                                 \n",
      "  77    0.05    0.23    0.11    0.00 * F  79 A F  83 A                                 \n",
      "  78    0.17    0.43    0.21    0.00   K  80 A D  84 A                                 \n",
      "  79    0.17    0.43    0.22    0.00   D  81 A G  85 A                                 \n",
      "  80    0.07    0.29    0.14    0.00 * A  82 A A  86 A                                 \n",
      "  81    0.47    0.35    0.17    0.00   D  83 A N  87 A                                 \n",
      "  82    0.13    0.18    0.09    0.00   V  84 A A  88 A                                 \n",
      "  83    0.11    0.63    0.32    0.00 * A  85 A A  89 A                                 \n",
      "  84    0.16    0.60    0.30    0.00   L  86 A F  90 A                                 \n",
      "  85    0.06    0.29    0.15    0.00 * L  87 A L  91 A                                 \n",
      "  86    0.08    0.47    0.24    0.00 * V  88 A V  92 A                                 \n",
      "  87    0.30    0.55    0.28    0.00 * G  89 A G  93 A                                 \n",
      "  88    0.36    0.82    0.41    0.00 * A  90 A A  94 A                                 \n",
      "  89    0.41    1.85    0.93    0.00   R  91 A K  95 A                                 \n",
      "  90    0.30    1.47    0.74    0.00 * P  92 A P  96 A                                 \n",
      "  91    0.00    0.00    0.00    0.00   R  93 A -                                       \n",
      "  92    0.00    0.00    0.00    0.00   S  94 A -                                       \n",
      "  93    0.00    0.00    0.00    0.00   M  97 A -                                       \n",
      "  94    0.40    2.40    1.20    0.00 * E  98 A E 102 A                                 \n",
      "  95    0.33    1.90    0.95    0.00 * R  99 A R 103 A                                 \n",
      "  96    0.37    2.26    1.13    0.00   K 100 A A 104 A                                 \n",
      "  97    0.26    1.97    0.98    0.00 * D 101 A D 105 A                                 \n",
      "  98    0.16    1.13    0.56    0.00 * L 102 A L 106 A                                 \n",
      "  99    0.15    1.05    0.52    0.00 * L 103 A L 107 A                                 \n",
      " 100    0.20    0.95    0.48    0.00   S 104 A A 108 A                                 \n",
      " 101    0.33    1.78    0.89    0.00   A 105 A N 109 A                                 \n",
      " 102    0.24    1.67    0.84    0.00 * N 106 A N 110 A                                 \n",
      " 103    0.32    1.93    0.96    0.00   A 107 A G 111 A                                 \n",
      " 104    0.27    1.38    0.69    0.00   E 108 A K 112 A                                 \n",
      " 105    0.11    0.60    0.30    0.00 * I 109 A I 113 A                                 \n",
      " 106    0.11    0.64    0.32    0.00 * F 110 A F 114 A                                 \n",
      " 107    0.43    0.59    0.30    0.00   T 111 A G 115 A                                 \n",
      " 108    0.20    0.48    0.24    0.00   V 112 A P 116 A                                 \n",
      " 109    0.09    0.47    0.24    0.00 * Q 113 A Q 117 A                                 \n",
      " 110    0.05    0.33    0.16    0.00 * G 114 A G 118 A                                 \n",
      " 111    0.18    0.59    0.29    0.00   A 115 A K 119 A                                 \n",
      " 112    0.10    0.55    0.28    0.00 * A 116 A A 120 A                                 \n",
      " 113    0.14    0.43    0.21    0.00   L 117 A I 121 A                                 \n",
      " 114    0.06    0.16    0.08    0.00 * N 118 A N 122 A                                 \n",
      " 115    0.12    0.26    0.13    0.00   E 119 A D 123 A                                 \n",
      " 116    0.20    0.39    0.19    0.00   V 120 A N 124 A                                 \n",
      " 117    0.10    0.41    0.20    0.00 * A 121 A A 125 A                                 \n",
      " 118    0.19    0.76    0.38    0.00   S 122 A A 126 A                                 \n",
      " 119    0.35    1.74    0.87    0.00   R 123 A D 127 A                                 \n",
      " 120    0.19    1.42    0.71    0.00 * D 124 A D 128 A                                 \n",
      " 121    0.11    0.33    0.16    0.00   V 125 A I 129 A                                 \n",
      " 122    0.12    0.27    0.13    0.00   K 126 A R 130 A                                 \n",
      " 123    0.09    0.52    0.26    0.00 * V 127 A V 131 A                                 \n",
      " 124    0.07    0.43    0.21    0.00 * L 128 A L 132 A                                 \n",
      " 125    0.04    0.11    0.06    0.00 * V 129 A V 133 A                                 \n",
      " 126    0.05    0.17    0.09    0.00 * V 130 A V 134 A                                 \n",
      " 127    0.04    0.27    0.13    0.00 * G 131 A G 135 A                                 \n",
      " 128    0.10    0.48    0.24    0.00 * N 132 A N 136 A                                 \n",
      " 129    0.05    0.21    0.11    0.00 * P 133 A P 137 A                                 \n",
      " 130    0.04    0.07    0.03    0.00 * A 134 A A 138 A                                 \n",
      " 131    0.07    0.27    0.13    0.00 * N 135 A N 139 A                                 \n",
      " 132    0.05    0.12    0.06    0.00 * T 136 A T 140 A                                 \n",
      " 133    0.08    0.27    0.14    0.00 * N 137 A N 141 A                                 \n",
      " 134    0.07    0.33    0.16    0.00 * A 138 A A 142 A                                 \n",
      " 135    0.15    0.15    0.08    0.00   Y 139 A L 143 A                                 \n",
      " 136    0.06    0.25    0.12    0.00 * I 140 A I 144 A                                 \n",
      " 137    0.08    0.42    0.21    0.00 * A 141 A A 145 A                                 \n",
      " 138    0.21    0.40    0.20    0.00   M 142 A S 146 A                                 \n",
      " 139    0.17    0.56    0.28    0.00   K 143 A A 147 A                                 \n",
      " 140    0.16    0.61    0.31    0.00   S 144 A A 148 A                                 \n",
      " 141    0.07    0.31    0.16    0.00 * A 145 A A 149 A                                 \n",
      " 142    0.08    0.46    0.23    0.00 * P 146 A P 150 A                                 \n",
      " 143    0.05    0.14    0.07    0.00 * D 147 A D 151 A                                 \n",
      " 144    0.16    0.64    0.32    0.00   L 148 A V 152 A                                 \n",
      " 145    0.14    0.98    0.49    0.00 * P 149 A P 153 A                                 \n",
      " 146    0.23    1.08    0.54    0.00   K 150 A A 154 A                                 \n",
      " 147    0.21    0.88    0.44    0.00   K 151 A S 155 A                                 \n",
      " 148    0.20    0.57    0.29    0.00   N 152 A R 156 A                                 \n",
      " 149    0.08    0.43    0.21    0.00 * F 153 A F 157 A                                 \n",
      " 150    0.15    0.35    0.17    0.00   T 154 A N 158 A                                 \n",
      " 151    0.07    0.29    0.14    0.00 * A 155 A A 159 A                                 \n",
      " 152    0.08    0.23    0.12    0.00 * M 156 A M 160 A                                 \n",
      " 153    0.15    0.58    0.29    0.00   L 157 A M 161 A                                 \n",
      " 154    0.09    0.48    0.24    0.00 * R 158 A R 162 A                                 \n",
      " 155    0.09    0.62    0.31    0.00 * L 159 A L 163 A                                 \n",
      " 156    0.07    0.35    0.18    0.00 * D 160 A D 164 A                                 \n",
      " 157    0.08    0.41    0.21    0.00 * H 161 A H 165 A                                 \n",
      " 158    0.10    0.48    0.24    0.00 * N 162 A N 166 A                                 \n",
      " 159    0.07    0.34    0.17    0.00 * R 163 A R 167 A                                 \n",
      " 160    0.05    0.09    0.04    0.00 * A 164 A A 168 A                                 \n",
      " 161    0.10    0.09    0.05    0.00   L 165 A I 169 A                                 \n",
      " 162    0.10    0.61    0.30    0.00 * S 166 A S 170 A                                 \n",
      " 163    0.12    0.82    0.41    0.00 * Q 167 A Q 171 A                                 \n",
      " 164    0.10    0.68    0.34    0.00 * L 168 A L 172 A                                 \n",
      " 165    0.14    0.96    0.48    0.00 * A 169 A A 173 A                                 \n",
      " 166    0.25    1.28    0.64    0.00   A 170 A T 174 A                                 \n",
      " 167    0.17    1.23    0.61    0.00 * K 171 A K 175 A                                 \n",
      " 168    0.32    1.56    0.78    0.00   S 172 A L 176 A                                 \n",
      " 169    0.19    1.51    0.76    0.00 * G 173 A G 177 A                                 \n",
      " 170    0.49    1.57    0.78    0.00   K 174 A R 178 A                                 \n",
      " 171    0.28    1.26    0.63    0.00   P 175 A G 179 A                                 \n",
      " 172    0.24    0.87    0.44    0.00   V 176 A S 180 A                                 \n",
      " 173    0.11    0.68    0.34    0.00 * A 177 A A 181 A                                 \n",
      " 174    0.23    0.95    0.47    0.00   S 178 A E 182 A                                 \n",
      " 175    0.19    0.65    0.33    0.00   I 179 A F 183 A                                 \n",
      " 176    0.20    0.70    0.35    0.00   E 180 A N 184 A                                 \n",
      " 177    0.19    0.73    0.37    0.00   K 181 A N 185 A                                 \n",
      " 178    0.17    0.79    0.40    0.00   L 182 A I 186 A                                 \n",
      " 179    0.20    0.85    0.43    0.00   A 183 A V 187 A                                 \n",
      " 180    0.14    0.93    0.46    0.00 * V 184 A V 188 A                                 \n",
      " 181    0.04    0.12    0.06    0.00 * W 185 A W 189 A                                 \n",
      " 182    0.05    0.29    0.15    0.00 * G 186 A G 190 A                                 \n",
      " 183    0.09    0.36    0.18    0.00 * N 187 A N 191 A                                 \n",
      " 184    0.11    0.65    0.32    0.00 * H 188 A H 192 A                                 \n",
      " 185    0.14    0.97    0.49    0.00 * S 189 A S 193 A                                 \n",
      " 186    0.23    1.01    0.51    0.00   P 190 A A 194 A                                 \n",
      " 187    0.14    0.94    0.47    0.00 * T 191 A T 195 A                                 \n",
      " 188    0.23    0.70    0.35    0.00   M 192 A Q 196 A                                 \n",
      " 189    0.13    0.36    0.18    0.00   Y 193 A F 197 A                                 \n",
      " 190    0.04    0.19    0.10    0.00 * P 194 A P 198 A                                 \n",
      " 191    0.08    0.45    0.23    0.00 * D 195 A D 199 A                                 \n",
      " 192    0.20    0.75    0.38    0.00   F 196 A I 200 A                                 \n",
      " 193    0.24    0.99    0.50    0.00   R 197 A T 201 A                                 \n",
      " 194    0.16    0.69    0.34    0.00   F 198 A Y 202 A                                 \n",
      " 195    0.10    0.55    0.28    0.00 * A 199 A A 203 A                                 \n",
      " 196    0.11    0.67    0.34    0.00 * T 200 A T 204 A                                 \n",
      " 197    0.20    0.86    0.43    0.00   A 201 A V 205 A                                 \n",
      " 198    0.40    2.47    1.23    0.00   E 202 A G 206 A                                 \n",
      " 199    0.20    1.61    0.80    0.00 * G 203 A G 207 A                                 \n",
      " 200    0.20    1.40    0.70    0.00 * E 204 A E 208 A                                 \n",
      " 201    0.24    1.17    0.58    0.00   S 205 A K 209 A                                 \n",
      " 202    0.17    0.75    0.38    0.00   L 206 A V 210 A                                 \n",
      " 203    0.26    1.04    0.52    0.00   L 207 A T 211 A                                 \n",
      " 204    0.27    1.30    0.65    0.00   K 208 A D 212 A                                 \n",
      " 205    0.29    2.01    1.01    0.00 * L 209 A L 213 A                                 \n",
      " 206    0.34    2.13    1.06    0.00   I 210 A V 214 A                                 \n",
      " 207    0.00    0.00    0.00    0.00   N 211 A -                                       \n",
      " 208    0.12    0.55    0.27    0.00 * D 212 A D 215 A                                 \n",
      " 209    0.30    0.27    0.14    0.00   D 213 A H 216 A                                 \n",
      " 210    0.36    0.79    0.40    0.00   V 214 A D 217 A                                 \n",
      " 211    0.21    0.73    0.37    0.00 * W 215 A W 218 A                                 \n",
      " 212    0.35    0.86    0.43    0.00   N 216 A Y 219 A                                 \n",
      " 213    0.38    1.08    0.54    0.00   R 217 A V 220 A                                 \n",
      " 214    0.20    0.92    0.46    0.00   D 218 A E 221 A                                 \n",
      " 215    0.18    0.53    0.26    0.00   T 219 A E 222 A                                 \n",
      " 216    0.06    0.29    0.14    0.00 * F 220 A F 223 A                                 \n",
      " 217    0.06    0.20    0.10    0.00 * I 221 A I 224 A                                 \n",
      " 218    0.04    0.19    0.09    0.00 * P 222 A P 225 A                                 \n",
      " 219    0.15    0.17    0.08    0.00   T 223 A R 226 A                                 \n",
      " 220    0.05    0.20    0.10    0.00 * V 224 A V 227 A                                 \n",
      " 221    0.17    0.67    0.33    0.00   G 225 A A 228 A                                 \n",
      " 222    0.18    0.64    0.32    0.00   K 226 A N 229 A                                 \n",
      " 223    0.06    0.23    0.12    0.00 * R 227 A R 230 A                                 \n",
      " 224    0.06    0.46    0.23    0.00 * G 228 A G 231 A                                 \n",
      " 225    0.12    0.81    0.40    0.00 * A 229 A A 232 A                                 \n",
      " 226    0.15    0.37    0.19    0.00   A 230 A E 233 A                                 \n",
      " 227    0.10    0.55    0.28    0.00 * I 231 A I 234 A                                 \n",
      " 228    0.13    0.87    0.44    0.00 * I 232 A I 235 A                                 \n",
      " 229    0.10    0.56    0.28    0.00 * E 233 A E 236 A                                 \n",
      " 230    0.15    0.40    0.20    0.00   A 234 A V 237 A                                 \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 231    0.12    0.76    0.38    0.00 * R 235 A R 238 A                                 \n",
      " 232    0.07    0.52    0.26    0.00 * G 236 A G 239 A                                 \n",
      " 233    0.20    0.57    0.29    0.00   L 237 A K 240 A                                 \n",
      " 234    0.13    0.91    0.45    0.00 * S 238 A S 241 A                                 \n",
      " 235    0.22    0.62    0.31    0.00 * S 239 A S 242 A                                 \n",
      " 236    0.10    0.52    0.26    0.00 * A 240 A A 243 A                                 \n",
      " 237    0.10    0.53    0.27    0.00 * A 241 A A 244 A                                 \n",
      " 238    0.08    0.48    0.24    0.00 * S 242 A S 245 A                                 \n",
      " 239    0.08    0.41    0.20    0.00 * A 243 A A 246 A                                 \n",
      " 240    0.07    0.35    0.17    0.00 * A 244 A A 247 A                                 \n",
      " 241    0.15    0.52    0.26    0.00   N 245 A S 248 A                                 \n",
      " 242    0.15    0.51    0.25    0.00   A 246 A S 249 A                                 \n",
      " 243    0.08    0.42    0.21    0.00 * A 247 A A 250 A                                 \n",
      " 244    0.09    0.52    0.26    0.00 * I 248 A I 251 A                                 \n",
      " 245    0.08    0.44    0.22    0.00 * D 249 A D 252 A                                 \n",
      " 246    0.08    0.41    0.20    0.00 * H 250 A H 253 A                                 \n",
      " 247    0.18    0.52    0.26    0.00   V 251 A M 254 A                                 \n",
      " 248    0.07    0.32    0.16    0.00 * R 252 A R 255 A                                 \n",
      " 249    0.05    0.21    0.11    0.00 * D 253 A D 256 A                                 \n",
      " 250    0.07    0.48    0.24    0.00 * W 254 A W 257 A                                 \n",
      " 251    0.08    0.47    0.23    0.00 * V 255 A V 258 A                                 \n",
      " 252    0.17    0.27    0.13    0.00   L 256 A Q 259 A                                 \n",
      " 253    0.11    0.70    0.35    0.00 * G 257 A G 260 A                                 \n",
      " 254    0.07    0.14    0.07    0.00 * T 258 A T 261 A                                 \n",
      " 255    0.47    2.77    1.39    0.00   N 259 A E 262 A                                 \n",
      " 256    0.00    0.00    0.00    0.00   G 260 A -                                       \n",
      " 257    0.54    1.65    0.83    0.00   K 261 A A 263 A                                 \n",
      " 258    0.16    1.20    0.60    0.00 * W 262 A W 264 A                                 \n",
      " 259    0.28    1.10    0.55    0.00   V 263 A S 265 A                                 \n",
      " 260    0.16    0.54    0.27    0.00   T 264 A S 266 A                                 \n",
      " 261    0.19    0.38    0.19    0.00   M 265 A A 267 A                                 \n",
      " 262    0.15    0.40    0.20    0.00   G 266 A A 268 A                                 \n",
      " 263    0.06    0.28    0.14    0.00 * I 267 A I 269 A                                 \n",
      " 264    0.04    0.20    0.10    0.00 * P 268 A P 270 A                                 \n",
      " 265    0.08    0.45    0.23    0.00 * S 269 A S 271 A                                 \n",
      " 266    0.17    0.35    0.18    0.00   D 270 A T 272 A                                 \n",
      " 267    0.06    0.39    0.20    0.00 * G 271 A G 273 A                                 \n",
      " 268    0.22    1.00    0.50    0.00   S 272 A A 274 A                                 \n",
      " 269    0.13    0.91    0.46    0.00 * Y 273 A Y 275 A                                 \n",
      " 270    0.18    1.47    0.73    0.00 * G 274 A G 276 A                                 \n",
      " 271    0.12    0.78    0.39    0.00 * I 275 A I 277 A                                 \n",
      " 272    0.11    0.81    0.40    0.00 * P 276 A P 278 A                                 \n",
      " 273    0.15    0.88    0.44    0.00 * E 277 A E 279 A                                 \n",
      " 274    0.63    2.50    1.25    0.00   D 278 A G 280 A                                 \n",
      " 275    0.18    1.25    0.63    0.00 * I 279 A I 281 A                                 \n",
      " 276    0.17    0.36    0.18    0.00   I 280 A F 282 A                                 \n",
      " 277    0.21    0.56    0.28    0.00   Y 281 A V 283 A                                 \n",
      " 278    0.08    0.55    0.28    0.00 * G 282 A G 284 A                                 \n",
      " 279    0.35    0.23    0.12    0.00   V 283 A L 285 A                                 \n",
      " 280    0.16    1.12    0.56    0.00 * P 284 A P 286 A                                 \n",
      " 281    0.22    0.82    0.41    0.00   V 285 A T 287 A                                 \n",
      " 282    0.43    1.23    0.62    0.00   I 286 A V 288 A                                 \n",
      " 283    0.24    0.59    0.30    0.00   C 287 A S 289 A                                 \n",
      " 284    0.29    1.37    0.69    0.00   E 288 A R 290 A                                 \n",
      " 285    0.15    0.93    0.47    0.00 * N 289 A N 291 A                                 \n",
      " 286    0.11    0.85    0.42    0.00 * G 290 A G 292 A                                 \n",
      " 287    0.18    1.21    0.61    0.00 * E 291 A E 293 A                                 \n",
      " 288    0.25    1.17    0.58    0.00   Y 292 A W 294 A                                 \n",
      " 289    0.20    0.78    0.39    0.00   K 293 A E 295 A                                 \n",
      " 290    0.22    0.46    0.23    0.00   R 294 A I 296 A                                 \n",
      " 291    0.07    0.38    0.19    0.00 * V 295 A V 297 A                                 \n",
      " 292    0.15    1.00    0.50    0.00 * E 296 A E 298 A                                 \n",
      " 293    0.21    1.78    0.89    0.00 * G 297 A G 299 A                                 \n",
      " 294    0.16    1.19    0.59    0.00 * L 298 A L 300 A                                 \n",
      " 295    0.17    1.11    0.56    0.00 * E 299 A E 301 A                                 \n",
      " 296    0.14    0.93    0.46    0.00 * I 300 A I 302 A                                 \n",
      " 297    0.20    0.80    0.40    0.00   D 301 A S 303 A                                 \n",
      " 298    0.25    1.19    0.59    0.00   A 302 A D 304 A                                 \n",
      " 299    0.19    1.45    0.72    0.00 * F 303 A F 305 A                                 \n",
      " 300    0.27    1.29    0.64    0.00   S 304 A Q 306 A                                 \n",
      " 301    0.17    1.24    0.62    0.00 * R 305 A R 307 A                                 \n",
      " 302    0.21    0.81    0.41    0.00   E 306 A A 308 A                                 \n",
      " 303    0.19    0.88    0.44    0.00   K 307 A R 309 A                                 \n",
      " 304    0.21    0.93    0.46    0.00   M 308 A I 310 A                                 \n",
      " 305    0.16    1.20    0.60    0.00 * D 309 A D 311 A                                 \n",
      " 306    0.23    1.13    0.57    0.00   G 310 A A 312 A                                 \n",
      " 307    0.28    1.48    0.74    0.00   T 311 A N 313 A                                 \n",
      " 308    0.30    1.55    0.77    0.00   L 312 A A 314 A                                 \n",
      " 309    0.30    1.58    0.79    0.00   A 313 A Q 315 A                                 \n",
      " 310    0.20    1.46    0.73    0.00 * E 314 A E 316 A                                 \n",
      " 311    0.14    1.02    0.51    0.00 * L 315 A L 317 A                                 \n",
      " 312    0.26    1.06    0.53    0.00   L 316 A Q 318 A                                 \n",
      " 313    0.23    1.01    0.51    0.00   E 317 A A 319 A                                 \n",
      " 314    0.09    0.48    0.24    0.00 * E 318 A E 320 A                                 \n",
      " 315    0.08    0.42    0.21    0.00 * R 319 A R 321 A                                 \n",
      " 316    0.10    0.05    0.03    0.00   D 320 A E 322 A                                 \n",
      " 317    0.15    0.41    0.21    0.00   G 321 A A 323 A                                 \n",
      " 318    0.11    0.62    0.31    0.00 * V 322 A V 324 A                                 \n",
      " 319    0.18    0.25    0.13    0.00   A 323 A R 325 A                                 \n",
      " 320    0.35    1.66    0.83    0.00   H 324 A D 326 A                                 \n",
      " 321    0.43    3.69    1.84    0.00 * L 325 A L 327 A                                 \n",
      " 322    0.00    0.00    0.00    0.00   -       L 328 A                                 \n",
      " 323    0.00    0.00    0.00    0.00   -       L 329 A                                 \n",
      " 324    0.00    0.00    0.00    0.00   -       E 330 A                                 \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    2\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      313      313        0.7780        0.7780        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      313      313        0.7780        0.7780        0.0000\n",
      "\n",
      "Current alignment total score:        67.64\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -               M   0 A                 \n",
      "   2    0.37    1.07    0.54    0.00   K   3 A -               K   1 A                 \n",
      "   3    0.15    0.38    0.68    0.30   P   4 A S   8 A         A   2 A                 \n",
      "   4    0.23    0.48    0.44    0.08   A   5 A T   9 A         P   3 A                 \n",
      "   5    0.16    0.25    0.15    0.04   K   6 A K  10 A         V   4 A                 \n",
      "   6    0.07    0.09    0.14    0.06   R   7 A K  11 A         R   5 A                 \n",
      "   7    0.06    0.27    0.17    0.01 * V   8 A V  12 A         V   6 A                 \n",
      "   8    0.10    0.28    0.17    0.02   A   9 A T  13 A         A   7 A                 \n",
      "   9    0.07    0.33    0.24    0.07 * V  10 A V  14 A         V   8 A                 \n",
      "  10    0.06    0.23    0.15    0.00 * T  11 A T  15 A         T   9 A                 \n",
      "  11    0.06    0.39    0.19    0.06 * G  12 A G  16 A         G  10 A                 \n",
      "  12    0.12    0.72    0.34    0.11 * A  13 A A  17 A         A  11 A                 \n",
      "  13    0.11    0.67    0.33    0.09 * A  14 A A  18 A         A  12 A                 \n",
      "  14    0.04    0.18    0.16    0.03 * G  15 A G  19 A         G  13 A                 \n",
      "  15    0.08    0.33    0.17    0.04 * Q  16 A Q  20 A         Q  14 A                 \n",
      "  16    0.06    0.17    0.16    0.04 * I  17 A I  21 A         I  15 A                 \n",
      "  17    0.15    0.36    0.21    0.07   A  18 A S  22 A         G  16 A                 \n",
      "  18    0.04    0.19    0.14    0.04 * Y  19 A Y  23 A         Y  17 A                 \n",
      "  19    0.06    0.25    0.19    0.05 * S  20 A S  24 A         S  18 A                 \n",
      "  20    0.05    0.23    0.18    0.03 * L  21 A L  25 A         L  19 A                 \n",
      "  21    0.05    0.22    0.21    0.05 * L  22 A L  26 A         L  20 A                 \n",
      "  22    0.12    0.34    0.34    0.12   F  23 A W  27 A         F  21 A                 \n",
      "  23    0.09    0.53    0.47    0.16 * R  24 A R  28 A         R  22 A                 \n",
      "  24    0.08    0.43    0.41    0.12 * I  25 A I  29 A         I  23 A                 \n",
      "  25    0.06    0.21    0.27    0.09 * A  26 A A  30 A         A  24 A                 \n",
      "  26    0.17    0.40    0.35    0.12   N  27 A N  31 A         A  25 A                 \n",
      "  27    0.07    0.49    0.42    0.13 * G  28 A G  32 A         G  26 A                 \n",
      "  28    0.14    0.69    0.56    0.14   D  29 A E  33 A         E  27 A                 \n",
      "  29    0.21    0.99    0.63    0.12   L  30 A V  34 A         M  28 A                 \n",
      "  30    0.18    1.05    0.58    0.24   L  31 A F  35 A         L  29 A                 \n",
      "  31    0.10    0.77    0.58    0.18 * G  32 A G  36 A         G  30 A                 \n",
      "  32    0.10    0.25    0.46    0.21   K  33 A T  37 A         K  31 A                 \n",
      "  33    0.09    0.48    0.54    0.18 * D  34 A D  38 A         D  32 A                 \n",
      "  34    0.27    0.67    0.41    0.16   Q  35 A T  39 A         Q  33 A                 \n",
      "  35    0.09    0.66    0.43    0.15 * P  36 A P  40 A         P  34 A                 \n",
      "  36    0.08    0.45    0.24    0.10 * V  37 A V  41 A         V  35 A                 \n",
      "  37    0.16    0.52    0.27    0.06   I  38 A E  42 A         I  36 A                 \n",
      "  38    0.06    0.32    0.15    0.05 * L  39 A L  43 A         L  37 A                 \n",
      "  39    0.10    0.28    0.14    0.07   Q  40 A K  44 A         Q  38 A                 \n",
      "  40    0.07    0.44    0.31    0.02 * L  41 A L  45 A         L  39 A                 \n",
      "  41    0.06    0.28    0.14    0.07 * L  42 A L  46 A         L  40 A                 \n",
      "  42    0.11    0.40    0.28    0.04   D  43 A E  47 A         E  41 A                 \n",
      "  43    0.22    0.39    0.31    0.04   L  44 A I  48 A         I  42 A                 \n",
      "  44    0.19    0.52    0.28    0.07 * P  45 A P  49 A         P  43 A                 \n",
      "  45    0.11    0.65    0.45    0.16 * Q  46 A Q  50 A         Q  44 A                 \n",
      "  46    0.10    0.51    0.28    0.12 * A  47 A A  51 A         A  45 A                 \n",
      "  47    0.18    0.56    0.31    0.07   Q  48 A L  52 A         M  46 A                 \n",
      "  48    0.18    0.55    0.33    0.10   A  49 A G  53 A         K  47 A                 \n",
      "  49    0.13    0.56    0.35    0.04   A  50 A G  54 A         A  48 A                 \n",
      "  50    0.17    0.57    0.32    0.09   V  51 A A  55 A         L  49 A                 \n",
      "  51    0.11    0.39    0.30    0.03   K  52 A E  56 A         E  50 A                 \n",
      "  52    0.03    0.18    0.21    0.08 * G  53 A G  57 A         G  51 A                 \n",
      "  53    0.04    0.12    0.26    0.13 * V  54 A V  58 A         V  52 A                 \n",
      "  54    0.10    0.32    0.39    0.13   V  55 A A  59 A         V  53 A                 \n",
      "  55    0.08    0.25    0.35    0.14 * M  56 A M  60 A         M  54 A                 \n",
      "  56    0.06    0.24    0.28    0.09 * E  57 A E  61 A         E  55 A                 \n",
      "  57    0.03    0.06    0.25    0.15 * L  58 A L  62 A         L  56 A                 \n",
      "  58    0.15    0.23    0.40    0.18   D  59 A L  63 A         E  57 A                 \n",
      "  59    0.06    0.31    0.46    0.19 * D  60 A D  64 A         D  58 A                 \n",
      "  60    0.12    0.23    0.33    0.13   C  61 A S  65 A         C  59 A                 \n",
      "  61    0.06    0.18    0.38    0.19 * A  62 A A  66 A         A  60 A                 \n",
      "  62    0.05    0.22    0.26    0.08 * F  63 A F  67 A         F  61 A                 \n",
      "  63    0.04    0.21    0.22    0.06 * P  64 A P  68 A         P  62 A                 \n",
      "  64    0.06    0.28    0.28    0.09 * L  65 A L  69 A         L  63 A                 \n",
      "  65    0.05    0.18    0.16    0.03 * L  66 A L  70 A         L  64 A                 \n",
      "  66    0.20    0.94    0.45    0.18   A  67 A R  71 A         A  65 A                 \n",
      "  67    0.16    0.83    0.41    0.11   G  68 A N  72 A         G  66 A                 \n",
      "  68    0.15    0.59    0.32    0.06   V  69 A I  73 A         L  67 A                 \n",
      "  69    0.20    0.63    0.38    0.07   V  70 A T  74 A         E  68 A                 \n",
      "  70    0.20    0.62    0.33    0.06   I  71 A I  75 A         A  69 A                 \n",
      "  71    0.10    0.57    0.26    0.09 * T  72 A T  76 A         T  70 A                 \n",
      "  72    0.23    0.42    0.22    0.06   D  73 A A  77 A         D  71 A                 \n",
      "  73    0.06    0.31    0.27    0.05 * D  74 A D  78 A         D  72 A                 \n",
      "  74    0.21    0.29    0.22    0.03   P  75 A A  79 A         P  73 A                 \n",
      "  75    0.15    0.33    0.23    0.05   K  76 A N  80 A         D  74 A                 \n",
      "  76    0.12    0.30    0.22    0.04   V  77 A E  81 A         V  75 A                 \n",
      "  77    0.10    0.52    0.26    0.10 * A  78 A A  82 A         A  76 A                 \n",
      "  78    0.05    0.18    0.13    0.02 * F  79 A F  83 A         F  77 A                 \n",
      "  79    0.11    0.22    0.19    0.06   K  80 A D  84 A         K  78 A                 \n",
      "  80    0.09    0.17    0.19    0.05   D  81 A G  85 A         D  79 A                 \n",
      "  81    0.08    0.39    0.21    0.05 * A  82 A A  86 A         A  80 A                 \n",
      "  82    0.20    0.20    0.17    0.03   D  83 A N  87 A         D  81 A                 \n",
      "  83    0.18    0.26    0.14    0.02   V  84 A A  88 A         Y  82 A                 \n",
      "  84    0.08    0.36    0.29    0.10 * A  85 A A  89 A         A  83 A                 \n",
      "  85    0.10    0.35    0.28    0.08   L  86 A F  90 A         L  84 A                 \n",
      "  86    0.06    0.27    0.17    0.06 * L  87 A L  91 A         L  85 A                 \n",
      "  87    0.07    0.36    0.25    0.07 * V  88 A V  92 A         V  86 A                 \n",
      "  88    0.17    0.37    0.27    0.09 * G  89 A G  93 A         G  87 A                 \n",
      "  89    0.26    0.86    0.47    0.23 * A  90 A A  94 A         A  88 A                 \n",
      "  90    0.57    1.83    1.05    0.47   R  91 A K  95 A         A  89 A                 \n",
      "  91    0.42    2.75    1.37    0.41 * P  92 A P  96 A         P  90 A                 \n",
      "  92    0.23    0.00    0.00    0.00   R  93 A -                                       \n",
      "  93    0.00    0.00    0.00    0.00   S  94 A -               -                       \n",
      "  94    0.00    0.00    0.00    0.00   M  97 A -               -                       \n",
      "  95    0.00    0.00    1.20    0.00   E  98 A E 102 A         -                       \n",
      "  96    0.00    0.00    0.95    0.00   R  99 A R 103 A         -                       \n",
      "  97    0.00    0.00    1.13    0.00   K 100 A A 104 A         -                       \n",
      "  98    0.00    0.00    0.98    0.00   D 101 A D 105 A         -                       \n",
      "  99    0.00    0.00    0.56    0.00   L 102 A L 106 A         -                       \n",
      " 100    0.58    2.40    1.09    0.51 * L 103 A L 107 A         L 101 A                 \n",
      " 101    0.39    2.38    1.08    0.50   S 104 A A 108 A         Q 102 A                 \n",
      " 102    0.33    1.73    0.99    0.46   A 105 A N 109 A         V 103 A                 \n",
      " 103    0.18    1.13    0.81    0.30 * N 106 A N 110 A         N 104 A                 \n",
      " 104    0.24    1.57    1.00    0.41   A 107 A G 111 A         G 105 A                 \n",
      " 105    0.22    1.28    0.75    0.35   E 108 A K 112 A         K 106 A                 \n",
      " 106    0.09    0.47    0.31    0.12 * I 109 A I 113 A         I 107 A                 \n",
      " 107    0.09    0.54    0.34    0.13 * F 110 A F 114 A         F 108 A                 \n",
      " 108    0.28    0.70    0.37    0.18   T 111 A G 115 A         T 109 A                 \n",
      " 109    0.20    0.52    0.31    0.04   V 112 A P 116 A         E 110 A                 \n",
      " 110    0.06    0.19    0.20    0.07 * Q 113 A Q 117 A         Q 111 A                 \n",
      " 111    0.07    0.48    0.24    0.11 * G 114 A G 118 A         G 112 A                 \n",
      " 112    0.15    0.36    0.29    0.04   A 115 A K 119 A         R 113 A                 \n",
      " 113    0.08    0.32    0.27    0.05 * A 116 A A 120 A         A 114 A                 \n",
      " 114    0.10    0.39    0.25    0.06   L 117 A I 121 A         L 115 A                 \n",
      " 115    0.18    0.50    0.24    0.07   N 118 A N 122 A         A 116 A                 \n",
      " 116    0.13    0.56    0.27    0.08   E 119 A D 123 A         E 117 A                 \n",
      " 117    0.14    0.39    0.24    0.02   V 120 A N 124 A         V 118 A                 \n",
      " 118    0.09    0.40    0.25    0.02 * A 121 A A 125 A         A 119 A                 \n",
      " 119    0.20    0.70    0.43    0.12   S 122 A A 126 A         K 120 A                 \n",
      " 120    0.24    1.20    0.85    0.31   R 123 A D 127 A         K 121 A                 \n",
      " 121    0.21    1.57    0.84    0.42 * D 124 A D 128 A         D 122 A                 \n",
      " 122    0.10    0.42    0.23    0.04   V 125 A I 129 A         V 123 A                 \n",
      " 123    0.11    0.33    0.18    0.06   K 126 A R 130 A         K 124 A                 \n",
      " 124    0.05    0.17    0.21    0.08 * V 127 A V 131 A         V 125 A                 \n",
      " 125    0.04    0.16    0.18    0.07 * L 128 A L 132 A         L 126 A                 \n",
      " 126    0.06    0.25    0.11    0.06 * V 129 A V 133 A         V 127 A                 \n",
      " 127    0.07    0.34    0.17    0.05 * V 130 A V 134 A         V 128 A                 \n",
      " 128    0.04    0.24    0.16    0.03 * G 131 A G 135 A         G 129 A                 \n",
      " 129    0.11    0.53    0.28    0.15 * N 132 A N 136 A         N 130 A                 \n",
      " 130    0.28    0.31    0.16    0.06 * P 133 A P 137 A         P 131 A                 \n",
      " 131    0.10    0.53    0.24    0.08 * A 134 A A 138 A         A 132 A                 \n",
      " 132    0.08    0.35    0.19    0.05 * N 135 A N 139 A         N 133 A                 \n",
      " 133    0.07    0.30    0.14    0.04 * T 136 A T 140 A         T 134 A                 \n",
      " 134    0.09    0.42    0.21    0.07 * N 137 A N 141 A         N 135 A                 \n",
      " 135    0.09    0.48    0.26    0.04 * A 138 A A 142 A         A 136 A                 \n",
      " 136    0.14    0.56    0.25    0.10   Y 139 A L 143 A         L 137 A                 \n",
      " 137    0.09    0.50    0.25    0.06 * I 140 A I 144 A         I 138 A                 \n",
      " 138    0.10    0.54    0.30    0.06 * A 141 A A 145 A         A 139 A                 \n",
      " 139    0.21    0.52    0.28    0.08   M 142 A S 146 A         Y 140 A                 \n",
      " 140    0.12    0.37    0.29    0.03   K 143 A A 147 A         K 141 A                 \n",
      " 141    0.18    0.59    0.37    0.02   S 144 A A 148 A         N 142 A                 \n",
      " 142    0.13    0.74    0.36    0.11 * A 145 A A 149 A         A 143 A                 \n",
      " 143    0.24    1.92    0.86    0.35 * P 146 A P 150 A         P 144 A                 \n",
      " 144    0.19    0.54    0.25    0.09   D 147 A D 151 A         G 145 A                 \n",
      " 145    0.14    0.60    0.38    0.04   L 148 A V 152 A         L 146 A                 \n",
      " 146    0.20    0.52    0.46    0.08   P 149 A P 153 A         N 147 A                 \n",
      " 147    0.19    0.51    0.49    0.11   K 150 A A 154 A         P 148 A                 \n",
      " 148    0.18    0.58    0.45    0.05   K 151 A S 155 A         R 149 A                 \n",
      " 149    0.14    0.40    0.28    0.09   N 152 A R 156 A         N 150 A                 \n",
      " 150    0.08    0.40    0.24    0.10 * F 153 A F 157 A         F 151 A                 \n",
      " 151    0.11    0.31    0.19    0.07   T 154 A N 158 A         T 152 A                 \n",
      " 152    0.06    0.13    0.13    0.04 * A 155 A A 159 A         A 153 A                 \n",
      " 153    0.10    0.37    0.19    0.04 * M 156 A M 160 A         M 154 A                 \n",
      " 154    0.18    0.32    0.27    0.08   L 157 A M 161 A         T 155 A                 \n",
      " 155    0.05    0.17    0.20    0.06 * R 158 A R 162 A         R 156 A                 \n",
      " 156    0.05    0.22    0.26    0.09 * L 159 A L 163 A         L 157 A                 \n",
      " 157    0.08    0.47    0.25    0.08 * D 160 A D 164 A         D 158 A                 \n",
      " 158    0.05    0.20    0.18    0.05 * H 161 A H 165 A         H 159 A                 \n",
      " 159    0.07    0.21    0.21    0.05 * N 162 A N 166 A         N 160 A                 \n",
      " 160    0.08    0.42    0.23    0.07 * R 163 A R 167 A         R 161 A                 \n",
      " 161    0.07    0.32    0.15    0.05 * A 164 A A 168 A         A 162 A                 \n",
      " 162    0.17    0.18    0.09    0.02   L 165 A I 169 A         K 163 A                 \n",
      " 163    0.12    0.19    0.25    0.09   S 166 A S 170 A         A 164 A                 \n",
      " 164    0.09    0.45    0.39    0.06 * Q 167 A Q 171 A         Q 165 A                 \n",
      " 165    0.07    0.35    0.32    0.06 * L 168 A L 172 A         L 166 A                 \n",
      " 166    0.09    0.50    0.44    0.11 * A 169 A A 173 A         A 167 A                 \n",
      " 167    0.16    0.44    0.53    0.18   A 170 A T 174 A         K 168 A                 \n",
      " 168    0.08    0.43    0.52    0.16 * K 171 A K 175 A         K 169 A                 \n",
      " 169    0.15    0.35    0.60    0.26   S 172 A L 176 A         T 170 A                 \n",
      " 170    0.06    0.42    0.60    0.24 * G 173 A G 177 A         G 171 A                 \n",
      " 171    0.18    0.52    0.65    0.22   K 174 A R 178 A         T 172 A                 \n",
      " 172    0.14    0.58    0.57    0.13   P 175 A G 179 A         G 173 A                 \n",
      " 173    0.17    0.74    0.49    0.11   V 176 A S 180 A         V 174 A                 \n",
      " 174    0.22    0.92    0.50    0.12   A 177 A A 181 A         D 175 A                 \n",
      " 175    0.19    0.49    0.43    0.13   S 178 A E 182 A         R 176 A                 \n",
      " 176    0.16    0.74    0.44    0.05   I 179 A F 183 A         I 177 A                 \n",
      " 177    0.26    1.13    0.60    0.12   E 180 A N 184 A         R 178 A                 \n",
      " 178    0.23    1.11    0.59    0.13   K 181 A N 185 A         R 179 A                 \n",
      " 179    0.21    1.00    0.53    0.21   L 182 A I 186 A         M 180 A                 \n",
      " 180    0.18    0.56    0.42    0.11   A 183 A V 187 A         T 181 A                 \n",
      " 181    0.08    0.42    0.40    0.14 * V 184 A V 188 A         V 182 A                 \n",
      " 182    0.06    0.22    0.11    0.03 * W 185 A W 189 A         W 183 A                 \n",
      " 183    0.06    0.25    0.16    0.05 * G 186 A G 190 A         G 184 A                 \n",
      " 184    0.44    1.46    0.65    0.25 * N 187 A N 191 A         N 185 A                 \n",
      " 185    0.26    1.01    0.54    0.10 * H 188 A H 192 A         H 186 A                 \n",
      " 186    0.09    0.37    0.42    0.12 * S 189 A S 193 A         S 187 A                 \n",
      " 187    0.17    0.56    0.47    0.14   P 190 A A 194 A         S 188 A                 \n",
      " 188    0.22    0.80    0.53    0.07   T 191 A T 195 A         I 189 A                 \n",
      " 189    0.15    0.36    0.31    0.11   M 192 A Q 196 A         M 190 A                 \n",
      " 190    0.09    0.31    0.19    0.08   Y 193 A F 197 A         F 191 A                 \n",
      " 191    0.04    0.19    0.12    0.01 * P 194 A P 198 A         P 192 A                 \n",
      " 192    0.05    0.16    0.19    0.06 * D 195 A D 199 A         D 193 A                 \n",
      " 193    0.21    0.93    0.49    0.22   F 196 A I 200 A         L 194 A                 \n",
      " 194    0.21    0.24    0.39    0.16   R 197 A T 201 A         F 195 A                 \n",
      " 195    0.23    0.75    0.44    0.10   F 198 A Y 202 A         H 196 A                 \n",
      " 196    0.12    0.68    0.38    0.09 * A 199 A A 203 A         A 197 A                 \n",
      " 197    0.22    0.81    0.47    0.08   T 200 A T 204 A         E 198 A                 \n",
      " 198    0.24    1.39    0.70    0.25   A 201 A V 205 A         V 199 A                 \n",
      " 199    0.25    1.19    1.13    0.24   E 202 A G 206 A         D 200 A                 \n",
      " 200    0.24    2.00    1.14    0.16 * G 203 A G 207 A         G 205 A                 \n",
      " 201    0.25    0.96    0.73    0.07   E 204 A E 208 A         R 206 A                 \n",
      " 202    0.23    0.87    0.62    0.12   S 205 A K 209 A         P 207 A                 \n",
      " 203    0.17    0.54    0.39    0.08   L 206 A V 210 A         A 208 A                 \n",
      " 204    0.12    0.29    0.42    0.16   L 207 A T 211 A         L 209 A                 \n",
      " 205    0.15    0.34    0.52    0.21   K 208 A D 212 A         E 210 A                 \n",
      " 206    0.09    0.30    0.74    0.39 * L 209 A L 213 A         L 211 A                 \n",
      " 207    0.12    0.38    0.80    0.39   I 210 A V 214 A         V 212 A                 \n",
      " 208    0.00    0.00    0.00    0.00   N 211 A -               -                       \n",
      " 209    0.21    1.41    0.67    0.20 * D 212 A D 215 A         D 214 A                 \n",
      " 210    0.30    0.45    0.21    0.11   D 213 A H 216 A         M 215 A                 \n",
      " 211    0.30    1.07    0.58    0.15   V 214 A D 217 A         E 216 A                 \n",
      " 212    0.20    1.14    0.59    0.18 * W 215 A W 218 A         W 217 A                 \n",
      " 213    0.22    0.72    0.48    0.08   N 216 A Y 219 A         Y 218 A                 \n",
      " 214    0.28    0.79    0.58    0.05   R 217 A V 220 A         E 219 A                 \n",
      " 215    0.25    1.22    0.64    0.24   D 218 A E 221 A         K 220 A                 \n",
      " 216    0.27    1.25    0.60    0.19   T 219 A E 222 A         V 221 A                 \n",
      " 217    0.11    0.71    0.34    0.10 * F 220 A F 223 A         F 222 A                 \n",
      " 218    0.12    0.78    0.36    0.12 * I 221 A I 224 A         I 223 A                 \n",
      " 219    0.17    1.40    0.63    0.22 * P 222 A P 225 A         P 224 A                 \n",
      " 220    0.21    1.11    0.50    0.17   T 223 A R 226 A         T 225 A                 \n",
      " 221    0.13    0.90    0.40    0.15 * V 224 A V 227 A         V 226 A                 \n",
      " 222    0.21    1.28    0.63    0.19   G 225 A A 228 A         A 227 A                 \n",
      " 223    0.30    1.67    0.76    0.33   K 226 A N 229 A         Q 228 A                 \n",
      " 224    0.16    1.12    0.51    0.18 * R 227 A R 230 A         R 229 A                 \n",
      " 225    0.14    1.12    0.53    0.18 * G 228 A G 231 A         G 230 A                 \n",
      " 226    0.18    1.26    0.65    0.21 * A 229 A A 232 A         A 231 A                 \n",
      " 227    0.22    1.36    0.61    0.25   A 230 A E 233 A         A 232 A                 \n",
      " 228    0.16    1.10    0.49    0.28 * I 231 A I 234 A         I 233 A                 \n",
      " 229    0.15    0.98    0.52    0.27 * I 232 A I 235 A         I 234 A                 \n",
      " 230    0.21    0.92    0.46    0.17   E 233 A E 236 A         Q 235 A                 \n",
      " 231    0.17    0.86    0.41    0.16   A 234 A V 237 A         A 236 A                 \n",
      " 232    0.15    0.99    0.52    0.21 * R 235 A R 238 A         R 237 A                 \n",
      " 233    0.14    1.12    0.51    0.25 * G 236 A G 239 A         G 238 A                 \n",
      " 234    0.20    0.74    0.38    0.17   L 237 A K 240 A         A 239 A                 \n",
      " 235    0.12    0.82    0.51    0.15 * S 238 A S 241 A         S 240 A                 \n",
      " 236    0.17    0.67    0.38    0.14 * S 239 A S 242 A         S 241 A                 \n",
      " 237    0.08    0.31    0.25    0.04 * A 240 A A 243 A         A 242 A                 \n",
      " 238    0.06    0.17    0.22    0.07 * A 241 A A 244 A         A 243 A                 \n",
      " 239    0.06    0.30    0.24    0.04 * S 242 A S 245 A         S 244 A                 \n",
      " 240    0.07    0.32    0.22    0.03 * A 243 A A 246 A         A 245 A                 \n",
      " 241    0.05    0.13    0.15    0.04 * A 244 A A 247 A         A 246 A                 \n",
      " 242    0.09    0.14    0.21    0.08   N 245 A S 248 A         N 247 A                 \n",
      " 243    0.08    0.13    0.20    0.08   A 246 A S 249 A         A 248 A                 \n",
      " 244    0.05    0.13    0.17    0.06 * A 247 A A 250 A         A 249 A                 \n",
      " 245    0.07    0.28    0.25    0.05 * I 248 A I 251 A         I 250 A                 \n",
      " 246    0.14    0.37    0.25    0.03   D 249 A D 252 A         E 251 A                 \n",
      " 247    0.07    0.27    0.21    0.02 * H 250 A H 253 A         H 252 A                 \n",
      " 248    0.12    0.19    0.22    0.07   V 251 A M 254 A         I 253 A                 \n",
      " 249    0.08    0.43    0.23    0.08 * R 252 A R 255 A         R 254 A                 \n",
      " 250    0.08    0.41    0.20    0.06 * D 253 A D 256 A         D 255 A                 \n",
      " 251    0.07    0.53    0.28    0.15 * W 254 A W 257 A         W 256 A                 \n",
      " 252    0.13    0.22    0.21    0.07   V 255 A V 258 A         A 257 A                 \n",
      " 253    0.14    0.41    0.22    0.05   L 256 A Q 259 A         L 258 A                 \n",
      " 254    0.09    0.58    0.38    0.11 * G 257 A G 260 A         G 259 A                 \n",
      " 255    0.12    0.61    0.27    0.10 * T 258 A T 261 A         T 260 A                 \n",
      " 256    0.43    2.28    1.56    0.10   N 259 A E 262 A         P 261 A                 \n",
      " 257    0.80    2.99    1.49    0.00   G 260 A -               E 262 A                 \n",
      " 258    0.00    0.00    0.00    0.00   -       -               G 263 A                 \n",
      " 259    0.48    1.89    1.08    0.29   K 261 A A 263 A         D 264 A                 \n",
      " 260    0.15    0.84    0.60    0.18 * W 262 A W 264 A         W 265 A                 \n",
      " 261    0.15    0.39    0.45    0.17   V 263 A S 265 A         V 266 A                 \n",
      " 262    0.08    0.13    0.21    0.09   T 264 A S 266 A         S 267 A                 \n",
      " 263    0.17    0.55    0.27    0.13   M 265 A A 267 A         M 268 A                 \n",
      " 264    0.12    0.46    0.27    0.03   G 266 A A 268 A         A 269 A                 \n",
      " 265    0.13    0.53    0.26    0.08   I 267 A I 269 A         V 270 A                 \n",
      " 266    0.08    0.57    0.26    0.10 * P 268 A P 270 A         P 271 A                 \n",
      " 267    0.30    0.43    0.27    0.01 * S 269 A S 271 A         S 272 A                 \n",
      " 268    0.46    0.91    0.43    0.12   D 270 A T 272 A         Q 273 A                 \n",
      " 269    0.03    0.12    0.16    0.06 * G 271 A G 273 A         G 274 A                 \n",
      " 270    0.37    1.10    0.62    0.24   S 272 A A 274 A         E 275 A                 \n",
      " 271    0.11    0.66    0.47    0.12 * Y 273 A Y 275 A         Y 277 A                 \n",
      " 272    0.16    1.27    0.80    0.29 * G 274 A G 276 A         G 278 A                 \n",
      " 273    0.09    0.37    0.35    0.10 * I 275 A I 277 A         I 279 A                 \n",
      " 274    0.09    0.50    0.40    0.08 * P 276 A P 278 A         P 280 A                 \n",
      " 275    0.10    0.42    0.40    0.10 * E 277 A E 279 A         E 281 A                 \n",
      " 276    0.22    0.25    0.89    0.51   D 278 A G 280 A         G 282 A                 \n",
      " 277    0.13    0.80    0.61    0.17 * I 279 A I 281 A         I 283 A                 \n",
      " 278    0.17    0.52    0.27    0.08   I 280 A F 282 A         V 284 A                 \n",
      " 279    0.14    0.45    0.30    0.09   Y 281 A V 283 A         Y 285 A                 \n",
      " 280    0.16    0.45    0.29    0.11   G 282 A G 284 A         S 286 A                 \n",
      " 281    0.25    0.21    0.14    0.01   V 283 A L 285 A         F 287 A                 \n",
      " 282    0.06    0.31    0.44    0.19 * P 284 A P 286 A         P 288 A                 \n",
      " 283    0.13    0.43    0.38    0.07   V 285 A T 287 A         V 289 A                 \n",
      " 284    0.29    0.47    0.53    0.16   I 286 A V 288 A         T 290 A                 \n",
      " 285    0.21    0.70    0.41    0.04   C 287 A S 289 A         A 291 A                 \n",
      " 286    0.19    0.78    0.65    0.17   E 288 A R 290 A         K 292 A                 \n",
      " 287    0.34    2.15    1.03    0.32   N 289 A N 291 A         D 293 A                 \n",
      " 288    0.11    0.85    0.51    0.14 * G 290 A G 292 A         G 294 A                 \n",
      " 289    0.21    0.73    0.60    0.09   E 291 A E 293 A         A 295 A                 \n",
      " 290    0.18    0.94    0.62    0.20   Y 292 A W 294 A         Y 296 A                 \n",
      " 291    0.21    0.87    0.52    0.05   K 293 A E 295 A         R 297 A                 \n",
      " 292    0.20    0.73    0.37    0.13   R 294 A I 296 A         V 298 A                 \n",
      " 293    0.11    0.64    0.32    0.12 * V 295 A V 297 A         V 299 A                 \n",
      " 294    0.17    1.13    0.63    0.22 * E 296 A E 298 A         E 300 A                 \n",
      " 295    0.08    0.55    0.73    0.26 * G 297 A G 299 A         G 301 A                 \n",
      " 296    0.12    0.74    0.59    0.07 * L 298 A L 300 A         L 302 A                 \n",
      " 297    0.15    0.99    0.65    0.04 * E 299 A E 301 A         E 303 A                 \n",
      " 298    0.12    0.72    0.51    0.03 * I 300 A I 302 A         I 304 A                 \n",
      " 299    0.16    0.55    0.41    0.08   D 301 A S 303 A         N 305 A                 \n",
      " 300    0.19    0.73    0.59    0.07   A 302 A D 304 A         E 306 A                 \n",
      " 301    0.10    0.65    0.65    0.16 * F 303 A F 305 A         F 307 A                 \n",
      " 302    0.15    0.38    0.52    0.19   S 304 A Q 306 A         A 308 A                 \n",
      " 303    0.06    0.26    0.47    0.21 * R 305 A R 307 A         R 309 A                 \n",
      " 304    0.14    0.23    0.32    0.13   E 306 A A 308 A         K 310 A                 \n",
      " 305    0.08    0.15    0.33    0.16   K 307 A R 309 A         R 311 A                 \n",
      " 306    0.09    0.08    0.33    0.19   M 308 A I 310 A         M 312 A                 \n",
      " 307    0.11    0.16    0.44    0.24   D 309 A D 311 A         E 313 A                 \n",
      " 308    0.19    0.36    0.46    0.16   G 310 A A 312 A         I 314 A                 \n",
      " 309    0.13    0.47    0.61    0.21   T 311 A N 313 A         T 315 A                 \n",
      " 310    0.10    0.21    0.56    0.30   L 312 A A 314 A         A 316 A                 \n",
      " 311    0.14    0.57    0.65    0.24   A 313 A Q 315 A         Q 317 A                 \n",
      " 312    0.14    0.90    0.71    0.16 * E 314 A E 316 A         E 318 A                 \n",
      " 313    0.07    0.39    0.44    0.13 * L 315 A L 317 A         L 319 A                 \n",
      " 314    0.18    0.80    0.53    0.21   L 316 A Q 318 A         L 320 A                 \n",
      " 315    0.26    1.34    0.70    0.28   E 317 A A 319 A         D 321 A                 \n",
      " 316    0.20    1.44    0.67    0.23 * E 318 A E 320 A         E 322 A                 \n",
      " 317    0.31    1.15    0.53    0.22   R 319 A R 321 A         M 323 A                 \n",
      " 318    0.23    1.35    0.60    0.21   D 320 A E 322 A         E 324 A                 \n",
      " 319    0.32    1.63    0.74    0.26   G 321 A A 323 A         Q 325 A                 \n",
      " 320    0.24    1.47    0.71    0.19 * V 322 A V 324 A         V 326 A                 \n",
      " 321    0.44    1.53    0.69    0.24   A 323 A R 325 A         K 327 A                 \n",
      " 322    0.71    3.99    1.83    0.82   H 324 A D 326 A         A 328 A                 \n",
      " 323    0.73    5.14    2.82    0.43 * L 325 A L 327 A         L 329 A                 \n",
      " 324    1.10    5.95    2.98    0.00   -       L 328 A         G 330 A                 \n",
      " 325    0.96    5.29    2.65    0.00   -       L 329 A         L 331 A                 \n",
      " 326    0.80    5.45    2.73    0.00   -       E 330 A         I 332 A                 \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    3\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      308      308        0.9821        0.9821        0.0019\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      308      308        0.9721        0.9721        0.0000\n",
      "\n",
      "Current alignment total score:        87.91\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -               M   0 A         -       \n",
      "   2    0.57    3.00    1.40    0.48   K   3 A -               K   1 A         M   1 A \n",
      "   3    0.34    2.30    1.12    0.38   P   4 A S   8 A         A   2 A         S   2 A \n",
      "   4    0.38    0.88    0.52    0.15   A   5 A T   9 A         P   3 A         A   3 A \n",
      "   5    0.17    0.23    0.16    0.04   K   6 A K  10 A         V   4 A         V   4 A \n",
      "   6    0.09    0.12    0.13    0.06   R   7 A K  11 A         R   5 A         K   5 A \n",
      "   7    0.06    0.24    0.18    0.03 * V   8 A V  12 A         V   6 A         V   6 A \n",
      "   8    0.11    0.49    0.25    0.07   A   9 A T  13 A         A   7 A         A   7 A \n",
      "   9    0.09    0.55    0.31    0.07 * V  10 A V  14 A         V   8 A         V   8 A \n",
      "  10    0.10    0.56    0.25    0.11 * T  11 A T  15 A         T   9 A         T   9 A \n",
      "  11    0.09    0.65    0.30    0.13 * G  12 A G  16 A         G  10 A         G  10 A \n",
      "  12    0.10    0.58    0.40    0.06 * A  13 A A  17 A         A  11 A         A  11 A \n",
      "  13    0.08    0.39    0.34    0.03 * A  14 A A  18 A         A  12 A         A  12 A \n",
      "  14    0.05    0.35    0.19    0.09 * G  15 A G  19 A         G  13 A         G  13 A \n",
      "  15    0.09    0.50    0.25    0.10 * Q  16 A Q  20 A         Q  14 A         Q  14 A \n",
      "  16    0.07    0.30    0.17    0.08 * I  17 A I  21 A         I  15 A         I  15 A \n",
      "  17    0.09    0.17    0.19    0.07   A  18 A S  22 A         G  16 A         G  16 A \n",
      "  18    0.05    0.30    0.18    0.04 * Y  19 A Y  23 A         Y  17 A         Y  17 A \n",
      "  19    0.11    0.19    0.19    0.03   S  20 A S  24 A         S  18 A         A  18 A \n",
      "  20    0.05    0.26    0.19    0.02 * L  21 A L  25 A         L  19 A         L  19 A \n",
      "  21    0.11    0.22    0.20    0.04   L  22 A L  26 A         L  20 A         V  20 A \n",
      "  22    0.23    0.38    0.35    0.05   F  23 A W  27 A         F  21 A         P  21 A \n",
      "  23    0.19    0.47    0.47    0.09   R  24 A R  28 A         R  22 A         L  22 A \n",
      "  24    0.06    0.27    0.37    0.11 * I  25 A I  29 A         I  23 A         I  23 A \n",
      "  25    0.06    0.25    0.26    0.09 * A  26 A A  30 A         A  24 A         A  24 A \n",
      "  26    0.16    0.22    0.31    0.12   N  27 A N  31 A         A  25 A         R  25 A \n",
      "  27    0.05    0.33    0.38    0.15 * G  28 A G  32 A         G  26 A         G  26 A \n",
      "  28    0.16    0.45    0.51    0.15   D  29 A E  33 A         E  27 A         A  27 A \n",
      "  29    0.11    0.42    0.55    0.19   L  30 A V  34 A         M  28 A         L  28 A \n",
      "  30    0.08    0.34    0.51    0.25   L  31 A F  35 A         L  29 A         L  29 A \n",
      "  31    0.03    0.20    0.47    0.26 * G  32 A G  36 A         G  30 A         G  30 A \n",
      "  32    0.18    0.45    0.42    0.24   K  33 A T  37 A         K  31 A         P  31 A \n",
      "  33    0.19    0.58    0.51    0.22   D  34 A D  38 A         D  32 A         T  32 A \n",
      "  34    0.38    0.51    0.41    0.17   Q  35 A T  39 A         Q  33 A         T  33 A \n",
      "  35    0.05    0.28    0.38    0.15 * P  36 A P  40 A         P  34 A         P  34 A \n",
      "  36    0.06    0.31    0.25    0.08 * V  37 A V  41 A         V  35 A         V  35 A \n",
      "  37    0.15    0.33    0.27    0.05   I  38 A E  42 A         I  36 A         E  36 A \n",
      "  38    0.07    0.39    0.21    0.05 * L  39 A L  43 A         L  37 A         L  37 A \n",
      "  39    0.16    0.48    0.24    0.08   Q  40 A K  44 A         Q  38 A         R  38 A \n",
      "  40    0.11    0.73    0.40    0.12 * L  41 A L  45 A         L  39 A         L  39 A \n",
      "  41    0.11    0.76    0.33    0.15 * L  42 A L  46 A         L  40 A         L  40 A \n",
      "  42    0.17    0.91    0.44    0.16   D  43 A E  47 A         E  41 A         D  41 A \n",
      "  43    0.27    0.63    0.36    0.13   L  44 A I  48 A         I  42 A         I  42 A \n",
      "  44    0.35    0.66    0.37    0.09   P  45 A P  49 A         P  43 A         E  43 A \n",
      "  45    0.20    0.40    0.43    0.13   Q  46 A Q  50 A         Q  44 A         P  44 A \n",
      "  46    0.09    0.51    0.34    0.08 * A  47 A A  51 A         A  45 A         A  45 A \n",
      "  47    0.14    0.50    0.34    0.08   Q  48 A L  52 A         M  46 A         L  46 A \n",
      "  48    0.15    0.53    0.37    0.07   A  49 A G  53 A         K  47 A         K  47 A \n",
      "  49    0.10    0.37    0.34    0.07   A  50 A G  54 A         A  48 A         A  48 A \n",
      "  50    0.13    0.46    0.33    0.13   V  51 A A  55 A         L  49 A         L  49 A \n",
      "  51    0.15    0.40    0.31    0.05   K  52 A E  56 A         E  50 A         A  50 A \n",
      "  52    0.06    0.44    0.27    0.05 * G  53 A G  57 A         G  51 A         G  51 A \n",
      "  53    0.07    0.39    0.29    0.09 * V  54 A V  58 A         V  52 A         V  52 A \n",
      "  54    0.31    0.29    0.36    0.12   V  55 A A  59 A         V  53 A         E  53 A \n",
      "  55    0.18    0.33    0.34    0.11   M  56 A M  60 A         M  54 A         A  54 A \n",
      "  56    0.06    0.23    0.26    0.10 * E  57 A E  61 A         E  55 A         E  55 A \n",
      "  57    0.05    0.22    0.24    0.13 * L  58 A L  62 A         L  56 A         L  56 A \n",
      "  58    0.13    0.30    0.37    0.18   D  59 A L  63 A         E  57 A         E  57 A \n",
      "  59    0.07    0.41    0.41    0.21 * D  60 A D  64 A         D  58 A         D  58 A \n",
      "  60    0.10    0.32    0.30    0.16   C  61 A S  65 A         C  59 A         C  59 A \n",
      "  61    0.08    0.40    0.37    0.17 * A  62 A A  66 A         A  60 A         A  60 A \n",
      "  62    0.05    0.27    0.25    0.08 * F  63 A F  67 A         F  61 A         F  61 A \n",
      "  63    0.05    0.28    0.23    0.06 * P  64 A P  68 A         P  62 A         P  62 A \n",
      "  64    0.07    0.37    0.28    0.11 * L  65 A L  69 A         L  63 A         L  63 A \n",
      "  65    0.07    0.41    0.20    0.11 * L  66 A L  70 A         L  64 A         L  64 A \n",
      "  66    0.17    0.36    0.40    0.19   A  67 A R  71 A         A  65 A         D  65 A \n",
      "  67    0.17    0.40    0.38    0.13   G  68 A N  72 A         G  66 A         K  66 A \n",
      "  68    0.12    0.52    0.36    0.03   V  69 A I  73 A         L  67 A         V  67 A \n",
      "  69    0.14    0.42    0.37    0.08   V  70 A T  74 A         E  68 A         V  68 A \n",
      "  70    0.15    0.65    0.39    0.07   I  71 A I  75 A         A  69 A         V  69 A \n",
      "  71    0.11    0.73    0.38    0.11 * T  72 A T  76 A         T  70 A         T  70 A \n",
      "  72    0.32    0.73    0.34    0.15   D  73 A A  77 A         D  71 A         A  71 A \n",
      "  73    0.10    0.61    0.32    0.15 * D  74 A D  78 A         D  72 A         D  72 A \n",
      "  74    0.11    0.54    0.29    0.09   P  75 A A  79 A         P  73 A         P  73 A \n",
      "  75    0.16    0.37    0.25    0.04   K  76 A N  80 A         D  74 A         R  74 A \n",
      "  76    0.13    0.52    0.29    0.08   V  77 A E  81 A         V  75 A         V  75 A \n",
      "  77    0.09    0.46    0.31    0.05 * A  78 A A  82 A         A  76 A         A  76 A \n",
      "  78    0.06    0.33    0.17    0.07 * F  79 A F  83 A         F  77 A         F  77 A \n",
      "  79    0.13    0.30    0.21    0.06   K  80 A D  84 A         K  78 A         D  78 A \n",
      "  80    0.11    0.18    0.18    0.04   D  81 A G  85 A         D  79 A         G  79 A \n",
      "  81    0.15    0.45    0.27    0.05   A  82 A A  86 A         A  80 A         V  80 A \n",
      "  82    0.25    0.51    0.24    0.12   D  83 A N  87 A         D  81 A         A  81 A \n",
      "  83    0.16    0.35    0.19    0.05   V  84 A A  88 A         Y  82 A         I  82 A \n",
      "  84    0.07    0.29    0.28    0.09 * A  85 A A  89 A         A  83 A         A  83 A \n",
      "  85    0.16    0.57    0.35    0.06   L  86 A F  90 A         L  84 A         I  84 A \n",
      "  86    0.16    0.69    0.31    0.12   L  87 A L  91 A         L  85 A         M  85 A \n",
      "  87    0.22    0.50    0.31    0.06   V  88 A V  92 A         V  86 A         C  86 A \n",
      "  88    0.12    0.29    0.25    0.11 * G  89 A G  93 A         G  87 A         G  87 A \n",
      "  89    0.15    0.30    0.40    0.25 * A  90 A A  94 A         A  88 A         A  88 A \n",
      "  90    0.32    0.37    0.84    0.55   R  91 A K  95 A         A  89 A         F  89 A \n",
      "  91    0.17    0.79    1.18    0.52 * P  92 A P  96 A         P  90 A         P  90 A \n",
      "  92    0.47    1.89    0.95    0.00   R  93 A -                               R  91 A \n",
      "  93    0.36    1.29    0.64    0.00   S  94 A -               -               K  92 A \n",
      "  94    0.00    0.00    0.00    0.00   -       -               -               A  93 A \n",
      "  95    0.00    0.00    0.00    0.00   -       -               -               G  94 A \n",
      "  96    0.55    3.01    1.50    0.00   M  97 A -               -               M  95 A \n",
      "  97    0.36    1.79    1.24    0.38   E  98 A E 102 A         -               E  96 A \n",
      "  98    0.33    1.63    1.00    0.43   R  99 A R 103 A         -               R  97 A \n",
      "  99    0.34    1.84    1.17    0.48   K 100 A A 104 A         -               K  98 A \n",
      " 100    0.24    1.25    0.94    0.34   D 101 A D 105 A         -               D  99 A \n",
      " 101    0.15    0.42    0.47    0.17   L 102 A L 106 A         -               L 100 A \n",
      " 102    0.22    1.04    1.00    0.54 * L 103 A L 107 A         L 101 A         L 101 A \n",
      " 103    0.24    1.11    0.96    0.60   S 104 A A 108 A         Q 102 A         E 102 A \n",
      " 104    0.48    0.99    0.98    0.33   A 105 A N 109 A         V 103 A         M 103 A \n",
      " 105    0.08    0.24    0.65    0.38 * N 106 A N 110 A         N 104 A         N 104 A \n",
      " 106    0.10    0.14    0.77    0.53   A 107 A G 111 A         G 105 A         A 105 A \n",
      " 107    0.13    0.29    0.62    0.38   E 108 A K 112 A         K 106 A         R 106 A \n",
      " 108    0.07    0.34    0.32    0.07 * I 109 A I 113 A         I 107 A         I 107 A \n",
      " 109    0.05    0.22    0.31    0.12 * F 110 A F 114 A         F 108 A         F 108 A \n",
      " 110    0.31    0.41    0.37    0.13   T 111 A G 115 A         T 109 A         K 109 A \n",
      " 111    0.16    0.52    0.35    0.05   V 112 A P 116 A         E 110 A         E 110 A \n",
      " 112    0.07    0.30    0.23    0.02 * Q 113 A Q 117 A         Q 111 A         Q 111 A \n",
      " 113    0.03    0.18    0.21    0.11 * G 114 A G 118 A         G 112 A         G 112 A \n",
      " 114    0.15    0.28    0.27    0.06   A 115 A K 119 A         R 113 A         E 113 A \n",
      " 115    0.06    0.23    0.25    0.05 * A 116 A A 120 A         A 114 A         A 114 A \n",
      " 116    0.10    0.26    0.24    0.05   L 117 A I 121 A         L 115 A         I 115 A \n",
      " 117    0.12    0.27    0.23    0.07   N 118 A N 122 A         A 116 A         A 116 A \n",
      " 118    0.16    0.45    0.30    0.09   E 119 A D 123 A         E 117 A         A 117 A \n",
      " 119    0.10    0.23    0.23    0.06   V 120 A N 124 A         V 118 A         V 118 A \n",
      " 120    0.06    0.19    0.23    0.05 * A 121 A A 125 A         A 119 A         A 119 A \n",
      " 121    0.09    0.10    0.35    0.18   S 122 A A 126 A         K 120 A         A 120 A \n",
      " 122    0.14    0.25    0.68    0.40   R 123 A D 127 A         K 121 A         S 121 A \n",
      " 123    0.10    0.59    0.72    0.45 * D 124 A D 128 A         D 122 A         D 122 A \n",
      " 124    0.20    0.26    0.23    0.03   V 125 A I 129 A         V 123 A         C 123 A \n",
      " 125    0.12    0.46    0.25    0.07   K 126 A R 130 A         K 124 A         R 124 A \n",
      " 126    0.06    0.23    0.20    0.10 * V 127 A V 131 A         V 125 A         V 125 A \n",
      " 127    0.14    0.49    0.24    0.12   L 128 A L 132 A         L 126 A         V 126 A \n",
      " 128    0.06    0.28    0.16    0.04 * V 129 A V 133 A         V 127 A         V 127 A \n",
      " 129    0.08    0.42    0.22    0.07 * V 130 A V 134 A         V 128 A         V 128 A \n",
      " 130    0.09    0.64    0.27    0.15 * G 131 A G 135 A         G 129 A         G 129 A \n",
      " 131    0.07    0.22    0.25    0.14 * N 132 A N 136 A         N 130 A         N 130 A \n",
      " 132    0.12    0.24    0.17    0.06 * P 133 A P 137 A         P 131 A         P 131 A \n",
      " 133    0.05    0.15    0.21    0.08 * A 134 A A 138 A         A 132 A         A 132 A \n",
      " 134    0.06    0.13    0.17    0.06 * N 135 A N 139 A         N 133 A         N 133 A \n",
      " 135    0.05    0.13    0.14    0.04 * T 136 A T 140 A         T 134 A         T 134 A \n",
      " 136    0.05    0.06    0.17    0.10 * N 137 A N 141 A         N 135 A         N 135 A \n",
      " 137    0.05    0.16    0.22    0.08 * A 138 A A 142 A         A 136 A         A 136 A \n",
      " 138    0.08    0.18    0.23    0.10   Y 139 A L 143 A         L 137 A         L 137 A \n",
      " 139    0.04    0.08    0.20    0.10 * I 140 A I 144 A         I 138 A         I 138 A \n",
      " 140    0.14    0.20    0.27    0.09   A 141 A A 145 A         A 139 A         L 139 A \n",
      " 141    0.16    0.35    0.28    0.08   M 142 A S 146 A         Y 140 A         L 140 A \n",
      " 142    0.10    0.35    0.29    0.05   K 143 A A 147 A         K 141 A         K 141 A \n",
      " 143    0.18    0.89    0.49    0.12   S 144 A A 148 A         N 142 A         S 142 A \n",
      " 144    0.20    1.22    0.59    0.20 * A 145 A A 149 A         A 143 A         A 143 A \n",
      " 145    0.00    0.00    0.00    0.00   -       -               -               Q 144 A \n",
      " 146    0.41    1.96    1.00    0.61   P 146 A P 150 A         P 144 A         G 145 A \n",
      " 147    0.26    0.97    0.44    0.17   D 147 A D 151 A         G 145 A         K 146 A \n",
      " 148    0.20    1.35    0.60    0.30   L 148 A V 152 A         L 146 A         L 147 A \n",
      " 149    0.17    0.57    0.46    0.12   P 149 A P 153 A         N 147 A         N 148 A \n",
      " 150    0.18    0.83    0.56    0.07   K 150 A A 154 A         P 148 A         P 149 A \n",
      " 151    0.18    0.85    0.52    0.12   K 151 A S 155 A         R 149 A         R 150 A \n",
      " 152    0.18    0.42    0.30    0.11   N 152 A R 156 A         N 150 A         H 151 A \n",
      " 153    0.16    0.31    0.24    0.09   F 153 A F 157 A         F 151 A         V 152 A \n",
      " 154    0.09    0.29    0.19    0.10   T 154 A N 158 A         T 152 A         T 153 A \n",
      " 155    0.06    0.22    0.14    0.05 * A 155 A A 159 A         A 153 A         A 154 A \n",
      " 156    0.08    0.25    0.20    0.03 * M 156 A M 160 A         M 154 A         M 155 A \n",
      " 157    0.24    0.22    0.24    0.11   L 157 A M 161 A         T 155 A         T 156 A \n",
      " 158    0.16    0.17    0.18    0.08 * R 158 A R 162 A         R 156 A         R 157 A \n",
      " 159    0.04    0.18    0.23    0.10 * L 159 A L 163 A         L 157 A         L 158 A \n",
      " 160    0.05    0.22    0.23    0.10 * D 160 A D 164 A         D 158 A         D 159 A \n",
      " 161    0.05    0.16    0.17    0.07 * H 161 A H 165 A         H 159 A         H 160 A \n",
      " 162    0.05    0.10    0.18    0.07 * N 162 A N 166 A         N 160 A         N 161 A \n",
      " 163    0.05    0.14    0.20    0.08 * R 163 A R 167 A         R 161 A         R 162 A \n",
      " 164    0.05    0.17    0.15    0.05 * A 164 A A 168 A         A 162 A         A 163 A \n",
      " 165    0.12    0.37    0.17    0.07   L 165 A I 169 A         K 163 A         L 164 A \n",
      " 166    0.10    0.47    0.28    0.13   S 166 A S 170 A         A 164 A         S 165 A \n",
      " 167    0.20    0.50    0.40    0.07   Q 167 A Q 171 A         Q 165 A         L 166 A \n",
      " 168    0.06    0.31    0.30    0.05 * L 168 A L 172 A         L 166 A         L 167 A \n",
      " 169    0.06    0.24    0.38    0.16 * A 169 A A 173 A         A 167 A         A 168 A \n",
      " 170    0.14    0.18    0.44    0.23   A 170 A T 174 A         K 168 A         R 169 A \n",
      " 171    0.09    0.48    0.50    0.12 * K 171 A K 175 A         K 169 A         K 170 A \n",
      " 172    0.16    0.42    0.55    0.23   S 172 A L 176 A         T 170 A         A 171 A \n",
      " 173    0.09    0.69    0.61    0.16 * G 173 A G 177 A         G 171 A         G 172 A \n",
      " 174    0.24    0.93    0.67    0.25   K 174 A R 178 A         T 172 A         V 173 A \n",
      " 175    0.20    0.92    0.61    0.21   P 175 A G 179 A         G 173 A         P 174 A \n",
      " 176    0.13    0.60    0.49    0.14   V 176 A S 180 A         V 174 A         V 175 A \n",
      " 177    0.21    0.98    0.59    0.19   A 177 A A 181 A         D 175 A         S 176 A \n",
      " 178    0.19    0.60    0.44    0.15   S 178 A E 182 A         R 176 A         Q 177 A \n",
      " 179    0.22    1.19    0.59    0.25   I 179 A F 183 A         I 177 A         V 178 A \n",
      " 180    0.57    2.17    0.96    0.49   E 180 A N 184 A         R 178 A         R 179 A \n",
      " 181    0.46    1.89    0.89    0.39   K 181 A N 185 A         R 179 A         N 180 A \n",
      " 182    0.27    1.60    0.78    0.33   L 182 A I 186 A         M 180 A         V 181 A \n",
      " 183    0.23    1.05    0.55    0.22   A 183 A V 187 A         T 181 A         I 182 A \n",
      " 184    0.16    0.82    0.44    0.26   V 184 A V 188 A         V 182 A         I 183 A \n",
      " 185    0.07    0.44    0.20    0.07 * W 185 A W 189 A         W 183 A         W 184 A \n",
      " 186    0.09    0.66    0.30    0.13 * G 186 A G 190 A         G 184 A         G 185 A \n",
      " 187    0.31    0.98    0.63    0.39 * N 187 A N 191 A         N 185 A         N 186 A \n",
      " 188    0.13    0.86    0.56    0.23 * H 188 A H 192 A         H 186 A         H 187 A \n",
      " 189    0.29    0.94    0.53    0.16 * S 189 A S 193 A         S 187 A         S 188 A \n",
      " 190    0.22    1.26    0.62    0.30   P 190 A A 194 A         S 188 A         S 189 A \n",
      " 191    0.21    1.30    0.68    0.26   T 191 A T 195 A         I 189 A         T 190 A \n",
      " 192    0.21    0.82    0.44    0.13   M 192 A Q 196 A         M 190 A         Q 191 A \n",
      " 193    0.23    0.89    0.39    0.16   Y 193 A F 197 A         F 191 A         V 192 A \n",
      " 194    0.15    1.20    0.47    0.25 * P 194 A P 198 A         P 192 A         P 193 A \n",
      " 195    0.14    0.98    0.41    0.21 * D 195 A D 199 A         D 193 A         D 194 A \n",
      " 196    0.27    1.09    0.64    0.16   F 196 A I 200 A         L 194 A         T 195 A \n",
      " 197    0.27    1.03    0.52    0.24   R 197 A T 201 A         F 195 A         D 196 A \n",
      " 198    0.26    0.98    0.56    0.15   F 198 A Y 202 A         H 196 A         S 197 A \n",
      " 199    0.15    0.99    0.53    0.15 * A 199 A A 203 A         A 197 A         A 198 A \n",
      " 200    0.24    0.97    0.57    0.14   T 200 A T 204 A         E 198 A         V 199 A \n",
      " 201    0.17    0.58    0.66    0.22   A 201 A V 205 A         V 199 A         I 200 A \n",
      " 202    0.33    2.16    1.31    0.40   E 202 A G 206 A         D 200 A         G 201 A \n",
      " 203    0.40    2.35    1.38    0.36   G 203 A G 207 A         G 205 A         T 202 A \n",
      " 204    0.26    1.21    0.80    0.20   E 204 A E 208 A         R 206 A         T 203 A \n",
      " 205    0.18    0.77    0.63    0.10   S 205 A K 209 A         P 207 A         P 204 A \n",
      " 206    0.22    1.16    0.58    0.18   L 206 A V 210 A         A 208 A         A 205 A \n",
      " 207    0.36    1.99    0.84    0.43   L 207 A T 211 A         L 209 A         R 206 A \n",
      " 208    0.31    2.07    0.89    0.50   K 208 A D 212 A         E 210 A         E 207 A \n",
      " 209    0.42    2.41    1.18    0.47   L 209 A L 213 A         L 211 A         A 208 A \n",
      " 210    0.75    2.93    1.37    0.56   I 210 A V 214 A         V 212 A         I 209 A \n",
      " 211    1.12    6.85    3.43    0.00   N 211 A -               -               K 210 A \n",
      " 212    0.00    0.00    0.67    0.20   D 212 A D 215 A         D 214 A         -       \n",
      " 213    0.00    0.00    0.21    0.11   D 213 A H 216 A         M 215 A         -       \n",
      " 214    0.00    0.00    0.58    0.15   V 214 A D 217 A         E 216 A         -       \n",
      " 215    0.00    0.00    0.59    0.18   W 215 A W 218 A         W 217 A         -       \n",
      " 216    0.00    0.00    0.48    0.08   N 216 A Y 219 A         Y 218 A         -       \n",
      " 217    0.00    0.00    0.58    0.05   R 217 A V 220 A         E 219 A         -       \n",
      " 218    0.00    0.00    0.64    0.24   D 218 A E 221 A         K 220 A         -       \n",
      " 219    0.41    0.66    0.56    0.24   T 219 A E 222 A         V 221 A         D 217 A \n",
      " 220    0.29    0.56    0.35    0.18 * F 220 A F 223 A         F 222 A         F 218 A \n",
      " 221    0.29    0.57    0.37    0.19   I 221 A I 224 A         I 223 A         V 219 A \n",
      " 222    0.18    0.27    0.53    0.25   P 222 A P 225 A         P 224 A         Q 220 A \n",
      " 223    0.16    0.31    0.42    0.23   T 223 A R 226 A         T 225 A         V 221 A \n",
      " 224    0.09    0.57    0.39    0.22 * V 224 A V 227 A         V 226 A         V 222 A \n",
      " 225    0.34    0.52    0.57    0.24   G 225 A A 228 A         A 227 A         R 223 A \n",
      " 226    0.14    0.10    0.59    0.42   K 226 A N 229 A         Q 228 A         G 224 A \n",
      " 227    0.06    0.23    0.41    0.24 * R 227 A R 230 A         R 229 A         R 225 A \n",
      " 228    0.05    0.32    0.45    0.24 * G 228 A G 231 A         G 230 A         G 226 A \n",
      " 229    0.10    0.57    0.58    0.27 * A 229 A A 232 A         A 231 A         A 227 A \n",
      " 230    0.15    0.61    0.55    0.30   A 230 A E 233 A         A 232 A         E 228 A \n",
      " 231    0.10    0.62    0.50    0.25 * I 231 A I 234 A         I 233 A         I 229 A \n",
      " 232    0.09    0.48    0.50    0.22 * I 232 A I 235 A         I 234 A         I 230 A \n",
      " 233    0.13    0.44    0.42    0.20   E 233 A E 236 A         Q 235 A         Q 231 A \n",
      " 234    0.20    0.77    0.47    0.17   A 234 A V 237 A         A 236 A         L 232 A \n",
      " 235    0.13    0.91    0.61    0.15 * R 235 A R 238 A         R 237 A         R 233 A \n",
      " 236    0.10    0.76    0.53    0.27 * G 236 A G 239 A         G 238 A         G 234 A \n",
      " 237    0.17    0.67    0.43    0.17   L 237 A K 240 A         A 239 A         L 235 A \n",
      " 238    0.11    0.67    0.53    0.15 * S 238 A S 241 A         S 240 A         S 236 A \n",
      " 239    0.13    0.54    0.40    0.13 * S 239 A S 242 A         S 241 A         S 237 A \n",
      " 240    0.08    0.36    0.27    0.04 * A 240 A A 243 A         A 242 A         A 238 A \n",
      " 241    0.17    0.29    0.21    0.10   A 241 A A 244 A         A 243 A         M 239 A \n",
      " 242    0.05    0.22    0.23    0.04 * S 242 A S 245 A         S 244 A         S 240 A \n",
      " 243    0.08    0.37    0.25    0.04 * A 243 A A 246 A         A 245 A         A 241 A \n",
      " 244    0.07    0.36    0.20    0.05 * A 244 A A 247 A         A 246 A         A 242 A \n",
      " 245    0.14    0.25    0.21    0.08   N 245 A S 248 A         N 247 A         K 243 A \n",
      " 246    0.07    0.17    0.19    0.07   A 246 A S 249 A         A 248 A         A 244 A \n",
      " 247    0.06    0.25    0.19    0.04 * A 247 A A 250 A         A 249 A         A 245 A \n",
      " 248    0.12    0.42    0.27    0.09   I 248 A I 251 A         I 250 A         V 246 A \n",
      " 249    0.08    0.29    0.24    0.07   D 249 A D 252 A         E 251 A         D 247 A \n",
      " 250    0.06    0.23    0.20    0.05 * H 250 A H 253 A         H 252 A         H 248 A \n",
      " 251    0.12    0.38    0.23    0.12   V 251 A M 254 A         I 253 A         V 249 A \n",
      " 252    0.19    0.51    0.28    0.11   R 252 A R 255 A         R 254 A         H 250 A \n",
      " 253    0.08    0.52    0.28    0.09 * D 253 A D 256 A         D 255 A         D 251 A \n",
      " 254    0.05    0.37    0.27    0.16 * W 254 A W 257 A         W 256 A         W 252 A \n",
      " 255    0.15    0.51    0.24    0.17   V 255 A V 258 A         A 257 A         I 253 A \n",
      " 256    0.20    0.52    0.28    0.09   L 256 A Q 259 A         L 258 A         H 254 A \n",
      " 257    0.09    0.61    0.40    0.17 * G 257 A G 260 A         G 259 A         G 255 A \n",
      " 258    0.10    0.54    0.34    0.05 * T 258 A T 261 A         T 260 A         T 256 A \n",
      " 259    0.25    1.25    1.40    0.38   N 259 A E 262 A         P 261 A         P 257 A \n",
      " 260    0.41    1.32    1.29    0.46   G 260 A -               E 262 A         E 258 A \n",
      " 261    0.25    0.74    0.37    0.00   -       -               G 263 A         G 259 A \n",
      " 262    0.29    0.62    0.94    0.35   K 261 A A 263 A         D 264 A         V 260 A \n",
      " 263    0.19    0.44    0.54    0.19   W 262 A W 264 A         W 265 A         Y 261 A \n",
      " 264    0.14    0.62    0.47    0.18   V 263 A S 265 A         V 266 A         V 262 A \n",
      " 265    0.10    0.41    0.25    0.08   T 264 A S 266 A         S 267 A         S 263 A \n",
      " 266    0.15    0.55    0.33    0.13   M 265 A A 267 A         M 268 A         M 264 A \n",
      " 267    0.12    0.43    0.30    0.03   G 266 A A 268 A         A 269 A         G 265 A \n",
      " 268    0.08    0.22    0.25    0.07   I 267 A I 269 A         V 270 A         V 266 A \n",
      " 269    0.20    0.26    0.25    0.10   P 268 A P 270 A         P 271 A         Y 267 A \n",
      " 270    0.15    0.35    0.28    0.05 * S 269 A S 271 A         S 272 A         S 268 A \n",
      " 271    0.44    2.22    0.90    0.52   D 270 A T 272 A         Q 273 A         D 269 A \n",
      " 272    0.59    4.05    1.52    0.89   G 271 A G 273 A         G 274 A         E 270 A \n",
      " 273    0.50    2.51    1.08    0.59   S 272 A A 274 A         E 275 A         N 271 A \n",
      " 274    0.00    0.00    0.00    0.00   -       -               -               P 272 A \n",
      " 275    0.13    0.72    0.48    0.21 * Y 273 A Y 275 A         Y 277 A         Y 273 A \n",
      " 276    0.24    1.74    0.92    0.51 * G 274 A G 276 A         G 278 A         G 274 A \n",
      " 277    0.14    0.60    0.38    0.15   I 275 A I 277 A         I 279 A         V 275 A \n",
      " 278    0.07    0.40    0.37    0.12 * P 276 A P 278 A         P 280 A         P 276 A \n",
      " 279    0.19    0.57    0.41    0.15   E 277 A E 279 A         E 281 A         S 277 A \n",
      " 280    0.23    0.89    0.85    0.47   D 278 A G 280 A         G 282 A         G 278 A \n",
      " 281    0.12    0.27    0.51    0.23   I 279 A I 281 A         I 283 A         L 279 A \n",
      " 282    0.12    0.40    0.29    0.06   I 280 A F 282 A         V 284 A         I 280 A \n",
      " 283    0.15    0.46    0.30    0.14   Y 281 A V 283 A         Y 285 A         F 281 A \n",
      " 284    0.14    0.48    0.29    0.17   G 282 A G 284 A         S 286 A         S 282 A \n",
      " 285    0.20    0.37    0.19    0.08   V 283 A L 285 A         F 287 A         F 283 A \n",
      " 286    0.08    0.47    0.42    0.20 * P 284 A P 286 A         P 288 A         P 284 A \n",
      " 287    0.21    0.40    0.37    0.09   V 285 A T 287 A         V 289 A         C 285 A \n",
      " 288    0.22    0.45    0.48    0.19   I 286 A V 288 A         T 290 A         T 286 A \n",
      " 289    0.18    0.61    0.44    0.05   C 287 A S 289 A         A 291 A         C 287 A \n",
      " 290    0.24    0.95    0.71    0.04   E 288 A R 290 A         K 292 A         H 288 A \n",
      " 291    0.26    1.25    0.99    0.43   N 289 A N 291 A         D 293 A         A 289 A \n",
      " 292    0.09    0.65    0.49    0.21 * G 290 A G 292 A         G 294 A         G 290 A \n",
      " 293    0.14    0.70    0.60    0.09   E 291 A E 293 A         A 295 A         E 291 A \n",
      " 294    0.17    0.75    0.64    0.10   Y 292 A W 294 A         Y 296 A         W 292 A \n",
      " 295    0.21    0.84    0.57    0.09   K 293 A E 295 A         R 297 A         T 293 A \n",
      " 296    0.18    0.85    0.47    0.17   R 294 A I 296 A         V 298 A         V 294 A \n",
      " 297    0.10    0.56    0.36    0.13 * V 295 A V 297 A         V 299 A         V 295 A \n",
      " 298    0.31    1.67    0.82    0.43   E 296 A E 298 A         E 300 A         S 296 A \n",
      " 299    0.35    2.89    1.27    0.63 * G 297 A G 299 A         G 301 A         G 297 A \n",
      " 300    0.52    3.05    1.26    0.66   L 298 A L 300 A         L 302 A         K 298 A \n",
      " 301    0.91    4.21    1.68    0.90   E 299 A E 301 A         E 303 A         L 299 A \n",
      " 302    0.00    0.00    0.51    0.03   I 300 A I 302 A         I 304 A         -       \n",
      " 303    0.00    0.00    0.41    0.08   D 301 A S 303 A         N 305 A         -       \n",
      " 304    0.00    0.00    0.59    0.07   A 302 A D 304 A         E 306 A         -       \n",
      " 305    0.00    0.00    0.65    0.16   F 303 A F 305 A         F 307 A         -       \n",
      " 306    0.00    0.00    0.52    0.19   S 304 A Q 306 A         A 308 A         -       \n",
      " 307    0.61    2.67    1.07    0.63   R 305 A R 307 A         R 309 A         K 305 A \n",
      " 308    0.62    3.02    1.15    0.69   E 306 A A 308 A         K 310 A         Q 306 A \n",
      " 309    0.49    2.78    1.08    0.63   K 307 A R 309 A         R 311 A         R 307 A \n",
      " 310    0.38    1.35    0.62    0.25   M 308 A I 310 A         M 312 A         L 308 A \n",
      " 311    0.31    1.47    0.71    0.28   D 309 A D 311 A         E 313 A         A 309 A \n",
      " 312    0.24    1.13    0.61    0.23   G 310 A A 312 A         I 314 A         S 310 A \n",
      " 313    0.14    0.72    0.58    0.28   T 311 A N 313 A         T 315 A         T 311 A \n",
      " 314    0.22    0.92    0.64    0.25   L 312 A A 314 A         A 316 A         I 312 A \n",
      " 315    0.19    0.87    0.69    0.19   A 313 A Q 315 A         Q 317 A         A 313 A \n",
      " 316    0.10    0.56    0.64    0.21 * E 314 A E 316 A         E 318 A         E 314 A \n",
      " 317    0.07    0.41    0.40    0.17 * L 315 A L 317 A         L 319 A         L 315 A \n",
      " 318    0.15    0.45    0.50    0.19   L 316 A Q 318 A         L 320 A         Q 316 A \n",
      " 319    0.15    0.63    0.67    0.22   E 317 A A 319 A         D 321 A         E 317 A \n",
      " 320    0.08    0.39    0.59    0.24 * E 318 A E 320 A         E 322 A         E 318 A \n",
      " 321    0.13    0.39    0.48    0.21   R 319 A R 321 A         M 323 A         R 319 A \n",
      " 322    0.21    0.86    0.61    0.26   D 320 A E 322 A         E 324 A         A 320 A \n",
      " 323    0.21    0.95    0.71    0.36   G 321 A A 323 A         Q 325 A         Q 321 A \n",
      " 324    0.34    0.88    0.70    0.24   V 322 A V 324 A         V 326 A         A 322 A \n",
      " 325    0.37    1.78    0.94    0.32   A 323 A R 325 A         K 327 A         G 323 A \n",
      " 326    0.78    5.35    2.72    0.99   H 324 A D 326 A         A 328 A         L 324 A \n",
      " 327    0.00    0.00    2.82    0.43   L 325 A L 327 A         L 329 A         -       \n",
      " 328    0.00    0.00    2.98    0.00   -       L 328 A         G 330 A         -       \n",
      " 329    0.00    0.00    2.65    0.00   -       L 329 A         L 331 A         -       \n",
      " 330    0.00    0.00    2.73    0.00   -       E 330 A         I 332 A         -       \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    4\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      320      320        1.0806        1.0806        0.0001\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      320      320        1.0806        1.0806        0.0000\n",
      "\n",
      "Current alignment total score:        88.19\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -       -       M   0 A         -       \n",
      "   2    0.26    0.58    1.17    0.57   K   3 A -       S   1 A K   1 A         M   1 A \n",
      "   3    0.20    0.66    1.02    0.40   P   4 A S   8 A E   2 A A   2 A         S   2 A \n",
      "   4    0.30    0.54    0.52    0.12   A   5 A T   9 A P   3 A P   3 A         A   3 A \n",
      "   5    0.35    0.58    0.25    0.11   K   6 A K  10 A I   4 A V   4 A         V   4 A \n",
      "   6    0.12    0.56    0.22    0.13   R   7 A K  11 A R   5 A R   5 A         K   5 A \n",
      "   7    0.06    0.27    0.19    0.04 * V   8 A V  12 A V   6 A V   6 A         V   6 A \n",
      "   8    0.17    0.35    0.27    0.05   A   9 A T  13 A L   7 A A   7 A         A   7 A \n",
      "   9    0.07    0.34    0.32    0.03 * V  10 A V  14 A V   8 A V   8 A         V   8 A \n",
      "  10    0.06    0.24    0.24    0.11 * T  11 A T  15 A T   9 A T   9 A         T   9 A \n",
      "  11    0.08    0.58    0.33    0.17 * G  12 A G  16 A G  10 A G  10 A         G  10 A \n",
      "  12    0.07    0.28    0.36    0.10 * A  13 A A  17 A A  11 A A  11 A         A  11 A \n",
      "  13    0.08    0.38    0.34    0.04 * A  14 A A  18 A A  12 A A  12 A         A  12 A \n",
      "  14    0.06    0.39    0.22    0.09 * G  15 A G  19 A G  13 A G  13 A         G  13 A \n",
      "  15    0.07    0.26    0.24    0.09 * Q  16 A Q  20 A Q  14 A Q  14 A         Q  14 A \n",
      "  16    0.06    0.16    0.17    0.07 * I  17 A I  21 A I  15 A I  15 A         I  15 A \n",
      "  17    0.13    0.39    0.22    0.09   A  18 A S  22 A A  16 A G  16 A         G  16 A \n",
      "  18    0.04    0.11    0.16    0.05 * Y  19 A Y  23 A Y  17 A Y  17 A         Y  17 A \n",
      "  19    0.09    0.37    0.22    0.06   S  20 A S  24 A S  18 A S  18 A         A  18 A \n",
      "  20    0.10    0.57    0.27    0.10 * L  21 A L  25 A L  19 A L  19 A         L  19 A \n",
      "  21    0.22    0.54    0.27    0.10   L  22 A L  26 A L  20 A L  20 A         V  20 A \n",
      "  22    0.19    0.67    0.40    0.11   F  23 A W  27 A Y  21 A F  21 A         P  21 A \n",
      "  23    0.18    0.44    0.45    0.10   R  24 A R  28 A S  22 A R  22 A         L  22 A \n",
      "  24    0.07    0.29    0.34    0.12 * I  25 A I  29 A I  23 A I  23 A         I  23 A \n",
      "  25    0.14    0.30    0.26    0.08   A  26 A A  30 A G  24 A A  24 A         A  24 A \n",
      "  26    0.14    0.39    0.32    0.11   N  27 A N  31 A N  25 A A  25 A         R  25 A \n",
      "  27    0.08    0.56    0.41    0.10 * G  28 A G  32 A G  26 A G  26 A         G  26 A \n",
      "  28    0.27    1.34    0.69    0.21   D  29 A E  33 A S  27 A E  27 A         A  27 A \n",
      "  29    0.22    1.22    0.70    0.17   L  30 A V  34 A V  28 A M  28 A         L  28 A \n",
      "  30    0.15    0.56    0.50    0.24   L  31 A F  35 A F  29 A L  29 A         L  29 A \n",
      "  31    0.11    0.80    0.55    0.16 * G  32 A G  36 A G  30 A G  30 A         G  30 A \n",
      "  32    0.14    0.51    0.44    0.18   K  33 A T  37 A K  31 A K  31 A         P  31 A \n",
      "  33    0.16    0.85    0.58    0.13   D  34 A D  38 A D  32 A D  32 A         T  32 A \n",
      "  34    0.18    0.34    0.39    0.15   Q  35 A T  39 A Q  33 A Q  33 A         T  33 A \n",
      "  35    0.09    0.59    0.43    0.10 * P  36 A P  40 A P  34 A P  34 A         P  34 A \n",
      "  36    0.14    0.50    0.30    0.08   V  37 A V  41 A I  35 A V  35 A         V  35 A \n",
      "  37    0.22    1.00    0.43    0.19   I  38 A E  42 A I  36 A I  36 A         E  36 A \n",
      "  38    0.10    0.57    0.27    0.12 * L  39 A L  43 A L  37 A L  37 A         L  37 A \n",
      "  39    0.18    0.30    0.25    0.04   Q  40 A K  44 A V  38 A Q  38 A         R  38 A \n",
      "  40    0.09    0.51    0.41    0.10 * L  41 A L  45 A L  39 A L  39 A         L  39 A \n",
      "  41    0.08    0.49    0.36    0.11 * L  42 A L  46 A L  40 A L  40 A         L  40 A \n",
      "  42    0.11    0.41    0.42    0.15   D  43 A E  47 A D  41 A E  41 A         D  41 A \n",
      "  43    0.28    1.07    0.51    0.21   L  44 A I  48 A I  42 A I  42 A         I  42 A \n",
      "  44    0.39    1.31    0.56    0.27   P  45 A P  49 A T  43 A P  43 A         E  43 A \n",
      "  45    0.30    1.62    0.68    0.33   Q  46 A Q  50 A P  44 A Q  44 A         P  44 A \n",
      "  46    0.27    1.17    0.51    0.23   A  47 A A  51 A M  45 A A  45 A         A  45 A \n",
      "  47    0.23    1.21    0.51    0.28   Q  48 A L  52 A M  46 A M  46 A         L  46 A \n",
      "  48    0.24    1.27    0.54    0.29   A  49 A G  53 A G  47 A K  47 A         K  47 A \n",
      "  49    0.24    1.08    0.48    0.23   A  50 A G  54 A V  48 A A  48 A         A  48 A \n",
      "  50    0.21    1.28    0.52    0.31   V  51 A A  55 A L  49 A L  49 A         L  49 A \n",
      "  51    0.28    1.56    0.56    0.40   K  52 A E  56 A D  50 A E  50 A         A  50 A \n",
      "  52    0.16    1.35    0.52    0.29 * G  53 A G  57 A G  51 A G  51 A         G  51 A \n",
      "  53    0.15    1.05    0.46    0.20 * V  54 A V  58 A V  52 A V  52 A         V  52 A \n",
      "  54    0.27    1.41    0.59    0.30   V  55 A A  59 A L  53 A V  53 A         E  53 A \n",
      "  55    0.26    1.65    0.62    0.39   M  56 A M  60 A M  54 A M  54 A         A  54 A \n",
      "  56    0.16    1.07    0.43    0.23 * E  57 A E  61 A E  55 A E  55 A         E  55 A \n",
      "  57    0.13    0.94    0.40    0.21 * L  58 A L  62 A L  56 A L  56 A         L  56 A \n",
      "  58    0.31    1.68    0.62    0.45   D  59 A L  63 A Q  57 A E  57 A         E  57 A \n",
      "  59    0.23    1.80    0.67    0.49 * D  60 A D  64 A D  58 A D  58 A         D  58 A \n",
      "  60    0.22    1.46    0.52    0.40   C  61 A S  65 A C  59 A C  59 A         C  59 A \n",
      "  61    0.30    2.29    0.79    0.60 * A  62 A A  66 A A  60 A A  60 A         A  60 A \n",
      "  62    0.16    0.49    0.29    0.09   F  63 A F  67 A L  61 A F  61 A         F  61 A \n",
      "  63    0.19    1.32    0.48    0.31 * P  64 A P  68 A P  62 A P  62 A         P  62 A \n",
      "  64    0.09    0.44    0.31    0.08 * L  65 A L  69 A L  63 A L  63 A         L  63 A \n",
      "  65    0.10    0.51    0.26    0.12 * L  66 A L  70 A L  64 A L  64 A         L  64 A \n",
      "  66    0.39    0.39    0.40    0.16   A  67 A R  71 A K  65 A A  65 A         D  65 A \n",
      "  67    0.16    0.26    0.35    0.14   G  68 A N  72 A D  66 A G  66 A         K  66 A \n",
      "  68    0.11    0.38    0.36    0.04   V  69 A I  73 A V  67 A L  67 A         V  67 A \n",
      "  69    0.15    0.28    0.34    0.12   V  70 A T  74 A I  68 A E  68 A         V  68 A \n",
      "  70    0.20    0.80    0.46    0.13   I  71 A I  75 A A  69 A A  69 A         V  69 A \n",
      "  71    0.34    0.76    0.45    0.12 * T  72 A T  76 A T  70 A T  70 A         T  70 A \n",
      "  72    0.29    0.83    0.44    0.18   D  73 A A  77 A D  71 A D  71 A         A  71 A \n",
      "  73    0.25    1.08    0.49    0.23   D  74 A D  78 A K  72 A D  72 A         D  72 A \n",
      "  74    0.36    1.61    0.56    0.42   P  75 A A  79 A E  73 A P  73 A         P  73 A \n",
      "  75    0.33    1.93    0.67    0.45   K  76 A N  80 A E  74 A D  74 A         R  74 A \n",
      "  76    0.18    0.68    0.34    0.16   V  77 A E  81 A I  75 A V  75 A         V  75 A \n",
      "  77    0.11    0.57    0.34    0.12 * A  78 A A  82 A A  76 A A  76 A         A  76 A \n",
      "  78    0.15    1.13    0.38    0.29 * F  79 A F  83 A F  77 A F  77 A         F  77 A \n",
      "  79    0.18    0.80    0.34    0.15   K  80 A D  84 A K  78 A K  78 A         D  78 A \n",
      "  80    0.20    0.91    0.35    0.20   D  81 A G  85 A D  79 A D  79 A         G  79 A \n",
      "  81    0.19    0.60    0.32    0.12   A  82 A A  86 A L  80 A A  80 A         V  80 A \n",
      "  82    0.16    0.30    0.24    0.11   D  83 A N  87 A D  81 A D  81 A         A  81 A \n",
      "  83    0.14    0.43    0.23    0.09   V  84 A A  88 A V  82 A Y  82 A         I  82 A \n",
      "  84    0.10    0.58    0.31    0.17 * A  85 A A  89 A A  83 A A  83 A         A  83 A \n",
      "  85    0.13    0.38    0.34    0.10   L  86 A F  90 A I  84 A L  84 A         I  84 A \n",
      "  86    0.06    0.18    0.27    0.14   L  87 A L  91 A L  85 A L  85 A         M  85 A \n",
      "  87    0.11    0.33    0.30    0.08   V  88 A V  92 A V  86 A V  86 A         C  86 A \n",
      "  88    0.26    0.62    0.34    0.08 * G  89 A G  93 A G  87 A G  87 A         G  87 A \n",
      "  89    0.29    0.97    0.54    0.20   A  90 A A  94 A S  88 A A  88 A         A  88 A \n",
      "  90    0.50    2.12    1.16    0.44   R  91 A K  95 A M  89 A A  89 A         F  89 A \n",
      "  91    0.44    3.10    1.46    0.86 * P  92 A P  96 A P  90 A P  90 A         P  90 A \n",
      "  92    0.64    3.77    1.69    0.94   R  93 A -       R  91 A                 R  91 A \n",
      "  93    0.89    6.31    2.81    1.11   S  94 A -       R  92 A -               K  92 A \n",
      "  94    1.05    7.61    3.81    0.00   -       -       D  93 A -               A  93 A \n",
      "  95    1.13    8.52    4.26    0.00   -       -       G  94 A -               G  94 A \n",
      "  96    1.06    8.00    3.60    1.66   M  97 A -       M  95 A -               M  95 A \n",
      "  97    1.13    7.77    3.00    1.89   E  98 A E 102 A E  96 A -               E  96 A \n",
      "  98    1.19    6.49    2.50    1.60   R  99 A R 103 A R  97 A -               R  97 A \n",
      "  99    0.95    6.29    2.47    1.59   K 100 A A 104 A K  98 A -               K  98 A \n",
      " 100    0.77    4.91    1.93    1.24   D 101 A D 105 A D  99 A -               D  99 A \n",
      " 101    0.44    1.81    0.80    0.41   L 102 A L 106 A L 100 A -               L 100 A \n",
      " 102    0.37    1.61    1.03    0.62 * L 103 A L 107 A L 101 A L 101 A         L 101 A \n",
      " 103    0.30    1.65    1.02    0.65   S 104 A A 108 A K 102 A Q 102 A         E 102 A \n",
      " 104    0.35    1.57    1.04    0.41   A 105 A N 109 A A 103 A V 103 A         M 103 A \n",
      " 105    0.18    1.10    0.76    0.26 * N 106 A N 110 A N 104 A N 104 A         N 104 A \n",
      " 106    0.33    1.75    1.03    0.37   A 107 A G 111 A V 105 A G 105 A         A 105 A \n",
      " 107    0.23    1.39    0.80    0.29   E 108 A K 112 A K 106 A K 106 A         R 106 A \n",
      " 108    0.10    0.52    0.34    0.09 * I 109 A I 113 A I 107 A I 107 A         I 107 A \n",
      " 109    0.11    0.67    0.37    0.15 * F 110 A F 114 A F 108 A F 108 A         F 108 A \n",
      " 110    0.31    0.94    0.48    0.19   T 111 A G 115 A K 109 A T 109 A         K 109 A \n",
      " 111    0.28    0.74    0.41    0.13   V 112 A P 116 A C 110 A E 110 A         E 110 A \n",
      " 112    0.08    0.41    0.26    0.05 * Q 113 A Q 117 A Q 111 A Q 111 A         Q 111 A \n",
      " 113    0.07    0.46    0.27    0.08 * G 114 A G 118 A G 112 A G 112 A         G 112 A \n",
      " 114    0.17    0.64    0.34    0.11   A 115 A K 119 A A 113 A R 113 A         E 113 A \n",
      " 115    0.10    0.60    0.32    0.09 * A 116 A A 120 A A 114 A A 114 A         A 114 A \n",
      " 116    0.10    0.44    0.27    0.09   L 117 A I 121 A L 115 A L 115 A         I 115 A \n",
      " 117    0.19    0.77    0.35    0.14   N 118 A N 122 A D 116 A A 116 A         A 116 A \n",
      " 118    0.20    0.83    0.41    0.15   E 119 A D 123 A K 117 A E 117 A         A 117 A \n",
      " 119    0.25    0.92    0.38    0.18   V 120 A N 124 A Y 118 A V 118 A         V 118 A \n",
      " 120    0.13    0.74    0.33    0.14 * A 121 A A 125 A A 119 A A 119 A         A 119 A \n",
      " 121    0.24    1.31    0.56    0.30   S 122 A A 126 A K 120 A K 120 A         A 120 A \n",
      " 122    0.27    1.58    0.78    0.54   R 123 A D 127 A K 121 A K 121 A         S 121 A \n",
      " 123    0.19    0.65    0.66    0.47   D 124 A D 128 A S 122 A D 122 A         D 122 A \n",
      " 124    0.12    0.30    0.23    0.07   V 125 A I 129 A V 123 A V 123 A         C 123 A \n",
      " 125    0.08    0.12    0.22    0.10   K 126 A R 130 A K 124 A K 124 A         R 124 A \n",
      " 126    0.09    0.43    0.24    0.11 * V 127 A V 131 A V 125 A V 125 A         V 125 A \n",
      " 127    0.13    0.45    0.25    0.16   L 128 A L 132 A I 126 A L 126 A         V 126 A \n",
      " 128    0.06    0.29    0.18    0.05 * V 129 A V 133 A V 127 A V 127 A         V 127 A \n",
      " 129    0.08    0.41    0.26    0.07 * V 130 A V 134 A V 128 A V 128 A         V 128 A \n",
      " 130    0.06    0.38    0.28    0.15 * G 131 A G 135 A G 129 A G 129 A         G 129 A \n",
      " 131    0.09    0.42    0.28    0.14 * N 132 A N 136 A N 130 A N 130 A         N 130 A \n",
      " 132    0.25    0.57    0.25    0.14 * P 133 A P 137 A P 131 A P 131 A         P 131 A \n",
      " 133    0.19    0.37    0.23    0.10 * A 134 A A 138 A A 132 A A 132 A         A 132 A \n",
      " 134    0.09    0.34    0.20    0.07 * N 135 A N 139 A N 133 A N 133 A         N 133 A \n",
      " 135    0.08    0.38    0.19    0.06 * T 136 A T 140 A T 134 A T 134 A         T 134 A \n",
      " 136    0.09    0.34    0.21    0.08 * N 137 A N 141 A N 135 A N 135 A         N 135 A \n",
      " 137    0.21    0.48    0.27    0.09   A 138 A A 142 A C 136 A A 136 A         A 136 A \n",
      " 138    0.11    0.46    0.26    0.12   Y 139 A L 143 A L 137 A L 137 A         L 137 A \n",
      " 139    0.20    0.61    0.29    0.14   I 140 A I 144 A T 138 A I 138 A         I 138 A \n",
      " 140    0.09    0.30    0.27    0.08   A 141 A A 145 A A 139 A A 139 A         L 139 A \n",
      " 141    0.15    0.28    0.28    0.08   M 142 A S 146 A S 140 A Y 140 A         L 140 A \n",
      " 142    0.11    0.42    0.31    0.04   K 143 A A 147 A K 141 A K 141 A         K 141 A \n",
      " 143    0.11    0.34    0.44    0.16   S 144 A A 148 A S 142 A N 142 A         S 142 A \n",
      " 144    0.11    0.53    0.54    0.24 * A 145 A A 149 A A 143 A A 143 A         A 143 A \n",
      " 145    0.00    0.00    0.00    0.00   -       -       -       -               Q 144 A \n",
      " 146    0.26    0.62    0.87    0.63   P 146 A P 150 A P 144 A P 144 A         G 145 A \n",
      " 147    0.15    0.13    0.37    0.22   D 147 A D 151 A S 145 A G 145 A         K 146 A \n",
      " 148    0.13    0.28    0.52    0.33   L 148 A V 152 A I 146 A L 146 A         L 147 A \n",
      " 149    0.13    0.38    0.43    0.12   P 149 A P 153 A P 147 A N 147 A         N 148 A \n",
      " 150    0.17    0.56    0.54    0.12   K 150 A A 154 A K 148 A P 148 A         P 149 A \n",
      " 151    0.19    0.56    0.51    0.15   K 151 A S 155 A E 149 A R 149 A         R 150 A \n",
      " 152    0.13    0.37    0.31    0.07   N 152 A R 156 A N 150 A N 150 A         H 151 A \n",
      " 153    0.11    0.44    0.27    0.11   F 153 A F 157 A F 151 A F 151 A         V 152 A \n",
      " 154    0.14    0.48    0.25    0.10   T 154 A N 158 A S 152 A T 152 A         T 153 A \n",
      " 155    0.22    0.53    0.21    0.13   A 155 A A 159 A C 153 A A 153 A         A 154 A \n",
      " 156    0.10    0.10    0.17    0.06   M 156 A M 160 A L 154 A M 154 A         M 155 A \n",
      " 157    0.13    0.13    0.22    0.10   L 157 A M 161 A T 155 A T 155 A         T 156 A \n",
      " 158    0.09    0.23    0.19    0.05 * R 158 A R 162 A R 156 A R 156 A         R 157 A \n",
      " 159    0.06    0.28    0.24    0.07 * L 159 A L 163 A L 157 A L 157 A         L 158 A \n",
      " 160    0.06    0.28    0.24    0.08 * D 160 A D 164 A D 158 A D 158 A         D 159 A \n",
      " 161    0.06    0.24    0.18    0.04 * H 161 A H 165 A H 159 A H 159 A         H 160 A \n",
      " 162    0.05    0.05    0.15    0.09 * N 162 A N 166 A N 160 A N 160 A         N 161 A \n",
      " 163    0.07    0.36    0.22    0.09 * R 163 A R 167 A R 161 A R 161 A         R 162 A \n",
      " 164    0.07    0.29    0.16    0.09 * A 164 A A 168 A A 162 A A 162 A         A 163 A \n",
      " 165    0.14    0.25    0.17    0.08   L 165 A I 169 A K 163 A K 163 A         L 164 A \n",
      " 166    0.13    0.45    0.31    0.10   S 166 A S 170 A A 164 A A 164 A         S 165 A \n",
      " 167    0.10    0.37    0.39    0.06   Q 167 A Q 171 A Q 165 A Q 165 A         L 166 A \n",
      " 168    0.11    0.26    0.29    0.05   L 168 A L 172 A I 166 A L 166 A         L 167 A \n",
      " 169    0.10    0.55    0.42    0.09 * A 169 A A 173 A A 167 A A 167 A         A 168 A \n",
      " 170    0.22    0.76    0.50    0.20   A 170 A T 174 A L 168 A K 168 A         R 169 A \n",
      " 171    0.10    0.55    0.49    0.15 * K 171 A K 175 A K 169 A K 169 A         K 170 A \n",
      " 172    0.24    1.11    0.64    0.28   S 172 A L 176 A L 170 A T 170 A         A 171 A \n",
      " 173    0.18    1.47    0.76    0.30 * G 173 A G 177 A G 171 A G 171 A         G 172 A \n",
      " 174    0.34    1.46    0.79    0.36   K 174 A R 178 A V 172 A T 172 A         V 173 A \n",
      " 175    0.30    1.41    0.76    0.26   P 175 A G 179 A T 173 A G 173 A         P 174 A \n",
      " 176    0.25    1.20    0.64    0.16   V 176 A S 180 A S 174 A V 174 A         V 175 A \n",
      " 177    0.19    0.82    0.61    0.21   A 177 A A 181 A D 175 A D 175 A         S 176 A \n",
      " 178    0.28    1.27    0.58    0.31   S 178 A E 182 A D 176 A R 176 A         Q 177 A \n",
      " 179    0.24    1.25    0.69    0.31   I 179 A F 183 A V 177 A I 177 A         V 178 A \n",
      " 180    0.26    1.41    1.02    0.44   E 180 A N 184 A K 178 A R 178 A         R 179 A \n",
      " 181    0.31    0.95    0.88    0.35   K 181 A N 185 A N 179 A R 179 A         N 180 A \n",
      " 182    0.17    0.79    0.79    0.25   L 182 A I 186 A V 180 A M 180 A         V 181 A \n",
      " 183    0.13    0.29    0.49    0.23   A 183 A V 187 A I 181 A T 181 A         I 182 A \n",
      " 184    0.11    0.42    0.43    0.23   V 184 A V 188 A I 182 A V 182 A         I 183 A \n",
      " 185    0.04    0.14    0.18    0.08 * W 185 A W 189 A W 183 A W 183 A         W 184 A \n",
      " 186    0.08    0.55    0.35    0.09 * G 186 A G 190 A G 184 A G 184 A         G 185 A \n",
      " 187    0.23    0.60    0.59    0.39 * N 187 A N 191 A N 185 A N 185 A         N 186 A \n",
      " 188    0.16    0.59    0.54    0.24 * H 188 A H 192 A H 186 A H 186 A         H 187 A \n",
      " 189    0.21    0.50    0.52    0.13 * S 189 A S 193 A S 187 A S 187 A         S 188 A \n",
      " 190    0.13    0.55    0.61    0.24   P 190 A A 194 A S 188 A S 188 A         S 189 A \n",
      " 191    0.11    0.42    0.61    0.28   T 191 A T 195 A T 189 A I 189 A         T 190 A \n",
      " 192    0.12    0.17    0.38    0.16   M 192 A Q 196 A Q 190 A M 190 A         Q 191 A \n",
      " 193    0.13    0.33    0.37    0.15   Y 193 A F 197 A Y 191 A F 191 A         V 192 A \n",
      " 194    0.04    0.15    0.40    0.26 * P 194 A P 198 A P 192 A P 192 A         P 193 A \n",
      " 195    0.07    0.28    0.37    0.20 * D 195 A D 199 A D 193 A D 193 A         D 194 A \n",
      " 196    0.14    0.34    0.57    0.20   F 196 A I 200 A V 194 A L 194 A         T 195 A \n",
      " 197    0.17    0.32    0.47    0.24   R 197 A T 201 A N 195 A F 195 A         D 196 A \n",
      " 198    0.17    0.43    0.51    0.18   F 198 A Y 202 A H 196 A H 196 A         S 197 A \n",
      " 199    0.30    0.35    0.48    0.18 * A 199 A A 203 A A 197 A A 197 A         A 198 A \n",
      " 200    0.42    0.62    0.56    0.16   T 200 A T 204 A K 198 A E 198 A         V 199 A \n",
      " 201    0.46    0.84    0.68    0.17   A 201 A V 205 A V 199 A V 199 A         I 200 A \n",
      " 202    0.72    1.84    1.35    0.45   E 202 A G 206 A K 200 A D 200 A         G 201 A \n",
      " 203    0.00    0.00    0.00    0.00   -       -       L 201 A -               -       \n",
      " 204    0.00    0.00    0.00    0.00   -       -       Q 202 A -               -       \n",
      " 205    0.00    0.00    0.00    0.00   -       -       A 203 A -               -       \n",
      " 206    0.00    0.00    0.00    0.00   -       -       K 204 A -               -       \n",
      " 207    0.46    1.79    1.42    0.36   G 203 A G 207 A E 205 A G 205 A         T 202 A \n",
      " 208    0.50    0.75    0.77    0.20   E 204 A E 208 A V 206 A R 206 A         T 203 A \n",
      " 209    0.23    0.82    0.64    0.15   S 205 A K 209 A G 207 A P 207 A         P 204 A \n",
      " 210    0.19    0.88    0.63    0.17   L 206 A V 210 A V 208 A A 208 A         A 205 A \n",
      " 211    0.23    0.70    0.80    0.38   L 207 A T 211 A Y 209 A L 209 A         R 206 A \n",
      " 212    0.13    0.52    0.80    0.48   K 208 A D 212 A E 210 A E 210 A         E 207 A \n",
      " 213    0.34    0.72    1.08    0.45   L 209 A L 213 A A 211 A L 211 A         A 208 A \n",
      " 214    0.29    0.80    1.26    0.53   I 210 A V 214 A V 212 A V 212 A         I 209 A \n",
      " 215    0.53    1.12    2.54    1.29   N 211 A -       K 213 A -               K 210 A \n",
      " 216    0.23    1.07    0.70    0.29   D 212 A D 215 A D 214 A D 214 A         -       \n",
      " 217    0.28    0.65    0.34    0.10   D 213 A H 216 A D 215 A M 215 A         -       \n",
      " 218    0.24    0.42    0.51    0.19   V 214 A D 217 A S 216 A E 216 A         -       \n",
      " 219    0.21    0.91    0.63    0.21   W 215 A W 218 A W 217 A W 217 A         -       \n",
      " 220    0.33    1.11    0.60    0.21   N 216 A Y 219 A L 218 A Y 218 A         -       \n",
      " 221    0.39    1.76    0.82    0.39   R 217 A V 220 A K 219 A E 219 A         -       \n",
      " 222    0.38    1.84    0.94    0.32   D 218 A E 221 A G 220 A K 220 A         -       \n",
      " 223    0.32    1.00    0.58    0.35   T 219 A E 222 A E 221 A V 221 A         D 217 A \n",
      " 224    0.18    0.42    0.34    0.20 * F 220 A F 223 A F 222 A F 222 A         F 218 A \n",
      " 225    0.16    0.49    0.36    0.20   I 221 A I 224 A I 223 A I 223 A         V 219 A \n",
      " 226    0.19    0.54    0.51    0.25   P 222 A P 225 A T 224 A P 224 A         Q 220 A \n",
      " 227    0.17    0.78    0.42    0.32   T 223 A R 226 A T 225 A T 225 A         V 221 A \n",
      " 228    0.13    0.87    0.43    0.31 * V 224 A V 227 A V 226 A V 226 A         V 222 A \n",
      " 229    0.25    1.09    0.63    0.31   G 225 A A 228 A Q 227 A A 227 A         R 223 A \n",
      " 230    0.26    1.42    0.71    0.47   K 226 A N 229 A Q 228 A Q 228 A         G 224 A \n",
      " 231    0.19    1.39    0.53    0.45 * R 227 A R 230 A R 229 A R 229 A         R 225 A \n",
      " 232    0.21    1.78    0.68    0.50 * G 228 A G 231 A G 230 A G 230 A         G 226 A \n",
      " 233    0.31    2.45    0.92    0.66 * A 229 A A 232 A A 231 A A 231 A         A 227 A \n",
      " 234    0.34    2.35    0.85    0.69   A 230 A E 233 A A 232 A A 232 A         E 228 A \n",
      " 235    0.29    1.96    0.75    0.54   I 231 A I 234 A V 233 A I 233 A         I 229 A \n",
      " 236    0.54    2.49    0.91    0.64 * I 232 A I 235 A I 234 A I 234 A         I 230 A \n",
      " 237    0.47    3.24    1.09    0.82   E 233 A E 236 A K 235 A Q 235 A         Q 231 A \n",
      " 238    0.42    3.10    1.09    0.74   A 234 A V 237 A A 236 A A 236 A         L 232 A \n",
      " 239    0.46    2.81    1.05    0.69 * R 235 A R 238 A R 237 A R 237 A         R 233 A \n",
      " 240    0.47    3.00    1.05    0.79   G 236 A G 239 A K 238 A G 238 A         G 234 A \n",
      " 241    0.35    2.36    0.81    0.64   L 237 A K 240 A L 239 A A 239 A         L 235 A \n",
      " 242    0.26    2.03    0.80    0.51 * S 238 A S 241 A S 240 A S 240 A         S 236 A \n",
      " 243    0.26    1.27    0.55    0.31 * S 239 A S 242 A S 241 A S 241 A         S 237 A \n",
      " 244    0.21    0.52    0.31    0.07 * A 240 A A 243 A A 242 A A 242 A         A 238 A \n",
      " 245    0.18    0.47    0.27    0.10   A 241 A A 244 A M 243 A A 243 A         M 239 A \n",
      " 246    0.09    0.46    0.27    0.05 * S 242 A S 245 A S 244 A S 244 A         S 240 A \n",
      " 247    0.06    0.20    0.23    0.05 * A 243 A A 246 A A 245 A A 245 A         A 241 A \n",
      " 248    0.05    0.11    0.18    0.06 * A 244 A A 247 A A 246 A A 246 A         A 242 A \n",
      " 249    0.11    0.12    0.18    0.08   N 245 A S 248 A K 247 A N 247 A         K 243 A \n",
      " 250    0.07    0.17    0.18    0.06   A 246 A S 249 A A 248 A A 248 A         A 244 A \n",
      " 251    0.18    0.42    0.23    0.06   A 247 A A 250 A I 249 A A 249 A         A 245 A \n",
      " 252    0.25    0.57    0.33    0.08   I 248 A I 251 A C 250 A I 250 A         V 246 A \n",
      " 253    0.09    0.35    0.26    0.06   D 249 A D 252 A D 251 A E 251 A         D 247 A \n",
      " 254    0.07    0.35    0.23    0.03 * H 250 A H 253 A H 252 A H 252 A         H 248 A \n",
      " 255    0.12    0.48    0.29    0.09   V 251 A M 254 A V 253 A I 253 A         V 249 A \n",
      " 256    0.10    0.31    0.29    0.09   R 252 A R 255 A R 254 A R 254 A         H 250 A \n",
      " 257    0.06    0.24    0.26    0.08 * D 253 A D 256 A D 255 A D 255 A         D 251 A \n",
      " 258    0.27    0.83    0.41    0.16   W 254 A W 257 A I 256 A W 256 A         W 252 A \n",
      " 259    0.29    1.02    0.42    0.23   V 255 A V 258 A W 257 A A 257 A         I 253 A \n",
      " 260    0.18    0.40    0.30    0.07   L 256 A Q 259 A F 258 A L 258 A         H 254 A \n",
      " 261    0.09    0.52    0.40    0.19 * G 257 A G 260 A G 259 A G 259 A         G 255 A \n",
      " 262    0.10    0.44    0.35    0.05 * T 258 A T 261 A T 260 A T 260 A         T 256 A \n",
      " 263    0.18    0.69    1.21    0.54   N 259 A E 262 A P 261 A P 261 A         P 257 A \n",
      " 264    0.33    1.09    1.11    0.63   G 260 A -       E 262 A E 262 A         E 258 A \n",
      " 265    0.17    0.42    0.36    0.07   -       -       G 263 A G 263 A         G 259 A \n",
      " 266    0.31    1.03    0.91    0.39   K 261 A A 263 A E 264 A D 264 A         V 260 A \n",
      " 267    0.23    0.76    0.55    0.24   W 262 A W 264 A F 265 A W 265 A         Y 261 A \n",
      " 268    0.11    0.40    0.42    0.21   V 263 A S 265 A V 266 A V 266 A         V 262 A \n",
      " 269    0.06    0.06    0.21    0.11   T 264 A S 266 A S 267 A S 267 A         S 263 A \n",
      " 270    0.11    0.31    0.31    0.13   M 265 A A 267 A M 268 A M 268 A         M 264 A \n",
      " 271    0.08    0.19    0.27    0.07   G 266 A A 268 A G 269 A A 269 A         G 265 A \n",
      " 272    0.31    0.26    0.24    0.07   I 267 A I 269 A I 270 A V 270 A         V 266 A \n",
      " 273    0.43    0.39    0.27    0.10   P 268 A P 270 A I 271 A P 271 A         Y 267 A \n",
      " 274    0.26    0.45    0.31    0.04 * S 269 A S 271 A S 272 A S 272 A         S 268 A \n",
      " 275    0.39    1.08    0.94    0.40   D 270 A T 272 A D 273 A Q 273 A         D 269 A \n",
      " 276    0.33    2.46    1.80    0.44   G 271 A G 273 A G 274 A G 274 A         E 270 A \n",
      " 277    0.30    0.87    1.04    0.49   S 272 A A 274 A N 275 A E 275 A         N 271 A \n",
      " 278    0.40    1.60    0.80    0.00   -       -       S 276 A -               P 272 A \n",
      " 279    0.15    0.79    0.51    0.24 * Y 273 A Y 275 A Y 277 A Y 277 A         Y 273 A \n",
      " 280    0.21    1.44    0.97    0.53 * G 274 A G 276 A G 278 A G 278 A         G 274 A \n",
      " 281    0.18    1.04    0.49    0.26   I 275 A I 277 A V 279 A I 279 A         V 275 A \n",
      " 282    0.15    1.09    0.47    0.31 * P 276 A P 278 A P 280 A P 280 A         P 276 A \n",
      " 283    0.23    1.11    0.51    0.29   E 277 A E 279 A D 281 A E 281 A         S 277 A \n",
      " 284    0.29    1.17    0.88    0.45   D 278 A G 280 A D 282 A G 282 A         G 278 A \n",
      " 285    0.18    0.89    0.59    0.18   I 279 A I 281 A L 283 A I 283 A         L 279 A \n",
      " 286    0.17    0.62    0.34    0.12   I 280 A F 282 A L 284 A V 284 A         I 280 A \n",
      " 287    0.13    0.48    0.31    0.17   Y 281 A V 283 A Y 285 A Y 285 A         F 281 A \n",
      " 288    0.12    0.44    0.30    0.18   G 282 A G 284 A S 286 A S 286 A         S 282 A \n",
      " 289    0.17    0.38    0.22    0.08   V 283 A L 285 A F 287 A F 287 A         F 283 A \n",
      " 290    0.05    0.25    0.36    0.22 * P 284 A P 286 A P 288 A P 288 A         P 284 A \n",
      " 291    0.15    0.52    0.39    0.09   V 285 A T 287 A V 289 A V 289 A         C 285 A \n",
      " 292    0.21    0.67    0.48    0.24   I 286 A V 288 A T 290 A T 290 A         T 286 A \n",
      " 293    0.24    0.70    0.46    0.14   C 287 A S 289 A I 291 A A 291 A         C 287 A \n",
      " 294    0.18    0.73    0.69    0.11   E 288 A R 290 A K 292 A K 292 A         H 288 A \n",
      " 295    0.17    0.74    0.91    0.43   N 289 A N 291 A D 293 A D 293 A         A 289 A \n",
      " 296    0.47    0.54    0.48    0.22   G 290 A G 292 A K 294 A G 294 A         G 290 A \n",
      " 297    0.20    0.64    0.58    0.14   E 291 A E 293 A T 295 A A 295 A         E 291 A \n",
      " 298    0.14    0.63    0.62    0.13   Y 292 A W 294 A W 296 A Y 296 A         W 292 A \n",
      " 299    0.15    0.59    0.55    0.13   K 293 A E 295 A K 297 A R 297 A         T 293 A \n",
      " 300    0.12    0.26    0.42    0.19   R 294 A I 296 A I 298 A V 298 A         V 294 A \n",
      " 301    0.07    0.30    0.33    0.14 * V 295 A V 297 A V 299 A V 299 A         V 295 A \n",
      " 302    0.16    0.85    0.80    0.40   E 296 A E 298 A E 300 A E 300 A         S 296 A \n",
      " 303    0.18    1.45    1.30    0.49 * G 297 A G 299 A G 301 A G 301 A         G 297 A \n",
      " 304    0.17    0.86    1.15    0.64   L 298 A L 300 A L 302 A L 302 A         K 298 A \n",
      " 305    0.41    1.69    1.56    0.96   E 299 A E 301 A P 303 A E 303 A         L 299 A \n",
      " 306    0.21    1.08    0.62    0.17   I 300 A I 302 A I 304 A I 304 A         -       \n",
      " 307    0.23    1.04    0.54    0.19   D 301 A S 303 A N 305 A N 305 A         -       \n",
      " 308    0.20    0.73    0.59    0.12   A 302 A D 304 A D 306 A E 306 A         -       \n",
      " 309    0.11    0.24    0.53    0.26   F 303 A F 305 A F 307 A F 307 A         -       \n",
      " 310    0.15    0.21    0.43    0.23   S 304 A Q 306 A S 308 A A 308 A         -       \n",
      " 311    0.24    0.93    0.98    0.65   R 305 A R 307 A R 309 A R 309 A         K 305 A \n",
      " 312    0.23    0.69    1.00    0.73   E 306 A A 308 A E 310 A K 310 A         Q 306 A \n",
      " 313    0.19    0.67    0.93    0.68   K 307 A R 309 A K 311 A R 311 A         R 307 A \n",
      " 314    0.17    0.45    0.56    0.29   M 308 A I 310 A M 312 A M 312 A         L 308 A \n",
      " 315    0.16    0.68    0.67    0.30   D 309 A D 311 A D 313 A E 313 A         A 309 A \n",
      " 316    0.20    0.63    0.58    0.26   G 310 A A 312 A L 314 A I 314 A         S 310 A \n",
      " 317    0.10    0.39    0.53    0.25   T 311 A N 313 A T 315 A T 315 A         T 311 A \n",
      " 318    0.10    0.17    0.53    0.31   L 312 A A 314 A A 316 A A 316 A         I 312 A \n",
      " 319    0.17    0.56    0.63    0.24   A 313 A Q 315 A K 317 A Q 317 A         A 313 A \n",
      " 320    0.28    0.65    0.63    0.20 * E 314 A E 316 A E 318 A E 318 A         E 314 A \n",
      " 321    0.08    0.44    0.40    0.15 * L 315 A L 317 A L 319 A L 319 A         L 315 A \n",
      " 322    0.16    0.36    0.47    0.17   L 316 A Q 318 A A 320 A L 320 A         Q 316 A \n",
      " 323    0.12    0.44    0.62    0.23   E 317 A A 319 A E 321 A D 321 A         E 317 A \n",
      " 324    0.10    0.58    0.57    0.24 * E 318 A E 320 A E 322 A E 322 A         E 318 A \n",
      " 325    0.17    0.58    0.49    0.19   R 319 A R 321 A K 323 A M 323 A         R 319 A \n",
      " 326    0.17    0.80    0.60    0.31   D 320 A E 322 A E 324 A E 324 A         A 320 A \n",
      " 327    0.22    0.77    0.68    0.38   G 321 A A 323 A T 325 A Q 325 A         Q 321 A \n",
      " 328    0.29    1.27    0.75    0.36   V 322 A V 324 A A 326 A V 326 A         A 322 A \n",
      " 329    0.42    1.09    0.93    0.35   A 323 A R 325 A F 327 A K 327 A         G 323 A \n",
      " 330    0.58    3.11    2.70    0.98   H 324 A D 326 A E 328 A A 328 A         L 324 A \n",
      " 331    0.74    4.53    3.13    0.40   L 325 A L 327 A F 329 A L 329 A         -       \n",
      " 332    0.92    4.05    3.05    0.50   -       L 328 A L 330 A G 330 A         -       \n",
      " 333    1.14    5.84    3.36    0.97   -       L 329 A S 331 A L 331 A         -       \n",
      " 334    1.61   11.86    5.70    1.93   -       E 330 A S 332 A I 332 A         -       \n",
      " 335    0.00    0.00    0.00    0.00   -       -       A 333 A -               -       \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    5\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      327      327        1.1664        1.1664        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      327      327        1.1664        1.1664        0.0000\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Current alignment total score:        107.2\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -       -       -       D  23 A -       \n",
      "   2    0.00    0.00    0.00    0.00   -       -       -       -       C  24 A -       \n",
      "   3    0.00    0.00    0.00    0.00   -       -       -       -       F  25 A -       \n",
      "   4    0.00    0.00    0.00    0.00   -       -       -       -       G  26 A -       \n",
      "   5    0.00    0.00    0.00    0.00   -       -       -       -       V  27 A -       \n",
      "   6    0.00    0.00    0.00    0.00   -       -       -       -       F  28 A -       \n",
      "   7    0.00    0.00    0.00    0.00   -       -       -       -       C  29 A -       \n",
      "   8    0.00    0.00    0.00    0.00   -       -       -       -       T  30 A -       \n",
      "   9    0.27    0.49    0.24    0.00   -       -       -       M   0 A W  43 A -       \n",
      "  10    0.34    1.85    1.22    0.64   K   3 A -       S   1 A K   1 A K  44 A M   1 A \n",
      "  11    0.35    2.10    1.13    0.58   P   4 A S   8 A E   2 A A   2 A K  45 A S   2 A \n",
      "  12    0.31    1.19    0.60    0.27   A   5 A T   9 A P   3 A P   3 A L  46 A A   3 A \n",
      "  13    0.16    0.52    0.29    0.13   K   6 A K  10 A I   4 A V   4 A V  47 A V   4 A \n",
      "  14    0.16    0.35    0.25    0.11   R   7 A K  11 A R   5 A R   5 A N  48 A K   5 A \n",
      "  15    0.10    0.30    0.21    0.03   V   8 A V  12 A V   6 A V   6 A I  49 A V   6 A \n",
      "  16    0.10    0.35    0.27    0.07   A   9 A T  13 A L   7 A A   7 A A  50 A A   7 A \n",
      "  17    0.07    0.39    0.32    0.05 * V  10 A V  14 A V   8 A V   8 A V  51 A V   8 A \n",
      "  18    0.11    0.30    0.24    0.11   T  11 A T  15 A T   9 A T   9 A S  52 A T   9 A \n",
      "  19    0.17    0.40    0.34    0.14 * G  12 A G  16 A G  10 A G  10 A G  53 A G  10 A \n",
      "  20    0.08    0.45    0.38    0.07 * A  13 A A  17 A A  11 A A  11 A A  54 A A  11 A \n",
      "  21    0.10    0.58    0.37    0.07 * A  14 A A  18 A A  12 A A  12 A A  55 A A  12 A \n",
      "  22    0.08    0.61    0.29    0.13 * G  15 A G  19 A G  13 A G  13 A G  56 A G  13 A \n",
      "  23    0.23    0.71    0.32    0.15   Q  16 A Q  20 A Q  14 A Q  14 A M  57 A Q  14 A \n",
      "  24    0.12    0.79    0.28    0.19 * I  17 A I  21 A I  15 A I  15 A I  58 A I  15 A \n",
      "  25    0.16    0.65    0.29    0.15   A  18 A S  22 A A  16 A G  16 A S  59 A G  16 A \n",
      "  26    0.20    0.56    0.23    0.13   Y  19 A Y  23 A Y  17 A Y  17 A N  60 A Y  17 A \n",
      "  27    0.27    1.14    0.38    0.28   S  20 A S  24 A S  18 A S  18 A H  61 A A  18 A \n",
      "  28    0.10    0.66    0.34    0.12 * L  21 A L  25 A L  19 A L  19 A L  62 A L  19 A \n",
      "  29    0.10    0.37    0.27    0.11   L  22 A L  26 A L  20 A L  20 A L  63 A V  20 A \n",
      "  30    0.13    0.37    0.39    0.12   F  23 A W  27 A Y  21 A F  21 A F  64 A P  21 A \n",
      "  31    0.13    0.27    0.42    0.12   R  24 A R  28 A S  22 A R  22 A K  65 A L  22 A \n",
      "  32    0.13    0.43    0.35    0.13   I  25 A I  29 A I  23 A I  23 A L  66 A I  23 A \n",
      "  33    0.08    0.33    0.27    0.07   A  26 A A  30 A G  24 A A  24 A A  67 A A  24 A \n",
      "  34    0.17    0.63    0.36    0.13   N  27 A N  31 A N  25 A A  25 A S  68 A R  25 A \n",
      "  35    0.08    0.57    0.43    0.11 * G  28 A G  32 A G  26 A G  26 A G  69 A G  26 A \n",
      "  36    0.25    1.53    0.83    0.24   D  29 A E  33 A S  27 A E  27 A E  70 A A  27 A \n",
      "  37    0.24    1.55    0.84    0.25   L  30 A V  34 A V  28 A M  28 A V  71 A L  28 A \n",
      "  38    0.19    1.12    0.60    0.25   L  31 A F  35 A F  29 A L  29 A F  72 A L  29 A \n",
      "  39    0.12    1.02    0.62    0.18 * G  32 A G  36 A G  30 A G  30 A G  73 A G  30 A \n",
      "  40    0.23    1.01    0.54    0.20   K  33 A T  37 A K  31 A K  31 A Q  74 A P  31 A \n",
      "  41    0.13    0.77    0.61    0.12   D  34 A D  38 A D  32 A D  32 A D  75 A T  32 A \n",
      "  42    0.20    0.38    0.38    0.14   Q  35 A T  39 A Q  33 A Q  33 A Q  76 A T  33 A \n",
      "  43    0.06    0.36    0.41    0.11 * P  36 A P  40 A P  34 A P  34 A P  77 A P  34 A \n",
      "  44    0.11    0.41    0.32    0.06   V  37 A V  41 A I  35 A V  35 A I  78 A V  35 A \n",
      "  45    0.16    0.29    0.40    0.20   I  38 A E  42 A I  36 A I  36 A A  79 A E  36 A \n",
      "  46    0.05    0.22    0.26    0.12 * L  39 A L  43 A L  37 A L  37 A L  80 A L  37 A \n",
      "  47    0.12    0.23    0.24    0.05   Q  40 A K  44 A V  38 A Q  38 A K  81 A R  38 A \n",
      "  48    0.04    0.10    0.36    0.16 * L  41 A L  45 A L  39 A L  39 A L  82 A L  39 A \n",
      "  49    0.04    0.13    0.32    0.13 * L  42 A L  46 A L  40 A L  40 A L  83 A L  40 A \n",
      "  50    0.17    0.50    0.43    0.13   D  43 A E  47 A D  41 A E  41 A G  84 A D  41 A \n",
      "  51    0.39    0.97    0.58    0.23   L  44 A I  48 A I  42 A I  42 A S  85 A I  42 A \n",
      "  52    0.37    1.13    0.63    0.32   P  45 A P  49 A T  43 A P  43 A E  86 A E  43 A \n",
      "  53    0.36    1.98    0.85    0.55   Q  46 A Q  50 A P  44 A Q  44 A R  87 A P  44 A \n",
      "  54    0.21    0.94    0.57    0.24   A  47 A A  51 A M  45 A A  45 A S  88 A A  45 A \n",
      "  55    0.24    1.00    0.58    0.28   Q  48 A L  52 A M  46 A M  46 A F  89 A L  46 A \n",
      "  56    0.23    0.97    0.61    0.26   A  49 A G  53 A G  47 A K  47 A Q  90 A K  47 A \n",
      "  57    0.15    0.80    0.52    0.23   A  50 A G  54 A V  48 A A  48 A A  91 A A  48 A \n",
      "  58    0.12    0.64    0.50    0.33   V  51 A A  55 A L  49 A L  49 A L  92 A L  49 A \n",
      "  59    0.15    0.66    0.55    0.39   K  52 A E  56 A D  50 A E  50 A E  93 A A  50 A \n",
      "  60    0.07    0.54    0.49    0.31 * G  53 A G  57 A G  51 A G  51 A G  94 A G  51 A \n",
      "  61    0.06    0.31    0.42    0.22 * V  54 A V  58 A V  52 A V  52 A V  95 A V  52 A \n",
      "  62    0.21    0.62    0.56    0.32   V  55 A A  59 A L  53 A V  53 A A  96 A E  53 A \n",
      "  63    0.15    0.75    0.60    0.41   M  56 A M  60 A M  54 A M  54 A M  97 A A  54 A \n",
      "  64    0.09    0.51    0.42    0.25 * E  57 A E  61 A E  55 A E  55 A E  98 A E  55 A \n",
      "  65    0.11    0.80    0.46    0.22 * L  58 A L  62 A L  56 A L  56 A L  99 A L  56 A \n",
      "  66    0.21    1.10    0.68    0.44   D  59 A L  63 A Q  57 A E  57 A E 100 A E  57 A \n",
      "  67    0.13    0.92    0.68    0.49 * D  60 A D  64 A D  58 A D  58 A D 101 A D  58 A \n",
      "  68    0.26    1.02    0.56    0.44   C  61 A S  65 A C  59 A C  59 A S 102 A C  59 A \n",
      "  69    0.24    1.03    0.76    0.62   A  62 A A  66 A A  60 A A  60 A L 103 A A  60 A \n",
      "  70    0.19    0.44    0.32    0.06   F  63 A F  67 A L  61 A F  61 A Y 104 A F  61 A \n",
      "  71    0.11    0.78    0.55    0.22 * P  64 A P  68 A P  62 A P  62 A P 105 A P  62 A \n",
      "  72    0.05    0.17    0.29    0.09 * L  65 A L  69 A L  63 A L  63 A L 106 A L  63 A \n",
      "  73    0.07    0.35    0.28    0.10 * L  66 A L  70 A L  64 A L  64 A L 107 A L  64 A \n",
      "  74    0.17    0.17    0.35    0.18   A  67 A R  71 A K  65 A A  65 A R 108 A D  65 A \n",
      "  75    0.42    0.73    0.42    0.11   G  68 A N  72 A D  66 A G  66 A E 109 A K  66 A \n",
      "  76    0.30    0.25    0.34    0.06   V  69 A I  73 A V  67 A L  67 A V 110 A V  67 A \n",
      "  77    0.53    0.46    0.34    0.13   V  70 A T  74 A I  68 A E  68 A S 111 A V  68 A \n",
      "  78    0.36    0.51    0.46    0.14   I  71 A I  75 A A  69 A A  69 A I 112 A V  69 A \n",
      "  79    0.55    0.49    0.45    0.12   T  72 A T  76 A T  70 A T  70 A G 113 A T  70 A \n",
      "  80    0.31    0.58    0.45    0.17   D  73 A A  77 A D  71 A D  71 A I 114 A A  71 A \n",
      "  81    0.16    1.01    0.54    0.29   D  74 A D  78 A K  72 A D  72 A D 115 A D  72 A \n",
      "  82    0.23    0.72    0.55    0.42   P  75 A A  79 A E  73 A P  73 A P 116 A P  73 A \n",
      "  83    0.29    1.21    0.72    0.48   K  76 A N  80 A E  74 A D  74 A Y 117 A R  74 A \n",
      "  84    0.28    1.35    0.53    0.31   V  77 A E  81 A I  75 A V  75 A E 118 A V  75 A \n",
      "  85    0.19    0.76    0.40    0.17   A  78 A A  82 A A  76 A A  76 A V 119 A A  76 A \n",
      "  86    0.07    0.35    0.38    0.25 * F  79 A F  83 A F  77 A F  77 A F 120 A F  77 A \n",
      "  87    0.13    0.30    0.34    0.12   K  80 A D  84 A K  78 A K  78 A E 121 A D  78 A \n",
      "  88    0.08    0.16    0.32    0.18   D  81 A G  85 A D  79 A D  79 A D 122 A G  79 A \n",
      "  89    0.10    0.20    0.29    0.13   A  82 A A  86 A L  80 A A  80 A V 123 A V  80 A \n",
      "  90    0.14    0.35    0.26    0.09   D  83 A N  87 A D  81 A D  81 A D 124 A A  81 A \n",
      "  91    0.21    0.38    0.25    0.08   V  84 A A  88 A V  82 A Y  82 A W 125 A I  82 A \n",
      "  92    0.09    0.47    0.32    0.17 * A  85 A A  89 A A  83 A A  83 A A 126 A A  83 A \n",
      "  93    0.08    0.13    0.30    0.13   L  86 A F  90 A I  84 A L  84 A L 127 A I  84 A \n",
      "  94    0.27    0.12    0.25    0.14   L  87 A L  91 A L  85 A L  85 A L 128 A M  85 A \n",
      "  95    0.34    0.21    0.28    0.08   V  88 A V  92 A V  86 A V  86 A I 129 A C  86 A \n",
      "  96    0.13    0.23    0.31    0.11 * G  89 A G  93 A G  87 A G  87 A G 130 A G  87 A \n",
      "  97    0.11    0.18    0.47    0.23   A  90 A A  94 A S  88 A A  88 A A 131 A A  88 A \n",
      "  98    0.22    0.43    1.03    0.49   R  91 A K  95 A M  89 A A  89 A K 132 A F  89 A \n",
      "  99    0.09    0.44    1.27    0.89 * P  92 A P  96 A P  90 A P  90 A P 133 A P  90 A \n",
      " 100    0.21    1.13    1.45    0.98   R  93 A -       R  91 A         R 134 A R  91 A \n",
      " 101    0.55    3.36    2.54    1.58   S  94 A -       R  92 A -       G 135 A K  92 A \n",
      " 102    0.90    6.42    3.98    1.71   -       -       D  93 A -       P 136 A A  93 A \n",
      " 103    0.71    5.07    3.97    1.44   -       -       G  94 A -       G 137 A G  94 A \n",
      " 104    0.43    2.70    3.07    1.94   M  97 A -       M  95 A -       M 138 A M  95 A \n",
      " 105    0.48    3.22    2.72    2.08   E  98 A E 102 A E  96 A -       E 139 A E  96 A \n",
      " 106    0.54    2.32    2.23    1.70   R  99 A R 103 A R  97 A -       R 140 A R  97 A \n",
      " 107    0.38    2.10    2.19    1.67   K 100 A A 104 A K  98 A -       A 141 A K  98 A \n",
      " 108    0.40    2.14    1.79    1.33   D 101 A D 105 A D  99 A -       A 142 A D  99 A \n",
      " 109    0.24    1.33    0.86    0.43   L 102 A L 106 A L 100 A -       L 143 A L 100 A \n",
      " 110    0.15    0.60    0.96    0.58 * L 103 A L 107 A L 101 A L 101 A L 144 A L 101 A \n",
      " 111    0.22    1.00    1.00    0.60   S 104 A A 108 A K 102 A Q 102 A D 145 A E 102 A \n",
      " 112    0.27    0.91    1.02    0.36   A 105 A N 109 A A 103 A V 103 A I 146 A M 103 A \n",
      " 113    0.08    0.27    0.67    0.32 * N 106 A N 110 A N 104 A N 104 A N 147 A N 104 A \n",
      " 114    0.18    0.91    1.01    0.31   A 107 A G 111 A V 105 A G 105 A G 148 A A 105 A \n",
      " 115    0.21    0.94    0.83    0.22   E 108 A K 112 A K 106 A K 106 A Q 149 A R 106 A \n",
      " 116    0.08    0.40    0.35    0.07 * I 109 A I 113 A I 107 A I 107 A I 150 A I 107 A \n",
      " 117    0.06    0.34    0.36    0.14 * F 110 A F 114 A F 108 A F 108 A F 151 A F 108 A \n",
      " 118    0.29    0.45    0.47    0.17   T 111 A G 115 A K 109 A T 109 A A 152 A K 109 A \n",
      " 119    0.18    0.42    0.41    0.12   V 112 A P 116 A C 110 A E 110 A D 153 A E 110 A \n",
      " 120    0.05    0.16    0.24    0.06 * Q 113 A Q 117 A Q 111 A Q 111 A Q 154 A Q 111 A \n",
      " 121    0.04    0.26    0.27    0.07 * G 114 A G 118 A G 112 A G 112 A G 155 A G 112 A \n",
      " 122    0.11    0.16    0.31    0.13   A 115 A K 119 A A 113 A R 113 A K 156 A E 113 A \n",
      " 123    0.06    0.23    0.30    0.09 * A 116 A A 120 A A 114 A A 114 A A 157 A A 114 A \n",
      " 124    0.08    0.26    0.26    0.09   L 117 A I 121 A L 115 A L 115 A L 158 A I 115 A \n",
      " 125    0.11    0.25    0.34    0.12   N 118 A N 122 A D 116 A A 116 A N 159 A A 116 A \n",
      " 126    0.13    0.30    0.39    0.13   E 119 A D 123 A K 117 A E 117 A A 160 A A 117 A \n",
      " 127    0.11    0.35    0.38    0.15   V 120 A N 124 A Y 118 A V 118 A V 161 A V 118 A \n",
      " 128    0.09    0.46    0.36    0.10 * A 121 A A 125 A A 119 A A 119 A A 162 A A 119 A \n",
      " 129    0.15    0.58    0.56    0.27   S 122 A A 126 A K 120 A K 120 A S 163 A A 120 A \n",
      " 130    0.18    0.96    0.77    0.53   R 123 A D 127 A K 121 A K 121 A K 164 A S 121 A \n",
      " 131    0.22    1.13    0.68    0.50   D 124 A D 128 A S 122 A D 122 A N 165 A D 122 A \n",
      " 132    0.12    0.54    0.26    0.14   V 125 A I 129 A V 123 A V 123 A V 166 A C 123 A \n",
      " 133    0.11    0.48    0.26    0.10   K 126 A R 130 A K 124 A K 124 A K 167 A R 124 A \n",
      " 134    0.07    0.42    0.26    0.11 * V 127 A V 131 A V 125 A V 125 A V 168 A V 125 A \n",
      " 135    0.06    0.11    0.23    0.16   L 128 A L 132 A I 126 A L 126 A L 169 A V 126 A \n",
      " 136    0.06    0.24    0.18    0.05 * V 129 A V 133 A V 127 A V 127 A V 170 A V 127 A \n",
      " 137    0.08    0.49    0.29    0.09 * V 130 A V 134 A V 128 A V 128 A V 171 A V 128 A \n",
      " 138    0.08    0.65    0.32    0.19 * G 131 A G 135 A G 129 A G 129 A G 172 A G 129 A \n",
      " 139    0.10    0.51    0.31    0.15 * N 132 A N 136 A N 130 A N 130 A N 173 A N 130 A \n",
      " 140    0.20    0.48    0.28    0.14 * P 133 A P 137 A P 131 A P 131 A P 174 A P 131 A \n",
      " 141    0.22    0.38    0.26    0.09   A 134 A A 138 A A 132 A A 132 A C 175 A A 132 A \n",
      " 142    0.08    0.30    0.21    0.08 * N 135 A N 139 A N 133 A N 133 A N 176 A N 133 A \n",
      " 143    0.06    0.31    0.21    0.04 * T 136 A T 140 A T 134 A T 134 A T 177 A T 134 A \n",
      " 144    0.08    0.35    0.23    0.06 * N 137 A N 141 A N 135 A N 135 A N 178 A N 135 A \n",
      " 145    0.08    0.21    0.26    0.08   A 138 A A 142 A C 136 A A 136 A A 179 A A 136 A \n",
      " 146    0.07    0.27    0.26    0.10   Y 139 A L 143 A L 137 A L 137 A L 180 A L 137 A \n",
      " 147    0.10    0.41    0.31    0.10   I 140 A I 144 A T 138 A I 138 A I 181 A I 138 A \n",
      " 148    0.19    0.31    0.27    0.08   A 141 A A 145 A A 139 A A 139 A C 182 A L 139 A \n",
      " 149    0.14    0.33    0.28    0.06   M 142 A S 146 A S 140 A Y 140 A L 183 A L 140 A \n",
      " 150    0.08    0.24    0.29    0.06   K 143 A A 147 A K 141 A K 141 A K 184 A K 141 A \n",
      " 151    0.13    0.29    0.40    0.19   S 144 A A 148 A S 142 A N 142 A N 185 A S 142 A \n",
      " 152    0.08    0.37    0.49    0.26 * A 145 A A 149 A A 143 A A 143 A A 186 A A 143 A \n",
      " 153    0.00    0.00    0.00    0.00   -       -       -       -       -       Q 144 A \n",
      " 154    0.21    0.64    0.79    0.62   P 146 A P 150 A P 144 A P 144 A P 187 A G 145 A \n",
      " 155    0.13    0.41    0.36    0.22   D 147 A D 151 A S 145 A G 145 A D 188 A K 146 A \n",
      " 156    0.11    0.30    0.47    0.32   L 148 A V 152 A I 146 A L 146 A I 189 A L 147 A \n",
      " 157    0.10    0.26    0.40    0.13   P 149 A P 153 A P 147 A N 147 A P 190 A N 148 A \n",
      " 158    0.13    0.28    0.49    0.15   K 150 A A 154 A K 148 A P 148 A A 191 A P 149 A \n",
      " 159    0.12    0.32    0.46    0.18   K 151 A S 155 A E 149 A R 149 A K 192 A R 150 A \n",
      " 160    0.14    0.54    0.34    0.12   N 152 A R 156 A N 150 A N 150 A N 193 A H 151 A \n",
      " 161    0.12    0.60    0.32    0.12   F 153 A F 157 A F 151 A F 151 A F 194 A V 152 A \n",
      " 162    0.22    0.67    0.33    0.14   T 154 A N 158 A S 152 A T 152 A H 195 A T 153 A \n",
      " 163    0.14    0.73    0.29    0.21   A 155 A A 159 A C 153 A A 153 A A 196 A A 154 A \n",
      " 164    0.09    0.13    0.17    0.05   M 156 A M 160 A L 154 A M 154 A L 197 A M 155 A \n",
      " 165    0.17    0.65    0.27    0.18   L 157 A M 161 A T 155 A T 155 A T 198 A T 156 A \n",
      " 166    0.09    0.30    0.20    0.06 * R 158 A R 162 A R 156 A R 156 A R 199 A R 157 A \n",
      " 167    0.08    0.50    0.27    0.10 * L 159 A L 163 A L 157 A L 157 A L 200 A L 158 A \n",
      " 168    0.10    0.63    0.30    0.13 * D 160 A D 164 A D 158 A D 158 A D 201 A D 159 A \n",
      " 169    0.19    0.44    0.21    0.11   H 161 A H 165 A H 159 A H 159 A E 202 A H 160 A \n",
      " 170    0.09    0.41    0.20    0.10 * N 162 A N 166 A N 160 A N 160 A N 203 A N 161 A \n",
      " 171    0.09    0.57    0.28    0.13 * R 163 A R 167 A R 161 A R 161 A R 204 A R 162 A \n",
      " 172    0.05    0.18    0.16    0.08 * A 164 A A 168 A A 162 A A 162 A A 205 A A 163 A \n",
      " 173    0.12    0.22    0.17    0.09   L 165 A I 169 A K 163 A K 163 A K 206 A L 164 A \n",
      " 174    0.22    0.47    0.33    0.11   S 166 A S 170 A A 164 A A 164 A C 207 A S 165 A \n",
      " 175    0.09    0.34    0.37    0.06   Q 167 A Q 171 A Q 165 A Q 165 A Q 208 A L 166 A \n",
      " 176    0.10    0.56    0.33    0.08   L 168 A L 172 A I 166 A L 166 A L 209 A L 167 A \n",
      " 177    0.12    0.81    0.46    0.18 * A 169 A A 173 A A 167 A A 167 A A 210 A A 168 A \n",
      " 178    0.21    0.90    0.56    0.20   A 170 A T 174 A L 168 A K 168 A L 211 A R 169 A \n",
      " 179    0.13    0.89    0.55    0.14 * K 171 A K 175 A K 169 A K 169 A K 212 A K 170 A \n",
      " 180    0.20    0.98    0.68    0.28   S 172 A L 176 A L 170 A T 170 A A 213 A A 171 A \n",
      " 181    0.27    1.34    0.83    0.34 * G 173 A G 177 A G 171 A G 171 A G 214 A G 172 A \n",
      " 182    0.18    0.82    0.79    0.32   K 174 A R 178 A V 172 A T 172 A V 215 A V 173 A \n",
      " 183    0.30    1.18    0.83    0.23   P 175 A G 179 A T 173 A G 173 A F 216 A P 174 A \n",
      " 184    0.22    0.71    0.64    0.16   V 176 A S 180 A S 174 A V 174 A Y 217 A V 175 A \n",
      " 185    0.17    0.85    0.65    0.16   A 177 A A 181 A D 175 A D 175 A D 218 A S 176 A \n",
      " 186    0.20    0.85    0.62    0.28   S 178 A E 182 A D 176 A R 176 A K 219 A Q 177 A \n",
      " 187    0.14    0.63    0.68    0.27   I 179 A F 183 A V 177 A I 177 A V 220 A V 178 A \n",
      " 188    0.49    0.94    1.00    0.39   E 180 A N 184 A K 178 A R 178 A S 221 A R 179 A \n",
      " 189    0.22    0.47    0.79    0.38   K 181 A N 185 A N 179 A R 179 A N 222 A N 180 A \n",
      " 190    0.15    0.77    0.76    0.27   L 182 A I 186 A V 180 A M 180 A V 223 A V 181 A \n",
      " 191    0.19    0.78    0.51    0.26   A 183 A V 187 A I 181 A T 181 A T 224 A I 182 A \n",
      " 192    0.13    0.68    0.45    0.24   V 184 A V 188 A I 182 A V 182 A I 225 A I 183 A \n",
      " 193    0.07    0.49    0.24    0.09 * W 185 A W 189 A W 183 A W 183 A W 226 A W 184 A \n",
      " 194    0.05    0.36    0.35    0.07 * G 186 A G 190 A G 184 A G 184 A G 227 A G 185 A \n",
      " 195    0.25    0.68    0.57    0.40 * N 187 A N 191 A N 185 A N 185 A N 228 A N 186 A \n",
      " 196    0.10    0.57    0.52    0.25 * H 188 A H 192 A H 186 A H 186 A H 229 A H 187 A \n",
      " 197    0.16    0.36    0.48    0.16 * S 189 A S 193 A S 187 A S 187 A S 230 A S 188 A \n",
      " 198    0.16    0.57    0.58    0.25   P 190 A A 194 A S 188 A S 188 A T 231 A S 189 A \n",
      " 199    0.11    0.55    0.60    0.26   T 191 A T 195 A T 189 A I 189 A T 232 A T 190 A \n",
      " 200    0.13    0.47    0.40    0.12   M 192 A Q 196 A Q 190 A M 190 A Q 233 A Q 191 A \n",
      " 201    0.17    0.50    0.39    0.12   Y 193 A F 197 A Y 191 A F 191 A V 234 A V 192 A \n",
      " 202    0.04    0.14    0.36    0.25 * P 194 A P 198 A P 192 A P 192 A P 235 A P 193 A \n",
      " 203    0.09    0.53    0.39    0.20 * D 195 A D 199 A D 193 A D 193 A D 236 A D 194 A \n",
      " 204    0.25    1.26    0.68    0.25   F 196 A I 200 A V 194 A L 194 A F 237 A T 195 A \n",
      " 205    0.26    1.03    0.56    0.25   R 197 A T 201 A N 195 A F 195 A L 238 A D 196 A \n",
      " 206    0.31    1.65    0.68    0.41   F 198 A Y 202 A H 196 A H 196 A N 239 A S 197 A \n",
      " 207    0.26    1.65    0.66    0.39 * A 199 A A 203 A A 197 A A 197 A A 240 A A 198 A \n",
      " 208    0.31    1.54    0.67    0.39   T 200 A T 204 A K 198 A E 198 A K 241 A V 199 A \n",
      " 209    0.27    1.26    0.76    0.22   A 201 A V 205 A V 199 A V 199 A I 242 A I 200 A \n",
      " 210    0.30    1.32    1.32    0.45   E 202 A G 206 A K 200 A D 200 A D 243 A G 201 A \n",
      " 211    0.00    0.00    0.00    0.00   -       -       L 201 A -       -       -       \n",
      " 212    0.00    0.00    0.00    0.00   -       -       Q 202 A -       -       -       \n",
      " 213    0.00    0.00    0.00    0.00   -       -       A 203 A -       -       -       \n",
      " 214    0.00    0.00    0.00    0.00   -       -       K 204 A -       -       -       \n",
      " 215    0.33    2.36    1.54    0.43   G 203 A G 207 A E 205 A G 205 A G 244 A T 202 A \n",
      " 216    0.40    2.09    0.95    0.48   E 204 A E 208 A V 206 A R 206 A R 245 A T 203 A \n",
      " 217    0.25    1.50    0.75    0.33   S 205 A K 209 A G 207 A P 207 A P 246 A P 204 A \n",
      " 218    0.16    0.86    0.63    0.24   L 206 A V 210 A V 208 A A 208 A V 247 A A 205 A \n",
      " 219    0.22    0.92    0.78    0.40   L 207 A T 211 A Y 209 A L 209 A K 248 A R 206 A \n",
      " 220    0.20    1.25    0.86    0.45   K 208 A D 212 A E 210 A E 210 A E 249 A E 207 A \n",
      " 221    0.20    0.88    1.02    0.46   L 209 A L 213 A A 211 A L 211 A V 250 A A 208 A \n",
      " 222    0.21    0.74    1.14    0.56   I 210 A V 214 A V 212 A V 212 A I 251 A I 209 A \n",
      " 223    0.31    1.05    2.04    1.46   N 211 A -       K 213 A -       K 252 A K 210 A \n",
      " 224    0.46    1.62    0.84    0.41   D 212 A D 215 A D 214 A D 214 A R 253 A -       \n",
      " 225    0.35    1.50    0.60    0.32   D 213 A H 216 A D 215 A M 215 A T 254 A -       \n",
      " 226    0.28    0.98    0.60    0.17   V 214 A D 217 A S 216 A E 216 A K 255 A -       \n",
      " 227    0.13    0.37    0.57    0.22   W 215 A W 218 A W 217 A W 217 A W 256 A -       \n",
      " 228    0.27    0.94    0.65    0.22   N 216 A Y 219 A L 218 A Y 218 A L 257 A -       \n",
      " 229    0.34    1.60    0.99    0.29   R 217 A V 220 A K 219 A E 219 A E 258 A -       \n",
      " 230    0.30    1.61    1.06    0.29   D 218 A E 221 A G 220 A K 220 A E 259 A -       \n",
      " 231    0.23    1.02    0.64    0.35   T 219 A E 222 A E 221 A V 221 A E 260 A D 217 A \n",
      " 232    0.13    0.74    0.41    0.19 * F 220 A F 223 A F 222 A F 222 A F 261 A F 218 A \n",
      " 233    0.33    0.62    0.38    0.23   I 221 A I 224 A I 223 A I 223 A T 262 A V 219 A \n",
      " 234    0.24    0.72    0.55    0.20   P 222 A P 225 A T 224 A P 224 A I 263 A Q 220 A \n",
      " 235    0.17    0.95    0.54    0.25   T 223 A R 226 A T 225 A T 225 A T 264 A V 221 A \n",
      " 236    0.10    0.67    0.47    0.27 * V 224 A V 227 A V 226 A V 226 A V 265 A V 222 A \n",
      " 237    0.33    1.25    0.73    0.31   G 225 A A 228 A Q 227 A A 227 A Q 266 A R 223 A \n",
      " 238    0.29    1.78    0.88    0.53   K 226 A N 229 A Q 228 A Q 228 A K 267 A G 224 A \n",
      " 239    0.22    1.72    0.76    0.49 * R 227 A R 230 A R 229 A R 229 A R 268 A R 225 A \n",
      " 240    0.22    1.91    0.86    0.60 * G 228 A G 231 A G 230 A G 230 A G 269 A G 226 A \n",
      " 241    0.35    2.31    1.11    0.77   A 229 A A 232 A A 231 A A 231 A G 270 A A 227 A \n",
      " 242    0.31    2.26    1.03    0.80   A 230 A E 233 A A 232 A A 232 A A 271 A E 228 A \n",
      " 243    0.31    2.03    0.93    0.66   I 231 A I 234 A V 233 A I 233 A L 272 A I 229 A \n",
      " 244    0.34    2.35    1.07    0.81 * I 232 A I 235 A I 234 A I 234 A I 273 A I 230 A \n",
      " 245    0.39    2.82    1.19    1.11   E 233 A E 236 A K 235 A Q 235 A Q 274 A Q 231 A \n",
      " 246    0.41    2.64    1.19    1.00   A 234 A V 237 A A 236 A A 236 A K 275 A L 232 A \n",
      " 247    0.49    2.64    1.25    0.85   R 235 A R 238 A R 237 A R 237 A W 276 A R 233 A \n",
      " 248    0.37    3.01    1.27    1.04   G 236 A G 239 A K 238 A G 238 A G 277 A G 234 A \n",
      " 249    0.33    1.75    0.91    0.70   L 237 A K 240 A L 239 A A 239 A R 278 A L 235 A \n",
      " 250    0.28    2.19    1.01    0.63 * S 238 A S 241 A S 240 A S 240 A S 279 A S 236 A \n",
      " 251    0.29    1.87    0.76    0.50 * S 239 A S 242 A S 241 A S 241 A S 280 A S 237 A \n",
      " 252    0.17    1.02    0.43    0.22 * A 240 A A 243 A A 242 A A 242 A A 281 A A 238 A \n",
      " 253    0.19    0.99    0.38    0.23   A 241 A A 244 A M 243 A A 243 A A 282 A M 239 A \n",
      " 254    0.16    1.17    0.42    0.28 * S 242 A S 245 A S 244 A S 244 A S 283 A S 240 A \n",
      " 255    0.23    1.10    0.40    0.24   A 243 A A 246 A A 245 A A 245 A T 284 A A 241 A \n",
      " 256    0.14    0.93    0.32    0.22 * A 244 A A 247 A A 246 A A 246 A A 285 A A 242 A \n",
      " 257    0.22    0.75    0.29    0.17   N 245 A S 248 A K 247 A N 247 A V 286 A K 243 A \n",
      " 258    0.16    0.75    0.28    0.17   A 246 A S 249 A A 248 A A 248 A S 287 A A 244 A \n",
      " 259    0.20    0.78    0.33    0.15   A 247 A A 250 A I 249 A A 249 A I 288 A A 245 A \n",
      " 260    0.20    0.63    0.38    0.08   I 248 A I 251 A C 250 A I 250 A A 289 A V 246 A \n",
      " 261    0.10    0.51    0.30    0.08   D 249 A D 252 A D 251 A E 251 A D 290 A D 247 A \n",
      " 262    0.21    0.56    0.28    0.09   H 250 A H 253 A H 252 A H 252 A A 291 A H 248 A \n",
      " 263    0.16    0.85    0.39    0.15   V 251 A M 254 A V 253 A I 253 A I 292 A V 249 A \n",
      " 264    0.23    1.22    0.45    0.27   R 252 A R 255 A R 254 A R 254 A K 293 A H 250 A \n",
      " 265    0.22    0.98    0.39    0.22   D 253 A D 256 A D 255 A D 255 A S 294 A D 251 A \n",
      " 266    0.21    0.70    0.46    0.14   W 254 A W 257 A I 256 A W 256 A L 295 A W 252 A \n",
      " 267    0.29    1.74    0.70    0.35   V 255 A V 258 A W 257 A A 257 A V 296 A I 253 A \n",
      " 268    0.39    2.08    0.65    0.51   L 256 A Q 259 A F 258 A L 258 A T 297 A H 254 A \n",
      " 269    0.48    2.62    0.88    0.59   G 257 A G 260 A G 259 A G 259 A P 298 A G 255 A \n",
      " 270    0.21    1.58    0.58    0.33 * T 258 A T 261 A T 260 A T 260 A T 299 A T 256 A \n",
      " 271    0.23    1.47    1.24    0.48   N 259 A E 262 A P 261 A P 261 A P 300 A P 257 A \n",
      " 272    0.34    1.72    1.22    0.55   G 260 A -       E 262 A E 262 A E 301 A E 258 A \n",
      " 273    0.32    2.01    0.83    0.42   -       -       G 263 A G 263 A G 302 A G 259 A \n",
      " 274    0.33    1.58    0.96    0.49   K 261 A A 263 A E 264 A D 264 A D 303 A V 260 A \n",
      " 275    0.12    0.53    0.53    0.24   W 262 A W 264 A F 265 A W 265 A W 304 A Y 261 A \n",
      " 276    0.18    0.45    0.42    0.19   V 263 A S 265 A V 266 A V 266 A F 305 A V 262 A \n",
      " 277    0.07    0.29    0.22    0.10   T 264 A S 266 A S 267 A S 267 A S 306 A S 263 A \n",
      " 278    0.23    0.81    0.37    0.22   M 265 A A 267 A M 268 A M 268 A T 307 A M 264 A \n",
      " 279    0.12    0.62    0.31    0.15   G 266 A A 268 A G 269 A A 269 A G 308 A G 265 A \n",
      " 280    0.17    0.70    0.30    0.17   I 267 A I 269 A I 270 A V 270 A V 309 A V 266 A \n",
      " 281    0.22    0.37    0.28    0.09   P 268 A P 270 A I 271 A P 271 A Y 310 A Y 267 A \n",
      " 282    0.20    0.28    0.29    0.06   S 269 A S 271 A S 272 A S 272 A T 311 A S 268 A \n",
      " 283    0.39    1.70    1.06    0.38   D 270 A T 272 A D 273 A Q 273 A T 312 A D 269 A \n",
      " 284    0.40    3.24    2.05    0.36   G 271 A G 273 A G 274 A G 274 A G 313 A E 270 A \n",
      " 285    0.36    1.94    1.17    0.50   S 272 A A 274 A N 275 A E 275 A N 314 A N 271 A \n",
      " 286    0.28    1.01    0.78    0.22   -       -       S 276 A -       P 315 A P 272 A \n",
      " 287    0.15    1.06    0.58    0.29 * Y 273 A Y 275 A Y 277 A Y 277 A Y 316 A Y 273 A \n",
      " 288    0.16    1.25    1.00    0.48 * G 274 A G 276 A G 278 A G 278 A G 317 A G 274 A \n",
      " 289    0.13    0.68    0.51    0.24   I 275 A I 277 A V 279 A I 279 A I 318 A V 275 A \n",
      " 290    0.22    0.96    0.55    0.29   P 276 A P 278 A P 280 A P 280 A A 319 A P 276 A \n",
      " 291    0.16    0.81    0.56    0.26   E 277 A E 279 A D 281 A E 281 A E 320 A S 277 A \n",
      " 292    0.21    0.69    0.84    0.42   D 278 A G 280 A D 282 A G 282 A D 321 A G 278 A \n",
      " 293    0.14    0.79    0.60    0.21   I 279 A I 281 A L 283 A I 283 A I 322 A L 279 A \n",
      " 294    0.11    0.27    0.32    0.13   I 280 A F 282 A L 284 A V 284 A V 323 A I 280 A \n",
      " 295    0.13    0.40    0.32    0.17   Y 281 A V 283 A Y 285 A Y 285 A F 324 A F 281 A \n",
      " 296    0.12    0.51    0.33    0.18   G 282 A G 284 A S 286 A S 286 A S 325 A S 282 A \n",
      " 297    0.23    0.55    0.28    0.10   V 283 A L 285 A F 287 A F 287 A M 326 A F 283 A \n",
      " 298    0.09    0.65    0.38    0.26 * P 284 A P 286 A P 288 A P 288 A P 327 A P 284 A \n",
      " 299    0.20    0.59    0.40    0.13   V 285 A T 287 A V 289 A V 289 A C 328 A C 285 A \n",
      " 300    0.46    0.43    0.46    0.23   I 286 A V 288 A T 290 A T 290 A R 329 A T 286 A \n",
      " 301    0.46    0.90    0.52    0.20   C 287 A S 289 A I 291 A A 291 A S 330 A C 287 A \n",
      " 302    0.43    1.07    0.75    0.08   E 288 A R 290 A K 292 A K 292 A K 331 A H 288 A \n",
      " 303    0.00    0.00    0.00    0.00   -       -       -       -       G 332 A -       \n",
      " 304    0.35    2.39    1.21    0.44   N 289 A N 291 A D 293 A D 293 A D 333 A A 289 A \n",
      " 305    0.37    2.12    0.78    0.50   G 290 A G 292 A K 294 A G 294 A G 334 A G 290 A \n",
      " 306    0.54    1.87    0.81    0.37   E 291 A E 293 A T 295 A A 295 A D 335 A E 291 A \n",
      " 307    0.41    0.93    0.65    0.17   Y 292 A W 294 A W 296 A Y 296 A Y 336 A W 292 A \n",
      " 308    0.43    0.92    0.59    0.20   K 293 A E 295 A K 297 A R 297 A E 337 A T 293 A \n",
      " 309    0.38    0.59    0.44    0.16   R 294 A I 296 A I 298 A V 298 A L 338 A V 294 A \n",
      " 310    0.41    0.73    0.40    0.15   V 295 A V 297 A V 299 A V 299 A A 339 A V 295 A \n",
      " 311    0.26    1.29    0.87    0.38   E 296 A E 298 A E 300 A E 300 A T 340 A S 296 A \n",
      " 312    0.65    4.67    1.92    0.98   G 297 A G 299 A G 301 A G 301 A D 341 A G 297 A \n",
      " 313    0.32    1.99    1.29    0.58   L 298 A L 300 A L 302 A L 302 A V 342 A K 298 A \n",
      " 314    0.43    2.17    1.64    0.88   E 299 A E 301 A P 303 A E 303 A S 343 A L 299 A \n",
      " 315    0.53    1.13    0.69    0.24   I 300 A I 302 A I 304 A I 304 A N 344 A -       \n",
      " 316    0.17    0.49    0.51    0.22   D 301 A S 303 A N 305 A N 305 A D 345 A -       \n",
      " 317    0.19    0.77    0.60    0.14   A 302 A D 304 A D 306 A E 306 A D 346 A -       \n",
      " 318    0.11    0.36    0.48    0.26   F 303 A F 305 A F 307 A F 307 A F 347 A -       \n",
      " 319    0.23    0.65    0.46    0.22   S 304 A Q 306 A S 308 A A 308 A L 348 A -       \n",
      " 320    0.32    0.99    0.97    0.59   R 305 A R 307 A R 309 A R 309 A W 349 A K 305 A \n",
      " 321    0.16    0.50    0.88    0.73   E 306 A A 308 A E 310 A K 310 A E 350 A Q 306 A \n",
      " 322    0.16    0.77    0.83    0.70   K 307 A R 309 A K 311 A R 311 A R 351 A R 307 A \n",
      " 323    0.19    0.66    0.55    0.28   M 308 A I 310 A M 312 A M 312 A I 352 A L 308 A \n",
      " 324    0.17    0.46    0.62    0.31   D 309 A D 311 A D 313 A E 313 A K 353 A A 309 A \n",
      " 325    0.16    0.33    0.54    0.25   G 310 A A 312 A L 314 A I 314 A K 354 A S 310 A \n",
      " 326    0.13    0.46    0.51    0.25   T 311 A N 313 A T 315 A T 315 A S 355 A T 311 A \n",
      " 327    0.18    0.49    0.53    0.25   L 312 A A 314 A A 316 A A 316 A E 356 A I 312 A \n",
      " 328    0.14    0.51    0.61    0.22   A 313 A Q 315 A K 317 A Q 317 A A 357 A A 313 A \n",
      " 329    0.21    0.39    0.58    0.22 * E 314 A E 316 A E 318 A E 318 A E 358 A E 314 A \n",
      " 330    0.07    0.40    0.39    0.16 * L 315 A L 317 A L 319 A L 319 A L 359 A L 315 A \n",
      " 331    0.12    0.27    0.43    0.18   L 316 A Q 318 A A 320 A L 320 A L 360 A Q 316 A \n",
      " 332    0.14    0.38    0.57    0.23   E 317 A A 319 A E 321 A D 321 A A 361 A E 317 A \n",
      " 333    0.08    0.37    0.53    0.22 * E 318 A E 320 A E 322 A E 322 A E 362 A E 318 A \n",
      " 334    0.13    0.39    0.47    0.17   R 319 A R 321 A K 323 A M 323 A K 363 A R 319 A \n",
      " 335    0.16    0.47    0.58    0.27   D 320 A E 322 A E 324 A E 324 A K 364 A A 320 A \n",
      " 336    0.45    0.38    0.63    0.35   G 321 A A 323 A T 325 A Q 325 A C 365 A Q 321 A \n",
      " 337    0.16    0.64    0.73    0.32   V 322 A V 324 A A 326 A V 326 A V 366 A A 322 A \n",
      " 338    0.46    0.71    0.88    0.33   A 323 A R 325 A F 327 A K 327 A A 367 A G 323 A \n",
      " 339    0.48    2.74    2.61    1.06   H 324 A D 326 A E 328 A A 328 A H 368 A L 324 A \n",
      " 340    0.66    3.66    3.14    0.52   L 325 A L 327 A F 329 A L 329 A L 369 A -       \n",
      " 341    0.82    4.65    3.33    0.38   -       L 328 A L 330 A G 330 A T 370 A -       \n",
      " 342    0.85    5.00    3.68    0.60   -       L 329 A S 331 A L 331 A G 371 A -       \n",
      " 343    0.86    5.95    5.45    2.01   -       E 330 A S 332 A I 332 A E 372 A -       \n",
      " 344    1.58   11.41    5.70    0.00   -       -       A 333 A -       G 373 A -       \n",
      " 345    0.00    0.00    0.00    0.00   -       -       -       -       N 374 A -       \n",
      " 346    0.00    0.00    0.00    0.00   -       -       -       -       A 375 A -       \n",
      " 347    0.00    0.00    0.00    0.00   -       -       -       -       Y 376 A -       \n",
      " 348    0.00    0.00    0.00    0.00   -       -       -       -       C 377 A -       \n",
      " 349    0.00    0.00    0.00    0.00   -       -       -       -       D 378 A -       \n",
      " 350    0.00    0.00    0.00    0.00   -       -       -       -       V 379 A -       \n",
      " 351    0.00    0.00    0.00    0.00   -       -       -       -       P 380 A -       \n",
      " 352    0.00    0.00    0.00    0.00   -       -       -       -       E 381 A -       \n",
      " 353    0.00    0.00    0.00    0.00   -       -       -       -       D 382 A -       \n",
      " 354    0.00    0.00    0.00    0.00   -       -       -       -       T 383 A -       \n",
      " 355    0.00    0.00    0.00    0.00   -       -       -       -       M 384 A -       \n",
      " 356    0.00    0.00    0.00    0.00   -       -       -       -       L 385 A -       \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "SALIGN______> Raw QUALITY_SCORE of the multiple alignment:     307.2\n",
      "              QUALITY_SCORE (percentage)                 :      96.7\n",
      "              Number of unique protein pairs             :        15\n",
      "              QUALITY_SCORE_min                          :       296\n",
      "              RMS_CUTOFF                                 :    3.5000\n",
      "              QS is the average number of structurally equivalent residue pairs.\n",
      "              Two residues are equivalent when closer than RMS_CUTOFF upon\n",
      "              pairwise least-squares superposition given current alignment.\n",
      "\n",
      "\n",
      "SALIGN______> Matrix of pairwise equivalences from individual pairwise alignments:\n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     314     312     310     318     302\n",
      "6itkA           0       0     307     308     314     297\n",
      "5mdhA           0       0       0     309     302     300\n",
      "1bdmA           0       0       0       0     310     297\n",
      "7mdhA           0       0       0       0       0     306\n",
      "4h7pA           0       0       0       0       0       0\n",
      "SALIGN______> Matrix of pairwise equivalences: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     314     313     311     318     303\n",
      "6itkA           0       0     308     308     314     296\n",
      "5mdhA           0       0       0     308     302     300\n",
      "1bdmA           0       0       0       0     310     297\n",
      "7mdhA           0       0       0       0       0     306\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n",
      "SALIGN______> Matrix of differences in pairwise equivalences inferred from the tree and individually: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0       0       1       1       0       1\n",
      "6itkA           0       0       1       0       0      -1\n",
      "5mdhA           0       0       0      -1       0       0\n",
      "1bdmA           0       0       0       0       0       0\n",
      "7mdhA           0       0       0       0       0       0\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n",
      "SALIGN______> Matrix of %differences in pairwise equivalences inferred from the tree and individually: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0       0       0       0       0       0\n",
      "6itkA           0       0       0       0       0       0\n",
      "5mdhA           0       0       0       0       0       0\n",
      "1bdmA           0       0       0       0       0       0\n",
      "7mdhA           0       0       0       0       0       0\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n",
      "  ALIGN_CODES                                   : 3d5tA 6itkA 5mdhA 1bdmA 7mdhA 4h7pA \n",
      "  Atom type for alignment (FIT_ATOMS)           : CA\n",
      "  FIT                                           :         T\n",
      "  IMPROVE_ALIGNMENT                             :         T\n",
      "  WATER_IO, HETATM_IO, HYDROGEN_IO              :         F        F        F\n",
      "  NO_TER                                        :         F\n",
      "  OUTPUT                                        : ALIGNMENT QUALITY\n",
      "  FIT_ON_FIRST                                  :         F\n",
      "  WRITE_FIT                                     :         T\n",
      "  FIT_PDBNAM                                    :         T\n",
      "  WRITE_WHOLE_PDB                               :         F\n",
      "  CURRENT_DIRECTORY                             :         T\n",
      "  Feature weights (FEATURE_WEIGHTS)             :    1.0000   1.0000   1.0000   1.0000   1.0000   0.0000\n",
      "  NORMALIZE_PP_SCORES                           :         F\n",
      "  Gap initiation penalty (GAP_PENALTIES[1])     :  -450.0000\n",
      "  Gap extension penalty (GAP_PENALTIES[2])      :   -50.0000\n",
      "  Break-break bonus                             : 10000.0000\n",
      "  Gap_function (align2d flag)                    :         F\n",
      "  2-D Gap penalties                             :  3.50  3.50  3.50  0.20  4.00  6.50  2.00  0.00\n",
      "  GAP-GAP_SCORE                                 :    0.0000\n",
      "  GAP-RESIDUE_SCORE                             :    0.0000\n",
      "  Perform gap-residue and gap-gap correction    :         T\n",
      "  Residue type - residue type file (RR_FILE)    : $(LIB)/as1.sim.mat\n",
      "  OFF_DIAGONAL                                  :       100\n",
      "  OVERHANG                                      :        30\n",
      "  LOCAL_ALIGNMENT                               :         F\n",
      "  MATRIX_OFFSET                                 :    0.0000\n",
      "  N_SUBOPT                                      :         0\n",
      "  SUBOPT_OFFSET                                 :    0.0000\n",
      "  Gap introduction penalty (GAP_PENALTIES_3D_1) :    0.0000\n",
      "  Gap extension penalty (GAP_PENALTIES_3D_2)    :    3.0000\n",
      "  Max dist for equiv (2 * GAP_PENALTIES_3D_2)   :    6.0000\n",
      "  ALIGN3D_TRF                                   :         F\n",
      "  RMS_CUTOFF                                    :    3.5\n",
      "\n",
      "\n",
      "\n",
      "salign__276_> 'align_block' changed to 1.\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      315      315        0.8822        0.8822        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      316      316        1.2115        1.2115        0.0003\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      316      316        1.2115        1.2115        0.0000\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      312      312        0.8224        0.8224        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      321      321        1.2299        1.2299        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      307      307        1.0979        1.0979        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      313      313        1.1002        1.1002        0.0004\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      313      313        1.1002        1.1002        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      311      311        0.9316        0.9316        0.0001\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      311      311        0.9316        0.9316        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      317      317        1.2407        1.2407        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      302      302        1.0694        1.0694        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      313      313        1.2063        1.2063        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      313      313        1.2063        1.2063        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      316      316        1.3291        1.3291        0.0008\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      314      314        1.2549        1.2549        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        3      314      314        1.2549        1.2549        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      306      306        1.1744        1.1744        0.0003\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      306      306        1.1744        1.1744        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      315      315        0.9869        0.9869        0.0001\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      300      300        1.0091        1.0091        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      300      300        1.0091        1.0091        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      312      312        1.1880        1.1880        0.0000\n",
      "\n",
      "SALIGN______> Matrix of pairwise protein distances for the initial tree:\n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA       0.000   0.266   0.367   0.266   0.389   0.328\n",
      "6itkA       0.266   0.000   0.364   0.276   0.404   0.356\n",
      "5mdhA       0.367   0.364   0.000   0.357   0.443   0.378\n",
      "1bdmA       0.266   0.276   0.357   0.000   0.360   0.339\n",
      "7mdhA       0.389   0.404   0.443   0.360   0.000   0.404\n",
      "4h7pA       0.328   0.356   0.378   0.339   0.404   0.000\n",
      "\n",
      "SALIGN____> Matrix of pairwise equivalences: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     314     312     311     318     303\n",
      "6itkA           0       0     308     308     314     297\n",
      "5mdhA           0       0       0     308     302     300\n",
      "1bdmA           0       0       0       0     313     297\n",
      "7mdhA           0       0       0       0       0     306\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "openf___224_> Open           ldh.tree\n",
      "                                                               .--- 3d5tA          0.2657\n",
      "                                                               |\n",
      "                                                             .----- 1bdmA          0.2714\n",
      "                                                             |\n",
      "                                   .------------------------------- 6itkA          0.3447\n",
      "                                   |\n",
      "                          .---------------------------------------- 4h7pA          0.3704\n",
      "                          |\n",
      "        .---------------------------------------------------------- 5mdhA          0.4199\n",
      "        |\n",
      "      .------------------------------------------------------------ 7mdhA\n",
      "\n",
      "      +----+----+----+----+----+----+----+----+----+----+----+----+\n",
      "     0.4260    0.3983    0.3705    0.3428    0.3150    0.2873    0.2595\n",
      "          0.4121    0.3844    0.3566    0.3289    0.3011    0.2734\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    1\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      312      312        0.8224        0.8224        0.0000\n",
      "\n",
      "Current alignment total score:        86.61\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -                       M   0 A                 \n",
      "   2    0.35    1.06    0.53    0.00 * K   3 A                 K   1 A                 \n",
      "   3    0.32    1.05    0.52    0.00   P   4 A                 A   2 A                 \n",
      "   4    0.33    0.61    0.31    0.00   A   5 A                 P   3 A                 \n",
      "   5    0.29    0.34    0.17    0.00   K   6 A                 V   4 A                 \n",
      "   6    0.07    0.18    0.09    0.00 * R   7 A                 R   5 A                 \n",
      "   7    0.09    0.31    0.15    0.00 * V   8 A                 V   6 A                 \n",
      "   8    0.09    0.28    0.14    0.00 * A   9 A                 A   7 A                 \n",
      "   9    0.07    0.22    0.11    0.00 * V  10 A                 V   8 A                 \n",
      "  10    0.08    0.25    0.13    0.00 * T  11 A                 T   9 A                 \n",
      "  11    0.09    0.37    0.19    0.00 * G  12 A                 G  10 A                 \n",
      "  12    0.20    0.82    0.41    0.00 * A  13 A                 A  11 A                 \n",
      "  13    0.18    0.74    0.37    0.00 * A  14 A                 A  12 A                 \n",
      "  14    0.07    0.29    0.14    0.00 * G  15 A                 G  13 A                 \n",
      "  15    0.11    0.33    0.17    0.00 * Q  16 A                 Q  14 A                 \n",
      "  16    0.10    0.32    0.16    0.00 * I  17 A                 I  15 A                 \n",
      "  17    0.14    0.26    0.13    0.00   A  18 A                 G  16 A                 \n",
      "  18    0.05    0.13    0.07    0.00 * Y  19 A                 Y  17 A                 \n",
      "  19    0.07    0.20    0.10    0.00 * S  20 A                 S  18 A                 \n",
      "  20    0.07    0.23    0.11    0.00 * L  21 A                 L  19 A                 \n",
      "  21    0.07    0.25    0.12    0.00 * L  22 A                 L  20 A                 \n",
      "  22    0.05    0.14    0.07    0.00 * F  23 A                 F  21 A                 \n",
      "  23    0.05    0.11    0.05    0.00 * R  24 A                 R  22 A                 \n",
      "  24    0.10    0.36    0.18    0.00 * I  25 A                 I  23 A                 \n",
      "  25    0.09    0.30    0.15    0.00 * A  26 A                 A  24 A                 \n",
      "  26    0.13    0.10    0.05    0.00   N  27 A                 A  25 A                 \n",
      "  27    0.06    0.24    0.12    0.00 * G  28 A                 G  26 A                 \n",
      "  28    0.19    0.54    0.27    0.00   D  29 A                 E  27 A                 \n",
      "  29    0.25    0.85    0.42    0.00   L  30 A                 M  28 A                 \n",
      "  30    0.16    0.68    0.34    0.00 * L  31 A                 L  29 A                 \n",
      "  31    0.10    0.42    0.21    0.00 * G  32 A                 G  30 A                 \n",
      "  32    0.12    0.42    0.21    0.00 * K  33 A                 K  31 A                 \n",
      "  33    0.14    0.50    0.25    0.00 * D  34 A                 D  32 A                 \n",
      "  34    0.31    0.36    0.18    0.00 * Q  35 A                 Q  33 A                 \n",
      "  35    0.09    0.38    0.19    0.00 * P  36 A                 P  34 A                 \n",
      "  36    0.09    0.30    0.15    0.00 * V  37 A                 V  35 A                 \n",
      "  37    0.14    0.52    0.26    0.00 * I  38 A                 I  36 A                 \n",
      "  38    0.09    0.30    0.15    0.00 * L  39 A                 L  37 A                 \n",
      "  39    0.07    0.18    0.09    0.00 * Q  40 A                 Q  38 A                 \n",
      "  40    0.12    0.48    0.24    0.00 * L  41 A                 L  39 A                 \n",
      "  41    0.10    0.37    0.18    0.00 * L  42 A                 L  40 A                 \n",
      "  42    0.20    0.55    0.27    0.00   D  43 A                 E  41 A                 \n",
      "  43    0.39    0.55    0.28    0.00   L  44 A                 I  42 A                 \n",
      "  44    0.35    0.63    0.31    0.00 * P  45 A                 P  43 A                 \n",
      "  45    0.26    1.09    0.55    0.00 * Q  46 A                 Q  44 A                 \n",
      "  46    0.19    0.74    0.37    0.00 * A  47 A                 A  45 A                 \n",
      "  47    0.28    0.71    0.35    0.00   Q  48 A                 M  46 A                 \n",
      "  48    0.26    0.80    0.40    0.00   A  49 A                 K  47 A                 \n",
      "  49    0.14    0.56    0.28    0.00 * A  50 A                 A  48 A                 \n",
      "  50    0.17    0.48    0.24    0.00   V  51 A                 L  49 A                 \n",
      "  51    0.20    0.50    0.25    0.00   K  52 A                 E  50 A                 \n",
      "  52    0.04    0.14    0.07    0.00 * G  53 A                 G  51 A                 \n",
      "  53    0.08    0.27    0.14    0.00 * V  54 A                 V  52 A                 \n",
      "  54    0.14    0.54    0.27    0.00 * V  55 A                 V  53 A                 \n",
      "  55    0.12    0.37    0.18    0.00 * M  56 A                 M  54 A                 \n",
      "  56    0.09    0.29    0.15    0.00 * E  57 A                 E  55 A                 \n",
      "  57    0.10    0.38    0.19    0.00 * L  58 A                 L  56 A                 \n",
      "  58    0.17    0.42    0.21    0.00   D  59 A                 E  57 A                 \n",
      "  59    0.12    0.44    0.22    0.00 * D  60 A                 D  58 A                 \n",
      "  60    0.08    0.36    0.18    0.00 * C  61 A                 C  59 A                 \n",
      "  61    0.12    0.44    0.22    0.00 * A  62 A                 A  60 A                 \n",
      "  62    0.09    0.30    0.15    0.00 * F  63 A                 F  61 A                 \n",
      "  63    0.08    0.32    0.16    0.00 * P  64 A                 P  62 A                 \n",
      "  64    0.07    0.21    0.10    0.00 * L  65 A                 L  63 A                 \n",
      "  65    0.07    0.21    0.10    0.00 * L  66 A                 L  64 A                 \n",
      "  66    0.19    0.77    0.38    0.00 * A  67 A                 A  65 A                 \n",
      "  67    0.18    0.81    0.41    0.00 * G  68 A                 G  66 A                 \n",
      "  68    0.19    0.55    0.27    0.00   V  69 A                 L  67 A                 \n",
      "  69    0.29    0.81    0.40    0.00   V  70 A                 E  68 A                 \n",
      "  70    0.24    0.62    0.31    0.00   I  71 A                 A  69 A                 \n",
      "  71    0.14    0.52    0.26    0.00 * T  72 A                 T  70 A                 \n",
      "  72    0.30    0.35    0.17    0.00 * D  73 A                 D  71 A                 \n",
      "  73    0.09    0.32    0.16    0.00 * D  74 A                 D  72 A                 \n",
      "  74    0.09    0.38    0.19    0.00 * P  75 A                 P  73 A                 \n",
      "  75    0.21    0.47    0.24    0.00   K  76 A                 D  74 A                 \n",
      "  76    0.11    0.42    0.21    0.00 * V  77 A                 V  75 A                 \n",
      "  77    0.16    0.64    0.32    0.00 * A  78 A                 A  76 A                 \n",
      "  78    0.08    0.23    0.12    0.00 * F  79 A                 F  77 A                 \n",
      "  79    0.05    0.09    0.04    0.00 * K  80 A                 K  78 A                 \n",
      "  80    0.09    0.30    0.15    0.00 * D  81 A                 D  79 A                 \n",
      "  81    0.13    0.47    0.23    0.00 * A  82 A                 A  80 A                 \n",
      "  82    0.08    0.23    0.11    0.00 * D  83 A                 D  81 A                 \n",
      "  83    0.19    0.27    0.14    0.00   V  84 A                 Y  82 A                 \n",
      "  84    0.05    0.08    0.04    0.00 * A  85 A                 A  83 A                 \n",
      "  85    0.07    0.21    0.10    0.00 * L  86 A                 L  84 A                 \n",
      "  86    0.06    0.18    0.09    0.00 * L  87 A                 L  85 A                 \n",
      "  87    0.08    0.25    0.13    0.00 * V  88 A                 V  86 A                 \n",
      "  88    0.05    0.21    0.10    0.00 * G  89 A                 G  87 A                 \n",
      "  89    0.13    0.47    0.24    0.00 * A  90 A                 A  88 A                 \n",
      "  90    0.54    0.99    0.50    0.00   R  91 A                 A  89 A                 \n",
      "  91    0.61    2.43    1.22    0.00 * P  92 A                 P  90 A                 \n",
      "  92    0.14    0.00    0.00    0.00 * R  93 A                                         \n",
      "  93    0.00    0.00    0.00    0.00   S  94 A                 -                       \n",
      "  94    0.00    0.00    0.00    0.00   M  97 A                 -                       \n",
      "  95    0.00    0.00    0.00    0.00   E  98 A                 -                       \n",
      "  96    0.00    0.00    0.00    0.00   R  99 A                 -                       \n",
      "  97    0.00    0.00    0.00    0.00   K 100 A                 -                       \n",
      "  98    0.00    0.00    0.00    0.00   D 101 A                 -                       \n",
      "  99    0.00    0.00    0.00    0.00   L 102 A                 -                       \n",
      " 100    0.71    1.90    0.95    0.00 * L 103 A                 L 101 A                 \n",
      " 101    0.50    1.91    0.96    0.00   S 104 A                 Q 102 A                 \n",
      " 102    0.28    0.87    0.44    0.00   A 105 A                 V 103 A                 \n",
      " 103    0.12    0.34    0.17    0.00 * N 106 A                 N 104 A                 \n",
      " 104    0.22    0.65    0.32    0.00   A 107 A                 G 105 A                 \n",
      " 105    0.21    0.59    0.29    0.00   E 108 A                 K 106 A                 \n",
      " 106    0.07    0.18    0.09    0.00 * I 109 A                 I 107 A                 \n",
      " 107    0.08    0.26    0.13    0.00 * F 110 A                 F 108 A                 \n",
      " 108    0.31    0.42    0.21    0.00 * T 111 A                 T 109 A                 \n",
      " 109    0.23    0.48    0.24    0.00   V 112 A                 E 110 A                 \n",
      " 110    0.06    0.11    0.05    0.00 * Q 113 A                 Q 111 A                 \n",
      " 111    0.09    0.35    0.17    0.00 * G 114 A                 G 112 A                 \n",
      " 112    0.22    0.43    0.22    0.00   A 115 A                 R 113 A                 \n",
      " 113    0.10    0.32    0.16    0.00 * A 116 A                 A 114 A                 \n",
      " 114    0.08    0.29    0.14    0.00 * L 117 A                 L 115 A                 \n",
      " 115    0.21    0.50    0.25    0.00   N 118 A                 A 116 A                 \n",
      " 116    0.16    0.61    0.31    0.00 * E 119 A                 E 117 A                 \n",
      " 117    0.12    0.46    0.23    0.00 * V 120 A                 V 118 A                 \n",
      " 118    0.13    0.46    0.23    0.00 * A 121 A                 A 119 A                 \n",
      " 119    0.22    0.52    0.26    0.00   S 122 A                 K 120 A                 \n",
      " 120    0.17    0.46    0.23    0.00   R 123 A                 K 121 A                 \n",
      " 121    0.22    0.93    0.46    0.00 * D 124 A                 D 122 A                 \n",
      " 122    0.13    0.48    0.24    0.00 * V 125 A                 V 123 A                 \n",
      " 123    0.12    0.43    0.21    0.00 * K 126 A                 K 124 A                 \n",
      " 124    0.05    0.13    0.07    0.00 * V 127 A                 V 125 A                 \n",
      " 125    0.04    0.09    0.04    0.00 * L 128 A                 L 126 A                 \n",
      " 126    0.07    0.20    0.10    0.00 * V 129 A                 V 127 A                 \n",
      " 127    0.09    0.31    0.15    0.00 * V 130 A                 V 128 A                 \n",
      " 128    0.08    0.32    0.16    0.00 * G 131 A                 G 129 A                 \n",
      " 129    0.19    0.75    0.38    0.00 * N 132 A                 N 130 A                 \n",
      " 130    0.30    0.38    0.19    0.00 * P 133 A                 P 131 A                 \n",
      " 131    0.14    0.52    0.26    0.00 * A 134 A                 A 132 A                 \n",
      " 132    0.10    0.30    0.15    0.00 * N 135 A                 N 133 A                 \n",
      " 133    0.09    0.27    0.14    0.00 * T 136 A                 T 134 A                 \n",
      " 134    0.11    0.33    0.17    0.00 * N 137 A                 N 135 A                 \n",
      " 135    0.13    0.48    0.24    0.00 * A 138 A                 A 136 A                 \n",
      " 136    0.24    0.61    0.31    0.00   Y 139 A                 L 137 A                 \n",
      " 137    0.13    0.48    0.24    0.00 * I 140 A                 I 138 A                 \n",
      " 138    0.13    0.48    0.24    0.00 * A 141 A                 A 139 A                 \n",
      " 139    0.27    0.64    0.32    0.00   M 142 A                 Y 140 A                 \n",
      " 140    0.13    0.43    0.22    0.00 * K 143 A                 K 141 A                 \n",
      " 141    0.23    0.64    0.32    0.00   S 144 A                 N 142 A                 \n",
      " 142    0.21    0.82    0.41    0.00 * A 145 A                 A 143 A                 \n",
      " 143    0.46    2.13    1.06    0.00 * P 146 A                 P 144 A                 \n",
      " 144    0.24    0.57    0.29    0.00   D 147 A                 G 145 A                 \n",
      " 145    0.20    0.75    0.38    0.00 * L 148 A                 L 146 A                 \n",
      " 146    0.28    0.72    0.36    0.00   P 149 A                 N 147 A                 \n",
      " 147    0.27    0.69    0.34    0.00   K 150 A                 P 148 A                 \n",
      " 148    0.23    0.67    0.34    0.00   K 151 A                 R 149 A                 \n",
      " 149    0.10    0.22    0.11    0.00 * N 152 A                 N 150 A                 \n",
      " 150    0.08    0.22    0.11    0.00 * F 153 A                 F 151 A                 \n",
      " 151    0.07    0.17    0.08    0.00 * T 154 A                 T 152 A                 \n",
      " 152    0.07    0.13    0.07    0.00 * A 155 A                 A 153 A                 \n",
      " 153    0.13    0.37    0.19    0.00 * M 156 A                 M 154 A                 \n",
      " 154    0.24    0.57    0.29    0.00   L 157 A                 T 155 A                 \n",
      " 155    0.10    0.34    0.17    0.00 * R 158 A                 R 156 A                 \n",
      " 156    0.12    0.49    0.25    0.00 * L 159 A                 L 157 A                 \n",
      " 157    0.15    0.62    0.31    0.00 * D 160 A                 D 158 A                 \n",
      " 158    0.10    0.38    0.19    0.00 * H 161 A                 H 159 A                 \n",
      " 159    0.11    0.33    0.16    0.00 * N 162 A                 N 160 A                 \n",
      " 160    0.14    0.56    0.28    0.00 * R 163 A                 R 161 A                 \n",
      " 161    0.11    0.35    0.17    0.00 * A 164 A                 A 162 A                 \n",
      " 162    0.17    0.21    0.11    0.00   L 165 A                 K 163 A                 \n",
      " 163    0.17    0.40    0.20    0.00   S 166 A                 A 164 A                 \n",
      " 164    0.17    0.68    0.34    0.00 * Q 167 A                 Q 165 A                 \n",
      " 165    0.12    0.48    0.24    0.00 * L 168 A                 L 166 A                 \n",
      " 166    0.13    0.49    0.25    0.00 * A 169 A                 A 167 A                 \n",
      " 167    0.22    0.61    0.31    0.00   A 170 A                 K 168 A                 \n",
      " 168    0.18    0.75    0.37    0.00 * K 171 A                 K 169 A                 \n",
      " 169    0.23    0.78    0.39    0.00   S 172 A                 T 170 A                 \n",
      " 170    0.14    0.63    0.31    0.00 * G 173 A                 G 171 A                 \n",
      " 171    0.30    0.99    0.50    0.00   K 174 A                 T 172 A                 \n",
      " 172    0.28    0.76    0.38    0.00   P 175 A                 G 173 A                 \n",
      " 173    0.16    0.61    0.30    0.00 * V 176 A                 V 174 A                 \n",
      " 174    0.27    0.83    0.42    0.00   A 177 A                 D 175 A                 \n",
      " 175    0.30    0.89    0.45    0.00   S 178 A                 R 176 A                 \n",
      " 176    0.20    0.83    0.42    0.00 * I 179 A                 I 177 A                 \n",
      " 177    0.36    1.21    0.61    0.00   E 180 A                 R 178 A                 \n",
      " 178    0.34    1.27    0.63    0.00   K 181 A                 R 179 A                 \n",
      " 179    0.35    1.32    0.66    0.00   L 182 A                 M 180 A                 \n",
      " 180    0.29    0.90    0.45    0.00   A 183 A                 T 181 A                 \n",
      " 181    0.21    0.86    0.43    0.00 * V 184 A                 V 182 A                 \n",
      " 182    0.10    0.25    0.13    0.00 * W 185 A                 W 183 A                 \n",
      " 183    0.07    0.16    0.08    0.00 * G 186 A                 G 184 A                 \n",
      " 184    0.52    1.33    0.66    0.00 * N 187 A                 N 185 A                 \n",
      " 185    0.45    1.06    0.53    0.00 * H 188 A                 H 186 A                 \n",
      " 186    0.20    0.72    0.36    0.00 * S 189 A                 S 187 A                 \n",
      " 187    0.32    1.00    0.50    0.00   P 190 A                 S 188 A                 \n",
      " 188    0.33    1.06    0.53    0.00   T 191 A                 I 189 A                 \n",
      " 189    0.20    0.71    0.35    0.00 * M 192 A                 M 190 A                 \n",
      " 190    0.18    0.47    0.24    0.00   Y 193 A                 F 191 A                 \n",
      " 191    0.07    0.23    0.11    0.00 * P 194 A                 P 192 A                 \n",
      " 192    0.09    0.25    0.12    0.00 * D 195 A                 D 193 A                 \n",
      " 193    0.36    1.26    0.63    0.00   F 196 A                 L 194 A                 \n",
      " 194    0.28    0.51    0.26    0.00   R 197 A                 F 195 A                 \n",
      " 195    0.34    0.95    0.48    0.00   F 198 A                 H 196 A                 \n",
      " 196    0.20    0.82    0.41    0.00 * A 199 A                 A 197 A                 \n",
      " 197    0.31    0.95    0.47    0.00   T 200 A                 E 198 A                 \n",
      " 198    0.45    1.69    0.84    0.00   A 201 A                 V 199 A                 \n",
      " 199    0.43    1.65    0.83    0.00   E 202 A                 D 200 A                 \n",
      " 200    0.48    2.27    1.14    0.00 * G 203 A                 G 205 A                 \n",
      " 201    0.38    1.23    0.61    0.00   E 204 A                 R 206 A                 \n",
      " 202    0.37    1.25    0.63    0.00   S 205 A                 P 207 A                 \n",
      " 203    0.27    0.77    0.39    0.00   L 206 A                 A 208 A                 \n",
      " 204    0.14    0.52    0.26    0.00 * L 207 A                 L 209 A                 \n",
      " 205    0.20    0.47    0.24    0.00   K 208 A                 E 210 A                 \n",
      " 206    0.30    1.20    0.60    0.00 * L 209 A                 L 211 A                 \n",
      " 207    0.35    1.24    0.62    0.00   I 210 A                 V 212 A                 \n",
      " 208    0.00    0.00    0.00    0.00   N 211 A                 -                       \n",
      " 209    0.34    1.36    0.68    0.00 * D 212 A                 D 214 A                 \n",
      " 210    0.42    0.60    0.30    0.00   D 213 A                 M 215 A                 \n",
      " 211    0.43    0.96    0.48    0.00   V 214 A                 E 216 A                 \n",
      " 212    0.32    1.01    0.51    0.00 * W 215 A                 W 217 A                 \n",
      " 213    0.44    1.04    0.52    0.00   N 216 A                 Y 218 A                 \n",
      " 214    0.44    1.08    0.54    0.00   R 217 A                 E 219 A                 \n",
      " 215    0.31    0.96    0.48    0.00   D 218 A                 K 220 A                 \n",
      " 216    0.35    1.19    0.59    0.00   T 219 A                 V 221 A                 \n",
      " 217    0.17    0.73    0.37    0.00 * F 220 A                 F 222 A                 \n",
      " 218    0.20    0.85    0.42    0.00 * I 221 A                 I 223 A                 \n",
      " 219    0.30    1.40    0.70    0.00 * P 222 A                 P 224 A                 \n",
      " 220    0.26    1.16    0.58    0.00 * T 223 A                 T 225 A                 \n",
      " 221    0.23    0.99    0.50    0.00 * V 224 A                 V 226 A                 \n",
      " 222    0.32    1.16    0.58    0.00   G 225 A                 A 227 A                 \n",
      " 223    0.38    1.42    0.71    0.00   K 226 A                 Q 228 A                 \n",
      " 224    0.25    1.08    0.54    0.00 * R 227 A                 R 229 A                 \n",
      " 225    0.22    1.02    0.51    0.00 * G 228 A                 G 230 A                 \n",
      " 226    0.25    1.09    0.54    0.00 * A 229 A                 A 231 A                 \n",
      " 227    0.27    1.21    0.61    0.00 * A 230 A                 A 232 A                 \n",
      " 228    0.20    0.84    0.42    0.00 * I 231 A                 I 233 A                 \n",
      " 229    0.15    0.59    0.29    0.00 * I 232 A                 I 234 A                 \n",
      " 230    0.25    0.76    0.38    0.00   E 233 A                 Q 235 A                 \n",
      " 231    0.18    0.74    0.37    0.00 * A 234 A                 A 236 A                 \n",
      " 232    0.18    0.72    0.36    0.00 * R 235 A                 R 237 A                 \n",
      " 233    0.20    0.90    0.45    0.00 * G 236 A                 G 238 A                 \n",
      " 234    0.22    0.53    0.26    0.00   L 237 A                 A 239 A                 \n",
      " 235    0.14    0.58    0.29    0.00 * S 238 A                 S 240 A                 \n",
      " 236    0.12    0.46    0.23    0.00 * S 239 A                 S 241 A                 \n",
      " 237    0.13    0.47    0.24    0.00 * A 240 A                 A 242 A                 \n",
      " 238    0.10    0.35    0.18    0.00 * A 241 A                 A 243 A                 \n",
      " 239    0.12    0.47    0.24    0.00 * S 242 A                 S 244 A                 \n",
      " 240    0.10    0.33    0.16    0.00 * A 243 A                 A 245 A                 \n",
      " 241    0.07    0.20    0.10    0.00 * A 244 A                 A 246 A                 \n",
      " 242    0.08    0.20    0.10    0.00 * N 245 A                 N 247 A                 \n",
      " 243    0.09    0.30    0.15    0.00 * A 246 A                 A 248 A                 \n",
      " 244    0.08    0.22    0.11    0.00 * A 247 A                 A 249 A                 \n",
      " 245    0.09    0.28    0.14    0.00 * I 248 A                 I 250 A                 \n",
      " 246    0.16    0.36    0.18    0.00   D 249 A                 E 251 A                 \n",
      " 247    0.09    0.31    0.15    0.00 * H 250 A                 H 252 A                 \n",
      " 248    0.11    0.21    0.10    0.00   V 251 A                 I 253 A                 \n",
      " 249    0.10    0.33    0.16    0.00 * R 252 A                 R 254 A                 \n",
      " 250    0.10    0.35    0.18    0.00 * D 253 A                 D 255 A                 \n",
      " 251    0.07    0.30    0.15    0.00 * W 254 A                 W 256 A                 \n",
      " 252    0.12    0.09    0.04    0.00   V 255 A                 A 257 A                 \n",
      " 253    0.11    0.38    0.19    0.00 * L 256 A                 L 258 A                 \n",
      " 254    0.20    0.85    0.43    0.00 * G 257 A                 G 259 A                 \n",
      " 255    0.16    0.54    0.27    0.00 * T 258 A                 T 260 A                 \n",
      " 256    0.68    2.44    1.22    0.00   N 259 A                 P 261 A                 \n",
      " 257    0.99    3.01    1.50    0.00   G 260 A                 E 262 A                 \n",
      " 258    0.00    0.00    0.00    0.00   -                       G 263 A                 \n",
      " 259    0.45    1.52    0.76    0.00   K 261 A                 D 264 A                 \n",
      " 260    0.15    0.52    0.26    0.00 * W 262 A                 W 265 A                 \n",
      " 261    0.08    0.20    0.10    0.00 * V 263 A                 V 266 A                 \n",
      " 262    0.12    0.25    0.12    0.00   T 264 A                 S 267 A                 \n",
      " 263    0.12    0.38    0.19    0.00 * M 265 A                 M 268 A                 \n",
      " 264    0.19    0.51    0.26    0.00   G 266 A                 A 269 A                 \n",
      " 265    0.16    0.45    0.23    0.00   I 267 A                 V 270 A                 \n",
      " 266    0.12    0.48    0.24    0.00 * P 268 A                 P 271 A                 \n",
      " 267    0.32    0.45    0.22    0.00 * S 269 A                 S 272 A                 \n",
      " 268    0.51    0.90    0.45    0.00   D 270 A                 Q 273 A                 \n",
      " 269    0.05    0.12    0.06    0.00 * G 271 A                 G 274 A                 \n",
      " 270    0.65    1.55    0.78    0.00   S 272 A                 E 275 A                 \n",
      " 271    0.25    1.06    0.53    0.00 * Y 273 A                 Y 277 A                 \n",
      " 272    0.41    1.96    0.98    0.00 * G 274 A                 G 278 A                 \n",
      " 273    0.19    0.74    0.37    0.00 * I 275 A                 I 279 A                 \n",
      " 274    0.19    0.79    0.40    0.00 * P 276 A                 P 280 A                 \n",
      " 275    0.20    0.76    0.38    0.00 * E 277 A                 E 281 A                 \n",
      " 276    0.39    1.39    0.70    0.00   D 278 A                 G 282 A                 \n",
      " 277    0.30    1.32    0.66    0.00 * I 279 A                 I 283 A                 \n",
      " 278    0.15    0.41    0.21    0.00   I 280 A                 V 284 A                 \n",
      " 279    0.08    0.26    0.13    0.00 * Y 281 A                 Y 285 A                 \n",
      " 280    0.15    0.22    0.11    0.00   G 282 A                 S 286 A                 \n",
      " 281    0.17    0.23    0.12    0.00   V 283 A                 F 287 A                 \n",
      " 282    0.08    0.27    0.14    0.00 * P 284 A                 P 288 A                 \n",
      " 283    0.15    0.55    0.28    0.00 * V 285 A                 V 289 A                 \n",
      " 284    0.23    0.53    0.27    0.00   I 286 A                 T 290 A                 \n",
      " 285    0.30    0.77    0.39    0.00   C 287 A                 A 291 A                 \n",
      " 286    0.22    0.60    0.30    0.00   E 288 A                 K 292 A                 \n",
      " 287    0.49    1.98    0.99    0.00   N 289 A                 D 293 A                 \n",
      " 288    0.25    1.16    0.58    0.00 * G 290 A                 G 294 A                 \n",
      " 289    0.28    0.80    0.40    0.00   E 291 A                 A 295 A                 \n",
      " 290    0.14    0.55    0.28    0.00 * Y 292 A                 Y 296 A                 \n",
      " 291    0.28    0.97    0.48    0.00   K 293 A                 R 297 A                 \n",
      " 292    0.32    0.89    0.44    0.00   R 294 A                 V 298 A                 \n",
      " 293    0.14    0.50    0.25    0.00 * V 295 A                 V 299 A                 \n",
      " 294    0.19    0.78    0.39    0.00 * E 296 A                 E 300 A                 \n",
      " 295    0.18    0.83    0.42    0.00 * G 297 A                 G 301 A                 \n",
      " 296    0.23    1.01    0.51    0.00 * L 298 A                 L 302 A                 \n",
      " 297    0.25    1.08    0.54    0.00 * E 299 A                 E 303 A                 \n",
      " 298    0.20    0.84    0.42    0.00 * I 300 A                 I 304 A                 \n",
      " 299    0.27    0.87    0.44    0.00   D 301 A                 N 305 A                 \n",
      " 300    0.30    0.98    0.49    0.00   A 302 A                 E 306 A                 \n",
      " 301    0.26    1.17    0.58    0.00 * F 303 A                 F 307 A                 \n",
      " 302    0.24    0.71    0.35    0.00   S 304 A                 A 308 A                 \n",
      " 303    0.17    0.68    0.34    0.00 * R 305 A                 R 309 A                 \n",
      " 304    0.22    0.63    0.32    0.00   E 306 A                 K 310 A                 \n",
      " 305    0.19    0.56    0.28    0.00   K 307 A                 R 311 A                 \n",
      " 306    0.16    0.53    0.27    0.00 * M 308 A                 M 312 A                 \n",
      " 307    0.23    0.70    0.35    0.00   D 309 A                 E 313 A                 \n",
      " 308    0.31    0.79    0.40    0.00   G 310 A                 I 314 A                 \n",
      " 309    0.21    0.88    0.44    0.00 * T 311 A                 T 315 A                 \n",
      " 310    0.31    0.97    0.49    0.00   L 312 A                 A 316 A                 \n",
      " 311    0.38    1.35    0.67    0.00   A 313 A                 Q 317 A                 \n",
      " 312    0.33    1.46    0.73    0.00 * E 314 A                 E 318 A                 \n",
      " 313    0.18    0.75    0.38    0.00 * L 315 A                 L 319 A                 \n",
      " 314    0.30    1.33    0.67    0.00 * L 316 A                 L 320 A                 \n",
      " 315    0.45    1.78    0.89    0.00   E 317 A                 D 321 A                 \n",
      " 316    0.36    1.58    0.79    0.00 * E 318 A                 E 322 A                 \n",
      " 317    0.44    1.33    0.67    0.00   R 319 A                 M 323 A                 \n",
      " 318    0.38    1.36    0.68    0.00   D 320 A                 E 324 A                 \n",
      " 319    0.45    1.53    0.77    0.00   G 321 A                 Q 325 A                 \n",
      " 320    0.36    1.46    0.73    0.00 * V 322 A                 V 326 A                 \n",
      " 321    0.57    1.49    0.74    0.00   A 323 A                 K 327 A                 \n",
      " 322    0.93    3.32    1.66    0.00   H 324 A                 A 328 A                 \n",
      " 323    0.00    0.00    0.00    0.00   -                       L 329 A                 \n",
      " 324    0.00    0.00    0.00    0.00   -                       G 330 A                 \n",
      " 325    0.00    0.00    0.00    0.00   -                       L 331 A                 \n",
      " 326    0.84    3.82    1.91    0.00   L 325 A                 I 332 A                 \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    2\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      316      316        0.8324        0.8324        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      316      316        0.8324        0.8324        0.0000\n",
      "\n",
      "Current alignment total score:        87.22\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -               M   0 A                 \n",
      "   2    0.00    0.00    0.53    0.00   K   3 A -               K   1 A                 \n",
      "   3    0.35    1.29    0.68    0.30   P   4 A S   8 A         A   2 A                 \n",
      "   4    0.31    0.79    0.44    0.09   A   5 A T   9 A         P   3 A                 \n",
      "   5    0.16    0.14    0.15    0.04   K   6 A K  10 A         V   4 A                 \n",
      "   6    0.14    0.28    0.14    0.06   R   7 A K  11 A         R   5 A                 \n",
      "   7    0.08    0.23    0.17    0.01 * V   8 A V  12 A         V   6 A                 \n",
      "   8    0.15    0.25    0.17    0.02   A   9 A T  13 A         A   7 A                 \n",
      "   9    0.13    0.50    0.24    0.07 * V  10 A V  14 A         V   8 A                 \n",
      "  10    0.07    0.23    0.15    0.00 * T  11 A T  15 A         T   9 A                 \n",
      "  11    0.07    0.27    0.19    0.06 * G  12 A G  16 A         G  10 A                 \n",
      "  12    0.09    0.31    0.35    0.11 * A  13 A A  17 A         A  11 A                 \n",
      "  13    0.10    0.36    0.34    0.08 * A  14 A A  18 A         A  12 A                 \n",
      "  14    0.06    0.25    0.16    0.03 * G  15 A G  19 A         G  13 A                 \n",
      "  15    0.09    0.26    0.18    0.04 * Q  16 A Q  20 A         Q  14 A                 \n",
      "  16    0.07    0.21    0.16    0.04 * I  17 A I  21 A         I  15 A                 \n",
      "  17    0.18    0.42    0.21    0.07   A  18 A S  22 A         G  16 A                 \n",
      "  18    0.08    0.29    0.14    0.04 * Y  19 A Y  23 A         Y  17 A                 \n",
      "  19    0.11    0.39    0.20    0.05 * S  20 A S  24 A         S  18 A                 \n",
      "  20    0.09    0.33    0.18    0.03 * L  21 A L  25 A         L  19 A                 \n",
      "  21    0.10    0.40    0.21    0.05 * L  22 A L  26 A         L  20 A                 \n",
      "  22    0.27    0.77    0.35    0.12   F  23 A W  27 A         F  21 A                 \n",
      "  23    0.24    1.06    0.47    0.16 * R  24 A R  28 A         R  22 A                 \n",
      "  24    0.20    0.86    0.41    0.12 * I  25 A I  29 A         I  23 A                 \n",
      "  25    0.14    0.55    0.27    0.10 * A  26 A A  30 A         A  24 A                 \n",
      "  26    0.24    0.79    0.35    0.12   N  27 A N  31 A         A  25 A                 \n",
      "  27    0.19    0.91    0.42    0.13 * G  28 A G  32 A         G  26 A                 \n",
      "  28    0.28    1.13    0.56    0.14   D  29 A E  33 A         E  27 A                 \n",
      "  29    0.32    1.15    0.63    0.13   L  30 A V  34 A         M  28 A                 \n",
      "  30    0.33    1.21    0.59    0.24   L  31 A F  35 A         L  29 A                 \n",
      "  31    0.26    1.25    0.58    0.18 * G  32 A G  36 A         G  30 A                 \n",
      "  32    0.30    1.02    0.46    0.22   K  33 A T  37 A         K  31 A                 \n",
      "  33    0.25    1.13    0.54    0.18 * D  34 A D  38 A         D  32 A                 \n",
      "  34    0.49    0.88    0.41    0.16   Q  35 A T  39 A         Q  33 A                 \n",
      "  35    0.20    0.92    0.44    0.15 * P  36 A P  40 A         P  34 A                 \n",
      "  36    0.12    0.49    0.24    0.10 * V  37 A V  41 A         V  35 A                 \n",
      "  37    0.22    0.37    0.27    0.05   I  38 A E  42 A         I  36 A                 \n",
      "  38    0.06    0.20    0.15    0.05 * L  39 A L  43 A         L  37 A                 \n",
      "  39    0.15    0.28    0.14    0.07   Q  40 A K  44 A         Q  38 A                 \n",
      "  40    0.12    0.51    0.31    0.02 * L  41 A L  45 A         L  39 A                 \n",
      "  41    0.04    0.08    0.14    0.06 * L  42 A L  46 A         L  40 A                 \n",
      "  42    0.13    0.37    0.28    0.04   D  43 A E  47 A         E  41 A                 \n",
      "  43    0.25    0.47    0.31    0.04   L  44 A I  48 A         I  42 A                 \n",
      "  44    0.18    0.28    0.28    0.07 * P  45 A P  49 A         P  43 A                 \n",
      "  45    0.11    0.39    0.45    0.16 * Q  46 A Q  50 A         Q  44 A                 \n",
      "  46    0.07    0.17    0.29    0.13 * A  47 A A  51 A         A  45 A                 \n",
      "  47    0.17    0.32    0.32    0.08   Q  48 A L  52 A         M  46 A                 \n",
      "  48    0.16    0.29    0.34    0.10   A  49 A G  53 A         K  47 A                 \n",
      "  49    0.20    0.57    0.35    0.04   A  50 A G  54 A         A  48 A                 \n",
      "  50    0.22    0.58    0.33    0.09   V  51 A A  55 A         L  49 A                 \n",
      "  51    0.16    0.48    0.30    0.03   K  52 A E  56 A         E  50 A                 \n",
      "  52    0.10    0.46    0.21    0.07 * G  53 A G  57 A         G  51 A                 \n",
      "  53    0.14    0.56    0.26    0.12 * V  54 A V  58 A         V  52 A                 \n",
      "  54    0.24    0.75    0.40    0.13   V  55 A A  59 A         V  53 A                 \n",
      "  55    0.20    0.74    0.35    0.13 * M  56 A M  60 A         M  54 A                 \n",
      "  56    0.15    0.58    0.29    0.09 * E  57 A E  61 A         E  55 A                 \n",
      "  57    0.13    0.53    0.26    0.15 * L  58 A L  62 A         L  56 A                 \n",
      "  58    0.32    0.88    0.41    0.18   D  59 A L  63 A         E  57 A                 \n",
      "  59    0.22    0.97    0.46    0.19 * D  60 A D  64 A         D  58 A                 \n",
      "  60    0.30    0.69    0.33    0.13   C  61 A S  65 A         C  59 A                 \n",
      "  61    0.20    0.82    0.38    0.19 * A  62 A A  66 A         A  60 A                 \n",
      "  62    0.13    0.53    0.26    0.08 * F  63 A F  67 A         F  61 A                 \n",
      "  63    0.11    0.43    0.23    0.06 * P  64 A P  68 A         P  62 A                 \n",
      "  64    0.14    0.60    0.28    0.09 * L  65 A L  69 A         L  63 A                 \n",
      "  65    0.09    0.31    0.16    0.04 * L  66 A L  70 A         L  64 A                 \n",
      "  66    0.28    0.76    0.44    0.18   A  67 A R  71 A         A  65 A                 \n",
      "  67    0.21    0.53    0.40    0.10   G  68 A N  72 A         G  66 A                 \n",
      "  68    0.17    0.49    0.32    0.06   V  69 A I  73 A         L  67 A                 \n",
      "  69    0.19    0.41    0.38    0.07   V  70 A T  74 A         E  68 A                 \n",
      "  70    0.16    0.45    0.32    0.06   I  71 A I  75 A         A  69 A                 \n",
      "  71    0.10    0.36    0.26    0.09 * T  72 A T  76 A         T  70 A                 \n",
      "  72    0.28    0.36    0.21    0.06   D  73 A A  77 A         D  71 A                 \n",
      "  73    0.13    0.51    0.27    0.06 * D  74 A D  78 A         D  72 A                 \n",
      "  74    0.27    0.34    0.22    0.03   P  75 A A  79 A         P  73 A                 \n",
      "  75    0.14    0.25    0.22    0.05   K  76 A N  80 A         D  74 A                 \n",
      "  76    0.19    0.29    0.22    0.03   V  77 A E  81 A         V  75 A                 \n",
      "  77    0.07    0.18    0.26    0.10 * A  78 A A  82 A         A  76 A                 \n",
      "  78    0.06    0.17    0.12    0.01 * F  79 A F  83 A         F  77 A                 \n",
      "  79    0.20    0.43    0.20    0.07   K  80 A D  84 A         K  78 A                 \n",
      "  80    0.17    0.32    0.19    0.05   D  81 A G  85 A         D  79 A                 \n",
      "  81    0.08    0.23    0.21    0.05 * A  82 A A  86 A         A  80 A                 \n",
      "  82    0.35    0.31    0.17    0.03   D  83 A N  87 A         D  81 A                 \n",
      "  83    0.15    0.19    0.14    0.02   V  84 A A  88 A         Y  82 A                 \n",
      "  84    0.17    0.65    0.29    0.10 * A  85 A A  89 A         A  83 A                 \n",
      "  85    0.21    0.60    0.28    0.08   L  86 A F  90 A         L  84 A                 \n",
      "  86    0.09    0.34    0.17    0.06 * L  87 A L  91 A         L  85 A                 \n",
      "  87    0.13    0.51    0.25    0.07 * V  88 A V  92 A         V  86 A                 \n",
      "  88    0.33    0.59    0.28    0.09 * G  89 A G  93 A         G  87 A                 \n",
      "  89    0.45    1.05    0.47    0.24 * A  90 A A  94 A         A  88 A                 \n",
      "  90    0.72    2.28    1.05    0.47   R  91 A K  95 A         A  89 A                 \n",
      "  91    0.54    2.15    1.36    0.41 * P  92 A P  96 A         P  90 A                 \n",
      "  92    0.00    0.00    0.00    0.00   R  93 A -                                       \n",
      "  93    0.00    0.00    0.00    0.00   S  94 A -               -                       \n",
      "  94    0.00    0.00    0.00    0.00   M  97 A -               -                       \n",
      "  95    0.70    2.40    1.20    0.00   E  98 A E 102 A         -                       \n",
      "  96    0.59    1.89    0.95    0.00   R  99 A R 103 A         -                       \n",
      "  97    0.61    2.25    1.13    0.00   K 100 A A 104 A         -                       \n",
      "  98    0.52    1.97    0.98    0.00   D 101 A D 105 A         -                       \n",
      "  99    0.35    1.13    0.56    0.00   L 102 A L 106 A         -                       \n",
      " 100    0.53    1.93    1.08    0.51 * L 103 A L 107 A         L 101 A                 \n",
      " 101    0.47    1.85    1.06    0.49   S 104 A A 108 A         Q 102 A                 \n",
      " 102    0.57    2.17    0.99    0.45   A 105 A N 109 A         V 103 A                 \n",
      " 103    0.41    1.81    0.81    0.30 * N 106 A N 110 A         N 104 A                 \n",
      " 104    0.50    2.20    0.99    0.41   A 107 A G 111 A         G 105 A                 \n",
      " 105    0.40    1.65    0.74    0.34   E 108 A K 112 A         K 106 A                 \n",
      " 106    0.17    0.67    0.30    0.12 * I 109 A I 113 A         I 107 A                 \n",
      " 107    0.18    0.73    0.34    0.13 * F 110 A F 114 A         F 108 A                 \n",
      " 108    0.38    0.77    0.36    0.17   T 111 A G 115 A         T 109 A                 \n",
      " 109    0.22    0.48    0.30    0.04   V 112 A P 116 A         E 110 A                 \n",
      " 110    0.12    0.43    0.20    0.07 * Q 113 A Q 117 A         Q 111 A                 \n",
      " 111    0.10    0.46    0.23    0.10 * G 114 A G 118 A         G 112 A                 \n",
      " 112    0.18    0.47    0.28    0.04   A 115 A K 119 A         R 113 A                 \n",
      " 113    0.13    0.49    0.26    0.05 * A 116 A A 120 A         A 114 A                 \n",
      " 114    0.17    0.46    0.24    0.06   L 117 A I 121 A         L 115 A                 \n",
      " 115    0.13    0.28    0.23    0.07   N 118 A N 122 A         A 116 A                 \n",
      " 116    0.14    0.25    0.26    0.08   E 119 A D 123 A         E 117 A                 \n",
      " 117    0.21    0.31    0.24    0.02   V 120 A N 124 A         V 118 A                 \n",
      " 118    0.12    0.34    0.24    0.01 * A 121 A A 125 A         A 119 A                 \n",
      " 119    0.27    0.85    0.44    0.12   S 122 A A 126 A         K 120 A                 \n",
      " 120    0.50    1.87    0.85    0.31   R 123 A D 127 A         K 121 A                 \n",
      " 121    0.40    1.82    0.84    0.42 * D 124 A D 128 A         D 122 A                 \n",
      " 122    0.13    0.29    0.23    0.03   V 125 A I 129 A         V 123 A                 \n",
      " 123    0.11    0.16    0.18    0.06   K 126 A R 130 A         K 124 A                 \n",
      " 124    0.12    0.47    0.22    0.08 * V 127 A V 131 A         V 125 A                 \n",
      " 125    0.10    0.39    0.17    0.07 * L 128 A L 132 A         L 126 A                 \n",
      " 126    0.07    0.21    0.12    0.05 * V 129 A V 133 A         V 127 A                 \n",
      " 127    0.08    0.26    0.17    0.05 * V 130 A V 134 A         V 128 A                 \n",
      " 128    0.05    0.19    0.16    0.02 * G 131 A G 135 A         G 129 A                 \n",
      " 129    0.07    0.12    0.28    0.14 * N 132 A N 136 A         N 130 A                 \n",
      " 130    0.14    0.11    0.15    0.06 * P 133 A P 137 A         P 131 A                 \n",
      " 131    0.08    0.26    0.23    0.08 * A 134 A A 138 A         A 132 A                 \n",
      " 132    0.10    0.31    0.19    0.04 * N 135 A N 139 A         N 133 A                 \n",
      " 133    0.07    0.19    0.14    0.04 * T 136 A T 140 A         T 134 A                 \n",
      " 134    0.11    0.37    0.21    0.07 * N 137 A N 141 A         N 135 A                 \n",
      " 135    0.10    0.34    0.25    0.04 * A 138 A A 142 A         A 136 A                 \n",
      " 136    0.11    0.18    0.25    0.10   Y 139 A L 143 A         L 137 A                 \n",
      " 137    0.10    0.34    0.24    0.06 * I 140 A I 144 A         I 138 A                 \n",
      " 138    0.13    0.49    0.30    0.06 * A 141 A A 145 A         A 139 A                 \n",
      " 139    0.19    0.27    0.28    0.07   M 142 A S 146 A         Y 140 A                 \n",
      " 140    0.20    0.50    0.29    0.03   K 143 A A 147 A         K 141 A                 \n",
      " 141    0.22    0.58    0.38    0.02   S 144 A A 148 A         N 142 A                 \n",
      " 142    0.10    0.32    0.35    0.11 * A 145 A A 149 A         A 143 A                 \n",
      " 143    0.14    0.61    0.85    0.35 * P 146 A P 150 A         P 144 A                 \n",
      " 144    0.12    0.22    0.24    0.08   D 147 A D 151 A         G 145 A                 \n",
      " 145    0.18    0.49    0.38    0.04   L 148 A V 152 A         L 146 A                 \n",
      " 146    0.23    0.77    0.46    0.09   P 149 A P 153 A         N 147 A                 \n",
      " 147    0.28    0.87    0.48    0.11   K 150 A A 154 A         P 148 A                 \n",
      " 148    0.26    0.74    0.44    0.05   K 151 A S 155 A         R 149 A                 \n",
      " 149    0.25    0.61    0.29    0.09   N 152 A R 156 A         N 150 A                 \n",
      " 150    0.13    0.50    0.24    0.10 * F 153 A F 157 A         F 151 A                 \n",
      " 151    0.18    0.40    0.19    0.07   T 154 A N 158 A         T 152 A                 \n",
      " 152    0.09    0.26    0.13    0.03 * A 155 A A 159 A         A 153 A                 \n",
      " 153    0.10    0.27    0.19    0.04 * M 156 A M 160 A         M 154 A                 \n",
      " 154    0.17    0.33    0.27    0.07   L 157 A M 161 A         T 155 A                 \n",
      " 155    0.10    0.34    0.20    0.06 * R 158 A R 162 A         R 156 A                 \n",
      " 156    0.10    0.39    0.26    0.09 * L 159 A L 163 A         L 157 A                 \n",
      " 157    0.07    0.20    0.26    0.08 * D 160 A D 164 A         D 158 A                 \n",
      " 158    0.08    0.25    0.19    0.05 * H 161 A H 165 A         H 159 A                 \n",
      " 159    0.11    0.37    0.22    0.05 * N 162 A N 166 A         N 160 A                 \n",
      " 160    0.07    0.20    0.24    0.07 * R 163 A R 167 A         R 161 A                 \n",
      " 161    0.07    0.19    0.16    0.05 * A 164 A A 168 A         A 162 A                 \n",
      " 162    0.13    0.12    0.10    0.03   L 165 A I 169 A         K 163 A                 \n",
      " 163    0.15    0.46    0.26    0.08   S 166 A S 170 A         A 164 A                 \n",
      " 164    0.16    0.62    0.40    0.06 * Q 167 A Q 171 A         Q 165 A                 \n",
      " 165    0.13    0.55    0.32    0.05 * L 168 A L 172 A         L 166 A                 \n",
      " 166    0.21    0.87    0.45    0.10 * A 169 A A 173 A         A 167 A                 \n",
      " 167    0.31    1.07    0.54    0.17   A 170 A T 174 A         K 168 A                 \n",
      " 168    0.22    0.95    0.52    0.15 * K 171 A K 175 A         K 169 A                 \n",
      " 169    0.37    1.21    0.60    0.25   S 172 A L 176 A         T 170 A                 \n",
      " 170    0.27    1.26    0.60    0.23 * G 173 A G 177 A         G 171 A                 \n",
      " 171    0.44    1.17    0.65    0.21   K 174 A R 178 A         T 172 A                 \n",
      " 172    0.28    1.05    0.57    0.13   P 175 A G 179 A         G 173 A                 \n",
      " 173    0.31    0.95    0.50    0.12   V 176 A S 180 A         V 174 A                 \n",
      " 174    0.24    0.85    0.51    0.12   A 177 A A 181 A         D 175 A                 \n",
      " 175    0.23    0.55    0.43    0.12   S 178 A E 182 A         R 176 A                 \n",
      " 176    0.23    0.59    0.44    0.04   I 179 A F 183 A         I 177 A                 \n",
      " 177    0.26    0.74    0.59    0.12   E 180 A N 184 A         R 178 A                 \n",
      " 178    0.24    0.66    0.59    0.12   K 181 A N 185 A         R 179 A                 \n",
      " 179    0.16    0.38    0.53    0.20   L 182 A I 186 A         M 180 A                 \n",
      " 180    0.20    0.51    0.43    0.10   A 183 A V 187 A         T 181 A                 \n",
      " 181    0.13    0.50    0.40    0.14 * V 184 A V 188 A         V 182 A                 \n",
      " 182    0.04    0.09    0.11    0.03 * W 185 A W 189 A         W 183 A                 \n",
      " 183    0.08    0.32    0.16    0.05 * G 186 A G 190 A         G 184 A                 \n",
      " 184    0.34    0.97    0.66    0.25 * N 187 A N 191 A         N 185 A                 \n",
      " 185    0.18    0.73    0.54    0.10 * H 188 A H 192 A         H 186 A                 \n",
      " 186    0.17    0.69    0.42    0.12 * S 189 A S 193 A         S 187 A                 \n",
      " 187    0.21    0.55    0.46    0.14   P 190 A A 194 A         S 188 A                 \n",
      " 188    0.21    0.69    0.54    0.06   T 191 A T 195 A         I 189 A                 \n",
      " 189    0.21    0.35    0.31    0.11   M 192 A Q 196 A         M 190 A                 \n",
      " 190    0.08    0.13    0.19    0.08   Y 193 A F 197 A         F 191 A                 \n",
      " 191    0.06    0.18    0.13    0.01 * P 194 A P 198 A         P 192 A                 \n",
      " 192    0.10    0.38    0.20    0.05 * D 195 A D 199 A         D 193 A                 \n",
      " 193    0.15    0.27    0.48    0.22   F 196 A I 200 A         L 194 A                 \n",
      " 194    0.29    0.78    0.39    0.17   R 197 A T 201 A         F 195 A                 \n",
      " 195    0.19    0.46    0.44    0.09   F 198 A Y 202 A         H 196 A                 \n",
      " 196    0.12    0.40    0.38    0.08 * A 199 A A 203 A         A 197 A                 \n",
      " 197    0.19    0.54    0.46    0.07   T 200 A T 204 A         E 198 A                 \n",
      " 198    0.17    0.51    0.68    0.25   A 201 A V 205 A         V 199 A                 \n",
      " 199    0.51    1.96    1.12    0.24   E 202 A G 206 A         D 200 A                 \n",
      " 200    0.30    1.43    1.14    0.15 * G 203 A G 207 A         G 205 A                 \n",
      " 201    0.30    1.11    0.72    0.07   E 204 A E 208 A         R 206 A                 \n",
      " 202    0.26    0.75    0.61    0.11   S 205 A K 209 A         P 207 A                 \n",
      " 203    0.18    0.48    0.38    0.07   L 206 A V 210 A         A 208 A                 \n",
      " 204    0.30    0.84    0.42    0.16   L 207 A T 211 A         L 209 A                 \n",
      " 205    0.33    1.12    0.52    0.20   K 208 A D 212 A         E 210 A                 \n",
      " 206    0.34    1.42    0.73    0.39 * L 209 A L 213 A         L 211 A                 \n",
      " 207    0.38    1.53    0.79    0.39   I 210 A V 214 A         V 212 A                 \n",
      " 208    0.00    0.00    0.00    0.00   N 211 A -               -                       \n",
      " 209    0.23    0.94    0.68    0.21 * D 212 A D 215 A         D 214 A                 \n",
      " 210    0.22    0.11    0.23    0.11   D 213 A H 216 A         M 215 A                 \n",
      " 211    0.37    1.01    0.60    0.16   V 214 A D 217 A         E 216 A                 \n",
      " 212    0.27    1.02    0.60    0.19 * W 215 A W 218 A         W 217 A                 \n",
      " 213    0.24    0.56    0.49    0.09   N 216 A Y 219 A         Y 218 A                 \n",
      " 214    0.35    0.83    0.59    0.05   R 217 A V 220 A         E 219 A                 \n",
      " 215    0.34    1.24    0.66    0.25   D 218 A E 221 A         K 220 A                 \n",
      " 216    0.30    0.90    0.61    0.19   T 219 A E 222 A         V 221 A                 \n",
      " 217    0.12    0.46    0.36    0.10 * F 220 A F 223 A         F 222 A                 \n",
      " 218    0.10    0.37    0.37    0.12 * I 221 A I 224 A         I 223 A                 \n",
      " 219    0.17    0.76    0.64    0.22 * P 222 A P 225 A         P 224 A                 \n",
      " 220    0.23    0.54    0.51    0.18   T 223 A R 226 A         T 225 A                 \n",
      " 221    0.10    0.35    0.41    0.15 * V 224 A V 227 A         V 226 A                 \n",
      " 222    0.26    1.00    0.64    0.19   G 225 A A 228 A         A 227 A                 \n",
      " 223    0.37    1.28    0.77    0.33   K 226 A N 229 A         Q 228 A                 \n",
      " 224    0.17    0.69    0.52    0.18 * R 227 A R 230 A         R 229 A                 \n",
      " 225    0.18    0.82    0.54    0.18 * G 228 A G 231 A         G 230 A                 \n",
      " 226    0.26    1.11    0.65    0.21 * A 229 A A 232 A         A 231 A                 \n",
      " 227    0.29    0.95    0.62    0.25   A 230 A E 233 A         A 232 A                 \n",
      " 228    0.22    0.96    0.50    0.28 * I 231 A I 234 A         I 233 A                 \n",
      " 229    0.26    1.14    0.52    0.27 * I 232 A I 235 A         I 234 A                 \n",
      " 230    0.23    0.82    0.46    0.17   E 233 A E 236 A         Q 235 A                 \n",
      " 231    0.23    0.67    0.41    0.15   A 234 A V 237 A         A 236 A                 \n",
      " 232    0.23    1.01    0.52    0.21 * R 235 A R 238 A         R 237 A                 \n",
      " 233    0.20    0.93    0.52    0.25 * G 236 A G 239 A         G 238 A                 \n",
      " 234    0.27    0.76    0.38    0.17   L 237 A K 240 A         A 239 A                 \n",
      " 235    0.23    1.02    0.51    0.15 * S 238 A S 241 A         S 240 A                 \n",
      " 236    0.29    0.77    0.38    0.15 * S 239 A S 242 A         S 241 A                 \n",
      " 237    0.12    0.37    0.26    0.04 * A 240 A A 243 A         A 242 A                 \n",
      " 238    0.11    0.39    0.22    0.07 * A 241 A A 244 A         A 243 A                 \n",
      " 239    0.09    0.32    0.24    0.04 * S 242 A S 245 A         S 244 A                 \n",
      " 240    0.11    0.39    0.23    0.03 * A 243 A A 246 A         A 245 A                 \n",
      " 241    0.09    0.28    0.15    0.04 * A 244 A A 247 A         A 246 A                 \n",
      " 242    0.17    0.43    0.20    0.08   N 245 A S 248 A         N 247 A                 \n",
      " 243    0.16    0.37    0.20    0.08   A 246 A S 249 A         A 248 A                 \n",
      " 244    0.10    0.33    0.17    0.06 * A 247 A A 250 A         A 249 A                 \n",
      " 245    0.12    0.47    0.24    0.05 * I 248 A I 251 A         I 250 A                 \n",
      " 246    0.14    0.42    0.25    0.03   D 249 A D 252 A         E 251 A                 \n",
      " 247    0.10    0.35    0.20    0.02 * H 250 A H 253 A         H 252 A                 \n",
      " 248    0.20    0.46    0.22    0.07   V 251 A M 254 A         I 253 A                 \n",
      " 249    0.11    0.41    0.22    0.08 * R 252 A R 255 A         R 254 A                 \n",
      " 250    0.09    0.32    0.20    0.06 * D 253 A D 256 A         D 255 A                 \n",
      " 251    0.14    0.63    0.28    0.15 * W 254 A W 257 A         W 256 A                 \n",
      " 252    0.15    0.45    0.20    0.07   V 255 A V 258 A         A 257 A                 \n",
      " 253    0.19    0.32    0.21    0.05   L 256 A Q 259 A         L 258 A                 \n",
      " 254    0.11    0.39    0.38    0.10 * G 257 A G 260 A         G 259 A                 \n",
      " 255    0.12    0.37    0.26    0.10 * T 258 A T 261 A         T 260 A                 \n",
      " 256    0.65    2.56    1.56    0.11   N 259 A E 262 A         P 261 A                 \n",
      " 257    0.00    0.00    1.50    0.00   G 260 A -               E 262 A                 \n",
      " 258    0.00    0.00    0.00    0.00   -       -               G 263 A                 \n",
      " 259    0.73    1.96    1.08    0.29   K 261 A A 263 A         D 264 A                 \n",
      " 260    0.29    1.26    0.61    0.18 * W 262 A W 264 A         W 265 A                 \n",
      " 261    0.35    1.01    0.45    0.17   V 263 A S 265 A         V 266 A                 \n",
      " 262    0.16    0.44    0.21    0.09   T 264 A S 266 A         S 267 A                 \n",
      " 263    0.25    0.53    0.27    0.12   M 265 A A 267 A         M 268 A                 \n",
      " 264    0.14    0.35    0.26    0.03   G 266 A A 268 A         A 269 A                 \n",
      " 265    0.13    0.40    0.25    0.08   I 267 A I 269 A         V 270 A                 \n",
      " 266    0.10    0.39    0.25    0.10 * P 268 A P 270 A         P 271 A                 \n",
      " 267    0.21    0.41    0.26    0.01 * S 269 A S 271 A         S 272 A                 \n",
      " 268    0.32    0.51    0.42    0.11   D 270 A T 272 A         Q 273 A                 \n",
      " 269    0.08    0.34    0.16    0.06 * G 271 A G 273 A         G 274 A                 \n",
      " 270    0.18    0.39    0.61    0.25   S 272 A A 274 A         E 275 A                 \n",
      " 271    0.12    0.47    0.47    0.13 * Y 273 A Y 275 A         Y 277 A                 \n",
      " 272    0.13    0.61    0.80    0.30 * G 274 A G 276 A         G 278 A                 \n",
      " 273    0.12    0.44    0.35    0.10 * I 275 A I 277 A         I 279 A                 \n",
      " 274    0.13    0.51    0.39    0.08 * P 276 A P 278 A         P 280 A                 \n",
      " 275    0.16    0.59    0.41    0.10 * E 277 A E 279 A         E 281 A                 \n",
      " 276    0.64    1.83    0.90    0.51   D 278 A G 280 A         G 282 A                 \n",
      " 277    0.18    0.70    0.61    0.17 * I 279 A I 281 A         I 283 A                 \n",
      " 278    0.22    0.46    0.26    0.08   I 280 A F 282 A         V 284 A                 \n",
      " 279    0.25    0.60    0.29    0.09   Y 281 A V 283 A         Y 285 A                 \n",
      " 280    0.18    0.61    0.28    0.10   G 282 A G 284 A         S 286 A                 \n",
      " 281    0.33    0.20    0.13    0.00   V 283 A L 285 A         F 287 A                 \n",
      " 282    0.23    0.98    0.44    0.19 * P 284 A P 286 A         P 288 A                 \n",
      " 283    0.25    0.67    0.38    0.07   V 285 A T 287 A         V 289 A                 \n",
      " 284    0.50    1.07    0.52    0.17   I 286 A V 288 A         T 290 A                 \n",
      " 285    0.24    0.56    0.41    0.04   C 287 A S 289 A         A 291 A                 \n",
      " 286    0.37    1.34    0.66    0.17   E 288 A R 290 A         K 292 A                 \n",
      " 287    0.37    1.54    1.04    0.32   N 289 A N 291 A         D 293 A                 \n",
      " 288    0.11    0.49    0.51    0.13 * G 290 A G 292 A         G 294 A                 \n",
      " 289    0.30    1.08    0.60    0.10   E 291 A E 293 A         A 295 A                 \n",
      " 290    0.38    1.30    0.62    0.21   Y 292 A W 294 A         Y 296 A                 \n",
      " 291    0.25    0.69    0.51    0.04   K 293 A E 295 A         R 297 A                 \n",
      " 292    0.16    0.26    0.36    0.13   R 294 A I 296 A         V 298 A                 \n",
      " 293    0.14    0.54    0.31    0.11 * V 295 A V 297 A         V 299 A                 \n",
      " 294    0.28    1.22    0.62    0.22 * E 296 A E 298 A         E 300 A                 \n",
      " 295    0.31    1.46    0.72    0.26 * G 297 A G 299 A         G 301 A                 \n",
      " 296    0.22    0.93    0.60    0.07 * L 298 A L 300 A         L 302 A                 \n",
      " 297    0.25    1.05    0.66    0.04 * E 299 A E 301 A         E 303 A                 \n",
      " 298    0.20    0.83    0.52    0.02 * I 300 A I 302 A         I 304 A                 \n",
      " 299    0.20    0.51    0.42    0.08   D 301 A S 303 A         N 305 A                 \n",
      " 300    0.29    0.96    0.60    0.06   A 302 A D 304 A         E 306 A                 \n",
      " 301    0.23    1.01    0.66    0.15 * F 303 A F 305 A         F 307 A                 \n",
      " 302    0.32    1.02    0.53    0.19   S 304 A Q 306 A         A 308 A                 \n",
      " 303    0.22    0.94    0.48    0.20 * R 305 A R 307 A         R 309 A                 \n",
      " 304    0.21    0.51    0.33    0.12   E 306 A A 308 A         K 310 A                 \n",
      " 305    0.18    0.61    0.33    0.17   K 307 A R 309 A         R 311 A                 \n",
      " 306    0.23    0.66    0.32    0.20   M 308 A I 310 A         M 312 A                 \n",
      " 307    0.23    0.86    0.44    0.22   D 309 A D 311 A         E 313 A                 \n",
      " 308    0.26    0.77    0.46    0.17   G 310 A A 312 A         I 314 A                 \n",
      " 309    0.32    1.11    0.60    0.21   T 311 A N 313 A         T 315 A                 \n",
      " 310    0.28    1.06    0.56    0.30   L 312 A A 314 A         A 316 A                 \n",
      " 311    0.25    0.91    0.66    0.24   A 313 A Q 315 A         Q 317 A                 \n",
      " 312    0.21    0.88    0.71    0.17 * E 314 A E 316 A         E 318 A                 \n",
      " 313    0.16    0.71    0.43    0.13 * L 315 A L 317 A         L 319 A                 \n",
      " 314    0.21    0.41    0.54    0.21   L 316 A Q 318 A         L 320 A                 \n",
      " 315    0.19    0.44    0.70    0.29   E 317 A A 319 A         D 321 A                 \n",
      " 316    0.15    0.59    0.66    0.23 * E 318 A E 320 A         E 322 A                 \n",
      " 317    0.17    0.37    0.53    0.22   R 319 A R 321 A         M 323 A                 \n",
      " 318    0.20    0.68    0.61    0.21   D 320 A E 322 A         E 324 A                 \n",
      " 319    0.31    1.03    0.75    0.26   G 321 A A 323 A         Q 325 A                 \n",
      " 320    0.24    0.94    0.72    0.20 * V 322 A V 324 A         V 326 A                 \n",
      " 321    0.35    0.85    0.69    0.24   A 323 A R 325 A         K 327 A                 \n",
      " 322    0.81    3.16    1.84    0.82   H 324 A D 326 A         A 328 A                 \n",
      " 323    1.36    5.56    2.78    0.00   -       L 327 A         L 329 A                 \n",
      " 324    1.58    5.94    2.97    0.00   -       L 328 A         G 330 A                 \n",
      " 325    1.39    5.30    2.65    0.00   -       L 329 A         L 331 A                 \n",
      " 326    1.36    6.09    3.16    0.84   L 325 A E 330 A         I 332 A                 \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    3\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      308      308        0.9725        0.9725        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      308      308        0.9725        0.9725        0.0000\n",
      "\n",
      "Current alignment total score:        104.9\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -               M   0 A         -       \n",
      "   2    0.81    3.00    1.40    0.48   K   3 A -               K   1 A         M   1 A \n",
      "   3    0.54    2.30    1.12    0.38   P   4 A S   8 A         A   2 A         S   2 A \n",
      "   4    0.43    0.88    0.52    0.15   A   5 A T   9 A         P   3 A         A   3 A \n",
      "   5    0.18    0.23    0.16    0.04   K   6 A K  10 A         V   4 A         V   4 A \n",
      "   6    0.09    0.12    0.13    0.06   R   7 A K  11 A         R   5 A         K   5 A \n",
      "   7    0.08    0.24    0.18    0.03 * V   8 A V  12 A         V   6 A         V   6 A \n",
      "   8    0.15    0.49    0.25    0.07   A   9 A T  13 A         A   7 A         A   7 A \n",
      "   9    0.14    0.55    0.31    0.07 * V  10 A V  14 A         V   8 A         V   8 A \n",
      "  10    0.14    0.56    0.25    0.11 * T  11 A T  15 A         T   9 A         T   9 A \n",
      "  11    0.14    0.65    0.30    0.13 * G  12 A G  16 A         G  10 A         G  10 A \n",
      "  12    0.15    0.58    0.40    0.05 * A  13 A A  17 A         A  11 A         A  11 A \n",
      "  13    0.11    0.39    0.34    0.03 * A  14 A A  18 A         A  12 A         A  12 A \n",
      "  14    0.08    0.35    0.19    0.08 * G  15 A G  19 A         G  13 A         G  13 A \n",
      "  15    0.13    0.50    0.25    0.10 * Q  16 A Q  20 A         Q  14 A         Q  14 A \n",
      "  16    0.09    0.30    0.18    0.08 * I  17 A I  21 A         I  15 A         I  15 A \n",
      "  17    0.10    0.17    0.20    0.07   A  18 A S  22 A         G  16 A         G  16 A \n",
      "  18    0.08    0.30    0.18    0.04 * Y  19 A Y  23 A         Y  17 A         Y  17 A \n",
      "  19    0.12    0.19    0.19    0.03   S  20 A S  24 A         S  18 A         A  18 A \n",
      "  20    0.07    0.26    0.19    0.01 * L  21 A L  25 A         L  19 A         L  19 A \n",
      "  21    0.12    0.22    0.21    0.04   L  22 A L  26 A         L  20 A         V  20 A \n",
      "  22    0.24    0.38    0.36    0.05   F  23 A W  27 A         F  21 A         P  21 A \n",
      "  23    0.22    0.47    0.47    0.09   R  24 A R  28 A         R  22 A         L  22 A \n",
      "  24    0.08    0.27    0.37    0.12 * I  25 A I  29 A         I  23 A         I  23 A \n",
      "  25    0.08    0.25    0.26    0.09 * A  26 A A  30 A         A  24 A         A  24 A \n",
      "  26    0.16    0.22    0.32    0.12   N  27 A N  31 A         A  25 A         R  25 A \n",
      "  27    0.08    0.33    0.38    0.15 * G  28 A G  32 A         G  26 A         G  26 A \n",
      "  28    0.19    0.45    0.51    0.15   D  29 A E  33 A         E  27 A         A  27 A \n",
      "  29    0.14    0.42    0.55    0.19   L  30 A V  34 A         M  28 A         L  28 A \n",
      "  30    0.11    0.34    0.51    0.25   L  31 A F  35 A         L  29 A         L  29 A \n",
      "  31    0.05    0.20    0.47    0.26 * G  32 A G  36 A         G  30 A         G  30 A \n",
      "  32    0.21    0.45    0.42    0.24   K  33 A T  37 A         K  31 A         P  31 A \n",
      "  33    0.23    0.58    0.51    0.23   D  34 A D  38 A         D  32 A         T  32 A \n",
      "  34    0.39    0.51    0.41    0.17   Q  35 A T  39 A         Q  33 A         T  33 A \n",
      "  35    0.07    0.28    0.39    0.15 * P  36 A P  40 A         P  34 A         P  34 A \n",
      "  36    0.09    0.31    0.25    0.08 * V  37 A V  41 A         V  35 A         V  35 A \n",
      "  37    0.17    0.33    0.27    0.05   I  38 A E  42 A         I  36 A         E  36 A \n",
      "  38    0.10    0.39    0.21    0.05 * L  39 A L  43 A         L  37 A         L  37 A \n",
      "  39    0.20    0.48    0.23    0.08   Q  40 A K  44 A         Q  38 A         R  38 A \n",
      "  40    0.17    0.73    0.40    0.12 * L  41 A L  45 A         L  39 A         L  39 A \n",
      "  41    0.17    0.76    0.32    0.15 * L  42 A L  46 A         L  40 A         L  40 A \n",
      "  42    0.25    0.91    0.44    0.16   D  43 A E  47 A         E  41 A         D  41 A \n",
      "  43    0.30    0.63    0.36    0.13   L  44 A I  48 A         I  42 A         I  42 A \n",
      "  44    0.38    0.66    0.37    0.09   P  45 A P  49 A         P  43 A         E  43 A \n",
      "  45    0.22    0.40    0.43    0.13   Q  46 A Q  50 A         Q  44 A         P  44 A \n",
      "  46    0.14    0.51    0.34    0.09 * A  47 A A  51 A         A  45 A         A  45 A \n",
      "  47    0.17    0.50    0.34    0.09   Q  48 A L  52 A         M  46 A         L  46 A \n",
      "  48    0.19    0.53    0.37    0.08   A  49 A G  53 A         K  47 A         K  47 A \n",
      "  49    0.12    0.37    0.34    0.07   A  50 A G  54 A         A  48 A         A  48 A \n",
      "  50    0.16    0.46    0.33    0.13   V  51 A A  55 A         L  49 A         L  49 A \n",
      "  51    0.18    0.40    0.31    0.05   K  52 A E  56 A         E  50 A         A  50 A \n",
      "  52    0.10    0.44    0.27    0.05 * G  53 A G  57 A         G  51 A         G  51 A \n",
      "  53    0.10    0.39    0.29    0.09 * V  54 A V  58 A         V  52 A         V  52 A \n",
      "  54    0.31    0.29    0.36    0.11   V  55 A A  59 A         V  53 A         E  53 A \n",
      "  55    0.19    0.33    0.34    0.11   M  56 A M  60 A         M  54 A         A  54 A \n",
      "  56    0.08    0.22    0.26    0.10 * E  57 A E  61 A         E  55 A         E  55 A \n",
      "  57    0.06    0.22    0.24    0.12 * L  58 A L  62 A         L  56 A         L  56 A \n",
      "  58    0.14    0.30    0.37    0.17   D  59 A L  63 A         E  57 A         E  57 A \n",
      "  59    0.11    0.41    0.41    0.21 * D  60 A D  64 A         D  58 A         D  58 A \n",
      "  60    0.12    0.32    0.30    0.16   C  61 A S  65 A         C  59 A         C  59 A \n",
      "  61    0.11    0.40    0.37    0.17 * A  62 A A  66 A         A  60 A         A  60 A \n",
      "  62    0.08    0.27    0.25    0.08 * F  63 A F  67 A         F  61 A         F  61 A \n",
      "  63    0.07    0.28    0.23    0.06 * P  64 A P  68 A         P  62 A         P  62 A \n",
      "  64    0.10    0.37    0.28    0.11 * L  65 A L  69 A         L  63 A         L  63 A \n",
      "  65    0.10    0.41    0.20    0.11 * L  66 A L  70 A         L  64 A         L  64 A \n",
      "  66    0.19    0.36    0.40    0.19   A  67 A R  71 A         A  65 A         D  65 A \n",
      "  67    0.19    0.40    0.38    0.13   G  68 A N  72 A         G  66 A         K  66 A \n",
      "  68    0.16    0.52    0.36    0.03   V  69 A I  73 A         L  67 A         V  67 A \n",
      "  69    0.17    0.42    0.37    0.08   V  70 A T  74 A         E  68 A         V  68 A \n",
      "  70    0.20    0.65    0.39    0.08   I  71 A I  75 A         A  69 A         V  69 A \n",
      "  71    0.17    0.73    0.38    0.11 * T  72 A T  76 A         T  70 A         T  70 A \n",
      "  72    0.36    0.73    0.34    0.15   D  73 A A  77 A         D  71 A         A  71 A \n",
      "  73    0.15    0.61    0.32    0.15 * D  74 A D  78 A         D  72 A         D  72 A \n",
      "  74    0.15    0.54    0.29    0.09   P  75 A A  79 A         P  73 A         P  73 A \n",
      "  75    0.18    0.37    0.25    0.05   K  76 A N  80 A         D  74 A         R  74 A \n",
      "  76    0.17    0.52    0.28    0.09   V  77 A E  81 A         V  75 A         V  75 A \n",
      "  77    0.12    0.46    0.31    0.04 * A  78 A A  82 A         A  76 A         A  76 A \n",
      "  78    0.09    0.33    0.16    0.07 * F  79 A F  83 A         F  77 A         F  77 A \n",
      "  79    0.14    0.30    0.21    0.06   K  80 A D  84 A         K  78 A         D  78 A \n",
      "  80    0.11    0.18    0.18    0.04   D  81 A G  85 A         D  79 A         G  79 A \n",
      "  81    0.18    0.45    0.27    0.04   A  82 A A  86 A         A  80 A         V  80 A \n",
      "  82    0.28    0.51    0.24    0.12   D  83 A N  87 A         D  81 A         A  81 A \n",
      "  83    0.17    0.35    0.19    0.05   V  84 A A  88 A         Y  82 A         I  82 A \n",
      "  84    0.09    0.29    0.28    0.09 * A  85 A A  89 A         A  83 A         A  83 A \n",
      "  85    0.20    0.57    0.35    0.06   L  86 A F  90 A         L  84 A         I  84 A \n",
      "  86    0.22    0.69    0.31    0.12   L  87 A L  91 A         L  85 A         M  85 A \n",
      "  87    0.25    0.50    0.31    0.06   V  88 A V  92 A         V  86 A         C  86 A \n",
      "  88    0.14    0.29    0.26    0.12 * G  89 A G  93 A         G  87 A         G  87 A \n",
      "  89    0.17    0.30    0.40    0.26 * A  90 A A  94 A         A  88 A         A  88 A \n",
      "  90    0.32    0.37    0.84    0.55   R  91 A K  95 A         A  89 A         F  89 A \n",
      "  91    0.23    0.79    1.17    0.51 * P  92 A P  96 A         P  90 A         P  90 A \n",
      "  92    0.61    1.90    0.95    0.00   R  93 A -                               R  91 A \n",
      "  93    0.45    1.30    0.65    0.00   S  94 A -               -               K  92 A \n",
      "  94    0.00    0.00    0.00    0.00   -       -               -               A  93 A \n",
      "  95    0.00    0.00    0.00    0.00   -       -               -               G  94 A \n",
      "  96    0.79    3.02    1.51    0.00   M  97 A -               -               M  95 A \n",
      "  97    0.51    1.80    1.24    0.38   E  98 A E 102 A         -               E  96 A \n",
      "  98    0.46    1.64    1.00    0.43   R  99 A R 103 A         -               R  97 A \n",
      "  99    0.49    1.85    1.17    0.48   K 100 A A 104 A         -               K  98 A \n",
      " 100    0.34    1.26    0.94    0.34   D 101 A D 105 A         -               D  99 A \n",
      " 101    0.18    0.43    0.47    0.17   L 102 A L 106 A         -               L 100 A \n",
      " 102    0.30    1.04    0.99    0.53 * L 103 A L 107 A         L 101 A         L 101 A \n",
      " 103    0.33    1.11    0.96    0.59   S 104 A A 108 A         Q 102 A         E 102 A \n",
      " 104    0.53    0.99    0.98    0.32   A 105 A N 109 A         V 103 A         M 103 A \n",
      " 105    0.10    0.24    0.65    0.38 * N 106 A N 110 A         N 104 A         N 104 A \n",
      " 106    0.11    0.14    0.77    0.53   A 107 A G 111 A         G 105 A         A 105 A \n",
      " 107    0.15    0.29    0.61    0.37   E 108 A K 112 A         K 106 A         R 106 A \n",
      " 108    0.10    0.34    0.31    0.06 * I 109 A I 113 A         I 107 A         I 107 A \n",
      " 109    0.07    0.22    0.30    0.12 * F 110 A F 114 A         F 108 A         F 108 A \n",
      " 110    0.32    0.41    0.37    0.13   T 111 A G 115 A         T 109 A         K 109 A \n",
      " 111    0.20    0.52    0.34    0.05   V 112 A P 116 A         E 110 A         E 110 A \n",
      " 112    0.09    0.30    0.22    0.03 * Q 113 A Q 117 A         Q 111 A         Q 111 A \n",
      " 113    0.05    0.18    0.21    0.10 * G 114 A G 118 A         G 112 A         G 112 A \n",
      " 114    0.16    0.28    0.27    0.05   A 115 A K 119 A         R 113 A         E 113 A \n",
      " 115    0.08    0.23    0.25    0.05 * A 116 A A 120 A         A 114 A         A 114 A \n",
      " 116    0.11    0.26    0.24    0.05   L 117 A I 121 A         L 115 A         I 115 A \n",
      " 117    0.14    0.27    0.23    0.07   N 118 A N 122 A         A 116 A         A 116 A \n",
      " 118    0.19    0.45    0.29    0.10   E 119 A D 123 A         E 117 A         A 117 A \n",
      " 119    0.11    0.23    0.22    0.06   V 120 A N 124 A         V 118 A         V 118 A \n",
      " 120    0.07    0.19    0.22    0.05 * A 121 A A 125 A         A 119 A         A 119 A \n",
      " 121    0.09    0.10    0.35    0.19   S 122 A A 126 A         K 120 A         A 120 A \n",
      " 122    0.15    0.25    0.68    0.40   R 123 A D 127 A         K 121 A         S 121 A \n",
      " 123    0.14    0.59    0.72    0.45 * D 124 A D 128 A         D 122 A         D 122 A \n",
      " 124    0.21    0.26    0.23    0.03   V 125 A I 129 A         V 123 A         C 123 A \n",
      " 125    0.15    0.46    0.25    0.08   K 126 A R 130 A         K 124 A         R 124 A \n",
      " 126    0.07    0.23    0.20    0.10 * V 127 A V 131 A         V 125 A         V 125 A \n",
      " 127    0.17    0.49    0.24    0.13   L 128 A L 132 A         L 126 A         V 126 A \n",
      " 128    0.08    0.28    0.16    0.04 * V 129 A V 133 A         V 127 A         V 127 A \n",
      " 129    0.11    0.42    0.22    0.07 * V 130 A V 134 A         V 128 A         V 128 A \n",
      " 130    0.14    0.64    0.27    0.15 * G 131 A G 135 A         G 129 A         G 129 A \n",
      " 131    0.09    0.22    0.25    0.14 * N 132 A N 136 A         N 130 A         N 130 A \n",
      " 132    0.13    0.24    0.17    0.06 * P 133 A P 137 A         P 131 A         P 131 A \n",
      " 133    0.06    0.15    0.21    0.08 * A 134 A A 138 A         A 132 A         A 132 A \n",
      " 134    0.07    0.13    0.17    0.05 * N 135 A N 139 A         N 133 A         N 133 A \n",
      " 135    0.05    0.14    0.13    0.04 * T 136 A T 140 A         T 134 A         T 134 A \n",
      " 136    0.05    0.06    0.17    0.10 * N 137 A N 141 A         N 135 A         N 135 A \n",
      " 137    0.06    0.16    0.22    0.07 * A 138 A A 142 A         A 136 A         A 136 A \n",
      " 138    0.09    0.18    0.22    0.10   Y 139 A L 143 A         L 137 A         L 137 A \n",
      " 139    0.05    0.08    0.20    0.10 * I 140 A I 144 A         I 138 A         I 138 A \n",
      " 140    0.15    0.20    0.26    0.08   A 141 A A 145 A         A 139 A         L 139 A \n",
      " 141    0.18    0.35    0.28    0.08   M 142 A S 146 A         Y 140 A         L 140 A \n",
      " 142    0.13    0.35    0.29    0.05   K 143 A A 147 A         K 141 A         K 141 A \n",
      " 143    0.25    0.89    0.49    0.12   S 144 A A 148 A         N 142 A         S 142 A \n",
      " 144    0.30    1.22    0.58    0.21 * A 145 A A 149 A         A 143 A         A 143 A \n",
      " 145    0.00    0.00    0.00    0.00   -       -               -               Q 144 A \n",
      " 146    0.57    1.96    0.99    0.61   P 146 A P 150 A         P 144 A         G 145 A \n",
      " 147    0.33    0.97    0.44    0.17   D 147 A D 151 A         G 145 A         K 146 A \n",
      " 148    0.31    1.35    0.59    0.31   L 148 A V 152 A         L 146 A         L 147 A \n",
      " 149    0.21    0.57    0.46    0.13   P 149 A P 153 A         N 147 A         N 148 A \n",
      " 150    0.25    0.83    0.55    0.08   K 150 A A 154 A         P 148 A         P 149 A \n",
      " 151    0.25    0.85    0.52    0.12   K 151 A S 155 A         R 149 A         R 150 A \n",
      " 152    0.20    0.42    0.30    0.11   N 152 A R 156 A         N 150 A         H 151 A \n",
      " 153    0.18    0.31    0.24    0.09   F 153 A F 157 A         F 151 A         V 152 A \n",
      " 154    0.11    0.29    0.19    0.10   T 154 A N 158 A         T 152 A         T 153 A \n",
      " 155    0.08    0.22    0.14    0.05 * A 155 A A 159 A         A 153 A         A 154 A \n",
      " 156    0.10    0.25    0.20    0.03 * M 156 A M 160 A         M 154 A         M 155 A \n",
      " 157    0.24    0.22    0.24    0.11   L 157 A M 161 A         T 155 A         T 156 A \n",
      " 158    0.16    0.17    0.18    0.07 * R 158 A R 162 A         R 156 A         R 157 A \n",
      " 159    0.06    0.18    0.23    0.10 * L 159 A L 163 A         L 157 A         L 158 A \n",
      " 160    0.07    0.22    0.23    0.10 * D 160 A D 164 A         D 158 A         D 159 A \n",
      " 161    0.06    0.16    0.17    0.06 * H 161 A H 165 A         H 159 A         H 160 A \n",
      " 162    0.06    0.10    0.18    0.07 * N 162 A N 166 A         N 160 A         N 161 A \n",
      " 163    0.06    0.14    0.21    0.08 * R 163 A R 167 A         R 161 A         R 162 A \n",
      " 164    0.07    0.17    0.15    0.06 * A 164 A A 168 A         A 162 A         A 163 A \n",
      " 165    0.15    0.37    0.17    0.06   L 165 A I 169 A         K 163 A         L 164 A \n",
      " 166    0.14    0.47    0.28    0.12   S 166 A S 170 A         A 164 A         S 165 A \n",
      " 167    0.23    0.50    0.41    0.07   Q 167 A Q 171 A         Q 165 A         L 166 A \n",
      " 168    0.08    0.31    0.31    0.05 * L 168 A L 172 A         L 166 A         L 167 A \n",
      " 169    0.08    0.24    0.38    0.15 * A 169 A A 173 A         A 167 A         A 168 A \n",
      " 170    0.14    0.18    0.44    0.22   A 170 A T 174 A         K 168 A         R 169 A \n",
      " 171    0.13    0.48    0.50    0.12 * K 171 A K 175 A         K 169 A         K 170 A \n",
      " 172    0.18    0.42    0.55    0.22   S 172 A L 176 A         T 170 A         A 171 A \n",
      " 173    0.15    0.69    0.61    0.16 * G 173 A G 177 A         G 171 A         G 172 A \n",
      " 174    0.31    0.93    0.67    0.25   K 174 A R 178 A         T 172 A         V 173 A \n",
      " 175    0.27    0.92    0.61    0.21   P 175 A G 179 A         G 173 A         P 174 A \n",
      " 176    0.18    0.60    0.49    0.14   V 176 A S 180 A         V 174 A         V 175 A \n",
      " 177    0.29    0.98    0.59    0.19   A 177 A A 181 A         D 175 A         S 176 A \n",
      " 178    0.23    0.60    0.44    0.14   S 178 A E 182 A         R 176 A         Q 177 A \n",
      " 179    0.32    1.19    0.59    0.24   I 179 A F 183 A         I 177 A         V 178 A \n",
      " 180    0.73    2.17    0.96    0.49   E 180 A N 184 A         R 178 A         R 179 A \n",
      " 181    0.61    1.89    0.89    0.38   K 181 A N 185 A         R 179 A         N 180 A \n",
      " 182    0.41    1.60    0.79    0.33   L 182 A I 186 A         M 180 A         V 181 A \n",
      " 183    0.31    1.05    0.55    0.22   A 183 A V 187 A         T 181 A         I 182 A \n",
      " 184    0.23    0.82    0.44    0.25   V 184 A V 188 A         V 182 A         I 183 A \n",
      " 185    0.11    0.44    0.20    0.08 * W 185 A W 189 A         W 183 A         W 184 A \n",
      " 186    0.15    0.66    0.29    0.13 * G 186 A G 190 A         G 184 A         G 185 A \n",
      " 187    0.37    0.98    0.64    0.39 * N 187 A N 191 A         N 185 A         N 186 A \n",
      " 188    0.21    0.86    0.56    0.23 * H 188 A H 192 A         H 186 A         H 187 A \n",
      " 189    0.35    0.94    0.53    0.16 * S 189 A S 193 A         S 187 A         S 188 A \n",
      " 190    0.33    1.26    0.62    0.30   P 190 A A 194 A         S 188 A         S 189 A \n",
      " 191    0.32    1.30    0.69    0.26   T 191 A T 195 A         I 189 A         T 190 A \n",
      " 192    0.27    0.82    0.44    0.12   M 192 A Q 196 A         M 190 A         Q 191 A \n",
      " 193    0.30    0.89    0.39    0.16   Y 193 A F 197 A         F 191 A         V 192 A \n",
      " 194    0.26    1.20    0.47    0.25 * P 194 A P 198 A         P 192 A         P 193 A \n",
      " 195    0.22    0.98    0.41    0.21 * D 195 A D 199 A         D 193 A         D 194 A \n",
      " 196    0.35    1.09    0.64    0.16   F 196 A I 200 A         L 194 A         T 195 A \n",
      " 197    0.34    1.03    0.52    0.25   R 197 A T 201 A         F 195 A         D 196 A \n",
      " 198    0.33    0.98    0.55    0.15   F 198 A Y 202 A         H 196 A         S 197 A \n",
      " 199    0.23    0.99    0.52    0.16 * A 199 A A 203 A         A 197 A         A 198 A \n",
      " 200    0.32    0.97    0.56    0.15   T 200 A T 204 A         E 198 A         V 199 A \n",
      " 201    0.21    0.58    0.65    0.21   A 201 A V 205 A         V 199 A         I 200 A \n",
      " 202    0.51    2.16    1.30    0.41   E 202 A G 206 A         D 200 A         G 201 A \n",
      " 203    0.60    2.35    1.37    0.36   G 203 A G 207 A         G 205 A         T 202 A \n",
      " 204    0.36    1.21    0.79    0.21   E 204 A E 208 A         R 206 A         T 203 A \n",
      " 205    0.24    0.77    0.63    0.10   S 205 A K 209 A         P 207 A         P 204 A \n",
      " 206    0.31    1.16    0.58    0.18   L 206 A V 210 A         A 208 A         A 205 A \n",
      " 207    0.52    1.99    0.84    0.44   L 207 A T 211 A         L 209 A         R 206 A \n",
      " 208    0.49    2.07    0.88    0.51   K 208 A D 212 A         E 210 A         E 207 A \n",
      " 209    0.62    2.41    1.17    0.48   L 209 A L 213 A         L 211 A         A 208 A \n",
      " 210    0.97    2.93    1.37    0.57   I 210 A V 214 A         V 212 A         I 209 A \n",
      " 211    1.69    6.86    3.43    0.00   N 211 A -               -               K 210 A \n",
      " 212    0.00    0.00    0.68    0.21   D 212 A D 215 A         D 214 A         -       \n",
      " 213    0.00    0.00    0.23    0.11   D 213 A H 216 A         M 215 A         -       \n",
      " 214    0.00    0.00    0.60    0.16   V 214 A D 217 A         E 216 A         -       \n",
      " 215    0.00    0.00    0.60    0.19   W 215 A W 218 A         W 217 A         -       \n",
      " 216    0.00    0.00    0.49    0.09   N 216 A Y 219 A         Y 218 A         -       \n",
      " 217    0.00    0.00    0.59    0.05   R 217 A V 220 A         E 219 A         -       \n",
      " 218    0.00    0.00    0.66    0.25   D 218 A E 221 A         K 220 A         -       \n",
      " 219    0.44    0.66    0.57    0.25   T 219 A E 222 A         V 221 A         D 217 A \n",
      " 220    0.32    0.56    0.36    0.18 * F 220 A F 223 A         F 222 A         F 218 A \n",
      " 221    0.31    0.57    0.37    0.19   I 221 A I 224 A         I 223 A         V 219 A \n",
      " 222    0.19    0.27    0.54    0.26   P 222 A P 225 A         P 224 A         Q 220 A \n",
      " 223    0.18    0.31    0.42    0.23   T 223 A R 226 A         T 225 A         V 221 A \n",
      " 224    0.14    0.57    0.39    0.22 * V 224 A V 227 A         V 226 A         V 222 A \n",
      " 225    0.36    0.52    0.57    0.24   G 225 A A 228 A         A 227 A         R 223 A \n",
      " 226    0.14    0.10    0.59    0.42   K 226 A N 229 A         Q 228 A         G 224 A \n",
      " 227    0.08    0.23    0.42    0.24 * R 227 A R 230 A         R 229 A         R 225 A \n",
      " 228    0.08    0.32    0.45    0.24 * G 228 A G 231 A         G 230 A         G 226 A \n",
      " 229    0.14    0.57    0.58    0.27 * A 229 A A 232 A         A 231 A         A 227 A \n",
      " 230    0.20    0.61    0.56    0.30   A 230 A E 233 A         A 232 A         E 228 A \n",
      " 231    0.15    0.62    0.50    0.25 * I 231 A I 234 A         I 233 A         I 229 A \n",
      " 232    0.13    0.48    0.50    0.23 * I 232 A I 235 A         I 234 A         I 230 A \n",
      " 233    0.16    0.44    0.43    0.20   E 233 A E 236 A         Q 235 A         Q 231 A \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 234    0.25    0.77    0.48    0.17   A 234 A V 237 A         A 236 A         L 232 A \n",
      " 235    0.21    0.91    0.61    0.15 * R 235 A R 238 A         R 237 A         R 233 A \n",
      " 236    0.16    0.76    0.53    0.28 * G 236 A G 239 A         G 238 A         G 234 A \n",
      " 237    0.22    0.67    0.43    0.18   L 237 A K 240 A         A 239 A         L 235 A \n",
      " 238    0.16    0.67    0.53    0.15 * S 238 A S 241 A         S 240 A         S 236 A \n",
      " 239    0.17    0.54    0.40    0.14 * S 239 A S 242 A         S 241 A         S 237 A \n",
      " 240    0.11    0.36    0.27    0.03 * A 240 A A 243 A         A 242 A         A 238 A \n",
      " 241    0.19    0.29    0.22    0.10   A 241 A A 244 A         A 243 A         M 239 A \n",
      " 242    0.07    0.22    0.23    0.04 * S 242 A S 245 A         S 244 A         S 240 A \n",
      " 243    0.10    0.37    0.25    0.04 * A 243 A A 246 A         A 245 A         A 241 A \n",
      " 244    0.10    0.36    0.20    0.05 * A 244 A A 247 A         A 246 A         A 242 A \n",
      " 245    0.15    0.25    0.20    0.08   N 245 A S 248 A         N 247 A         K 243 A \n",
      " 246    0.08    0.17    0.19    0.07   A 246 A S 249 A         A 248 A         A 244 A \n",
      " 247    0.08    0.25    0.18    0.04 * A 247 A A 250 A         A 249 A         A 245 A \n",
      " 248    0.15    0.42    0.27    0.09   I 248 A I 251 A         I 250 A         V 246 A \n",
      " 249    0.10    0.29    0.24    0.07   D 249 A D 252 A         E 251 A         D 247 A \n",
      " 250    0.08    0.23    0.20    0.05 * H 250 A H 253 A         H 252 A         H 248 A \n",
      " 251    0.14    0.38    0.23    0.12   V 251 A M 254 A         I 253 A         V 249 A \n",
      " 252    0.23    0.51    0.28    0.11   R 252 A R 255 A         R 254 A         H 250 A \n",
      " 253    0.13    0.52    0.27    0.09 * D 253 A D 256 A         D 255 A         D 251 A \n",
      " 254    0.08    0.37    0.27    0.16 * W 254 A W 257 A         W 256 A         W 252 A \n",
      " 255    0.18    0.51    0.23    0.17   V 255 A V 258 A         A 257 A         I 253 A \n",
      " 256    0.24    0.52    0.28    0.09   L 256 A Q 259 A         L 258 A         H 254 A \n",
      " 257    0.14    0.61    0.40    0.17 * G 257 A G 260 A         G 259 A         G 255 A \n",
      " 258    0.15    0.54    0.33    0.06 * T 258 A T 261 A         T 260 A         T 256 A \n",
      " 259    0.35    1.25    1.41    0.38   N 259 A E 262 A         P 261 A         P 257 A \n",
      " 260    0.50    1.32    1.29    0.46   G 260 A -               E 262 A         E 258 A \n",
      " 261    0.30    0.76    0.38    0.00   -       -               G 263 A         G 259 A \n",
      " 262    0.33    0.62    0.94    0.35   K 261 A A 263 A         D 264 A         V 260 A \n",
      " 263    0.21    0.44    0.55    0.19   W 262 A W 264 A         W 265 A         Y 261 A \n",
      " 264    0.19    0.62    0.47    0.17   V 263 A S 265 A         V 266 A         V 262 A \n",
      " 265    0.13    0.41    0.26    0.08   T 264 A S 266 A         S 267 A         S 263 A \n",
      " 266    0.19    0.55    0.32    0.13   M 265 A A 267 A         M 268 A         M 264 A \n",
      " 267    0.15    0.43    0.30    0.03   G 266 A A 268 A         A 269 A         G 265 A \n",
      " 268    0.09    0.22    0.24    0.06   I 267 A I 269 A         V 270 A         V 266 A \n",
      " 269    0.20    0.26    0.24    0.10   P 268 A P 270 A         P 271 A         Y 267 A \n",
      " 270    0.17    0.35    0.27    0.06 * S 269 A S 271 A         S 272 A         S 268 A \n",
      " 271    0.61    2.22    0.89    0.52   D 270 A T 272 A         Q 273 A         D 269 A \n",
      " 272    0.93    4.05    1.52    0.89   G 271 A G 273 A         G 274 A         E 270 A \n",
      " 273    0.70    2.51    1.08    0.59   S 272 A A 274 A         E 275 A         N 271 A \n",
      " 274    0.00    0.00    0.00    0.00   -       -               -               P 272 A \n",
      " 275    0.19    0.72    0.48    0.22 * Y 273 A Y 275 A         Y 277 A         Y 273 A \n",
      " 276    0.39    1.74    0.92    0.52 * G 274 A G 276 A         G 278 A         G 274 A \n",
      " 277    0.19    0.60    0.38    0.15   I 275 A I 277 A         I 279 A         V 275 A \n",
      " 278    0.10    0.40    0.37    0.13 * P 276 A P 278 A         P 280 A         P 276 A \n",
      " 279    0.23    0.57    0.41    0.14   E 277 A E 279 A         E 281 A         S 277 A \n",
      " 280    0.29    0.89    0.86    0.46   D 278 A G 280 A         G 282 A         G 278 A \n",
      " 281    0.13    0.27    0.51    0.23   I 279 A I 281 A         I 283 A         L 279 A \n",
      " 282    0.15    0.40    0.29    0.06   I 280 A F 282 A         V 284 A         I 280 A \n",
      " 283    0.18    0.46    0.30    0.14   Y 281 A V 283 A         Y 285 A         F 281 A \n",
      " 284    0.17    0.48    0.29    0.17   G 282 A G 284 A         S 286 A         S 282 A \n",
      " 285    0.22    0.37    0.18    0.08   V 283 A L 285 A         F 287 A         F 283 A \n",
      " 286    0.12    0.47    0.42    0.19 * P 284 A P 286 A         P 288 A         P 284 A \n",
      " 287    0.23    0.40    0.37    0.09   V 285 A T 287 A         V 289 A         C 285 A \n",
      " 288    0.24    0.45    0.48    0.18   I 286 A V 288 A         T 290 A         T 286 A \n",
      " 289    0.22    0.61    0.44    0.05   C 287 A S 289 A         A 291 A         C 287 A \n",
      " 290    0.31    0.95    0.71    0.04   E 288 A R 290 A         K 292 A         H 288 A \n",
      " 291    0.36    1.25    1.00    0.43   N 289 A N 291 A         D 293 A         A 289 A \n",
      " 292    0.14    0.65    0.49    0.21 * G 290 A G 292 A         G 294 A         G 290 A \n",
      " 293    0.20    0.70    0.60    0.09   E 291 A E 293 A         A 295 A         E 291 A \n",
      " 294    0.23    0.75    0.64    0.10   Y 292 A W 294 A         Y 296 A         W 292 A \n",
      " 295    0.28    0.84    0.57    0.09   K 293 A E 295 A         R 297 A         T 293 A \n",
      " 296    0.25    0.85    0.46    0.18   R 294 A I 296 A         V 298 A         V 294 A \n",
      " 297    0.14    0.56    0.35    0.13 * V 295 A V 297 A         V 299 A         V 295 A \n",
      " 298    0.45    1.67    0.81    0.43   E 296 A E 298 A         E 300 A         S 296 A \n",
      " 299    0.60    2.89    1.27    0.63 * G 297 A G 299 A         G 301 A         G 297 A \n",
      " 300    0.77    3.06    1.26    0.66   L 298 A L 300 A         L 302 A         K 298 A \n",
      " 301    1.24    4.21    1.69    0.90   E 299 A E 301 A         E 303 A         L 299 A \n",
      " 302    0.00    0.00    0.52    0.02   I 300 A I 302 A         I 304 A         -       \n",
      " 303    0.00    0.00    0.42    0.08   D 301 A S 303 A         N 305 A         -       \n",
      " 304    0.00    0.00    0.60    0.06   A 302 A D 304 A         E 306 A         -       \n",
      " 305    0.00    0.00    0.66    0.15   F 303 A F 305 A         F 307 A         -       \n",
      " 306    0.00    0.00    0.53    0.19   S 304 A Q 306 A         A 308 A         -       \n",
      " 307    0.82    2.67    1.07    0.64   R 305 A R 307 A         R 309 A         K 305 A \n",
      " 308    0.86    3.02    1.15    0.69   E 306 A A 308 A         K 310 A         Q 306 A \n",
      " 309    0.72    2.78    1.08    0.63   K 307 A R 309 A         R 311 A         R 307 A \n",
      " 310    0.48    1.35    0.62    0.25   M 308 A I 310 A         M 312 A         L 308 A \n",
      " 311    0.43    1.47    0.72    0.27   D 309 A D 311 A         E 313 A         A 309 A \n",
      " 312    0.33    1.13    0.61    0.22   G 310 A A 312 A         I 314 A         S 310 A \n",
      " 313    0.20    0.72    0.57    0.28   T 311 A N 313 A         T 315 A         T 311 A \n",
      " 314    0.29    0.92    0.64    0.24   L 312 A A 314 A         A 316 A         I 312 A \n",
      " 315    0.26    0.87    0.69    0.18   A 313 A Q 315 A         Q 317 A         A 313 A \n",
      " 316    0.14    0.56    0.64    0.21 * E 314 A E 316 A         E 318 A         E 314 A \n",
      " 317    0.10    0.41    0.40    0.17 * L 315 A L 317 A         L 319 A         L 315 A \n",
      " 318    0.18    0.45    0.50    0.20   L 316 A Q 318 A         L 320 A         Q 316 A \n",
      " 319    0.20    0.63    0.67    0.23   E 317 A A 319 A         D 321 A         E 317 A \n",
      " 320    0.11    0.39    0.58    0.24 * E 318 A E 320 A         E 322 A         E 318 A \n",
      " 321    0.15    0.39    0.48    0.22   R 319 A R 321 A         M 323 A         R 319 A \n",
      " 322    0.28    0.86    0.61    0.26   D 320 A E 322 A         E 324 A         A 320 A \n",
      " 323    0.29    0.95    0.71    0.36   G 321 A A 323 A         Q 325 A         Q 321 A \n",
      " 324    0.39    0.88    0.71    0.24   V 322 A V 324 A         V 326 A         A 322 A \n",
      " 325    0.51    1.78    0.94    0.33   A 323 A R 325 A         K 327 A         G 323 A \n",
      " 326    0.00    0.00    1.84    0.82   H 324 A D 326 A         A 328 A         -       \n",
      " 327    0.00    0.00    2.78    0.00   -       L 327 A         L 329 A         -       \n",
      " 328    0.00    0.00    2.97    0.00   -       L 328 A         G 330 A         -       \n",
      " 329    0.00    0.00    2.65    0.00   -       L 329 A         L 331 A         -       \n",
      " 330    1.20    5.55    3.39    1.51   L 325 A E 330 A         I 332 A         L 324 A \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    4\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      320      320        1.0733        1.0733        0.0002\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      320      320        1.0733        1.0733        0.0000\n",
      "\n",
      "Current alignment total score:        113.3\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -       -       M   0 A         -       \n",
      "   2    0.29    0.60    1.18    0.57   K   3 A -       S   1 A K   1 A         M   1 A \n",
      "   3    0.24    0.67    1.02    0.39   P   4 A S   8 A E   2 A A   2 A         S   2 A \n",
      "   4    0.32    0.55    0.52    0.12   A   5 A T   9 A P   3 A P   3 A         A   3 A \n",
      "   5    0.37    0.59    0.25    0.12   K   6 A K  10 A I   4 A V   4 A         V   4 A \n",
      "   6    0.17    0.57    0.23    0.13   R   7 A K  11 A R   5 A R   5 A         K   5 A \n",
      "   7    0.08    0.27    0.19    0.04 * V   8 A V  12 A V   6 A V   6 A         V   6 A \n",
      "   8    0.18    0.35    0.27    0.05   A   9 A T  13 A L   7 A A   7 A         A   7 A \n",
      "   9    0.10    0.35    0.32    0.03 * V  10 A V  14 A V   8 A V   8 A         V   8 A \n",
      "  10    0.08    0.24    0.24    0.11 * T  11 A T  15 A T   9 A T   9 A         T   9 A \n",
      "  11    0.13    0.58    0.34    0.17 * G  12 A G  16 A G  10 A G  10 A         G  10 A \n",
      "  12    0.09    0.28    0.37    0.09 * A  13 A A  17 A A  11 A A  11 A         A  11 A \n",
      "  13    0.11    0.39    0.34    0.04 * A  14 A A  18 A A  12 A A  12 A         A  12 A \n",
      "  14    0.09    0.39    0.22    0.09 * G  15 A G  19 A G  13 A G  13 A         G  13 A \n",
      "  15    0.09    0.25    0.24    0.10 * Q  16 A Q  20 A Q  14 A Q  14 A         Q  14 A \n",
      "  16    0.07    0.15    0.17    0.07 * I  17 A I  21 A I  15 A I  15 A         I  15 A \n",
      "  17    0.16    0.39    0.22    0.09   A  18 A S  22 A A  16 A G  16 A         G  16 A \n",
      "  18    0.05    0.12    0.16    0.05 * Y  19 A Y  23 A Y  17 A Y  17 A         Y  17 A \n",
      "  19    0.12    0.37    0.22    0.06   S  20 A S  24 A S  18 A S  18 A         A  18 A \n",
      "  20    0.14    0.57    0.27    0.10 * L  21 A L  25 A L  19 A L  19 A         L  19 A \n",
      "  21    0.25    0.54    0.27    0.10   L  22 A L  26 A L  20 A L  20 A         V  20 A \n",
      "  22    0.24    0.67    0.40    0.12   F  23 A W  27 A Y  21 A F  21 A         P  21 A \n",
      "  23    0.21    0.44    0.45    0.10   R  24 A R  28 A S  22 A R  22 A         L  22 A \n",
      "  24    0.09    0.29    0.35    0.13 * I  25 A I  29 A I  23 A I  23 A         I  23 A \n",
      "  25    0.15    0.30    0.26    0.08   A  26 A A  30 A G  24 A A  24 A         A  24 A \n",
      "  26    0.17    0.39    0.32    0.11   N  27 A N  31 A N  25 A A  25 A         R  25 A \n",
      "  27    0.13    0.55    0.41    0.10 * G  28 A G  32 A G  26 A G  26 A         G  26 A \n",
      "  28    0.37    1.34    0.69    0.21   D  29 A E  33 A S  27 A E  27 A         A  27 A \n",
      "  29    0.32    1.22    0.70    0.17   L  30 A V  34 A V  28 A M  28 A         L  28 A \n",
      "  30    0.19    0.56    0.50    0.24   L  31 A F  35 A F  29 A L  29 A         L  29 A \n",
      "  31    0.18    0.80    0.55    0.16 * G  32 A G  36 A G  30 A G  30 A         G  30 A \n",
      "  32    0.18    0.51    0.44    0.18   K  33 A T  37 A K  31 A K  31 A         P  31 A \n",
      "  33    0.22    0.84    0.58    0.14   D  34 A D  38 A D  32 A D  32 A         T  32 A \n",
      "  34    0.19    0.34    0.39    0.15   Q  35 A T  39 A Q  33 A Q  33 A         T  33 A \n",
      "  35    0.14    0.60    0.43    0.10 * P  36 A P  40 A P  34 A P  34 A         P  34 A \n",
      "  36    0.18    0.51    0.30    0.08   V  37 A V  41 A I  35 A V  35 A         V  35 A \n",
      "  37    0.30    1.00    0.43    0.19   I  38 A E  42 A I  36 A I  36 A         E  36 A \n",
      "  38    0.15    0.56    0.27    0.12 * L  39 A L  43 A L  37 A L  37 A         L  37 A \n",
      "  39    0.19    0.30    0.25    0.04   Q  40 A K  44 A V  38 A Q  38 A         R  38 A \n",
      "  40    0.13    0.51    0.41    0.10 * L  41 A L  45 A L  39 A L  39 A         L  39 A \n",
      "  41    0.12    0.49    0.36    0.11 * L  42 A L  46 A L  40 A L  40 A         L  40 A \n",
      "  42    0.14    0.42    0.42    0.16   D  43 A E  47 A D  41 A E  41 A         D  41 A \n",
      "  43    0.36    1.08    0.51    0.21   L  44 A I  48 A I  42 A I  42 A         I  42 A \n",
      "  44    0.49    1.32    0.57    0.27   P  45 A P  49 A T  43 A P  43 A         E  43 A \n",
      "  45    0.44    1.63    0.69    0.34   Q  46 A Q  50 A P  44 A Q  44 A         P  44 A \n",
      "  46    0.36    1.18    0.52    0.23   A  47 A A  51 A M  45 A A  45 A         A  45 A \n",
      "  47    0.33    1.22    0.51    0.28   Q  48 A L  52 A M  46 A M  46 A         L  46 A \n",
      "  48    0.34    1.28    0.54    0.29   A  49 A G  53 A G  47 A K  47 A         K  47 A \n",
      "  49    0.32    1.09    0.48    0.23   A  50 A G  54 A V  48 A A  48 A         A  48 A \n",
      "  50    0.32    1.28    0.52    0.31   V  51 A A  55 A L  49 A L  49 A         L  49 A \n",
      "  51    0.41    1.56    0.56    0.40   K  52 A E  56 A D  50 A E  50 A         A  50 A \n",
      "  52    0.28    1.36    0.52    0.30 * G  53 A G  57 A G  51 A G  51 A         G  51 A \n",
      "  53    0.24    1.06    0.46    0.20 * V  54 A V  58 A V  52 A V  52 A         V  52 A \n",
      "  54    0.38    1.42    0.59    0.30   V  55 A A  59 A L  53 A V  53 A         E  53 A \n",
      "  55    0.40    1.66    0.63    0.39   M  56 A M  60 A M  54 A M  54 A         A  54 A \n",
      "  56    0.25    1.08    0.44    0.23 * E  57 A E  61 A E  55 A E  55 A         E  55 A \n",
      "  57    0.21    0.95    0.40    0.21 * L  58 A L  62 A L  56 A L  56 A         L  56 A \n",
      "  58    0.44    1.68    0.62    0.45   D  59 A L  63 A Q  57 A E  57 A         E  57 A \n",
      "  59    0.39    1.81    0.67    0.49 * D  60 A D  64 A D  58 A D  58 A         D  58 A \n",
      "  60    0.35    1.46    0.52    0.40   C  61 A S  65 A C  59 A C  59 A         C  59 A \n",
      "  61    0.50    2.29    0.79    0.60 * A  62 A A  66 A A  60 A A  60 A         A  60 A \n",
      "  62    0.19    0.49    0.29    0.09   F  63 A F  67 A L  61 A F  61 A         F  61 A \n",
      "  63    0.30    1.32    0.48    0.31 * P  64 A P  68 A P  62 A P  62 A         P  62 A \n",
      "  64    0.12    0.44    0.31    0.09 * L  65 A L  69 A L  63 A L  63 A         L  63 A \n",
      "  65    0.14    0.51    0.26    0.12 * L  66 A L  70 A L  64 A L  64 A         L  64 A \n",
      "  66    0.39    0.40    0.40    0.16   A  67 A R  71 A K  65 A A  65 A         D  65 A \n",
      "  67    0.17    0.26    0.34    0.14   G  68 A N  72 A D  66 A G  66 A         K  66 A \n",
      "  68    0.13    0.38    0.35    0.04   V  69 A I  73 A V  67 A L  67 A         V  67 A \n",
      "  69    0.16    0.28    0.34    0.12   V  70 A T  74 A I  68 A E  68 A         V  68 A \n",
      "  70    0.26    0.81    0.46    0.13   I  71 A I  75 A A  69 A A  69 A         V  69 A \n",
      "  71    0.39    0.76    0.45    0.13 * T  72 A T  76 A T  70 A T  70 A         T  70 A \n",
      "  72    0.35    0.84    0.43    0.19   D  73 A A  77 A D  71 A D  71 A         A  71 A \n",
      "  73    0.33    1.09    0.49    0.23   D  74 A D  78 A K  72 A D  72 A         D  72 A \n",
      "  74    0.49    1.61    0.56    0.42   P  75 A A  79 A E  73 A P  73 A         P  73 A \n",
      "  75    0.49    1.94    0.67    0.45   K  76 A N  80 A E  74 A D  74 A         R  74 A \n",
      "  76    0.23    0.68    0.34    0.16   V  77 A E  81 A I  75 A V  75 A         V  75 A \n",
      "  77    0.15    0.57    0.34    0.12 * A  78 A A  82 A A  76 A A  76 A         A  76 A \n",
      "  78    0.25    1.14    0.38    0.29 * F  79 A F  83 A F  77 A F  77 A         F  77 A \n",
      "  79    0.25    0.81    0.35    0.15   K  80 A D  84 A K  78 A K  78 A         D  78 A \n",
      "  80    0.27    0.92    0.35    0.20   D  81 A G  85 A D  79 A D  79 A         G  79 A \n",
      "  81    0.23    0.60    0.32    0.12   A  82 A A  86 A L  80 A A  80 A         V  80 A \n",
      "  82    0.18    0.30    0.24    0.11   D  83 A N  87 A D  81 A D  81 A         A  81 A \n",
      "  83    0.17    0.44    0.23    0.09   V  84 A A  88 A V  82 A Y  82 A         I  82 A \n",
      "  84    0.15    0.59    0.31    0.17 * A  85 A A  89 A A  83 A A  83 A         A  83 A \n",
      "  85    0.15    0.38    0.34    0.10   L  86 A F  90 A I  84 A L  84 A         I  84 A \n",
      "  86    0.08    0.18    0.27    0.14   L  87 A L  91 A L  85 A L  85 A         M  85 A \n",
      "  87    0.13    0.33    0.30    0.08   V  88 A V  92 A V  86 A V  86 A         C  86 A \n",
      "  88    0.30    0.62    0.34    0.08 * G  89 A G  93 A G  87 A G  87 A         G  87 A \n",
      "  89    0.36    0.98    0.54    0.20   A  90 A A  94 A S  88 A A  88 A         A  88 A \n",
      "  90    0.66    2.13    1.16    0.44   R  91 A K  95 A M  89 A A  89 A         F  89 A \n",
      "  91    0.71    3.11    1.46    0.86 * P  92 A P  96 A P  90 A P  90 A         P  90 A \n",
      "  92    0.95    3.78    1.70    0.95   R  93 A -       R  91 A                 R  91 A \n",
      "  93    1.44    6.32    2.81    1.11   S  94 A -       R  92 A -               K  92 A \n",
      "  94    1.71    7.62    3.81    0.00   -       -       D  93 A -               A  93 A \n",
      "  95    1.87    8.53    4.27    0.00   -       -       G  94 A -               G  94 A \n",
      "  96    1.76    8.01    3.61    1.66   M  97 A -       M  95 A -               M  95 A \n",
      "  97    1.79    7.78    3.00    1.90   E  98 A E 102 A E  96 A -               E  96 A \n",
      "  98    1.72    6.49    2.50    1.60   R  99 A R 103 A R  97 A -               R  97 A \n",
      "  99    1.49    6.30    2.47    1.59   K 100 A A 104 A K  98 A -               K  98 A \n",
      " 100    1.18    4.93    1.94    1.24   D 101 A D 105 A D  99 A -               D  99 A \n",
      " 101    0.58    1.82    0.80    0.41   L 102 A L 106 A L 100 A -               L 100 A \n",
      " 102    0.49    1.62    1.03    0.61 * L 103 A L 107 A L 101 A L 101 A         L 101 A \n",
      " 103    0.44    1.67    1.02    0.64   S 104 A A 108 A K 102 A Q 102 A         E 102 A \n",
      " 104    0.47    1.59    1.04    0.41   A 105 A N 109 A A 103 A V 103 A         M 103 A \n",
      " 105    0.28    1.11    0.77    0.26 * N 106 A N 110 A N 104 A N 104 A         N 104 A \n",
      " 106    0.48    1.76    1.03    0.36   A 107 A G 111 A V 105 A G 105 A         A 105 A \n",
      " 107    0.34    1.39    0.80    0.29   E 108 A K 112 A K 106 A K 106 A         R 106 A \n",
      " 108    0.14    0.53    0.34    0.09 * I 109 A I 113 A I 107 A I 107 A         I 107 A \n",
      " 109    0.16    0.67    0.37    0.15 * F 110 A F 114 A F 108 A F 108 A         F 108 A \n",
      " 110    0.38    0.95    0.48    0.19   T 111 A G 115 A K 109 A T 109 A         K 109 A \n",
      " 111    0.32    0.75    0.41    0.13   V 112 A P 116 A C 110 A E 110 A         E 110 A \n",
      " 112    0.11    0.41    0.25    0.05 * Q 113 A Q 117 A Q 111 A Q 111 A         Q 111 A \n",
      " 113    0.11    0.47    0.27    0.08 * G 114 A G 118 A G 112 A G 112 A         G 112 A \n",
      " 114    0.22    0.65    0.34    0.11   A 115 A K 119 A A 113 A R 113 A         E 113 A \n",
      " 115    0.16    0.61    0.32    0.09 * A 116 A A 120 A A 114 A A 114 A         A 114 A \n",
      " 116    0.14    0.44    0.27    0.09   L 117 A I 121 A L 115 A L 115 A         I 115 A \n",
      " 117    0.25    0.78    0.35    0.14   N 118 A N 122 A D 116 A A 116 A         A 116 A \n",
      " 118    0.27    0.85    0.41    0.16   E 119 A D 123 A K 117 A E 117 A         A 117 A \n",
      " 119    0.32    0.93    0.38    0.19   V 120 A N 124 A Y 118 A V 118 A         V 118 A \n",
      " 120    0.19    0.75    0.33    0.14 * A 121 A A 125 A A 119 A A 119 A         A 119 A \n",
      " 121    0.35    1.32    0.56    0.30   S 122 A A 126 A K 120 A K 120 A         A 120 A \n",
      " 122    0.40    1.60    0.79    0.54   R 123 A D 127 A K 121 A K 121 A         S 121 A \n",
      " 123    0.23    0.66    0.66    0.46   D 124 A D 128 A S 122 A D 122 A         D 122 A \n",
      " 124    0.14    0.31    0.23    0.07   V 125 A I 129 A V 123 A V 123 A         C 123 A \n",
      " 125    0.09    0.13    0.22    0.10   K 126 A R 130 A K 124 A K 124 A         R 124 A \n",
      " 126    0.12    0.43    0.24    0.11 * V 127 A V 131 A V 125 A V 125 A         V 125 A \n",
      " 127    0.16    0.45    0.25    0.16   L 128 A L 132 A I 126 A L 126 A         V 126 A \n",
      " 128    0.08    0.28    0.18    0.05 * V 129 A V 133 A V 127 A V 127 A         V 127 A \n",
      " 129    0.11    0.40    0.25    0.07 * V 130 A V 134 A V 128 A V 128 A         V 128 A \n",
      " 130    0.09    0.37    0.28    0.15 * G 131 A G 135 A G 129 A G 129 A         G 129 A \n",
      " 131    0.13    0.41    0.27    0.13 * N 132 A N 136 A N 130 A N 130 A         N 130 A \n",
      " 132    0.28    0.56    0.24    0.13 * P 133 A P 137 A P 131 A P 131 A         P 131 A \n",
      " 133    0.21    0.36    0.23    0.09 * A 134 A A 138 A A 132 A A 132 A         A 132 A \n",
      " 134    0.11    0.34    0.19    0.07 * N 135 A N 139 A N 133 A N 133 A         N 133 A \n",
      " 135    0.11    0.38    0.18    0.06 * T 136 A T 140 A T 134 A T 134 A         T 134 A \n",
      " 136    0.11    0.34    0.20    0.08 * N 137 A N 141 A N 135 A N 135 A         N 135 A \n",
      " 137    0.24    0.48    0.27    0.09   A 138 A A 142 A C 136 A A 136 A         A 136 A \n",
      " 138    0.14    0.45    0.26    0.12   Y 139 A L 143 A L 137 A L 137 A         L 137 A \n",
      " 139    0.24    0.60    0.28    0.13   I 140 A I 144 A T 138 A I 138 A         I 138 A \n",
      " 140    0.12    0.31    0.27    0.08   A 141 A A 145 A A 139 A A 139 A         L 139 A \n",
      " 141    0.17    0.28    0.27    0.08   M 142 A S 146 A S 140 A Y 140 A         L 140 A \n",
      " 142    0.13    0.39    0.31    0.04   K 143 A A 147 A K 141 A K 141 A         K 141 A \n",
      " 143    0.13    0.32    0.44    0.16   S 144 A A 148 A S 142 A N 142 A         S 142 A \n",
      " 144    0.15    0.51    0.54    0.24 * A 145 A A 149 A A 143 A A 143 A         A 143 A \n",
      " 145    0.00    0.00    0.00    0.00   -       -       -       -               Q 144 A \n",
      " 146    0.30    0.62    0.87    0.62   P 146 A P 150 A P 144 A P 144 A         G 145 A \n",
      " 147    0.14    0.11    0.37    0.22   D 147 A D 151 A S 145 A G 145 A         K 146 A \n",
      " 148    0.14    0.26    0.51    0.33   L 148 A V 152 A I 146 A L 146 A         L 147 A \n",
      " 149    0.15    0.38    0.43    0.13   P 149 A P 153 A P 147 A N 147 A         N 148 A \n",
      " 150    0.21    0.55    0.53    0.12   K 150 A A 154 A K 148 A P 148 A         P 149 A \n",
      " 151    0.23    0.55    0.50    0.15   K 151 A S 155 A E 149 A R 149 A         R 150 A \n",
      " 152    0.16    0.36    0.31    0.08   N 152 A R 156 A N 150 A N 150 A         H 151 A \n",
      " 153    0.14    0.42    0.26    0.11   F 153 A F 157 A F 151 A F 151 A         V 152 A \n",
      " 154    0.17    0.47    0.25    0.10   T 154 A N 158 A S 152 A T 152 A         T 153 A \n",
      " 155    0.25    0.52    0.21    0.13   A 155 A A 159 A C 153 A A 153 A         A 154 A \n",
      " 156    0.10    0.09    0.17    0.06   M 156 A M 160 A L 154 A M 154 A         M 155 A \n",
      " 157    0.13    0.13    0.22    0.10   L 157 A M 161 A T 155 A T 155 A         T 156 A \n",
      " 158    0.10    0.22    0.19    0.05 * R 158 A R 162 A R 156 A R 156 A         R 157 A \n",
      " 159    0.08    0.27    0.24    0.07 * L 159 A L 163 A L 157 A L 157 A         L 158 A \n",
      " 160    0.08    0.28    0.24    0.08 * D 160 A D 164 A D 158 A D 158 A         D 159 A \n",
      " 161    0.08    0.24    0.18    0.04 * H 161 A H 165 A H 159 A H 159 A         H 160 A \n",
      " 162    0.05    0.05    0.15    0.09 * N 162 A N 166 A N 160 A N 160 A         N 161 A \n",
      " 163    0.10    0.35    0.23    0.09 * R 163 A R 167 A R 161 A R 161 A         R 162 A \n",
      " 164    0.09    0.29    0.16    0.09 * A 164 A A 168 A A 162 A A 162 A         A 163 A \n",
      " 165    0.15    0.25    0.18    0.08   L 165 A I 169 A K 163 A K 163 A         L 164 A \n",
      " 166    0.16    0.45    0.32    0.10   S 166 A S 170 A A 164 A A 164 A         S 165 A \n",
      " 167    0.13    0.36    0.39    0.06   Q 167 A Q 171 A Q 165 A Q 165 A         L 166 A \n",
      " 168    0.13    0.26    0.29    0.05   L 168 A L 172 A I 166 A L 166 A         L 167 A \n",
      " 169    0.14    0.54    0.42    0.09 * A 169 A A 173 A A 167 A A 167 A         A 168 A \n",
      " 170    0.27    0.75    0.49    0.20   A 170 A T 174 A L 168 A K 168 A         R 169 A \n",
      " 171    0.14    0.54    0.49    0.15 * K 171 A K 175 A K 169 A K 169 A         K 170 A \n",
      " 172    0.32    1.10    0.63    0.28   S 172 A L 176 A L 170 A T 170 A         A 171 A \n",
      " 173    0.31    1.46    0.76    0.30 * G 173 A G 177 A G 171 A G 171 A         G 172 A \n",
      " 174    0.45    1.45    0.79    0.36   K 174 A R 178 A V 172 A T 172 A         V 173 A \n",
      " 175    0.41    1.41    0.76    0.25   P 175 A G 179 A T 173 A G 173 A         P 174 A \n",
      " 176    0.35    1.19    0.64    0.16   V 176 A S 180 A S 174 A V 174 A         V 175 A \n",
      " 177    0.25    0.81    0.61    0.21   A 177 A A 181 A D 175 A D 175 A         S 176 A \n",
      " 178    0.38    1.26    0.58    0.30   S 178 A E 182 A D 176 A R 176 A         Q 177 A \n",
      " 179    0.34    1.25    0.70    0.30   I 179 A F 183 A V 177 A I 177 A         V 178 A \n",
      " 180    0.37    1.41    1.03    0.43   E 180 A N 184 A K 178 A R 178 A         R 179 A \n",
      " 181    0.37    0.95    0.88    0.35   K 181 A N 185 A N 179 A R 179 A         N 180 A \n",
      " 182    0.24    0.79    0.79    0.25   L 182 A I 186 A V 180 A M 180 A         V 181 A \n",
      " 183    0.15    0.29    0.49    0.23   A 183 A V 187 A I 181 A T 181 A         I 182 A \n",
      " 184    0.14    0.42    0.43    0.23   V 184 A V 188 A I 182 A V 182 A         I 183 A \n",
      " 185    0.05    0.13    0.18    0.09 * W 185 A W 189 A W 183 A W 183 A         W 184 A \n",
      " 186    0.13    0.54    0.34    0.10 * G 186 A G 190 A G 184 A G 184 A         G 185 A \n",
      " 187    0.26    0.60    0.59    0.39 * N 187 A N 191 A N 185 A N 185 A         N 186 A \n",
      " 188    0.20    0.58    0.54    0.25 * H 188 A H 192 A H 186 A H 186 A         H 187 A \n",
      " 189    0.23    0.49    0.52    0.13 * S 189 A S 193 A S 187 A S 187 A         S 188 A \n",
      " 190    0.17    0.53    0.60    0.24   P 190 A A 194 A S 188 A S 188 A         S 189 A \n",
      " 191    0.14    0.41    0.61    0.29   T 191 A T 195 A T 189 A I 189 A         T 190 A \n",
      " 192    0.12    0.16    0.38    0.16   M 192 A Q 196 A Q 190 A M 190 A         Q 191 A \n",
      " 193    0.15    0.32    0.37    0.15   Y 193 A F 197 A Y 191 A F 191 A         V 192 A \n",
      " 194    0.05    0.15    0.40    0.26 * P 194 A P 198 A P 192 A P 192 A         P 193 A \n",
      " 195    0.09    0.28    0.38    0.20 * D 195 A D 199 A D 193 A D 193 A         D 194 A \n",
      " 196    0.16    0.34    0.57    0.19   F 196 A I 200 A V 194 A L 194 A         T 195 A \n",
      " 197    0.18    0.32    0.47    0.25   R 197 A T 201 A N 195 A F 195 A         D 196 A \n",
      " 198    0.19    0.43    0.51    0.19   F 198 A Y 202 A H 196 A H 196 A         S 197 A \n",
      " 199    0.31    0.35    0.47    0.18 * A 199 A A 203 A A 197 A A 197 A         A 198 A \n",
      " 200    0.44    0.62    0.55    0.16   T 200 A T 204 A K 198 A E 198 A         V 199 A \n",
      " 201    0.50    0.83    0.67    0.17   A 201 A V 205 A V 199 A V 199 A         I 200 A \n",
      " 202    0.83    1.83    1.34    0.46   E 202 A G 206 A K 200 A D 200 A         G 201 A \n",
      " 203    0.00    0.00    0.00    0.00   -       -       L 201 A -               -       \n",
      " 204    0.00    0.00    0.00    0.00   -       -       Q 202 A -               -       \n",
      " 205    0.00    0.00    0.00    0.00   -       -       A 203 A -               -       \n",
      " 206    0.00    0.00    0.00    0.00   -       -       K 204 A -               -       \n",
      " 207    0.59    1.78    1.41    0.37   G 203 A G 207 A E 205 A G 205 A         T 202 A \n",
      " 208    0.52    0.75    0.76    0.21   E 204 A E 208 A V 206 A R 206 A         T 203 A \n",
      " 209    0.29    0.81    0.64    0.15   S 205 A K 209 A G 207 A P 207 A         P 204 A \n",
      " 210    0.26    0.88    0.62    0.17   L 206 A V 210 A V 208 A A 208 A         A 205 A \n",
      " 211    0.28    0.69    0.80    0.39   L 207 A T 211 A Y 209 A L 209 A         R 206 A \n",
      " 212    0.17    0.52    0.80    0.49   K 208 A D 212 A E 210 A E 210 A         E 207 A \n",
      " 213    0.37    0.73    1.08    0.46   L 209 A L 213 A A 211 A L 211 A         A 208 A \n",
      " 214    0.34    0.80    1.25    0.54   I 210 A V 214 A V 212 A V 212 A         I 209 A \n",
      " 215    0.59    1.12    2.54    1.29   N 211 A -       K 213 A -               K 210 A \n",
      " 216    0.32    1.07    0.71    0.29   D 212 A D 215 A D 214 A D 214 A         -       \n",
      " 217    0.32    0.66    0.34    0.10   D 213 A H 216 A D 215 A M 215 A         -       \n",
      " 218    0.26    0.41    0.53    0.20   V 214 A D 217 A S 216 A E 216 A         -       \n",
      " 219    0.28    0.90    0.64    0.22   W 215 A W 218 A W 217 A W 217 A         -       \n",
      " 220    0.41    1.11    0.61    0.21   N 216 A Y 219 A L 218 A Y 218 A         -       \n",
      " 221    0.53    1.76    0.83    0.39   R 217 A V 220 A K 219 A E 219 A         -       \n",
      " 222    0.52    1.83    0.95    0.31   D 218 A E 221 A G 220 A K 220 A         -       \n",
      " 223    0.39    0.99    0.59    0.36   T 219 A E 222 A E 221 A V 221 A         D 217 A \n",
      " 224    0.20    0.41    0.34    0.20 * F 220 A F 223 A F 222 A F 222 A         F 218 A \n",
      " 225    0.19    0.48    0.37    0.21   I 221 A I 224 A I 223 A I 223 A         V 219 A \n",
      " 226    0.22    0.53    0.51    0.26   P 222 A P 225 A T 224 A P 224 A         Q 220 A \n",
      " 227    0.23    0.78    0.42    0.32   T 223 A R 226 A T 225 A T 225 A         V 221 A \n",
      " 228    0.20    0.86    0.43    0.31 * V 224 A V 227 A V 226 A V 226 A         V 222 A \n",
      " 229    0.33    1.08    0.63    0.32   G 225 A A 228 A Q 227 A A 227 A         R 223 A \n",
      " 230    0.38    1.41    0.72    0.47   K 226 A N 229 A Q 228 A Q 228 A         G 224 A \n",
      " 231    0.31    1.38    0.54    0.45 * R 227 A R 230 A R 229 A R 229 A         R 225 A \n",
      " 232    0.37    1.77    0.68    0.50 * G 228 A G 231 A G 230 A G 230 A         G 226 A \n",
      " 233    0.52    2.44    0.92    0.66 * A 229 A A 232 A A 231 A A 231 A         A 227 A \n",
      " 234    0.54    2.33    0.85    0.69   A 230 A E 233 A A 232 A A 232 A         E 228 A \n",
      " 235    0.46    1.94    0.75    0.54   I 231 A I 234 A V 233 A I 233 A         I 229 A \n",
      " 236    0.73    2.48    0.90    0.64 * I 232 A I 235 A I 234 A I 234 A         I 230 A \n",
      " 237    0.74    3.22    1.08    0.82   E 233 A E 236 A K 235 A Q 235 A         Q 231 A \n",
      " 238    0.69    3.09    1.09    0.74   A 234 A V 237 A A 236 A A 236 A         L 232 A \n",
      " 239    0.69    2.80    1.04    0.69 * R 235 A R 238 A R 237 A R 237 A         R 233 A \n",
      " 240    0.72    2.98    1.04    0.78   G 236 A G 239 A K 238 A G 238 A         G 234 A \n",
      " 241    0.55    2.35    0.80    0.64   L 237 A K 240 A L 239 A A 239 A         L 235 A \n",
      " 242    0.43    2.02    0.80    0.51 * S 238 A S 241 A S 240 A S 240 A         S 236 A \n",
      " 243    0.36    1.26    0.55    0.31 * S 239 A S 242 A S 241 A S 241 A         S 237 A \n",
      " 244    0.24    0.52    0.31    0.07 * A 240 A A 243 A A 242 A A 242 A         A 238 A \n",
      " 245    0.21    0.47    0.27    0.09   A 241 A A 244 A M 243 A A 243 A         M 239 A \n",
      " 246    0.12    0.45    0.27    0.05 * S 242 A S 245 A S 244 A S 244 A         S 240 A \n",
      " 247    0.08    0.20    0.23    0.06 * A 243 A A 246 A A 245 A A 245 A         A 241 A \n",
      " 248    0.06    0.11    0.18    0.06 * A 244 A A 247 A A 246 A A 246 A         A 242 A \n",
      " 249    0.11    0.12    0.18    0.08   N 245 A S 248 A K 247 A N 247 A         K 243 A \n",
      " 250    0.08    0.17    0.18    0.06   A 246 A S 249 A A 248 A A 248 A         A 244 A \n",
      " 251    0.20    0.42    0.23    0.06   A 247 A A 250 A I 249 A A 249 A         A 245 A \n",
      " 252    0.28    0.57    0.33    0.08   I 248 A I 251 A C 250 A I 250 A         V 246 A \n",
      " 253    0.11    0.34    0.26    0.06   D 249 A D 252 A D 251 A E 251 A         D 247 A \n",
      " 254    0.10    0.34    0.23    0.03 * H 250 A H 253 A H 252 A H 252 A         H 248 A \n",
      " 255    0.15    0.48    0.29    0.09   V 251 A M 254 A V 253 A I 253 A         V 249 A \n",
      " 256    0.12    0.31    0.28    0.09   R 252 A R 255 A R 254 A R 254 A         H 250 A \n",
      " 257    0.08    0.24    0.26    0.08 * D 253 A D 256 A D 255 A D 255 A         D 251 A \n",
      " 258    0.32    0.83    0.41    0.16   W 254 A W 257 A I 256 A W 256 A         W 252 A \n",
      " 259    0.36    1.01    0.42    0.23   V 255 A V 258 A W 257 A A 257 A         I 253 A \n",
      " 260    0.20    0.39    0.29    0.07   L 256 A Q 259 A F 258 A L 258 A         H 254 A \n",
      " 261    0.13    0.53    0.40    0.19 * G 257 A G 260 A G 259 A G 259 A         G 255 A \n",
      " 262    0.14    0.45    0.35    0.04 * T 258 A T 261 A T 260 A T 260 A         T 256 A \n",
      " 263    0.23    0.68    1.22    0.55   N 259 A E 262 A P 261 A P 261 A         P 257 A \n",
      " 264    0.40    1.08    1.11    0.63   G 260 A -       E 262 A E 262 A         E 258 A \n",
      " 265    0.20    0.44    0.37    0.07   -       -       G 263 A G 263 A         G 259 A \n",
      " 266    0.39    1.04    0.91    0.39   K 261 A A 263 A E 264 A D 264 A         V 260 A \n",
      " 267    0.28    0.76    0.55    0.24   W 262 A W 264 A F 265 A W 265 A         Y 261 A \n",
      " 268    0.14    0.40    0.43    0.21   V 263 A S 265 A V 266 A V 266 A         V 262 A \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 269    0.06    0.06    0.21    0.11   T 264 A S 266 A S 267 A S 267 A         S 263 A \n",
      " 270    0.13    0.31    0.31    0.13   M 265 A A 267 A M 268 A M 268 A         M 264 A \n",
      " 271    0.09    0.20    0.27    0.07   G 266 A A 268 A G 269 A A 269 A         G 265 A \n",
      " 272    0.30    0.27    0.24    0.07   I 267 A I 269 A I 270 A V 270 A         V 266 A \n",
      " 273    0.42    0.39    0.26    0.09   P 268 A P 270 A I 271 A P 271 A         Y 267 A \n",
      " 274    0.28    0.44    0.30    0.04 * S 269 A S 271 A S 272 A S 272 A         S 268 A \n",
      " 275    0.46    1.10    0.95    0.39   D 270 A T 272 A D 273 A Q 273 A         D 269 A \n",
      " 276    0.54    2.44    1.80    0.45   G 271 A G 273 A G 274 A G 274 A         E 270 A \n",
      " 277    0.35    0.86    1.04    0.49   S 272 A A 274 A N 275 A E 275 A         N 271 A \n",
      " 278    0.52    1.61    0.81    0.00   -       -       S 276 A -               P 272 A \n",
      " 279    0.21    0.78    0.51    0.24 * Y 273 A Y 275 A Y 277 A Y 277 A         Y 273 A \n",
      " 280    0.33    1.44    0.97    0.53 * G 274 A G 276 A G 278 A G 278 A         G 274 A \n",
      " 281    0.27    1.04    0.49    0.26   I 275 A I 277 A V 279 A I 279 A         V 275 A \n",
      " 282    0.25    1.09    0.46    0.31 * P 276 A P 278 A P 280 A P 280 A         P 276 A \n",
      " 283    0.32    1.11    0.52    0.28   E 277 A E 279 A D 281 A E 281 A         S 277 A \n",
      " 284    0.38    1.18    0.88    0.45   D 278 A G 280 A D 282 A G 282 A         G 278 A \n",
      " 285    0.25    0.89    0.59    0.18   I 279 A I 281 A L 283 A I 283 A         L 279 A \n",
      " 286    0.21    0.62    0.34    0.13   I 280 A F 282 A L 284 A V 284 A         I 280 A \n",
      " 287    0.16    0.48    0.31    0.17   Y 281 A V 283 A Y 285 A Y 285 A         F 281 A \n",
      " 288    0.15    0.44    0.30    0.18   G 282 A G 284 A S 286 A S 286 A         S 282 A \n",
      " 289    0.19    0.38    0.22    0.07   V 283 A L 285 A F 287 A F 287 A         F 283 A \n",
      " 290    0.07    0.25    0.36    0.22 * P 284 A P 286 A P 288 A P 288 A         P 284 A \n",
      " 291    0.18    0.52    0.39    0.09   V 285 A T 287 A V 289 A V 289 A         C 285 A \n",
      " 292    0.26    0.67    0.48    0.23   I 286 A V 288 A T 290 A T 290 A         T 286 A \n",
      " 293    0.29    0.70    0.47    0.14   C 287 A S 289 A I 291 A A 291 A         C 287 A \n",
      " 294    0.24    0.74    0.70    0.11   E 288 A R 290 A K 292 A K 292 A         H 288 A \n",
      " 295    0.23    0.75    0.92    0.43   N 289 A N 291 A D 293 A D 293 A         A 289 A \n",
      " 296    0.48    0.55    0.48    0.22   G 290 A G 292 A K 294 A G 294 A         G 290 A \n",
      " 297    0.25    0.65    0.59    0.14   E 291 A E 293 A T 295 A A 295 A         E 291 A \n",
      " 298    0.19    0.63    0.62    0.13   Y 292 A W 294 A W 296 A Y 296 A         W 292 A \n",
      " 299    0.20    0.60    0.55    0.13   K 293 A E 295 A K 297 A R 297 A         T 293 A \n",
      " 300    0.13    0.26    0.41    0.19   R 294 A I 296 A I 298 A V 298 A         V 294 A \n",
      " 301    0.09    0.29    0.32    0.13 * V 295 A V 297 A V 299 A V 299 A         V 295 A \n",
      " 302    0.22    0.83    0.79    0.40   E 296 A E 298 A E 300 A E 300 A         S 296 A \n",
      " 303    0.30    1.43    1.30    0.50 * G 297 A G 299 A G 301 A G 301 A         G 297 A \n",
      " 304    0.24    0.85    1.15    0.64   L 298 A L 300 A L 302 A L 302 A         K 298 A \n",
      " 305    0.54    1.68    1.57    0.96   E 299 A E 301 A P 303 A E 303 A         L 299 A \n",
      " 306    0.29    1.07    0.63    0.16   I 300 A I 302 A I 304 A I 304 A         -       \n",
      " 307    0.31    1.03    0.55    0.19   D 301 A S 303 A N 305 A N 305 A         -       \n",
      " 308    0.25    0.72    0.60    0.12   A 302 A D 304 A D 306 A E 306 A         -       \n",
      " 309    0.12    0.24    0.53    0.26   F 303 A F 305 A F 307 A F 307 A         -       \n",
      " 310    0.15    0.20    0.44    0.23   S 304 A Q 306 A S 308 A A 308 A         -       \n",
      " 311    0.30    0.93    0.98    0.65   R 305 A R 307 A R 309 A R 309 A         K 305 A \n",
      " 312    0.28    0.70    1.00    0.73   E 306 A A 308 A E 310 A K 310 A         Q 306 A \n",
      " 313    0.24    0.68    0.93    0.68   K 307 A R 309 A K 311 A R 311 A         R 307 A \n",
      " 314    0.20    0.44    0.56    0.28   M 308 A I 310 A M 312 A M 312 A         L 308 A \n",
      " 315    0.21    0.66    0.67    0.30   D 309 A D 311 A D 313 A E 313 A         A 309 A \n",
      " 316    0.24    0.62    0.57    0.26   G 310 A A 312 A L 314 A I 314 A         S 310 A \n",
      " 317    0.12    0.38    0.52    0.26   T 311 A N 313 A T 315 A T 315 A         T 311 A \n",
      " 318    0.11    0.16    0.53    0.30   L 312 A A 314 A A 316 A A 316 A         I 312 A \n",
      " 319    0.21    0.55    0.63    0.23   A 313 A Q 315 A K 317 A Q 317 A         A 313 A \n",
      " 320    0.32    0.65    0.62    0.20 * E 314 A E 316 A E 318 A E 318 A         E 314 A \n",
      " 321    0.11    0.45    0.40    0.15 * L 315 A L 317 A L 319 A L 319 A         L 315 A \n",
      " 322    0.19    0.38    0.47    0.17   L 316 A Q 318 A A 320 A L 320 A         Q 316 A \n",
      " 323    0.16    0.46    0.62    0.23   E 317 A A 319 A E 321 A D 321 A         E 317 A \n",
      " 324    0.15    0.59    0.56    0.23 * E 318 A E 320 A E 322 A E 322 A         E 318 A \n",
      " 325    0.22    0.59    0.49    0.19   R 319 A R 321 A K 323 A M 323 A         R 319 A \n",
      " 326    0.24    0.83    0.60    0.32   D 320 A E 322 A E 324 A E 324 A         A 320 A \n",
      " 327    0.28    0.80    0.68    0.39   G 321 A A 323 A T 325 A Q 325 A         Q 321 A \n",
      " 328    0.39    1.29    0.76    0.37   V 322 A V 324 A A 326 A V 326 A         A 322 A \n",
      " 329    0.49    1.11    0.93    0.36   A 323 A R 325 A F 327 A K 327 A         G 323 A \n",
      " 330    0.71    2.46    1.99    0.46   H 324 A D 326 A E 328 A A 328 A         -       \n",
      " 331    0.92    3.44    2.69    0.74   -       L 327 A F 329 A L 329 A         -       \n",
      " 332    1.23    4.07    3.05    0.49   -       L 328 A L 330 A G 330 A         -       \n",
      " 333    1.61    5.85    3.37    0.97   -       L 329 A S 331 A L 331 A         -       \n",
      " 334    2.24   10.03    4.61    2.42   L 325 A E 330 A S 332 A I 332 A         L 324 A \n",
      " 335    0.00    0.00    0.00    0.00   -       -       A 333 A -               -       \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    5\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        1      327      327        1.1699        1.1699        0.0000\n",
      "rr_dist_318_> I,NEQVCUT,NEQV,RMSCUT,RMS,RMSROT:        2      327      327        1.1699        1.1699        0.0000\n",
      "\n",
      "Current alignment total score:        132.1\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -       -       -       D  23 A -       \n",
      "   2    0.00    0.00    0.00    0.00   -       -       -       -       C  24 A -       \n",
      "   3    0.00    0.00    0.00    0.00   -       -       -       -       F  25 A -       \n",
      "   4    0.00    0.00    0.00    0.00   -       -       -       -       G  26 A -       \n",
      "   5    0.00    0.00    0.00    0.00   -       -       -       -       V  27 A -       \n",
      "   6    0.00    0.00    0.00    0.00   -       -       -       -       F  28 A -       \n",
      "   7    0.00    0.00    0.00    0.00   -       -       -       -       C  29 A -       \n",
      "   8    0.00    0.00    0.00    0.00   -       -       -       -       T  30 A -       \n",
      "   9    0.29    0.50    0.25    0.00   -       -       -       M   0 A W  43 A -       \n",
      "  10    0.49    1.85    1.23    0.64   K   3 A -       S   1 A K   1 A K  44 A M   1 A \n",
      "  11    0.52    2.11    1.13    0.58   P   4 A S   8 A E   2 A A   2 A K  45 A S   2 A \n",
      "  12    0.42    1.20    0.60    0.27   A   5 A T   9 A P   3 A P   3 A L  46 A A   3 A \n",
      "  13    0.20    0.53    0.29    0.14   K   6 A K  10 A I   4 A V   4 A V  47 A V   4 A \n",
      "  14    0.18    0.36    0.25    0.10   R   7 A K  11 A R   5 A R   5 A N  48 A K   5 A \n",
      "  15    0.13    0.32    0.21    0.04   V   8 A V  12 A V   6 A V   6 A I  49 A V   6 A \n",
      "  16    0.13    0.37    0.27    0.08   A   9 A T  13 A L   7 A A   7 A A  50 A A   7 A \n",
      "  17    0.11    0.40    0.32    0.06 * V  10 A V  14 A V   8 A V   8 A V  51 A V   8 A \n",
      "  18    0.13    0.32    0.25    0.11   T  11 A T  15 A T   9 A T   9 A S  52 A T   9 A \n",
      "  19    0.19    0.41    0.35    0.14 * G  12 A G  16 A G  10 A G  10 A G  53 A G  10 A \n",
      "  20    0.12    0.46    0.38    0.07 * A  13 A A  17 A A  11 A A  11 A A  54 A A  11 A \n",
      "  21    0.15    0.58    0.37    0.07 * A  14 A A  18 A A  12 A A  12 A A  55 A A  12 A \n",
      "  22    0.13    0.61    0.29    0.13 * G  15 A G  19 A G  13 A G  13 A G  56 A G  13 A \n",
      "  23    0.27    0.69    0.32    0.15   Q  16 A Q  20 A Q  14 A Q  14 A M  57 A Q  14 A \n",
      "  24    0.18    0.78    0.28    0.19 * I  17 A I  21 A I  15 A I  15 A I  58 A I  15 A \n",
      "  25    0.21    0.64    0.29    0.15   A  18 A S  22 A A  16 A G  16 A S  59 A G  16 A \n",
      "  26    0.23    0.54    0.23    0.12   Y  19 A Y  23 A Y  17 A Y  17 A N  60 A Y  17 A \n",
      "  27    0.36    1.13    0.38    0.27   S  20 A S  24 A S  18 A S  18 A H  61 A A  18 A \n",
      "  28    0.15    0.65    0.34    0.12 * L  21 A L  25 A L  19 A L  19 A L  62 A L  19 A \n",
      "  29    0.13    0.37    0.27    0.11   L  22 A L  26 A L  20 A L  20 A L  63 A V  20 A \n",
      "  30    0.15    0.37    0.39    0.12   F  23 A W  27 A Y  21 A F  21 A F  64 A P  21 A \n",
      "  31    0.14    0.26    0.42    0.12   R  24 A R  28 A S  22 A R  22 A K  65 A L  22 A \n",
      "  32    0.16    0.42    0.35    0.13   I  25 A I  29 A I  23 A I  23 A L  66 A I  23 A \n",
      "  33    0.11    0.32    0.27    0.07   A  26 A A  30 A G  24 A A  24 A A  67 A A  24 A \n",
      "  34    0.22    0.63    0.37    0.13   N  27 A N  31 A N  25 A A  25 A S  68 A R  25 A \n",
      "  35    0.12    0.56    0.43    0.11 * G  28 A G  32 A G  26 A G  26 A G  69 A G  26 A \n",
      "  36    0.38    1.52    0.83    0.23   D  29 A E  33 A S  27 A E  27 A E  70 A A  27 A \n",
      "  37    0.37    1.54    0.83    0.24   L  30 A V  34 A V  28 A M  28 A V  71 A L  28 A \n",
      "  38    0.28    1.11    0.60    0.25   L  31 A F  35 A F  29 A L  29 A F  72 A L  29 A \n",
      "  39    0.21    1.01    0.62    0.18 * G  32 A G  36 A G  30 A G  30 A G  73 A G  30 A \n",
      "  40    0.31    1.00    0.54    0.21   K  33 A T  37 A K  31 A K  31 A Q  74 A P  31 A \n",
      "  41    0.20    0.77    0.61    0.12   D  34 A D  38 A D  32 A D  32 A D  75 A T  32 A \n",
      "  42    0.22    0.38    0.38    0.14   Q  35 A T  39 A Q  33 A Q  33 A Q  76 A T  33 A \n",
      "  43    0.09    0.35    0.41    0.11 * P  36 A P  40 A P  34 A P  34 A P  77 A P  34 A \n",
      "  44    0.14    0.42    0.32    0.06   V  37 A V  41 A I  35 A V  35 A I  78 A V  35 A \n",
      "  45    0.17    0.30    0.40    0.20   I  38 A E  42 A I  36 A I  36 A A  79 A E  36 A \n",
      "  46    0.07    0.25    0.26    0.12 * L  39 A L  43 A L  37 A L  37 A L  80 A L  37 A \n",
      "  47    0.13    0.26    0.24    0.05   Q  40 A K  44 A V  38 A Q  38 A K  81 A R  38 A \n",
      "  48    0.04    0.07    0.35    0.16 * L  41 A L  45 A L  39 A L  39 A L  82 A L  39 A \n",
      "  49    0.05    0.13    0.32    0.14 * L  42 A L  46 A L  40 A L  40 A L  83 A L  40 A \n",
      "  50    0.20    0.48    0.42    0.14   D  43 A E  47 A D  41 A E  41 A G  84 A D  41 A \n",
      "  51    0.45    0.98    0.58    0.23   L  44 A I  48 A I  42 A I  42 A S  85 A I  42 A \n",
      "  52    0.45    1.14    0.63    0.33   P  45 A P  49 A T  43 A P  43 A E  86 A E  43 A \n",
      "  53    0.52    2.00    0.86    0.55   Q  46 A Q  50 A P  44 A Q  44 A R  87 A P  44 A \n",
      "  54    0.29    0.95    0.58    0.25   A  47 A A  51 A M  45 A A  45 A S  88 A A  45 A \n",
      "  55    0.32    1.02    0.59    0.28   Q  48 A L  52 A M  46 A M  46 A F  89 A L  46 A \n",
      "  56    0.30    0.98    0.62    0.27   A  49 A G  53 A G  47 A K  47 A Q  90 A K  47 A \n",
      "  57    0.21    0.81    0.53    0.23   A  50 A G  54 A V  48 A A  48 A A  91 A A  48 A \n",
      "  58    0.18    0.67    0.50    0.33   V  51 A A  55 A L  49 A L  49 A L  92 A L  49 A \n",
      "  59    0.20    0.68    0.55    0.39   K  52 A E  56 A D  50 A E  50 A E  93 A A  50 A \n",
      "  60    0.12    0.56    0.49    0.31 * G  53 A G  57 A G  51 A G  51 A G  94 A G  51 A \n",
      "  61    0.09    0.33    0.42    0.22 * V  54 A V  58 A V  52 A V  52 A V  95 A V  52 A \n",
      "  62    0.25    0.65    0.57    0.32   V  55 A A  59 A L  53 A V  53 A A  96 A E  53 A \n",
      "  63    0.22    0.78    0.60    0.42   M  56 A M  60 A M  54 A M  54 A M  97 A A  54 A \n",
      "  64    0.14    0.54    0.43    0.26 * E  57 A E  61 A E  55 A E  55 A E  98 A E  55 A \n",
      "  65    0.18    0.81    0.46    0.23 * L  58 A L  62 A L  56 A L  56 A L  99 A L  56 A \n",
      "  66    0.30    1.12    0.68    0.44   D  59 A L  63 A Q  57 A E  57 A E 100 A E  57 A \n",
      "  67    0.21    0.94    0.68    0.49 * D  60 A D  64 A D  58 A D  58 A D 101 A D  58 A \n",
      "  68    0.34    1.04    0.56    0.44   C  61 A S  65 A C  59 A C  59 A S 102 A C  59 A \n",
      "  69    0.32    1.06    0.76    0.62   A  62 A A  66 A A  60 A A  60 A L 103 A A  60 A \n",
      "  70    0.22    0.46    0.32    0.06   F  63 A F  67 A L  61 A F  61 A Y 104 A F  61 A \n",
      "  71    0.18    0.80    0.56    0.22 * P  64 A P  68 A P  62 A P  62 A P 105 A P  62 A \n",
      "  72    0.06    0.18    0.29    0.09 * L  65 A L  69 A L  63 A L  63 A L 106 A L  63 A \n",
      "  73    0.10    0.36    0.28    0.10 * L  66 A L  70 A L  64 A L  64 A L 107 A L  64 A \n",
      "  74    0.17    0.15    0.35    0.18   A  67 A R  71 A K  65 A A  65 A R 108 A D  65 A \n",
      "  75    0.45    0.75    0.42    0.11   G  68 A N  72 A D  66 A G  66 A E 109 A K  66 A \n",
      "  76    0.30    0.26    0.33    0.05   V  69 A I  73 A V  67 A L  67 A V 110 A V  67 A \n",
      "  77    0.52    0.45    0.34    0.13   V  70 A T  74 A I  68 A E  68 A S 111 A V  68 A \n",
      "  78    0.38    0.54    0.46    0.14   I  71 A I  75 A A  69 A A  69 A I 112 A V  69 A \n",
      "  79    0.55    0.52    0.45    0.12   T  72 A T  76 A T  70 A T  70 A G 113 A T  70 A \n",
      "  80    0.34    0.61    0.45    0.18   D  73 A A  77 A D  71 A D  71 A I 114 A A  71 A \n",
      "  81    0.25    1.04    0.55    0.30   D  74 A D  78 A K  72 A D  72 A D 115 A D  72 A \n",
      "  82    0.28    0.74    0.55    0.42   P  75 A A  79 A E  73 A P  73 A P 116 A P  73 A \n",
      "  83    0.38    1.24    0.72    0.48   K  76 A N  80 A E  74 A D  74 A Y 117 A R  74 A \n",
      "  84    0.39    1.38    0.53    0.32   V  77 A E  81 A I  75 A V  75 A E 118 A V  75 A \n",
      "  85    0.24    0.76    0.39    0.18   A  78 A A  82 A A  76 A A  76 A V 119 A A  76 A \n",
      "  86    0.09    0.34    0.38    0.25 * F  79 A F  83 A F  77 A F  77 A F 120 A F  77 A \n",
      "  87    0.15    0.32    0.35    0.12   K  80 A D  84 A K  78 A K  78 A E 121 A D  78 A \n",
      "  88    0.09    0.18    0.33    0.18   D  81 A G  85 A D  79 A D  79 A D 122 A G  79 A \n",
      "  89    0.11    0.19    0.29    0.13   A  82 A A  86 A L  80 A A  80 A V 123 A V  80 A \n",
      "  90    0.17    0.37    0.26    0.09   D  83 A N  87 A D  81 A D  81 A D 124 A A  81 A \n",
      "  91    0.23    0.39    0.26    0.08   V  84 A A  88 A V  82 A Y  82 A W 125 A I  82 A \n",
      "  92    0.13    0.48    0.33    0.17 * A  85 A A  89 A A  83 A A  83 A A 126 A A  83 A \n",
      "  93    0.08    0.13    0.30    0.13   L  86 A F  90 A I  84 A L  84 A L 127 A I  84 A \n",
      "  94    0.25    0.11    0.25    0.14   L  87 A L  91 A L  85 A L  85 A L 128 A M  85 A \n",
      "  95    0.33    0.22    0.28    0.08   V  88 A V  92 A V  86 A V  86 A I 129 A C  86 A \n",
      "  96    0.14    0.25    0.32    0.11 * G  89 A G  93 A G  87 A G  87 A G 130 A G  87 A \n",
      "  97    0.12    0.20    0.48    0.23   A  90 A A  94 A S  88 A A  88 A A 131 A A  88 A \n",
      "  98    0.23    0.41    1.03    0.49   R  91 A K  95 A M  89 A A  89 A K 132 A F  89 A \n",
      "  99    0.13    0.45    1.27    0.89 * P  92 A P  96 A P  90 A P  90 A P 133 A P  90 A \n",
      " 100    0.31    1.13    1.45    0.98   R  93 A -       R  91 A         R 134 A R  91 A \n",
      " 101    0.83    3.36    2.55    1.58   S  94 A -       R  92 A -       G 135 A K  92 A \n",
      " 102    1.45    6.42    3.99    1.71   -       -       D  93 A -       P 136 A A  93 A \n",
      " 103    1.14    5.07    3.97    1.44   -       -       G  94 A -       G 137 A G  94 A \n",
      " 104    0.66    2.69    3.07    1.94   M  97 A -       M  95 A -       M 138 A M  95 A \n",
      " 105    0.75    3.20    2.72    2.08   E  98 A E 102 A E  96 A -       E 139 A E  96 A \n",
      " 106    0.71    2.31    2.23    1.71   R  99 A R 103 A R  97 A -       R 140 A R  97 A \n",
      " 107    0.55    2.08    2.20    1.67   K 100 A A 104 A K  98 A -       A 141 A K  98 A \n",
      " 108    0.58    2.13    1.79    1.33   D 101 A D 105 A D  99 A -       A 142 A D  99 A \n",
      " 109    0.35    1.32    0.86    0.44   L 102 A L 106 A L 100 A -       L 143 A L 100 A \n",
      " 110    0.20    0.60    0.95    0.58 * L 103 A L 107 A L 101 A L 101 A L 144 A L 101 A \n",
      " 111    0.30    1.00    1.00    0.59   S 104 A A 108 A K 102 A Q 102 A D 145 A E 102 A \n",
      " 112    0.33    0.92    1.02    0.36   A 105 A N 109 A A 103 A V 103 A I 146 A M 103 A \n",
      " 113    0.10    0.25    0.67    0.32 * N 106 A N 110 A N 104 A N 104 A N 147 A N 104 A \n",
      " 114    0.25    0.92    1.01    0.31   A 107 A G 111 A V 105 A G 105 A G 148 A A 105 A \n",
      " 115    0.29    0.97    0.83    0.22   E 108 A K 112 A K 106 A K 106 A Q 149 A R 106 A \n",
      " 116    0.11    0.42    0.36    0.06 * I 109 A I 113 A I 107 A I 107 A I 150 A I 107 A \n",
      " 117    0.09    0.32    0.35    0.14 * F 110 A F 114 A F 108 A F 108 A F 151 A F 108 A \n",
      " 118    0.31    0.47    0.47    0.17   T 111 A G 115 A K 109 A T 109 A A 152 A K 109 A \n",
      " 119    0.21    0.45    0.41    0.12   V 112 A P 116 A C 110 A E 110 A D 153 A E 110 A \n",
      " 120    0.06    0.17    0.24    0.06 * Q 113 A Q 117 A Q 111 A Q 111 A Q 154 A Q 111 A \n",
      " 121    0.07    0.27    0.27    0.07 * G 114 A G 118 A G 112 A G 112 A G 155 A G 112 A \n",
      " 122    0.11    0.17    0.31    0.13   A 115 A K 119 A A 113 A R 113 A K 156 A E 113 A \n",
      " 123    0.08    0.24    0.30    0.09 * A 116 A A 120 A A 114 A A 114 A A 157 A A 114 A \n",
      " 124    0.09    0.26    0.26    0.09   L 117 A I 121 A L 115 A L 115 A L 158 A I 115 A \n",
      " 125    0.13    0.26    0.34    0.12   N 118 A N 122 A D 116 A A 116 A N 159 A A 116 A \n",
      " 126    0.15    0.32    0.40    0.13   E 119 A D 123 A K 117 A E 117 A A 160 A A 117 A \n",
      " 127    0.14    0.36    0.38    0.15   V 120 A N 124 A Y 118 A V 118 A V 161 A V 118 A \n",
      " 128    0.13    0.47    0.36    0.10 * A 121 A A 125 A A 119 A A 119 A A 162 A A 119 A \n",
      " 129    0.20    0.59    0.56    0.26   S 122 A A 126 A K 120 A K 120 A S 163 A A 120 A \n",
      " 130    0.26    0.97    0.78    0.53   R 123 A D 127 A K 121 A K 121 A K 164 A S 121 A \n",
      " 131    0.31    1.14    0.69    0.50   D 124 A D 128 A S 122 A D 122 A N 165 A D 122 A \n",
      " 132    0.17    0.55    0.27    0.14   V 125 A I 129 A V 123 A V 123 A V 166 A C 123 A \n",
      " 133    0.15    0.49    0.26    0.10   K 126 A R 130 A K 124 A K 124 A K 167 A R 124 A \n",
      " 134    0.11    0.43    0.27    0.11 * V 127 A V 131 A V 125 A V 125 A V 168 A V 125 A \n",
      " 135    0.06    0.11    0.23    0.16   L 128 A L 132 A I 126 A L 126 A L 169 A V 126 A \n",
      " 136    0.07    0.24    0.18    0.05 * V 129 A V 133 A V 127 A V 127 A V 170 A V 127 A \n",
      " 137    0.12    0.49    0.29    0.09 * V 130 A V 134 A V 128 A V 128 A V 171 A V 128 A \n",
      " 138    0.14    0.65    0.32    0.19 * G 131 A G 135 A G 129 A G 129 A G 172 A G 129 A \n",
      " 139    0.14    0.51    0.30    0.15 * N 132 A N 136 A N 130 A N 130 A N 173 A N 130 A \n",
      " 140    0.23    0.48    0.27    0.14 * P 133 A P 137 A P 131 A P 131 A P 174 A P 131 A \n",
      " 141    0.24    0.38    0.25    0.09   A 134 A A 138 A A 132 A A 132 A C 175 A A 132 A \n",
      " 142    0.10    0.31    0.21    0.07 * N 135 A N 139 A N 133 A N 133 A N 176 A N 133 A \n",
      " 143    0.09    0.32    0.21    0.05 * T 136 A T 140 A T 134 A T 134 A T 177 A T 134 A \n",
      " 144    0.11    0.36    0.23    0.06 * N 137 A N 141 A N 135 A N 135 A N 178 A N 135 A \n",
      " 145    0.09    0.21    0.26    0.08   A 138 A A 142 A C 136 A A 136 A A 179 A A 136 A \n",
      " 146    0.10    0.29    0.26    0.10   Y 139 A L 143 A L 137 A L 137 A L 180 A L 137 A \n",
      " 147    0.13    0.44    0.31    0.10   I 140 A I 144 A T 138 A I 138 A I 181 A I 138 A \n",
      " 148    0.21    0.34    0.27    0.08   A 141 A A 145 A A 139 A A 139 A C 182 A L 139 A \n",
      " 149    0.16    0.33    0.28    0.06   M 142 A S 146 A S 140 A Y 140 A L 183 A L 140 A \n",
      " 150    0.10    0.26    0.29    0.06   K 143 A A 147 A K 141 A K 141 A K 184 A K 141 A \n",
      " 151    0.15    0.30    0.40    0.19   S 144 A A 148 A S 142 A N 142 A N 185 A S 142 A \n",
      " 152    0.11    0.37    0.49    0.26 * A 145 A A 149 A A 143 A A 143 A A 186 A A 143 A \n",
      " 153    0.00    0.00    0.00    0.00   -       -       -       -       -       Q 144 A \n",
      " 154    0.25    0.64    0.79    0.62   P 146 A P 150 A P 144 A P 144 A P 187 A G 145 A \n",
      " 155    0.16    0.40    0.36    0.22   D 147 A D 151 A S 145 A G 145 A D 188 A K 146 A \n",
      " 156    0.12    0.28    0.46    0.32   L 148 A V 152 A I 146 A L 146 A I 189 A L 147 A \n",
      " 157    0.11    0.27    0.40    0.13   P 149 A P 153 A P 147 A N 147 A P 190 A N 148 A \n",
      " 158    0.14    0.29    0.49    0.14   K 150 A A 154 A K 148 A P 148 A A 191 A P 149 A \n",
      " 159    0.14    0.33    0.46    0.18   K 151 A S 155 A E 149 A R 149 A K 192 A R 150 A \n",
      " 160    0.18    0.56    0.34    0.12   N 152 A R 156 A N 150 A N 150 A N 193 A H 151 A \n",
      " 161    0.17    0.61    0.32    0.13   F 153 A F 157 A F 151 A F 151 A F 194 A V 152 A \n",
      " 162    0.27    0.67    0.32    0.14   T 154 A N 158 A S 152 A T 152 A H 195 A T 153 A \n",
      " 163    0.20    0.74    0.29    0.21   A 155 A A 159 A C 153 A A 153 A A 196 A A 154 A \n",
      " 164    0.10    0.14    0.17    0.05   M 156 A M 160 A L 154 A M 154 A L 197 A M 155 A \n",
      " 165    0.22    0.65    0.27    0.18   L 157 A M 161 A T 155 A T 155 A T 198 A T 156 A \n",
      " 166    0.11    0.30    0.20    0.06 * R 158 A R 162 A R 156 A R 156 A R 199 A R 157 A \n",
      " 167    0.12    0.50    0.27    0.10 * L 159 A L 163 A L 157 A L 157 A L 200 A L 158 A \n",
      " 168    0.15    0.63    0.30    0.13 * D 160 A D 164 A D 158 A D 158 A D 201 A D 159 A \n",
      " 169    0.22    0.44    0.21    0.10   H 161 A H 165 A H 159 A H 159 A E 202 A H 160 A \n",
      " 170    0.12    0.41    0.20    0.10 * N 162 A N 166 A N 160 A N 160 A N 203 A N 161 A \n",
      " 171    0.14    0.57    0.28    0.13 * R 163 A R 167 A R 161 A R 161 A R 204 A R 162 A \n",
      " 172    0.07    0.18    0.16    0.08 * A 164 A A 168 A A 162 A A 162 A A 205 A A 163 A \n",
      " 173    0.13    0.22    0.17    0.09   L 165 A I 169 A K 163 A K 163 A K 206 A L 164 A \n",
      " 174    0.24    0.47    0.33    0.10   S 166 A S 170 A A 164 A A 164 A C 207 A S 165 A \n",
      " 175    0.12    0.34    0.38    0.06   Q 167 A Q 171 A Q 165 A Q 165 A Q 208 A L 166 A \n",
      " 176    0.14    0.55    0.33    0.08   L 168 A L 172 A I 166 A L 166 A L 209 A L 167 A \n",
      " 177    0.19    0.81    0.46    0.17 * A 169 A A 173 A A 167 A A 167 A A 210 A A 168 A \n",
      " 178    0.28    0.89    0.56    0.19   A 170 A T 174 A L 168 A K 168 A L 211 A R 169 A \n",
      " 179    0.21    0.87    0.55    0.13 * K 171 A K 175 A K 169 A K 169 A K 212 A K 170 A \n",
      " 180    0.28    0.97    0.68    0.27   S 172 A L 176 A L 170 A T 170 A A 213 A A 171 A \n",
      " 181    0.38    1.33    0.83    0.34 * G 173 A G 177 A G 171 A G 171 A G 214 A G 172 A \n",
      " 182    0.24    0.81    0.79    0.31   K 174 A R 178 A V 172 A T 172 A V 215 A V 173 A \n",
      " 183    0.39    1.18    0.83    0.22   P 175 A G 179 A T 173 A G 173 A F 216 A P 174 A \n",
      " 184    0.27    0.71    0.64    0.16   V 176 A S 180 A S 174 A V 174 A Y 217 A V 175 A \n",
      " 185    0.24    0.85    0.65    0.16   A 177 A A 181 A D 175 A D 175 A D 218 A S 176 A \n",
      " 186    0.27    0.85    0.62    0.28   S 178 A E 182 A D 176 A R 176 A K 219 A Q 177 A \n",
      " 187    0.19    0.63    0.68    0.26   I 179 A F 183 A V 177 A I 177 A V 220 A V 178 A \n",
      " 188    0.53    0.94    1.01    0.38   E 180 A N 184 A K 178 A R 178 A S 221 A R 179 A \n",
      " 189    0.24    0.46    0.79    0.38   K 181 A N 185 A N 179 A R 179 A N 222 A N 180 A \n",
      " 190    0.22    0.77    0.76    0.26   L 182 A I 186 A V 180 A M 180 A V 223 A V 181 A \n",
      " 191    0.24    0.77    0.51    0.25   A 183 A V 187 A I 181 A T 181 A T 224 A I 182 A \n",
      " 192    0.18    0.67    0.45    0.24   V 184 A V 188 A I 182 A V 182 A I 225 A I 183 A \n",
      " 193    0.11    0.48    0.24    0.08 * W 185 A W 189 A W 183 A W 183 A W 226 A W 184 A \n",
      " 194    0.08    0.35    0.34    0.08 * G 186 A G 190 A G 184 A G 184 A G 227 A G 185 A \n",
      " 195    0.30    0.68    0.56    0.41 * N 187 A N 191 A N 185 A N 185 A N 228 A N 186 A \n",
      " 196    0.14    0.57    0.52    0.26 * H 188 A H 192 A H 186 A H 186 A H 229 A H 187 A \n",
      " 197    0.18    0.36    0.48    0.16 * S 189 A S 193 A S 187 A S 187 A S 230 A S 188 A \n",
      " 198    0.20    0.57    0.58    0.25   P 190 A A 194 A S 188 A S 188 A T 231 A S 189 A \n",
      " 199    0.15    0.54    0.59    0.27   T 191 A T 195 A T 189 A I 189 A T 232 A T 190 A \n",
      " 200    0.16    0.45    0.39    0.13   M 192 A Q 196 A Q 190 A M 190 A Q 233 A Q 191 A \n",
      " 201    0.20    0.49    0.39    0.12   Y 193 A F 197 A Y 191 A F 191 A V 234 A V 192 A \n",
      " 202    0.05    0.14    0.36    0.25 * P 194 A P 198 A P 192 A P 192 A P 235 A P 193 A \n",
      " 203    0.13    0.52    0.39    0.20 * D 195 A D 199 A D 193 A D 193 A D 236 A D 194 A \n",
      " 204    0.35    1.25    0.68    0.24   F 196 A I 200 A V 194 A L 194 A F 237 A T 195 A \n",
      " 205    0.34    1.02    0.56    0.25   R 197 A T 201 A N 195 A F 195 A L 238 A D 196 A \n",
      " 206    0.44    1.63    0.67    0.40   F 198 A Y 202 A H 196 A H 196 A N 239 A S 197 A \n",
      " 207    0.40    1.64    0.66    0.38 * A 199 A A 203 A A 197 A A 197 A A 240 A A 198 A \n",
      " 208    0.43    1.53    0.67    0.39   T 200 A T 204 A K 198 A E 198 A K 241 A V 199 A \n",
      " 209    0.36    1.25    0.75    0.22   A 201 A V 205 A V 199 A V 199 A I 242 A I 200 A \n",
      " 210    0.40    1.31    1.31    0.45   E 202 A G 206 A K 200 A D 200 A D 243 A G 201 A \n",
      " 211    0.00    0.00    0.00    0.00   -       -       L 201 A -       -       -       \n",
      " 212    0.00    0.00    0.00    0.00   -       -       Q 202 A -       -       -       \n",
      " 213    0.00    0.00    0.00    0.00   -       -       A 203 A -       -       -       \n",
      " 214    0.00    0.00    0.00    0.00   -       -       K 204 A -       -       -       \n",
      " 215    0.53    2.35    1.53    0.43   G 203 A G 207 A E 205 A G 205 A G 244 A T 202 A \n",
      " 216    0.56    2.07    0.94    0.48   E 204 A E 208 A V 206 A R 206 A R 245 A T 203 A \n",
      " 217    0.37    1.48    0.75    0.32   S 205 A K 209 A G 207 A P 207 A P 246 A P 204 A \n",
      " 218    0.23    0.85    0.63    0.24   L 206 A V 210 A V 208 A A 208 A V 247 A A 205 A \n",
      " 219    0.29    0.91    0.78    0.41   L 207 A T 211 A Y 209 A L 209 A K 248 A R 206 A \n",
      " 220    0.31    1.25    0.86    0.46   K 208 A D 212 A E 210 A E 210 A E 249 A E 207 A \n",
      " 221    0.26    0.88    1.02    0.46   L 209 A L 213 A A 211 A L 211 A V 250 A A 208 A \n",
      " 222    0.26    0.74    1.14    0.57   I 210 A V 214 A V 212 A V 212 A I 251 A I 209 A \n",
      " 223    0.38    1.03    2.04    1.46   N 211 A -       K 213 A -       K 252 A K 210 A \n",
      " 224    0.58    1.63    0.85    0.41   D 212 A D 215 A D 214 A D 214 A R 253 A -       \n",
      " 225    0.47    1.52    0.60    0.33   D 213 A H 216 A D 215 A M 215 A T 254 A -       \n",
      " 226    0.35    1.00    0.62    0.17   V 214 A D 217 A S 216 A E 216 A K 255 A -       \n",
      " 227    0.15    0.37    0.58    0.23   W 215 A W 218 A W 217 A W 217 A W 256 A -       \n",
      " 228    0.34    0.95    0.65    0.22   N 216 A Y 219 A L 218 A Y 218 A L 257 A -       \n",
      " 229    0.46    1.61    1.00    0.29   R 217 A V 220 A K 219 A E 219 A E 258 A -       \n",
      " 230    0.43    1.60    1.06    0.28   D 218 A E 221 A G 220 A K 220 A E 259 A -       \n",
      " 231    0.31    1.02    0.64    0.36   T 219 A E 222 A E 221 A V 221 A E 260 A D 217 A \n",
      " 232    0.19    0.74    0.41    0.19 * F 220 A F 223 A F 222 A F 222 A F 261 A F 218 A \n",
      " 233    0.36    0.61    0.38    0.24   I 221 A I 224 A I 223 A I 223 A T 262 A V 219 A \n",
      " 234    0.29    0.73    0.55    0.20   P 222 A P 225 A T 224 A P 224 A I 263 A Q 220 A \n",
      " 235    0.25    0.96    0.55    0.25   T 223 A R 226 A T 225 A T 225 A T 264 A V 221 A \n",
      " 236    0.16    0.67    0.47    0.27 * V 224 A V 227 A V 226 A V 226 A V 265 A V 222 A \n",
      " 237    0.43    1.25    0.73    0.31   G 225 A A 228 A Q 227 A A 227 A Q 266 A R 223 A \n",
      " 238    0.44    1.79    0.89    0.52   K 226 A N 229 A Q 228 A Q 228 A K 267 A G 224 A \n",
      " 239    0.37    1.72    0.76    0.49 * R 227 A R 230 A R 229 A R 229 A R 268 A R 225 A \n",
      " 240    0.39    1.92    0.86    0.59 * G 228 A G 231 A G 230 A G 230 A G 269 A G 226 A \n",
      " 241    0.55    2.32    1.11    0.77   A 229 A A 232 A A 231 A A 231 A G 270 A A 227 A \n",
      " 242    0.51    2.26    1.04    0.79   A 230 A E 233 A A 232 A A 232 A A 271 A E 228 A \n",
      " 243    0.48    2.03    0.93    0.65   I 231 A I 234 A V 233 A I 233 A L 272 A I 229 A \n",
      " 244    0.54    2.35    1.07    0.80 * I 232 A I 235 A I 234 A I 234 A I 273 A I 230 A \n",
      " 245    0.63    2.82    1.18    1.11   E 233 A E 236 A K 235 A Q 235 A Q 274 A Q 231 A \n",
      " 246    0.63    2.64    1.19    0.99   A 234 A V 237 A A 236 A A 236 A K 275 A L 232 A \n",
      " 247    0.70    2.63    1.24    0.85   R 235 A R 238 A R 237 A R 237 A W 276 A R 233 A \n",
      " 248    0.64    3.01    1.27    1.04   G 236 A G 239 A K 238 A G 238 A G 277 A G 234 A \n",
      " 249    0.47    1.74    0.90    0.70   L 237 A K 240 A L 239 A A 239 A R 278 A L 235 A \n",
      " 250    0.46    2.18    1.00    0.63 * S 238 A S 241 A S 240 A S 240 A S 279 A S 236 A \n",
      " 251    0.44    1.87    0.76    0.49 * S 239 A S 242 A S 241 A S 241 A S 280 A S 237 A \n",
      " 252    0.26    1.01    0.42    0.22 * A 240 A A 243 A A 242 A A 242 A A 281 A A 238 A \n",
      " 253    0.27    0.98    0.38    0.23   A 241 A A 244 A M 243 A A 243 A A 282 A M 239 A \n",
      " 254    0.26    1.16    0.42    0.28 * S 242 A S 245 A S 244 A S 244 A S 283 A S 240 A \n",
      " 255    0.31    1.10    0.40    0.23   A 243 A A 246 A A 245 A A 245 A T 284 A A 241 A \n",
      " 256    0.21    0.92    0.32    0.21 * A 244 A A 247 A A 246 A A 246 A A 285 A A 242 A \n",
      " 257    0.28    0.74    0.28    0.17   N 245 A S 248 A K 247 A N 247 A V 286 A K 243 A \n",
      " 258    0.22    0.74    0.28    0.17   A 246 A S 249 A A 248 A A 248 A S 287 A A 244 A \n",
      " 259    0.25    0.77    0.33    0.15   A 247 A A 250 A I 249 A A 249 A I 288 A A 245 A \n",
      " 260    0.24    0.63    0.38    0.08   I 248 A I 251 A C 250 A I 250 A A 289 A V 246 A \n",
      " 261    0.14    0.51    0.29    0.08   D 249 A D 252 A D 251 A E 251 A D 290 A D 247 A \n",
      " 262    0.25    0.56    0.28    0.09   H 250 A H 253 A H 252 A H 252 A A 291 A H 248 A \n",
      " 263    0.23    0.85    0.39    0.15   V 251 A M 254 A V 253 A I 253 A I 292 A V 249 A \n",
      " 264    0.33    1.22    0.45    0.27   R 252 A R 255 A R 254 A R 254 A K 293 A H 250 A \n",
      " 265    0.29    0.99    0.39    0.22   D 253 A D 256 A D 255 A D 255 A S 294 A D 251 A \n",
      " 266    0.26    0.70    0.46    0.14   W 254 A W 257 A I 256 A W 256 A L 295 A W 252 A \n",
      " 267    0.44    1.74    0.70    0.35   V 255 A V 258 A W 257 A A 257 A V 296 A I 253 A \n",
      " 268    0.55    2.08    0.64    0.51   L 256 A Q 259 A F 258 A L 258 A T 297 A H 254 A \n",
      " 269    0.69    2.62    0.88    0.59   G 257 A G 260 A G 259 A G 259 A P 298 A G 255 A \n",
      " 270    0.35    1.58    0.58    0.33 * T 258 A T 261 A T 260 A T 260 A T 299 A T 256 A \n",
      " 271    0.36    1.47    1.24    0.48   N 259 A E 262 A P 261 A P 261 A P 300 A P 257 A \n",
      " 272    0.48    1.72    1.22    0.55   G 260 A -       E 262 A E 262 A E 301 A E 258 A \n",
      " 273    0.49    2.01    0.83    0.42   -       -       G 263 A G 263 A G 302 A G 259 A \n",
      " 274    0.45    1.58    0.96    0.49   K 261 A A 263 A E 264 A D 264 A D 303 A V 260 A \n",
      " 275    0.16    0.52    0.53    0.24   W 262 A W 264 A F 265 A W 265 A W 304 A Y 261 A \n",
      " 276    0.21    0.44    0.42    0.19   V 263 A S 265 A V 266 A V 266 A F 305 A V 262 A \n",
      " 277    0.09    0.28    0.22    0.10   T 264 A S 266 A S 267 A S 267 A S 306 A S 263 A \n",
      " 278    0.29    0.81    0.37    0.22   M 265 A A 267 A M 268 A M 268 A T 307 A M 264 A \n",
      " 279    0.17    0.63    0.31    0.15   G 266 A A 268 A G 269 A A 269 A G 308 A G 265 A \n",
      " 280    0.23    0.70    0.30    0.17   I 267 A I 269 A I 270 A V 270 A V 309 A V 266 A \n",
      " 281    0.23    0.36    0.28    0.08   P 268 A P 270 A I 271 A P 271 A Y 310 A Y 267 A \n",
      " 282    0.21    0.26    0.29    0.06   S 269 A S 271 A S 272 A S 272 A T 311 A S 268 A \n",
      " 283    0.52    1.69    1.06    0.37   D 270 A T 272 A D 273 A Q 273 A T 312 A D 269 A \n",
      " 284    0.68    3.23    2.04    0.36   G 271 A G 273 A G 274 A G 274 A G 313 A E 270 A \n",
      " 285    0.51    1.92    1.17    0.50   S 272 A A 274 A N 275 A E 275 A N 314 A N 271 A \n",
      " 286    0.35    1.00    0.78    0.22   -       -       S 276 A -       P 315 A P 272 A \n",
      " 287    0.24    1.06    0.58    0.30 * Y 273 A Y 275 A Y 277 A Y 277 A Y 316 A Y 273 A \n",
      " 288    0.27    1.25    1.00    0.48 * G 274 A G 276 A G 278 A G 278 A G 317 A G 274 A \n",
      " 289    0.19    0.71    0.52    0.24   I 275 A I 277 A V 279 A I 279 A I 318 A V 275 A \n",
      " 290    0.30    0.97    0.55    0.29   P 276 A P 278 A P 280 A P 280 A A 319 A P 276 A \n",
      " 291    0.23    0.82    0.57    0.26   E 277 A E 279 A D 281 A E 281 A E 320 A S 277 A \n",
      " 292    0.26    0.70    0.84    0.42   D 278 A G 280 A D 282 A G 282 A D 321 A G 278 A \n",
      " 293    0.21    0.77    0.60    0.20   I 279 A I 281 A L 283 A I 283 A I 322 A L 279 A \n",
      " 294    0.12    0.27    0.32    0.13   I 280 A F 282 A L 284 A V 284 A V 323 A I 280 A \n",
      " 295    0.15    0.40    0.31    0.17   Y 281 A V 283 A Y 285 A Y 285 A F 324 A F 281 A \n",
      " 296    0.16    0.50    0.32    0.17   G 282 A G 284 A S 286 A S 286 A S 325 A S 282 A \n",
      " 297    0.26    0.54    0.28    0.10   V 283 A L 285 A F 287 A F 287 A M 326 A F 283 A \n",
      " 298    0.15    0.64    0.38    0.26 * P 284 A P 286 A P 288 A P 288 A P 327 A P 284 A \n",
      " 299    0.24    0.58    0.40    0.12   V 285 A T 287 A V 289 A V 289 A C 328 A C 285 A \n",
      " 300    0.45    0.43    0.46    0.22   I 286 A V 288 A T 290 A T 290 A R 329 A T 286 A \n",
      " 301    0.51    0.90    0.52    0.21   C 287 A S 289 A I 291 A A 291 A S 330 A C 287 A \n",
      " 302    0.50    1.07    0.75    0.07   E 288 A R 290 A K 292 A K 292 A K 331 A H 288 A \n",
      " 303    0.00    0.00    0.00    0.00   -       -       -       -       G 332 A -       \n",
      " 304    0.56    2.38    1.21    0.43   N 289 A N 291 A D 293 A D 293 A D 333 A A 289 A \n",
      " 305    0.55    2.12    0.78    0.50   G 290 A G 292 A K 294 A G 294 A G 334 A G 290 A \n",
      " 306    0.67    1.86    0.81    0.37   E 291 A E 293 A T 295 A A 295 A D 335 A E 291 A \n",
      " 307    0.46    0.92    0.65    0.17   Y 292 A W 294 A W 296 A Y 296 A Y 336 A W 292 A \n",
      " 308    0.47    0.91    0.58    0.20   K 293 A E 295 A K 297 A R 297 A E 337 A T 293 A \n",
      " 309    0.40    0.58    0.44    0.17   R 294 A I 296 A I 298 A V 298 A L 338 A V 294 A \n",
      " 310    0.44    0.72    0.39    0.15   V 295 A V 297 A V 299 A V 299 A A 339 A V 295 A \n",
      " 311    0.37    1.30    0.86    0.38   E 296 A E 298 A E 300 A E 300 A T 340 A S 296 A \n",
      " 312    1.05    4.68    1.92    0.98   G 297 A G 299 A G 301 A G 301 A D 341 A G 297 A \n",
      " 313    0.49    2.00    1.29    0.58   L 298 A L 300 A L 302 A L 302 A V 342 A K 298 A \n",
      " 314    0.60    2.19    1.65    0.88   E 299 A E 301 A P 303 A E 303 A S 343 A L 299 A \n",
      " 315    0.59    1.15    0.69    0.25   I 300 A I 302 A I 304 A I 304 A N 344 A -       \n",
      " 316    0.20    0.48    0.51    0.21   D 301 A S 303 A N 305 A N 305 A D 345 A -       \n",
      " 317    0.24    0.75    0.61    0.14   A 302 A D 304 A D 306 A E 306 A D 346 A -       \n",
      " 318    0.13    0.35    0.48    0.26   F 303 A F 305 A F 307 A F 307 A F 347 A -       \n",
      " 319    0.27    0.65    0.46    0.22   S 304 A Q 306 A S 308 A A 308 A L 348 A -       \n",
      " 320    0.38    0.99    0.97    0.59   R 305 A R 307 A R 309 A R 309 A W 349 A K 305 A \n",
      " 321    0.19    0.49    0.88    0.73   E 306 A A 308 A E 310 A K 310 A E 350 A Q 306 A \n",
      " 322    0.22    0.76    0.83    0.70   K 307 A R 309 A K 311 A R 311 A R 351 A R 307 A \n",
      " 323    0.23    0.65    0.55    0.28   M 308 A I 310 A M 312 A M 312 A I 352 A L 308 A \n",
      " 324    0.19    0.44    0.62    0.31   D 309 A D 311 A D 313 A E 313 A K 353 A A 309 A \n",
      " 325    0.18    0.32    0.53    0.24   G 310 A A 312 A L 314 A I 314 A K 354 A S 310 A \n",
      " 326    0.16    0.45    0.50    0.25   T 311 A N 313 A T 315 A T 315 A S 355 A T 311 A \n",
      " 327    0.21    0.49    0.53    0.25   L 312 A A 314 A A 316 A A 316 A E 356 A I 312 A \n",
      " 328    0.18    0.51    0.61    0.21   A 313 A Q 315 A K 317 A Q 317 A A 357 A A 313 A \n",
      " 329    0.23    0.39    0.58    0.22 * E 314 A E 316 A E 318 A E 318 A E 358 A E 314 A \n",
      " 330    0.10    0.40    0.39    0.15 * L 315 A L 317 A L 319 A L 319 A L 359 A L 315 A \n",
      " 331    0.13    0.27    0.43    0.19   L 316 A Q 318 A A 320 A L 320 A L 360 A Q 316 A \n",
      " 332    0.16    0.38    0.57    0.23   E 317 A A 319 A E 321 A D 321 A A 361 A E 317 A \n",
      " 333    0.11    0.37    0.53    0.22 * E 318 A E 320 A E 322 A E 322 A E 362 A E 318 A \n",
      " 334    0.16    0.39    0.47    0.17   R 319 A R 321 A K 323 A M 323 A K 363 A R 319 A \n",
      " 335    0.19    0.46    0.59    0.27   D 320 A E 322 A E 324 A E 324 A K 364 A A 320 A \n",
      " 336    0.44    0.37    0.64    0.35   G 321 A A 323 A T 325 A Q 325 A C 365 A Q 321 A \n",
      " 337    0.21    0.65    0.74    0.33   V 322 A V 324 A A 326 A V 326 A V 366 A A 322 A \n",
      " 338    0.49    0.72    0.89    0.33   A 323 A R 325 A F 327 A K 327 A A 367 A G 323 A \n",
      " 339    0.74    2.83    2.05    0.64   H 324 A D 326 A E 328 A A 328 A H 368 A -       \n",
      " 340    1.16    4.56    3.10    0.33   -       L 327 A F 329 A L 329 A L 369 A -       \n",
      " 341    1.20    4.66    3.33    0.39   -       L 328 A L 330 A G 330 A T 370 A -       \n",
      " 342    1.27    5.03    3.69    0.61   -       L 329 A S 331 A L 331 A G 371 A -       \n",
      " 343    1.18    5.25    4.66    2.16   L 325 A E 330 A S 332 A I 332 A E 372 A L 324 A \n",
      " 344    2.57   11.45    5.73    0.00   -       -       A 333 A -       G 373 A -       \n",
      " 345    0.00    0.00    0.00    0.00   -       -       -       -       N 374 A -       \n",
      " 346    0.00    0.00    0.00    0.00   -       -       -       -       A 375 A -       \n",
      " 347    0.00    0.00    0.00    0.00   -       -       -       -       Y 376 A -       \n",
      " 348    0.00    0.00    0.00    0.00   -       -       -       -       C 377 A -       \n",
      " 349    0.00    0.00    0.00    0.00   -       -       -       -       D 378 A -       \n",
      " 350    0.00    0.00    0.00    0.00   -       -       -       -       V 379 A -       \n",
      " 351    0.00    0.00    0.00    0.00   -       -       -       -       P 380 A -       \n",
      " 352    0.00    0.00    0.00    0.00   -       -       -       -       E 381 A -       \n",
      " 353    0.00    0.00    0.00    0.00   -       -       -       -       D 382 A -       \n",
      " 354    0.00    0.00    0.00    0.00   -       -       -       -       T 383 A -       \n",
      " 355    0.00    0.00    0.00    0.00   -       -       -       -       M 384 A -       \n",
      " 356    0.00    0.00    0.00    0.00   -       -       -       -       L 385 A -       \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "openf___224_> Open           3d5t_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      321     2376     2376\n",
      "openf___224_> Open           6itk_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      318     2387     2387\n",
      "openf___224_> Open           5mdh_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      333     2550     2550\n",
      "openf___224_> Open           1bdm_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      318     2406     2406\n",
      "openf___224_> Open           7mdh_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      351     2688     2688\n",
      "openf___224_> Open           4h7p_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      313     2244     2244\n",
      "\n",
      "\n",
      "SALIGN______> Raw QUALITY_SCORE of the multiple alignment:     307.1\n",
      "              QUALITY_SCORE (percentage)                 :      96.7\n",
      "              Number of unique protein pairs             :        15\n",
      "              QUALITY_SCORE_min                          :       296\n",
      "              RMS_CUTOFF                                 :    3.5000\n",
      "              QS is the average number of structurally equivalent residue pairs.\n",
      "              Two residues are equivalent when closer than RMS_CUTOFF upon\n",
      "              pairwise least-squares superposition given current alignment.\n",
      "\n",
      "\n",
      "SALIGN______> Matrix of pairwise equivalences from individual pairwise alignments:\n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     314     312     311     318     303\n",
      "6itkA           0       0     308     308     314     297\n",
      "5mdhA           0       0       0     308     302     300\n",
      "1bdmA           0       0       0       0     313     297\n",
      "7mdhA           0       0       0       0       0     306\n",
      "4h7pA           0       0       0       0       0       0\n",
      "SALIGN______> Matrix of pairwise equivalences: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     314     313     311     317     303\n",
      "6itkA           0       0     308     308     314     296\n",
      "5mdhA           0       0       0     308     302     300\n",
      "1bdmA           0       0       0       0     310     297\n",
      "7mdhA           0       0       0       0       0     306\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n",
      "SALIGN______> Matrix of differences in pairwise equivalences inferred from the tree and individually: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0       0       1       0      -1       0\n",
      "6itkA           0       0       0       0       0      -1\n",
      "5mdhA           0       0       0       0       0       0\n",
      "1bdmA           0       0       0       0      -3       0\n",
      "7mdhA           0       0       0       0       0       0\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n",
      "SALIGN______> Matrix of %differences in pairwise equivalences inferred from the tree and individually: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0       0       0       0       0       0\n",
      "6itkA           0       0       0       0       0       0\n",
      "5mdhA           0       0       0       0       0       0\n",
      "1bdmA           0       0       0       0      -1       0\n",
      "7mdhA           0       0       0       0       0       0\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           ldh.pap\n",
      "openf___224_> Open           ldh.ali\n",
      "  ALIGN_CODES                                   : 3d5tA 6itkA 5mdhA 1bdmA 7mdhA 4h7pA \n",
      "  Atom type for alignment (FIT_ATOMS)           : CA\n",
      "  FIT                                           :         F\n",
      "  IMPROVE_ALIGNMENT                             :         F\n",
      "  WATER_IO, HETATM_IO, HYDROGEN_IO              :         F        F        F\n",
      "  NO_TER                                        :         F\n",
      "  OUTPUT                                        : QUALITY\n",
      "  FIT_ON_FIRST                                  :         F\n",
      "  WRITE_FIT                                     :         T\n",
      "  FIT_PDBNAM                                    :         T\n",
      "  WRITE_WHOLE_PDB                               :         F\n",
      "  CURRENT_DIRECTORY                             :         T\n",
      "  Feature weights (FEATURE_WEIGHTS)             :    0.0000   0.0000   0.0000   0.0000   0.0000   0.0000\n",
      "  NORMALIZE_PP_SCORES                           :         F\n",
      "  Gap initiation penalty (GAP_PENALTIES[1])     :  -450.0000\n",
      "  Gap extension penalty (GAP_PENALTIES[2])      :   -50.0000\n",
      "  Break-break bonus                             : 10000.0000\n",
      "  Gap_function (align2d flag)                    :         F\n",
      "  2-D Gap penalties                             :  3.50  3.50  3.50  0.20  4.00  6.50  2.00  0.00\n",
      "  GAP-GAP_SCORE                                 :    0.0000\n",
      "  GAP-RESIDUE_SCORE                             :    0.0000\n",
      "  Perform gap-residue and gap-gap correction    :         T\n",
      "  Residue type - residue type file (RR_FILE)    : $(LIB)/as1.sim.mat\n",
      "  OFF_DIAGONAL                                  :       100\n",
      "  OVERHANG                                      :        30\n",
      "  LOCAL_ALIGNMENT                               :         F\n",
      "  MATRIX_OFFSET                                 :    0.0000\n",
      "  N_SUBOPT                                      :         0\n",
      "  SUBOPT_OFFSET                                 :    0.0000\n",
      "  Gap introduction penalty (GAP_PENALTIES_3D_1) :    0.0000\n",
      "  Gap extension penalty (GAP_PENALTIES_3D_2)    :    3.0000\n",
      "  Max dist for equiv (2 * GAP_PENALTIES_3D_2)   :    6.0000\n",
      "  ALIGN3D_TRF                                   :         F\n",
      "  RMS_CUTOFF                                    :    1.0\n",
      "\n",
      "\n",
      "\n",
      "salign__276_> 'align_block' changed to 1.\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "\n",
      "Current alignment total score:            0\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "--------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -       -       -       D  23 A -       \n",
      "   2    0.00    0.00    0.00    0.00   -       -       -       -       C  24 A -       \n",
      "   3    0.00    0.00    0.00    0.00   -       -       -       -       F  25 A -       \n",
      "   4    0.00    0.00    0.00    0.00   -       -       -       -       G  26 A -       \n",
      "   5    0.00    0.00    0.00    0.00   -       -       -       -       V  27 A -       \n",
      "   6    0.00    0.00    0.00    0.00   -       -       -       -       F  28 A -       \n",
      "   7    0.00    0.00    0.00    0.00   -       -       -       -       C  29 A -       \n",
      "   8    0.00    0.00    0.00    0.00   -       -       -       -       T  30 A -       \n",
      "   9    0.00    0.00    0.25    0.00   -       -       -       M   0 A W  43 A -       \n",
      "  10    0.00    2.90    1.23    0.64   K   3 A -       S   1 A K   1 A K  44 A M   1 A \n",
      "  11    0.00    2.41    1.13    0.58   P   4 A S   8 A E   2 A A   2 A K  45 A S   2 A \n",
      "  12    0.00    1.11    0.60    0.27   A   5 A T   9 A P   3 A P   3 A L  46 A A   3 A \n",
      "  13    0.00    0.39    0.29    0.14   K   6 A K  10 A I   4 A V   4 A V  47 A V   4 A \n",
      "  14    0.00    0.21    0.25    0.10   R   7 A K  11 A R   5 A R   5 A N  48 A K   5 A \n",
      "  15    0.00    0.26    0.21    0.04   V   8 A V  12 A V   6 A V   6 A I  49 A V   6 A \n",
      "  16    0.00    0.46    0.27    0.08   A   9 A T  13 A L   7 A A   7 A A  50 A A   7 A \n",
      "  17    0.00    0.50    0.32    0.06 * V  10 A V  14 A V   8 A V   8 A V  51 A V   8 A \n",
      "  18    0.00    0.55    0.25    0.11   T  11 A T  15 A T   9 A T   9 A S  52 A T   9 A \n",
      "  19    0.00    0.65    0.35    0.14 * G  12 A G  16 A G  10 A G  10 A G  53 A G  10 A \n",
      "  20    0.00    0.52    0.38    0.07 * A  13 A A  17 A A  11 A A  11 A A  54 A A  11 A \n",
      "  21    0.00    0.36    0.37    0.07 * A  14 A A  18 A A  12 A A  12 A A  55 A A  12 A \n",
      "  22    0.00    0.29    0.29    0.13 * G  15 A G  19 A G  13 A G  13 A G  56 A G  13 A \n",
      "  23    0.00    0.32    0.32    0.15   Q  16 A Q  20 A Q  14 A Q  14 A M  57 A Q  14 A \n",
      "  24    0.00    0.18    0.28    0.19 * I  17 A I  21 A I  15 A I  15 A I  58 A I  15 A \n",
      "  25    0.00    0.20    0.29    0.15   A  18 A S  22 A A  16 A G  16 A S  59 A G  16 A \n",
      "  26    0.00    0.24    0.23    0.12   Y  19 A Y  23 A Y  17 A Y  17 A N  60 A Y  17 A \n",
      "  27    0.00    0.17    0.38    0.27   S  20 A S  24 A S  18 A S  18 A H  61 A A  18 A \n",
      "  28    0.00    0.19    0.34    0.12 * L  21 A L  25 A L  19 A L  19 A L  62 A L  19 A \n",
      "  29    0.00    0.12    0.27    0.11   L  22 A L  26 A L  20 A L  20 A L  63 A V  20 A \n",
      "  30    0.00    0.24    0.39    0.12   F  23 A W  27 A Y  21 A F  21 A F  64 A P  21 A \n",
      "  31    0.00    0.41    0.42    0.12   R  24 A R  28 A S  22 A R  22 A K  65 A L  22 A \n",
      "  32    0.00    0.22    0.35    0.13   I  25 A I  29 A I  23 A I  23 A L  66 A I  23 A \n",
      "  33    0.00    0.22    0.27    0.07   A  26 A A  30 A G  24 A A  24 A A  67 A A  24 A \n",
      "  34    0.00    0.28    0.37    0.13   N  27 A N  31 A N  25 A A  25 A S  68 A R  25 A \n",
      "  35    0.00    0.46    0.43    0.11 * G  28 A G  32 A G  26 A G  26 A G  69 A G  26 A \n",
      "  36    0.00    1.00    0.83    0.23   D  29 A E  33 A S  27 A E  27 A E  70 A A  27 A \n",
      "  37    0.00    0.96    0.83    0.24   L  30 A V  34 A V  28 A M  28 A V  71 A L  28 A \n",
      "  38    0.00    0.48    0.60    0.25   L  31 A F  35 A F  29 A L  29 A F  72 A L  29 A \n",
      "  39    0.00    0.41    0.62    0.18 * G  32 A G  36 A G  30 A G  30 A G  73 A G  30 A \n",
      "  40    0.00    0.70    0.54    0.21   K  33 A T  37 A K  31 A K  31 A Q  74 A P  31 A \n",
      "  41    0.00    0.68    0.61    0.12   D  34 A D  38 A D  32 A D  32 A D  75 A T  32 A \n",
      "  42    0.00    0.49    0.38    0.14   Q  35 A T  39 A Q  33 A Q  33 A Q  76 A T  33 A \n",
      "  43    0.00    0.28    0.41    0.11 * P  36 A P  40 A P  34 A P  34 A P  77 A P  34 A \n",
      "  44    0.00    0.36    0.32    0.06   V  37 A V  41 A I  35 A V  35 A I  78 A V  35 A \n",
      "  45    0.00    0.46    0.40    0.20   I  38 A E  42 A I  36 A I  36 A A  79 A E  36 A \n",
      "  46    0.00    0.39    0.26    0.12 * L  39 A L  43 A L  37 A L  37 A L  80 A L  37 A \n",
      "  47    0.00    0.41    0.24    0.05   Q  40 A K  44 A V  38 A Q  38 A K  81 A R  38 A \n",
      "  48    0.00    0.60    0.35    0.16 * L  41 A L  45 A L  39 A L  39 A L  82 A L  39 A \n",
      "  49    0.00    0.63    0.32    0.14 * L  42 A L  46 A L  40 A L  40 A L  83 A L  40 A \n",
      "  50    0.00    0.80    0.42    0.14   D  43 A E  47 A D  41 A E  41 A G  84 A D  41 A \n",
      "  51    0.00    0.53    0.58    0.23   L  44 A I  48 A I  42 A I  42 A S  85 A I  42 A \n",
      "  52    0.00    0.69    0.63    0.33   P  45 A P  49 A T  43 A P  43 A E  86 A E  43 A \n",
      "  53    0.00    0.39    0.86    0.55   Q  46 A Q  50 A P  44 A Q  44 A R  87 A P  44 A \n",
      "  54    0.00    0.50    0.58    0.25   A  47 A A  51 A M  45 A A  45 A S  88 A A  45 A \n",
      "  55    0.00    0.42    0.59    0.28   Q  48 A L  52 A M  46 A M  46 A F  89 A L  46 A \n",
      "  56    0.00    0.40    0.62    0.27   A  49 A G  53 A G  47 A K  47 A Q  90 A K  47 A \n",
      "  57    0.00    0.29    0.53    0.23   A  50 A G  54 A V  48 A A  48 A A  91 A A  48 A \n",
      "  58    0.00    0.32    0.50    0.33   V  51 A A  55 A L  49 A L  49 A L  92 A L  49 A \n",
      "  59    0.00    0.31    0.55    0.39   K  52 A E  56 A D  50 A E  50 A E  93 A A  50 A \n",
      "  60    0.00    0.39    0.49    0.31 * G  53 A G  57 A G  51 A G  51 A G  94 A G  51 A \n",
      "  61    0.00    0.39    0.42    0.22 * V  54 A V  58 A V  52 A V  52 A V  95 A V  52 A \n",
      "  62    0.00    0.29    0.57    0.32   V  55 A A  59 A L  53 A V  53 A A  96 A E  53 A \n",
      "  63    0.00    0.28    0.60    0.42   M  56 A M  60 A M  54 A M  54 A M  97 A A  54 A \n",
      "  64    0.00    0.19    0.43    0.26 * E  57 A E  61 A E  55 A E  55 A E  98 A E  55 A \n",
      "  65    0.00    0.25    0.46    0.23 * L  58 A L  62 A L  56 A L  56 A L  99 A L  56 A \n",
      "  66    0.00    0.26    0.68    0.44   D  59 A L  63 A Q  57 A E  57 A E 100 A E  57 A \n",
      "  67    0.00    0.22    0.68    0.49 * D  60 A D  64 A D  58 A D  58 A D 101 A D  58 A \n",
      "  68    0.00    0.16    0.56    0.44   C  61 A S  65 A C  59 A C  59 A S 102 A C  59 A \n",
      "  69    0.00    0.12    0.76    0.62   A  62 A A  66 A A  60 A A  60 A L 103 A A  60 A \n",
      "  70    0.00    0.31    0.32    0.06   F  63 A F  67 A L  61 A F  61 A Y 104 A F  61 A \n",
      "  71    0.00    0.68    0.56    0.22 * P  64 A P  68 A P  62 A P  62 A P 105 A P  62 A \n",
      "  72    0.00    0.46    0.29    0.09 * L  65 A L  69 A L  63 A L  63 A L 106 A L  63 A \n",
      "  73    0.00    0.46    0.28    0.10 * L  66 A L  70 A L  64 A L  64 A L 107 A L  64 A \n",
      "  74    0.00    0.39    0.35    0.18   A  67 A R  71 A K  65 A A  65 A R 108 A D  65 A \n",
      "  75    0.00    0.48    0.42    0.11   G  68 A N  72 A D  66 A G  66 A E 109 A K  66 A \n",
      "  76    0.00    0.43    0.33    0.05   V  69 A I  73 A V  67 A L  67 A V 110 A V  67 A \n",
      "  77    0.00    0.32    0.34    0.13   V  70 A T  74 A I  68 A E  68 A S 111 A V  68 A \n",
      "  78    0.00    0.54    0.46    0.14   I  71 A I  75 A A  69 A A  69 A I 112 A V  69 A \n",
      "  79    0.00    0.62    0.45    0.12   T  72 A T  76 A T  70 A T  70 A G 113 A T  70 A \n",
      "  80    0.00    0.61    0.45    0.18   D  73 A A  77 A D  71 A D  71 A I 114 A A  71 A \n",
      "  81    0.00    0.53    0.55    0.30   D  74 A D  78 A K  72 A D  72 A D 115 A D  72 A \n",
      "  82    0.00    0.74    0.55    0.42   P  75 A A  79 A E  73 A P  73 A P 116 A P  73 A \n",
      "  83    0.00    0.56    0.72    0.48   K  76 A N  80 A E  74 A D  74 A Y 117 A R  74 A \n",
      "  84    0.00    0.62    0.53    0.32   V  77 A E  81 A I  75 A V  75 A E 118 A V  75 A \n",
      "  85    0.00    0.66    0.39    0.18   A  78 A A  82 A A  76 A A  76 A V 119 A A  76 A \n",
      "  86    0.00    0.59    0.38    0.25 * F  79 A F  83 A F  77 A F  77 A F 120 A F  77 A \n",
      "  87    0.00    0.46    0.35    0.12   K  80 A D  84 A K  78 A K  78 A E 121 A D  78 A \n",
      "  88    0.00    0.33    0.33    0.18   D  81 A G  85 A D  79 A D  79 A D 122 A G  79 A \n",
      "  89    0.00    0.51    0.29    0.13   A  82 A A  86 A L  80 A A  80 A V 123 A V  80 A \n",
      "  90    0.00    0.41    0.26    0.09   D  83 A N  87 A D  81 A D  81 A D 124 A A  81 A \n",
      "  91    0.00    0.44    0.26    0.08   V  84 A A  88 A V  82 A Y  82 A W 125 A I  82 A \n",
      "  92    0.00    0.39    0.33    0.17 * A  85 A A  89 A A  83 A A  83 A A 126 A A  83 A \n",
      "  93    0.00    0.60    0.30    0.13   L  86 A F  90 A I  84 A L  84 A L 127 A I  84 A \n",
      "  94    0.00    0.65    0.25    0.14   L  87 A L  91 A L  85 A L  85 A L 128 A M  85 A \n",
      "  95    0.00    0.49    0.28    0.08   V  88 A V  92 A V  86 A V  86 A I 129 A C  86 A \n",
      "  96    0.00    0.31    0.32    0.11 * G  89 A G  93 A G  87 A G  87 A G 130 A G  87 A \n",
      "  97    0.00    0.41    0.48    0.23   A  90 A A  94 A S  88 A A  88 A A 131 A A  88 A \n",
      "  98    0.00    0.81    1.03    0.49   R  91 A K  95 A M  89 A A  89 A K 132 A F  89 A \n",
      "  99    0.00    0.39    1.27    0.89 * P  92 A P  96 A P  90 A P  90 A P 133 A P  90 A \n",
      " 100    0.00    0.32    1.45    0.98   R  93 A -       R  91 A         R 134 A R  91 A \n",
      " 101    0.00    0.90    2.55    1.58   S  94 A -       R  92 A -       G 135 A K  92 A \n",
      " 102    0.00    2.64    3.99    1.71   -       -       D  93 A -       P 136 A A  93 A \n",
      " 103    0.00    3.87    3.97    1.44   -       -       G  94 A -       G 137 A G  94 A \n",
      " 104    0.00    1.06    3.07    1.94   M  97 A -       M  95 A -       M 138 A M  95 A \n",
      " 105    0.00    1.00    2.72    2.08   E  98 A E 102 A E  96 A -       E 139 A E  96 A \n",
      " 106    0.00    0.71    2.23    1.71   R  99 A R 103 A R  97 A -       R 140 A R  97 A \n",
      " 107    0.00    0.63    2.20    1.67   K 100 A A 104 A K  98 A -       A 141 A K  98 A \n",
      " 108    0.00    0.58    1.79    1.33   D 101 A D 105 A D  99 A -       A 142 A D  99 A \n",
      " 109    0.00    0.42    0.86    0.44   L 102 A L 106 A L 100 A -       L 143 A L 100 A \n",
      " 110    0.00    0.63    0.95    0.58 * L 103 A L 107 A L 101 A L 101 A L 144 A L 101 A \n",
      " 111    0.00    0.75    1.00    0.59   S 104 A A 108 A K 102 A Q 102 A D 145 A E 102 A \n",
      " 112    0.00    0.68    1.02    0.36   A 105 A N 109 A A 103 A V 103 A I 146 A M 103 A \n",
      " 113    0.00    0.34    0.67    0.32 * N 106 A N 110 A N 104 A N 104 A N 147 A N 104 A \n",
      " 114    0.00    0.69    1.01    0.31   A 107 A G 111 A V 105 A G 105 A G 148 A A 105 A \n",
      " 115    0.00    0.64    0.83    0.22   E 108 A K 112 A K 106 A K 106 A Q 149 A R 106 A \n",
      " 116    0.00    0.33    0.36    0.06 * I 109 A I 113 A I 107 A I 107 A I 150 A I 107 A \n",
      " 117    0.00    0.16    0.35    0.14 * F 110 A F 114 A F 108 A F 108 A F 151 A F 108 A \n",
      " 118    0.00    0.51    0.47    0.17   T 111 A G 115 A K 109 A T 109 A A 152 A K 109 A \n",
      " 119    0.00    0.59    0.41    0.12   V 112 A P 116 A C 110 A E 110 A D 153 A E 110 A \n",
      " 120    0.00    0.31    0.24    0.06 * Q 113 A Q 117 A Q 111 A Q 111 A Q 154 A Q 111 A \n",
      " 121    0.00    0.25    0.27    0.07 * G 114 A G 118 A G 112 A G 112 A G 155 A G 112 A \n",
      " 122    0.00    0.32    0.31    0.13   A 115 A K 119 A A 113 A R 113 A K 156 A E 113 A \n",
      " 123    0.00    0.30    0.30    0.09 * A 116 A A 120 A A 114 A A 114 A A 157 A A 114 A \n",
      " 124    0.00    0.35    0.26    0.09   L 117 A I 121 A L 115 A L 115 A L 158 A I 115 A \n",
      " 125    0.00    0.40    0.34    0.12   N 118 A N 122 A D 116 A A 116 A N 159 A A 116 A \n",
      " 126    0.00    0.45    0.40    0.13   E 119 A D 123 A K 117 A E 117 A A 160 A A 117 A \n",
      " 127    0.00    0.40    0.38    0.15   V 120 A N 124 A Y 118 A V 118 A V 161 A V 118 A \n",
      " 128    0.00    0.35    0.36    0.10 * A 121 A A 125 A A 119 A A 119 A A 162 A A 119 A \n",
      " 129    0.00    0.43    0.56    0.26   S 122 A A 126 A K 120 A K 120 A S 163 A A 120 A \n",
      " 130    0.00    0.57    0.78    0.53   R 123 A D 127 A K 121 A K 121 A K 164 A S 121 A \n",
      " 131    0.00    0.21    0.69    0.50   D 124 A D 128 A S 122 A D 122 A N 165 A D 122 A \n",
      " 132    0.00    0.11    0.27    0.14   V 125 A I 129 A V 123 A V 123 A V 166 A C 123 A \n",
      " 133    0.00    0.35    0.26    0.10   K 126 A R 130 A K 124 A K 124 A K 167 A R 124 A \n",
      " 134    0.00    0.24    0.27    0.11 * V 127 A V 131 A V 125 A V 125 A V 168 A V 125 A \n",
      " 135    0.00    0.53    0.23    0.16   L 128 A L 132 A I 126 A L 126 A L 169 A V 126 A \n",
      " 136    0.00    0.25    0.18    0.05 * V 129 A V 133 A V 127 A V 127 A V 170 A V 127 A \n",
      " 137    0.00    0.47    0.29    0.09 * V 130 A V 134 A V 128 A V 128 A V 171 A V 128 A \n",
      " 138    0.00    0.76    0.32    0.19 * G 131 A G 135 A G 129 A G 129 A G 172 A G 129 A \n",
      " 139    0.00    0.27    0.30    0.15 * N 132 A N 136 A N 130 A N 130 A N 173 A N 130 A \n",
      " 140    0.00    0.07    0.27    0.14 * P 133 A P 137 A P 131 A P 131 A P 174 A P 131 A \n",
      " 141    0.00    0.08    0.25    0.09   A 134 A A 138 A A 132 A A 132 A C 175 A A 132 A \n",
      " 142    0.00    0.11    0.21    0.07 * N 135 A N 139 A N 133 A N 133 A N 176 A N 133 A \n",
      " 143    0.00    0.19    0.21    0.05 * T 136 A T 140 A T 134 A T 134 A T 177 A T 134 A \n",
      " 144    0.00    0.16    0.23    0.06 * N 137 A N 141 A N 135 A N 135 A N 178 A N 135 A \n",
      " 145    0.00    0.24    0.26    0.08   A 138 A A 142 A C 136 A A 136 A A 179 A A 136 A \n",
      " 146    0.00    0.26    0.26    0.10   Y 139 A L 143 A L 137 A L 137 A L 180 A L 137 A \n",
      " 147    0.00    0.26    0.31    0.10   I 140 A I 144 A T 138 A I 138 A I 181 A I 138 A \n",
      " 148    0.00    0.23    0.27    0.08   A 141 A A 145 A A 139 A A 139 A C 182 A L 139 A \n",
      " 149    0.00    0.36    0.28    0.06   M 142 A S 146 A S 140 A Y 140 A L 183 A L 140 A \n",
      " 150    0.00    0.42    0.29    0.06   K 143 A A 147 A K 141 A K 141 A K 184 A K 141 A \n",
      " 151    0.00    0.91    0.40    0.19   S 144 A A 148 A S 142 A N 142 A N 185 A S 142 A \n",
      " 152    0.00    1.29    0.49    0.26 * A 145 A A 149 A A 143 A A 143 A A 186 A A 143 A \n",
      " 153    0.00    0.00    0.00    0.00   -       -       -       -       -       Q 144 A \n",
      " 154    0.00    1.76    0.79    0.62   P 146 A P 150 A P 144 A P 144 A P 187 A G 145 A \n",
      " 155    0.00    0.92    0.36    0.22   D 147 A D 151 A S 145 A G 145 A D 188 A K 146 A \n",
      " 156    0.00    1.27    0.46    0.32   L 148 A V 152 A I 146 A L 146 A I 189 A L 147 A \n",
      " 157    0.00    0.57    0.40    0.13   P 149 A P 153 A P 147 A N 147 A P 190 A N 148 A \n",
      " 158    0.00    0.66    0.49    0.14   K 150 A A 154 A K 148 A P 148 A A 191 A P 149 A \n",
      " 159    0.00    0.66    0.46    0.18   K 151 A S 155 A E 149 A R 149 A K 192 A R 150 A \n",
      " 160    0.00    0.32    0.34    0.12   N 152 A R 156 A N 150 A N 150 A N 193 A H 151 A \n",
      " 161    0.00    0.27    0.32    0.13   F 153 A F 157 A F 151 A F 151 A F 194 A V 152 A \n",
      " 162    0.00    0.41    0.32    0.14   T 154 A N 158 A S 152 A T 152 A H 195 A T 153 A \n",
      " 163    0.00    0.21    0.29    0.21   A 155 A A 159 A C 153 A A 153 A A 196 A A 154 A \n",
      " 164    0.00    0.23    0.17    0.05   M 156 A M 160 A L 154 A M 154 A L 197 A M 155 A \n",
      " 165    0.00    0.17    0.27    0.18   L 157 A M 161 A T 155 A T 155 A T 198 A T 156 A \n",
      " 166    0.00    0.20    0.20    0.06 * R 158 A R 162 A R 156 A R 156 A R 199 A R 157 A \n",
      " 167    0.00    0.15    0.27    0.10 * L 159 A L 163 A L 157 A L 157 A L 200 A L 158 A \n",
      " 168    0.00    0.17    0.30    0.13 * D 160 A D 164 A D 158 A D 158 A D 201 A D 159 A \n",
      " 169    0.00    0.16    0.21    0.10   H 161 A H 165 A H 159 A H 159 A E 202 A H 160 A \n",
      " 170    0.00    0.13    0.20    0.10 * N 162 A N 166 A N 160 A N 160 A N 203 A N 161 A \n",
      " 171    0.00    0.12    0.28    0.13 * R 163 A R 167 A R 161 A R 161 A R 204 A R 162 A \n",
      " 172    0.00    0.15    0.16    0.08 * A 164 A A 168 A A 162 A A 162 A A 205 A A 163 A \n",
      " 173    0.00    0.40    0.17    0.09   L 165 A I 169 A K 163 A K 163 A K 206 A L 164 A \n",
      " 174    0.00    0.52    0.33    0.10   S 166 A S 170 A A 164 A A 164 A C 207 A S 165 A \n",
      " 175    0.00    0.53    0.38    0.06   Q 167 A Q 171 A Q 165 A Q 165 A Q 208 A L 166 A \n",
      " 176    0.00    0.25    0.33    0.08   L 168 A L 172 A I 166 A L 166 A L 209 A L 167 A \n",
      " 177    0.00    0.19    0.46    0.17 * A 169 A A 173 A A 167 A A 167 A A 210 A A 168 A \n",
      " 178    0.00    0.21    0.56    0.19   A 170 A T 174 A L 168 A K 168 A L 211 A R 169 A \n",
      " 179    0.00    0.37    0.55    0.13 * K 171 A K 175 A K 169 A K 169 A K 212 A K 170 A \n",
      " 180    0.00    0.28    0.68    0.27   S 172 A L 176 A L 170 A T 170 A A 213 A A 171 A \n",
      " 181    0.00    0.33    0.83    0.34 * G 173 A G 177 A G 171 A G 171 A G 214 A G 172 A \n",
      " 182    0.00    0.59    0.79    0.31   K 174 A R 178 A V 172 A T 172 A V 215 A V 173 A \n",
      " 183    0.00    0.70    0.83    0.22   P 175 A G 179 A T 173 A G 173 A F 216 A P 174 A \n",
      " 184    0.00    0.56    0.64    0.16   V 176 A S 180 A S 174 A V 174 A Y 217 A V 175 A \n",
      " 185    0.00    0.90    0.65    0.16   A 177 A A 181 A D 175 A D 175 A D 218 A S 176 A \n",
      " 186    0.00    0.38    0.62    0.28   S 178 A E 182 A D 176 A R 176 A K 219 A Q 177 A \n",
      " 187    0.00    0.76    0.68    0.26   I 179 A F 183 A V 177 A I 177 A V 220 A V 178 A \n",
      " 188    0.00    1.44    1.01    0.38   E 180 A N 184 A K 178 A R 178 A S 221 A R 179 A \n",
      " 189    0.00    1.40    0.79    0.38   K 181 A N 185 A N 179 A R 179 A N 222 A N 180 A \n",
      " 190    0.00    1.23    0.76    0.26   L 182 A I 186 A V 180 A M 180 A V 223 A V 181 A \n",
      " 191    0.00    0.84    0.51    0.25   A 183 A V 187 A I 181 A T 181 A T 224 A I 182 A \n",
      " 192    0.00    0.60    0.45    0.24   V 184 A V 188 A I 182 A V 182 A I 225 A I 183 A \n",
      " 193    0.00    0.36    0.24    0.08 * W 185 A W 189 A W 183 A W 183 A W 226 A W 184 A \n",
      " 194    0.00    0.45    0.34    0.08 * G 186 A G 190 A G 184 A G 184 A G 227 A G 185 A \n",
      " 195    0.00    0.64    0.56    0.41 * N 187 A N 191 A N 185 A N 185 A N 228 A N 186 A \n",
      " 196    0.00    0.58    0.52    0.26 * H 188 A H 192 A H 186 A H 186 A H 229 A H 187 A \n",
      " 197    0.00    0.73    0.48    0.16 * S 189 A S 193 A S 187 A S 187 A S 230 A S 188 A \n",
      " 198    0.00    0.93    0.58    0.25   P 190 A A 194 A S 188 A S 188 A T 231 A S 189 A \n",
      " 199    0.00    1.03    0.59    0.27   T 191 A T 195 A T 189 A I 189 A T 232 A T 190 A \n",
      " 200    0.00    0.65    0.39    0.13   M 192 A Q 196 A Q 190 A M 190 A Q 233 A Q 191 A \n",
      " 201    0.00    0.77    0.39    0.12   Y 193 A F 197 A Y 191 A F 191 A V 234 A V 192 A \n",
      " 202    0.00    1.04    0.36    0.25 * P 194 A P 198 A P 192 A P 192 A P 235 A P 193 A \n",
      " 203    0.00    0.92    0.39    0.20 * D 195 A D 199 A D 193 A D 193 A D 236 A D 194 A \n",
      " 204    0.00    0.91    0.68    0.24   F 196 A I 200 A V 194 A L 194 A F 237 A T 195 A \n",
      " 205    0.00    0.96    0.56    0.25   R 197 A T 201 A N 195 A F 195 A L 238 A D 196 A \n",
      " 206    0.00    0.74    0.67    0.40   F 198 A Y 202 A H 196 A H 196 A N 239 A S 197 A \n",
      " 207    0.00    0.73    0.66    0.38 * A 199 A A 203 A A 197 A A 197 A A 240 A A 198 A \n",
      " 208    0.00    0.66    0.67    0.39   T 200 A T 204 A K 198 A E 198 A K 241 A V 199 A \n",
      " 209    0.00    0.72    0.75    0.22   A 201 A V 205 A V 199 A V 199 A I 242 A I 200 A \n",
      " 210    0.00    2.08    1.31    0.45   E 202 A G 206 A K 200 A D 200 A D 243 A G 201 A \n",
      " 211    0.00    0.00    0.00    0.00   -       -       L 201 A -       -       -       \n",
      " 212    0.00    0.00    0.00    0.00   -       -       Q 202 A -       -       -       \n",
      " 213    0.00    0.00    0.00    0.00   -       -       A 203 A -       -       -       \n",
      " 214    0.00    0.00    0.00    0.00   -       -       K 204 A -       -       -       \n",
      " 215    0.00    2.04    1.53    0.43   G 203 A G 207 A E 205 A G 205 A G 244 A T 202 A \n",
      " 216    0.00    1.01    0.94    0.48   E 204 A E 208 A V 206 A R 206 A R 245 A T 203 A \n",
      " 217    0.00    0.63    0.75    0.32   S 205 A K 209 A G 207 A P 207 A P 246 A P 204 A \n",
      " 218    0.00    1.26    0.63    0.24   L 206 A V 210 A V 208 A A 208 A V 247 A A 205 A \n",
      " 219    0.00    1.90    0.78    0.41   L 207 A T 211 A Y 209 A L 209 A K 248 A R 206 A \n",
      " 220    0.00    1.86    0.86    0.46   K 208 A D 212 A E 210 A E 210 A E 249 A E 207 A \n",
      " 221    0.00    2.04    1.02    0.46   L 209 A L 213 A A 211 A L 211 A V 250 A A 208 A \n",
      " 222    0.00    2.45    1.14    0.57   I 210 A V 214 A V 212 A V 212 A I 251 A I 209 A \n",
      " 223    0.00    5.31    2.04    1.46   N 211 A -       K 213 A -       K 252 A K 210 A \n",
      " 224    0.00    0.00    0.85    0.41   D 212 A D 215 A D 214 A D 214 A R 253 A -       \n",
      " 225    0.00    0.00    0.60    0.33   D 213 A H 216 A D 215 A M 215 A T 254 A -       \n",
      " 226    0.00    0.00    0.62    0.17   V 214 A D 217 A S 216 A E 216 A K 255 A -       \n",
      " 227    0.00    0.00    0.58    0.23   W 215 A W 218 A W 217 A W 217 A W 256 A -       \n",
      " 228    0.00    0.00    0.65    0.22   N 216 A Y 219 A L 218 A Y 218 A L 257 A -       \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 229    0.00    0.00    1.00    0.29   R 217 A V 220 A K 219 A E 219 A E 258 A -       \n",
      " 230    0.00    0.00    1.06    0.28   D 218 A E 221 A G 220 A K 220 A E 259 A -       \n",
      " 231    0.00    0.23    0.64    0.36   T 219 A E 222 A E 221 A V 221 A E 260 A D 217 A \n",
      " 232    0.00    0.36    0.41    0.19 * F 220 A F 223 A F 222 A F 222 A F 261 A F 218 A \n",
      " 233    0.00    0.44    0.38    0.24   I 221 A I 224 A I 223 A I 223 A T 262 A V 219 A \n",
      " 234    0.00    0.32    0.55    0.20   P 222 A P 225 A T 224 A P 224 A I 263 A Q 220 A \n",
      " 235    0.00    0.26    0.55    0.25   T 223 A R 226 A T 225 A T 225 A T 264 A V 221 A \n",
      " 236    0.00    0.43    0.47    0.27 * V 224 A V 227 A V 226 A V 226 A V 265 A V 222 A \n",
      " 237    0.00    0.41    0.73    0.31   G 225 A A 228 A Q 227 A A 227 A Q 266 A R 223 A \n",
      " 238    0.00    0.10    0.89    0.52   K 226 A N 229 A Q 228 A Q 228 A K 267 A G 224 A \n",
      " 239    0.00    0.28    0.76    0.49 * R 227 A R 230 A R 229 A R 229 A R 268 A R 225 A \n",
      " 240    0.00    0.28    0.86    0.59 * G 228 A G 231 A G 230 A G 230 A G 269 A G 226 A \n",
      " 241    0.00    0.45    1.11    0.77   A 229 A A 232 A A 231 A A 231 A G 270 A A 227 A \n",
      " 242    0.00    0.51    1.04    0.79   A 230 A E 233 A A 232 A A 232 A A 271 A E 228 A \n",
      " 243    0.00    0.55    0.93    0.65   I 231 A I 234 A V 233 A I 233 A L 272 A I 229 A \n",
      " 244    0.00    0.35    1.07    0.80 * I 232 A I 235 A I 234 A I 234 A I 273 A I 230 A \n",
      " 245    0.00    0.19    1.18    1.11   E 233 A E 236 A K 235 A Q 235 A Q 274 A Q 231 A \n",
      " 246    0.00    0.48    1.19    0.99   A 234 A V 237 A A 236 A A 236 A K 275 A L 232 A \n",
      " 247    0.00    0.61    1.24    0.85   R 235 A R 238 A R 237 A R 237 A W 276 A R 233 A \n",
      " 248    0.00    0.55    1.27    1.04   G 236 A G 239 A K 238 A G 238 A G 277 A G 234 A \n",
      " 249    0.00    0.39    0.90    0.70   L 237 A K 240 A L 239 A A 239 A R 278 A L 235 A \n",
      " 250    0.00    0.45    1.00    0.63 * S 238 A S 241 A S 240 A S 240 A S 279 A S 236 A \n",
      " 251    0.00    0.50    0.76    0.49 * S 239 A S 242 A S 241 A S 241 A S 280 A S 237 A \n",
      " 252    0.00    0.22    0.42    0.22 * A 240 A A 243 A A 242 A A 242 A A 281 A A 238 A \n",
      " 253    0.00    0.23    0.38    0.23   A 241 A A 244 A M 243 A A 243 A A 282 A M 239 A \n",
      " 254    0.00    0.35    0.42    0.28 * S 242 A S 245 A S 244 A S 244 A S 283 A S 240 A \n",
      " 255    0.00    0.36    0.40    0.23   A 243 A A 246 A A 245 A A 245 A T 284 A A 241 A \n",
      " 256    0.00    0.31    0.32    0.21 * A 244 A A 247 A A 246 A A 246 A A 285 A A 242 A \n",
      " 257    0.00    0.25    0.28    0.17   N 245 A S 248 A K 247 A N 247 A V 286 A K 243 A \n",
      " 258    0.00    0.25    0.28    0.17   A 246 A S 249 A A 248 A A 248 A S 287 A A 244 A \n",
      " 259    0.00    0.32    0.33    0.15   A 247 A A 250 A I 249 A A 249 A I 288 A A 245 A \n",
      " 260    0.00    0.52    0.38    0.08   I 248 A I 251 A C 250 A I 250 A A 289 A V 246 A \n",
      " 261    0.00    0.38    0.29    0.08   D 249 A D 252 A D 251 A E 251 A D 290 A D 247 A \n",
      " 262    0.00    0.36    0.28    0.09   H 250 A H 253 A H 252 A H 252 A A 291 A H 248 A \n",
      " 263    0.00    0.51    0.39    0.15   V 251 A M 254 A V 253 A I 253 A I 292 A V 249 A \n",
      " 264    0.00    0.42    0.45    0.27   R 252 A R 255 A R 254 A R 254 A K 293 A H 250 A \n",
      " 265    0.00    0.32    0.39    0.22   D 253 A D 256 A D 255 A D 255 A S 294 A D 251 A \n",
      " 266    0.00    0.57    0.46    0.14   W 254 A W 257 A I 256 A W 256 A L 295 A W 252 A \n",
      " 267    0.00    0.80    0.70    0.35   V 255 A V 258 A W 257 A A 257 A V 296 A I 253 A \n",
      " 268    0.00    0.52    0.64    0.51   L 256 A Q 259 A F 258 A L 258 A T 297 A H 254 A \n",
      " 269    0.00    0.70    0.88    0.59   G 257 A G 260 A G 259 A G 259 A P 298 A G 255 A \n",
      " 270    0.00    0.50    0.58    0.33 * T 258 A T 261 A T 260 A T 260 A T 299 A T 256 A \n",
      " 271    0.00    1.05    1.24    0.48   N 259 A E 262 A P 261 A P 261 A P 300 A P 257 A \n",
      " 272    0.00    0.91    1.22    0.55   G 260 A -       E 262 A E 262 A E 301 A E 258 A \n",
      " 273    0.00    0.90    0.83    0.42   -       -       G 263 A G 263 A G 302 A G 259 A \n",
      " 274    0.00    0.31    0.96    0.49   K 261 A A 263 A E 264 A D 264 A D 303 A V 260 A \n",
      " 275    0.00    0.23    0.53    0.24   W 262 A W 264 A F 265 A W 265 A W 304 A Y 261 A \n",
      " 276    0.00    0.58    0.42    0.19   V 263 A S 265 A V 266 A V 266 A F 305 A V 262 A \n",
      " 277    0.00    0.38    0.22    0.10   T 264 A S 266 A S 267 A S 267 A S 306 A S 263 A \n",
      " 278    0.00    0.31    0.37    0.22   M 265 A A 267 A M 268 A M 268 A T 307 A M 264 A \n",
      " 279    0.00    0.24    0.31    0.15   G 266 A A 268 A G 269 A A 269 A G 308 A G 265 A \n",
      " 280    0.00    0.19    0.30    0.17   I 267 A I 269 A I 270 A V 270 A V 309 A V 266 A \n",
      " 281    0.00    0.22    0.28    0.08   P 268 A P 270 A I 271 A P 271 A Y 310 A Y 267 A \n",
      " 282    0.00    0.36    0.29    0.06   S 269 A S 271 A S 272 A S 272 A T 311 A S 268 A \n",
      " 283    0.00    1.42    1.06    0.37   D 270 A T 272 A D 273 A Q 273 A T 312 A D 269 A \n",
      " 284    0.00    2.46    2.04    0.36   G 271 A G 273 A G 274 A G 274 A G 313 A E 270 A \n",
      " 285    0.00    1.69    1.17    0.50   S 272 A A 274 A N 275 A E 275 A N 314 A N 271 A \n",
      " 286    0.00    0.88    0.78    0.22   -       -       S 276 A -       P 315 A P 272 A \n",
      " 287    0.00    0.57    0.58    0.30 * Y 273 A Y 275 A Y 277 A Y 277 A Y 316 A Y 273 A \n",
      " 288    0.00    1.57    1.00    0.48 * G 274 A G 276 A G 278 A G 278 A G 317 A G 274 A \n",
      " 289    0.00    0.44    0.52    0.24   I 275 A I 277 A V 279 A I 279 A I 318 A V 275 A \n",
      " 290    0.00    0.30    0.55    0.29   P 276 A P 278 A P 280 A P 280 A A 319 A P 276 A \n",
      " 291    0.00    0.48    0.57    0.26   E 277 A E 279 A D 281 A E 281 A E 320 A S 277 A \n",
      " 292    0.00    0.60    0.84    0.42   D 278 A G 280 A D 282 A G 282 A D 321 A G 278 A \n",
      " 293    0.00    0.24    0.60    0.20   I 279 A I 281 A L 283 A I 283 A I 322 A L 279 A \n",
      " 294    0.00    0.37    0.32    0.13   I 280 A F 282 A L 284 A V 284 A V 323 A I 280 A \n",
      " 295    0.00    0.34    0.31    0.17   Y 281 A V 283 A Y 285 A Y 285 A F 324 A F 281 A \n",
      " 296    0.00    0.26    0.32    0.17   G 282 A G 284 A S 286 A S 286 A S 325 A S 282 A \n",
      " 297    0.00    0.23    0.28    0.10   V 283 A L 285 A F 287 A F 287 A M 326 A F 283 A \n",
      " 298    0.00    0.38    0.38    0.26 * P 284 A P 286 A P 288 A P 288 A P 327 A P 284 A \n",
      " 299    0.00    0.39    0.40    0.12   V 285 A T 287 A V 289 A V 289 A C 328 A C 285 A \n",
      " 300    0.00    0.35    0.46    0.22   I 286 A V 288 A T 290 A T 290 A R 329 A T 286 A \n",
      " 301    0.00    0.50    0.52    0.21   C 287 A S 289 A I 291 A A 291 A S 330 A C 287 A \n",
      " 302    0.00    0.90    0.75    0.07   E 288 A R 290 A K 292 A K 292 A K 331 A H 288 A \n",
      " 303    0.00    0.00    0.00    0.00   -       -       -       -       G 332 A -       \n",
      " 304    0.00    1.41    1.21    0.43   N 289 A N 291 A D 293 A D 293 A D 333 A A 289 A \n",
      " 305    0.00    0.82    0.78    0.50   G 290 A G 292 A K 294 A G 294 A G 334 A G 290 A \n",
      " 306    0.00    0.62    0.81    0.37   E 291 A E 293 A T 295 A A 295 A D 335 A E 291 A \n",
      " 307    0.00    0.76    0.65    0.17   Y 292 A W 294 A W 296 A Y 296 A Y 336 A W 292 A \n",
      " 308    0.00    0.72    0.58    0.20   K 293 A E 295 A K 297 A R 297 A E 337 A T 293 A \n",
      " 309    0.00    0.81    0.44    0.17   R 294 A I 296 A I 298 A V 298 A L 338 A V 294 A \n",
      " 310    0.00    0.64    0.39    0.15   V 295 A V 297 A V 299 A V 299 A A 339 A V 295 A \n",
      " 311    0.00    1.66    0.86    0.38   E 296 A E 298 A E 300 A E 300 A T 340 A S 296 A \n",
      " 312    0.00    2.58    1.92    0.98   G 297 A G 299 A G 301 A G 301 A D 341 A G 297 A \n",
      " 313    0.00    2.64    1.29    0.58   L 298 A L 300 A L 302 A L 302 A V 342 A K 298 A \n",
      " 314    0.00    4.11    1.65    0.88   E 299 A E 301 A P 303 A E 303 A S 343 A L 299 A \n",
      " 315    0.00    0.00    0.69    0.25   I 300 A I 302 A I 304 A I 304 A N 344 A -       \n",
      " 316    0.00    0.00    0.51    0.21   D 301 A S 303 A N 305 A N 305 A D 345 A -       \n",
      " 317    0.00    0.00    0.61    0.14   A 302 A D 304 A D 306 A E 306 A D 346 A -       \n",
      " 318    0.00    0.00    0.48    0.26   F 303 A F 305 A F 307 A F 307 A F 347 A -       \n",
      " 319    0.00    0.00    0.46    0.22   S 304 A Q 306 A S 308 A A 308 A L 348 A -       \n",
      " 320    0.00    2.59    0.97    0.59   R 305 A R 307 A R 309 A R 309 A W 349 A K 305 A \n",
      " 321    0.00    2.90    0.88    0.73   E 306 A A 308 A E 310 A K 310 A E 350 A Q 306 A \n",
      " 322    0.00    2.81    0.83    0.70   K 307 A R 309 A K 311 A R 311 A R 351 A R 307 A \n",
      " 323    0.00    1.35    0.55    0.28   M 308 A I 310 A M 312 A M 312 A I 352 A L 308 A \n",
      " 324    0.00    1.48    0.62    0.31   D 309 A D 311 A D 313 A E 313 A K 353 A A 309 A \n",
      " 325    0.00    1.08    0.53    0.24   G 310 A A 312 A L 314 A I 314 A K 354 A S 310 A \n",
      " 326    0.00    0.63    0.50    0.25   T 311 A N 313 A T 315 A T 315 A S 355 A T 311 A \n",
      " 327    0.00    0.76    0.53    0.25   L 312 A A 314 A A 316 A A 316 A E 356 A I 312 A \n",
      " 328    0.00    0.76    0.61    0.21   A 313 A Q 315 A K 317 A Q 317 A A 357 A A 313 A \n",
      " 329    0.00    0.56    0.58    0.22 * E 314 A E 316 A E 318 A E 318 A E 358 A E 314 A \n",
      " 330    0.00    0.41    0.39    0.15 * L 315 A L 317 A L 319 A L 319 A L 359 A L 315 A \n",
      " 331    0.00    0.40    0.43    0.19   L 316 A Q 318 A A 320 A L 320 A L 360 A Q 316 A \n",
      " 332    0.00    0.63    0.57    0.23   E 317 A A 319 A E 321 A D 321 A A 361 A E 317 A \n",
      " 333    0.00    0.48    0.53    0.22 * E 318 A E 320 A E 322 A E 322 A E 362 A E 318 A \n",
      " 334    0.00    0.48    0.47    0.17   R 319 A R 321 A K 323 A M 323 A K 363 A R 319 A \n",
      " 335    0.00    0.76    0.59    0.27   D 320 A E 322 A E 324 A E 324 A K 364 A A 320 A \n",
      " 336    0.00    0.87    0.64    0.35   G 321 A A 323 A T 325 A Q 325 A C 365 A Q 321 A \n",
      " 337    0.00    0.83    0.74    0.33   V 322 A V 324 A A 326 A V 326 A V 366 A A 322 A \n",
      " 338    0.00    1.57    0.89    0.33   A 323 A R 325 A F 327 A K 327 A A 367 A G 323 A \n",
      " 339    0.00    0.00    2.05    0.64   H 324 A D 326 A E 328 A A 328 A H 368 A -       \n",
      " 340    0.00    0.00    3.10    0.33   -       L 327 A F 329 A L 329 A L 369 A -       \n",
      " 341    0.00    0.00    3.33    0.39   -       L 328 A L 330 A G 330 A T 370 A -       \n",
      " 342    0.00    0.00    3.69    0.61   -       L 329 A S 331 A L 331 A G 371 A -       \n",
      " 343    0.00    4.77    4.66    2.16   L 325 A E 330 A S 332 A I 332 A E 372 A L 324 A \n",
      " 344    0.00    0.00    5.73    0.00   -       -       A 333 A -       G 373 A -       \n",
      " 345    0.00    0.00    0.00    0.00   -       -       -       -       N 374 A -       \n",
      " 346    0.00    0.00    0.00    0.00   -       -       -       -       A 375 A -       \n",
      " 347    0.00    0.00    0.00    0.00   -       -       -       -       Y 376 A -       \n",
      " 348    0.00    0.00    0.00    0.00   -       -       -       -       C 377 A -       \n",
      " 349    0.00    0.00    0.00    0.00   -       -       -       -       D 378 A -       \n",
      " 350    0.00    0.00    0.00    0.00   -       -       -       -       V 379 A -       \n",
      " 351    0.00    0.00    0.00    0.00   -       -       -       -       P 380 A -       \n",
      " 352    0.00    0.00    0.00    0.00   -       -       -       -       E 381 A -       \n",
      " 353    0.00    0.00    0.00    0.00   -       -       -       -       D 382 A -       \n",
      " 354    0.00    0.00    0.00    0.00   -       -       -       -       T 383 A -       \n",
      " 355    0.00    0.00    0.00    0.00   -       -       -       -       M 384 A -       \n",
      " 356    0.00    0.00    0.00    0.00   -       -       -       -       L 385 A -       \n",
      "--------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "openf___224_> Open           3d5t_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      321     2376     2376\n",
      "openf___224_> Open           6itk_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      318     2387     2387\n",
      "openf___224_> Open           5mdh_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      333     2550     2550\n",
      "openf___224_> Open           1bdm_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      318     2406     2406\n",
      "openf___224_> Open           7mdh_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      351     2688     2688\n",
      "openf___224_> Open           4h7p_fit.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      313     2244     2244\n",
      "openf___224_> Open           6itk_fit.pdb\n",
      "openf___224_> Open           6itk_fit.pdb\n",
      "openf___224_> Open           5mdh_fit.pdb\n",
      "openf___224_> Open           5mdh_fit.pdb\n",
      "openf___224_> Open           1bdm_fit.pdb\n",
      "openf___224_> Open           1bdm_fit.pdb\n",
      "openf___224_> Open           7mdh_fit.pdb\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           7mdh_fit.pdb\n",
      "openf___224_> Open           4h7p_fit.pdb\n",
      "openf___224_> Open           4h7p_fit.pdb\n",
      "\n",
      "\n",
      "SALIGN______> Raw QUALITY_SCORE of the multiple alignment:     221.1\n",
      "              QUALITY_SCORE (percentage)                 :      69.7\n",
      "              Number of unique protein pairs             :        15\n",
      "              QUALITY_SCORE_min                          :       191\n",
      "              RMS_CUTOFF                                 :    1.0000\n",
      "              QS is the average number of structurally equivalent residue pairs.\n",
      "              Two residues are equivalent when closer than RMS_CUTOFF upon\n",
      "              pairwise least-squares superposition given current alignment.\n",
      "\n",
      "SALIGN______> Matrix of pairwise equivalences: \n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA           0     246     191     254     202     230\n",
      "6itkA           0       0     220     238     200     239\n",
      "5mdhA           0       0       0     207     195     213\n",
      "1bdmA           0       0       0       0     236     238\n",
      "7mdhA           0       0       0       0       0     208\n",
      "4h7pA           0       0       0       0       0       0\n",
      "\n",
      "\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "<modeller.salign.SalignData at 0x7fa4ab5516d0>"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Illustrates the SALIGN multiple structure/sequence alignment\n",
    "from modeller import *\n",
    "\n",
    "log.verbose()\n",
    "env = environ()\n",
    "env.io.atom_files_directory = './:../atom_files/'\n",
    "\n",
    "aln = alignment(env)\n",
    "\n",
    "#replace the PDB ID and chain as per the alignment of the target sequence against the PDB database sequence\n",
    "for (code, chain) in (('3d5t', 'A'), ('6itk', 'A'), ('5mdh', 'A'),\n",
    "                     ('1bdm', 'A'), ('7mdh', 'A'), ('4h7p', 'A')):\n",
    "    mdl = model(env, file=code, model_segment=('FIRST:'+chain, 'LAST:'+chain))\n",
    "    aln.append_model(mdl, atom_files=code, align_codes=code+chain)\n",
    "\n",
    "for (weights, write_fit, whole) in (((1., 0., 0., 0., 1., 0.), False, True),\n",
    "                                    ((1., 0.5, 1., 1., 1., 0.), False, True),\n",
    "                                    ((1., 1., 1., 1., 1., 0.), True, False)):\n",
    "    aln.salign(rms_cutoff=3.5, normalize_pp_scores=False, #customizable\n",
    "               rr_file='$(LIB)/as1.sim.mat', overhang=30, #customizable\n",
    "               gap_penalties_1d=(-450, -50), #customizable\n",
    "               gap_penalties_3d=(0, 3), gap_gap_score=0, gap_residue_score=0, #customizable\n",
    "               dendrogram_file='ldh.tree', #rename with the family name\n",
    "               alignment_type='tree', # If 'progresive', the tree is not\n",
    "                                      # computed and all structues will be\n",
    "                                      # aligned sequentially to the first\n",
    "               feature_weights=weights, # For a multiple sequence alignment only\n",
    "                                        # the first feature needs to be non-zero\n",
    "               improve_alignment=True, fit=True, write_fit=write_fit,\n",
    "               write_whole_pdb=whole, output='ALIGNMENT QUALITY')\n",
    "\n",
    "aln.write(file='ldh.pap', alignment_format='PAP')  #rename with the family name\n",
    "aln.write(file='ldh.ali', alignment_format='PIR')  #rename with the family name\n",
    "\n",
    "aln.salign(rms_cutoff=1.0, normalize_pp_scores=False,\n",
    "           rr_file='$(LIB)/as1.sim.mat', overhang=30,\n",
    "           gap_penalties_1d=(-450, -50), gap_penalties_3d=(0, 3),\n",
    "           gap_gap_score=0, gap_residue_score=0, dendrogram_file='multA.tree',\n",
    "           alignment_type='progressive', feature_weights=[0]*6,\n",
    "           improve_alignment=False, fit=False, write_fit=True,\n",
    "           write_whole_pdb=False, output='QUALITY')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                                               .--- 3d5tA          0.2657\r\n",
      "                                                               |\r\n",
      "                                                             .----- 1bdmA          0.2714\r\n",
      "                                                             |\r\n",
      "                                   .------------------------------- 6itkA          0.3447\r\n",
      "                                   |\r\n",
      "                          .---------------------------------------- 4h7pA          0.3704\r\n",
      "                          |\r\n",
      "        .---------------------------------------------------------- 5mdhA          0.4199\r\n",
      "        |\r\n",
      "      .------------------------------------------------------------ 7mdhA\r\n",
      "\r\n",
      "      +----+----+----+----+----+----+----+----+----+----+----+----+\r\n",
      "     0.4260    0.3983    0.3705    0.3428    0.3150    0.2873    0.2595\r\n",
      "          0.4121    0.3844    0.3566    0.3289    0.3011    0.2734\r\n",
      "\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "#To display the distance/tree\n",
    "cat ldh.tree"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      ">P1;3d5tA\r\n",
      "structureX:3d5t_fit.pdb:   3 :A:+321 :A:MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B, C, D; EC  1.1.1.37; ENGINEERED  YES:MOL_ID  1; ORGANISM_SCIENTIFIC  BURKHOLDERIA PSEUDOMALLEI; ORGANISM_TAXID  320372; STRAIN  1710B; GENE  MDH, BURPS1710B_A0795; EXPRESSION_SYSTEM  ESCHERICHIA COLI; EXPRESSION_SYSTEM_TAXID  562; EXPRESSION_SYSTEM_STRAIN  BL21(DE3); EXPRESSION_SYSTEM_VECTOR_TYPE  PLASMID; EXPRESSION_SYSTEM_PLASMID  AVA0421: 2.51: 0.18\r\n",
      "---------KPAKRVAVTGAAGQIAYSLLFRIANGDLLGKDQPVILQLLDLPQAQAAVKGVVMELDDCAFPLLAG\r\n",
      "VVITDDPKVAFKDADVALLVGARPRS--MERKDLLSANAEIFTVQGAALNEVASRDVKVLVVGNPANTNAYIAMK\r\n",
      "SA-PDLPKKNFTAMLRLDHNRALSQLAAKSGKPVASIEKLAVWGNHSPTMYPDFRFATAE----GESLLKLINDD\r\n",
      "VWNRDTFIPTVGKRGAAIIEARGLSSAASAANAAIDHVRDWVLGTNG-KWVTMGIPSDGS-YGIPEDIIYGVPVI\r\n",
      "CE-NGEYKRVEGLEIDAFSREKMDGTLAELLEERDGVAH---L-------------*\r\n",
      "\r\n",
      ">P1;6itkA\r\n",
      "structureX:6itk_fit.pdb:   8 :A:+318 :A:MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED  YES:MOL_ID  1; ORGANISM_SCIENTIFIC  CORYNEBACTERIUM GLUTAMICUM ATCC 13032; ORGANISM_TAXID  196627; STRAIN  ATCC 13032 / DSM 20300 / JCM 1318 / LMG 3730 / NCIM GENE  MDH, CGL2380, CG2613; EXPRESSION_SYSTEM  ESCHERICHIA COLI BL21(DE3); EXPRESSION_SYSTEM_TAXID  469008; EXPRESSION_SYSTEM_STRAIN  BL21(DE3); EXPRESSION_SYSTEM_VECTOR_TYPE  PLASMID; EXPRESSION_SYSTEM_PLASMID  PET30A: 2.00: 0.19\r\n",
      "----------STKKVTVTGAAGQISYSLLWRIANGEVFGTDTPVELKLLEIPQALGGAEGVAMELLDSAFPLLRN\r\n",
      "ITITADANEAFDGANAAFLVGAKP-----ERADLLANNGKIFGPQGKAINDNAADDIRVLVVGNPANTNALIASA\r\n",
      "AA-PDVPASRFNAMMRLDHNRAISQLATKLGRGSAEFNNIVVWGNHSATQFPDITYATVG----GEKVTDLV-DH\r\n",
      "DWYVEEFIPRVANRGAEIIEVRGKSSAASAASSAIDHMRDWVQGTE--AWSSAAIPSTGA-YGIPEGIFVGLPTV\r\n",
      "SR-NGEWEIVEGLEISDFQRARIDANAQELQAEREAVRDLLLE-------------*\r\n",
      "\r\n",
      ">P1;5mdhA\r\n",
      "structureX:5mdh_fit.pdb:   1 :A:+333 :A:MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED  YES:MOL_ID  1; ORGANISM_SCIENTIFIC  SUS SCROFA; ORGANISM_COMMON  PIG; ORGANISM_TAXID  9823; ORGAN  HEART; TISSUE  MUSCLE; CELLULAR_LOCATION  CYTOPLASM; EXPRESSION_SYSTEM  ESCHERICHIA COLI; EXPRESSION_SYSTEM_TAXID  562: 2.40: 0.20\r\n",
      "---------SEPIRVLVTGAAGQIAYSLLYSIGNGSVFGKDQPIILVLLDITPMMGVLDGVLMELQDCALPLLKD\r\n",
      "VIATDKEEIAFKDLDVAILVGSMPRRDGMERKDLLKANVKIFKCQGAALDKYAKKSVKVIVVGNPANTNCLTASK\r\n",
      "SA-PSIPKENFSCLTRLDHNRAKAQIALKLGVTSDDVKNVIIWGNHSSTQYPDVNHAKVKLQAKEVGVYEAVKDD\r\n",
      "SWLKGEFITTVQQRGAAVIKARKLSSAMSAAKAICDHVRDIWFGTPEGEFVSMGIISDGNSYGVPDDLLYSFPVT\r\n",
      "IK-DKTWKIVEGLPINDFSREKMDLTAKELAEEKETAFEFLSSA------------*\r\n",
      "\r\n",
      ">P1;1bdmA\r\n",
      "structureX:1bdm_fit.pdb:   0 :A:+318 :A:MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED  YES:MOL_ID  1; ORGANISM_SCIENTIFIC  THERMUS THERMOPHILUS; ORGANISM_TAXID  274; EXPRESSION_SYSTEM_VECTOR_TYPE  BACTERIUM: 1.80: 0.17\r\n",
      "--------MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGKDQPVILQLLEIPQAMKALEGVVMELEDCAFPLLAG\r\n",
      "LEATDDPDVAFKDADYALLVGAAPR---------LQVNGKIFTEQGRALAEVAKKDVKVLVVGNPANTNALIAYK\r\n",
      "NA-PGLNPRNFTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSIMFPDLFHAEVD----GRPALELV-DM\r\n",
      "EWYEKVFIPTVAQRGAAIIQARGASSAASAANAAIEHIRDWALGTPEGDWVSMAVPSQGE-YGIPEGIVYSFPVT\r\n",
      "AK-DGAYRVVEGLEINEFARKRMEITAQELLDEMEQVKALGLI-------------*\r\n",
      "\r\n",
      ">P1;7mdhA\r\n",
      "structureX:7mdh_fit.pdb:  23 :A:+351 :A:MOL_ID  1; MOLECULE  PROTEIN (MALATE DEHYDROGENASE); CHAIN  A, B, C, D; EC  1.1.1.82; ENGINEERED  YES; MUTATION  YES:MOL_ID  1; ORGANISM_SCIENTIFIC  SORGHUM BICOLOR; ORGANISM_COMMON  SORGHUM; ORGANISM_TAXID  4558; TISSUE  LEAF; ORGANELLE  CHLOROPLAST; EXPRESSION_SYSTEM  ESCHERICHIA COLI BL21(DE3); EXPRESSION_SYSTEM_TAXID  469008; EXPRESSION_SYSTEM_STRAIN  BL21(DE3); EXPRESSION_SYSTEM_PLASMID  PET: 2.40: 0.22\r\n",
      "DCFGVFCTWKKLVNIAVSGAAGMISNHLLFKLASGEVFGQDQPIALKLLGSERSFQALEGVAMELEDSLYPLLRE\r\n",
      "VSIGIDPYEVFEDVDWALLIGAKPRGPGMERAALLDINGQIFADQGKALNAVASKNVKVLVVGNPCNTNALICLK\r\n",
      "NA-PDIPAKNFHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDFLNAKID----GRPVKEVIKRT\r\n",
      "KWLEEEFTITVQKRGGALIQKWGRSSAASTAVSIADAIKSLVTPTPEGDWFSTGVYTTGNPYGIAEDIVFSMPCR\r\n",
      "SKGDGDYELATDVSNDDFLWERIKKSEAELLAEKKCVAHLTGEGNAYCDVPEDTML*\r\n",
      "\r\n",
      ">P1;4h7pA\r\n",
      "structureX:4h7p_fit.pdb:   1 :A:+313 :A:MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED  YES:MOL_ID  1; ORGANISM_SCIENTIFIC  LEISHMANIA MAJOR; ORGANISM_TAXID  347515; STRAIN  FRIEDLIN; GENE  CMDH, LMJF_28_2860; EXPRESSION_SYSTEM  ESCHERICHIA COLI; EXPRESSION_SYSTEM_TAXID  469008; EXPRESSION_SYSTEM_STRAIN  BL21(DE3); EXPRESSION_SYSTEM_VECTOR_TYPE  PLASMID; EXPRESSION_SYSTEM_PLASMID  AVA0421: 1.30: 0.15\r\n",
      "---------MSAVKVAVTGAAGQIGYALVPLIARGALLGPTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDK\r\n",
      "VVVTADPRVAFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLK\r\n",
      "SAQGKLNPRHVTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVIG----TTPAREAIK--\r\n",
      "-----DFVQVVRGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDENPYGVPSGLIFSFPCT\r\n",
      "CH-AGEWTVVSGKL-----KQRLASTIAELQEERAQAG----L-------------*\r\n"
     ]
    }
   ],
   "source": [
    "#To display the alignment\n",
    "cat ldh.ali"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Align query/target sequence to the template structures"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      2738666    2674.479     2.612\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      2739194    2674.994     2.612\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      2787794    2722.455     2.659\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      2801094    2735.443     2.671\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ldh.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1799214    1757.045     1.716\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1800664    1758.461     1.717\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1803564    1761.293     1.720\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1809364    1766.957     1.726\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1934666    1889.322     1.845\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1935946    1890.572     1.846\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1937214    1891.811     1.847\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1938542    1893.107     1.849\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1939810    1894.346     1.850\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1941210    1895.713     1.851\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1942458    1896.932     1.852\n",
      "\n",
      "Read the alignment from file       : ldh.ali\n",
      "\n",
      "Total number of alignment positions:   356\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      3d5tA     321      1 3d5t_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B, C, D; EC  1.1.1.37; ENGI\n",
      "  2      6itkA     318      1 6itk_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  3      5mdhA     333      1 5mdh_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  4      1bdmA     318      1 1bdm_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  5      7mdhA     351      1 7mdh_fit.pd MOL_ID  1; MOLECULE  PROTEIN (MALATE DEHYDROGENASE); CHAIN  A, B, C, D; EC  1.1.\n",
      "  6      4h7pA     313      1 4h7p_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "openf___224_> Open           TvLDH.ali\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1943794    1898.236     1.854\n",
      "\n",
      "Read the alignment from file       : TvLDH.ali\n",
      "\n",
      "Total number of alignment positions:   356\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      3d5tA     321      1 3d5t_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B, C, D; EC  1.1.1.37; ENGI\n",
      "  2      6itkA     318      1 6itk_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  3      5mdhA     333      1 5mdh_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  4      1bdmA     318      1 1bdm_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  5      7mdhA     351      1 7mdh_fit.pd MOL_ID  1; MOLECULE  PROTEIN (MALATE DEHYDROGENASE); CHAIN  A, B, C, D; EC  1.1.\n",
      "  6      4h7pA     313      1 4h7p_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  7      TvLDH     335      1       TvLDH\n",
      "  ALIGN_CODES                                   : 3d5tA 6itkA 5mdhA 1bdmA 7mdhA 4h7pA TvLDH \n",
      "  Atom type for alignment (FIT_ATOMS)           : CA\n",
      "  FIT                                           :         T\n",
      "  IMPROVE_ALIGNMENT                             :         T\n",
      "  WATER_IO, HETATM_IO, HYDROGEN_IO              :         F        F        F\n",
      "  NO_TER                                        :         F\n",
      "  OUTPUT                                        :\n",
      "  FIT_ON_FIRST                                  :         F\n",
      "  WRITE_FIT                                     :         F\n",
      "  FIT_PDBNAM                                    :         T\n",
      "  WRITE_WHOLE_PDB                               :         T\n",
      "  CURRENT_DIRECTORY                             :         T\n",
      "  Feature weights (FEATURE_WEIGHTS)             :    1.0000   0.0000   0.0000   0.0000   0.0000   0.0000\n",
      "  NORMALIZE_PP_SCORES                           :         F\n",
      "  Gap initiation penalty (GAP_PENALTIES[1])     :  -450.0000\n",
      "  Gap extension penalty (GAP_PENALTIES[2])      :     0.0000\n",
      "  Break-break bonus                             : 10000.0000\n",
      "  Gap_function (align2d flag)                    :         T\n",
      "  2-D Gap penalties                             :  0.35  1.20  0.90  1.20  0.60  8.60  1.20  0.00\n",
      "  GAP-GAP_SCORE                                 :    0.0000\n",
      "  GAP-RESIDUE_SCORE                             :    0.0000\n",
      "  Perform gap-residue and gap-gap correction    :         T\n",
      "  Residue type - residue type file (RR_FILE)    : $(LIB)/as1.sim.mat\n",
      "  OFF_DIAGONAL                                  :       100\n",
      "  OVERHANG                                      :         0\n",
      "  LOCAL_ALIGNMENT                               :         F\n",
      "  MATRIX_OFFSET                                 :    0.0000\n",
      "  N_SUBOPT                                      :         0\n",
      "  SUBOPT_OFFSET                                 :    0.0000\n",
      "  Gap introduction penalty (GAP_PENALTIES_3D_1) :    0.0000\n",
      "  Gap extension penalty (GAP_PENALTIES_3D_2)    :    1.7500\n",
      "  Max dist for equiv (2 * GAP_PENALTIES_3D_2)   :    3.5000\n",
      "  ALIGN3D_TRF                                   :         F\n",
      "  RMS_CUTOFF                                    :    3.5\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           3d5t_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2041819    1993.964     1.947\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2041819    1993.964     1.947\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2041990    1994.131     1.947\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2042840    1994.961     1.948\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2044098    1996.189     1.949\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2046002    1998.049     1.951\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2046002    1998.049     1.951\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2046092    1998.137     1.951\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2048948    2000.926     1.954\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2048948    2000.926     1.954\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2049083    2001.058     1.954\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2053367    2005.241     1.958\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2053367    2005.241     1.958\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2053565    2005.435     1.958\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2059991    2011.710     1.965\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2059991    2011.710     1.965\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2060288    2012.000     1.965\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2069910    2021.396     1.974\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2069910    2021.396     1.974\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2070360    2021.836     1.974\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2084810    2035.947     1.988\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2084810    2035.947     1.988\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2085485    2036.606     1.989\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2107143    2057.757     2.010\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2107143    2057.757     2.010\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2108151    2058.741     2.010\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2140655    2090.483     2.041\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2140511    2090.343     2.041\n",
      "openf___224_> Open           6itk_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2237636    2185.191     2.134\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2237636    2185.191     2.134\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2237807    2185.358     2.134\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2238657    2186.188     2.135\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2239915    2187.417     2.136\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2241819    2189.276     2.138\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2241819    2189.276     2.138\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2241909    2189.364     2.138\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2244765    2192.153     2.141\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2244765    2192.153     2.141\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2244900    2192.285     2.141\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2249184    2196.469     2.145\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2249184    2196.469     2.145\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2249382    2196.662     2.145\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2255808    2202.938     2.151\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2255808    2202.938     2.151\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2256105    2203.228     2.152\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2265727    2212.624     2.161\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2265727    2212.624     2.161\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2266177    2213.063     2.161\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2280627    2227.175     2.175\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2280627    2227.175     2.175\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2281302    2227.834     2.176\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2302960    2248.984     2.196\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2302960    2248.984     2.196\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2303968    2249.969     2.197\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2336472    2281.711     2.228\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2336301    2281.544     2.228\n",
      "openf___224_> Open           5mdh_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2437926    2380.787     2.325\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2437926    2380.787     2.325\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2438097    2380.954     2.325\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2438947    2381.784     2.326\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2440205    2383.013     2.327\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2442109    2384.872     2.329\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2442109    2384.872     2.329\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2442199    2384.960     2.329\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2445055    2387.749     2.332\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2445055    2387.749     2.332\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2445190    2387.881     2.332\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2449474    2392.064     2.336\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2449474    2392.064     2.336\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2449672    2392.258     2.336\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2456098    2398.533     2.342\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2456098    2398.533     2.342\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2456395    2398.823     2.343\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2466017    2408.220     2.352\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2466017    2408.220     2.352\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2466467    2408.659     2.352\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2480917    2422.771     2.366\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2480917    2422.771     2.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2481592    2423.430     2.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2503250    2444.580     2.387\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2503250    2444.580     2.387\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2504258    2445.564     2.388\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2536762    2477.307     2.419\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2536726    2477.271     2.419\n",
      "openf___224_> Open           1bdm_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2633851    2572.120     2.512\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2633851    2572.120     2.512\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2634022    2572.287     2.512\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2634872    2573.117     2.513\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2636130    2574.346     2.514\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2638034    2576.205     2.516\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2638034    2576.205     2.516\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2638124    2576.293     2.516\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2640980    2579.082     2.519\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2640980    2579.082     2.519\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2641115    2579.214     2.519\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2645399    2583.397     2.523\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2645399    2583.397     2.523\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2645597    2583.591     2.523\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2652023    2589.866     2.529\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2652023    2589.866     2.529\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2652320    2590.156     2.529\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2661942    2599.553     2.539\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2661942    2599.553     2.539\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2662392    2599.992     2.539\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2676842    2614.104     2.553\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2676842    2614.104     2.553\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2677517    2614.763     2.553\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2699175    2635.913     2.574\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2699175    2635.913     2.574\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2700183    2636.897     2.575\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2732687    2668.640     2.606\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2732516    2668.473     2.606\n",
      "openf___224_> Open           7mdh_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2839541    2772.989     2.708\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2839541    2772.989     2.708\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2839712    2773.156     2.708\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2840562    2773.986     2.709\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2841820    2775.215     2.710\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2843724    2777.074     2.712\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2843724    2777.074     2.712\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2843814    2777.162     2.712\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2846670    2779.951     2.715\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2846670    2779.951     2.715\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2846805    2780.083     2.715\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2851089    2784.267     2.719\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2851089    2784.267     2.719\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2851287    2784.460     2.719\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2857713    2790.735     2.725\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2857713    2790.735     2.725\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2858010    2791.025     2.726\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2867632    2800.422     2.735\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2867632    2800.422     2.735\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2868082    2800.861     2.735\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2882532    2814.973     2.749\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2882532    2814.973     2.749\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2883207    2815.632     2.750\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2904865    2836.782     2.770\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2904865    2836.782     2.770\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2905873    2837.767     2.771\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2938377    2869.509     2.802\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2938377    2869.509     2.802\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2939889    2870.985     2.804\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2938503    2869.632     2.802\n",
      "openf___224_> Open           4h7p_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3034128    2963.016     2.894\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3034128    2963.016     2.894\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3034299    2963.183     2.894\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3035149    2964.013     2.895\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3036407    2965.241     2.896\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3038311    2967.101     2.898\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3038311    2967.101     2.898\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3038401    2967.188     2.898\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3041257    2969.978     2.900\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3041257    2969.978     2.900\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3041392    2970.109     2.900\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3045676    2974.293     2.905\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3045676    2974.293     2.905\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3045874    2974.486     2.905\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3052300    2980.762     2.911\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3052300    2980.762     2.911\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3052597    2981.052     2.911\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3062219    2990.448     2.920\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3062219    2990.448     2.920\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3062669    2990.888     2.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3077119    3004.999     2.935\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3077119    3004.999     2.935\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3077794    3005.658     2.935\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3099452    3026.809     2.956\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3099452    3026.809     2.956\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3100460    3027.793     2.957\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3132964    3059.535     2.988\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      3132748    3059.324     2.988\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "SALIGN_____> adding the next group to the alignment; iteration    1\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "\n",
      "Current alignment total score:        200.8\n",
      "\n",
      "pos_scr ... position-position dissimilarity from the dynamic programming matrix\n",
      "avr_avr ... distance between the two averages\n",
      "avr_dst ... for all aligned structures, average distance to the average structure\n",
      "std_dev ... for all aligned structures, standard deviation of distance to ave str\n",
      "\n",
      "group:                                 1       1       1       1       1       1       2       \n",
      "   N pos_scr avr_ave avr_dst std_dev   3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   TvLDH   \n",
      "----------------------------------------------------------------------------------------------\n",
      "   1    0.00    0.00    0.00    0.00   -       -       -       -       D       -       -       \n",
      "   2    0.00    0.00    0.00    0.00   -       -       -       -       C       -       -       \n",
      "   3    0.00    0.00    0.00    0.00   -       -       -       -       F       -       -       \n",
      "   4    0.00    0.00    0.00    0.00   -       -       -       -       G       -       -       \n",
      "   5    0.00    0.00    0.00    0.00   -       -       -       -       V       -       -       \n",
      "   6    0.00    0.00    0.00    0.00   -       -       -       -       F       -       -       \n",
      "   7    0.00    0.00    0.00    0.00   -       -       -       -       C       -       -       \n",
      "   8    0.00    0.00    0.00    0.00   -       -       -       -       T       -       -       \n",
      "   9    0.18    0.00    0.00    0.00   -       -       -       M       W       -       M       \n",
      "  10    0.45    0.00    0.00    0.00   K       -       S       K       K       M       S       \n",
      "  11    0.55    0.00    0.00    0.00   P       S       E       A       K       S       E       \n",
      "  12    0.63    0.00    0.00    0.00   A       T       P       P       L       A       A       \n",
      "  13    0.53    0.00    0.00    0.00   K       K       I       V       V       V       A       \n",
      "  14    0.43    0.00    0.00    0.00   R       K       R       R       N       K       H       \n",
      "  15    0.85    0.00    0.00    0.00   V       V       V       V       I       V       V       \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  16    0.53    0.00    0.00    0.00   A       T       L       A       A       A       L       \n",
      "  17    0.70    0.00    0.00    0.00   V       V       V       V       V       V       I       \n",
      "  18    0.84    0.00    0.00    0.00   T       T       T       T       S       T       T       \n",
      "  19    0.96    0.00    0.00    0.00 * G       G       G       G       G       G       G       \n",
      "  20    0.86    0.00    0.00    0.00 * A       A       A       A       A       A       A       \n",
      "  21    0.86    0.00    0.00    0.00 * A       A       A       A       A       A       A       \n",
      "  22    0.96    0.00    0.00    0.00 * G       G       G       G       G       G       G       \n",
      "  23    0.78    0.00    0.00    0.00   Q       Q       Q       Q       M       Q       Q       \n",
      "  24    0.87    0.00    0.00    0.00 * I       I       I       I       I       I       I       \n",
      "  25    0.69    0.00    0.00    0.00   A       S       A       G       S       G       G       \n",
      "  26    0.83    0.00    0.00    0.00   Y       Y       Y       Y       N       Y       Y       \n",
      "  27    0.35    0.00    0.00    0.00   S       S       S       S       H       A       I       \n",
      "  28    0.91    0.00    0.00    0.00 * L       L       L       L       L       L       L       \n",
      "  29    0.37    0.00    0.00    0.00   L       L       L       L       L       V       S       \n",
      "  30    0.31    0.00    0.00    0.00   F       W       Y       F       F       P       H       \n",
      "  31    0.25    0.00    0.00    0.00   R       R       S       R       K       L       W       \n",
      "  32    0.83    0.00    0.00    0.00   I       I       I       I       L       I       I       \n",
      "  33    0.81    0.00    0.00    0.00   A       A       G       A       A       A       A       \n",
      "  34    0.62    0.00    0.00    0.00   N       N       N       A       S       R       S       \n",
      "  35    0.96    0.00    0.00    0.00 * G       G       G       G       G       G       G       \n",
      "  36    0.69    0.00    0.00    0.00   D       E       S       E       E       A       E       \n",
      "  37    0.73    0.00    0.00    0.00   L       V       V       M       V       L       L       \n",
      "  38    0.53    0.00    0.00    0.00   L       F       F       L       F       L       Y       \n",
      "  39    0.96    0.00    0.00    0.00 * G       G       G       G       G       G       G       \n",
      "  40    0.00    0.00    0.00    0.00   K       T       K       K       Q       P       -       \n",
      "  41    0.82    0.00    0.00    0.00   D       D       D       D       D       T       D       \n",
      "  42    0.46    0.00    0.00    0.00   Q       T       Q       Q       Q       T       R       \n",
      "  43    0.35    0.00    0.00    0.00   P       P       P       P       P       P       Q       \n",
      "  44    0.82    0.00    0.00    0.00   V       V       I       V       I       V       V       \n",
      "  45    0.36    0.00    0.00    0.00   I       E       I       I       A       E       Y       \n",
      "  46    0.91    0.00    0.00    0.00 * L       L       L       L       L       L       L       \n",
      "  47    0.42    0.00    0.00    0.00   Q       K       V       Q       K       R       H       \n",
      "  48    0.91    0.00    0.00    0.00 * L       L       L       L       L       L       L       \n",
      "  49    0.91    0.00    0.00    0.00 * L       L       L       L       L       L       L       \n",
      "  50    0.72    0.00    0.00    0.00   D       E       D       E       G       D       D       \n",
      "  51    0.74    0.00    0.00    0.00   L       I       I       I       S       I       I       \n",
      "  52    0.68    0.00    0.00    0.00   P       P       T       P       E       E       P       \n",
      "  53    0.54    0.00    0.00    0.00   Q       Q       P       Q       R       P       P       \n",
      "  54    0.74    0.00    0.00    0.00   A       A       M       A       S       A       A       \n",
      "  55    0.59    0.00    0.00    0.00   Q       L       M       M       F       L       M       \n",
      "  56    0.48    0.00    0.00    0.00   A       G       G       K       Q       K       N       \n",
      "  57    0.39    0.00    0.00    0.00   A       G       V       A       A       A       R       \n",
      "  58    0.79    0.00    0.00    0.00   V       A       L       L       L       L       L       \n",
      "  59    0.49    0.00    0.00    0.00   K       E       D       E       E       A       T       \n",
      "  60    0.58    0.00    0.00    0.00   G       G       G       G       G       G       A       \n",
      "  61    0.65    0.00    0.00    0.00   V       V       V       V       V       V       L       \n",
      "  62    0.49    0.00    0.00    0.00   V       A       L       V       A       E       T       \n",
      "  63    0.72    0.00    0.00    0.00   M       M       M       M       M       A       M       \n",
      "  64    0.86    0.00    0.00    0.00 * E       E       E       E       E       E       E       \n",
      "  65    0.91    0.00    0.00    0.00 * L       L       L       L       L       L       L       \n",
      "  66    0.68    0.00    0.00    0.00   D       L       Q       E       E       E       E       \n",
      "  67    0.89    0.00    0.00    0.00 * D       D       D       D       D       D       D       \n",
      "  68    0.74    0.00    0.00    0.00   C       S       C       C       S       C       C       \n",
      "  69    0.80    0.00    0.00    0.00   A       A       A       A       L       A       A       \n",
      "  70    0.80    0.00    0.00    0.00   F       F       L       F       Y       F       F       \n",
      "  71    0.93    0.00    0.00    0.00 * P       P       P       P       P       P       P       \n",
      "  72    0.32    0.00    0.00    0.00   L       L       L       L       L       L       H       \n",
      "  73    0.91    0.00    0.00    0.00 * L       L       L       L       L       L       L       \n",
      "  74    0.59    0.00    0.00    0.00   A       R       K       A       R       D       A       \n",
      "  75    0.63    0.00    0.00    0.00   G       N       D       G       E       K       G       \n",
      "  76    0.49    0.00    0.00    0.00   V       I       V       L       V       V       F       \n",
      "  77    0.63    0.00    0.00    0.00   V       T       I       E       S       V       V       \n",
      "  78    0.60    0.00    0.00    0.00   I       I       A       A       I       V       A       \n",
      "  79    0.80    0.00    0.00    0.00   T       T       T       T       G       T       T       \n",
      "  80    0.48    0.00    0.00    0.00   D       A       D       D       I       A       T       \n",
      "  81    0.82    0.00    0.00    0.00   D       D       K       D       D       D       D       \n",
      "  82    0.78    0.00    0.00    0.00   P       A       E       P       P       P       P       \n",
      "  83    0.56    0.00    0.00    0.00   K       N       E       D       Y       R       K       \n",
      "  84    0.53    0.00    0.00    0.00   V       E       I       V       E       V       A       \n",
      "  85    0.81    0.00    0.00    0.00   A       A       A       A       V       A       A       \n",
      "  86    0.90    0.00    0.00    0.00 * F       F       F       F       F       F       F       \n",
      "  87    0.68    0.00    0.00    0.00   K       D       K       K       E       D       K       \n",
      "  88    0.75    0.00    0.00    0.00   D       G       D       D       D       G       D       \n",
      "  89    0.56    0.00    0.00    0.00   A       A       L       A       V       V       I       \n",
      "  90    0.78    0.00    0.00    0.00   D       N       D       D       D       A       D       \n",
      "  91    0.20    0.00    0.00    0.00   V       A       V       Y       W       I       C       \n",
      "  92    0.86    0.00    0.00    0.00 * A       A       A       A       A       A       A       \n",
      "  93    0.61    0.00    0.00    0.00   L       F       I       L       L       I       F       \n",
      "  94    0.86    0.00    0.00    0.00   L       L       L       L       L       M       L       \n",
      "  95    0.75    0.00    0.00    0.00   V       V       V       V       I       C       V       \n",
      "  96    0.58    0.00    0.00    0.00   G       G       G       G       G       G       A       \n",
      "  97    0.65    0.00    0.00    0.00   A       A       S       A       A       A       S       \n",
      "  98    0.41    0.00    0.00    0.00   R       K       M       A       K       F       M       \n",
      "  99    0.93    0.00    0.00    0.00 * P       P       P       P       P       P       P       \n",
      " 100    0.33    0.00    0.00    0.00   R       -       R       R       R       R       L       \n",
      " 101    0.41    0.00    0.00    0.00   S       -       R       -       G       K       K       \n",
      " 102    0.32    0.00    0.00    0.00   -       -       D       -       P       A       P       \n",
      " 103    0.48    0.00    0.00    0.00   -       -       G       -       G       G       G       \n",
      " 104    0.22    0.00    0.00    0.00   M       -       M       -       M       M       Q       \n",
      " 105    0.32    0.00    0.00    0.00   E       E       E       -       E       E       V       \n",
      " 106    0.73    0.00    0.00    0.00   R       R       R       -       R       R       R       \n",
      " 107    0.55    0.00    0.00    0.00   K       A       K       -       A       K       A       \n",
      " 108    0.68    0.00    0.00    0.00   D       D       D       -       A       D       D       \n",
      " 109    0.76    0.00    0.00    0.00   L       L       L       -       L       L       L       \n",
      " 110    0.65    0.00    0.00    0.00   L       L       L       L       L       L       I       \n",
      " 111    0.58    0.00    0.00    0.00   S       A       K       Q       D       E       S       \n",
      " 112    0.47    0.00    0.00    0.00   A       N       A       V       I       M       S       \n",
      " 113    0.82    0.00    0.00    0.00 * N       N       N       N       N       N       N       \n",
      " 114    0.53    0.00    0.00    0.00   A       G       V       G       G       A       S       \n",
      " 115    0.39    0.00    0.00    0.00   E       K       K       K       Q       R       V       \n",
      " 116    0.87    0.00    0.00    0.00 * I       I       I       I       I       I       I       \n",
      " 117    0.90    0.00    0.00    0.00 * F       F       F       F       F       F       F       \n",
      " 118    0.62    0.00    0.00    0.00   T       G       K       T       A       K       K       \n",
      " 119    0.40    0.00    0.00    0.00   V       P       C       E       D       E       N       \n",
      " 120    0.44    0.00    0.00    0.00   Q       Q       Q       Q       Q       Q       T       \n",
      " 121    0.96    0.00    0.00    0.00 * G       G       G       G       G       G       G       \n",
      " 122    0.57    0.00    0.00    0.00   A       K       A       R       K       E       E       \n",
      " 123    0.36    0.00    0.00    0.00   A       A       A       A       A       A       Y       \n",
      " 124    0.82    0.00    0.00    0.00   L       I       L       L       L       I       L       \n",
      " 125    0.58    0.00    0.00    0.00   N       N       D       A       N       A       S       \n",
      " 126    0.58    0.00    0.00    0.00   E       D       K       E       A       A       K       \n",
      " 127    0.26    0.00    0.00    0.00   V       N       Y       V       V       V       W       \n",
      " 128    0.86    0.00    0.00    0.00 * A       A       A       A       A       A       A       \n",
      " 129    0.63    0.00    0.00    0.00   S       A       K       K       S       A       K       \n",
      " 130    0.42    0.00    0.00    0.00   R       D       K       K       K       S       P       \n",
      " 131    0.61    0.00    0.00    0.00   D       D       S       D       N       D       S       \n",
      " 132    0.75    0.00    0.00    0.00   V       I       V       V       V       C       V       \n",
      " 133    0.78    0.00    0.00    0.00   K       R       K       K       K       R       K       \n",
      " 134    0.89    0.00    0.00    0.00 * V       V       V       V       V       V       V       \n",
      " 135    0.82    0.00    0.00    0.00   L       L       I       L       L       V       L       \n",
      " 136    0.89    0.00    0.00    0.00 * V       V       V       V       V       V       V       \n",
      " 137    0.70    0.00    0.00    0.00   V       V       V       V       V       V       I       \n",
      " 138    0.96    0.00    0.00    0.00 * G       G       G       G       G       G       G       \n",
      " 139    0.82    0.00    0.00    0.00 * N       N       N       N       N       N       N       \n",
      " 140    0.93    0.00    0.00    0.00 * P       P       P       P       P       P       P       \n",
      " 141    0.43    0.00    0.00    0.00   A       A       A       A       C       A       D       \n",
      " 142    0.82    0.00    0.00    0.00 * N       N       N       N       N       N       N       \n",
      " 143    0.88    0.00    0.00    0.00 * T       T       T       T       T       T       T       \n",
      " 144    0.82    0.00    0.00    0.00 * N       N       N       N       N       N       N       \n",
      " 145    0.43    0.00    0.00    0.00   A       A       C       A       A       A       C       \n",
      " 146    0.33    0.00    0.00    0.00   Y       L       L       L       L       L       E       \n",
      " 147    0.80    0.00    0.00    0.00   I       I       T       I       I       I       I       \n",
      " 148    0.71    0.00    0.00    0.00   A       A       A       A       C       L       A       \n",
      " 149    0.49    0.00    0.00    0.00   M       S       S       Y       L       L       M       \n",
      " 150    0.43    0.00    0.00    0.00   K       A       K       K       K       K       L       \n",
      " 151    0.40    0.00    0.00    0.00   S       A       S       N       N       S       H       \n",
      " 152    0.86    0.00    0.00    0.00 * A       A       A       A       A       A       A       \n",
      " 153    0.00    0.00    0.00    0.00   -       -       -       -       -       Q       -       \n",
      " 154    0.43    0.00    0.00    0.00   P       P       P       P       P       G       K       \n",
      " 155    0.56    0.00    0.00    0.00   D       D       S       G       D       K       N       \n",
      " 156    0.78    0.00    0.00    0.00   L       V       I       L       I       L       L       \n",
      " 157    0.45    0.00    0.00    0.00   P       P       P       N       P       N       K       \n",
      " 158    0.62    0.00    0.00    0.00   K       A       K       P       A       P       P       \n",
      " 159    0.54    0.00    0.00    0.00   K       S       E       R       K       R       E       \n",
      " 160    0.70    0.00    0.00    0.00   N       R       N       N       N       H       N       \n",
      " 161    0.83    0.00    0.00    0.00   F       F       F       F       F       V       F       \n",
      " 162    0.64    0.00    0.00    0.00   T       N       S       T       H       T       S       \n",
      " 163    0.53    0.00    0.00    0.00   A       A       C       A       A       A       S       \n",
      " 164    0.73    0.00    0.00    0.00   M       M       L       M       L       M       L       \n",
      " 165    0.56    0.00    0.00    0.00   L       M       T       T       T       T       S       \n",
      " 166    0.22    0.00    0.00    0.00   R       R       R       R       R       R       M       \n",
      " 167    0.91    0.00    0.00    0.00 * L       L       L       L       L       L       L       \n",
      " 168    0.89    0.00    0.00    0.00 * D       D       D       D       D       D       D       \n",
      " 169    0.48    0.00    0.00    0.00   H       H       H       H       E       H       Q       \n",
      " 170    0.82    0.00    0.00    0.00 * N       N       N       N       N       N       N       \n",
      " 171    0.87    0.00    0.00    0.00 * R       R       R       R       R       R       R       \n",
      " 172    0.86    0.00    0.00    0.00 * A       A       A       A       A       A       A       \n",
      " 173    0.37    0.00    0.00    0.00   L       I       K       K       K       L       Y       \n",
      " 174    0.34    0.00    0.00    0.00   S       S       A       A       C       S       Y       \n",
      " 175    0.53    0.00    0.00    0.00   Q       Q       Q       Q       Q       L       E       \n",
      " 176    0.65    0.00    0.00    0.00   L       L       I       L       L       L       V       \n",
      " 177    0.86    0.00    0.00    0.00 * A       A       A       A       A       A       A       \n",
      " 178    0.50    0.00    0.00    0.00   A       T       L       K       L       R       S       \n",
      " 179    0.86    0.00    0.00    0.00 * K       K       K       K       K       K       K       \n",
      " 180    0.59    0.00    0.00    0.00   S       L       L       T       A       A       L       \n",
      " 181    0.96    0.00    0.00    0.00 * G       G       G       G       G       G       G       \n",
      " 182    0.65    0.00    0.00    0.00   K       R       V       T       V       V       V       \n",
      " 183    0.43    0.00    0.00    0.00   P       G       T       G       F       P       D       \n",
      " 184    0.64    0.00    0.00    0.00   V       S       S       V       Y       V       V       \n",
      " 185    0.50    0.00    0.00    0.00   A       A       D       D       D       S       K       \n",
      " 186    0.55    0.00    0.00    0.00   S       E       D       R       K       Q       D       \n",
      " 187    0.75    0.00    0.00    0.00   I       F       V       I       V       V       V       \n",
      " 188    0.41    0.00    0.00    0.00   E       N       K       R       S       R       H       \n",
      " 189    0.54    0.00    0.00    0.00   K       N       N       R       N       N       D       \n",
      " 190    0.69    0.00    0.00    0.00   L       I       V       M       V       V       I       \n",
      " 191    0.62    0.00    0.00    0.00   A       V       I       T       T       I       I       \n",
      " 192    0.79    0.00    0.00    0.00   V       V       I       V       I       I       V       \n",
      " 193    0.95    0.00    0.00    0.00 * W       W       W       W       W       W       W       \n",
      " 194    0.96    0.00    0.00    0.00 * G       G       G       G       G       G       G       \n",
      " 195    0.82    0.00    0.00    0.00 * N       N       N       N       N       N       N       \n",
      " 196    0.87    0.00    0.00    0.00 * H       H       H       H       H       H       H       \n",
      " 197    0.52    0.00    0.00    0.00   S       S       S       S       S       S       G       \n",
      " 198    0.47    0.00    0.00    0.00   P       A       S       S       T       S       E       \n",
      " 199    0.62    0.00    0.00    0.00   T       T       T       I       T       T       S       \n",
      " 200    0.49    0.00    0.00    0.00   M       Q       Q       M       Q       Q       M       \n",
      " 201    0.57    0.00    0.00    0.00   Y       F       Y       F       V       V       V       \n",
      " 202    0.52    0.00    0.00    0.00   P       P       P       P       P       P       A       \n",
      " 203    0.89    0.00    0.00    0.00 * D       D       D       D       D       D       D       \n",
      " 204    0.63    0.00    0.00    0.00   F       I       V       L       F       T       L       \n",
      " 205    0.50    0.00    0.00    0.00   R       T       N       F       L       D       T       \n",
      " 206    0.39    0.00    0.00    0.00   F       Y       H       H       N       S       Q       \n",
      " 207    0.86    0.00    0.00    0.00 * A       A       A       A       A       A       A       \n",
      " 208    0.63    0.00    0.00    0.00   T       T       K       E       K       V       T       \n",
      " 209    0.46    0.00    0.00    0.00   A       V       V       V       I       I       F       \n",
      " 210    0.45    0.00    0.00    0.00   E       G       K       D       D       G       T       \n",
      " 211    0.07    0.00    0.00    0.00   -       -       L       -       -       -       K       \n",
      " 212    0.09    0.00    0.00    0.00   -       -       Q       -       -       -       E       \n",
      " 213    0.10    0.00    0.00    0.00   -       -       A       -       -       -       G       \n",
      " 214    0.14    0.00    0.00    0.00   -       -       K       -       -       -       K       \n",
      " 215    0.49    0.00    0.00    0.00   G       G       E       G       G       T       T       \n",
      " 216    0.48    0.00    0.00    0.00   E       E       V       R       R       T       Q       \n",
      " 217    0.51    0.00    0.00    0.00   S       K       G       P       P       P       K       \n",
      " 218    0.74    0.00    0.00    0.00   L       V       V       A       V       A       V       \n",
      " 219    0.49    0.00    0.00    0.00   L       T       Y       L       K       R       V       \n",
      " 220    0.62    0.00    0.00    0.00   K       D       E       E       E       E       D       \n",
      " 221    0.66    0.00    0.00    0.00   L       L       A       L       V       A       V       \n",
      " 222    0.65    0.00    0.00    0.00   I       V       V       V       I       I       L       \n",
      " 223    0.00    0.00    0.00    0.00   N       -       K       -       K       K       -       \n",
      " 224    0.65    0.00    0.00    0.00   D       D       D       D       R       -       D       \n",
      " 225    0.37    0.00    0.00    0.00   D       H       D       M       T       -       H       \n",
      " 226    0.47    0.00    0.00    0.00   V       D       S       E       K       -       D       \n",
      " 227    0.40    0.00    0.00    0.00   W       W       W       W       W       -       Y       \n",
      " 228    0.40    0.00    0.00    0.00   N       Y       L       Y       L       -       V       \n",
      " 229    0.25    0.00    0.00    0.00   R       V       K       E       E       -       F       \n",
      " 230    0.51    0.00    0.00    0.00   D       E       G       K       E       -       D       \n",
      " 231    0.54    0.00    0.00    0.00   T       E       E       V       E       D       T       \n",
      " 232    0.90    0.00    0.00    0.00 * F       F       F       F       F       F       F       \n",
      " 233    0.46    0.00    0.00    0.00   I       I       I       I       T       V       F       \n",
      " 234    0.45    0.00    0.00    0.00   P       P       T       P       I       Q       K       \n",
      " 235    0.53    0.00    0.00    0.00   T       R       T       T       T       V       K       \n",
      " 236    0.70    0.00    0.00    0.00   V       V       V       V       V       V       I       \n",
      " 237    0.54    0.00    0.00    0.00   G       A       Q       A       Q       R       G       \n",
      " 238    0.44    0.00    0.00    0.00   K       N       Q       Q       K       G       H       \n",
      " 239    0.87    0.00    0.00    0.00 * R       R       R       R       R       R       R       \n",
      " 240    0.58    0.00    0.00    0.00   G       G       G       G       G       G       A       \n",
      " 241    0.23    0.00    0.00    0.00   A       A       A       A       G       A       W       \n",
      " 242    0.53    0.00    0.00    0.00   A       E       A       A       A       E       D       \n",
      " 243    0.80    0.00    0.00    0.00   I       I       V       I       L       I       I       \n",
      " 244    0.65    0.00    0.00    0.00   I       I       I       I       I       I       L       \n",
      " 245    0.66    0.00    0.00    0.00   E       E       K       Q       Q       Q       E       \n",
      " 246    0.35    0.00    0.00    0.00   A       V       A       A       K       L       H       \n",
      " 247    0.77    0.00    0.00    0.00   R       R       R       R       W       R       R       \n",
      " 248    0.87    0.00    0.00    0.00   G       G       K       G       G       G       G       \n",
      " 249    0.44    0.00    0.00    0.00   L       K       L       A       R       L       F       \n",
      " 250    0.67    0.00    0.00    0.00   S       S       S       S       S       S       T       \n",
      " 251    0.89    0.00    0.00    0.00 * S       S       S       S       S       S       S       \n",
      " 252    0.86    0.00    0.00    0.00 * A       A       A       A       A       A       A       \n",
      " 253    0.70    0.00    0.00    0.00   A       A       M       A       A       M       A       \n",
      " 254    0.89    0.00    0.00    0.00 * S       S       S       S       S       S       S       \n",
      " 255    0.50    0.00    0.00    0.00   A       A       A       A       T       A       P       \n",
      " 256    0.54    0.00    0.00    0.00   A       A       A       A       A       A       T       \n",
      " 257    0.61    0.00    0.00    0.00   N       S       K       N       V       K       K       \n",
      " 258    0.77    0.00    0.00    0.00   A       S       A       A       S       A       A       \n",
      " 259    0.72    0.00    0.00    0.00   A       A       I       A       I       A       A       \n",
      " 260    0.65    0.00    0.00    0.00   I       I       C       I       A       V       I       \n",
      " 261    0.47    0.00    0.00    0.00   D       D       D       E       D       D       Q       \n",
      " 262    0.78    0.00    0.00    0.00   H       H       H       H       A       H       H       \n",
      " 263    0.54    0.00    0.00    0.00   V       M       V       I       I       V       M       \n",
      " 264    0.64    0.00    0.00    0.00   R       R       R       R       K       H       K       \n",
      " 265    0.52    0.00    0.00    0.00   D       D       D       D       S       D       A       \n",
      " 266    0.74    0.00    0.00    0.00   W       W       I       W       L       W       W       \n",
      " 267    0.57    0.00    0.00    0.00   V       V       W       A       V       I       L       \n",
      " 268    0.49    0.00    0.00    0.00   L       Q       F       L       T       H       F       \n",
      " 269    0.87    0.00    0.00    0.00   G       G       G       G       P       G       G       \n",
      " 270    0.88    0.00    0.00    0.00 * T       T       T       T       T       T       T       \n",
      " 271    0.51    0.00    0.00    0.00   N       E       P       P       P       P       A       \n",
      " 272    0.35    0.00    0.00    0.00   G       -       E       E       E       E       P       \n",
      " 273    0.64    0.00    0.00    0.00   -       -       G       G       G       G       G       \n",
      " 274    0.58    0.00    0.00    0.00   K       A       E       D       D       V       E       \n",
      " 275    0.29    0.00    0.00    0.00   W       W       F       W       W       Y       V       \n",
      " 276    0.59    0.00    0.00    0.00   V       S       V       V       F       V       L       \n",
      " 277    0.85    0.00    0.00    0.00   T       S       S       S       S       S       S       \n",
      " 278    0.65    0.00    0.00    0.00   M       A       M       M       T       M       M       \n",
      " 279    0.83    0.00    0.00    0.00   G       A       G       A       G       G       G       \n",
      " 280    0.78    0.00    0.00    0.00   I       I       I       V       V       V       I       \n",
      " 281    0.61    0.00    0.00    0.00   P       P       I       P       Y       Y       P       \n",
      " 282    0.42    0.00    0.00    0.00   S       S       S       S       T       S       V       \n",
      " 283    0.42    0.00    0.00    0.00   D       T       D       Q       T       D       P       \n",
      " 284    0.00    0.00    0.00    0.00   -       -       -       -       -       -       E       \n",
      " 285    0.88    0.00    0.00    0.00   G       G       G       G       G       E       G       \n",
      " 286    0.66    0.00    0.00    0.00   S       A       N       E       N       N       N       \n",
      " 287    0.39    0.00    0.00    0.00   -       -       S       -       P       P       P       \n",
      " 288    0.92    0.00    0.00    0.00 * Y       Y       Y       Y       Y       Y       Y       \n",
      " 289    0.96    0.00    0.00    0.00 * G       G       G       G       G       G       G       \n",
      " 290    0.81    0.00    0.00    0.00   I       I       V       I       I       V       I       \n",
      " 291    0.44    0.00    0.00    0.00   P       P       P       P       A       P       K       \n",
      " 292    0.44    0.00    0.00    0.00   E       E       D       E       E       S       P       \n",
      " 293    0.72    0.00    0.00    0.00   D       G       D       G       D       G       G       \n",
      " 294    0.68    0.00    0.00    0.00   I       I       L       I       I       L       V       \n",
      " 295    0.71    0.00    0.00    0.00   I       F       L       V       V       I       V       \n",
      " 296    0.69    0.00    0.00    0.00   Y       V       Y       Y       F       F       F       \n",
      " 297    0.76    0.00    0.00    0.00   G       G       S       S       S       S       S       \n",
      " 298    0.69    0.00    0.00    0.00   V       L       F       F       M       F       F       \n",
      " 299    0.93    0.00    0.00    0.00 * P       P       P       P       P       P       P       \n",
      " 300    0.51    0.00    0.00    0.00   V       T       V       V       C       C       C       \n",
      " 301    0.44    0.00    0.00    0.00   I       V       T       T       R       T       N       \n",
      " 302    0.44    0.00    0.00    0.00   C       S       I       A       S       C       V       \n",
      " 303    0.46    0.00    0.00    0.00   E       R       K       K       K       H       D       \n",
      " 304    0.07    0.00    0.00    0.00   -       -       -       -       G       -       K       \n",
      " 305    0.54    0.00    0.00    0.00   N       N       D       D       D       A       E       \n",
      " 306    0.87    0.00    0.00    0.00   G       G       K       G       G       G       G       \n",
      " 307    0.53    0.00    0.00    0.00   E       E       T       A       D       E       K       \n",
      " 308    0.32    0.00    0.00    0.00   Y       W       W       Y       Y       W       I       \n",
      " 309    0.40    0.00    0.00    0.00   K       E       K       R       E       T       H       \n",
      " 310    0.69    0.00    0.00    0.00   R       I       I       V       L       V       V       \n",
      " 311    0.83    0.00    0.00    0.00   V       V       V       V       A       V       V       \n",
      " 312    0.73    0.00    0.00    0.00   E       E       E       E       T       S       E       \n",
      " 313    0.88    0.00    0.00    0.00   G       G       G       G       D       G       G       \n",
      " 314    0.51    0.00    0.00    0.00   L       L       L       L       V       K       F       \n",
      " 315    0.50    0.00    0.00    0.00   E       E       P       E       S       L       K       \n",
      " 316    0.52    0.00    0.00    0.00   I       I       I       I       N       -       V       \n",
      " 317    0.57    0.00    0.00    0.00   D       S       N       N       D       -       N       \n",
      " 318    0.63    0.00    0.00    0.00   A       D       D       E       D       -       D       \n",
      " 319    0.36    0.00    0.00    0.00   F       F       F       F       F       -       W       \n",
      " 320    0.42    0.00    0.00    0.00   S       Q       S       A       L       -       L       \n",
      " 321    0.73    0.00    0.00    0.00   R       R       R       R       W       K       R       \n",
      " 322    0.70    0.00    0.00    0.00   E       A       E       K       E       Q       E       \n",
      " 323    0.71    0.00    0.00    0.00   K       R       K       R       R       R       K       \n",
      " 324    0.68    0.00    0.00    0.00   M       I       M       M       I       L       L       \n",
      " 325    0.71    0.00    0.00    0.00   D       D       D       E       K       A       D       \n",
      " 326    0.38    0.00    0.00    0.00   G       A       L       I       K       S       F       \n",
      " 327    0.78    0.00    0.00    0.00   T       N       T       T       S       T       T       \n",
      " 328    0.50    0.00    0.00    0.00   L       A       A       A       E       I       E       \n",
      " 329    0.59    0.00    0.00    0.00   A       Q       K       Q       A       A       K       \n",
      " 330    0.59    0.00    0.00    0.00   E       E       E       E       E       E       D       \n",
      " 331    0.91    0.00    0.00    0.00 * L       L       L       L       L       L       L       \n",
      " 332    0.45    0.00    0.00    0.00   L       Q       A       L       L       Q       F       \n",
      " 333    0.37    0.00    0.00    0.00   E       A       E       D       A       E       H       \n",
      " 334    0.86    0.00    0.00    0.00 * E       E       E       E       E       E       E       \n",
      " 335    0.66    0.00    0.00    0.00   R       R       K       M       K       R       K       \n",
      " 336    0.70    0.00    0.00    0.00   D       E       E       E       K       A       E       \n",
      " 337    0.30    0.00    0.00    0.00   G       A       T       Q       C       Q       I       \n",
      " 338    0.67    0.00    0.00    0.00   V       V       A       V       V       A       A       \n",
      " 339    0.44    0.00    0.00    0.00   A       R       F       K       A       G       L       \n",
      " 340    0.41    0.00    0.00    0.00   H       D       E       A       H       -       N       \n",
      " 341    0.22    0.00    0.00    0.00   -       L       F       L       L       -       H       \n",
      " 342    0.42    0.00    0.00    0.00   -       L       L       G       T       -       L       \n",
      " 343    0.35    0.00    0.00    0.00   -       L       S       L       G       -       A       \n",
      " 344    0.44    0.00    0.00    0.00   L       E       S       I       E       L       Q       \n",
      " 345    0.26    0.00    0.00    0.00   -       -       A       -       G       -       G       \n",
      " 346    0.08    0.00    0.00    0.00   -       -       -       -       N       -       G       \n",
      " 347    0.00    0.00    0.00    0.00   -       -       -       -       A       -       -       \n",
      " 348    0.00    0.00    0.00    0.00   -       -       -       -       Y       -       -       \n",
      " 349    0.00    0.00    0.00    0.00   -       -       -       -       C       -       -       \n",
      " 350    0.00    0.00    0.00    0.00   -       -       -       -       D       -       -       \n",
      " 351    0.00    0.00    0.00    0.00   -       -       -       -       V       -       -       \n",
      " 352    0.00    0.00    0.00    0.00   -       -       -       -       P       -       -       \n",
      " 353    0.00    0.00    0.00    0.00   -       -       -       -       E       -       -       \n",
      " 354    0.00    0.00    0.00    0.00   -       -       -       -       D       -       -       \n",
      " 355    0.00    0.00    0.00    0.00   -       -       -       -       T       -       -       \n",
      " 356    0.00    0.00    0.00    0.00   -       -       -       -       M       -       -       \n",
      " 357    0.00    0.00    0.00    0.00   -       -       -       -       L       -       -       \n",
      "----------------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH-mult.ali\n",
      "openf___224_> Open           TvLDH-mult.pap\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "\n",
    "log.verbose()\n",
    "env = environ()\n",
    "\n",
    "env.libs.topology.read(file='$(LIB)/top_heav.lib')\n",
    "\n",
    "# Read aligned structure(s):\n",
    "aln = alignment(env)\n",
    "aln.append(file='ldh.ali', align_codes='all') #<family name>.ali from previous script\n",
    "aln_block = len(aln)\n",
    "\n",
    "# Read aligned sequence(s):\n",
    "aln.append(file='TvLDH.ali', align_codes='TvLDH') #<query sequence>.ali and code in the 1st line of sequence\n",
    "\n",
    "\n",
    "# Structure sensitive variable gap penalty sequence-sequence alignment:\n",
    "aln.salign(output='', max_gap_length=20,\n",
    "           gap_function=True,   # to use structure-dependent gap penalty\n",
    "           alignment_type='PAIRWISE', align_block=aln_block,\n",
    "           feature_weights=(1., 0., 0., 0., 0., 0.), overhang=0,\n",
    "           gap_penalties_1d=(-450, 0),\n",
    "           gap_penalties_2d=(0.35, 1.2, 0.9, 1.2, 0.6, 8.6, 1.2, 0., 0.),\n",
    "           similarity_flag=True)\n",
    "\n",
    "aln.write(file='TvLDH-mult.ali', alignment_format='PIR') #filename could be renamed\n",
    "aln.write(file='TvLDH-mult.pap', alignment_format='PAP') #filename could be renamed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " _aln.pos         10        20        30        40        50        60\r\n",
      "3d5tA     ---------KPAKRVAVTGAAGQIAYSLLFRIANGDLLGKDQPVILQLLDLPQAQAAVKGVVMELDDC \r\n",
      "6itkA     ----------STKKVTVTGAAGQISYSLLWRIANGEVFGTDTPVELKLLEIPQALGGAEGVAMELLDS \r\n",
      "5mdhA     ---------SEPIRVLVTGAAGQIAYSLLYSIGNGSVFGKDQPIILVLLDITPMMGVLDGVLMELQDC \r\n",
      "1bdmA     --------MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGKDQPVILQLLEIPQAMKALEGVVMELEDC \r\n",
      "7mdhA     DCFGVFCTWKKLVNIAVSGAAGMISNHLLFKLASGEVFGQDQPIALKLLGSERSFQALEGVAMELEDS \r\n",
      "4h7pA     ---------MSAVKVAVTGAAGQIGYALVPLIARGALLGPTTPVELRLLDIEPALKALAGVEAELEDC \r\n",
      "TvLDH     --------MSEAAHVLITGAAGQIGYILSHWIASGELYG-DRQVYLHLLDIPPAMNRLTALTMELEDC \r\n",
      " _consrvd                   **** *   *      *   *      * **              ** *\r\n",
      "\r\n",
      " _aln.p   70        80        90       100       110       120       130\r\n",
      "3d5tA     AFPLLAGVVITDDPKVAFKDADVALLVGARPRS--MERKDLLSANAEIFTVQGAALNEVASRDVKVLV \r\n",
      "6itkA     AFPLLRNITITADANEAFDGANAAFLVGAKP-----ERADLLANNGKIFGPQGKAINDNAADDIRVLV \r\n",
      "5mdhA     ALPLLKDVIATDKEEIAFKDLDVAILVGSMPRRDGMERKDLLKANVKIFKCQGAALDKYAKKSVKVIV \r\n",
      "1bdmA     AFPLLAGLEATDDPDVAFKDADYALLVGAAPR---------LQVNGKIFTEQGRALAEVAKKDVKVLV \r\n",
      "7mdhA     LYPLLREVSIGIDPYEVFEDVDWALLIGAKPRGPGMERAALLDINGQIFADQGKALNAVASKNVKVLV \r\n",
      "4h7pA     AFPLLDKVVVTADPRVAFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVV \r\n",
      "TvLDH     AFPHLAGFVATTDPKAAFKDIDCAFLVASMPLKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLV \r\n",
      " _consrvd   * *            *     *      *             *  **   *      *     * *\r\n",
      "\r\n",
      " _aln.pos  140       150       160       170       180       190       200\r\n",
      "3d5tA     VGNPANTNAYIAMKSA-PDLPKKNFTAMLRLDHNRALSQLAAKSGKPVASIEKLAVWGNHSPTMYPDF \r\n",
      "6itkA     VGNPANTNALIASAAA-PDVPASRFNAMMRLDHNRAISQLATKLGRGSAEFNNIVVWGNHSATQFPDI \r\n",
      "5mdhA     VGNPANTNCLTASKSA-PSIPKENFSCLTRLDHNRAKAQIALKLGVTSDDVKNVIIWGNHSSTQYPDV \r\n",
      "1bdmA     VGNPANTNALIAYKNA-PGLNPRNFTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSIMFPDL \r\n",
      "7mdhA     VGNPCNTNALICLKNA-PDIPAKNFHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDF \r\n",
      "4h7pA     VGNPANTNALILLKSAQGKLNPRHVTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDT \r\n",
      "TvLDH     IGNPDNTNCEIAMLHA-KNLKPENFSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADL \r\n",
      " _consrvd  *** ***       *              ** ***    * * *           ****      *\r\n",
      "\r\n",
      " _aln.pos    210       220       230       240       250       260       270\r\n",
      "3d5tA     RFATAE----GESLLKLINDDVWNRDTFIPTVGKRGAAIIEARGLSSAASAANAAIDHVRDWVLGTNG \r\n",
      "6itkA     TYATVG----GEKVTDLV-DHDWYVEEFIPRVANRGAEIIEVRGKSSAASAASSAIDHMRDWVQGTE- \r\n",
      "5mdhA     NHAKVKLQAKEVGVYEAVKDDSWLKGEFITTVQQRGAAVIKARKLSSAMSAAKAICDHVRDIWFGTPE \r\n",
      "1bdmA     FHAEVD----GRPALELV-DMEWYEKVFIPTVAQRGAAIIQARGASSAASAANAAIEHIRDWALGTPE \r\n",
      "7mdhA     LNAKID----GRPVKEVIKRTKWLEEEFTITVQKRGGALIQKWGRSSAASTAVSIADAIKSLVTPTPE \r\n",
      "4h7pA     DSAVIG----TTPAREAIK-------DFVQVVRGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPE \r\n",
      "TvLDH     TQATFTKEGKTQKVVDVL-DHDYVFDTFFKKIGHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAP \r\n",
      " _consrvd   *                        *      *           ** *               *\r\n",
      "\r\n",
      " _aln.pos      280       290       300       310       320       330       340\r\n",
      "3d5tA     -KWVTMGIPSD-GS-YGIPEDIIYGVPVICE-NGEYKRVEGLEIDAFSREKMDGTLAELLEERDGVAH \r\n",
      "6itkA     -AWSSAAIPST-GA-YGIPEGIFVGLPTVSR-NGEWEIVEGLEISDFQRARIDANAQELQAEREAVRD \r\n",
      "5mdhA     GEFVSMGIISD-GNSYGVPDDLLYSFPVTIK-DKTWKIVEGLPINDFSREKMDLTAKELAEEKETAFE \r\n",
      "1bdmA     GDWVSMAVPSQ-GE-YGIPEGIVYSFPVTAK-DGAYRVVEGLEINEFARKRMEITAQELLDEMEQVKA \r\n",
      "7mdhA     GDWFSTGVYTT-GNPYGIAEDIVFSMPCRSKGDGDYELATDVSNDDFLWERIKKSEAELLAEKKCVAH \r\n",
      "4h7pA     GVYVSMGVYSD-ENPYGVPSGLIFSFPCTCH-AGEWTVVSGKL-----KQRLASTIAELQEERAQAG- \r\n",
      "TvLDH     GEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVVEGFKVNDWLREKLDFTEKDLFHEKEIALN \r\n",
      " _consrvd                **         *                               *  *\r\n",
      "\r\n",
      " _aln.pos        350\r\n",
      "3d5tA     ---L------------- \r\n",
      "6itkA     LLLE------------- \r\n",
      "5mdhA     FLSSA------------ \r\n",
      "1bdmA     LGLI------------- \r\n",
      "7mdhA     LTGEGNAYCDVPEDTML \r\n",
      "4h7pA     ---L------------- \r\n",
      "TvLDH     HLAQGG----------- \r\n",
      " _consrvd\r\n"
     ]
    }
   ],
   "source": [
    "#Alignment between target sequence and selected template\n",
    "!cat TvLDH-mult.pap"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Model building against multiple templates "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      3324314    3246.400     3.170\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      3324842    3246.916     3.171\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      3373442    3294.377     3.217\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      3386742    3307.365     3.230\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           TvLDH-mult.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      3398355    3318.706     3.241\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      3399805    3320.122     3.242\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      3402705    3322.954     3.245\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      3408505    3328.618     3.251\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      3533807    3450.983     3.370\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3535087    3452.233     3.371\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3536355    3453.472     3.373\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3537683    3454.769     3.374\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3538951    3456.007     3.375\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3540351    3457.374     3.376\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3541599    3458.593     3.378\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3542935    3459.897     3.379\n",
      "\n",
      "Read the alignment from file       : TvLDH-mult.ali\n",
      "\n",
      "Total number of alignment positions:   357\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      3d5tA     321      1 3d5t_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B, C, D; EC  1.1.1.37; ENGI\n",
      "  2      6itkA     318      1 6itk_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  3      5mdhA     333      1 5mdh_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  4      1bdmA     318      1 1bdm_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  5      7mdhA     351      1 7mdh_fit.pd MOL_ID  1; MOLECULE  PROTEIN (MALATE DEHYDROGENASE); CHAIN  A, B, C, D; EC  1.1.\n",
      "  6      4h7pA     313      1 4h7p_fit.pd MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  7      TvLDH     335      1       TvLDH\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           3d5t_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3640960    3555.625     3.472\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3640960    3555.625     3.472\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3641131    3555.792     3.472\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3641981    3556.622     3.473\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3643239    3557.851     3.474\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3645143    3559.710     3.476\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3645143    3559.710     3.476\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3645233    3559.798     3.476\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3648089    3562.587     3.479\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3648089    3562.587     3.479\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3648224    3562.719     3.479\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3652508    3566.902     3.483\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3652508    3566.902     3.483\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3652706    3567.096     3.483\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3659132    3573.371     3.490\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3659132    3573.371     3.490\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3659429    3573.661     3.490\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3669051    3583.058     3.499\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3669051    3583.058     3.499\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3669501    3583.497     3.500\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3683951    3597.608     3.513\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3683951    3597.608     3.513\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3684626    3598.268     3.514\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3706284    3619.418     3.535\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3706284    3619.418     3.535\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3707292    3620.402     3.536\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3739796    3652.145     3.567\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      3739652    3652.004     3.566\n",
      "openf___224_> Open           6itk_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3836777    3746.853     3.659\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3836777    3746.853     3.659\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3836948    3747.020     3.659\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3837798    3747.850     3.660\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3839056    3749.078     3.661\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3840960    3750.938     3.663\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3840960    3750.938     3.663\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3841050    3751.025     3.663\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3843906    3753.814     3.666\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3843906    3753.814     3.666\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3844041    3753.946     3.666\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3848325    3758.130     3.670\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3848325    3758.130     3.670\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3848523    3758.323     3.670\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3854949    3764.599     3.676\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3854949    3764.599     3.676\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3855246    3764.889     3.677\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3864868    3774.285     3.686\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3864868    3774.285     3.686\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3865318    3774.725     3.686\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3879768    3788.836     3.700\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3879768    3788.836     3.700\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3880443    3789.495     3.701\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3902101    3810.646     3.721\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3902101    3810.646     3.721\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3903109    3811.630     3.722\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3935613    3843.372     3.753\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      3935442    3843.205     3.753\n",
      "openf___224_> Open           5mdh_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4037067    3942.448     3.850\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4037067    3942.448     3.850\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4037238    3942.615     3.850\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4038088    3943.445     3.851\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4039346    3944.674     3.852\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4041250    3946.533     3.854\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4041250    3946.533     3.854\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4041340    3946.621     3.854\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4044196    3949.410     3.857\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4044196    3949.410     3.857\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4044331    3949.542     3.857\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4048615    3953.726     3.861\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4048615    3953.726     3.861\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4048813    3953.919     3.861\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4055239    3960.194     3.867\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4055239    3960.194     3.867\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4055536    3960.484     3.868\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4065158    3969.881     3.877\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4065158    3969.881     3.877\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4065608    3970.320     3.877\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4080058    3984.432     3.891\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4080058    3984.432     3.891\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4080733    3985.091     3.892\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4102391    4006.241     3.912\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4102391    4006.241     3.912\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4103399    4007.226     3.913\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4135903    4038.968     3.944\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      4135867    4038.933     3.944\n",
      "openf___224_> Open           1bdm_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4232992    4133.781     4.037\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4232992    4133.781     4.037\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4233163    4133.948     4.037\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4234013    4134.778     4.038\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4235271    4136.007     4.039\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4237175    4137.866     4.041\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4237175    4137.866     4.041\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4237265    4137.954     4.041\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4240121    4140.743     4.044\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4240121    4140.743     4.044\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4240256    4140.875     4.044\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4244540    4145.059     4.048\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4244540    4145.059     4.048\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4244738    4145.252     4.048\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4251164    4151.527     4.054\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4251164    4151.527     4.054\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4251461    4151.817     4.055\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4261083    4161.214     4.064\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4261083    4161.214     4.064\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4261533    4161.653     4.064\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4275983    4175.765     4.078\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4275983    4175.765     4.078\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4276658    4176.424     4.079\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4298316    4197.574     4.099\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4298316    4197.574     4.099\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4299324    4198.559     4.100\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4331828    4230.301     4.131\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      4331657    4230.134     4.131\n",
      "openf___224_> Open           7mdh_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4438682    4334.650     4.233\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4438682    4334.650     4.233\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4438853    4334.817     4.233\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4439703    4335.647     4.234\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4440961    4336.876     4.235\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4442865    4338.735     4.237\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4442865    4338.735     4.237\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4442955    4338.823     4.237\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4445811    4341.612     4.240\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4445811    4341.612     4.240\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4445946    4341.744     4.240\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4450230    4345.928     4.244\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4450230    4345.928     4.244\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4450428    4346.121     4.244\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4456854    4352.396     4.250\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4456854    4352.396     4.250\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4457151    4352.687     4.251\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4466773    4362.083     4.260\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4466773    4362.083     4.260\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4467223    4362.522     4.260\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4481673    4376.634     4.274\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4481673    4376.634     4.274\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4482348    4377.293     4.275\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4504006    4398.443     4.295\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4504006    4398.443     4.295\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4505014    4399.428     4.296\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4537518    4431.170     4.327\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4537518    4431.170     4.327\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4539030    4432.646     4.329\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      4537644    4431.293     4.327\n",
      "openf___224_> Open           4h7p_fit.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4633269    4524.677     4.419\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4633269    4524.677     4.419\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4633440    4524.844     4.419\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4634290    4525.674     4.420\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4635548    4526.902     4.421\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4637452    4528.762     4.423\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4637452    4528.762     4.423\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4637542    4528.850     4.423\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4640398    4531.639     4.425\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4640398    4531.639     4.425\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4640533    4531.771     4.426\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4644817    4535.954     4.430\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4644817    4535.954     4.430\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4645015    4536.147     4.430\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4651441    4542.423     4.436\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4651441    4542.423     4.436\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4651738    4542.713     4.436\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4661360    4552.109     4.445\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4661360    4552.109     4.445\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4661810    4552.549     4.446\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4676260    4566.660     4.460\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4676260    4566.660     4.460\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4676935    4567.319     4.460\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4698593    4588.470     4.481\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      4698593    4588.470     4.481\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4699601    4589.454     4.482\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      4732105    4621.196     4.513\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      4731889    4620.985     4.513\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "\n",
      "check_ali___> Checking pairwise structural superpositions. \n",
      "\n",
      "Equivalent CA pairs with distance difference larger than  6.0 angstroms:\n",
      "\n",
      "ALN_POS TMPL1 TMPL2  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------------\n",
      "    344     1     2  325   330      L     E    7.088\n",
      "    101     1     3   94    92      S     R    6.687\n",
      "    104     1     3   97    95      M     M    9.177\n",
      "    105     1     3   98    96      E     E    9.213\n",
      "    106     1     3   99    97      R     R    7.689\n",
      "    107     1     3  100    98      K     K    7.675\n",
      "    344     1     3  325   332      L     S    6.772\n",
      "    344     1     5  325   372      L     E    6.363\n",
      "    223     1     6  211   210      N     K    6.751\n",
      "    105     2     3  102    96      E     E    6.927\n",
      "    106     2     3  103    97      R     R    6.028\n",
      "    343     2     3  329   331      L     S    7.530\n",
      "    344     2     3  330   332      E     S   13.284\n",
      "    344     2     5  330   372      E     E    8.116\n",
      "    344     2     6  330   324      E     L    8.804\n",
      "    344     3     4  332   332      S     I   10.395\n",
      "    101     3     5   92   135      R     G    7.075\n",
      "    102     3     5   93   136      D     P    9.754\n",
      "    103     3     5   94   137      G     G    8.908\n",
      "    104     3     5   95   138      M     M    7.664\n",
      "    105     3     5   96   139      E     E    8.713\n",
      "    106     3     5   97   140      R     R    6.932\n",
      "    107     3     5   98   141      K     A    6.504\n",
      "    341     3     5  329   369      F     L    6.426\n",
      "    342     3     5  330   370      L     T    6.686\n",
      "    343     3     5  331   371      S     G    6.324\n",
      "    344     3     5  332   372      S     E   10.400\n",
      "    345     3     5  333   373      A     G   11.003\n",
      "    102     3     6   93    93      D     A    7.192\n",
      "    103     3     6   94    94      G     G    8.091\n",
      "    104     3     6   95    95      M     M    6.227\n",
      "    105     3     6   96    96      E     E    6.552\n",
      "    344     3     6  332   324      S     L   10.613\n",
      "    341     4     5  329   369      L     L    6.059\n",
      "    342     4     5  330   370      G     T    6.304\n",
      "END OF TABLE\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4871141    4756.974     4.645\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4872741    4758.536     4.647\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4876721    4762.423     4.651\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4878721    4764.376     4.653\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4881721    4767.306     4.656\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4886221    4771.700     4.660\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4891793    4777.142     4.665\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4894593    4779.876     4.668\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4898793    4783.978     4.672\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4905093    4790.130     4.678\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4907865    4792.837     4.681\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      4909265    4794.204     4.682\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      4961703    4845.413     4.732\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      5218757    5096.442     4.977\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      5258293    5135.052     5.015\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2604   2670      0         0      1136\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      5370749    5244.872     5.122\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2604   2670   3623      4313      1136\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      5481229    5352.763     5.227\n",
      "patch_______> segment topology patched using RTF:   335  ;  GLY  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2605   2671   3625      4314      1137\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2605   2671   3625      4314      1137\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        6\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     2342      263\n",
      "The following 9 residues contain 6-membered rings with poor geometries\n",
      "after transfer from templates. Rebuilding rings from internal coordinates:\n",
      "   <Residue 30 (type TYR)>\n",
      "   <Residue 84 (type PHE)>\n",
      "   <Residue 108 (type PHE)>\n",
      "   <Residue 216 (type TYR)>\n",
      "   <Residue 221 (type PHE)>\n",
      "   <Residue 257 (type PHE)>\n",
      "   <Residue 285 (type PHE)>\n",
      "   <Residue 308 (type TRP)>\n",
      "   <Residue 321 (type PHE)>\n",
      "openf___224_> Open           TvLDH.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5588544    5457.562     5.330\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5596736    5465.562     5.337\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5613120    5481.562     5.353\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5645888    5513.562     5.384\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5678656    5545.562     5.416\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5744192    5609.562     5.478\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5776960    5641.562     5.509\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5809728    5673.562     5.541\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      5875264    5737.562     5.603\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      6006336    5865.562     5.728\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      6071872    5929.562     5.791\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      6137408    5993.562     5.853\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      6268480    6121.562     5.978\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      6530624    6377.562     6.228\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      6661696    6505.562     6.353\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     2671    3625    4132    1121\n",
      "              Total number of restraints before, now         :        0    11549\n",
      "make_re_422_> Number of previous, current restraints         :        0    11549\n",
      "make_re_423_> Number of previous, current selected restraints:        0    11549\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      6852950    6692.334     6.535\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5        1  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       78  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       91  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       94  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       96  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      104  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      170  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      204  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      295  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      313  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6        1  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       78  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       91  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       94  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       96  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      104  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      170  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      204  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      295  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      313  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7        1  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       78  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       91  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       94  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       96  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      104  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      170  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      204  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      295  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      313  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8        1  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       78  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       91  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       96  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      170  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      204  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       90  ALA\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       91  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       93  LEU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        2       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        2       93  LEU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        3       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        4       91  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        4       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5        1  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      103  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      110  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      211  ASP\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      214  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      225  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      252  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      282  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      294  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      295  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      306  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6        1  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      103  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      110  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      211  ASP\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      214  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      225  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      252  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      282  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      294  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      295  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      306  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7        1  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      103  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      110  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      214  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      225  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      252  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      294  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      295  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      306  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      294  LYS\n",
      "getdata_643_> Protein accepted:  3d5tA\n",
      "getdata_643_> Protein accepted:  6itkA\n",
      "getdata_643_> Protein accepted:  5mdhA\n",
      "getdata_643_> Protein accepted:  1bdmA\n",
      "getdata_643_> Protein accepted:  7mdhA\n",
      "getdata_643_> Protein accepted:  4h7pA\n",
      "getdata_289_> Proteins (all/accepted):        6        6\n",
      "make_re_422_> Number of previous, current restraints         :    11549    11882\n",
      "make_re_423_> Number of previous, current selected restraints:    11549    11882\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      6853146    6692.525     6.536\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  3d5tA\n",
      "getdata_643_> Protein accepted:  6itkA\n",
      "getdata_643_> Protein accepted:  5mdhA\n",
      "getdata_643_> Protein accepted:  1bdmA\n",
      "getdata_643_> Protein accepted:  7mdhA\n",
      "getdata_643_> Protein accepted:  4h7pA\n",
      "getdata_289_> Proteins (all/accepted):        6        6\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      350\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :    11882    12216\n",
      "make_re_423_> Number of previous, current selected restraints:    11882    11866\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      6853146    6692.525     6.536\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1.mdt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "getdata_643_> Protein accepted:  3d5tA\n",
      "getdata_643_> Protein accepted:  6itkA\n",
      "getdata_643_> Protein accepted:  5mdhA\n",
      "getdata_643_> Protein accepted:  1bdmA\n",
      "getdata_643_> Protein accepted:  7mdhA\n",
      "getdata_643_> Protein accepted:  4h7pA\n",
      "getdata_289_> Proteins (all/accepted):        6        6\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      6984218    6820.525     6.661\n",
      "make_re_422_> Number of previous, current restraints         :    12216    12495\n",
      "make_re_423_> Number of previous, current selected restraints:    11866    12145\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      6984218    6820.525     6.661\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  3d5tA\n",
      "getdata_643_> Protein accepted:  6itkA\n",
      "getdata_643_> Protein accepted:  5mdhA\n",
      "getdata_643_> Protein accepted:  1bdmA\n",
      "getdata_643_> Protein accepted:  7mdhA\n",
      "getdata_643_> Protein accepted:  4h7pA\n",
      "getdata_289_> Proteins (all/accepted):        6        6\n",
      "make_re_422_> Number of previous, current restraints         :    12495    12709\n",
      "make_re_423_> Number of previous, current selected restraints:    12145    12359\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      6984218    6820.525     6.661\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  3d5tA\n",
      "getdata_643_> Protein accepted:  6itkA\n",
      "getdata_643_> Protein accepted:  5mdhA\n",
      "getdata_643_> Protein accepted:  1bdmA\n",
      "getdata_643_> Protein accepted:  7mdhA\n",
      "getdata_643_> Protein accepted:  4h7pA\n",
      "getdata_289_> Proteins (all/accepted):        6        6\n",
      "make_re_422_> Number of previous, current restraints         :    12709    12793\n",
      "make_re_423_> Number of previous, current selected restraints:    12359    12443\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      6984218    6820.525     6.661\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  3d5tA\n",
      "getdata_643_> Protein accepted:  6itkA\n",
      "getdata_643_> Protein accepted:  5mdhA\n",
      "getdata_643_> Protein accepted:  1bdmA\n",
      "getdata_643_> Protein accepted:  7mdhA\n",
      "getdata_643_> Protein accepted:  4h7pA\n",
      "getdata_289_> Proteins (all/accepted):        6        6\n",
      "make_re_422_> Number of previous, current restraints         :    12793    12825\n",
      "make_re_423_> Number of previous, current selected restraints:    12443    12475\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:      7734422    7553.146     7.376\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      7996566    7809.146     7.626\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8520854    8321.146     8.126\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8782998    8577.146     8.376\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      9307286    9089.146     8.876\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     10355862   10113.146     9.876\n",
      "make_re_422_> Number of previous, current restraints         :    12825    21110\n",
      "make_re_423_> Number of previous, current selected restraints:    12475    20760\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     10618006   10369.146    10.126\n",
      "make_re_422_> Number of previous, current restraints         :    21110    29838\n",
      "make_re_423_> Number of previous, current selected restraints:    20760    29488\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     12715158   12417.146    12.126\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     13239446   12929.146    12.626\n",
      "make_re_422_> Number of previous, current restraints         :    29838    34821\n",
      "make_re_423_> Number of previous, current selected restraints:    29488    34471\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14288022   13953.146    13.626\n",
      "make_re_422_> Number of previous, current restraints         :    34821    37377\n",
      "make_re_423_> Number of previous, current selected restraints:    34471    37027\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37377    37377\n",
      "make_re_423_> Number of previous, current selected restraints:    37027    37027\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37377    37377\n",
      "make_re_423_> Number of previous, current selected restraints:    37027    37027\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37377    37377\n",
      "make_re_423_> Number of previous, current selected restraints:    37027    37027\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37377    37377\n",
      "make_re_423_> Number of previous, current selected restraints:    37027    37027\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    37377    37377\n",
      "make_re_423_> Number of previous, current selected restraints:    37027    37027\n",
      "rmdupl__427_>     2039 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37377    34988\n",
      "openf___224_> Open           TvLDH.rsr\n",
      "openf___224_> Open           TvLDH.rsr\n",
      "rdcsr2__307_> Number of restraints read    :    34988\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    34988    34988\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           TvLDH.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     12759947   12460.886    12.169\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16617\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18806\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    20098\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21731\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    23388\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25723\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    29083\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    30473\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    32123\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     13115047   12807.663    12.507\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34005\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "iupac_m_485_> OD1/2 will be swapped:      156.8951       32    32\n",
      "iupac_m_487_> NH1/2 swapped:      178.8245       33    33\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.1997       36    36\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4397       48    48\n",
      "iupac_m_486_> OE1/2 will be swapped:      120.9298       55    55\n",
      "iupac_m_485_> OD1/2 will be swapped:      159.0310       79    79\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.1013      108   108\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -108.6035      113   113\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      115.7904      114   114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      174.1182      132   132\n",
      "iupac_m_486_> OE1/2 will be swapped:      117.7962      137   137\n",
      "iupac_m_486_> OE1/2 will be swapped:      156.3520      165   165\n",
      "iupac_m_486_> OE1/2 will be swapped:      107.3149      188   188\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       99.8562      199   199\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -163.6162      202   202\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.6391      222   222\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -170.3542      228   228\n",
      "iupac_m_485_> OD1/2 will be swapped:      156.7059      231   231\n",
      "iupac_m_486_> OE1/2 will be swapped:      149.8977      234   234\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.2372      273   273\n",
      "iupac_m_485_> OD1/2 will be swapped:      157.8017      292   292\n",
      "iupac_m_486_> OE1/2 will be swapped:       95.1779      294   294\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.3072      303   303\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.7376      310   310\n",
      "iupac_m_485_> OD1/2 will be swapped:      123.7799      314   314\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.2005      315   315\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34988   34988\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5699\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :       12252.2324\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      19.042       1.000\n",
      " 2 Bond angle potential               :    3625       0      4   2.014   2.014      287.61       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     37  45.389  45.389      554.55       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      0   1.216   1.216      35.112       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5699       0      0   0.002   0.002      3.3817       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    8285      95      6   0.687   0.687      2713.3       1.000\n",
      "10 Distance restraints 2 (N-O)        :    8728     115     22   0.788   0.788      3038.4       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       0      0   4.097   4.097      64.686       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      3  64.165  64.165      58.626       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      1  67.800  67.800      72.657       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      0  72.363  72.363      52.326       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  77.290  77.290      21.511       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    4983       0      0   0.401   0.401      3422.0       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      14     25  22.385  38.168     -26.674       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2556       4      4   0.937   0.937      1935.8       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990001\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   36976.4531\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13163  92K 231D CA  CA    690 1790   17.91   10.83    7.08   10.11   10.83    7.08   10.11\n",
      "    2  13183  93P 229A CA  CA    699 1771   16.33    9.93    6.41   11.31    9.93    6.41   11.31\n",
      "    3  13184  93P 230W CA  CA    699 1776   14.26    6.84    7.43   14.52    6.84    7.43   14.52\n",
      "    4  13185  93P 231D CA  CA    699 1790   17.47    8.59    8.88   14.94    8.59    8.88   14.94\n",
      "    5  13203  94G 187G CA  CA    706 1433   18.14   13.68    4.45    6.22   13.68    4.45    6.22\n",
      "    6  13204  94G 188E CA  CA    706 1437   18.39   12.73    5.66    8.06   12.73    5.66    8.06\n",
      "    7  13205  94G 227H CA  CA    706 1750   19.29   11.36    7.94   10.92   11.36    7.94   10.92\n",
      "    8  13206  94G 228R CA  CA    706 1760   21.33   13.09    8.24   11.13   13.09    8.24   11.13\n",
      "    9  13207  94G 229A CA  CA    706 1771   18.72   11.28    7.44   11.63   11.28    7.44   11.63\n",
      "   10  13208  94G 230W CA  CA    706 1776   16.63    7.81    8.82   15.44    7.81    8.82   15.44\n",
      "   11  13209  94G 231D CA  CA    706 1790   19.96   10.08    9.88   14.33   10.08    9.88   14.33\n",
      "   12  13210  94G 232I CA  CA    706 1798   20.63   11.87    8.76   12.95   11.87    8.76   12.95\n",
      "   13  13211  94G 233L CA  CA    706 1806   17.53    9.21    8.32   14.37    9.21    8.32   14.37\n",
      "   14  13212  94G 234E CA  CA    706 1814   18.70    8.27   10.43   16.78    8.27   10.43   16.78\n",
      "   15  13214  94G 236R CA  CA    706 1833   21.89   13.26    8.62   11.44   13.26    8.62   11.44\n",
      "   16  13215  94G 237G CA  CA    706 1844   18.68   10.86    7.82   10.68   10.86    7.82   10.68\n",
      "   17  13216  94G 238F CA  CA    706 1848   18.22   12.28    5.94    7.90   12.28    5.94    7.90\n",
      "   18  13217  94G 239T CA  CA    706 1859   18.01   12.15    5.87    8.59   12.15    5.87    8.59\n",
      "   19  13228  95Q 187G CA  CA    710 1433   15.72   11.42    4.30    7.95   11.42    4.30    7.95\n",
      "   20  13229  95Q 188E CA  CA    710 1437   16.58   11.24    5.34    9.47   11.24    5.34    9.47\n",
      "   21  13230  95Q 226G CA  CA    710 1746   19.82   13.81    6.02   10.74   13.81    6.02   10.74\n",
      "   22  13233  95Q 229A CA  CA    710 1771   17.74   10.81    6.92   13.44   10.81    6.92   13.44\n",
      "   23  13235  95Q 231D CA  CA    710 1790   19.95   11.57    8.38   13.62   11.57    8.38   13.62\n",
      "   24  13237  95Q 233L CA  CA    710 1806   17.63   10.12    7.51   15.07   10.12    7.51   15.07\n",
      "   25  13239  95Q 237G CA  CA    710 1844   19.31   12.31    7.01   10.66   12.31    7.01   10.66\n",
      "   26  13334  98A 130N CA  CA    737  981   11.88    7.85    4.04    7.52    7.85    4.04    7.52\n",
      "   27  13336  98A 132D CA  CA    737  996   13.63    9.85    3.77    6.87    9.85    3.77    6.87\n",
      "   28  13337  98A 133N CA  CA    737 1004   13.83   10.65    3.18    5.86   10.65    3.18    5.86\n",
      "   29  13341  98A 185N CA  CA    737 1415   14.95   10.87    4.08    7.56   10.87    4.08    7.56\n",
      "   30  13342  98A 186H CA  CA    737 1423   15.15   10.26    4.89    9.48   10.26    4.89    9.48\n",
      "   31  13343  98A 187G CA  CA    737 1433   13.62    8.20    5.43   11.15    8.20    5.43   11.15\n",
      "   32  13344  98A 188E CA  CA    737 1437   15.31    9.91    5.40    9.61    9.91    5.40    9.61\n",
      "   33  13345  98A 189S CA  CA    737 1446   15.71   11.51    4.20    7.10   11.51    4.20    7.10\n",
      "   34  13346  98A 190M CA  CA    737 1452   17.97   13.24    4.74    8.68   13.24    4.74    8.68\n",
      "   35  13347  98A 226G CA  CA    737 1746   19.62   14.00    5.62    9.57   14.00    5.62    9.57\n",
      "   36  13349  98A 229A CA  CA    737 1771   19.72   13.20    6.52   11.32   13.20    6.52   11.32\n",
      "   37  13351  98A 316T CA  CA    737 2448   15.99   12.90    3.09    5.67   12.90    3.09    5.67\n",
      "   38  13523 103S 334G CA  CA    772 2598   17.90   13.14    4.75    5.85   13.14    4.75    5.85\n",
      "   39  13628 106V 330H CA  CA    792 2566    9.42    6.79    2.62    4.67    6.79    2.62    4.67\n",
      "   40  13632 106V 334G CA  CA    792 2598   15.81   10.50    5.31    7.33   10.50    5.31    7.33\n",
      "   41  13660 107I 330H CA  CA    799 2566   12.99    9.90    3.08    5.10    9.90    3.08    5.10\n",
      "   42  13664 107I 334G CA  CA    799 2598   19.05   13.98    5.07    6.17   13.98    5.07    6.17\n",
      "   43  13703 108F 330H CA  CA    807 2566   13.12    9.20    3.92    7.11    9.20    3.92    7.11\n",
      "   44  13740 109K 329N CA  CA    818 2558   13.75   10.46    3.29    4.79   10.46    3.29    4.79\n",
      "   45  13741 109K 330H CA  CA    818 2566   11.66    6.72    4.94   11.85    6.72    4.94   11.85\n",
      "   46  13742 109K 331L CA  CA    818 2576    8.80    6.85    1.95    4.90    6.85    1.95    4.90\n",
      "   47  13770 110N 330H CA  CA    827 2566   14.33    9.72    4.60    7.41    9.72    4.60    7.41\n",
      "   48  13774 110N 334G CA  CA    827 2598   18.66   13.78    4.89    5.72   13.78    4.89    5.72\n",
      "   49  13806 111T 330H CA  CA    835 2566   16.88   12.19    4.69    8.55   12.19    4.69    8.55\n",
      "   50  13809 111T 333Q CA  CA    835 2589   17.86   13.69    4.17    4.77   13.69    4.17    4.77\n",
      "   51  13848 112G 330H CA  CA    842 2566   16.42   11.12    5.30    8.31   11.12    5.30    8.31\n",
      "   52  13850 112G 332A CA  CA    842 2584   13.22    9.25    3.97    9.68    9.25    3.97    9.68\n",
      "   53  13851 112G 333Q CA  CA    842 2589   15.78   10.97    4.81    9.24   10.97    4.81    9.24\n",
      "   54  13958 116S 332A CA  CA    875 2584   16.56   11.97    4.59    8.27   11.97    4.59    8.27\n",
      "   55  14531 133N 330H CA  CA   1004 2566   16.75   14.08    2.68    4.94   14.08    2.68    4.94\n",
      "   56  14579 134T 330H CA  CA   1012 2566   12.97   10.46    2.51    5.28   10.46    2.51    5.28\n",
      "   57  14620 135N 330H CA  CA   1019 2566   13.35   10.09    3.26    6.24   10.09    3.26    6.24\n",
      "   58  14662 136C 330H CA  CA   1027 2566   15.89   11.77    4.11    7.00   11.77    4.11    7.00\n",
      "   59  14663 136C 331L CA  CA   1027 2576   13.99   10.83    3.16    6.07   10.83    3.16    6.07\n",
      "   60  14710 137E 330H CA  CA   1033 2566   13.83    9.82    4.01    7.68    9.82    4.01    7.68\n",
      "   61  14711 137E 331L CA  CA   1033 2576   11.77    7.96    3.81    8.00    7.96    3.81    8.00\n",
      "   62  14746 138I 330H CA  CA   1042 2566   11.24    6.47    4.77    9.77    6.47    4.77    9.77\n",
      "   63  14747 138I 331L CA  CA   1042 2576    8.78    5.50    3.29    8.61    5.50    3.29    8.61\n",
      "   64  14776 139A 330H CA  CA   1050 2566   14.35    9.20    5.15    8.68    9.20    5.15    8.68\n",
      "   65  14777 139A 331L CA  CA   1050 2576   11.56    8.20    3.36    7.02    8.20    3.36    7.02\n",
      "   66  14806 140M 331L CA  CA   1055 2576   13.21    8.72    4.49    9.84    8.72    4.49    9.84\n",
      "   67  14807 140M 332A CA  CA   1055 2584   12.34    9.41    2.93    5.98    9.41    2.93    5.98\n",
      "   68  14808 140M 333Q CA  CA   1055 2589   13.46    9.31    4.15    6.94    9.31    4.15    6.94\n",
      "   69  14834 141L 330H CA  CA   1063 2566   13.97    8.57    5.40    7.66    8.57    5.40    7.66\n",
      "   70  14835 141L 331L CA  CA   1063 2576   10.67    5.55    5.12   12.24    5.55    5.12   12.24\n",
      "   71  14836 141L 332A CA  CA   1063 2584    9.05    6.24    2.81    5.49    6.24    2.81    5.49\n",
      "   72  14837 141L 333Q CA  CA   1063 2589   10.14    5.81    4.33    9.50    5.81    4.33    9.50\n",
      "   73  14855 142H 330H CA  CA   1071 2566   14.00    8.60    5.40    5.66    8.60    5.40    5.66\n",
      "   74  14856 142H 331L CA  CA   1071 2576   10.42    6.72    3.70    7.73    6.72    3.70    7.73\n",
      "   75  14857 142H 332A CA  CA   1071 2584    9.13    5.13    4.01   10.11    5.13    4.01   10.11\n",
      "   76  14858 142H 333Q CA  CA   1071 2589   11.47    5.92    5.55    8.94    5.92    5.55    8.94\n",
      "   77  14869 143A 330H CA  CA   1081 2566   17.72   12.09    5.63    5.28   12.09    5.63    5.28\n",
      "   78  14870 143A 331L CA  CA   1081 2576   14.15   10.19    3.96    8.40   10.19    3.96    8.40\n",
      "   79  14871 143A 332A CA  CA   1081 2584   12.64    8.67    3.97    7.96    8.67    3.97    7.96\n",
      "   80  14872 143A 333Q CA  CA   1081 2589   14.68    8.05    6.63   13.35    8.05    6.63   13.35\n",
      "   81  14881 144K 330H CA  CA   1086 2566   19.38   13.91    5.47    5.63   13.91    5.47    5.63\n",
      "   82  14882 144K 331L CA  CA   1086 2576   15.69   10.99    4.70    7.93   10.99    4.70    7.93\n",
      "   83  14883 144K 332A CA  CA   1086 2584   13.35    9.47    3.88    6.47    9.47    3.88    6.47\n",
      "   84  14941 148P 331L CA  CA   1120 2576   18.26   13.02    5.24    8.04   13.02    5.24    8.04\n",
      "   85  14942 148P 333Q CA  CA   1120 2589   17.06   13.03    4.04    5.29   13.03    4.04    5.29\n",
      "   86  18000 270P 331L CA  CA   2080 2576   18.95   13.86    5.09    4.60   13.86    5.09    4.60\n",
      "   87  18288 282G 331L CA  CA   2169 2576   15.65   10.82    4.82    6.34   10.82    4.82    6.34\n",
      "   88  18311 283V 331L CA  CA   2173 2576   15.65   10.73    4.93    8.78   10.73    4.93    8.78\n",
      "   89  18335 284V 331L CA  CA   2180 2576   17.09   12.75    4.33    5.36   12.75    4.33    5.36\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21010  93P 229A N   O     698 1774   15.23    8.04    7.19   12.66    8.04    7.19   12.66\n",
      "    2  21011  93P 230W N   O     698 1788   14.94    6.68    8.26   14.30    6.68    8.26   14.30\n",
      "    3  21012  93P 231D N   O     698 1796   17.73    9.26    8.48   12.31    9.26    8.48   12.31\n",
      "    4  21026  94G 227H N   O     705 1758   18.34    9.79    8.55   11.23    9.79    8.55   11.23\n",
      "    5  21027  94G 228R N   O     705 1769   19.33   10.90    8.43   11.39   10.90    8.43   11.39\n",
      "    6  21028  94G 229A N   O     705 1774   16.79    8.66    8.13   13.06    8.66    8.13   13.06\n",
      "    7  21029  94G 230W N   O     705 1788   16.18    6.53    9.64   15.68    6.53    9.64   15.68\n",
      "    8  21030  94G 231D N   O     705 1796   19.16    9.76    9.41   12.43    9.76    9.41   12.43\n",
      "    9  21031  94G 232I N   O     705 1804   18.84   10.18    8.65   12.68   10.18    8.65   12.68\n",
      "   10  21032  94G 233L N   O     705 1812   16.69    7.51    9.18   15.43    7.51    9.18   15.43\n",
      "   11  21034  94G 237G N   O     705 1846   15.81   10.03    5.78    7.37   10.03    5.78    7.37\n",
      "   12  21035  94G 238F N   O     705 1857   17.50   10.42    7.08    9.13   10.42    7.08    9.13\n",
      "   13  21046  95Q 229A N   O     709 1774   17.16    9.38    7.78   15.22    9.38    7.78   15.22\n",
      "   14  21048  95Q 233L N   O     709 1812   17.89    9.47    8.42   16.86    9.47    8.42   16.86\n",
      "   15  21108  98A 185N N   O     736 1421   15.24   10.36    4.87    9.44   10.36    4.87    9.44\n",
      "   16  21109  98A 186H N   O     736 1431   12.37    7.08    5.29   10.71    7.08    5.29   10.71\n",
      "   17  21110  98A 187G N   O     736 1435   14.76    9.30    5.46   10.62    9.30    5.46   10.62\n",
      "   18  21111  98A 188E N   O     736 1444   15.97   10.81    5.16    8.52   10.81    5.16    8.52\n",
      "   19  21112  98A 226G N   O     736 1748   16.94   10.95    5.99   10.61   10.95    5.99   10.61\n",
      "   20  21433 109K 329N N   O     817 2564   13.43    9.50    3.93    5.39    9.50    3.93    5.39\n",
      "   21  21434 109K 330H N   O     817 2574   10.97    5.50    5.47    8.64    5.50    5.47    8.64\n",
      "   22  21460 110N 329N N   O     826 2564   14.79   10.08    4.70    5.21   10.08    4.70    5.21\n",
      "   23  21461 110N 330H N   O     826 2574   11.98    7.33    4.65    5.39    7.33    4.65    5.39\n",
      "   24  21528 112G 330H N   O     841 2574   15.20    9.21    5.99   11.92    9.21    5.99   11.92\n",
      "   25  22272 135N 330H N   O    1018 2574   14.14    9.70    4.44    8.50    9.70    4.44    8.50\n",
      "   26  22318 136C 330H N   O    1026 2574   15.57   10.12    5.46   10.04   10.12    5.46   10.04\n",
      "   27  22363 137E 330H N   O    1032 2574   14.86    9.17    5.69   10.07    9.17    5.69   10.07\n",
      "   28  22364 137E 331L N   O    1032 2582   14.25   11.46    2.79    5.18   11.46    2.79    5.18\n",
      "   29  22406 138I 330H N   O    1041 2574   12.31    6.29    6.02   12.04    6.29    6.02   12.04\n",
      "   30  22452 139A 330H N   O    1049 2574   13.13    6.50    6.63   13.23    6.50    6.63   13.23\n",
      "   31  22453 139A 331L N   O    1049 2582   12.48    9.04    3.44    5.42    9.04    3.44    5.42\n",
      "   32  22455 139A 333Q N   O    1049 2596   13.63    9.79    3.85    6.43    9.79    3.85    6.43\n",
      "   33  22496 140M 330H N   O    1054 2574   15.07    8.16    6.91   11.36    8.16    6.91   11.36\n",
      "   34  22497 140M 331L N   O    1054 2582   13.93    9.94    3.99    5.80    9.94    3.99    5.80\n",
      "   35  22498 140M 332A N   O    1054 2587   13.63   10.79    2.84    5.35   10.79    2.84    5.35\n",
      "   36  22499 140M 333Q N   O    1054 2596   14.70   10.90    3.79    6.51   10.90    3.79    6.51\n",
      "   37  22530 141L 330H N   O    1062 2574   14.01    6.86    7.15    7.44    6.86    7.15    7.44\n",
      "   38  22531 141L 331L N   O    1062 2582   12.36    7.93    4.43    8.66    7.93    4.43    8.66\n",
      "   39  22533 141L 333Q N   O    1062 2596   12.84    8.93    3.91    6.20    8.93    3.91    6.20\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   40  22563 142H 329N N   O    1070 2564   14.33   10.02    4.31    6.15   10.02    4.31    6.15\n",
      "   41  22564 142H 330H N   O    1070 2574   12.94    6.05    6.89    6.65    6.05    6.89    6.65\n",
      "   42  22565 142H 331L N   O    1070 2582   11.29    6.65    4.63    8.25    6.65    4.63    8.25\n",
      "   43  22566 142H 332A N   O    1070 2587   10.61    6.64    3.97    7.25    6.64    3.97    7.25\n",
      "   44  22567 142H 333Q N   O    1070 2596   12.63    5.78    6.85   13.92    5.78    6.85   13.92\n",
      "   45  22598 143A 330H N   O    1080 2574   15.38    8.37    7.02    7.04    8.37    7.02    7.04\n",
      "   46  22599 143A 331L N   O    1080 2582   13.72    8.95    4.78    7.90    8.95    4.78    7.90\n",
      "   47  22600 143A 332A N   O    1080 2587   12.91    8.38    4.53    8.55    8.38    4.53    8.55\n",
      "   48  22601 143A 333Q N   O    1080 2596   15.21    7.49    7.72   13.51    7.49    7.72   13.51\n",
      "   49  22626 144K 330H N   O    1085 2574   17.34   10.88    6.45    4.86   10.88    6.45    4.86\n",
      "   50  22627 144K 331L N   O    1085 2582   15.29   10.20    5.09    5.53   10.20    5.09    5.53\n",
      "   51  22628 144K 332A N   O    1085 2587   13.84    8.86    4.97    7.40    8.86    4.97    7.40\n",
      "   52  22629 144K 333Q N   O    1085 2596   16.61    7.86    8.74   13.37    7.86    8.74   13.37\n",
      "   53  23815 184G 275N N   O    1410 2120   13.03   10.28    2.75    5.50   10.28    2.75    5.50\n",
      "   54  23911 187G  95Q N   O    1432  717   14.79   10.65    4.14    7.75   10.65    4.14    7.75\n",
      "   55  23946 188E  95Q N   O    1436  717   13.55    8.76    4.79    8.81    8.76    4.79    8.81\n",
      "   56  24848 229A  95Q N   O    1770  717   16.54    9.29    7.25   14.18    9.29    7.25   14.18\n",
      "   57  24874 230W  94G N   O    1775  708   18.45    9.83    8.62   12.42    9.83    8.62   12.42\n",
      "   58  24896 231D  93P N   O    1789  704   18.77   10.06    8.71   12.43   10.06    8.71   12.43\n",
      "   59  24897 231D  94G N   O    1789  708   20.22   10.64    9.59   12.32   10.64    9.59   12.32\n",
      "   60  24898 231D  95Q N   O    1789  717   16.86    8.60    8.27   13.93    8.60    8.27   13.93\n",
      "   61  24913 232I  91L N   O    1797  688   15.26   10.23    5.03    9.91   10.23    5.03    9.91\n",
      "   62  24915 232I  95Q N   O    1797  717   18.31   10.29    8.02   14.60   10.29    8.02   14.60\n",
      "   63  24934 233L  95Q N   O    1805  717   16.83    9.35    7.48   14.98    9.35    7.48   14.98\n",
      "   64  26193 283V 331L N   O    2172 2582   16.83   10.94    5.88   10.64   10.94    5.88   10.64\n",
      "   65  26827 309L 277Y N   O    2383 2139   13.70   10.59    3.12    4.63   10.59    3.12    4.63\n",
      "   66  26843 310R 277Y N   O    2391 2139   11.25    8.10    3.15    4.65    8.10    3.15    4.65\n",
      "   67  26904 313L 275N N   O    2420 2120   13.64   10.67    2.96    5.33   10.67    2.96    5.33\n",
      "   68  26997 317E 275N N   O    2454 2120   13.46   10.14    3.32    5.43   10.14    3.32    5.43\n",
      "   69  27313 330H 105S N   O    2565  790    9.94    7.23    2.71    5.66    7.23    2.71    5.66\n",
      "   70  27315 330H 108F N   O    2565  816   14.14   10.34    3.80    7.03   10.34    3.80    7.03\n",
      "   71  27316 330H 109K N   O    2565  825   14.25    9.87    4.38    6.81    9.87    4.38    6.81\n",
      "   72  27317 330H 134T N   O    2565 1017   11.72    9.32    2.40    4.58    9.32    2.40    4.58\n",
      "   73  27318 330H 135N N   O    2565 1025   13.71   10.77    2.93    5.68   10.77    2.93    5.68\n",
      "   74  27319 330H 137E N   O    2565 1040   12.84    9.36    3.48    6.49    9.36    3.48    6.49\n",
      "   75  27320 330H 138I N   O    2565 1048   12.18    7.93    4.25    7.77    7.93    4.25    7.77\n",
      "   76  27321 330H 139A N   O    2565 1053   15.52   11.13    4.39    6.90   11.13    4.39    6.90\n",
      "   77  27322 330H 141L N   O    2565 1069   14.40    9.47    4.93    7.74    9.47    4.93    7.74\n",
      "   78  27343 331L 109K N   O    2575  825   11.70    8.77    2.94    5.57    8.77    2.94    5.57\n",
      "   79  27344 331L 134T N   O    2575 1017   10.80    8.32    2.49    5.69    8.32    2.49    5.69\n",
      "   80  27345 331L 135N N   O    2575 1025   12.25    9.46    2.79    6.07    9.46    2.79    6.07\n",
      "   81  27346 331L 136C N   O    2575 1031   13.63    9.92    3.71    7.53    9.92    3.71    7.53\n",
      "   82  27347 331L 137E N   O    2575 1040   11.22    7.20    4.03    8.49    7.20    4.03    8.49\n",
      "   83  27348 331L 138I N   O    2575 1048   10.06    6.03    4.02    9.06    6.03    4.02    9.06\n",
      "   84  27349 331L 139A N   O    2575 1053   13.30    9.13    4.16    8.88    9.13    4.16    8.88\n",
      "   85  27350 331L 140M N   O    2575 1061   14.17    9.52    4.65    8.92    9.52    4.65    8.92\n",
      "   86  27351 331L 141L N   O    2575 1069   12.10    7.04    5.05   10.17    7.04    5.05   10.17\n",
      "   87  27352 331L 142H N   O    2575 1079   13.62    9.85    3.77    7.62    9.85    3.77    7.62\n",
      "   88  27353 331L 282G N   O    2575 2171   14.95   10.10    4.85    6.04   10.10    4.85    6.04\n",
      "   89  27381 332A 140M N   O    2583 1061   11.85    7.81    4.04    7.66    7.81    4.04    7.66\n",
      "   90  27382 332A 141L N   O    2583 1069    9.27    4.61    4.66   10.04    4.61    4.66   10.04\n",
      "   91  27383 332A 142H N   O    2583 1079   11.09    7.04    4.05    8.00    7.04    4.05    8.00\n",
      "   92  27384 332A 143A N   O    2583 1084   14.10   10.07    4.03    7.22   10.07    4.03    7.22\n",
      "   93  27411 333Q 140M N   O    2588 1061   12.20    7.32    4.89    8.53    7.32    4.89    8.53\n",
      "   94  27412 333Q 141L N   O    2588 1069    9.67    3.51    6.17   11.71    3.51    6.17   11.71\n",
      "   95  27414 333Q 143A N   O    2588 1084   14.57    8.81    5.76    9.77    8.81    5.76    9.77\n",
      "   96  27430 334G 102S N   O    2597  770   15.05   10.61    4.43    5.14   10.61    4.43    5.14\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9191  31G  32D C   N     225  227  -83.42  -96.50   71.33    3.00  -63.30   86.45    9.63\n",
      "    1         32D  32D N   CA    227  228   44.08  114.20                  -40.00\n",
      "    2   9303 143A 144K C   N    1083 1085  -72.59  -70.20   10.80    0.73  -62.90  168.55   22.43\n",
      "    2        144K 144K N   CA   1085 1086  150.93  140.40                  -40.80\n",
      "    3   9304 144K 145N C   N    1092 1094   83.85   55.90   48.00    2.44  -63.20  152.81   24.63\n",
      "    3        145N 145N N   CA   1094 1095    0.48   39.50                  -41.10\n",
      "    4   9386 226G 227H C   N    1747 1749 -104.70  -63.20   79.16    8.43  -63.20   79.16    8.43\n",
      "    4        227H 227H N   CA   1749 1750   25.10  -42.30                  -42.30\n",
      "    5   9431 271V 272P C   N    2091 2093  -58.22  -64.50    6.41    0.60  -58.70  176.37   14.61\n",
      "    5        272P 272P N   CA   2093 2094  145.87  147.20                  -30.50\n",
      "    6   9432 272P 273E C   N    2098 2100  135.20  -63.60  163.87   24.51  -63.60  163.87   24.51\n",
      "    6        273E 273E N   CA   2100 2101  -10.90  -40.30                  -40.30\n",
      "    7   9436 276P 277Y C   N    2126 2128 -102.50  -98.40   27.79    2.95  -63.50  165.38   28.24\n",
      "    7        277Y 277Y N   CA   2128 2129  155.88  128.40                  -43.40\n",
      "    8   9437 277Y 278G C   N    2138 2140  -54.56  -62.40   22.70    3.81   82.20  139.68   10.26\n",
      "    8        278G 278G N   CA   2140 2141  -19.90  -41.20                    8.50\n",
      "    9   9440 280K 281P C   N    2159 2161  -60.99  -58.70   21.54    1.97  -64.50  160.92   12.19\n",
      "    9        281P 281P N   CA   2161 2162  -51.92  -30.50                  147.20\n",
      "   10   9441 281P 282G C   N    2166 2168  -64.17  -62.40   16.27    2.35   82.20  150.16   11.10\n",
      "   10        282G 282G N   CA   2168 2169  -25.03  -41.20                    8.50\n",
      "   11   9452 292D 293K C   N    2248 2250  -63.52  -62.90   20.83    2.67   56.60  133.64   17.12\n",
      "   11        293K 293K N   CA   2250 2251  -19.98  -40.80                   38.60\n",
      "   12   9453 293K 294E C   N    2257 2259  -52.77  -63.60   15.87    2.68   54.60  128.78   16.82\n",
      "   12        294E 294E N   CA   2259 2260  -28.70  -40.30                   42.40\n",
      "   13   9490 330H 331L C   N    2573 2575  -91.15  -70.70   87.23    5.88  -63.50   96.45   15.10\n",
      "   13        331L 331L N   CA   2575 2576 -133.60  141.60                  -41.20\n",
      "   14   9492 332A 333Q C   N    2586 2588  -89.05  -73.00   19.11    1.49  -63.80  172.49   24.10\n",
      "   14        333Q 333Q N   CA   2588 2589  130.32  140.70                  -40.30\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33726 144K 333Q CB  CG   1087 2591   13.35    4.50    8.85    5.50    4.50    8.85    5.50\n",
      "    2  33727 144K 333Q CB  CD   1087 2592   12.89    4.80    8.09    4.97    4.80    8.09    4.97\n",
      "    3  34969 324K 331L CE  CD1  2523 2579   12.49    4.30    8.19    5.85    4.30    8.19    5.85\n",
      "    4  34972 324K 331L NZ  CD1  2524 2579   13.21    4.10    9.11    6.59    4.10    9.11    6.59\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   18   21   92  193  268  225  327  402  440  442\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "openf___224_> Open           TvLDH.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13115071   12807.687    12.508\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16617\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18806\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    20098\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21731\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    23388\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25723\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    29083\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    30473\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    32123\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34005\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "iupac_m_486_> OE1/2 will be swapped:     -177.2651        3     3\n",
      "iupac_m_487_> NH1/2 swapped:      176.3733       33    33\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.7864       36    36\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      179.6088       55    55\n",
      "iupac_m_486_> OE1/2 will be swapped:     -100.9899       57    57\n",
      "iupac_m_485_> OD1/2 will be swapped:      172.2735       79    79\n",
      "iupac_m_487_> NH1/2 swapped:      177.7083       97    97\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      108.1466      114   114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -121.1002      149   149\n",
      "iupac_m_485_> OD1/2 will be swapped:      137.5837      173   173\n",
      "iupac_m_485_> OD1/2 will be swapped:      173.9164      179   179\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      102.1719      199   199\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      119.3117      202   202\n",
      "iupac_m_485_> OD1/2 will be swapped:     -125.7391      210   210\n",
      "iupac_m_486_> OE1/2 will be swapped:     -132.6771      234   234\n",
      "iupac_m_485_> OD1/2 will be swapped:      119.9623      292   292\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.1525      311   311\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34988   34988\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5737\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :       12125.7930\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      17.384       1.000\n",
      " 2 Bond angle potential               :    3625       0      3   1.903   1.903      259.62       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     35  45.310  45.310      552.03       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      0   1.140   1.140      31.050       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5737       0      0   0.002   0.002      3.1028       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    8285      96      6   0.693   0.693      2710.0       1.000\n",
      "10 Distance restraints 2 (N-O)        :    8728     111     10   0.768   0.768      2999.5       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       0      0   4.018   4.018      62.135       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      0  66.910  66.910      47.619       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      0  73.585  73.585      76.620       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      0  66.661  66.661      53.282       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  65.875  65.875      19.702       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    4983       0      1   0.385   0.385      3415.3       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      16     23  20.796  38.500     -40.403       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2556       4      2   0.918   0.918      1918.9       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990002\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   36949.7656\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13163  92K 231D CA  CA    690 1790   18.00   10.83    7.16   10.23   10.83    7.16   10.23\n",
      "    2  13183  93P 229A CA  CA    699 1771   16.34    9.93    6.42   11.33    9.93    6.42   11.33\n",
      "    3  13184  93P 230W CA  CA    699 1776   14.40    6.84    7.56   14.79    6.84    7.56   14.79\n",
      "    4  13185  93P 231D CA  CA    699 1790   17.59    8.59    9.00   15.14    8.59    9.00   15.14\n",
      "    5  13203  94G 187G CA  CA    706 1433   18.37   13.68    4.68    6.54   13.68    4.68    6.54\n",
      "    6  13204  94G 188E CA  CA    706 1437   18.64   12.73    5.91    8.42   12.73    5.91    8.42\n",
      "    7  13205  94G 227H CA  CA    706 1750   19.46   11.36    8.11   11.15   11.36    8.11   11.15\n",
      "    8  13206  94G 228R CA  CA    706 1760   21.37   13.09    8.28   11.18   13.09    8.28   11.18\n",
      "    9  13207  94G 229A CA  CA    706 1771   18.70   11.28    7.41   11.60   11.28    7.41   11.60\n",
      "   10  13208  94G 230W CA  CA    706 1776   16.70    7.81    8.89   15.56    7.81    8.89   15.56\n",
      "   11  13209  94G 231D CA  CA    706 1790   20.03   10.08    9.95   14.43   10.08    9.95   14.43\n",
      "   12  13210  94G 232I CA  CA    706 1798   20.62   11.87    8.75   12.94   11.87    8.75   12.94\n",
      "   13  13211  94G 233L CA  CA    706 1806   17.53    9.21    8.32   14.36    9.21    8.32   14.36\n",
      "   14  13212  94G 234E CA  CA    706 1814   18.49    8.27   10.22   16.44    8.27   10.22   16.44\n",
      "   15  13214  94G 236R CA  CA    706 1833   21.67   13.26    8.40   11.15   13.26    8.40   11.15\n",
      "   16  13215  94G 237G CA  CA    706 1844   18.38   10.86    7.51   10.27   10.86    7.51   10.27\n",
      "   17  13216  94G 238F CA  CA    706 1848   18.37   12.28    6.08    8.09   12.28    6.08    8.09\n",
      "   18  13217  94G 239T CA  CA    706 1859   18.08   12.15    5.93    8.69   12.15    5.93    8.69\n",
      "   19  13228  95Q 187G CA  CA    710 1433   15.95   11.42    4.53    8.37   11.42    4.53    8.37\n",
      "   20  13229  95Q 188E CA  CA    710 1437   16.77   11.24    5.53    9.82   11.24    5.53    9.82\n",
      "   21  13230  95Q 226G CA  CA    710 1746   19.98   13.81    6.17   11.01   13.81    6.17   11.01\n",
      "   22  13233  95Q 229A CA  CA    710 1771   17.76   10.81    6.95   13.49   10.81    6.95   13.49\n",
      "   23  13235  95Q 231D CA  CA    710 1790   20.04   11.57    8.48   13.78   11.57    8.48   13.78\n",
      "   24  13237  95Q 233L CA  CA    710 1806   17.73   10.12    7.61   15.27   10.12    7.61   15.27\n",
      "   25  13239  95Q 237G CA  CA    710 1844   19.18   12.31    6.87   10.46   12.31    6.87   10.46\n",
      "   26  13334  98A 130N CA  CA    737  981   12.07    7.85    4.22    7.86    7.85    4.22    7.86\n",
      "   27  13336  98A 132D CA  CA    737  996   13.74    9.85    3.89    7.09    9.85    3.89    7.09\n",
      "   28  13337  98A 133N CA  CA    737 1004   13.96   10.65    3.30    6.09   10.65    3.30    6.09\n",
      "   29  13341  98A 185N CA  CA    737 1415   15.09   10.87    4.22    7.83   10.87    4.22    7.83\n",
      "   30  13342  98A 186H CA  CA    737 1423   15.15   10.26    4.89    9.49   10.26    4.89    9.49\n",
      "   31  13343  98A 187G CA  CA    737 1433   13.54    8.20    5.34   10.97    8.20    5.34   10.97\n",
      "   32  13344  98A 188E CA  CA    737 1437   15.04    9.91    5.13    9.13    9.91    5.13    9.13\n",
      "   33  13345  98A 189S CA  CA    737 1446   15.53   11.51    4.02    6.81   11.51    4.02    6.81\n",
      "   34  13346  98A 190M CA  CA    737 1452   17.93   13.24    4.69    8.59   13.24    4.69    8.59\n",
      "   35  13347  98A 226G CA  CA    737 1746   19.42   14.00    5.42    9.24   14.00    5.42    9.24\n",
      "   36  13349  98A 229A CA  CA    737 1771   19.54   13.20    6.34   11.00   13.20    6.34   11.00\n",
      "   37  13351  98A 316T CA  CA    737 2448   16.02   12.90    3.12    5.73   12.90    3.12    5.73\n",
      "   38  13523 103S 334G CA  CA    772 2598   18.26   13.14    5.12    6.30   13.14    5.12    6.30\n",
      "   39  13628 106V 330H CA  CA    792 2566    9.47    6.79    2.68    4.77    6.79    2.68    4.77\n",
      "   40  13632 106V 334G CA  CA    792 2598   15.96   10.50    5.45    7.53   10.50    5.45    7.53\n",
      "   41  13660 107I 330H CA  CA    799 2566   13.07    9.90    3.16    5.23    9.90    3.16    5.23\n",
      "   42  13664 107I 334G CA  CA    799 2598   19.13   13.98    5.15    6.27   13.98    5.15    6.27\n",
      "   43  13703 108F 330H CA  CA    807 2566   13.20    9.20    4.00    7.25    9.20    4.00    7.25\n",
      "   44  13740 109K 329N CA  CA    818 2558   13.74   10.46    3.28    4.76   10.46    3.28    4.76\n",
      "   45  13741 109K 330H CA  CA    818 2566   11.73    6.72    5.01   12.01    6.72    5.01   12.01\n",
      "   46  13742 109K 331L CA  CA    818 2576    8.86    6.85    2.01    5.04    6.85    2.01    5.04\n",
      "   47  13770 110N 330H CA  CA    827 2566   14.42    9.72    4.70    7.55    9.72    4.70    7.55\n",
      "   48  13774 110N 334G CA  CA    827 2598   18.55   13.78    4.78    5.59   13.78    4.78    5.59\n",
      "   49  13806 111T 330H CA  CA    835 2566   16.97   12.19    4.78    8.71   12.19    4.78    8.71\n",
      "   50  13848 112G 330H CA  CA    842 2566   16.52   11.12    5.40    8.46   11.12    5.40    8.46\n",
      "   51  13850 112G 332A CA  CA    842 2584   13.12    9.25    3.87    9.44    9.25    3.87    9.44\n",
      "   52  13851 112G 333Q CA  CA    842 2589   15.27   10.97    4.30    8.26   10.97    4.30    8.26\n",
      "   53  13958 116S 332A CA  CA    875 2584   16.45   11.97    4.48    8.07   11.97    4.48    8.07\n",
      "   54  14531 133N 330H CA  CA   1004 2566   16.73   14.08    2.65    4.90   14.08    2.65    4.90\n",
      "   55  14579 134T 330H CA  CA   1012 2566   12.95   10.46    2.49    5.24   10.46    2.49    5.24\n",
      "   56  14620 135N 330H CA  CA   1019 2566   13.36   10.09    3.28    6.26   10.09    3.28    6.26\n",
      "   57  14621 135N 331L CA  CA   1019 2576   12.13   10.14    2.00    4.62   10.14    2.00    4.62\n",
      "   58  14662 136C 330H CA  CA   1027 2566   15.92   11.77    4.14    7.06   11.77    4.14    7.06\n",
      "   59  14663 136C 331L CA  CA   1027 2576   14.10   10.83    3.27    6.29   10.83    3.27    6.29\n",
      "   60  14710 137E 330H CA  CA   1033 2566   13.88    9.82    4.06    7.76    9.82    4.06    7.76\n",
      "   61  14711 137E 331L CA  CA   1033 2576   11.92    7.96    3.96    8.32    7.96    3.96    8.32\n",
      "   62  14746 138I 330H CA  CA   1042 2566   11.30    6.47    4.83    9.89    6.47    4.83    9.89\n",
      "   63  14747 138I 331L CA  CA   1042 2576    8.92    5.50    3.42    8.97    5.50    3.42    8.97\n",
      "   64  14776 139A 330H CA  CA   1050 2566   14.43    9.20    5.23    8.82    9.20    5.23    8.82\n",
      "   65  14777 139A 331L CA  CA   1050 2576   11.69    8.20    3.50    7.30    8.20    3.50    7.30\n",
      "   66  14806 140M 331L CA  CA   1055 2576   13.42    8.72    4.70   10.30    8.72    4.70   10.30\n",
      "   67  14807 140M 332A CA  CA   1055 2584   12.38    9.41    2.97    6.06    9.41    2.97    6.06\n",
      "   68  14808 140M 333Q CA  CA   1055 2589   12.95    9.31    3.64    6.10    9.31    3.64    6.10\n",
      "   69  14834 141L 330H CA  CA   1063 2566   14.07    8.57    5.50    7.80    8.57    5.50    7.80\n",
      "   70  14835 141L 331L CA  CA   1063 2576   10.88    5.55    5.33   12.75    5.55    5.33   12.75\n",
      "   71  14836 141L 332A CA  CA   1063 2584    9.11    6.24    2.86    5.61    6.24    2.86    5.61\n",
      "   72  14837 141L 333Q CA  CA   1063 2589    9.62    5.81    3.81    8.36    5.81    3.81    8.36\n",
      "   73  14855 142H 330H CA  CA   1071 2566   14.13    8.60    5.53    5.80    8.60    5.53    5.80\n",
      "   74  14856 142H 331L CA  CA   1071 2576   10.61    6.72    3.89    8.12    6.72    3.89    8.12\n",
      "   75  14857 142H 332A CA  CA   1071 2584    9.12    5.13    3.99   10.06    5.13    3.99   10.06\n",
      "   76  14858 142H 333Q CA  CA   1071 2589   10.98    5.92    5.06    8.15    5.92    5.06    8.15\n",
      "   77  14869 143A 330H CA  CA   1081 2566   17.83   12.09    5.74    5.39   12.09    5.74    5.39\n",
      "   78  14870 143A 331L CA  CA   1081 2576   14.33   10.19    4.14    8.78   10.19    4.14    8.78\n",
      "   79  14871 143A 332A CA  CA   1081 2584   12.63    8.67    3.96    7.94    8.67    3.96    7.94\n",
      "   80  14872 143A 333Q CA  CA   1081 2589   14.17    8.05    6.12   12.32    8.05    6.12   12.32\n",
      "   81  14881 144K 330H CA  CA   1086 2566   19.41   13.91    5.50    5.67   13.91    5.50    5.67\n",
      "   82  14882 144K 331L CA  CA   1086 2576   15.79   10.99    4.80    8.11   10.99    4.80    8.11\n",
      "   83  14883 144K 332A CA  CA   1086 2584   13.28    9.47    3.81    6.36    9.47    3.81    6.36\n",
      "   84  14941 148P 331L CA  CA   1120 2576   18.45   13.02    5.43    8.33   13.02    5.43    8.33\n",
      "   85  14942 148P 333Q CA  CA   1120 2589   16.52   13.03    3.50    4.58   13.03    3.50    4.58\n",
      "   86  18000 270P 331L CA  CA   2080 2576   19.12   13.86    5.26    4.75   13.86    5.26    4.75\n",
      "   87  18288 282G 331L CA  CA   2169 2576   15.84   10.82    5.02    6.59   10.82    5.02    6.59\n",
      "   88  18311 283V 331L CA  CA   2173 2576   15.78   10.73    5.06    9.01   10.73    5.06    9.01\n",
      "   89  18335 284V 331L CA  CA   2180 2576   17.12   12.75    4.36    5.40   12.75    4.36    5.40\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21010  93P 229A N   O     698 1774   15.15    8.04    7.10   12.52    8.04    7.10   12.52\n",
      "    2  21011  93P 230W N   O     698 1788   15.00    6.68    8.33   14.41    6.68    8.33   14.41\n",
      "    3  21012  93P 231D N   O     698 1796   17.82    9.26    8.56   12.43    9.26    8.56   12.43\n",
      "    4  21026  94G 227H N   O     705 1758   18.47    9.79    8.68   11.41    9.79    8.68   11.41\n",
      "    5  21027  94G 228R N   O     705 1769   19.36   10.90    8.47   11.44   10.90    8.47   11.44\n",
      "    6  21028  94G 229A N   O     705 1774   16.71    8.66    8.05   12.94    8.66    8.05   12.94\n",
      "    7  21029  94G 230W N   O     705 1788   16.21    6.53    9.68   15.73    6.53    9.68   15.73\n",
      "    8  21030  94G 231D N   O     705 1796   19.24    9.76    9.49   12.54    9.76    9.49   12.54\n",
      "    9  21031  94G 232I N   O     705 1804   18.88   10.18    8.69   12.74   10.18    8.69   12.74\n",
      "   10  21032  94G 233L N   O     705 1812   16.52    7.51    9.01   15.14    7.51    9.01   15.14\n",
      "   11  21034  94G 237G N   O     705 1846   15.65   10.03    5.62    7.17   10.03    5.62    7.17\n",
      "   12  21035  94G 238F N   O     705 1857   17.45   10.42    7.03    9.06   10.42    7.03    9.06\n",
      "   13  21046  95Q 229A N   O     709 1774   17.08    9.38    7.70   15.06    9.38    7.70   15.06\n",
      "   14  21048  95Q 233L N   O     709 1812   17.77    9.47    8.30   16.62    9.47    8.30   16.62\n",
      "   15  21108  98A 185N N   O     736 1421   15.18   10.36    4.82    9.33   10.36    4.82    9.33\n",
      "   16  21109  98A 186H N   O     736 1431   12.32    7.08    5.24   10.61    7.08    5.24   10.61\n",
      "   17  21110  98A 187G N   O     736 1435   14.68    9.30    5.38   10.47    9.30    5.38   10.47\n",
      "   18  21111  98A 188E N   O     736 1444   15.77   10.81    4.96    8.18   10.81    4.96    8.18\n",
      "   19  21112  98A 226G N   O     736 1748   16.78   10.95    5.84   10.34   10.95    5.84   10.34\n",
      "   20  21226 103S  99D N   O     771  748    6.03    2.89    3.14    6.08    2.89    3.14    6.08\n",
      "   21  21433 109K 329N N   O     817 2564   13.49    9.50    3.99    5.47    9.50    3.99    5.47\n",
      "   22  21434 109K 330H N   O     817 2574   11.07    5.50    5.57    8.80    5.50    5.57    8.80\n",
      "   23  21460 110N 329N N   O     826 2564   14.81   10.08    4.73    5.24   10.08    4.73    5.24\n",
      "   24  21461 110N 330H N   O     826 2574   12.08    7.33    4.75    5.50    7.33    4.75    5.50\n",
      "   25  21528 112G 330H N   O     841 2574   15.30    9.21    6.09   12.12    9.21    6.09   12.12\n",
      "   26  22272 135N 330H N   O    1018 2574   14.15    9.70    4.44    8.51    9.70    4.44    8.51\n",
      "   27  22318 136C 330H N   O    1026 2574   15.60   10.12    5.48   10.09   10.12    5.48   10.09\n",
      "   28  22363 137E 330H N   O    1032 2574   14.93    9.17    5.77   10.20    9.17    5.77   10.20\n",
      "   29  22364 137E 331L N   O    1032 2582   14.17   11.46    2.70    5.02   11.46    2.70    5.02\n",
      "   30  22406 138I 330H N   O    1041 2574   12.40    6.29    6.10   12.21    6.29    6.10   12.21\n",
      "   31  22452 139A 330H N   O    1049 2574   13.23    6.50    6.73   13.42    6.50    6.73   13.42\n",
      "   32  22453 139A 331L N   O    1049 2582   12.46    9.04    3.42    5.40    9.04    3.42    5.40\n",
      "   33  22455 139A 333Q N   O    1049 2596   13.49    9.79    3.70    6.19    9.79    3.70    6.19\n",
      "   34  22496 140M 330H N   O    1054 2574   15.22    8.16    7.07   11.61    8.16    7.07   11.61\n",
      "   35  22497 140M 331L N   O    1054 2582   13.94    9.94    4.00    5.82    9.94    4.00    5.82\n",
      "   36  22498 140M 332A N   O    1054 2587   13.45   10.79    2.66    5.00   10.79    2.66    5.00\n",
      "   37  22499 140M 333Q N   O    1054 2596   14.53   10.90    3.63    6.23   10.90    3.63    6.23\n",
      "   38  22530 141L 330H N   O    1062 2574   14.20    6.86    7.34    7.64    6.86    7.34    7.64\n",
      "   39  22531 141L 331L N   O    1062 2582   12.39    7.93    4.45    8.71    7.93    4.45    8.71\n",
      "   40  22533 141L 333Q N   O    1062 2596   12.63    8.93    3.70    5.87    8.93    3.70    5.87\n",
      "   41  22563 142H 329N N   O    1070 2564   14.36   10.02    4.34    6.19   10.02    4.34    6.19\n",
      "   42  22564 142H 330H N   O    1070 2574   13.14    6.05    7.09    6.84    6.05    7.09    6.84\n",
      "   43  22565 142H 331L N   O    1070 2582   11.35    6.65    4.70    8.36    6.65    4.70    8.36\n",
      "   44  22566 142H 332A N   O    1070 2587   10.45    6.64    3.81    6.96    6.64    3.81    6.96\n",
      "   45  22567 142H 333Q N   O    1070 2596   12.38    5.78    6.61   13.42    5.78    6.61   13.42\n",
      "   46  22598 143A 330H N   O    1080 2574   15.56    8.37    7.20    7.23    8.37    7.20    7.23\n",
      "   47  22599 143A 331L N   O    1080 2582   13.79    8.95    4.84    8.01    8.95    4.84    8.01\n",
      "   48  22600 143A 332A N   O    1080 2587   12.75    8.38    4.37    8.24    8.38    4.37    8.24\n",
      "   49  22601 143A 333Q N   O    1080 2596   14.93    7.49    7.44   13.01    7.49    7.44   13.01\n",
      "   50  22626 144K 330H N   O    1085 2574   17.46   10.88    6.58    4.95   10.88    6.58    4.95\n",
      "   51  22627 144K 331L N   O    1085 2582   15.29   10.20    5.09    5.53   10.20    5.09    5.53\n",
      "   52  22628 144K 332A N   O    1085 2587   13.62    8.86    4.75    7.07    8.86    4.75    7.07\n",
      "   53  22629 144K 333Q N   O    1085 2596   16.19    7.86    8.33   12.73    7.86    8.33   12.73\n",
      "   54  23815 184G 275N N   O    1410 2120   13.25   10.28    2.97    5.93   10.28    2.97    5.93\n",
      "   55  23911 187G  95Q N   O    1432  717   14.89   10.65    4.24    7.94   10.65    4.24    7.94\n",
      "   56  23946 188E  95Q N   O    1436  717   13.74    8.76    4.98    9.16    8.76    4.98    9.16\n",
      "   57  24848 229A  95Q N   O    1770  717   16.44    9.29    7.15   13.99    9.29    7.15   13.99\n",
      "   58  24874 230W  94G N   O    1775  708   18.46    9.83    8.63   12.44    9.83    8.63   12.44\n",
      "   59  24896 231D  93P N   O    1789  704   18.90   10.06    8.84   12.62   10.06    8.84   12.62\n",
      "   60  24897 231D  94G N   O    1789  708   20.25   10.64    9.61   12.36   10.64    9.61   12.36\n",
      "   61  24898 231D  95Q N   O    1789  717   16.83    8.60    8.23   13.88    8.60    8.23   13.88\n",
      "   62  24913 232I  91L N   O    1797  688   15.24   10.23    5.02    9.88   10.23    5.02    9.88\n",
      "   63  24915 232I  95Q N   O    1797  717   18.22   10.29    7.93   14.43   10.29    7.93   14.43\n",
      "   64  24934 233L  95Q N   O    1805  717   16.77    9.35    7.42   14.85    9.35    7.42   14.85\n",
      "   65  26193 283V 331L N   O    2172 2582   16.76   10.94    5.82   10.53   10.94    5.82   10.53\n",
      "   66  26904 313L 275N N   O    2420 2120   13.74   10.67    3.06    5.51   10.67    3.06    5.51\n",
      "   67  26997 317E 275N N   O    2454 2120   13.42   10.14    3.28    5.37   10.14    3.28    5.37\n",
      "   68  27313 330H 105S N   O    2565  790   10.01    7.23    2.78    5.81    7.23    2.78    5.81\n",
      "   69  27315 330H 108F N   O    2565  816   14.22   10.34    3.89    7.19   10.34    3.89    7.19\n",
      "   70  27316 330H 109K N   O    2565  825   14.30    9.87    4.43    6.89    9.87    4.43    6.89\n",
      "   71  27317 330H 134T N   O    2565 1017   11.77    9.32    2.45    4.68    9.32    2.45    4.68\n",
      "   72  27318 330H 135N N   O    2565 1025   13.79   10.77    3.02    5.84   10.77    3.02    5.84\n",
      "   73  27319 330H 137E N   O    2565 1040   12.88    9.36    3.52    6.57    9.36    3.52    6.57\n",
      "   74  27320 330H 138I N   O    2565 1048   12.23    7.93    4.30    7.87    7.93    4.30    7.87\n",
      "   75  27321 330H 139A N   O    2565 1053   15.59   11.13    4.46    7.02   11.13    4.46    7.02\n",
      "   76  27322 330H 141L N   O    2565 1069   14.40    9.47    4.93    7.74    9.47    4.93    7.74\n",
      "   77  27343 331L 109K N   O    2575  825   11.80    8.77    3.03    5.74    8.77    3.03    5.74\n",
      "   78  27344 331L 134T N   O    2575 1017   10.89    8.32    2.58    5.90    8.32    2.58    5.90\n",
      "   79  27345 331L 135N N   O    2575 1025   12.37    9.46    2.91    6.33    9.46    2.91    6.33\n",
      "   80  27346 331L 136C N   O    2575 1031   13.74    9.92    3.83    7.75    9.92    3.83    7.75\n",
      "   81  27347 331L 137E N   O    2575 1040   11.35    7.20    4.15    8.75    7.20    4.15    8.75\n",
      "   82  27348 331L 138I N   O    2575 1048   10.19    6.03    4.15    9.35    6.03    4.15    9.35\n",
      "   83  27349 331L 139A N   O    2575 1053   13.45    9.13    4.32    9.20    9.13    4.32    9.20\n",
      "   84  27350 331L 140M N   O    2575 1061   14.33    9.52    4.81    9.22    9.52    4.81    9.22\n",
      "   85  27351 331L 141L N   O    2575 1069   12.20    7.04    5.16   10.39    7.04    5.16   10.39\n",
      "   86  27352 331L 142H N   O    2575 1079   13.76    9.85    3.92    7.91    9.85    3.92    7.91\n",
      "   87  27353 331L 282G N   O    2575 2171   14.17   10.10    4.07    5.07   10.10    4.07    5.07\n",
      "   88  27381 332A 140M N   O    2583 1061   11.89    7.81    4.08    7.73    7.81    4.08    7.73\n",
      "   89  27382 332A 141L N   O    2583 1069    9.28    4.61    4.67   10.06    4.61    4.67   10.06\n",
      "   90  27383 332A 142H N   O    2583 1079   11.06    7.04    4.02    7.96    7.04    4.02    7.96\n",
      "   91  27384 332A 143A N   O    2583 1084   14.13   10.07    4.06    7.26   10.07    4.06    7.26\n",
      "   92  27411 333Q 140M N   O    2588 1061   11.88    7.32    4.56    7.96    7.32    4.56    7.96\n",
      "   93  27412 333Q 141L N   O    2588 1069    9.32    3.51    5.82   11.05    3.51    5.82   11.05\n",
      "   94  27414 333Q 143A N   O    2588 1084   14.25    8.81    5.44    9.22    8.81    5.44    9.22\n",
      "   95  27430 334G 102S N   O    2597  770   15.08   10.61    4.46    5.17   10.61    4.46    5.17\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9191  31G  32D C   N     225  227  -79.10  -96.50   62.07    2.62  -63.30   95.92   10.97\n",
      "    1         32D  32D N   CA    227  228   54.62  114.20                  -40.00\n",
      "    2   9258  98A  99D C   N     739  741  -74.07  -70.90   13.11    0.50  -63.30  157.36   20.09\n",
      "    2         99D  99D N   CA    741  742  163.02  150.30                  -40.00\n",
      "    3   9259  99D 100L C   N     747  749   81.26   60.20   70.72    2.65  -63.50  144.80   25.12\n",
      "    3        100L 100L N   CA    749  750  -37.91   29.60                  -41.20\n",
      "    4   9303 143A 144K C   N    1083 1085  -70.39  -70.20   14.01    1.01  -62.90  164.96   21.84\n",
      "    4        144K 144K N   CA   1085 1086  154.41  140.40                  -40.80\n",
      "    5   9304 144K 145N C   N    1092 1094   83.91   55.90   51.22    2.53  -63.20  151.86   24.39\n",
      "    5        145N 145N N   CA   1094 1095   -3.39   39.50                  -41.10\n",
      "    6   9359 199F 200T C   N    1524 1526  -60.75  -63.20    9.16    1.03 -124.80  177.53    9.43\n",
      "    6        200T 200T N   CA   1526 1527  -50.93  -42.10                  143.50\n",
      "    7   9360 200T 201K C   N    1531 1533   53.59   56.60  107.63    7.46 -118.00  171.73    6.60\n",
      "    7        201K 201K N   CA   1533 1534  146.19   38.60                  139.10\n",
      "    8   9386 226G 227H C   N    1747 1749 -102.73  -63.20   77.69    8.26  -63.20   77.69    8.26\n",
      "    8        227H 227H N   CA   1749 1750   24.57  -42.30                  -42.30\n",
      "    9   9430 270P 271V C   N    2084 2086  -68.23  -73.50   12.11    0.94  -62.40  170.80   21.29\n",
      "    9        271V 271V N   CA   2086 2087  128.30  139.20                  -42.40\n",
      "   10   9431 271V 272P C   N    2091 2093  -68.32  -64.50    4.75    0.53  -58.70  175.15   13.71\n",
      "   10        272P 272P N   CA   2093 2094  144.38  147.20                  -30.50\n",
      "   11   9432 272P 273E C   N    2098 2100  -63.32  -69.30   30.75    2.13  -63.60  152.64   20.37\n",
      "   11        273E 273E N   CA   2100 2101  112.34  142.50                  -40.30\n",
      "   12   9452 292D 293K C   N    2248 2250  -58.47  -62.90   23.50    3.31   56.60  128.11   16.41\n",
      "   12        293K 293K N   CA   2250 2251  -17.73  -40.80                   38.60\n",
      "   13   9453 293K 294E C   N    2257 2259  -53.03  -63.60   15.15    2.56   54.60  129.41   16.90\n",
      "   13        294E 294E N   CA   2259 2260  -29.45  -40.30                   42.40\n",
      "   14   9490 330H 331L C   N    2573 2575 -103.15  -70.70   87.21    5.57  -63.50  104.10   16.91\n",
      "   14        331L 331L N   CA   2575 2576 -137.45  141.60                  -41.20\n",
      "   15   9492 332A 333Q C   N    2586 2588  -77.18  -73.00    7.43    0.56  -63.80  175.37   25.10\n",
      "   15        333Q 333Q N   CA   2588 2589  134.56  140.70                  -40.30\n",
      "   16   9493 333Q 334G C   N    2595 2597  -70.17  -80.20   23.55    0.55   78.70  154.45    8.50\n",
      "   16        334G 334G N   CA   2597 2598  152.79  174.10                 -166.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33726 144K 333Q CB  CG   1087 2591   13.12    4.50    8.62    5.36    4.50    8.62    5.36\n",
      "    2  34966 324K 331L CG  CD1  2521 2579   11.81    4.87    6.95    4.79    4.87    6.95    4.79\n",
      "    3  34969 324K 331L CE  CD1  2523 2579   12.56    4.30    8.26    5.90    4.30    8.26    5.90\n",
      "    4  34972 324K 331L NZ  CD1  2524 2579   12.08    4.10    7.98    5.77    4.10    7.98    5.77\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   20   33  108  194  272  213  357  418  419  435\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "openf___224_> Open           TvLDH.ini\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13115071   12807.687    12.508\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16617\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18806\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    20098\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21731\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    23388\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25723\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    29083\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    30473\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    32123\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34005\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "iupac_m_485_> OD1/2 will be swapped:      146.6626       32    32\n",
      "iupac_m_487_> NH1/2 swapped:      176.2878       33    33\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.3444       36    36\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5110       48    48\n",
      "iupac_m_486_> OE1/2 will be swapped:      135.3377       55    55\n",
      "iupac_m_485_> OD1/2 will be swapped:      121.3716       58    58\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       97.9441       67    67\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      177.3174       97    97\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.0629      137   137\n",
      "iupac_m_486_> OE1/2 will be swapped:     -137.6589      149   149\n",
      "iupac_m_487_> NH1/2 swapped:     -179.5429      161   161\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -91.6865      163   163\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      101.4223      165   165\n",
      "iupac_m_485_> OD1/2 will be swapped:      152.4579      179   179\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -98.9904      199   199\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      132.4426      219   219\n",
      "iupac_m_485_> OD1/2 will be swapped:      172.6959      231   231\n",
      "iupac_m_486_> OE1/2 will be swapped:      121.6540      234   234\n",
      "iupac_m_487_> NH1/2 swapped:      177.0814      236   236\n",
      "iupac_m_486_> OE1/2 will be swapped:      154.8272      273   273\n",
      "iupac_m_486_> OE1/2 will be swapped:     -156.4403      311   311\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.3425      315   315\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      148.4142      319   319\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34988   34988\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5771\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :       12247.2529\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      18.827       1.000\n",
      " 2 Bond angle potential               :    3625       0      3   1.959   1.959      272.23       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     35  45.426  45.426      554.19       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      1   1.266   1.266      37.231       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5771       0      0   0.002   0.002      2.9052       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    8285      99      8   0.707   0.707      2703.7       1.000\n",
      "10 Distance restraints 2 (N-O)        :    8728     100     23   0.776   0.776      3027.8       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       0      1   4.138   4.138      66.106       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      5  63.523  63.523      59.414       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      1  68.851  68.851      75.881       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      0  72.335  72.335      53.326       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  68.825  68.825      18.297       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    4983       0      0   0.439   0.439      3433.6       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      10     25  19.683  33.162     -40.882       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2556       3     13   0.911   0.911      1964.7       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990003\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   36461.8398\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13163  92K 231D CA  CA    690 1790   17.99   10.83    7.15   10.21   10.83    7.15   10.21\n",
      "    2  13183  93P 229A CA  CA    699 1771   16.44    9.93    6.52   11.50    9.93    6.52   11.50\n",
      "    3  13184  93P 230W CA  CA    699 1776   14.45    6.84    7.62   14.90    6.84    7.62   14.90\n",
      "    4  13185  93P 231D CA  CA    699 1790   17.62    8.59    9.03   15.18    8.59    9.03   15.18\n",
      "    5  13203  94G 187G CA  CA    706 1433   18.38   13.68    4.70    6.56   13.68    4.70    6.56\n",
      "    6  13204  94G 188E CA  CA    706 1437   18.72   12.73    5.98    8.52   12.73    5.98    8.52\n",
      "    7  13205  94G 227H CA  CA    706 1750   19.73   11.36    8.37   11.52   11.36    8.37   11.52\n",
      "    8  13206  94G 228R CA  CA    706 1760   21.46   13.09    8.37   11.31   13.09    8.37   11.31\n",
      "    9  13207  94G 229A CA  CA    706 1771   18.77   11.28    7.49   11.71   11.28    7.49   11.71\n",
      "   10  13208  94G 230W CA  CA    706 1776   16.70    7.81    8.89   15.57    7.81    8.89   15.57\n",
      "   11  13209  94G 231D CA  CA    706 1790   19.98   10.08    9.90   14.36   10.08    9.90   14.36\n",
      "   12  13210  94G 232I CA  CA    706 1798   20.62   11.87    8.75   12.94   11.87    8.75   12.94\n",
      "   13  13211  94G 233L CA  CA    706 1806   17.51    9.21    8.30   14.34    9.21    8.30   14.34\n",
      "   14  13212  94G 234E CA  CA    706 1814   18.47    8.27   10.20   16.41    8.27   10.20   16.41\n",
      "   15  13214  94G 236R CA  CA    706 1833   21.47   13.26    8.20   10.88   13.26    8.20   10.88\n",
      "   16  13215  94G 237G CA  CA    706 1844   18.14   10.86    7.28    9.95   10.86    7.28    9.95\n",
      "   17  13216  94G 238F CA  CA    706 1848   18.14   12.28    5.86    7.79   12.28    5.86    7.79\n",
      "   18  13217  94G 239T CA  CA    706 1859   18.03   12.15    5.88    8.62   12.15    5.88    8.62\n",
      "   19  13228  95Q 187G CA  CA    710 1433   15.83   11.42    4.41    8.15   11.42    4.41    8.15\n",
      "   20  13229  95Q 188E CA  CA    710 1437   16.73   11.24    5.50    9.75   11.24    5.50    9.75\n",
      "   21  13230  95Q 226G CA  CA    710 1746   19.74   13.81    5.93   10.58   13.81    5.93   10.58\n",
      "   22  13233  95Q 229A CA  CA    710 1771   17.67   10.81    6.86   13.32   10.81    6.86   13.32\n",
      "   23  13235  95Q 231D CA  CA    710 1790   19.86   11.57    8.29   13.48   11.57    8.29   13.48\n",
      "   24  13237  95Q 233L CA  CA    710 1806   17.56   10.12    7.44   14.93   10.12    7.44   14.93\n",
      "   25  13239  95Q 237G CA  CA    710 1844   18.80   12.31    6.50    9.89   12.31    6.50    9.89\n",
      "   26  13334  98A 130N CA  CA    737  981   11.92    7.85    4.07    7.58    7.85    4.07    7.58\n",
      "   27  13336  98A 132D CA  CA    737  996   13.66    9.85    3.81    6.94    9.85    3.81    6.94\n",
      "   28  13337  98A 133N CA  CA    737 1004   14.00   10.65    3.35    6.17   10.65    3.35    6.17\n",
      "   29  13341  98A 185N CA  CA    737 1415   15.07   10.87    4.20    7.79   10.87    4.20    7.79\n",
      "   30  13342  98A 186H CA  CA    737 1423   15.17   10.26    4.91    9.54   10.26    4.91    9.54\n",
      "   31  13343  98A 187G CA  CA    737 1433   13.61    8.20    5.41   11.12    8.20    5.41   11.12\n",
      "   32  13344  98A 188E CA  CA    737 1437   15.24    9.91    5.33    9.49    9.91    5.33    9.49\n",
      "   33  13345  98A 189S CA  CA    737 1446   15.61   11.51    4.10    6.94   11.51    4.10    6.94\n",
      "   34  13346  98A 190M CA  CA    737 1452   18.01   13.24    4.78    8.75   13.24    4.78    8.75\n",
      "   35  13347  98A 226G CA  CA    737 1746   19.43   14.00    5.43    9.25   14.00    5.43    9.25\n",
      "   36  13349  98A 229A CA  CA    737 1771   19.72   13.20    6.51   11.30   13.20    6.51   11.30\n",
      "   37  13351  98A 316T CA  CA    737 2448   16.11   12.90    3.21    5.89   12.90    3.21    5.89\n",
      "   38  13354  98A 320L CA  CA    737 2481   12.22   10.01    2.22    4.56   10.01    2.22    4.56\n",
      "   39  13523 103S 334G CA  CA    772 2598   17.77   13.14    4.63    5.70   13.14    4.63    5.70\n",
      "   40  13628 106V 330H CA  CA    792 2566    9.47    6.79    2.67    4.76    6.79    2.67    4.76\n",
      "   41  13632 106V 334G CA  CA    792 2598   14.56   10.50    4.06    5.60   10.50    4.06    5.60\n",
      "   42  13660 107I 330H CA  CA    799 2566   13.04    9.90    3.14    5.19    9.90    3.14    5.19\n",
      "   43  13703 108F 330H CA  CA    807 2566   13.17    9.20    3.97    7.20    9.20    3.97    7.20\n",
      "   44  13740 109K 329N CA  CA    818 2558   13.74   10.46    3.28    4.76   10.46    3.28    4.76\n",
      "   45  13741 109K 330H CA  CA    818 2566   11.80    6.72    5.08   12.18    6.72    5.08   12.18\n",
      "   46  13742 109K 331L CA  CA    818 2576    8.78    6.85    1.93    4.85    6.85    1.93    4.85\n",
      "   47  13770 110N 330H CA  CA    827 2566   14.46    9.72    4.74    7.62    9.72    4.74    7.62\n",
      "   48  13806 111T 330H CA  CA    835 2566   17.01   12.19    4.82    8.78   12.19    4.82    8.78\n",
      "   49  13848 112G 330H CA  CA    842 2566   16.68   11.12    5.56    8.72   11.12    5.56    8.72\n",
      "   50  13850 112G 332A CA  CA    842 2584   15.36    9.25    6.11   14.91    9.25    6.11   14.91\n",
      "   51  13851 112G 333Q CA  CA    842 2589   13.64   10.97    2.67    5.13   10.97    2.67    5.13\n",
      "   52  13958 116S 332A CA  CA    875 2584   18.91   11.97    6.94   12.51   11.97    6.94   12.51\n",
      "   53  14531 133N 330H CA  CA   1004 2566   16.78   14.08    2.70    4.98   14.08    2.70    4.98\n",
      "   54  14579 134T 330H CA  CA   1012 2566   13.00   10.46    2.54    5.35   10.46    2.54    5.35\n",
      "   55  14620 135N 330H CA  CA   1019 2566   13.23   10.09    3.14    6.00   10.09    3.14    6.00\n",
      "   56  14662 136C 330H CA  CA   1027 2566   16.01   11.77    4.23    7.21   11.77    4.23    7.21\n",
      "   57  14663 136C 331L CA  CA   1027 2576   14.22   10.83    3.39    6.51   10.83    3.39    6.51\n",
      "   58  14664 136C 332A CA  CA   1027 2584   17.20   12.69    4.51    5.51   12.69    4.51    5.51\n",
      "   59  14710 137E 330H CA  CA   1033 2566   14.05    9.82    4.22    8.08    9.82    4.22    8.08\n",
      "   60  14711 137E 331L CA  CA   1033 2576   12.16    7.96    4.20    8.81    7.96    4.20    8.81\n",
      "   61  14712 137E 332A CA  CA   1033 2584   14.81   10.75    4.05    5.69   10.75    4.05    5.69\n",
      "   62  14746 138I 330H CA  CA   1042 2566   11.44    6.47    4.97   10.18    6.47    4.97   10.18\n",
      "   63  14747 138I 331L CA  CA   1042 2576    9.07    5.50    3.57    9.36    5.50    3.57    9.36\n",
      "   64  14776 139A 330H CA  CA   1050 2566   14.57    9.20    5.37    9.05    9.20    5.37    9.05\n",
      "   65  14777 139A 331L CA  CA   1050 2576   11.83    8.20    3.63    7.58    8.20    3.63    7.58\n",
      "   66  14778 139A 332A CA  CA   1050 2584   14.23    9.13    5.10    7.31    9.13    5.10    7.31\n",
      "   67  14806 140M 331L CA  CA   1055 2576   13.75    8.72    5.03   11.03    8.72    5.03   11.03\n",
      "   68  14807 140M 332A CA  CA   1055 2584   15.64    9.41    6.23   12.71    9.41    6.23   12.71\n",
      "   69  14808 140M 333Q CA  CA   1055 2589   13.00    9.31    3.68    6.17    9.31    3.68    6.17\n",
      "   70  14834 141L 330H CA  CA   1063 2566   14.36    8.57    5.79    8.22    8.57    5.79    8.22\n",
      "   71  14835 141L 331L CA  CA   1063 2576   11.28    5.55    5.73   13.69    5.55    5.73   13.69\n",
      "   72  14836 141L 332A CA  CA   1063 2584   12.59    6.24    6.34   12.41    6.24    6.34   12.41\n",
      "   73  14837 141L 333Q CA  CA   1063 2589    9.57    5.81    3.76    8.26    5.81    3.76    8.26\n",
      "   74  14855 142H 330H CA  CA   1071 2566   14.38    8.60    5.78    6.06    8.60    5.78    6.06\n",
      "   75  14856 142H 331L CA  CA   1071 2576   10.88    6.72    4.16    8.69    6.72    4.16    8.69\n",
      "   76  14857 142H 332A CA  CA   1071 2584   11.94    5.13    6.82   17.19    5.13    6.82   17.19\n",
      "   77  14858 142H 333Q CA  CA   1071 2589    9.38    5.92    3.46    5.58    5.92    3.46    5.58\n",
      "   78  14869 143A 330H CA  CA   1081 2566   18.14   12.09    6.06    5.69   12.09    6.06    5.69\n",
      "   79  14870 143A 331L CA  CA   1081 2576   14.66   10.19    4.47    9.48   10.19    4.47    9.48\n",
      "   80  14871 143A 332A CA  CA   1081 2584   15.57    8.67    6.90   13.83    8.67    6.90   13.83\n",
      "   81  14872 143A 333Q CA  CA   1081 2589   12.77    8.05    4.72    9.51    8.05    4.72    9.51\n",
      "   82  14881 144K 330H CA  CA   1086 2566   19.68   13.91    5.77    5.94   13.91    5.77    5.94\n",
      "   83  14882 144K 331L CA  CA   1086 2576   16.18   10.99    5.19    8.76   10.99    5.19    8.76\n",
      "   84  14883 144K 332A CA  CA   1086 2584   16.44    9.47    6.97   11.63    9.47    6.97   11.63\n",
      "   85  14941 148P 331L CA  CA   1120 2576   18.79   13.02    5.77    8.84   13.02    5.77    8.84\n",
      "   86  14942 148P 333Q CA  CA   1120 2589   17.14   13.03    4.11    5.38   13.03    4.11    5.38\n",
      "   87  18000 270P 331L CA  CA   2080 2576   19.54   13.86    5.68    5.13   13.86    5.68    5.13\n",
      "   88  18079 274G 317E CA  CA   2110 2455   14.74   11.35    3.39    4.68   11.35    3.39    4.68\n",
      "   89  18288 282G 331L CA  CA   2169 2576   16.22   10.82    5.40    7.09   10.82    5.40    7.09\n",
      "   90  18311 283V 331L CA  CA   2173 2576   16.10   10.73    5.37    9.57   10.73    5.37    9.57\n",
      "   91  18313 283V 333Q CA  CA   2173 2589   16.05   12.07    3.98    4.82   12.07    3.98    4.82\n",
      "   92  18335 284V 331L CA  CA   2180 2576   17.37   12.75    4.62    5.71   12.75    4.62    5.71\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21010  93P 229A N   O     698 1774   15.11    8.04    7.06   12.44    8.04    7.06   12.44\n",
      "    2  21011  93P 230W N   O     698 1788   15.00    6.68    8.32   14.40    6.68    8.32   14.40\n",
      "    3  21012  93P 231D N   O     698 1796   17.87    9.26    8.61   12.51    9.26    8.61   12.51\n",
      "    4  21026  94G 227H N   O     705 1758   18.13    9.79    8.34   10.96    9.79    8.34   10.96\n",
      "    5  21027  94G 228R N   O     705 1769   19.40   10.90    8.50   11.48   10.90    8.50   11.48\n",
      "    6  21028  94G 229A N   O     705 1774   16.69    8.66    8.03   12.90    8.66    8.03   12.90\n",
      "    7  21029  94G 230W N   O     705 1788   16.18    6.53    9.65   15.69    6.53    9.65   15.69\n",
      "    8  21030  94G 231D N   O     705 1796   19.24    9.76    9.49   12.54    9.76    9.49   12.54\n",
      "    9  21031  94G 232I N   O     705 1804   18.86   10.18    8.68   12.72   10.18    8.68   12.72\n",
      "   10  21032  94G 233L N   O     705 1812   16.45    7.51    8.94   15.02    7.51    8.94   15.02\n",
      "   11  21034  94G 237G N   O     705 1846   15.40   10.03    5.37    6.85   10.03    5.37    6.85\n",
      "   12  21035  94G 238F N   O     705 1857   17.46   10.42    7.05    9.08   10.42    7.05    9.08\n",
      "   13  21046  95Q 229A N   O     709 1774   16.98    9.38    7.60   14.86    9.38    7.60   14.86\n",
      "   14  21048  95Q 233L N   O     709 1812   17.65    9.47    8.18   16.38    9.47    8.18   16.38\n",
      "   15  21108  98A 185N N   O     736 1421   15.40   10.36    5.04    9.76   10.36    5.04    9.76\n",
      "   16  21109  98A 186H N   O     736 1431   12.40    7.08    5.32   10.77    7.08    5.32   10.77\n",
      "   17  21110  98A 187G N   O     736 1435   14.80    9.30    5.49   10.69    9.30    5.49   10.69\n",
      "   18  21111  98A 188E N   O     736 1444   15.91   10.81    5.10    8.42   10.81    5.10    8.42\n",
      "   19  21112  98A 226G N   O     736 1748   16.97   10.95    6.03   10.68   10.95    6.03   10.68\n",
      "   20  21433 109K 329N N   O     817 2564   13.41    9.50    3.91    5.36    9.50    3.91    5.36\n",
      "   21  21434 109K 330H N   O     817 2574   11.07    5.50    5.57    8.80    5.50    5.57    8.80\n",
      "   22  21460 110N 329N N   O     826 2564   14.75   10.08    4.67    5.17   10.08    4.67    5.17\n",
      "   23  21461 110N 330H N   O     826 2574   12.09    7.33    4.76    5.51    7.33    4.76    5.51\n",
      "   24  21528 112G 330H N   O     841 2574   15.42    9.21    6.22   12.37    9.21    6.22   12.37\n",
      "   25  22272 135N 330H N   O    1018 2574   14.08    9.70    4.37    8.38    9.70    4.37    8.38\n",
      "   26  22318 136C 330H N   O    1026 2574   15.64   10.12    5.52   10.16   10.12    5.52   10.16\n",
      "   27  22363 137E 330H N   O    1032 2574   15.06    9.17    5.89   10.42    9.17    5.89   10.42\n",
      "   28  22406 138I 330H N   O    1041 2574   12.52    6.29    6.23   12.45    6.29    6.23   12.45\n",
      "   29  22452 139A 330H N   O    1049 2574   13.35    6.50    6.85   13.66    6.50    6.85   13.66\n",
      "   30  22453 139A 331L N   O    1049 2582   12.02    9.04    2.98    4.70    9.04    2.98    4.70\n",
      "   31  22496 140M 330H N   O    1054 2574   15.46    8.16    7.30   12.00    8.16    7.30   12.00\n",
      "   32  22497 140M 331L N   O    1054 2582   13.64    9.94    3.69    5.38    9.94    3.69    5.38\n",
      "   33  22498 140M 332A N   O    1054 2587   15.20   10.79    4.41    8.31   10.79    4.41    8.31\n",
      "   34  22530 141L 330H N   O    1062 2574   14.50    6.86    7.64    7.95    6.86    7.64    7.95\n",
      "   35  22531 141L 331L N   O    1062 2582   12.16    7.93    4.23    8.26    7.93    4.23    8.26\n",
      "   36  22532 141L 332A N   O    1062 2587   13.31    8.62    4.69    5.51    8.62    4.69    5.51\n",
      "   37  22563 142H 329N N   O    1070 2564   14.55   10.02    4.53    6.46   10.02    4.53    6.46\n",
      "   38  22564 142H 330H N   O    1070 2574   13.43    6.05    7.38    7.12    6.05    7.38    7.12\n",
      "   39  22565 142H 331L N   O    1070 2582   11.26    6.65    4.60    8.19    6.65    4.60    8.19\n",
      "   40  22566 142H 332A N   O    1070 2587   12.12    6.64    5.48   10.02    6.64    5.48   10.02\n",
      "   41  22567 142H 333Q N   O    1070 2596    9.13    5.78    3.36    6.82    5.78    3.36    6.82\n",
      "   42  22598 143A 330H N   O    1080 2574   15.87    8.37    7.50    7.53    8.37    7.50    7.53\n",
      "   43  22599 143A 331L N   O    1080 2582   13.75    8.95    4.80    7.94    8.95    4.80    7.94\n",
      "   44  22600 143A 332A N   O    1080 2587   14.32    8.38    5.94   11.22    8.38    5.94   11.22\n",
      "   45  22601 143A 333Q N   O    1080 2596   11.56    7.49    4.07    7.12    7.49    4.07    7.12\n",
      "   46  22626 144K 330H N   O    1085 2574   18.00   10.88    7.11    5.35   10.88    7.11    5.35\n",
      "   47  22627 144K 331L N   O    1085 2582   15.51   10.20    5.31    5.77   10.20    5.31    5.77\n",
      "   48  22628 144K 332A N   O    1085 2587   15.47    8.86    6.61    9.83    8.86    6.61    9.83\n",
      "   49  22629 144K 333Q N   O    1085 2596   12.87    7.86    5.01    7.65    7.86    5.01    7.65\n",
      "   50  23815 184G 275N N   O    1410 2120   13.11   10.28    2.82    5.64   10.28    2.82    5.64\n",
      "   51  23911 187G  95Q N   O    1432  717   14.88   10.65    4.23    7.93   10.65    4.23    7.93\n",
      "   52  23946 188E  95Q N   O    1436  717   13.74    8.76    4.97    9.15    8.76    4.97    9.15\n",
      "   53  24848 229A  95Q N   O    1770  717   16.54    9.29    7.25   14.19    9.29    7.25   14.19\n",
      "   54  24874 230W  94G N   O    1775  708   18.61    9.83    8.79   12.66    9.83    8.79   12.66\n",
      "   55  24896 231D  93P N   O    1789  704   18.89   10.06    8.83   12.60   10.06    8.83   12.60\n",
      "   56  24897 231D  94G N   O    1789  708   20.35   10.64    9.72   12.49   10.64    9.72   12.49\n",
      "   57  24898 231D  95Q N   O    1789  717   16.81    8.60    8.22   13.85    8.60    8.22   13.85\n",
      "   58  24913 232I  91L N   O    1797  688   15.14   10.23    4.91    9.67   10.23    4.91    9.67\n",
      "   59  24915 232I  95Q N   O    1797  717   18.26   10.29    7.97   14.51   10.29    7.97   14.51\n",
      "   60  24934 233L  95Q N   O    1805  717   16.82    9.35    7.47   14.95    9.35    7.47   14.95\n",
      "   61  26193 283V 331L N   O    2172 2582   16.09   10.94    5.15    9.31   10.94    5.15    9.31\n",
      "   62  26904 313L 275N N   O    2420 2120   13.84   10.67    3.17    5.70   10.67    3.17    5.70\n",
      "   63  26997 317E 275N N   O    2454 2120   13.45   10.14    3.31    5.41   10.14    3.31    5.41\n",
      "   64  27313 330H 105S N   O    2565  790   10.01    7.23    2.78    5.81    7.23    2.78    5.81\n",
      "   65  27315 330H 108F N   O    2565  816   14.23   10.34    3.90    7.21   10.34    3.90    7.21\n",
      "   66  27316 330H 109K N   O    2565  825   14.42    9.87    4.54    7.07    9.87    4.54    7.07\n",
      "   67  27317 330H 134T N   O    2565 1017   11.77    9.32    2.45    4.69    9.32    2.45    4.69\n",
      "   68  27318 330H 135N N   O    2565 1025   13.86   10.77    3.09    5.98   10.77    3.09    5.98\n",
      "   69  27319 330H 137E N   O    2565 1040   13.02    9.36    3.66    6.83    9.36    3.66    6.83\n",
      "   70  27320 330H 138I N   O    2565 1048   12.38    7.93    4.45    8.14    7.93    4.45    8.14\n",
      "   71  27321 330H 139A N   O    2565 1053   15.75   11.13    4.62    7.26   11.13    4.62    7.26\n",
      "   72  27322 330H 141L N   O    2565 1069   14.56    9.47    5.09    7.98    9.47    5.09    7.98\n",
      "   73  27343 331L 109K N   O    2575  825   11.76    8.77    2.99    5.66    8.77    2.99    5.66\n",
      "   74  27344 331L 134T N   O    2575 1017   10.79    8.32    2.47    5.66    8.32    2.47    5.66\n",
      "   75  27345 331L 135N N   O    2575 1025   12.29    9.46    2.82    6.15    9.46    2.82    6.15\n",
      "   76  27346 331L 136C N   O    2575 1031   13.76    9.92    3.85    7.80    9.92    3.85    7.80\n",
      "   77  27347 331L 137E N   O    2575 1040   11.44    7.20    4.24    8.94    7.20    4.24    8.94\n",
      "   78  27348 331L 138I N   O    2575 1048   10.24    6.03    4.21    9.46    6.03    4.21    9.46\n",
      "   79  27349 331L 139A N   O    2575 1053   13.51    9.13    4.38    9.33    9.13    4.38    9.33\n",
      "   80  27350 331L 140M N   O    2575 1061   14.54    9.52    5.02    9.63    9.52    5.02    9.63\n",
      "   81  27351 331L 141L N   O    2575 1069   12.36    7.04    5.32   10.71    7.04    5.32   10.71\n",
      "   82  27352 331L 142H N   O    2575 1079   13.91    9.85    4.06    8.20    9.85    4.06    8.20\n",
      "   83  27353 331L 282G N   O    2575 2171   14.40   10.10    4.29    5.35   10.10    4.29    5.35\n",
      "   84  27375 332A 134T N   O    2583 1017   12.95    9.97    2.98    5.26    9.97    2.98    5.26\n",
      "   85  27376 332A 135N N   O    2583 1025   13.90   10.02    3.88    5.76   10.02    3.88    5.76\n",
      "   86  27378 332A 137E N   O    2583 1040   12.41    7.41    5.00    8.62    7.41    5.00    8.62\n",
      "   87  27381 332A 140M N   O    2583 1061   14.41    7.81    6.59   12.50    7.81    6.59   12.50\n",
      "   88  27382 332A 141L N   O    2583 1069   11.61    4.61    7.01   15.09    4.61    7.01   15.09\n",
      "   89  27383 332A 142H N   O    2583 1079   12.85    7.04    5.82   11.50    7.04    5.82   11.50\n",
      "   90  27384 332A 143A N   O    2583 1084   16.47   10.07    6.40   11.46   10.07    6.40   11.46\n",
      "   91  27411 333Q 140M N   O    2588 1061   12.83    7.32    5.51    9.63    7.32    5.51    9.63\n",
      "   92  27412 333Q 141L N   O    2588 1069    9.75    3.51    6.24   11.85    3.51    6.24   11.85\n",
      "   93  27414 333Q 143A N   O    2588 1084   14.71    8.81    5.90   10.01    8.81    5.90   10.01\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9191  31G  32D C   N     225  227  -77.61  -96.50   59.39    2.50  -63.30   98.94   11.42\n",
      "    1         32D  32D N   CA    227  228   57.90  114.20                  -40.00\n",
      "    2   9378 218F 219D C   N    1680 1682  -76.86  -63.30   53.80    7.51  -63.30   53.80    7.51\n",
      "    2        219D 219D N   CA   1682 1683  -92.06  -40.00                  -40.00\n",
      "    3   9420 260A 261P C   N    2016 2018  -60.87  -58.70    4.07    0.51  -64.50  178.90   13.54\n",
      "    3        261P 261P N   CA   2018 2019  -33.94  -30.50                  147.20\n",
      "    4   9421 261P 262G C   N    2023 2025  -76.57  -62.40   20.71    2.90   82.20  162.50   11.97\n",
      "    4        262G 262G N   CA   2025 2026  -26.10  -41.20                    8.50\n",
      "    5   9436 276P 277Y C   N    2126 2128  -94.76  -98.40   23.32    2.44  -63.50  168.10   28.28\n",
      "    5        277Y 277Y N   CA   2128 2129  151.43  128.40                  -43.40\n",
      "    6   9437 277Y 278G C   N    2138 2140  -50.72  -62.40   23.85    4.18   82.20  136.03   10.02\n",
      "    6        278G 278G N   CA   2140 2141  -20.40  -41.20                    8.50\n",
      "    7   9452 292D 293K C   N    2248 2250  -59.35  -62.90   22.34    3.11   56.60  129.36   16.57\n",
      "    7        293K 293K N   CA   2250 2251  -18.75  -40.80                   38.60\n",
      "    8   9453 293K 294E C   N    2257 2259  -53.20  -63.60   16.27    2.72   54.60  128.63   16.81\n",
      "    8        294E 294E N   CA   2259 2260  -27.79  -40.30                   42.40\n",
      "    9   9490 330H 331L C   N    2573 2575  -73.93  -70.70   12.12    1.04  -63.50  171.44   23.23\n",
      "    9        331L 331L N   CA   2575 2576  129.92  141.60                  -41.20\n",
      "   10   9492 332A 333Q C   N    2586 2588  -69.01  -73.00   29.71    1.86  -63.80  151.66   22.03\n",
      "   10        333Q 333Q N   CA   2588 2589  111.26  140.70                  -40.30\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33718 142H 332A CB  CB   1074 2585   11.86    4.17    7.69    5.24    4.17    7.69    5.24\n",
      "    2  34969 324K 331L CE  CD1  2523 2579   10.74    4.30    6.44    4.60    4.30    6.44    4.60\n",
      "    3  34972 324K 331L NZ  CD1  2524 2579   12.11    4.10    8.01    5.79    4.10    8.01    5.79\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   15   23   95  191  265  262  336  412  426  444\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "openf___224_> Open           TvLDH.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13115071   12807.687    12.508\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16617\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18806\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    20098\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21731\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    23388\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25723\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    29083\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    30473\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    32123\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34005\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "iupac_m_486_> OE1/2 will be swapped:     -105.4395        3     3\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.8980       28    28\n",
      "iupac_m_485_> OD1/2 will be swapped:      179.5256       32    32\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.1488       36    36\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.1429\n",
      "iupac_m_486_> OE1/2 will be swapped:       91.3846       55    55\n",
      "iupac_m_485_> OD1/2 will be swapped:      123.1407       58    58\n",
      "iupac_m_485_> OD1/2 will be swapped:      115.1370       72    72\n",
      "iupac_m_485_> OD1/2 will be swapped:     -176.0906       79    79\n",
      "iupac_m_487_> NH1/2 swapped:     -178.0734       97    97\n",
      "iupac_m_485_> OD1/2 will be swapped:     -165.9727       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.9570      114   114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:       94.8655      165   165\n",
      "iupac_m_485_> OD1/2 will be swapped:      176.7082      179   179\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -95.2510      199   199\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -179.2678      202   202\n",
      "iupac_m_485_> OD1/2 will be swapped:     -179.5888      210   210\n",
      "iupac_m_487_> NH1/2 swapped:      179.5684      228   228\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.6968      234   234\n",
      "iupac_m_485_> OD1/2 will be swapped:      167.6302      292   292\n",
      "iupac_m_486_> OE1/2 will be swapped:     -171.3331      294   294\n",
      "iupac_m_485_> OD1/2 will be swapped:      120.8861      307   307\n",
      "iupac_m_486_> OE1/2 will be swapped:      117.8420      311   311\n",
      "iupac_m_485_> OD1/2 will be swapped:       98.3488      314   314\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34988   34988\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5817\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :       12095.7949\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      18.115       1.000\n",
      " 2 Bond angle potential               :    3625       0      3   1.913   1.913      260.74       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     33  45.643  45.643      554.33       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      0   1.151   1.151      31.787       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5817       0      0   0.002   0.002      3.6074       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    8285      66      3   0.628   0.628      2667.2       1.000\n",
      "10 Distance restraints 2 (N-O)        :    8728      68     11   0.683   0.683      2966.2       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       0      0   4.074   4.074      63.846       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      0  67.770  67.770      52.600       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      0  76.150  76.150      82.655       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      0  59.228  59.228      47.130       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  83.502  83.502      21.373       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    4983       0      0   0.416   0.416      3428.1       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      12     24  21.981  35.935     -22.277       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2556       3      2   0.945   0.945      1920.4       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990004\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   35543.2695\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13163  92K 231D CA  CA    690 1790   17.99   10.83    7.16   10.22   10.83    7.16   10.22\n",
      "    2  13183  93P 229A CA  CA    699 1771   16.28    9.93    6.35   11.20    9.93    6.35   11.20\n",
      "    3  13184  93P 230W CA  CA    699 1776   14.33    6.84    7.49   14.66    6.84    7.49   14.66\n",
      "    4  13185  93P 231D CA  CA    699 1790   17.53    8.59    8.94   15.03    8.59    8.94   15.03\n",
      "    5  13203  94G 187G CA  CA    706 1433   18.27   13.68    4.58    6.40   13.68    4.58    6.40\n",
      "    6  13204  94G 188E CA  CA    706 1437   18.51   12.73    5.78    8.23   12.73    5.78    8.23\n",
      "    7  13205  94G 227H CA  CA    706 1750   19.49   11.36    8.14   11.19   11.36    8.14   11.19\n",
      "    8  13206  94G 228R CA  CA    706 1760   21.36   13.09    8.27   11.17   13.09    8.27   11.17\n",
      "    9  13207  94G 229A CA  CA    706 1771   18.67   11.28    7.39   11.56   11.28    7.39   11.56\n",
      "   10  13208  94G 230W CA  CA    706 1776   16.68    7.81    8.87   15.52    7.81    8.87   15.52\n",
      "   11  13209  94G 231D CA  CA    706 1790   20.01   10.08    9.93   14.40   10.08    9.93   14.40\n",
      "   12  13210  94G 232I CA  CA    706 1798   20.70   11.87    8.83   13.06   11.87    8.83   13.06\n",
      "   13  13211  94G 233L CA  CA    706 1806   17.54    9.21    8.33   14.38    9.21    8.33   14.38\n",
      "   14  13212  94G 234E CA  CA    706 1814   18.47    8.27   10.19   16.40    8.27   10.19   16.40\n",
      "   15  13214  94G 236R CA  CA    706 1833   21.78   13.26    8.52   11.30   13.26    8.52   11.30\n",
      "   16  13215  94G 237G CA  CA    706 1844   18.43   10.86    7.57   10.34   10.86    7.57   10.34\n",
      "   17  13216  94G 238F CA  CA    706 1848   18.40   12.28    6.12    8.13   12.28    6.12    8.13\n",
      "   18  13217  94G 239T CA  CA    706 1859   18.11   12.15    5.96    8.73   12.15    5.96    8.73\n",
      "   19  13228  95Q 187G CA  CA    710 1433   15.82   11.42    4.40    8.14   11.42    4.40    8.14\n",
      "   20  13229  95Q 188E CA  CA    710 1437   16.65   11.24    5.41    9.61   11.24    5.41    9.61\n",
      "   21  13230  95Q 226G CA  CA    710 1746   19.92   13.81    6.11   10.90   13.81    6.11   10.90\n",
      "   22  13233  95Q 229A CA  CA    710 1771   17.66   10.81    6.85   13.30   10.81    6.85   13.30\n",
      "   23  13235  95Q 231D CA  CA    710 1790   19.97   11.57    8.41   13.66   11.57    8.41   13.66\n",
      "   24  13237  95Q 233L CA  CA    710 1806   17.64   10.12    7.52   15.08   10.12    7.52   15.08\n",
      "   25  13239  95Q 237G CA  CA    710 1844   19.12   12.31    6.81   10.37   12.31    6.81   10.37\n",
      "   26  13334  98A 130N CA  CA    737  981   11.90    7.85    4.05    7.54    7.85    4.05    7.54\n",
      "   27  13336  98A 132D CA  CA    737  996   13.58    9.85    3.72    6.78    9.85    3.72    6.78\n",
      "   28  13337  98A 133N CA  CA    737 1004   13.90   10.65    3.25    5.98   10.65    3.25    5.98\n",
      "   29  13341  98A 185N CA  CA    737 1415   14.98   10.87    4.11    7.61   10.87    4.11    7.61\n",
      "   30  13342  98A 186H CA  CA    737 1423   15.13   10.26    4.87    9.46   10.26    4.87    9.46\n",
      "   31  13343  98A 187G CA  CA    737 1433   13.53    8.20    5.34   10.96    8.20    5.34   10.96\n",
      "   32  13344  98A 188E CA  CA    737 1437   15.09    9.91    5.18    9.23    9.91    5.18    9.23\n",
      "   33  13345  98A 189S CA  CA    737 1446   15.58   11.51    4.06    6.88   11.51    4.06    6.88\n",
      "   34  13346  98A 190M CA  CA    737 1452   17.92   13.24    4.68    8.58   13.24    4.68    8.58\n",
      "   35  13347  98A 226G CA  CA    737 1746   19.52   14.00    5.53    9.41   14.00    5.53    9.41\n",
      "   36  13349  98A 229A CA  CA    737 1771   19.55   13.20    6.34   11.01   13.20    6.34   11.01\n",
      "   37  13351  98A 316T CA  CA    737 2448   16.00   12.90    3.10    5.68   12.90    3.10    5.68\n",
      "   38  13523 103S 334G CA  CA    772 2598   17.65   13.14    4.51    5.55   13.14    4.51    5.55\n",
      "   39  13558 104N 333Q CA  CA    778 2589   17.06   13.31    3.74    5.21   13.31    3.74    5.21\n",
      "   40  13632 106V 334G CA  CA    792 2598   14.91   10.50    4.40    6.08   10.50    4.40    6.08\n",
      "   41  13664 107I 334G CA  CA    799 2598   18.13   13.98    4.15    5.06   13.98    4.15    5.06\n",
      "   42  13744 109K 333Q CA  CA    818 2589   13.30    9.22    4.08    5.14    9.22    4.08    5.14\n",
      "   43  13809 111T 333Q CA  CA    835 2589   18.45   13.69    4.76    5.44   13.69    4.76    5.44\n",
      "   44  13850 112G 332A CA  CA    842 2584   12.97    9.25    3.72    9.09    9.25    3.72    9.09\n",
      "   45  13851 112G 333Q CA  CA    842 2589   16.14   10.97    5.17    9.92   10.97    5.17    9.92\n",
      "   46  13958 116S 332A CA  CA    875 2584   15.48   11.97    3.51    6.33   11.97    3.51    6.33\n",
      "   47  14808 140M 333Q CA  CA   1055 2589   13.19    9.31    3.87    6.49    9.31    3.87    6.49\n",
      "   48  14837 141L 333Q CA  CA   1063 2589    9.82    5.81    4.01    8.80    5.81    4.01    8.80\n",
      "   49  14857 142H 332A CA  CA   1071 2584    8.24    5.13    3.12    7.86    5.13    3.12    7.86\n",
      "   50  14858 142H 333Q CA  CA   1071 2589   11.72    5.92    5.80    9.35    5.92    5.80    9.35\n",
      "   51  14871 143A 332A CA  CA   1081 2584   11.00    8.67    2.33    4.67    8.67    2.33    4.67\n",
      "   52  14872 143A 333Q CA  CA   1081 2589   14.59    8.05    6.54   13.17    8.05    6.54   13.17\n",
      "   53  18687 326I 329N CA  CA   2537 2558    8.37    5.27    3.10    6.18    5.27    3.10    6.18\n",
      "   54  18688 326I 330H CA  CA   2537 2566    8.71    6.16    2.55    4.89    6.16    2.55    4.89\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21010  93P 229A N   O     698 1774   15.03    8.04    6.98   12.30    8.04    6.98   12.30\n",
      "    2  21011  93P 230W N   O     698 1788   14.95    6.68    8.28   14.33    6.68    8.28   14.33\n",
      "    3  21012  93P 231D N   O     698 1796   17.87    9.26    8.61   12.51    9.26    8.61   12.51\n",
      "    4  21026  94G 227H N   O     705 1758   18.43    9.79    8.64   11.35    9.79    8.64   11.35\n",
      "    5  21027  94G 228R N   O     705 1769   19.32   10.90    8.42   11.38   10.90    8.42   11.38\n",
      "    6  21028  94G 229A N   O     705 1774   16.59    8.66    7.93   12.74    8.66    7.93   12.74\n",
      "    7  21029  94G 230W N   O     705 1788   16.18    6.53    9.64   15.68    6.53    9.64   15.68\n",
      "    8  21030  94G 231D N   O     705 1796   19.30    9.76    9.54   12.61    9.76    9.54   12.61\n",
      "    9  21031  94G 232I N   O     705 1804   18.93   10.18    8.75   12.82   10.18    8.75   12.82\n",
      "   10  21032  94G 233L N   O     705 1812   16.55    7.51    9.04   15.19    7.51    9.04   15.19\n",
      "   11  21034  94G 237G N   O     705 1846   15.74   10.03    5.70    7.28   10.03    5.70    7.28\n",
      "   12  21035  94G 238F N   O     705 1857   17.52   10.42    7.11    9.16   10.42    7.11    9.16\n",
      "   13  21046  95Q 229A N   O     709 1774   16.95    9.38    7.57   14.81    9.38    7.57   14.81\n",
      "   14  21048  95Q 233L N   O     709 1812   17.77    9.47    8.29   16.61    9.47    8.29   16.61\n",
      "   15  21108  98A 185N N   O     736 1421   15.25   10.36    4.89    9.47   10.36    4.89    9.47\n",
      "   16  21109  98A 186H N   O     736 1431   12.34    7.08    5.26   10.65    7.08    5.26   10.65\n",
      "   17  21110  98A 187G N   O     736 1435   14.69    9.30    5.38   10.48    9.30    5.38   10.48\n",
      "   18  21111  98A 188E N   O     736 1444   15.82   10.81    5.01    8.27   10.81    5.01    8.27\n",
      "   19  21112  98A 226G N   O     736 1748   16.90   10.95    5.95   10.55   10.95    5.95   10.55\n",
      "   20  22455 139A 333Q N   O    1049 2596   12.76    9.79    2.97    4.97    9.79    2.97    4.97\n",
      "   21  22499 140M 333Q N   O    1054 2596   13.68   10.90    2.78    4.77   10.90    2.78    4.77\n",
      "   22  22566 142H 332A N   O    1070 2587    9.94    6.64    3.29    6.02    6.64    3.29    6.02\n",
      "   23  22567 142H 333Q N   O    1070 2596   11.52    5.78    5.75   11.67    5.78    5.75   11.67\n",
      "   24  22600 143A 332A N   O    1080 2587   12.05    8.38    3.67    6.93    8.38    3.67    6.93\n",
      "   25  22601 143A 333Q N   O    1080 2596   14.02    7.49    6.53   11.42    7.49    6.53   11.42\n",
      "   26  22628 144K 332A N   O    1085 2587   12.60    8.86    3.74    5.56    8.86    3.74    5.56\n",
      "   27  22629 144K 333Q N   O    1085 2596   15.23    7.86    7.36   11.26    7.86    7.36   11.26\n",
      "   28  23815 184G 275N N   O    1410 2120   13.16   10.28    2.87    5.75   10.28    2.87    5.75\n",
      "   29  23911 187G  95Q N   O    1432  717   14.83   10.65    4.18    7.84   10.65    4.18    7.84\n",
      "   30  23946 188E  95Q N   O    1436  717   13.65    8.76    4.89    8.99    8.76    4.89    8.99\n",
      "   31  24848 229A  95Q N   O    1770  717   16.44    9.29    7.15   14.00    9.29    7.15   14.00\n",
      "   32  24874 230W  94G N   O    1775  708   18.51    9.83    8.68   12.50    9.83    8.68   12.50\n",
      "   33  24896 231D  93P N   O    1789  704   18.82   10.06    8.76   12.51   10.06    8.76   12.51\n",
      "   34  24897 231D  94G N   O    1789  708   20.30   10.64    9.66   12.42   10.64    9.66   12.42\n",
      "   35  24898 231D  95Q N   O    1789  717   16.82    8.60    8.22   13.86    8.60    8.22   13.86\n",
      "   36  24913 232I  91L N   O    1797  688   15.35   10.23    5.12   10.08   10.23    5.12   10.08\n",
      "   37  24915 232I  95Q N   O    1797  717   18.27   10.29    7.98   14.53   10.29    7.98   14.53\n",
      "   38  24934 233L  95Q N   O    1805  717   16.85    9.35    7.50   15.01    9.35    7.50   15.01\n",
      "   39  26193 283V 331L N   O    2172 2582   13.73   10.94    2.79    5.05   10.94    2.79    5.05\n",
      "   40  26904 313L 275N N   O    2420 2120   13.72   10.67    3.05    5.47   10.67    3.05    5.47\n",
      "   41  26997 317E 275N N   O    2454 2120   13.31   10.14    3.17    5.19   10.14    3.17    5.19\n",
      "   42  27300 329N 325E N   O    2557 2535    5.74    2.87    2.87    4.69    2.87    2.87    4.69\n",
      "   43  27328 330H 326I N   O    2565 2543    6.11    2.97    3.13    6.72    2.97    3.13    6.72\n",
      "   44  27358 331L 326I N   O    2575 2543    7.72    4.93    2.79    5.63    4.93    2.79    5.63\n",
      "   45  27383 332A 142H N   O    2583 1079    9.32    7.04    2.29    4.52    7.04    2.29    4.52\n",
      "   46  27391 332A 328L N   O    2583 2556    7.32    3.20    4.12    6.37    3.20    4.12    6.37\n",
      "   47  27411 333Q 140M N   O    2588 1061   11.09    7.32    3.77    6.59    7.32    3.77    6.59\n",
      "   48  27412 333Q 141L N   O    2588 1069    8.69    3.51    5.18    9.84    3.51    5.18    9.84\n",
      "   49  27414 333Q 143A N   O    2588 1084   13.43    8.81    4.61    7.82    8.81    4.61    7.82\n",
      "   50  27430 334G 102S N   O    2597  770   15.19   10.61    4.58    5.31   10.61    4.58    5.31\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9191  31G  32D C   N     225  227  -81.58  -96.50   59.95    2.53  -63.30   97.86   11.09\n",
      "    1         32D  32D N   CA    227  228   56.14  114.20                  -40.00\n",
      "    2   9359 199F 200T C   N    1524 1526  -61.99  -63.20    9.30    1.12 -124.80  176.72    9.37\n",
      "    2        200T 200T N   CA   1526 1527  -51.33  -42.10                  143.50\n",
      "    3   9360 200T 201K C   N    1531 1533   50.76   56.60  109.30    7.37 -118.00  168.98    6.53\n",
      "    3        201K 201K N   CA   1533 1534  147.75   38.60                  139.10\n",
      "    4   9386 226G 227H C   N    1747 1749  -99.79  -63.20   72.37    7.70  -63.20   72.37    7.70\n",
      "    4        227H 227H N   CA   1749 1750   20.13  -42.30                  -42.30\n",
      "    5   9430 270P 271V C   N    2084 2086  -72.94  -73.50   17.76    1.51  -62.40  164.18   20.19\n",
      "    5        271V 271V N   CA   2086 2087  121.44  139.20                  -42.40\n",
      "    6   9432 272P 273E C   N    2098 2100 -143.79 -117.80  101.32    5.45  -63.60  112.69   13.46\n",
      "    6        273E 273E N   CA   2100 2101   38.87  136.80                  -40.30\n",
      "    7   9452 292D 293K C   N    2248 2250  -61.87  -62.90   26.20    3.46   56.60  129.87   16.63\n",
      "    7        293K 293K N   CA   2250 2251  -14.62  -40.80                   38.60\n",
      "    8   9453 293K 294E C   N    2257 2259  -50.84  -63.60   16.10    2.76   54.60  128.17   16.72\n",
      "    8        294E 294E N   CA   2259 2260  -30.49  -40.30                   42.40\n",
      "    9   9487 327A 328L C   N    2547 2549  -48.10  -70.70   23.16    2.24  -63.50  172.82   23.16\n",
      "    9        328L 328L N   CA   2549 2550  146.67  141.60                  -41.20\n",
      "   10   9488 328L 329N C   N    2555 2557   85.75   55.90   66.69    3.18  -63.20  150.42   23.64\n",
      "   10        329N 329N N   CA   2557 2558  -20.13   39.50                  -41.10\n",
      "   11   9490 330H 331L C   N    2573 2575 -111.79 -108.50   81.87    4.39  -63.50  115.11   18.97\n",
      "   11        331L 331L N   CA   2575 2576 -145.69  132.50                  -41.20\n",
      "   12   9492 332A 333Q C   N    2586 2588  -67.61  -73.00   25.79    1.58  -63.80  155.83   22.72\n",
      "   12        333Q 333Q N   CA   2588 2589  115.48  140.70                  -40.30\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33319 106V 331L CB  CD1   793 2579   12.01    4.50    7.51    5.13    4.50    7.51    5.13\n",
      "    2  33726 144K 333Q CB  CG   1087 2591   14.59    4.50   10.09    6.27    4.50   10.09    6.27\n",
      "    3  33727 144K 333Q CB  CD   1087 2592   13.33    4.80    8.53    5.24    4.80    8.53    5.24\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   20   23  116  217  254  219  341  432  444  434\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "openf___224_> Open           TvLDH.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13115071   12807.687    12.508\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16617\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18806\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    20098\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21731\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    23388\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25723\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    29083\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    30473\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    32123\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34005\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    34988\n",
      "iupac_m_486_> OE1/2 will be swapped:      173.4285        3     3\n",
      "iupac_m_486_> OE1/2 will be swapped:      147.0396       28    28\n",
      "iupac_m_487_> NH1/2 swapped:      179.0876       33    33\n",
      "iupac_m_485_> OD1/2 will be swapped:      160.9320       99    99\n",
      "iupac_m_486_> OE1/2 will be swapped:     -114.6090      113   113\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.6661      114   114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -161.3349      132   132\n",
      "iupac_m_486_> OE1/2 will be swapped:      150.4681      137   137\n",
      "iupac_m_487_> NH1/2 swapped:     -178.0002      161   161\n",
      "iupac_m_486_> OE1/2 will be swapped:      118.1198      165   165\n",
      "iupac_m_485_> OD1/2 will be swapped:      174.8963      179   179\n",
      "iupac_m_485_> OD1/2 will be swapped:       99.4489      215   215\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.3828      218   218\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -175.1240      228   228\n",
      "iupac_m_485_> OD1/2 will be swapped:     -131.4262      231   231\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       98.3342      315   315\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34988   34988\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5561\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :       12150.6123\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      16.935       1.000\n",
      " 2 Bond angle potential               :    3625       0      5   1.889   1.889      256.49       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     37  45.754  45.754      557.77       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      0   1.124   1.124      30.656       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5561       0      0   0.002   0.002      3.2018       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    8285     101      7   0.713   0.713      2693.6       1.000\n",
      "10 Distance restraints 2 (N-O)        :    8728     105     21   0.790   0.790      3019.6       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       0      1   4.048   4.048      63.190       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      0  70.089  70.089      50.518       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      0  70.686  70.686      68.616       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      0  75.856  75.856      53.576       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  87.171  87.171      21.298       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    4983       0      0   0.424   0.424      3423.9       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      14     21  20.768  37.695     -38.017       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    2556       3      5   0.887   0.887      1929.3       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990005\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   37351.2266\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13163  92K 231D CA  CA    690 1790   17.91   10.83    7.07   10.10   10.83    7.07   10.10\n",
      "    2  13183  93P 229A CA  CA    699 1771   16.22    9.93    6.29   11.10    9.93    6.29   11.10\n",
      "    3  13184  93P 230W CA  CA    699 1776   14.29    6.84    7.45   14.57    6.84    7.45   14.57\n",
      "    4  13185  93P 231D CA  CA    699 1790   17.49    8.59    8.90   14.96    8.59    8.90   14.96\n",
      "    5  13203  94G 187G CA  CA    706 1433   18.28   13.68    4.59    6.41   13.68    4.59    6.41\n",
      "    6  13204  94G 188E CA  CA    706 1437   18.47   12.73    5.74    8.17   12.73    5.74    8.17\n",
      "    7  13205  94G 227H CA  CA    706 1750   19.42   11.36    8.06   11.09   11.36    8.06   11.09\n",
      "    8  13206  94G 228R CA  CA    706 1760   21.33   13.09    8.24   11.13   13.09    8.24   11.13\n",
      "    9  13207  94G 229A CA  CA    706 1771   18.62   11.28    7.34   11.48   11.28    7.34   11.48\n",
      "   10  13208  94G 230W CA  CA    706 1776   16.65    7.81    8.84   15.48    7.81    8.84   15.48\n",
      "   11  13209  94G 231D CA  CA    706 1790   19.98   10.08    9.90   14.36   10.08    9.90   14.36\n",
      "   12  13210  94G 232I CA  CA    706 1798   20.69   11.87    8.82   13.04   11.87    8.82   13.04\n",
      "   13  13211  94G 233L CA  CA    706 1806   17.53    9.21    8.32   14.36    9.21    8.32   14.36\n",
      "   14  13212  94G 234E CA  CA    706 1814   18.51    8.27   10.23   16.46    8.27   10.23   16.46\n",
      "   15  13214  94G 236R CA  CA    706 1833   21.79   13.26    8.53   11.32   13.26    8.53   11.32\n",
      "   16  13215  94G 237G CA  CA    706 1844   18.62   10.86    7.76   10.60   10.86    7.76   10.60\n",
      "   17  13216  94G 238F CA  CA    706 1848   18.26   12.28    5.98    7.95   12.28    5.98    7.95\n",
      "   18  13217  94G 239T CA  CA    706 1859   18.06   12.15    5.92    8.67   12.15    5.92    8.67\n",
      "   19  13228  95Q 187G CA  CA    710 1433   15.81   11.42    4.39    8.11   11.42    4.39    8.11\n",
      "   20  13229  95Q 188E CA  CA    710 1437   16.60   11.24    5.36    9.52   11.24    5.36    9.52\n",
      "   21  13230  95Q 226G CA  CA    710 1746   19.83   13.81    6.03   10.75   13.81    6.03   10.75\n",
      "   22  13233  95Q 229A CA  CA    710 1771   17.53   10.81    6.71   13.04   10.81    6.71   13.04\n",
      "   23  13235  95Q 231D CA  CA    710 1790   19.87   11.57    8.31   13.50   11.57    8.31   13.50\n",
      "   24  13237  95Q 233L CA  CA    710 1806   17.51   10.12    7.39   14.83   10.12    7.39   14.83\n",
      "   25  13239  95Q 237G CA  CA    710 1844   19.19   12.31    6.88   10.47   12.31    6.88   10.47\n",
      "   26  13334  98A 130N CA  CA    737  981   12.11    7.85    4.26    7.93    7.85    4.26    7.93\n",
      "   27  13336  98A 132D CA  CA    737  996   13.85    9.85    4.00    7.29    9.85    4.00    7.29\n",
      "   28  13337  98A 133N CA  CA    737 1004   14.12   10.65    3.47    6.39   10.65    3.47    6.39\n",
      "   29  13341  98A 185N CA  CA    737 1415   15.15   10.87    4.28    7.93   10.87    4.28    7.93\n",
      "   30  13342  98A 186H CA  CA    737 1423   15.31   10.26    5.05    9.80   10.26    5.05    9.80\n",
      "   31  13343  98A 187G CA  CA    737 1433   13.70    8.20    5.51   11.31    8.20    5.51   11.31\n",
      "   32  13344  98A 188E CA  CA    737 1437   15.22    9.91    5.32    9.46    9.91    5.32    9.46\n",
      "   33  13345  98A 189S CA  CA    737 1446   15.71   11.51    4.20    7.10   11.51    4.20    7.10\n",
      "   34  13346  98A 190M CA  CA    737 1452   18.06   13.24    4.83    8.85   13.24    4.83    8.85\n",
      "   35  13347  98A 226G CA  CA    737 1746   19.61   14.00    5.61    9.56   14.00    5.61    9.56\n",
      "   36  13349  98A 229A CA  CA    737 1771   19.61   13.20    6.41   11.13   13.20    6.41   11.13\n",
      "   37  13351  98A 316T CA  CA    737 2448   16.09   12.90    3.18    5.84   12.90    3.18    5.84\n",
      "   38  13523 103S 334G CA  CA    772 2598   17.46   13.14    4.32    5.32   13.14    4.32    5.32\n",
      "   39  13632 106V 334G CA  CA    792 2598   14.38   10.50    3.87    5.35   10.50    3.87    5.35\n",
      "   40  13660 107I 330H CA  CA    799 2566   12.79    9.90    2.89    4.77    9.90    2.89    4.77\n",
      "   41  13703 108F 330H CA  CA    807 2566   12.98    9.20    3.78    6.85    9.20    3.78    6.85\n",
      "   42  13741 109K 330H CA  CA    818 2566   11.67    6.72    4.95   11.88    6.72    4.95   11.88\n",
      "   43  13742 109K 331L CA  CA    818 2576    8.82    6.85    1.97    4.95    6.85    1.97    4.95\n",
      "   44  13770 110N 330H CA  CA    827 2566   14.28    9.72    4.55    7.32    9.72    4.55    7.32\n",
      "   45  13806 111T 330H CA  CA    835 2566   16.79   12.19    4.60    8.38   12.19    4.60    8.38\n",
      "   46  13848 112G 330H CA  CA    842 2566   16.46   11.12    5.35    8.38   11.12    5.35    8.38\n",
      "   47  13850 112G 332A CA  CA    842 2584   15.46    9.25    6.21   15.16    9.25    6.21   15.16\n",
      "   48  13851 112G 333Q CA  CA    842 2589   14.08   10.97    3.11    5.96   10.97    3.11    5.96\n",
      "   49  13958 116S 332A CA  CA    875 2584   19.09   11.97    7.12   12.83   11.97    7.12   12.83\n",
      "   50  14532 133N 331L CA  CA   1004 2576   16.17   13.35    2.82    4.60   13.35    2.82    4.60\n",
      "   51  14579 134T 330H CA  CA   1012 2566   12.71   10.46    2.25    4.74   10.46    2.25    4.74\n",
      "   52  14580 134T 331L CA  CA   1012 2576   12.46   10.13    2.33    4.63   10.13    2.33    4.63\n",
      "   53  14620 135N 330H CA  CA   1019 2566   13.10   10.09    3.02    5.77   10.09    3.02    5.77\n",
      "   54  14621 135N 331L CA  CA   1019 2576   12.20   10.14    2.07    4.78   10.14    2.07    4.78\n",
      "   55  14662 136C 330H CA  CA   1027 2566   15.76   11.77    3.99    6.79   11.77    3.99    6.79\n",
      "   56  14663 136C 331L CA  CA   1027 2576   14.33   10.83    3.50    6.72   10.83    3.50    6.72\n",
      "   57  14664 136C 332A CA  CA   1027 2584   17.28   12.69    4.59    5.61   12.69    4.59    5.61\n",
      "   58  14710 137E 330H CA  CA   1033 2566   13.84    9.82    4.02    7.69    9.82    4.02    7.69\n",
      "   59  14711 137E 331L CA  CA   1033 2576   12.32    7.96    4.36    9.14    7.96    4.36    9.14\n",
      "   60  14712 137E 332A CA  CA   1033 2584   14.89   10.75    4.13    5.80   10.75    4.13    5.80\n",
      "   61  14746 138I 330H CA  CA   1042 2566   11.26    6.47    4.79    9.81    6.47    4.79    9.81\n",
      "   62  14747 138I 331L CA  CA   1042 2576    9.22    5.50    3.72    9.74    5.50    3.72    9.74\n",
      "   63  14776 139A 330H CA  CA   1050 2566   14.36    9.20    5.16    8.70    9.20    5.16    8.70\n",
      "   64  14777 139A 331L CA  CA   1050 2576   11.89    8.20    3.69    7.72    8.20    3.69    7.72\n",
      "   65  14778 139A 332A CA  CA   1050 2584   14.33    9.13    5.21    7.46    9.13    5.21    7.46\n",
      "   66  14806 140M 331L CA  CA   1055 2576   13.83    8.72    5.11   11.20    8.72    5.11   11.20\n",
      "   67  14807 140M 332A CA  CA   1055 2584   15.72    9.41    6.31   12.87    9.41    6.31   12.87\n",
      "   68  14808 140M 333Q CA  CA   1055 2589   13.28    9.31    3.96    6.64    9.31    3.96    6.64\n",
      "   69  14834 141L 330H CA  CA   1063 2566   14.18    8.57    5.61    7.97    8.57    5.61    7.97\n",
      "   70  14835 141L 331L CA  CA   1063 2576   11.35    5.55    5.80   13.87    5.55    5.80   13.87\n",
      "   71  14836 141L 332A CA  CA   1063 2584   12.64    6.24    6.39   12.51    6.24    6.39   12.51\n",
      "   72  14837 141L 333Q CA  CA   1063 2589    9.80    5.81    3.99    8.77    5.81    3.99    8.77\n",
      "   73  14855 142H 330H CA  CA   1071 2566   14.32    8.60    5.72    5.99    8.60    5.72    5.99\n",
      "   74  14856 142H 331L CA  CA   1071 2576   11.00    6.72    4.28    8.94    6.72    4.28    8.94\n",
      "   75  14857 142H 332A CA  CA   1071 2584   12.16    5.13    7.04   17.75    5.13    7.04   17.75\n",
      "   76  14858 142H 333Q CA  CA   1071 2589    9.92    5.92    4.00    6.45    5.92    4.00    6.45\n",
      "   77  14869 143A 330H CA  CA   1081 2566   18.07   12.09    5.98    5.62   12.09    5.98    5.62\n",
      "   78  14870 143A 331L CA  CA   1081 2576   14.77   10.19    4.58    9.71   10.19    4.58    9.71\n",
      "   79  14871 143A 332A CA  CA   1081 2584   15.77    8.67    7.10   14.23    8.67    7.10   14.23\n",
      "   80  14872 143A 333Q CA  CA   1081 2589   13.26    8.05    5.22   10.50    8.05    5.22   10.50\n",
      "   81  14881 144K 330H CA  CA   1086 2566   19.62   13.91    5.71    5.88   13.91    5.71    5.88\n",
      "   82  14882 144K 331L CA  CA   1086 2576   16.29   10.99    5.30    8.94   10.99    5.30    8.94\n",
      "   83  14883 144K 332A CA  CA   1086 2584   16.60    9.47    7.13   11.90    9.47    7.13   11.90\n",
      "   84  14941 148P 331L CA  CA   1120 2576   18.93   13.02    5.91    9.07   13.02    5.91    9.07\n",
      "   85  14942 148P 333Q CA  CA   1120 2589   17.35   13.03    4.33    5.66   13.03    4.33    5.66\n",
      "   86  18000 270P 331L CA  CA   2080 2576   19.58   13.86    5.72    5.17   13.86    5.72    5.17\n",
      "   87  18076 274G 313L CA  CA   2110 2421   16.43   13.09    3.33    4.79   13.09    3.33    4.79\n",
      "   88  18079 274G 317E CA  CA   2110 2455   15.39   11.35    4.04    5.57   11.35    4.04    5.57\n",
      "   89  18081 274G 320L CA  CA   2110 2481   17.74   14.07    3.67    5.03   14.07    3.67    5.03\n",
      "   90  18082 274G 321F CA  CA   2110 2489   17.49   13.22    4.27    5.34   13.22    4.27    5.34\n",
      "   91  18288 282G 331L CA  CA   2169 2576   16.47   10.82    5.65    7.42   10.82    5.65    7.42\n",
      "   92  18311 283V 331L CA  CA   2173 2576   16.31   10.73    5.58    9.95   10.73    5.58    9.95\n",
      "   93  18313 283V 333Q CA  CA   2173 2589   16.01   12.07    3.94    4.76   12.07    3.94    4.76\n",
      "   94  18335 284V 331L CA  CA   2180 2576   17.55   12.75    4.80    5.93   12.75    4.80    5.93\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21010  93P 229A N   O     698 1774   14.96    8.04    6.91   12.18    8.04    6.91   12.18\n",
      "    2  21011  93P 230W N   O     698 1788   14.89    6.68    8.22   14.22    6.68    8.22   14.22\n",
      "    3  21012  93P 231D N   O     698 1796   17.83    9.26    8.57   12.45    9.26    8.57   12.45\n",
      "    4  21026  94G 227H N   O     705 1758   18.49    9.79    8.70   11.43    9.79    8.70   11.43\n",
      "    5  21027  94G 228R N   O     705 1769   19.32   10.90    8.42   11.37   10.90    8.42   11.37\n",
      "    6  21028  94G 229A N   O     705 1774   16.58    8.66    7.92   12.72    8.66    7.92   12.72\n",
      "    7  21029  94G 230W N   O     705 1788   16.18    6.53    9.64   15.68    6.53    9.64   15.68\n",
      "    8  21030  94G 231D N   O     705 1796   19.31    9.76    9.56   12.63    9.76    9.56   12.63\n",
      "    9  21031  94G 232I N   O     705 1804   18.94   10.18    8.76   12.84   10.18    8.76   12.84\n",
      "   10  21032  94G 233L N   O     705 1812   16.58    7.51    9.07   15.24    7.51    9.07   15.24\n",
      "   11  21034  94G 237G N   O     705 1846   15.81   10.03    5.78    7.38   10.03    5.78    7.38\n",
      "   12  21035  94G 238F N   O     705 1857   17.50   10.42    7.08    9.12   10.42    7.08    9.12\n",
      "   13  21046  95Q 229A N   O     709 1774   16.89    9.38    7.51   14.69    9.38    7.51   14.69\n",
      "   14  21048  95Q 233L N   O     709 1812   17.74    9.47    8.27   16.55    9.47    8.27   16.55\n",
      "   15  21108  98A 185N N   O     736 1421   15.38   10.36    5.02    9.71   10.36    5.02    9.71\n",
      "   16  21109  98A 186H N   O     736 1431   12.48    7.08    5.40   10.94    7.08    5.40   10.94\n",
      "   17  21110  98A 187G N   O     736 1435   14.80    9.30    5.50   10.70    9.30    5.50   10.70\n",
      "   18  21111  98A 188E N   O     736 1444   15.86   10.81    5.05    8.34   10.81    5.05    8.34\n",
      "   19  21112  98A 226G N   O     736 1748   16.96   10.95    6.01   10.66   10.95    6.01   10.66\n",
      "   20  21226 103S  99D N   O     771  748    6.00    2.89    3.11    6.02    2.89    3.11    6.02\n",
      "   21  21433 109K 329N N   O     817 2564   13.36    9.50    3.86    5.29    9.50    3.86    5.29\n",
      "   22  21434 109K 330H N   O     817 2574   11.08    5.50    5.57    8.81    5.50    5.57    8.81\n",
      "   23  21460 110N 329N N   O     826 2564   14.61   10.08    4.53    5.02   10.08    4.53    5.02\n",
      "   24  21461 110N 330H N   O     826 2574   12.07    7.33    4.74    5.49    7.33    4.74    5.49\n",
      "   25  21528 112G 330H N   O     841 2574   15.42    9.21    6.21   12.35    9.21    6.21   12.35\n",
      "   26  22272 135N 330H N   O    1018 2574   14.08    9.70    4.38    8.39    9.70    4.38    8.39\n",
      "   27  22318 136C 330H N   O    1026 2574   15.65   10.12    5.53   10.18   10.12    5.53   10.18\n",
      "   28  22363 137E 330H N   O    1032 2574   15.11    9.17    5.94   10.52    9.17    5.94   10.52\n",
      "   29  22406 138I 330H N   O    1041 2574   12.60    6.29    6.31   12.62    6.29    6.31   12.62\n",
      "   30  22452 139A 330H N   O    1049 2574   13.42    6.50    6.92   13.80    6.50    6.92   13.80\n",
      "   31  22453 139A 331L N   O    1049 2582   12.01    9.04    2.97    4.68    9.04    2.97    4.68\n",
      "   32  22496 140M 330H N   O    1054 2574   15.54    8.16    7.38   12.13    8.16    7.38   12.13\n",
      "   33  22497 140M 331L N   O    1054 2582   13.62    9.94    3.67    5.34    9.94    3.67    5.34\n",
      "   34  22498 140M 332A N   O    1054 2587   15.50   10.79    4.70    8.85   10.79    4.70    8.85\n",
      "   35  22530 141L 330H N   O    1062 2574   14.62    6.86    7.76    8.08    6.86    7.76    8.08\n",
      "   36  22531 141L 331L N   O    1062 2582   12.15    7.93    4.21    8.24    7.93    4.21    8.24\n",
      "   37  22532 141L 332A N   O    1062 2587   13.59    8.62    4.97    5.85    8.62    4.97    5.85\n",
      "   38  22562 142H 328L N   O    1070 2556   13.59    9.90    3.69    5.09    9.90    3.69    5.09\n",
      "   39  22563 142H 329N N   O    1070 2564   14.88   10.02    4.86    6.94   10.02    4.86    6.94\n",
      "   40  22564 142H 330H N   O    1070 2574   13.63    6.05    7.58    7.31    6.05    7.58    7.31\n",
      "   41  22565 142H 331L N   O    1070 2582   11.31    6.65    4.66    8.29    6.65    4.66    8.29\n",
      "   42  22566 142H 332A N   O    1070 2587   12.49    6.64    5.85   10.69    6.64    5.85   10.69\n",
      "   43  22567 142H 333Q N   O    1070 2596    9.23    5.78    3.45    7.02    5.78    3.45    7.02\n",
      "   44  22598 143A 330H N   O    1080 2574   16.10    8.37    7.73    7.76    8.37    7.73    7.76\n",
      "   45  22599 143A 331L N   O    1080 2582   13.87    8.95    4.92    8.14    8.95    4.92    8.14\n",
      "   46  22600 143A 332A N   O    1080 2587   14.79    8.38    6.41   12.10    8.38    6.41   12.10\n",
      "   47  22601 143A 333Q N   O    1080 2596   11.75    7.49    4.26    7.44    7.49    4.26    7.44\n",
      "   48  22626 144K 330H N   O    1085 2574   18.23   10.88    7.35    5.53   10.88    7.35    5.53\n",
      "   49  22627 144K 331L N   O    1085 2582   15.61   10.20    5.42    5.88   10.20    5.42    5.88\n",
      "   50  22628 144K 332A N   O    1085 2587   15.90    8.86    7.04   10.47    8.86    7.04   10.47\n",
      "   51  22629 144K 333Q N   O    1085 2596   13.02    7.86    5.16    7.88    7.86    5.16    7.88\n",
      "   52  23815 184G 275N N   O    1410 2120   13.20   10.28    2.92    5.83   10.28    2.92    5.83\n",
      "   53  23911 187G  95Q N   O    1432  717   14.80   10.65    4.15    7.78   10.65    4.15    7.78\n",
      "   54  23946 188E  95Q N   O    1436  717   13.60    8.76    4.83    8.89    8.76    4.83    8.89\n",
      "   55  24848 229A  95Q N   O    1770  717   16.36    9.29    7.07   13.82    9.29    7.07   13.82\n",
      "   56  24874 230W  94G N   O    1775  708   18.52    9.83    8.69   12.52    9.83    8.69   12.52\n",
      "   57  24896 231D  93P N   O    1789  704   18.78   10.06    8.72   12.45   10.06    8.72   12.45\n",
      "   58  24897 231D  94G N   O    1789  708   20.33   10.64    9.69   12.46   10.64    9.69   12.46\n",
      "   59  24898 231D  95Q N   O    1789  717   16.78    8.60    8.18   13.79    8.60    8.18   13.79\n",
      "   60  24913 232I  91L N   O    1797  688   15.30   10.23    5.07    9.99   10.23    5.07    9.99\n",
      "   61  24915 232I  95Q N   O    1797  717   18.22   10.29    7.93   14.44   10.29    7.93   14.44\n",
      "   62  24934 233L  95Q N   O    1805  717   16.79    9.35    7.45   14.90    9.35    7.45   14.90\n",
      "   63  26193 283V 331L N   O    2172 2582   16.14   10.94    5.20    9.41   10.94    5.20    9.41\n",
      "   64  26904 313L 275N N   O    2420 2120   13.88   10.67    3.20    5.76   10.67    3.20    5.76\n",
      "   65  26997 317E 275N N   O    2454 2120   13.39   10.14    3.24    5.31   10.14    3.24    5.31\n",
      "   66  27313 330H 105S N   O    2565  790    9.89    7.23    2.67    5.57    7.23    2.67    5.57\n",
      "   67  27315 330H 108F N   O    2565  816   14.00   10.34    3.66    6.77   10.34    3.66    6.77\n",
      "   68  27316 330H 109K N   O    2565  825   14.08    9.87    4.20    6.54    9.87    4.20    6.54\n",
      "   69  27318 330H 135N N   O    2565 1025   13.60   10.77    2.82    5.47   10.77    2.82    5.47\n",
      "   70  27319 330H 137E N   O    2565 1040   12.81    9.36    3.45    6.44    9.36    3.45    6.44\n",
      "   71  27320 330H 138I N   O    2565 1048   12.12    7.93    4.19    7.66    7.93    4.19    7.66\n",
      "   72  27321 330H 139A N   O    2565 1053   15.46   11.13    4.33    6.81   11.13    4.33    6.81\n",
      "   73  27322 330H 141L N   O    2565 1069   14.34    9.47    4.87    7.63    9.47    4.87    7.63\n",
      "   74  27343 331L 109K N   O    2575  825   11.61    8.77    2.84    5.38    8.77    2.84    5.38\n",
      "   75  27344 331L 134T N   O    2575 1017   10.82    8.32    2.50    5.72    8.32    2.50    5.72\n",
      "   76  27345 331L 135N N   O    2575 1025   12.22    9.46    2.75    5.99    9.46    2.75    5.99\n",
      "   77  27346 331L 136C N   O    2575 1031   13.77    9.92    3.86    7.82    9.92    3.86    7.82\n",
      "   78  27347 331L 137E N   O    2575 1040   11.51    7.20    4.31    9.10    7.20    4.31    9.10\n",
      "   79  27348 331L 138I N   O    2575 1048   10.25    6.03    4.22    9.49    6.03    4.22    9.49\n",
      "   80  27349 331L 139A N   O    2575 1053   13.49    9.13    4.35    9.28    9.13    4.35    9.28\n",
      "   81  27350 331L 140M N   O    2575 1061   14.55    9.52    5.04    9.65    9.52    5.04    9.65\n",
      "   82  27351 331L 141L N   O    2575 1069   12.47    7.04    5.43   10.94    7.04    5.43   10.94\n",
      "   83  27352 331L 142H N   O    2575 1079   13.93    9.85    4.09    8.26    9.85    4.09    8.26\n",
      "   84  27353 331L 282G N   O    2575 2171   14.54   10.10    4.44    5.53   10.10    4.44    5.53\n",
      "   85  27375 332A 134T N   O    2583 1017   13.00    9.97    3.03    5.35    9.97    3.03    5.35\n",
      "   86  27376 332A 135N N   O    2583 1025   13.93   10.02    3.91    5.80   10.02    3.91    5.80\n",
      "   87  27378 332A 137E N   O    2583 1040   12.50    7.41    5.09    8.77    7.41    5.09    8.77\n",
      "   88  27381 332A 140M N   O    2583 1061   14.46    7.81    6.65   12.60    7.81    6.65   12.60\n",
      "   89  27382 332A 141L N   O    2583 1069   11.75    4.61    7.15   15.39    4.61    7.15   15.39\n",
      "   90  27383 332A 142H N   O    2583 1079   13.01    7.04    5.97   11.81    7.04    5.97   11.81\n",
      "   91  27384 332A 143A N   O    2583 1084   16.59   10.07    6.52   11.68   10.07    6.52   11.68\n",
      "   92  27391 332A 328L N   O    2583 2556    6.93    3.20    3.73    5.77    3.20    3.73    5.77\n",
      "   93  27411 333Q 140M N   O    2588 1061   13.04    7.32    5.72    9.99    7.32    5.72    9.99\n",
      "   94  27412 333Q 141L N   O    2588 1069   10.05    3.51    6.54   12.42    3.51    6.54   12.42\n",
      "   95  27414 333Q 143A N   O    2588 1084   15.03    8.81    6.22   10.55    8.81    6.22   10.55\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9191  31G  32D C   N     225  227  -80.91  -96.50   56.30    2.37  -63.30  101.64   11.59\n",
      "    1         32D  32D N   CA    227  228   60.10  114.20                  -40.00\n",
      "    2   9258  98A  99D C   N     739  741  -75.36  -70.90   15.23    0.56  -63.30  155.61   19.96\n",
      "    2         99D  99D N   CA    741  742  164.86  150.30                  -40.00\n",
      "    3   9259  99D 100L C   N     747  749   81.22   60.20   72.48    2.75  -63.50  144.73   25.03\n",
      "    3        100L 100L N   CA    749  750  -39.77   29.60                  -41.20\n",
      "    4   9386 226G 227H C   N    1747 1749  -97.96  -63.20   74.92    7.97  -63.20   74.92    7.97\n",
      "    4        227H 227H N   CA   1749 1750   24.06  -42.30                  -42.30\n",
      "    5   9430 270P 271V C   N    2084 2086  -92.26  -73.50   24.56    2.07  -62.40  168.41   19.75\n",
      "    5        271V 271V N   CA   2086 2087  123.35  139.20                  -42.40\n",
      "    6   9431 271V 272P C   N    2091 2093  -86.48  -64.50   33.04    2.04  -58.70  160.06   15.52\n",
      "    6        272P 272P N   CA   2093 2094  171.87  147.20                  -30.50\n",
      "    7   9433 273E 274G C   N    2107 2109  -57.74  -62.40   10.54    1.82   82.20  145.61   10.94\n",
      "    7        274G 274G N   CA   2109 2110  -31.75  -41.20                    8.50\n",
      "    8   9453 293K 294E C   N    2257 2259  -65.59  -63.60   14.30    1.80   54.60  138.36   18.13\n",
      "    8        294E 294E N   CA   2259 2260  -26.14  -40.30                   42.40\n",
      "    9   9460 300V 301E C   N    2311 2313  -78.92  -63.60   16.00    2.71 -117.80 -177.55    9.74\n",
      "    9        301E 301E N   CA   2313 2314  -44.94  -40.30                  136.80\n",
      "   10   9461 301E 302G C   N    2320 2322  -59.47  -62.40   11.86    1.92   82.20  146.74   10.97\n",
      "   10        302G 302G N   CA   2322 2323  -29.71  -41.20                    8.50\n",
      "   11   9487 327A 328L C   N    2547 2549  -70.89  -70.70   16.54    1.25  -63.50  160.82   22.76\n",
      "   11        328L 328L N   CA   2549 2550  158.14  141.60                  -41.20\n",
      "   12   9488 328L 329N C   N    2555 2557   76.85   55.90   80.58    4.27  -63.20  140.08   21.35\n",
      "   12        329N 329N N   CA   2557 2558  -38.31   39.50                  -41.10\n",
      "   13   9490 330H 331L C   N    2573 2575  -82.37  -70.70   13.55    1.37  -63.50  176.93   23.55\n",
      "   13        331L 331L N   CA   2575 2576  134.72  141.60                  -41.20\n",
      "   14   9492 332A 333Q C   N    2586 2588  -79.54  -73.00   35.04    2.43  -63.80  147.43   20.88\n",
      "   14        333Q 333Q N   CA   2588 2589  106.28  140.70                  -40.30\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33718 142H 332A CB  CB   1074 2585   12.08    4.17    7.92    5.39    4.17    7.92    5.39\n",
      "    2  34969 324K 331L CE  CD1  2523 2579   12.13    4.30    7.83    5.59    4.30    7.83    5.59\n",
      "    3  34972 324K 331L NZ  CD1  2524 2579   13.57    4.10    9.47    6.85    4.10    9.47    6.85\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   22   23   99  172  267  240  311  402  425  396\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "TvLDH.B99990001.pdb          12252.23242\n",
      "TvLDH.B99990002.pdb          12125.79297\n",
      "TvLDH.B99990003.pdb          12247.25293\n",
      "TvLDH.B99990004.pdb          12095.79492\n",
      "TvLDH.B99990005.pdb          12150.61230\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "from modeller.automodel import *\n",
    "\n",
    "env = environ()\n",
    "a = automodel(env, alnfile='TvLDH-mult.ali',\n",
    "              knowns=(('3d5tA'), ('6itkA'), ('5mdhA'),\n",
    "                     ('1bdmA'), ('7mdhA'), ('4h7pA')), sequence='TvLDH')\n",
    "a.starting_model = 1\n",
    "a.ending_model = 5\n",
    "a.make()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Compute DOPE Scores for all the models"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'\\nfrom modeller import *\\nfrom modeller.scripts import complete_pdb\\n\\nlog.verbose()    # request verbose output\\nenv = environ()\\nenv.libs.topology.read(file=\\'$(LIB)/top_heav.lib\\') # read topology\\nenv.libs.parameters.read(file=\\'$(LIB)/par.lib\\') # read parameters\\n\\nfor i in range(1, 6): #give the numbers as per the models generated ie. if 5 models, then give n+1 ie. 6\\n    # read model file\\n    code = \"TvLDH.B9999%04d.pdb\" % i\\n    mdl = complete_pdb(env, code)\\n    s = selection(mdl)\\n    s.assess_dope(output=\\'ENERGY_PROFILE NO_REPORT\\', file=\\'TvLDH.profile\\',\\n                  normalize_profile=True, smoothing_window=15)\\n'"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#do not run the following script, script will be updated in the next version\n",
    "\"\"\"\n",
    "from modeller import *\n",
    "from modeller.scripts import complete_pdb\n",
    "\n",
    "log.verbose()    # request verbose output\n",
    "env = environ()\n",
    "env.libs.topology.read(file='$(LIB)/top_heav.lib') # read topology\n",
    "env.libs.parameters.read(file='$(LIB)/par.lib') # read parameters\n",
    "\n",
    "for i in range(1, 6): #give the numbers as per the models generated ie. if 5 models, then give n+1 ie. 6\n",
    "    # read model file\n",
    "    code = \"TvLDH.B9999%04d.pdb\" % i\n",
    "    mdl = complete_pdb(env, code)\n",
    "    s = selection(mdl)\n",
    "    s.assess_dope(output='ENERGY_PROFILE NO_REPORT', file='TvLDH.profile',\n",
    "                  normalize_profile=True, smoothing_window=15)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Model evaluation - using DOPE potential"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     13306613   12994.739    12.690\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     13307141   12995.255    12.691\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     13355741   13042.716    12.737\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     13369041   13055.704    12.750\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13508293   13191.692    12.883\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13509893   13193.255    12.884\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13513873   13197.142    12.888\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13515873   13199.095    12.890\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13518873   13202.024    12.893\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13523373   13206.419    12.897\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13528945   13211.860    12.902\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13531745   13214.595    12.905\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13535945   13218.696    12.909\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13542245   13224.849    12.915\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13545017   13227.556    12.918\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13546417   13228.923    12.919\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           TvLDH.B99990001.pdb\n",
      "readlinef__W> File: TvLDH.B99990001.pdb, Line: 6\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13557428   13239.676    12.929\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13557504   13239.750    12.929\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13557675   13239.917    12.930\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13558525   13240.747    12.930\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13559783   13241.976    12.932\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13561687   13243.835    12.933\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13561727   13243.874    12.933\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13561817   13243.962    12.934\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13564673   13246.751    12.936\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13564733   13246.810    12.936\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13564868   13246.941    12.936\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13569152   13251.125    12.941\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13569240   13251.211    12.941\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13569438   13251.404    12.941\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13575864   13257.680    12.947\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13575996   13257.809    12.947\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13576293   13258.099    12.947\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13585915   13267.495    12.957\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13586115   13267.690    12.957\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13586565   13268.130    12.957\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13601015   13282.241    12.971\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13601315   13282.534    12.971\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13601990   13283.193    12.972\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13623648   13304.344    12.993\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13624096   13304.781    12.993\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13625104   13305.766    12.994\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13657608   13337.508    13.025\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13889480   13563.945    13.246\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     13900013   13574.231    13.256\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     14093450   13763.135    13.441\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     14096907   13766.511    13.444\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     14290344   13955.414    13.628\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     14329880   13994.023    13.666\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2604   2670      0         0      1136\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     14442336   14103.844    13.773\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2604   2670   3623      4313      1136\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     14552816   14211.734    13.879\n",
      "patch_______> segment topology patched using RTF:   335  ;  GLY  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2605   2671   3625      4314      1137\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2605   2671   3625      4314      1137\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     2605        0\n",
      "openf___224_> Open           ${LIB}/atmcls-mf.lib\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13555315   13237.612    12.927\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13555379   13237.675    12.927\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13555437   13237.731    12.927\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13555501   13237.794    12.928\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13555567   13237.858    12.928\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13555631   13237.921    12.928\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13555787   13238.073    12.928\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13555915   13238.198    12.928\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13556209   13238.485    12.928\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13556401   13238.673    12.928\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13556905   13239.165    12.929\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13557161   13239.415    12.929\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13558157   13240.388    12.930\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13558541   13240.763    12.930\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13559117   13241.325    12.931\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13561151   13243.312    12.933\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13562047   13244.187    12.934\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13566499   13248.534    12.938\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13567843   13249.847    12.939\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13577461   13259.239    12.948\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13579445   13261.177    12.950\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13600091   13281.339    12.970\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13603099   13284.276    12.973\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13649065   13329.165    13.017\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13653545   13333.540    13.021\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13754765   13432.388    13.118\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${LIB}/dist-mf.lib\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13755161   13432.774    13.118\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13755957   13433.552    13.119\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13756753   13434.329    13.119\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13757941   13435.489    13.121\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13758341   13435.880    13.121\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13758941   13436.466    13.122\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13759841   13437.345    13.122\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13761189   13438.661    13.124\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13761389   13438.856    13.124\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13763413   13440.833    13.126\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13763713   13441.126    13.126\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13766749   13444.091    13.129\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13767197   13444.528    13.129\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13771749   13448.974    13.134\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13772421   13449.630    13.134\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13779249   13456.298    13.141\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13780257   13457.282    13.142\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13790501   13467.286    13.152\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13791101   13467.872    13.152\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13791501   13468.263    13.153\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13793013   13469.739    13.154\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13808377   13484.743    13.169\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13809277   13485.622    13.170\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13809877   13486.208    13.170\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13812145   13488.423    13.172\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13835193   13510.931    13.194\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13836537   13512.243    13.196\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13837437   13513.122    13.196\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13840841   13516.446    13.200\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13875413   13550.208    13.233\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13877429   13552.177    13.235\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13878777   13553.493    13.236\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13883881   13558.478    13.241\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13935737   13609.118    13.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13938761   13612.071    13.293\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13940785   13614.048    13.295\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     13948441   13621.524    13.302\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14026225   13697.485    13.376\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14030761   13701.915    13.381\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14033797   13704.880    13.384\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14045281   13716.095    13.395\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14161957   13830.036    13.506\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14168761   13836.681    13.512\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14173313   13841.126    13.517\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14190541   13857.950    13.533\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14365557   14028.864    13.700\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14375769   14038.837    13.710\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14382597   14045.505    13.716\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14408437   14070.739    13.741\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14670961   14327.110    13.991\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14686273   14342.063    14.006\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14696517   14352.067    14.016\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     14735277   14389.919    14.053\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     15129061   14774.474    14.428\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     15152029   14796.903    14.450\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     15167393   14811.907    14.465\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     15225533   14868.685    14.520\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     15816209   15445.517    15.084\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     15850661   15479.161    15.116\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     15873709   15501.669    15.138\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     15960921   15586.837    15.222\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     16846937   16452.086    16.066\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     16898621   16502.559    16.116\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     16933193   16536.320    16.149\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17064009   16664.070    16.274\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     18393033   17961.945    17.541\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                                0       0       0       0   12561\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLY:OXT  GLY\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     19143397   18694.723    18.257\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19498521   19041.523    18.595\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19561041   19102.578    18.655\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19686081   19224.688    18.774\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19936161   19468.906    19.013\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     20436321   19957.344    19.490\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     21436641   20934.219    20.444\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     23437281   22887.969    22.352\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27438561   26795.469    26.167\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   529908\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -39547.6719\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "31 Non-bonded restraints              :  529692       0    512   6.869   6.869     -39548.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.profile\n",
      "# Energy of each residue is written to: TvLDH.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:     -12.2981\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -39547.671875\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "-39547.671875"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from modeller import *\n",
    "from modeller.scripts import complete_pdb\n",
    "\n",
    "log.verbose()    # request verbose output\n",
    "env = environ()\n",
    "env.libs.topology.read(file='$(LIB)/top_heav.lib') # read topology\n",
    "env.libs.parameters.read(file='$(LIB)/par.lib') # read parameters\n",
    "\n",
    "# read model file - the best ones\n",
    "mdl = complete_pdb(env, 'TvLDH.B99990001.pdb')\n",
    "\n",
    "# Assess all atoms with DOPE:\n",
    "s = selection(mdl)\n",
    "s.assess_dope(output='ENERGY_PROFILE NO_REPORT', file='TvLDH.profile',\n",
    "              normalize_profile=True, smoothing_window=15)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Energy of each residue is written to: TvLDH.profile\r\n",
      "# The profile IS normalized by the number of restraints.\r\n",
      "# The profiles are smoothed over a window of residues:   13\r\n",
      "# The sum of all numbers in the file:     -12.2981\r\n",
      "#COLUMNS S S  N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N N \r\n",
      "# RESID  RES            1           2           3           4           5           6           7           8           9          10          11          12          13          14          15          16          17          18          19          20          21          22          23          24          25          26          27          28          29          30          31          32          33          34          35          36          37          38          39     TOTAL\r\n",
      "\r\n",
      "     1  MET       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "     2  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "     3  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "     4  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "     5  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "     6  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "     7  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "     8  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "     9  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "    10  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "    11  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "    12  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "    13  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "    14  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "    15  GLN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "    16  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "    17  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "    18  TYR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "    19  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01\r\n",
      "    20  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "    21  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "    22  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "    23  TRP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "    24  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "    25  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "    26  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "    27  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "    28  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "    29  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "    30  TYR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "    31  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "    32  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "    33  ARG       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "    34  GLN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "    35  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "    36  TYR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "    37  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "    38  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "    39  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "    40  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "    41  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "    42  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "    43  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "    44  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "    45  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "    46  MET       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "    47  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01\r\n",
      "    48  ARG       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01\r\n",
      "    49  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.52E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.52E-01\r\n",
      "    50  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.55E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.55E-01\r\n",
      "    51  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.57E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.57E-01\r\n",
      "    52  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.57E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.57E-01\r\n",
      "    53  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.56E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.56E-01\r\n",
      "    54  MET       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.56E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.56E-01\r\n",
      "    55  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.54E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.54E-01\r\n",
      "    56  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.53E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.53E-01\r\n",
      "    57  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.51E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.51E-01\r\n",
      "    58  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.49E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.49E-01\r\n",
      "    59  CYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01\r\n",
      "    60  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.47E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.47E-01\r\n",
      "    61  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.46E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.46E-01\r\n",
      "    62  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "    63  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "    64  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "    65  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "    66  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "    67  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "    68  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "    69  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "    70  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "    71  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "    72  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "    73  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "    74  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "    75  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "    76  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "    77  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "    78  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "    79  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "    80  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "    81  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "    82  CYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "    83  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01\r\n",
      "    84  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "    85  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01\r\n",
      "    86  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "    87  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "    88  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "    89  MET       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "    90  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "    91  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "    92  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "    93  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "    94  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "    95  GLN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "    96  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "    97  ARG       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "    98  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "    99  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   100  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   101  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   102  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   103  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   104  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   105  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   106  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   107  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   108  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   109  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   110  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   111  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   112  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   113  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   114  TYR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   115  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   116  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   117  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   118  TRP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   119  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   120  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   121  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   122  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   123  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   124  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "   125  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "   126  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "   127  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   128  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   129  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   130  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   131  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   132  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   133  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01\r\n",
      "   134  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   135  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   136  CYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   137  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   138  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   139  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   140  MET       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   141  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   142  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   143  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   144  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   145  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   146  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   147  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   148  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   149  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   150  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   151  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   152  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   153  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   154  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   155  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   156  MET       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   157  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   158  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   159  GLN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   160  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   161  ARG       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   162  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   163  TYR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   164  TYR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   165  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01\r\n",
      "   166  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01\r\n",
      "   167  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.49E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.49E-01\r\n",
      "   168  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01\r\n",
      "   169  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.48E-01\r\n",
      "   170  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.47E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.47E-01\r\n",
      "   171  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.45E-01\r\n",
      "   172  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "   173  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   174  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   175  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   176  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   177  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   178  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   179  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   180  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   181  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   182  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   183  TRP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   184  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   185  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   186  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   187  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.26E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.26E-01\r\n",
      "   188  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   189  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01\r\n",
      "   190  MET       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   191  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   192  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   193  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   194  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   195  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   196  GLN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   197  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   198  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   199  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   200  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   201  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   202  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   203  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   204  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   205  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   206  GLN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   207  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   208  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   209  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   210  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   211  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   212  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   213  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   214  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "   215  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "   216  TYR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01\r\n",
      "   217  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01\r\n",
      "   218  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "   219  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   220  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   221  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   222  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   223  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   224  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   225  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   226  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   227  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   228  ARG       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   229  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   230  TRP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   231  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "   232  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01\r\n",
      "   233  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.44E-01\r\n",
      "   234  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.43E-01\r\n",
      "   235  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   236  ARG       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   237  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   238  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   239  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   240  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   241  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   242  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   243  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   244  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   245  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   246  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   247  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   248  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   249  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   250  GLN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   251  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   252  MET       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   253  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   254  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   255  TRP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   256  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   257  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   258  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   259  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   260  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   261  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   262  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   263  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   264  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   265  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   266  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   267  MET       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   268  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   269  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   270  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   271  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   272  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01\r\n",
      "   273  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   274  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   275  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   276  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   277  TYR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   278  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01\r\n",
      "   279  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   280  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01\r\n",
      "   281  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   282  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   283  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   284  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   285  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   286  SER       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   287  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   288  PRO       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   289  CYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   290  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   291  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.30E-01\r\n",
      "   292  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   293  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.28E-01\r\n",
      "   294  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   295  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   296  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   297  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   298  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.29E-01\r\n",
      "   299  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   300  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   301  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.32E-01\r\n",
      "   302  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.33E-01\r\n",
      "   303  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   304  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   305  VAL       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   306  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   307  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   308  TRP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   309  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   310  ARG       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   311  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.41E-01\r\n",
      "   312  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   313  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   314  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.39E-01\r\n",
      "   315  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   316  THR       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   317  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   318  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   319  ASP       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   320  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   321  PHE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.35E-01\r\n",
      "   322  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.36E-01\r\n",
      "   323  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   324  LYS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.38E-01\r\n",
      "   325  GLU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   326  ILE       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "   327  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.42E-01\r\n",
      "   328  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.40E-01\r\n",
      "   329  ASN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.37E-01\r\n",
      "   330  HIS       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.34E-01\r\n",
      "   331  LEU       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.31E-01\r\n",
      "   332  ALA       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.27E-01\r\n",
      "   333  GLN       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.21E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.21E-01\r\n",
      "   334  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.16E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.16E-01\r\n",
      "   335  GLY       0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.12E-01     0.0         0.0         0.0         0.0         0.0         0.0         0.0         0.0       -0.12E-01\r\n"
     ]
    }
   ],
   "source": [
    "cat TvLDH.profile"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Template Evaluation "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Following template evaluation script will be updated in the next version within few days"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'\\nfrom modeller import *\\nfrom modeller.scripts import complete_pdb\\n\\nlog.verbose()    # request verbose output\\nenv = environ()\\nenv.libs.topology.read(file=\\'$(LIB)/top_heav.lib\\') # read topology\\nenv.libs.parameters.read(file=\\'$(LIB)/par.lib\\') # read parameters\\n\\n# directories for input atom files\\nenv.io.atom_files_directory = \\'./:../atom_files\\'\\n\\nfor i in range(1, 6): #give the numbers as per the models generated ie. if 5 models, then give n+1 ie. 6\\n    # read model file\\n    code = \"B%02d0001.pdb\" % i\\n    mdl = complete_pdb(env, code)\\n    s = selection(mdl)\\n\\ns.assess_dope(output=\\'ENERGY_PROFILE NO_REPORT\\', file=\\'mult.profile\\',\\n              normalize_profile=True, smoothing_window=15)\\n'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"\n",
    "from modeller import *\n",
    "from modeller.scripts import complete_pdb\n",
    "\n",
    "log.verbose()    # request verbose output\n",
    "env = environ()\n",
    "env.libs.topology.read(file='$(LIB)/top_heav.lib') # read topology\n",
    "env.libs.parameters.read(file='$(LIB)/par.lib') # read parameters\n",
    "\n",
    "# directories for input atom files\n",
    "env.io.atom_files_directory = './:../atom_files'\n",
    "\n",
    "for i in range(1, 6): #give the numbers as per the models generated ie. if 5 models, then give n+1 ie. 6\n",
    "    # read model file\n",
    "    code = \"B%02d0001.pdb\" % i\n",
    "    mdl = complete_pdb(env, code)\n",
    "    s = selection(mdl)\n",
    "\n",
    "s.assess_dope(output='ENERGY_PROFILE NO_REPORT', file='mult.profile',\n",
    "              normalize_profile=True, smoothing_window=15)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"\\nfrom modeller import *\\nenv = environ()\\n\\n# Read in the alignment file\\naln = alignment(env)\\naln.append(file='ldh.ali', alignment_format='PIR', align_codes='ALL')\\n\\n# Convert the alignment to profile format\\nprf = aln.to_profile()\\n\\n# Write out the profile\\n\\n# in text file\\nprf.write(file='mult.profile', profile_format='TEXT')\\n\\n# in binary format\\nprf.write(file='mult.bin', profile_format='BINARY')\\n\""
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#do not run the following script, it will be updated soon\n",
    "\"\"\"\n",
    "from modeller import *\n",
    "env = environ()\n",
    "\n",
    "# Read in the alignment file\n",
    "aln = alignment(env)\n",
    "aln.append(file='ldh.ali', alignment_format='PIR', align_codes='ALL')\n",
    "\n",
    "# Convert the alignment to profile format\n",
    "prf = aln.to_profile()\n",
    "\n",
    "# Write out the profile\n",
    "\n",
    "# in text file\n",
    "prf.write(file='mult.profile', profile_format='TEXT')\n",
    "\n",
    "# in binary format\n",
    "prf.write(file='mult.bin', profile_format='BINARY')\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "                         MODELLER 9.25, 2020/09/03, r11894\n",
      "\n",
      "     PROTEIN STRUCTURE MODELLING BY SATISFACTION OF SPATIAL RESTRAINTS\n",
      "\n",
      "\n",
      "                     Copyright(c) 1989-2020 Andrej Sali\n",
      "                            All Rights Reserved\n",
      "\n",
      "                             Written by A. Sali\n",
      "                               with help from\n",
      "              B. Webb, M.S. Madhusudhan, M-Y. Shen, G.Q. Dong,\n",
      "          M.A. Marti-Renom, N. Eswar, F. Alber, M. Topf, B. Oliva,\n",
      "             A. Fiser, R. Sanchez, B. Yerkovich, A. Badretdinov,\n",
      "                     F. Melo, J.P. Overington, E. Feyfant\n",
      "                 University of California, San Francisco, USA\n",
      "                    Rockefeller University, New York, USA\n",
      "                      Harvard University, Cambridge, USA\n",
      "                   Imperial Cancer Research Fund, London, UK\n",
      "              Birkbeck College, University of London, London, UK\n",
      "\n",
      "\n",
      "Kind, OS, HostName, Kernel, Processor: 4, Darwin Girinaths-MacBook-Pro.local 19.6.0 x86_64\n",
      "Date and time of compilation         : 2020/09/03 08:25:47\n",
      "MODELLER executable type             : mac10v4-intel64\n",
      "Job starting time (YY/MM/DD HH:MM:SS): 2020/12/05 22:23:53\n",
      "\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAnYAAAFzCAYAAACzcUTXAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/Il7ecAAAACXBIWXMAAAsTAAALEwEAmpwYAABpIUlEQVR4nO3debgcdZkv8O97TvaV7CRkJRzQE8hGjDgIKDuIkNERwS3KXJUZcHcUr3dGHcXrONfrXEfBwRGMM7IJOjAQhRhFREUIZCc7SSAkJCH7vpzz3j/eKrtOp7q79qru/n6ep586Xb39uk511VvvbxNVBRERERHVv5a8C0BEREREyWBgR0RERNQgGNgRERERNQgGdkREREQNgoEdERERUYNgYEdERETUILrlXYAiGDp0qI4fPz7vYhARERHV9Nxzz72mqsP8HmNgB2D8+PFYsGBB3sUgIiIiqklENlZ6jFWxRERERA2CgR0RERFRg2BgR0RERNQg2MaugmPHjmHTpk04fPhw3kXJTa9evTB69Gh0794976IQERFRAAzsKti0aRP69++P8ePHQ0TyLk7mVBU7duzApk2bMGHChLyLQ0RERAGwKraCw4cPY8iQIU0Z1AGAiGDIkCFNnbEkIiKqNwzsqmjWoM7V7N+fiIio3jCwKzARwfvf//4/3z9+/DiGDRuGq666KtT7jB8/Hq+99lrs5xAREVGxMbArsL59+2LZsmU4dOgQAGDevHk45ZRTci4VERERFRUDu4K74oor8OijjwIA7rnnHlx//fV/fmznzp2YNWsWJk+ejHPOOQdLliwBAOzYsQOXXnoppk2bho9+9KNQ1T+/5j//8z8xc+ZMTJ06FR/96EfR0dGR7RciIiKi1DCwC0IknVsA1113He69914cPnwYS5YswRvf+MY/P/alL30J06ZNw5IlS/D1r38dH/jABwAAX/nKV/DmN78ZCxcuxNVXX42XXnoJALBixQrcd999+P3vf49FixahtbUVP/nJT5LfXkRERJQLDndScJMnT8aGDRtwzz334Morr+zy2FNPPYUHH3wQAHDhhRdix44d2LNnD5588kn87Gc/AwC87W1vw6BBgwAA8+fPx3PPPYc3vOENAIBDhw5h+PDhGX4bIiKiBtXZCXz2s0DfvsBXv5pbMRjYBeGpyszD1Vdfjc9+9rN44oknsGPHjj+vV59yuT1Z/Xq0qipmz56N//2//3d6hSUiImpGhw4B3/420Lt3roEdq2LrwA033IB/+Id/wFlnndVl/fnnn//nqtQnnngCQ4cOxYABA7qs/8UvfoFdu3YBAC666CI88MAD2LZtGwBro7dx48YMvwkREVGDOnjQln365FoMZuzqwOjRo/GJT3zihPVf/vKX8aEPfQiTJ09Gnz59MGfOHADW9u7666/H9OnTccEFF2Ds2LEAgPb2dnzta1/DpZdeis7OTnTv3h3f+973MG7cuEy/DxERUcMpSGAnftV5zWbGjBm6YMGCLutWrFiB17/+9TmVqDi4HYiIiAJYsQJobwfOOANYuTLVjxKR51R1ht9jrIolIiIiiqsgGTsGdkRERERxMbAjIiIiahDOLFHo3TvXYjCwq6LZ2x82+/cnIiIKjBm7YuvVqxd27NjRtMGNqmLHjh3o1atX3kUhIiIqvoIEdhzupILRo0dj06ZN2L59e95FyU2vXr0wevTovItBRERUfAzsiq179+6YMGFC3sUgIiKielCQwI5VsURERERxMbAjIiIiahDNHNiJyGARmScia5zloArPu1xEVonIWhG5xbP+n0VkpYgsEZGfi8hJnse+4Dx/lYhclsHXISIiombXzIEdgFsAzFfVNgDznftdiEgrgO8BuAJAO4DrRaTdeXgegDNVdTKA1QC+4LymHcB1ACYBuBzAbc77EBEREaWnycexuwbAHOfvOQBm+TxnJoC1qvqiqh4FcK/zOqjq46p63Hne0wDcrpvXALhXVY+o6noAa533ISIiIkpPk2fsRqjqFgBwlsN9nnMKgJc99zc568rdAOAXIV8DEfmIiCwQkQXNPKQJERERJaAggV1qw52IyK8AnOzz0BeDvoXPui6jBYvIFwEcB/CToK/580rVOwDcAQAzZsxozlGIiYiIKBmNHtip6sWVHhORrSIyUlW3iMhIANt8nrYJwBjP/dEANnveYzaAqwBcpKXpIaq+hoiIiCgVBQns8qqKfRjAbOfv2QAe8nnOswDaRGSCiPSAdYp4GLDesgA+D+BqVT1Y9r7XiUhPEZkAoA3AMyl9ByIiIiLT5IHdNwBcIiJrAFzi3IeIjBKRuQDgdI64GcBjAFYAuF9Vlzuv/y6A/gDmicgiEfm+85rlAO4H8AKAXwK4SVU7svtaRERE1JQKEtjlMqWYqu4AcJHP+s0ArvTcnwtgrs/zTqvy3rcCuDWZkhIREREF4A530qQZOyIiIqLG4WbsmnQcOyIiIqLGUZCqWAZ2RERERHExsCMiIiJqAB0dwJEj9nevXrkWhYEdERERURzejhPiN1dCdhjYEREREcVRkGpYgIEdERERUTwM7IiIiIgaREHGsAMY2BERERHFU5Ax7AAGdkRERETxsCqWiIiIqEEwsCMiIiJqEAzsiIiIiBoEAzsiIiKiBsHAjoiIiKhBMLAjIiIiahDuOHYc7oSIiIiozjFjR0RERNQgDhywJQM7IiIiojq3f78t+/XLtxxgYEdEREQUjxvY9e+fbznAwI6IiIgoHmbsiIiIiBrEvn22ZGBHREREVOdYFUtERETUIFgVS0RERNQg3KpYZuyIiIiI6hwzdkREREQNQJWBHREREVFDOHIEOH4c6NHDbjljYEdEREQUVYGydQADOyIiIqLoGNgBIjJYROaJyBpnOajC8y4XkVUislZEbvGs/2cRWSkiS0Tk5yJykrN+vIgcEpFFzu37GX0lIiIiakYF6hEL5JexuwXAfFVtAzDfud+FiLQC+B6AKwC0A7heRNqdh+cBOFNVJwNYDeALnpeuU9Wpzu3GNL8EERERNTlm7AAA1wCY4/w9B8Asn+fMBLBWVV9U1aMA7nVeB1V9XFWPO897GsDodItLRERENe3aBVxwAXDHHXmXJDsFmnUCyC+wG6GqWwDAWQ73ec4pAF723N/krCt3A4BfeO5PEJGFIvJbETmvUgFE5CMiskBEFmzfvj38NyAiIqKuHnkEePJJ4Nvfzrsk2SnQPLEA0C2tNxaRXwE42eehLwZ9C591WvYZXwRwHMBPnFVbAIxV1R0icjaA/xKRSaq694Q3Ur0DwB0AMGPGDC1/nIiIiEJauNCWq1YBBw4AffvmW54sFKwqNrXATlUvrvSYiGwVkZGqukVERgLY5vO0TQDGeO6PBrDZ8x6zAVwF4CJVVeczjwA44vz9nIisA3A6gAVxvw8RERHV4AZ2qsCSJcCb3pRvebLAqlgAwMMAZjt/zwbwkM9zngXQJiITRKQHgOuc10FELgfweQBXq+pB9wUiMszpdAERORVAG4AXU/sWREREZFRLgR0APP98fmXJUsGqYvMK7L4B4BIRWQPgEuc+RGSUiMwFAKdzxM0AHgOwAsD9qrrcef13AfQHMK9sWJPzASwRkcUAHgBwo6ruzOpLERERNa3164E9e0r3vUFeI2uWqthqVHUHgIt81m8GcKXn/lwAc32ed1qF930QwIPJlZSIiIgCcQO5IUOAHTuaL7Br8qpYIiJKQ2enVYkRZc2ten3PewARYOlS4KWXGn9/ZFUsERGl4oUXgD59gFtvzbsk1IwWLbLl+ecDbW3AsWPAuHHAzTfnWqzUFawqloEdEVGjeOQR4MgR4IEH8i4JNaMNG2zZ1gZ84hPAaGfugIcfzq1ImWBVLBERpcKtClu+3AI8oixt2WLLkSOBv/1bYONGG8du0yagkScCYFUsERGlwm2sfvw4sGxZvmWh5nL4sE0n1q0bMHSorWtpAaZOtb8buSMFM3ZERJS4ffuANWtK95tlDDEqhq1bbTlihAV0rmnTbNkMgR0zdkRElJjFi7v2PmzkEykVj1sNe3LZTKJuYNfIFxqsiiUiosS5gdyECV3vE2XB277Oa/p0Wzby/siqWCIix6FDNp8kxedmRD70IVsuXgx0dORXHiqGbduAhx4C5s4FDh6s/fyoXn3VluWBXXs70L27NRPYuze9z8+Laimw69s337I4GNgRUX6+/nVgyhQ76VA8bkbkoouAsWMtaF61Kt8yUf7e9jZg1ixbfu5z6X1OparYHj2AM8+0vxuxQ8/Bgxbc9e4NtLbmXRoADOyIKE9uMPKrX+Vbjnp35IgNcSJigbJb/dXI7Zqotv37geeeK91/8sn0PqtSVSwAnHKKLRtxyBM3C1qQbB3AwI6I8uSeDBiAxLNsmQ1xcsYZdoJphp6IVJvboeb0062n6gsv2LAkaahUFQsAgwbZcteudD47T25g16dPvuXwYGBHRPlxA7tFixp/Psk0uQGcG9A1Q4N1qs39/597rgX9HR02f2saqmXsTjrJlrt3p/PZeWJgR0Tk6Oiwht0AsGcPsH59vuWpZ27G0w3ovBk7BszNyw3spk9PP9iv1MYOKGXsGNhlgoEdEeXjtde69tpkdWx05Rm7UaOAYcPsROrO30nNx/1NTZuWbvV8Z2dpgGK/wM7N2DVyVWzv3vmWw4OBHRHlw73Cd7HaMJqODmtLBZRO3iLsQNHsyjvUpDlQsHuRNngw0LPniY83csbu0CFbMmNHRE3PbWztTj/EACSaVavs5DJunJ1YXexA0dyWLweOHbOOE/36lfaHJUuAr30N2Lgxuc+qVg0LNEfGjoEdETU992TwpjfZkgFINOXVsC538vVGHDuMaivfLwYNAtrarFfs3/898PnPJ/dZL71ky1Gj/B9v5IwdAzsiIoc3sBs40NrolFfPUm3eBvJeY8bYcvPmbMtDxeC3X9x7L/CpT9nfzzyT3GctWmTLs87yf5wZu0wxsCOifLhVsaNGNcdE4WnxNpD3coedYLDcnPz2i+nTgW9+E+jVy3qhJxVoVbq4cDFjlykGdkSUD2+7HLYHi0a1clWsG9ht3Wq9Fql5+HWocXXrBkyebH+7mba4Ku2DLmbsMsXAjojy4R3QlD04o9mwwbIgw4ef2L6pVy87oR47BuzcmUPhKDerV1vAMXYsMGTIiY8neSG1c6fth7162SDIfvr1s05SBw7Y/thIGNgRETm8UxAxYxeNN1MicuLjbi9FVsc2l1oZtCQHK3azflOmWDbQT0tLKWu3Z0/8zywSjmNHRASrQvRWxZ5xhl3xb9jQmNU1aal1Amc7u+ZUa79Isk1rrc9yNWp1LDN2Teqpp4BrrrGxg4gI2LfPDoi9ewMDBtiV/pQp9ljaWbvbbgP+/d/T/YyslE8lVs4N7NzsKAW3ahVw7bXAVVcB3/1u3qUJp9Z+cdZZQGsrsHJlKTCJ+1m1ArtG7UDBwK5J7doFPPww8Pvf510SomLwtq9zqxDdoRJWrEjvc197DbjpJuCjH7Xgst6tXGnLSsNMsCo2un/5F+CnPwUefRT4xCesfVi9cH9DlfaLXr2A9nbrVLNkSbzPWr3almeeWf15jZqx48wTTYpXzURdedvXuU45xZZpBiFuNjCJE1revNXZ7rYrx6rY6LzVlPW0v3R0lOZtrTRgMJBcu1Y3UBs6tPrzmLHLDAO7LPDgStSV3xREWfxOvCfreu+Bu3evZQv69gX69/d/Di8qozl+vBTIzZply3rp2LN9uwWiQ4YAPXpUfl5SgZ0bqLkZuUoaNWPHwK5JDR9u1U3bttkBg6jZeatiXVkEId6TWL2cqCvxy3qW40VlNCtX2tRbEyYAb32rrauX/SXIfgEkM8SQavDAjhm7zDCwy0L37pamVrXgjqjZ+Z18smgP1kiBXa2J172PMbALx9shoN5mRQmyXwCluYSXLo0+ttz+/Vb126dP9ewg0PgZu2Yf7kREBovIPBFZ4ywHVXje5SKySkTWisgtnvVfFZElIrJIRB4XkVGex77gPH+ViFyWxfcJhFUiRCXVqmLT+o3s22cNvbt1swz6smXAkSPpfFYW/LKe5Zixi8Y7RZYbAC1bBhw9mluRAguyXwDWG/200+w7vfBCtM9ys2+DfE/hXbmBHTN2qasZ2IlIHxH5exH5gXO/TUSuivm5twCYr6ptAOY798s/txXA9wBcAaAdwPUi0u48/M+qOllVpwJ4BMA/OK9pB3AdgEkALgdwm/M++eMBlqjE7+QzYoQtt261LEDS3CmWzjrLxs07fhxYvjz5z8lKkBP4SScBPXtaZmX//kyK1RC8Y7P17w+0tcULgLIUtCoWiN/Ozs2+1aqGBUrBX6Nm7OopsANwF4AjAN7k3N8EIO6AbNcAmOP8PQfALJ/nzASwVlVfVNWjAO51XgdV3et5Xl8A6nnfe1X1iKquB7DWeZ/8sUqEqMTv5NOjhzVZ6Oy0BuBJUQWuuw54+9vtvrd67eKLreeg93baaTb2ZNG527BalZtI6XHWFgTjnX/XbYdWTzOjBK2KBUrf62MfAy64oDR0R1BRMnYM7FIXJLCbqKrfBHAMAFT1EACfuWtCGaGqW5z32wJguM9zTgHwsuf+JmcdAEBEbhWRlwG8F07GrtZrvETkIyKyQEQWbE/yJFIJq2KJSiqdfNIIQjZuBO67z05CIhbgXX21/b1rl5XFe1u3Dpgzp+bb5i5oldtw5/CaxXGuEWzfbj2OTzqptD8mOQVX2oLuFwBw5ZV2QbV/P/Dkk+HHWg2TsRs92pbr14f7jKKr08DuqIj0hpMVE5GJsAxeVSLyKxFZ5nO7JmDZ/IJH/fMfql9U1TEAfgLg5iCv6bJS9Q5VnaGqM4YNGxawSDGwKpbIHD0K7Nhh80eW//bS+J24J+O3vtUmLJ81yzJ4O3YAr7zS9fbzn3d9TZEFPYE3am/EtLjb1TsGXD11oAhTFTtlig3afcMNdj/sfh8mY/e611mzgLVrLXBuBB0ddjwTse9WEEECuy8B+CWAMSLyE1ibuM/VepGqXqyqZ/rcHgKwVURGAoCz9OsqugnAGM/90QA2+zzvbgDvDPma7LEqlsi4g6eOGGHTGnmlEdi5J+M3vrFrZmHQoBOrYS+80B6L01MwK0GqYoHGbbSeFr/AyA3sFi2ypgJFFqYqFrA2hH/xF/Z32MA1TMaue/fS7BSLFoX7nKLyzjohcSsyk1M1sBORFgCDALwDwAcB3ANghqo+EfNzHwYw2/l7NoCHfJ7zLIA2EZkgIj1gnSIedsrV5nne1QBWet73OhHpKSITALQBeCZmWZPBqlgiU+3Ek0ZVbHl7qWqS6CmYlbAZu0Zr25QWv+06bJhVJR44AKxZk0+5gvDORhIkY+eKWtUcdAy7uJ9TVAUc6gSoEdipaieAm1V1h6o+qqqPqOprCXzuNwBcIiJrAFzi3IeIjBKRuc5nH4dVsT4GYAWA+1XV7cL2DadadwmASwF8wnnNcgD3A3gBlmW8SVVT6F4XAatiiUy1E0+aVbG1Jil31UND+SNHrFq5tbX2VE7M2IVT6cKjHvaLffssi9SnT+XZSPxMmmQZtdWrw/WeDlMVC9RXlXYQBWxfBwSrip0nIp8VkTHO+HODRWRwnA91AsWLVLXNWe501m9W1Ss9z5urqqer6kRVvdWz/p1Ote5kVX27qr7ieexW5/lnqOov4pQzUd6qWPVt9kfUHKq1AUo6sNu6Fdi82U5yp54a7DX1cPLxVme31DiMM2MXTqX9sx4CO29QGqZqsEcPC+5US8MCBRGmKhaoj20YRh0HdjcAuAnAkwCec24L0ixUQ+rXz25HjgB79uRdGqL8VMvYJV0V683W1QqAXG510bPPdu0tW6TAKEx1GzN24VTatklMwZW2KNWwrijVpGEzdpMn2+/whRfCD61SRPUa2KnqBJ9bwEtf6oLVsZSmj3/cquWKvn9Va2OX9G/EOzVUUO5zn366a8eKIUOAe+5JplxxhTmBM2MXTlGqYi+/3HqShpkdJU5gFyVTHTZj16ePfaeODpvJo97Va2AnIt1F5OMi8oBzu1lEumdRuIbDwI7Sogrce68N4TF/ft6lqW7HDlv6DTPkrnOfE1eYjhOu4cOB2bPtxO7eBgywbfzb3yZTrrjC9Hxkxi6cSlWxY8YAvXrZvnngQLpl2LULeOwxYNWqcFWjboef004L/5lZZOwA4PTTbblxY/DXFFW9BnYAbgdwNoDbnNvZzjoKiyPAU1o2by4NQFv09ivVetINGGDLvXuTGVYibMcJ149+1LUa9s47bX1RLsrCjFXGjF04lbJe3lk80t4PvMOBhPk9u9m2MBcyrsmT7TsuXx48Sxg2Ywc0VoKjjgO7N6jqbFX9tXP7EIA3pF2whtRIOzQVi/fgX/TAzj0Z+F3lt7YCAwdadixuW9Q9e2wWiZ49rfonjqINV8Q2dunYt8+ycb17ly4yvLLaD6L+nqNeyADWBvz00238xqBzKEfJ2BXttxRHHQd2Hc5sEwAAETkVQDGGEKk3DOwoLd52MQsXFrvnda2xr5IKRNysx+TJNpRDHEUbYDxMVSxnngjOO+izX6/SrPYD7+85aJu3V1+1coXpAV4uTHXssWM2NEpLiwWFQRXttxSH2wGknsaxc/wdgN+IyBMi8lsAvwbwmXSL1aAaaYemYvEeiHfvBjZsyKsktVXL2HnXx606jNJxohJvlqEIQXOYqtiBA225e3fxZ03IW61MaFYX597f85IlwWZBidIDvFyYDhRuRn3gwHCf10gJjnrN2KnqfNgMDh93bmeo6m/SLlhDaqQUNBWLe1AfP77r/aLp7Ox6QvCTVMYuSseJSnr3tvIeO2YDA+ctTFVs9+5A37627cMMPtuMagXMWRzDDx4EVq60ZgmnnGLt3VaurP26ONWwrjA9f2tdoFXSSOfBeg3sROQmAL1VdYmqLgbQR0T+Nv2iNaBGulIJ69gxm3+zCNmOsLZtA156Ke9S+Nu92yau37jRgo9rr7X1P/tZMedj3LfPAox+/SpXj7qBXdyMXRInOq+iZNw7O7sOUBxEo3Wg2LDBJq9PWq0q7rT2gc5O4He/Ax59FPjhD+3+618PnHOOPR4k0IrTccLl/lYWL7YhSaoJO52Yqyi/oyTUa2AH4MOqutu9o6q7AHw4tRI1skbaocP6p3+ytk6PPpp3ScK78EJg6lTrqVk073kP8I532N9TpgBvcPo1/eQndpAO2gg6K0FOBkm0CTt0CFixwrIeZ50V/X28inJhtmMHcPy4badevYK9ppE6UGzfbv/Tq65K/r29bez8pLUP/Od/Auefb9/p4x+3ddOnh2vzlsSFzJAhwNixFrCsXl39ue6QRGEzdiNGWPvFbdtsP65n7rA3ddjGrkWk1IpURFoB9EivSA1syBCgWzc7uB4+nHdpsvX447Z87rl8yxHWli0WHO3aVbzqzc5O4Mkn7e+rrgK+9jVb/o//AYwbZ+t/97v8yucnyPAISWTsli61jMPrX5/cQbcoVUhRBqFtpIzdn/5kVcp/+lPyF1tuNXul+XfTCuzc8RGnTAGuvNIu1j772eBt3nbvBl58MZke4EGDSXeAYXdcuqC6dbPxKlVLQzTVq1rNSnISJLB7DMD9InKRiFwI4B4Av0y3WA2qpaU5x7Lr7CwdJPLOdoTlPbgVbSqhNWvsinH0aOC//xu46CLL4PzgB8AnPmHPKVqZgwyPkETGLsmOE66iZNyjBHaNlLHz7tNJNzeo1W4sreDe/U7f/a7Vajz4oGUl3f130aLqHV+S7AEetJ1dnKrfovyW4ooy3EsGggR2nwcwH8DfwOaMnQ/gc2kWqqE1yg4dxrp1pUbb9RbQFnl8uGqdA6KMIp+FIFWxSWTskuw44SpKVWyYoU5cjTTkSZq/yVr757BhVo24fXty1YhHj1qtgIhl7LxOPtn2u717gfXrK79Hku1Jg2YJ43xmUX5LcUVtZ5iyIL1iO1X1+6r6V7C2dX9UVY5jF1VRqnOy5D1A1NsPOcp4UlmplpWaOtWWS5cGGyohK0F60iURhCTdcQIozskozFAnrqQ6pBRBmln0Wk0FunWzKedUSx1Y4lq+3H6jbW02Bl25IIFWkhcy3ovCSp3d9u+36c66dQPOPDP8ZzTKeTBqz+CUBekV+4SIDBCRwQAWAbhLRP5v6iVrVEU5OWTJeyCut+/tLfvKlaUBKYug2sF84EBg4kQbKmHFimzLVU0WGbtjx2zsL6AU4CahKCejOG3s6j1jt2NH1zlG08rYVTtRJ70f1Go2EKRqNMmmB6NGWWZy167K87kuWWJB36RJ1q4vrEY5D9Zrxg7AQFXdC+AdAO5S1bMBXJxusRpYo+zQYXgPSEUZ4DWIXbus+qNXL2uQ3NFhGbAiUE3mhJC1LDJ2K1daQDtxYrKNmovSjCJKVWyjZOy8bclaWmzS+yQ7ogU5USe9H9TKtrnrK2XsDh0qjXuXRA9wkdpNOeJmxIvyW4qrXjN2ALqJyEgA1wJ4JOXyNL5m6zzhDUBaWoozwGstv/89cMst9vfkyaVhRIpSHfvyy7YdhwyxzhN+ihjYZZGxS6MaFsjnoqyzE/iP/+ha7RelKrZRMnbu//bcc5O/2FIN1ms76f2g1v7q/R37XRSn0QPc/czbb7ehk8rFHTOvKNnvOIIMtp6TIIHdP8J6xq5V1WeduWLXpFusBtZsGbtXXrGBRAcNKnWLL/p37+wErr4auOMOuz9zZvGCJHeogalT/ee0dB8DStWSRRDkCjduD840esQCVuYePUoTxWfhnnuAD3wA+PznS+vcwbJHjQr+PoMH2zKpdmF5ccdlnDIl+d/kgQMWIPXpY//nSpI8hnd0lLKQlfbX8ePtN7Ftm/9nprG/z5xpy3nzgPe978RtHPfiyd13izrwexB791qg3b+/tTUskCCdJ36qqpNV9W+d+y+q6jvTL1qDarbAznvQqZfvvm6dZcMGDwZuvRX44hdrV4dk7ZVXbDl2bOXnuI9t3px+eYIKM0Bx3Ixdkj1iAQugs864//GPtnz6aVu67Z569bKq5qAmTbLl4sXJli9r7qC4I0Yk3/M7aLVakvvAmjU2GPCYMZXHzhOp3oEijQz11VcD3/kOcPbZdv+ZZ0qPHT1qF5Z+vXiDcvfHZcvqd5DigravA4Jl7ChJzVYV6z3J1st3d8v8pjcB//N/WrmL1ss0SHVcEas7gpw8e/e2sbiOHAnffso7ZmLSGTsg+4sT97usXm09Eb1tzMJkCSZMsOqirVuLf2FVjXf/CTNhfRBBT9RJ7gNB99Vq2ck0LmRaW4GPfcxmtin/3Fq9eIM46STbJw8fDjYPbhEVdAw7gIFd9tzgZuvW2nPxNQLvgateMnZ+7Ue8vUyLcCAK0oB+8GALkPbsKU5v3iAnT5HoWbsXX7Sq0lGjgs+jGkaWwbK3mk7Vsm1Rq91qZX3qhXf/8TY1SCLrEzRjl+RxLGhbtUr/u7R6gFf73KQunIpWCxJWkPaYOWFgl7UePazBe0dHOpNYF433wFUvPaEqHbiKdGIMMuSFt+qwKNs86Mkzaju7NLN1QLbb062mcy1cGC87U7R2olF4sySDBiWb9QmasUuy5iHo/lqp2jmtHuAut1xLl5aC57gdJ8rfu173x3rO2InICBH5oYj8wrnfLiJ/nX7RGljRTrZpee01673Zp4+l7YtYNViu2jAiRToQBe0ZWbRtHvTkGTVjl1bHCVeWWWd3P3M7x3gDuyjfr0gXJlGVZ0mS/E1GydjFGbpJNfj/84wzrInCxo1dRxVI+0LGr8o0qc8s0vE0ijrP2P0I1ivW7YK1GsAnUypPcyjayTYt7g92yhRrs1EPVbHeXrzjxnV9rEhVB0HHMivSRcTRo5aBam0F+vWr/ty4GbukO064svztuvvZxc6woU89FW+8sqJOMxfUsWPWc7WlpdS2K8nvFPSio08fYMAA25/jjAv40ksWpA0dWnnIIldrq7WrBPxn3kgrsPO+9/PPB+vFG5T3f1dtHtyiqueMHYChqno/gE4AUNXjAJqgcViK6iHASUL5SbZIQUYl3gNl+TAiQSfkTptq8NkHirSveU+clYZocUXJ2AUZtDmutPfhu+8GZs2ymzt+2Ac/aNtr9Wrb79rbrVdsWGecYa/bsKE+Byr223+SzEKGGWw2iQDfm/mq9XsASsfRz362tI/cd1/Xx9LgvvfXvw687W21e/EGdfLJdqs1D25R1XnG7oCIDAGgACAi5wDYk2qpGl2RTrZpWr3alu5cgu5Jcdu2fMoTRLWMz4gRNk/kvn35DiGye7dlCwYMsOxBNUXKDocZHsAdd80d3iKIzZttcna/bGtS0vztqgI33QQ89JDdtmyxNrkXXlgadgIAzj8/2vt361YanqIes3Z+GRJvdV7ci60w+2cSAb471V/Q7Kv7f1+0qLSPvPqqBeszZkQvR9DPXbUKeOyxruviKlItSFgFztgF6S//aQAPA5goIr8HMAzAX6VaqkZXL8N+xOVe0bhXdm7j3j177CDcUsC+O7Xaj5xySmmg0FrVJ2kJM1dokS4iwmREovxGhg4Ffvtb+/8EyYBEkWagvH69nSyGDgV+8ANb19Zm2+KRR2xMu+7dgbe8JfpnTJsG/OlPtp9feGESpc6OX4Zk5EjbPq++atsvzNh+ld4/TMYuzu/KvTgMehy59lr7ruXNE04/PX72rJrzzgP+8IfS4NbduiUX2E2bBsyda/vju96VzHtmpcDj2NUM7FT1eRG5AMAZAATAKlUtwEBedaxIJ9s0lR+Iu3WztjH79ln6vYA/iJpVeSNH2kEoz6A8zFyhRar+Tjsj0rNnciecSoYPt6Bx2zbrJZjkiPPuRcWMGVbN5jVixInromjEDMm0acAvfmHbL05gF2b/TOIYHnZquJaWeEF9HG96UzrvW88dKOq5KlZEPgDgPQDOBjAdwPXOOoqqWQI7vwNx3Omi0uTtxetOf1auCIFSmBNCkapis86IpKF7d8uOqCbfpCDLhvCNdCJN6juFOVEnUesS5gKtUXkvNOL0MM5Dgatig9SFvcFzOw/AlwFcnWKZGl+zVcV6D5Rxp4tKU3kvXj9FCDjqtSo2SkakiL+RtLZp2j16AWvv2tpqvWuzmu82KZVOpEllIcOcqJPYB8L8jhtVrXlwi6yeM3aq+jHP7cMApgGoMkNybSIyWETmicgaZ+n7SxKRy0VklYisFZFbPOu/KiJLRGSRiDwuIqOc9eNF5JCzfpGIfD9OOVOT1DhIRVdvGbsgJ9YiBEphrvSHD7fltm35z3QSpY1dEQ/2aQWdaY9JBlhD+0mT7LjjzlhQL4qUsYt7HFANXxXbiOp5RpQ6z9iVOwigLebn3gJgvqq2AZjv3O9CRFoBfA/AFQDaYVXA7c7D/6yqk1V1KoBHAPyD56XrVHWqc7sxZjnT0b+/VfcdPGjtzYoqTiDQ2WmdJICuI6LXQ8au2om1CNnWMCeEHj2s6rCz03qM5ilMxm7ECDvob99evEnC0wg6t2yx/+uAATYgbJrqtTq20ok0qXlww5yo4x4H9u2z43/v3tHnW20U9bo/1nPGTkT+W0Qedm6PAFgF4KGYn3sNgDnO33MAzPJ5zkwAa1X1RVU9CuBe53VQ1b2e5/WFMxRL3fBO9VTEqibADnIDBlhj3ShZxb177XUDBnSt1ixyxi7IVDlFytgFvdIvQpmBcBm7bt2AYcPSacsWVxrb03tRkXZv8aQ7UPzwh1alNmbMibc3vCG5C4pKJ9Iksj7HjwP799u2rzV4NhB/H/D+htPqwV0v6jGwO3LE5t/u1g3o2zfv0pwgSJeu/+P5+ziAjaq6KebnjlDVLQCgqltEZLjPc04B8LLn/iYAb3TviMitAD4AG1PvrZ7nTRCRhQD2Avhfqvo7vwKIyEcAfAQAxo4dG+OrRDRypE1YvmVL5Yb6eVq0yK4oDx6MduCpdBAuasZO1QZuBWwg10qKECS54wMG3W/Hj7e5HlevTrear5awwwOcfHKp7c2oUbWfnxW3ejvJDGgW1bCuSZNsuXZtMu/3ne/YVFd+Nm0Cfv1r4N3vjv851TJqkyYBTzwR/Tt5980ggfXgwdaRZs8eO8H37h3u81gNW1KPPbXd8TUHDy5kYB6kjd1vPbffBw3qRORXIrLM53ZNwLL5ba0/p45U9YuqOgbATwDc7KzeAmCsqk6Djb93t4gMqPC97lDVGao6Y9iwYQGLlKAiBAjVxD3RVDoIFzVjd+iQDfrbs2f1QX+9mdY82kdu22bTnvXrB5x2WrDXTJ1qy7yviMNk7IDi/kbc8ie5D2fRI9aVZBvBw4eB5cstGFqzxqbJcm8f/GBynwNUvzCI+53CXnR4a12i7J/sEVtSaR7cIit4x5eKgZ2I7BORvZVutd5YVS9W1TN9bg8B2CoiI53PGQnAr65lE4AxnvujAfgN9383gHc6n3lEVXc4fz8HYB2AAqbDUPyq2LgnmkoHyjROikkIemDv3dva8xw7ls9ByA3Opk4NXmVXlCvisCfPovaMTePiJIsesa4k2wguW2ZtcV/3OrvQ8FbDuhceSQXm1S4M4l4EhL3o8H5mlP2z4IFBplpb629GlIL//yqeGVS1v6oOAPAvsM4Np8CCq88D+FrMz30YwGzn79nwb7P3LIA2EZkgIj0AXOe8DiLi7bxxNYCVzvphTqcLiMipsE4eL8YsazqKmo1wxT3RVDpQuifFolXF1ssYa1Eyqd42LHn2wg7b2LioPWOTbk6wa5fNmtCrlwVIaRs0yDLTe/daU4s4qu2PSf9Oql0YxN1XoswiEOf7sSq2q3prZ1fw/1+QS/7LVPU2Vd2nqntV9XY4GbIYvgHgEhFZA+AS5z5EZJSIzAUAVT0Oq2J9DMAKAPer6nL39U617hIAlwL4hLP+fABLRGQxgAcA3KiqxcztFjmwO3jQ5jFsbQ0+j2G5Rs3YAfkGHFECu7Fjbbu/9ppV4+Yl7PAARf2NJJ2xW7TIlpMnJzuTRSVJdt6qltlPOuNatIwdq2KTU2+BXcEzdkGOIh0i8l5Yr1QFcD2AWANiOdWlF/ms3wzgSs/9uQDm+jzPN7BU1QcBPBinbJkp6kkLsIb2nZ02mGnYRsGuesvYZT04aVRBeu6WE7Hnz59vr89jjlvVxqmKTTpjl2XHCdfIkdamacsW4NRTo79Ptcx+khdAtfafrNvYxf3MggcGmStKc5GgCh6YB8nYvQfAtQC2Ord3OesojiK3sfM70ezfb714g6q3jF2UwUmz/t/t3Wu9/nr0ANrbaz/fy/1f/uEP1uA9a/v3W1usPn2s/EEEDaBXrrSOL1lJOmOXR2CXRNB1/HhpkGO3g45XkhdABw7Y51Xaf4YNszan27db+9ew4rSxY1VsfGeeadnqVavSnxHl6FHbT+J8TsED8yC9Yjeo6jWqOlRVh6nqLFXdkEHZGluRM3blV+HPPGPj0YUZsqBSoNRIGbvNfn15UuRW2Z11lg21EIYbNPzTP9lJsNLwFGmJkhEJEnwcPWpVmAMHWs/mLPTvb0HEgQPRgohyWXaccCVx/Fm1yrb5hAn+vxs32Hrttfjbqdb+09paGoZm69bk399PnG3oHjsKGhhkrmfP0owoixen9zn791uGevhw22effjra+xQ8MK/WK/ZzzvJfReQ75bfsitighg61g9GOHdlmG4IobzfzutfZD27p0uAH6Hob7iRMxs7t7ffCC6kVx1eczM7ll1tWpVcvO7j96leJFq2muBmRSp0+li+3fXLs2OjNBsISSW4/VrWOE0C241kmkXWutT+2ttoMIkC0YMvLHaS62tBUcb5TnDZ2YT9v61brUd+/f/Xv02yyaGf3zDOldsbHjgGPPhrtfeq4KnaFs1wA4DmfG8UR9wozLceOWQAHlKpXBgwAJk600bZXrKj40i4qBUp9+1rK/dAhe7+iCJOxy6uXaZzMzuDB9vqvfKXre2UlSkakb187+R09WjmAitLmMAlJNSnYv780tdQA3yE305FEVWyQIZGSamcX5EQaJ4OWZcYuypBFzSCLwM5976FDbRmlTZ9q/VbFqup/O8s57g3AfwD4ufM3xVXE6tgVKyzgmjix6xyv7okz6I+uUqCUZLYjSWEydnn1Mk1iENu8JtyOkhEBagcGebRPA5JrUuANWLIcwT6JY0+QC42kjnFBTqRxgsgo+6ebjdy2Ldy82nnts0WXRQcK971vuMGWUYLI3bvtYnPAgOqD2ecoyFyxd4vIABHpC+AFAKtE5O/SL1oTKGJgV+mgEzYgqBYoFbEDRZiMndvLFMguQDp82Kp+W1qsTVlU7v9x8eJwJ6O4omREgNq/kSxnbPBKah/O68o/blWsarAAJamORkHaNMX5rCj7Z/fulvnp7Aw3n3FeWeaimzLFjq3LlqXXPMndZ9/5TgvMtmwJv78UvBoWCNYrtl1V9wKYBRt6ZCyA96dZqKZRxJ6xla7Cw6bJqwVKRexAEXbw3KzHXVq6tDTCf5yrxKFDbVaAgwdLc85mIWrGrtrJuqOj1NC6XjN2eTXCjltFun69zZN68snpZdG80q6KDfv7L//MMMdwZuz89e8PtLVZc6Dly2s/P6wDB6zDT2urXRxHnWqx4NWwQLDArruIdIcFdg+p6jF45mylGIqYsauUAXHvL1pkV6i1NHLGDsg+sEvyZJDHYKBRM3bVAoPVqy1AHTOm1GYmK/WesRsxwrIj27bZMCJhBd0f66UqNuzv3xX2++3ZA6xbZ71AX//6cJ/VDNI8Ni1ZYueu9nbrRBb1s+ogsAsyQPG/AdgAYDGAJ0VkHICac8VSAFEOekuWlLrKt7QAb3qTXenEdfgw8Pvfl4bUKD9gjxgBjBpln71unV1ZVXuvw4etqsIvu9QIGbs0qmJXrarcuzPJITGmTwceftje8z0hhqQ8eNDGwfMGAiLAOed0bY/pJ27Gzu83kscwIa402thlqVs365G5bRvwwAP2fUaPtvHE/KxbZ8eAfv3sftAq8Hqoio0yeLYrbDAZZ8iiZjBtGnDffckFdu557dgx4LHHbJ17vHCX8+aV/j777No9lQs+1AkQILBT1e8A8A5vslFE3ppekZpI2APR0qWlyZJds2YBP/95/LL83d8B3/2u/T1qVKlhsNe0aRbYLVpUPbDzHiT9GoQnPXJ/EsJesbe1WdD68svWiSJuxuh3vwPOPx/4+MeB//f/Tnw8ybZkUTtQfOxjwJ13nrj+4ovt4FhN3DZ2fr+RPKu06j1jB9jvfNs24Prr7b6IteMsn692+XI77lx7LXD33bYuaFDtfq+4nYyCbKeo2cGDB+3E37u3ZdLCCHsMz6vpQL1I+oL5lltOPJ662979rCeesBtgVbS1xtFz9+V6bmMnIiNE5Ici8gvnfjuA2amXrBmEvdr77W9tOW6cnUwB2yGTGHLDHeLknHOAb33L/znu1EObNlV/L3f4Fnc4l3JuhnH//nBlTEtnp83qAAQfcqK1tRRkJ3F16QZGfgFSrRH+w/L2cA6z77gHvwsuAC67DHirc323bFnt10Ztw1TtN5JXxwmg/tvYATb0zRVX2P9yzBjbF373uxOf95vfWHvGxx8v7S9Bg+ozzrDl8uXRO+t4h5eodjJ1/yd7Q1YoRb3oAMIHk+6xM840bo0s6c5d7jHr3HNtP7/+euC977V1kyZZQuOyy+zWo4cdZ3fWmF7ePVeWXwAVSJA2dj8C8BiAUc791QA+mVJ5mkvYg4J7MP3MZ+wgO2yYHZSSmEXALcMPfwhcd53/c4KWt9bVddECuz177OQxcKAFbEGFHQKmGvc9/KbUWbnSqhQqjfAf1imnWIZx1y7gpZeCvWbPHptSrmdPCz5/+Uur2nDbadU6CCfdhsnbKzOPqthGyNhdfTUwd679Lz/+cVvnty+763bssMDE7Uk4cKDtk9UMHmwXoocO2b4dxe7dNgRT//42tmElbjXxvn3hLliiNhMAwl+c10H7rFy5nbsOHADWrIn3XkeO2AWFiO3jv/ylZZzd2hUR4JvfLD3mHkfc6nI/qnXRqzlIYDdUVe8H0AkAqnocQIbjJDQw96CwdWuwDgneq2SRZBuaJtk4uVYWwnsALoKoV+xJbn/3PTo7S9m58seSykx5952gVR5+bYPCDPeQdFXsxo12Qh461ALVrCU1FmNRhk6oti971y1ceOJxKM57BxE0q9m9u114dHSEG/w8iYxd0KrYovy/iyyp4+ry5VbbcfrppXNO3M996aV8jzsBBQnsDojIEDg9YUXkHAB7Ui1Vs+jVyw4mx47VTv8ePWpVXiKlKsCkBps9cMCCrJ49qx/cgh7Eah283B9ZUTJ2Ua/Yk9r+27Z1bYNUfmBJIzMVNttY6So16D4RdRsPHmwN/Xfv7joXrHebZDmwryuJdqJHj1oWrKUl/6mlKlWBuccd1/PPh7/QiNtuKkwwFOXYEidjF7bWhRm72pIK7MJm1oIcz73vmcdxJ6Aggd2nATwMYKKI/B7AjwF8LNVSNZOgBwZ3Tsy2tlJVZlJVgd4r4mo7a6NWxUa9Yp80ybIEa9bE+y7l/7/yA0sabcnCBqWVTuZBs7hRt3FLi/94j3mPBZZExs5tizpiRLgmAGmoVGXqHndcCxdGP2FGPU6FCYai1AYk1cYuSPVvHfSozF1SHSiiXoBU20/zPu4EVDWwE5FWABc4t78A8FEAk1R1SbXXUQhBgyW/k3tSVzZBr4iDnsRrHYiLVhUb9Yq9Z08L7lRr96Sqxjt3pPc+YNWcaRxMombsyssQZP89dswC35aWaEPz+GUF8+w4ASSTsStatZzfPuG3b4bdH+POrRwmGIpy0RgnY9evn7X7O3zY2qFWc+SI1cx06wYMGRL+s5pFUnNxhz1GnHmm/W9WrjyxnbMrz3a9IVQN7FS1A8A1qnpcVZer6jJngGJKStDZJ/x2qIkT7UC2eXPp6j+KoFfEw4ZZZmHnzuptWNzvUi9VsXGu2ONcXR4/DvzbvwE//and/9CHbLl0qTXqfeklG+F/797aI/yHFWbfOXTIDnbuiO1eQQI794Q3cGC0Sc/9PiPvA6w3Yxf15FO07I17AvzhD4Gvf91uc+bYune/uzS8z/r11ozE7fFay6hR1kN+925gw4bw5YqSsQtzbInz+weCN0dwHx8xItrvoFmMHm1t2HbuDN65q1xHR6mtctDAznuhXt7O2ZX3BWVAQfau34vId0XkPBGZ7t5SL1mzCJqx87tKbmmJPi2KV9ATTEtLaXy7asFA0KrYomTs3PaNUa7Y3faOUabAefBB4MYbSweLCy+00eiPHQM+/3ng059OL/Xf0hJ8uBbvdGblgycHOanFyYgAFhgApYP81q0WkPbvbwFqHnr0sEDn+PHoFyjub6goGbuZM235m98AX/yi3Z580ta98Y3AjBml506fbtmNIOJ29Hr5ZVumVRUb5/cPBD+GFy2QLyqR0nnNHVokLHdWmrFjw2VHq+2nu3fbcadv3/yOOwEF+WX+hbP8R886BXBh8sVpQkEOCh0dlWeEmDbNxp5auBC4/PJoZQhTJTRypO3cW7bYj6acd8ypWlWxRcnYea+kw3Jfs2NH+Nf+6U+2vPhi4H3vs6qAH//Ybv/6r8Azz5SyImlkpqZPB556qva+U+0qNUj1fNyMiDsbglvd7a0ezDPzMWKEZa9efTVaFbM73lqtWTuycskltt+5M9u4Ro8G3vIW4PbbgXvusd94pSGRKnnvey04jDL2l3vsqzQrhleUqtg4v38gfBOVogTyReaeW6JORRd1SJJp04Af/ci/BsY9xg8fXviMa5CZJzjLRJqCZDzWrKk8J2YSDU3DVHXUOojt329l7d278smuaJ0n4vRUizOemRugfPKTwNveZn/PmGEHlzvvtEzF44/b+jRS/0E7UFSr9gxyYRI3Y1dezqI0YB450gK7LVuqz8RSibv/BxmKIQstLcDNN1d+vL0d+OpXo733+98f7XX79ln2pXt3qyarJcpFY9yeqmFHC2DGrrY48/4C0Y8R1doeRx2LMwehwk4ReSStgjStIDtwtYxJEh0owlQR1DqIeQ9elXrYFq3zRJwqkqgzEFQbYNc7q8WCBbZMM7Crte9U2/+CnNTiZuymTClNd3XkSHHaucSdB7VogV0RuVnaM8+06u9aohxb4laRsio2eVGnh3NFPUa4x5qlS22oH6+os+fkIGw+sbgj8tWrIDtwtYzJ619vjT7XravdK6uSsFWx3tdUeq9qBy939Pj9+5OZDi2uOFUkUTN269fb/2vECP9t5f1fBxnhP4r2djtZVtt3jh0rtXPxm87Me2FS6X8ZN2PXt69VSR8/bmOq5d1xwhU3q+AGH1GqcZtF2MxLlNqAuFWkrIpNXpyLpjiz0vTvD5x2mh33Xnih62ONkrETkRYRudazKoEh9qmLIDtwtYNb9+42GwBQfSqUapKsig1y8OrWzapqVbsOOpuXOFUkUTN2tU5Y5cPapDEYZpB9Z+VKy5Kdeqr/lWq/fnarNtxD3IwdUNoev/mNBaI9e9pFTZ7iZhWYsastbFupsBm7oIOzV5PU+J5UEueiaeNGO+YMG1bqeBVGpeZNjZKxU9VOADd77t+QeomazcCBdlDZt89/7Bzv3HS1goAo1bHHjwPbt1vgMHx47efXCkSDVjcUpTr20CELSLp3t0Faw4o67EWtE5Z3fZqZqVrj2QWp0qi1TyRxQHTL+aMf2dI7tVleWBWbvrAZu7Bt7IIOzl5N2OFOGNjVFueiKe7sEJXOp42SsXPME5HPisgYERns3lIvWbMQqb4Te+emGz3a/z3iBHbbtllAMnx4sOELglbF1qpuKErPWO+Ye1EOAt27W1VhZ2e4ILXWCau9vRS4pNmWzH3v734XeOc7gXe9C3jssRPLWS24rHV1ncQB0S2nO6xM3u3rAFbFRvWtb1kP3PKqrnLeSdzdNqe1hK2KTaJ6tNZ+sGkT8MEPlr4vq2Jr8wbLYZvrxO1cVel8WkcZuyDDnbhZups86xTAqckXp0mNHGkDd776qtXvewWZcNsdNLbWgdJP2ANbrYPY+vW2rBSEuooyll0S1SODBlm2dfduYMCAYK+pdfDp0QM47zwbjuTcc6OXrZY3v9mW69bZDbD9yA2gghwkawX7SVTFnn22Vd+7VffnnRf9vZLCqthonn4a+NWvgGeftQuYSpYtsxqF172u1C63lrA1AUn8/ocOtVqXXbtsXy/fz2+7rTTQ80knMWMXRO/eVpu1Z49t1zC1KXE7V1U6nyZxHMtIzYydqk7wuTGoS1K1YClIGxM3iIpyggl7YHPLunWrZanKlU9BVElRMnZJXLGHnTd0yxYL4gcOtLZrldx3n7V9S6PjhOuss2w8vQceAO6/33rkrlxpQ9Z4pzOrtv8FrYqNk7E76STrIfzAA8C8ecB73hP9vZLCqthowg6zE+YEHTZjl0T1aEtLKRjwa6vqfs8vfMHGpgzSu5eiZ8Tjdq5ya6927eo6w1ISx7GM1AzsRKSPiPwvEbnDud8mIlelX7QmUu3KP8jBzTsbhF+wVU3YA1uvXrZjHz9+4qC8e/cCa9fagavalThQnLHskjiwh5031Bv8Vqv+HTo0mw4CM2eWqmHb220fWrIEePFFy3yMHFl98NagVbFxr3Tb262cF19sAWjehg2zk/r27daLLqxmrYoNOk9xlEFmw14wJtVTtVL1nbeN9Ic/HG28w2YVJSP+6qv2/P79q180V+OdYcl70dZIGTsAdwE4itIMFJsAfC21EjWjalf+QdLK3mDrtdfCfXaUA1ulH5x7tRqkYXtROk8kURUTNmNXlHHY/Hh7hAUtZ60DcB1d6YbS2lrqcBRlruZmz9gtWlT9QjRKxi6PqligchZy82YL/E86CRg/Pt5nNJsoGfGkZqXxO6bV0XEsyDefqKrfBHAMAFT1EIAUxl5oYpUyHtu22YGhX78T296Vi1otFOXA5pa3/LPCHIgbqSo2asYu73HY/HgzD0HLWWvfq6Mr3dDiVMc2a2Dnjt24b1+pXWe5jo7S4MRFr4oFKmchg7SRJn9RqmKTOrb6/a7r6DgWJLA7KiK9YR0mICITARyp/pLqnJ6180RkjbP0DYFF5HIRWSUia0XkFp/HPysiKiJDPeu+4Dx/lYhcFqecmamU8Qhz9RG1PUKUA1ut8gY5EDdSVWzYjF1RpsTy4xfY1SpntX1Pta56k4UW9Xd35IhV33bvbg3vm02t6thVq6yjzLhx4RrO51UVe9ZZlsFdscLap7qKnJ0vuihVsUltb7/fdYMNd/IlAL8EMEZEfgJgPoDPxfzcWwDMV9U25/38grZWAN8DcAWAdgDXi0i75/ExAC4B8JJnXTuA6wBMAnA5gNuc9ym2SjtwmDYmUXvoJVkVG6a8jVQVGyZjt2uX9Rzu1SvahOhpczu9LF1qnSqAeFWxBw9aE4FevezWaKL+7po1W+eq1YEi6sVPXlWxvXtbe9jOztJMLUCxs/NFF6cqNqmMnbt/1NkFapBesfMAvAPABwHcA2CGqj4R83OvAeD0/8YcALN8njMTwFpVfVFVjwK413md69uwANM7yM01AO5V1SOquh7AWud9iq3SDhzm4JZ3Vezhw9Y93NtDrJpGqooNk7Fz2yFOnhxs3MCsDRhg1f5Hj5aGbqjVNmjIEP9eZEBdtUuJJOrvrtkDu6ADY4c9QbvDohw4ULsjWdjB2Wtxj9Nvfztwww1dp7Zixi4895j8858Db3xj7Xasu3dbh6+ePeNfNJf/rg8ftmNinVygBm1deAGAiwC8FUASA0iNUNUtAOAs/X5VpwB42XN/k7MOInI1gFdUdXHQ15QTkY+IyAIRWbB9+/Zo3yIpw4bZwWX7djvYuNautWWtHqZAtMyBanJVsRs3WruY8eOBPn1qv0cRqmL37rXv36NHdm3s3HH+ipitc7397aW/r7qqdtugSr3IgFJnniizetQDNyDYti3c65q1R6zLW+XvNwBt1ICotdWOP6pdq0T97Nxpz3MvTOJ629tsuX07cNddNg7fxo2WzTvjjPjv32ymTLHfx+HDNkzMQw9Vf36Yznu1lFfF1lG2Dgg23MltAG4EsBTAMgAfFZHvBXjdr0Rkmc/tmlqvdd/CZ52KSB8AXwTwD0Ff4/fmqnqHqs5Q1RnDhg0LWKSUdOtmJwjVrlcl7k4VZL67KG19du+2DMuAAcGCMZdfYBemrEAxqmKTOhCEydi5zylyoPOtb9kJacMG4Mc/DvaaShcWjT4/phvUB21f6Wr2jN24cbbttm8HXnml62NxM11BawOSzia/+912YeMOKH7XXbacMqUYw/PUm2HDrPPg3/+93a81PE6S1d7lx7M66jgBBMvYXQDgMlW9S1XvAnAlgLfUepGqXqyqZ/rcHgKwVURGAoCz9Lvc3QRgjOf+aACbAUwEMAHAYhHZ4Kx/XkROrvKa4itP/XZ0WBZApPoYYpVeH0TUaki/zwp7Ai9CVWxS1SRhTu71cOUnAowdayffoD35KgV2jT4/ZtiOM65mD+xEKo/9tmGDbc/hw6NN4h60NiCNk/WIEVZtCAD/8R+2ZPu66Pr1A976Vvs7aGCXRLV3+TmuzpqUBAnsVgEY67k/BsCSmJ/7MIDZzt+zAfjlWJ8F0CYiE0SkB6xTxMOqulRVh6vqeFUdDwvmpqvqq877XiciPUVkAoA2AM/ELGs2yjNu27dbG5GhQ4Nlk6JUxUbNpvhlB8OewIswpVhSBwL3xBCkKraOelaFUmkInKR6HRZVmP+9V7NXxQKVA7u4Q4QErQ1I6yLLDeTcZghsXxeP26Fr8eKuTZXKRW2X6ad80P8GzNgNAbBCRJ4QkScAvABgmIg8LCIPR/zcbwC4RETWwHq2fgMARGSUiMwFAFU9DuBmAI8BWAHgflVdXu1Nncfvd8r4SwA3qWpHxDJmqzwwS3oOVz9RsykDB1oD0v37S1fFYctbhIxdUgeCRsvYRcGq2HCva/aMHdB1MGyvuL/LoMeWtC6yygM5BnbxDBpk0yoePmzD4Pg5eNCGmmlttaY1cfXsac1l3EH/6yxjF6TFqF9btlhUdQesM0b5+s2wql73/lwAc2u81/iy+7cCuDWRgmapPPUb9oToBlsHDtgBLcgJI2o2RcTKtX69lfe008KXN+/OE2F78VbDjF3zBnZRM3YM7IJl7KIIemxJ6yLrjDOsw8ShQ9Z++swzk33/ZjRtmp1vnn8emDTpxMeXLrXM2qRJtu2TMHKkdbDZsqXxMnaq+ttqtywK2RTKM25hT4husOV9bS1xTrpxy5t354lly6wd4xlnhOs44qdfP7tSPHCg9pyhjZqxc/eHl1+2oNnV6G3s4mbsmrkq9vTT7bf30ktd552OG9gFPbakdZHV2modJgALNJpxAOqkVboIcKUxXqB7TFuzpnR+q5ML8hiTqVGi3BPfZqevR5IzQlQS56Qbt7x5B3ZJtscQCZ65qbMrv8Dc//tjj1kVxhKnGW6jt7Hr08eyMocOnTiGXzXuft/MGTtvAOSemLdutX1mwIDok7iHDezS+C26xxV2nEhGrXEP0xgv0D2mvetdwNe/bn/XyXGbgV1RTJxoyxUrbBnlhFipAXslcU66ccs7cKAt9+4N/9lJSPpAMG6cLVevrv68Rq2KnTIFOOccaw5w6BAwd64NW9HoVbEi0bJ2rIo15ZmYDRts2dYWfRL3oMeWNNtN3XCDVcH+9V8n/97NqNa4h0leqLuuvdaOW4MGldr5XXppcu+fooq/HBEZUOWxsZUeo4gmTbLer2vW2JVmlBNillWx3h/akSPWFqFbN+vFG0SvXjYw8JEjXavuspJ0YFdriiRXo1bF9uoF/PGPwG232f3nn7f9+NAhy2o1cpVjlCFPWBVryjtQJHEhEPT/kWbG7uyzrd2XO6YdxTNypCUN9uwpDfLuOnasNI2b24M2CW9/u9VI7dxptxdfjN8eOyPVLomecP8Qkfllj/1XGoVpaj162BWeqnXrLnpVrPeA7A6qPGJE8Ktsb6YjbMPzuI4ft20MJBfY1aoqACyIdRtUu1MfNRrvdvBmcaMMW1EvouzHrIo15Rm7LOdurrOejk2v0sXzihV2bD311FK2tslVOwt7j8Tlw+Q38FE6R96DXNpVsYcO2RVr9+7RZkFoa7PgZNOm0tVS2CrdqIO7xrVqlWUJx49P7qAeJGPnzRA0aqDT3m4XKWvXloYmaNRqWFecjF2zB3aTJtmFzurVtk2S6GxThIwdJa9WL2q2Z/yzaoGdVvjb7z4lwRscpF0V6x5Ao2ZTWlpKDZ/nzu36+UFFHSoirjQa2k6ebNvxhRcqVy03avs6r+7dS8M7/PKXtmz0wC5Kxo5VsaZnz641FUl0tmHGrjFVqhVJ43he56oFdsNF5NMi8hnP3+79nCdXbVDujvvEExYc9OsX7oo+SmAX56TrljdqYBd1qIi40mho27cv8LrXWTXvsmX+z2nU9nXl4u4X9SZKxo5VsSV+NRXM2FG5SrUiaRzP61y1wO4HAPoD6Of5273/7+kXrQm5WR+3Z1jYq9YwVbFJHEDdH1rU8jZSxs77fpXa2TVDxg4obYeNG23ZqEOduOJk7BjYdT1hJ3HBGeT/ocrArt5MmGBt6NwhcQ4ftiTIokX2ODN2f1Zx5glV/QoAiMhQVX0tuyI1sX79bMDclSvtftgJsIcPtyrS7dutp1C1OWaTqPIov0IKW968MnZpHQimTQPuvrv0/uWaLWPnijKRez1hxi4ebxXb9u32d5zjUpD/x/79NkB5377B5uKm/InYMfaJJ+wi4NFHgdtvt8dGjizN70qVAzsRuQrAXQCOiUgngGtV9Q+ZlaxZff3rtrOKAJ/6VLjXtrZacPfqq3ZVM3p05ecmkbGbMgX45Cet88TgwcA73hHu9Xlk7PbvtwN+z57JVxG6A6pu2uT/eLNkCGbOBG66yS5Qhg4Frr467xKlK0rGbs8eWzb6vhDElCl2vFu+vDRGWRKBXbX/R7NcZDUaN7BbuBD4zW9s3bnnAn/zN7kWq2iqzRX7dQDnqepKEXkjgG8CuCCbYjWxv/xLu0U1cqQFdq++Wj2wS6LKQwT49rejvz6PjJ03oE26Z2qtNo7N0li7pQX47nfzLkV2wmbsjh2z6edaW5mxA2wbnH56qRf14MHxpuFyp/g7eBA4etR6aZdrlmYRjcbN7j71lO0v3boB8+dz2rYy1drYHVfVlQCgqn+Cta+joiufw7WSIkz1lMdwJ2nOhFArsGuWjF2zCZt5boZhb8LyNouIe0zyTvFX6djCjF19cveTefMsu8u5eH1Vy9gNF5FPV7qvqv83vWJRZEF7xhZhqqc8BihOc1J694S0dSvQ2XniYM3MEjSmsJlnBhUnmjYNuPde+zuJ3+agQcCOHfY/GT78xMf5W6xPZ5xhs9y4Q0qxw4SvIL1i3Vv5fSoi96BYq2dsmgFOUHlm7NLIVPbqZd/p2DGbgqYcT+iNKWrGjkFFibfDTRLHpFr/E2bP61O3bqXxUwEOcVJBzV6xVGeCVMV2dJSmAfO7ms1KHhm7tDOVI0faSWPLlhPnzeUJvTExYxefN/OSVMYOqF0Vy99i/Zk2DfjTn0p/0wmqTuwpIleIyJMi8pqIbBeR34rIlVkVjiIIUhW7fbtVFQ4d6t+wOCt5ZOzSzlRWy5jyhN6Y3Pkpd++231UtDPBPNGQIMHas/Z1ENr3WsYUZu/rlZulEumbv6M8qBnYi8mEAXwXwZQCnApgI4CsAviwiH8mkdBSeO2bYyy9Xfo77WN7ji+Ux3EnanUaqZUy3bbNllLl5qbi6d7eemJ2dpYGHq2FQ4e+Nb7SlO2xQHLVqA3iRVb9mzrRlezun5KugWueJTwF4s6p6Gwv9WkSuAPAUgDtSLRlF095uy+XLKw9SvHixLSdNyq5cftxMx549/p0N0pBFVaz3c1w7d1pA3bu3jaBOjeWkkyyo27ULGDCg+nNZDejvW98CLroomXEPg2bs+D+oP1OmAPfcYx0pyFe1M6mUBXUAAFXdkWJ5KK6BA4GJE4EjR4AVK/yfU5S59bp1sysuVWDv3mw+M6+qWHc2iilTbIwtaixh2tkxY+dvzBjgox9N5gKPGbvGdt11bF9XRbVf0F4ROaEC21m3L70iUWy15ixNa67UKLIcpPjYMWtfKAIMG5bOZ1Sqii3SNqfkhWkvyoxd+pixoyZWLbD7DICHReTLIvJ2EblKRL4C4CEAn67yOsqbm4lzM3NeHR2lqtgiBBlZdqDw9gTuVq0VQgyVqmLd/0URtjklL0wPb2bs0seMHTWxioGdqj4F4I3Ocz4I4Abn73Ocx6ioqmXsVq0CDh0Cxo0rRiP+LIc8yWLsvkpVse7/Iu/qb0oHM3bFwowdNbGqaQtVfRXAP4jIMOf+9kxKRfG4gd2iRSd2SihalWCWGbssplHzq4o9cABYudKyhGeemd5nU36YsSsWZuyoiVUM7EREAHwJwE2wTJ2ISAeAf1XVf8yofBTFiBE2lMnmzcA73tF1Lr3ly21ZlMAuy4xdFtOonXSSbe99+yyg69sXWLLEOoi0t3New0bFjF2xVPt/HDtmv82WFg6XQQ2pWsbukwDOBTBTVdcDgIicCuB2EfmUqn47g/JRVOedB9x3H/DQQ5UfLwLv4K5p27DBlqNHp/cZIvb+69YB69dbhq4ovZApPWHGZGTGLn1uM5MdPoM4eLe/SFYlIspMtcDuAwAuUdXX3BWq+qKIvA/A4wAY2BXZ7bcDf/VX1lmi3PDhwFveknmRfLlXzEEGdo3LrYZOe7TyyZMtsFu0yAK7olV/U/KC9u5WZTVgFgYPtqYPu3bZ0E/eTDnb11GDqxbYdfcGdS5V3S4iPqPeUqEMGmSBXdH162fLtAM71ex6pk6fDvz85/Z573sfA7tmEDRjd+CAXWz16ZPvdH6NrqXFmqS88op1ZBo3rvQYM6bU4KoNd3I04mNEwWWVsduyxab0Gjgw/ZkfvL2Sjx4Fli61+1Onpvu5lJ+gGTtm67JTaegh/g+owVXL2E0REb/pAARAr5TKQ83GzdjtS3nMa2/WLO12NW5buoULS1O7tbWxoXYjC5qxYzVgdioFdvwfUIOrNo5dq6oO8Ln1V9VYVbEiMlhE5onIGmfp+wsTkctFZJWIrBWRW3we/6yIqIgMde6PF5FDIrLIuX0/TjkpA1lVxWY5QPDIkVYNtGcP8LOfZfe5lB9m7IrHHXqofExJ/g+owWUw67qvWwDMV9U2APOd+12ISCuA7wG4AkA7gOtFpN3z+BgAlwB4qeyl61R1qnO7Ma0vQAlxs1hZZeyy6pnqfs6dd2b7uZQPZuyKhxk7alJ5BXbXAJjj/D0HwCyf58wEsFZVX1TVowDudV7n+jaAzwHQFMtJaUsjY7dggbVna2sr3R591B7LKnPmfs7mzdl+LuWjXz+gtRU4eNDaVVbChvvZYRs7alJ5BXYjVHULADjL4T7POQXAy577m5x1EJGrAbyiqot9XjdBRBaKyG9FpOJgbSLyERFZICILtm/nhBq5SSOwu+sumw937drS7ehR6zRxxhnJfU41l19eass3eDAwc2Y2n0v5EAk2SLF7rBkyJO0SUaWqWGbsqMGlNBM6ICK/AuA3d9MXg76FzzoVkT7Oe1zq8/gWAGNVdYeInA3gv0Rkkqqe0AlEVe8AcAcAzJgxg1m/vKRRFetWu/74x8A555TWjx5tY1tl4bzz7ISyZ4+dYNhxovENGmQD4u7ebWNF+slivmIyzNhRk0rtLKeqF1d6TES2ishIVd0iIiMBbPN52iYAYzz3RwPYDGAigAkAFtusZxgN4HkRmenMbXvE+fznRGQdgNMBLEjiO1EKks7YdXRYtg4Arrwy38zI8OGVT/DUeIK0s8tivmIybGNHTSqvqtiHAcx2/p4NwG/eq2cBtInIBBHpAeA6AA+r6lJVHa6q41V1PCwAnK6qr4rIMKfThTv9WRuAF9P+MhRD0oHd6tXWzmnsWFZ3UbaC9IzNYr5iMiNG2HLr1q4z8DBjRw0ur8DuGwAuEZE1sJ6t3wAAERklInMBQFWPA7gZwGMAVgC4X1WX13jf8wEsEZHFAB4AcKOq7kzpO1ASeva06tGjR6s3Og+KszxQXoK0sWNVbHZ69rT2rR0dXeeMZcaOGlxGDY66UtUdAC7yWb8ZwJWe+3MBzK3xXuM9fz8I4MHECkrpE7Gs3e7dlrVzJ++OioEd5cUNFIJUxTKwy8bIkcDOnbbd3WYRzNhRg8srY0dUkmR1rDsQMceNo6zVytgdPmxBRbdu8S9gKBi3LaMbUKtyyBlqeAzsKH9J9YxVBRYtsr+ZsaOs1eo84VbDnnyyTVJP6SvvQHHgAHD8ONC7t1XVEjUgHl0of0ll7PbutWqXPn2AU06JXy6iMGp1nmD7uuxNnGjLFStsuXq1LceNy6c8RBlgYEf5cwO7uBk7b/sl8RsGkShFtTJ2HOoke27m3m2ikeWc0UQ5YWBH+XOrYuNm7JgRoTzVytix40T23La2CxdaU42s54wmygEDO8pfUlWxzIhQnoK2sWNgl53Ro208y507gZdfZsaOmgIDO8pfGlWxRFljxq54RErZuWefLc1Kw8COGhgDO8ofq2KpEbCNXTG5Qdx99wGHDlnHCQ43Qw2MgR3lj1Wx1Ai849ipnvg4M3b5cAO7n/60632iBsXAjvLHqlhqBD162FA7HR3+FymbN9uS+2e2yjtKMLCjBpfLlGJEXbAqlhrFoEHAwYOWtXP3awB47TXbP/v25RiLWWtrAz7zGWDBAmDgQOBDH8q7RESpYmBH+Us6Y8eqWMrLSScBr7xi7ezGjCmtd4fZmDKFs05kTQT4P/8n71IQZYZHGMpfEm3sjhyxIQ1aW4Fhw5IpF1FYlXrGcg5jIsoIAzvKXxJVsW417IgRzIhQfir1jHUzdmzfRUQp4xmQ8pdEVax3gnWivFTK2DGwI6KMMLCj/CVRFcsesVQEfhm7ffts8vnu3YFJk3IpFhE1DwZ2lL+4VbF/+ANw9932NwM7ypN3LDuXO9vBmWfakChERClir1jKn5ux27s3/GuPHwcuu6wUFI4dm1y5iMLyq4plxwkiyhADO8rfgAG23LfPBndtbQ3+2q1bLajr1w/49KeBv/mbdMpIFIRfVSzb1xFRhlgVS/lrbS0Fd3v2hHut22nitNOAr3wFGDo02bIRheGXsWNgR0QZYmBHxVCpN2EtHJSYiqQ8Y3fkCLB8uQ2SO2VKbsUioubBwI6KodL4X7WwNywVSfkFyrJl1g70jDNsOjEiopQxsKNiiJqx4/ywVCTlFyhuNSw7ThBRRhjYUTHEzdixKpaKoPwCxe0Ry/Z1RJQRBnZUDHHb2DFjR0XQr59Nabd/P3DsGDtOEFHmGNhRMfgN7BoEAzsqkpaWUs/sl18uDU7MwI6IMsLAjorBzdiFrYrlHLFUNGedZcv77wcOHQLGjQMGD863TETUNBjYUTFEydipMmNHxeN2lLjzTlsyW0dEGWJgR8UQJWO3axdw9KgNbtynTzrlIgrLDeTWrLEle8QSUYYY2FExRMnYsRqWiqg8Q8eMHRFlKJfATkQGi8g8EVnjLAdVeN7lIrJKRNaKyC2e9V8WkVdEZJFzu9Lz2Bec568Skcuy+D6UgCjDnbAaloqora3rYMTM2BFRhvLK2N0CYL6qtgGY79zvQkRaAXwPwBUA2gFcLyLtnqd8W1WnOre5zmvaAVwHYBKAywHc5rwPFV2U4U4Y2FERtbaWpg8bPpz7JxFlKq/A7hoAc5y/5wCY5fOcmQDWquqLqnoUwL3O62q9772qekRV1wNY67wPFV2cjB2rYqlo3OrXadNsnlgioozkFdiNUNUtAOAsh/s85xQAL3vub3LWuW4WkSUicqenKrfWa/5MRD4iIgtEZMH27dujfg9KSpSMHacTo6KaNavrkogoI93SemMR+RUAv1TKF4O+hc86dZa3A/iqc/+rAL4F4IYar+m6UvUOAHcAwIwZM3yfQxnq3Rvo3h04csTG/urdu/ZrWBVLRXXxxcDevTYTBRFRhlIL7FT14kqPichWERmpqltEZCSAbT5P2wRgjOf+aACbnffe6nmvHwB4pNZrqOBELGu3bZtl7cIEdqyKpSLq3z/vEhBRE8qrKvZhALOdv2cDeMjnOc8CaBORCSLSA9Yp4mEAcIJB118CWOZ53+tEpKeITADQBuCZFMpPaQg75AmrYomIiLpILWNXwzcA3C8ifw3gJQDvAgARGQXg31X1SlU9LiI3A3gMQCuAO1V1ufP6b4rIVFg16wYAHwUAVV0uIvcDeAHAcQA3qWpHdl+LYgk7SDGrYomIiLrIJbBT1R0ALvJZvxnAlZ77cwHM9Xne+6u8960Abk2mpJSpMBm7Q4eAPXusXR7n4SQiIgLAmSeoSMJk7LyzTnA4CSIiIgAM7KhIwmTsWA1LRER0AgZ2VBxhMnYM7IiIiE7AwI6KI0rGjkOdEBER/RkDOyqOMNOKcagTIiKiEzCwo+IIM60Yq2KJiIhOwMCOiiNMxo5VsURERCdgYEfFESZjx6pYIiKiEzCwo+LgcCdERESxMLCj4gg63MmxY8C2bfb38OHplomIiKiOMLCj4hg40JZ79gCdnZWft2KFPT5xItCjRzZlIyIiqgMM7Kg4unUD+vcHVIG9eys/7/nnbTl9ejblIiIiqhMM7KhYgrSzW7jQltOmpV0aIiKiusLAjoolSDs7BnZERES+GNhRsdTK2HV2MrAjIiKqgIEdFUutjN26dcD+/cCoUcCIEdmVi4iIqA4wsKNiqZWxY8cJIiKiihjYUbHUmlaM1bBEREQVMbCjYqk1rRgDOyIioooY2FGxVKuKVWVVLBERURUM7KhYqnWeeOUV4LXX7Dljx2ZbLiIiojrAwI6KpVrGzs3WTZsGiGRVIiIiorrBwI6KpVrGzm1fx2pYIiIiXwzsqFiqZezYcYKIiKgqBnZULNUyduw4QUREVBUDOyoWN7DbudN6wbpeew14+WWgTx+grS2fshERERUcAzsqlr59gX79gCNHulbHutWwU6YAra25FI2IiKjoGNhR8YwcacstW0rr2HGCiIioJgZ2VDxuYPfqq6V17DhBRERUEwM7Kp6TT7alN2PHjhNEREQ15RLYichgEZknImuc5aAKz7tcRFaJyFoRucWz/ssi8oqILHJuVzrrx4vIIc/672f1nShB5VWx+/YBa9YA3bsDkyblVy4iIqKCyytjdwuA+araBmC+c78LEWkF8D0AVwBoB3C9iLR7nvJtVZ3q3OZ61q/zrL8xxe9AaSmvil282HrITpoE9OiRX7mIiIgKLq/A7hoAc5y/5wCY5fOcmQDWquqLqnoUwL3O66jReatib7wRuO46u89qWCIioqryCuxGqOoWAHCWw32ecwqAlz33NznrXDeLyBIRubOsKneCiCwUkd+KyHmVCiAiHxGRBSKyYPv27TG+CiXOzdg9/TTwb/8GvPKK3b/00vzKREREVAdSC+xE5FcissznFjTr5jfLuzti7e0AJgKYCmALgG8567cAGKuq0wB8GsDdIjLA781V9Q5VnaGqM4YNGxb0a1EW3MDuxRdt+eY3A+vXA+9+d35lIiIiqgPd0npjVb240mMislVERqrqFhEZCWCbz9M2ARjjuT8awGbnvbd63usHAB5x1h8BcMT5+zkRWQfgdAALYn4dypIb2LnOPRcYPz6XohAREdWTvKpiHwYw2/l7NoCHfJ7zLIA2EZkgIj0AXOe8Dk4w6PpLAMuc9cOcThcQkVMBtAF4MZVvQOkZPBjo5rnm4Nh1REREgeQV2H0DwCUisgbAJc59iMgoEZkLAKp6HMDNAB4DsALA/aq63Hn9N0VkqYgsAfBWAJ9y1p8PYImILAbwAIAbVXVnVl+KEtLSUupAAbDTBBERUUCi3onWm9SMGTN0wQLW1hbKzJnAs88C/fvbnLEtHEubiIgIAETkOVWd4fcYz5ZUTG7GbupUBnVEREQB8YxJxeR2oGD7OiIiosAY2FExvetdwGmnAR/4QN4lISIiqhupDXdCFMvFF9v8sERERBQYM3ZEREREDYKBHREREVGDYGBHRERE1CAY2BERERE1CAZ2RERERA2CgR0RERFRg2BgR0RERNQgGNgRERERNQgGdkREREQNgoEdERERUYNgYEdERETUIBjYERERETUIBnZEREREDUJUNe8y5E5EtgPYGOIlQwG8llJxGg23VTjcXsFxWwXHbRUct1U43F7BJbmtxqnqML8HGNhFICILVHVG3uWoB9xW4XB7BcdtFRy3VXDcVuFwewWX1bZiVSwRERFRg2BgR0RERNQgGNhFc0feBagj3FbhcHsFx20VHLdVcNxW4XB7BZfJtmIbOyIiIqIGwYwdERERUYNgYBeSiFwuIqtEZK2I3JJ3eYpGRDaIyFIRWSQiC5x1g0VknoiscZaD8i5nHkTkThHZJiLLPOsqbhsR+YKzn60SkcvyKXU+KmyrL4vIK86+tUhErvQ81szbaoyI/EZEVojIchH5hLOe+5aPKtuL+1cZEeklIs+IyGJnW33FWc99q0yVbZX9fqWqvAW8AWgFsA7AqQB6AFgMoD3vchXpBmADgKFl674J4Bbn71sA/FPe5cxp25wPYDqAZbW2DYB2Z//qCWCCs9+15v0dct5WXwbwWZ/nNvu2GglguvN3fwCrnW3CfSvc9uL+deJ3FwD9nL+7A/gTgHO4b4XaVpnvV8zYhTMTwFpVfVFVjwK4F8A1OZepHlwDYI7z9xwAs/IrSn5U9UkAO8tWV9o21wC4V1WPqOp6AGth+19TqLCtKmn2bbVFVZ93/t4HYAWAU8B9y1eV7VVJ024vNfudu92dm4L71gmqbKtKUttWDOzCOQXAy577m1D9gNCMFMDjIvKciHzEWTdCVbcAdlAFMDy30hVPpW3Dfc3fzSKyxKmqdat/uK0cIjIewDRYtoD7Vg1l2wvg/nUCEWkVkUUAtgGYp6rctyqosK2AjPcrBnbhiM86divu6lxVnQ7gCgA3icj5eReoTnFfO9HtACYCmApgC4BvOeu5rQCISD8ADwL4pKrurfZUn3XcXty/fKlqh6pOBTAawEwRObPK07mtTtxWme9XDOzC2QRgjOf+aACbcypLIanqZme5DcDPYanlrSIyEgCc5bb8Slg4lbYN97UyqrrVOXB2AvgBStUWTb+tRKQ7LEj5iar+zFnNfasCv+3F/as6Vd0N4AkAl4P7VlXebZXHfsXALpxnAbSJyAQR6QHgOgAP51ymwhCRviLS3/0bwKUAlsG20WznabMBPJRPCQup0rZ5GMB1ItJTRCYAaAPwTA7lKwz3ROL4S9i+BTT5thIRAfBDACtU9f96HuK+5aPS9uL+dSIRGSYiJzl/9wZwMYCV4L51gkrbKo/9qlsSb9IsVPW4iNwM4DFYD9k7VXV5zsUqkhEAfm7HTXQDcLeq/lJEngVwv4j8NYCXALwrxzLmRkTuAfAWAENFZBOALwH4Bny2jaouF5H7AbwA4DiAm1S1I5eC56DCtnqLiEyFVVdsAPBRgNsKwLkA3g9gqdO+BwD+J7hvVVJpe13P/esEIwHMEZFWWCLoflV9RET+CO5b5Sptq//Ier/izBNEREREDYJVsUREREQNgoEdERERUYNgYEdERETUIBjYERERETUIBnZEREREDYKBHRHVBRH5SxFREXmdZ914EVnm/D1DRL6TXwn9icgsEWlP+TP+4CzHi8h7POsLuU2IKD0M7IioXlwP4CnYwOAnUNUFqvrxbIsUyCwAqQZ2qvoXzp/jAbzHs76o24SIUsLAjogKz5nX81wAf40KgZ2IvEVEHnH+HiYi80TkeRH5NxHZKCJDnYzWChH5gYgsF5HHnVHiISJPiMi3ReRJ5zlvEJGficgaEfma53PeJyLPiMgi571bnfX7ReRWEVksIk+LyAgR+QsAVwP4Z+f5E8vK/CMR+b6I/E5EVovIVc76XiJyl4gsFZGFIvJWZ/0kz2cvEZE297Odt/wGgPOcxz9Vtk0Gi8h/Oa97WkQmO+u/LDY5+RMi8qKIMBAkqmMM7IioHswC8EtVXQ1gp4hMr/H8LwH4tapOh81ZPNbzWBuA76nqJAC7AbzT89hRVT0fwPdh0yTdBOBMAB8UkSEi8noA7wZwrjPZdweA9zqv7QvgaVWdAuBJAB9W1T/Apg76O1WdqqrrfMo6HsAFAN4G4Psi0sv5XKjqWbBM5Rxn/Y0A/p/z2TNg80163QLgd85nfbvssa8AWKiqk2EzLfzY89jrAFwGm8fyS2JzqRJRHeKUYkRUD64H8C/O3/c695+v8vw3w+ZlhDOt3S7PY+tVdZHz93OwwMrlzv28FMByVd0CACLyImzC7jcDOBvAs87Ueb1RmgD9KIBHPO97ScDvdr8zQfga53Ne53zOvzrlXykiGwGcDuCPAL4oIqMB/ExV1wT8DDjv+U7nPX/tBKoDncceVdUjAI6IyDbY9IDlQSMR1QEGdkRUaCIyBMCFAM4UEYXN06wi8rlqL6vy2BHP3x2w4Kz8sc6y53XCjpcCYI6qfsHnfY9paY7GDgQ/vpbP66ioUH5VvVtE/gTL7j0mIv9DVX8d8HP83tP97PJtwnMDUZ1iVSwRFd1fAfixqo5T1fGqOgbAelgGqpKnAFwLACJyKYBBCZVlPoC/EpHhznsPFpFxNV6zD0D/Ko+/S0RanPZ3pwJYBavKfa/zGafDqpJXicipAF5U1e/AsouTQ3yW9z3fAuA1Vd1bo+xEVGcY2BFR0V0Payfn9SA8vT99fAXApSLyPIArAGyBBT2xqOoLAP4XgMdFZAmAeQBG1njZvQD+zukEMdHn8VUAfgvgFwBuVNXDAG4D0CoiSwHcB+CDTlXpuwEsE5FFsCrbH5e91xIAx50OHJ8qe+zLAGY45f4GgNlBvjMR1Rcp1RwQETUGEekJoENVj4vImwDc7nQ4KBQR+RGAR1T1gbzLQkSNge0oiKgRjQVwv4i0wDo1fDjn8hARZYIZOyIiIqIGwTZ2RERERA2CgR0RERFRg2BgR0RERNQgGNgRERERNQgGdkREREQNgoEdERERUYP4/x+F1Uqg86GgAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 720x432 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "import pylab\n",
    "import modeller\n",
    "\n",
    "def r_enumerate(seq):\n",
    "    \"\"\"Enumerate a sequence in reverse order\"\"\"\n",
    "    # Note that we don't use reversed() since Python 2.3 doesn't have it\n",
    "    num = len(seq) - 1\n",
    "    while num >= 0:\n",
    "        yield num, seq[num]\n",
    "        num -= 1\n",
    "\n",
    "def get_profile(profile_file, seq):\n",
    "    \"\"\"Read `profile_file` into a Python array, and add gaps corresponding to\n",
    "       the alignment sequence `seq`.\"\"\"\n",
    "    # Read all non-comment and non-blank lines from the file:\n",
    "    f = open(profile_file)\n",
    "    vals = []\n",
    "    for line in f:\n",
    "        if not line.startswith('#') and len(line) > 10:\n",
    "            spl = line.split()\n",
    "            vals.append(float(spl[-1]))\n",
    "    # Insert gaps into the profile corresponding to those in seq:\n",
    "    for n, res in r_enumerate(seq.residues):\n",
    "        for gap in range(res.get_leading_gaps()):\n",
    "            vals.insert(n, None)\n",
    "    # Add a gap at position '0', so that we effectively count from 1:\n",
    "    vals.insert(0, None)\n",
    "    return vals\n",
    "\n",
    "e = modeller.environ()\n",
    "a = modeller.alignment(e, file='TvLDH-mult.ali')\n",
    "\n",
    "#only uncomment the next, if you have generated profile for templates\n",
    "#template = get_profile('mult.profile', a['mult'])\n",
    "model = get_profile('TvLDH.profile', a['TvLDH'])\n",
    "\n",
    "# Plot the template and model profiles in the same plot for comparison:\n",
    "pylab.figure(1, figsize=(10,6))\n",
    "pylab.xlabel('Alignment position')\n",
    "pylab.ylabel('DOPE per-residue score')\n",
    "pylab.plot(model, color='red', linewidth=2, label='Model')\n",
    "#only uncomment the next, if you have generated profile for templates\n",
    "#pylab.plot(template, color='green', linewidth=2, label='Template')\n",
    "pylab.legend()\n",
    "pylab.savefig('dope_profile.png', dpi=65)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Visually compare the model vs templates "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#interactively view molecular structures and trajectories\n",
    "!conda install nglview -c conda-forge"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# if you already installed nglview, you can `upgrade`\n",
    "!conda upgrade nglview --force"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!jupyter labextension install  nglview-js-widgets"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run this if installed nglview via pip install\n",
    "!jupyter-nbextension enable nglview --py --sys-prefix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "52605903e3a0409498bdd3832b3cd4ef",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "_ColormakerRegistry()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import nglview as nv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "b700efb4516443fd955686fdb8da5196",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "NGLWidget()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "view = nv.NGLWidget()\n",
    "\n",
    "#load the model/ all models\n",
    "view.add_component('TvLDH.B99990001.pdb')\n",
    "view.add_component('TvLDH.B99990002.pdb')\n",
    "view.add_component('TvLDH.B99990003.pdb')\n",
    "view.add_component('TvLDH.B99990004.pdb')\n",
    "view.add_component('TvLDH.B99990005.pdb')\n",
    "view.background = 'white'\n",
    "\n",
    "#view.stage.set_parameters(mouse_preset='coot')\n",
    "#view.color_by('ssctr')\n",
    "view\n",
    "\n",
    "#mouse hover on the chains to know the atoms|PDBID|residue ID|etc\n",
    "#scroll to zoom\n",
    "#click and drag to rotate"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "779b9ddf03ab40638a357cdbe9b7dfb1",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "NGLWidget()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "view1 = nv.NGLWidget()\n",
    "\n",
    "#load the best model\n",
    "view1.add_component('TvLDH.B99990001.pdb')\n",
    "\n",
    "#load the templates\n",
    "view1.add_component('4h7p_fit.pdb')\n",
    "view1.add_component('1bdm_fit.pdb')\n",
    "view1.add_component('3d5t_fit.pdb')\n",
    "view1.add_component('5mdh_fit.pdb')\n",
    "view1.add_component('6itk_fit.pdb')\n",
    "view1.add_component('7mdh_fit.pdb')\n",
    "view1.background = 'white'\n",
    "\n",
    "#view.stage.set_parameters(mouse_preset='coot')\n",
    "view1.color_by('chain')\n",
    "view1\n",
    "\n",
    "#mouse hover on the chains to know the atoms|PDBID|residue ID|etc\n",
    "#scroll to zoom\n",
    "#click and drag to rotate"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "NOTE: The notebook is not yet completed. Multi template profile to generated and plots to be generated.\n",
    "Loop refinement will be uploaded once this notebook is complete! Thanks for your patience!"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## This notebook is compiled and verfied by Girinath G. Pillai, @giribio\n",
    "\n",
    "### Kindly cite this notebook in your publications if you have used it for calculations.\n",
    "\n",
    "### Cite as : NyroMDNotebook v1, Girinath G. Pillai : https://github.com/giribio/MDNotebooks"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
