{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Kindly cite this notebook in your publications if you use it for calculations.\n",
    "\n",
    "### Cite as : NyroMDNotebook v1, Girinath G. Pillai : https://github.com/giribio/ModellerNotebooks"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###### Used the tutorial from Salilab : https://salilab.org/modeller/tutorial/basic.html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Install anaconda from the web\n",
    "# https://www.anaconda.com/products/individual\n",
    "\n",
    "###### numpy and matplotlib required for result analysis and ploting"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Install MODELLER using the following commands\n",
    "Ensure Anaconda is installed, therefore python3, conda, pip3, jupyter notebook and others are installed\n",
    "\n",
    "TESTED in LINUX and MACOSX, In Windows this notebook should work, \n",
    "but change commands for curl/wget/cat/etc as per the OS requirements.\n",
    "Create a folder and run the notebook within folder and download all supporting files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Warning: 'salilab' already in 'channels' list, moving to the top\n",
      "\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "conda config --add channels salilab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting package metadata (current_repodata.json): done\n",
      "Solving environment: done\n",
      "\n",
      "# All requested packages already installed.\n",
      "\n",
      "\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "conda install modeller"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Go to https://salilab.org/modeller/ to get a license key,\n",
    "and then set the 'license' variable to it in the file\n",
    "~/<miniconda or anaconda>/lib/modeller-9.25/modlib/modeller/config.py"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Basic Homology Modeling"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Model a sequence with high identity to a template.\n",
    "This exercise introduces the use of MODELLER in a simple case where \n",
    "the template selection and target-template alignments are not a problem.\n",
    "\n",
    "Download the latest PDB sequence database from https://salilab.org/modeller/downloads/pdb_95.pir.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#To download latest PDB sequence database in PIR file\n",
    "!curl https://salilab.org/modeller/downloads/pdb_95.pir.gz -o pdb_95.pir.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Unzip the file\n",
    "!gunzip pdb_95.pir.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Searching template structures related to target sequence"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Which ever Target sequence, you would like to model, get the FASTA format of the sequence\n",
    "And make generate PIR format which is readable by MODELLER with file extension .ali\n",
    "\n",
    "SAMPLE\n",
    ">P1;CODE ##CODE (used in next line too) - some letters to represent target sequence##\n",
    "sequence:CODE:::::0.00:0.00 ##\n",
    "##FASTA FORMAT SEQUENCE##\n",
    "and at the end * (symbol without gap or space represents end of sequence)\n",
    "Then save the file with .ali as file extension, in this example we used the following \n",
    "sequence and filename with TvLDH.ali (copy paste the following to editor and save as TvLDH.ali)\n",
    "''''\n",
    ">P1;TvLDH\n",
    "sequence:TvLDH:::::::0.00: 0.00\n",
    "MSEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKA\n",
    "AFKDIDCAFLVASMPLKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPEN\n",
    "FSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKI\n",
    "GHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVV\n",
    "EGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQGG*\n",
    "''''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "                         MODELLER 9.25, 2020/09/03, r11894\n",
      "\n",
      "     PROTEIN STRUCTURE MODELLING BY SATISFACTION OF SPATIAL RESTRAINTS\n",
      "\n",
      "\n",
      "                     Copyright(c) 1989-2020 Andrej Sali\n",
      "                            All Rights Reserved\n",
      "\n",
      "                             Written by A. Sali\n",
      "                               with help from\n",
      "              B. Webb, M.S. Madhusudhan, M-Y. Shen, G.Q. Dong,\n",
      "          M.A. Marti-Renom, N. Eswar, F. Alber, M. Topf, B. Oliva,\n",
      "             A. Fiser, R. Sanchez, B. Yerkovich, A. Badretdinov,\n",
      "                     F. Melo, J.P. Overington, E. Feyfant\n",
      "                 University of California, San Francisco, USA\n",
      "                    Rockefeller University, New York, USA\n",
      "                      Harvard University, Cambridge, USA\n",
      "                   Imperial Cancer Research Fund, London, UK\n",
      "              Birkbeck College, University of London, London, UK\n",
      "\n",
      "\n",
      "Kind, OS, HostName, Kernel, Processor: 4, Darwin Girinaths-MacBook-Pro.local 19.6.0 x86_64\n",
      "Date and time of compilation         : 2020/09/03 08:25:47\n",
      "MODELLER executable type             : mac10v4-intel64\n",
      "Job starting time (YY/MM/DD HH:MM:SS): 2020/11/28 18:20:25\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       191566     187.076     0.183\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       192094     187.592     0.183\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       240694     235.053     0.230\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       253994     248.041     0.242\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           pdb_95.pir\n",
      "readlinef__W> File: pdb_95.pir, Line: 6025\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 6137\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 8219\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 9814\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31139\n",
      "              Modeller will only read the first 4096 characters of this line.\n",
      "\n",
      "readlinef__W> File: pdb_95.pir, Line: 31145\n",
      "              No more warnings will be printed for truncated lines in this file.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       523244     510.980     0.499\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:      4084367    3988.640     3.895\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4086475    3990.698     3.897\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4087925    3992.114     3.899\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4090825    3994.946     3.901\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      4096625    4000.610     3.907\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     17535024   17124.047    16.723\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17546624   17135.375    16.734\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17569824   17158.031    16.756\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17616224   17203.344    16.800\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17709024   17293.969    16.889\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:     24428224   23855.688    23.297\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.pir\n",
      "SEQ_DATABASE_FORMAT       : PIR\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :           30        4000\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "SEARCH_CHAINS_LIST        : ALL\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "\n",
      "SEQ_DATABASE_FILE         : pdb_95.bin\n",
      "SEQ_DATABASE_FORMAT       : BINARY\n",
      "CHAINS_LIST               : ALL\n",
      "CLEAN_SEQUENCES           :            T\n",
      "MINMAX_DB_SEQ_LEN         :            0      999999\n",
      "Number of sequences       :        67192\n",
      "Number of residues        :     17199305\n",
      "Length of longest sequence:         3725\n",
      "\n",
      "openf___224_> Open           TvLDH.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       525430     513.115     0.501\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       526880     514.531     0.502\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       529780     517.363     0.505\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       535580     523.027     0.511\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       536916     524.332     0.512\n",
      "\n",
      "Read the alignment from file       : TvLDH.ali\n",
      "\n",
      "Total number of alignment positions:   335\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1      TvLDH     335      1       TvLDH\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:       538581     525.958     0.514\n",
      "openf___224_> Open           ${LIB}/blosum62.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       863756     843.512     0.824\n",
      "profile_iteration_> processing sequence:       1    335      1     0.0000000     0.0000000     0.0000000     1\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       864834     844.564     0.825\n",
      "profile_iteration_> processing sequence:    6720    335      1     2.3599999     0.0003512     0.0000013  2847\n",
      "profile_iteration_> processing sequence:   13439    335      1     4.6300001     0.0003445     0.0000013  2903\n",
      "profile_iteration_> processing sequence:   20158    335      1     6.8699999     0.0003408     0.0000013  2934\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       865209     844.931     0.825\n",
      "profile_iteration_> processing sequence:   26877    335      1     9.1300001     0.0003397     0.0000013  2944\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       865445     845.161     0.825\n",
      "profile_iteration_> processing sequence:   33596    335      1    11.3900003     0.0003390     0.0000013  2950\n",
      "profile_iteration_> processing sequence:   40315    335      1    13.6300001     0.0003381     0.0000013  2958\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       868395     848.042     0.828\n",
      "profile_iteration_> processing sequence:   47034    335      1    15.8999996     0.0003381     0.0000013  2958\n",
      "profile_iteration_> processing sequence:   53753    335      1    18.2199993     0.0003390     0.0000013  2950\n",
      "profile_iteration_> processing sequence:   60472    335      1    20.4799995     0.0003387     0.0000013  2953\n",
      "\n",
      "Dynamically allocated memory at amaxsequence_db [B,KiB,MiB]:       871905     851.470     0.832\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "profile_iteration_> processing sequence:   67191    335      1    22.8400002     0.0003399     0.0000013  2942\n",
      " AVERAGE RESIDUAL STANDARD ERROR:    26.38443\n",
      " AVERAGE RESIDUAL STANDARD DEVIATION:    247.6263\n",
      "profile_660_> Quality of the fit between observed/expected score distributions.\n",
      "              [Iteration, Chi2, nbins, KS-Stat, nscores ] =        1,        0.0182,      121,        0.0067,    67192\n",
      "\n",
      "HISTOGRAM OF Z-SCORES:\n",
      "\n",
      "Z:         1  -3.95000   0.00000   0.00000\n",
      "Z:         1  -3.85000   0.00000   0.00000\n",
      "Z:         1  -3.75000   0.00000   0.00000\n",
      "Z:         1  -3.65000   0.00000   0.00000\n",
      "Z:         1  -3.55000   0.00000   0.00000\n",
      "Z:         1  -3.45000   0.00000   0.00000\n",
      "Z:         1  -3.35000   0.00000   0.00000\n",
      "Z:         1  -3.25000   0.00000   0.00000\n",
      "Z:         1  -3.15000   0.00000   0.00000\n",
      "Z:         1  -3.05000   0.00000   0.00000\n",
      "Z:         1  -2.95000   0.00000   0.00000\n",
      "Z:         1  -2.85000   0.00000   0.00000\n",
      "Z:         1  -2.75000   0.00000   0.00000\n",
      "Z:         1  -2.65000   0.00000   0.00000\n",
      "Z:         1  -2.55000   0.00000   0.00001\n",
      "Z:         1  -2.45000   0.00045   0.00004\n",
      "Z:         1  -2.35000   0.00089   0.00016\n",
      "Z:         1  -2.25000   0.00089   0.00055\n",
      "Z:         1  -2.15000   0.00610   0.00163\n",
      "Z:         1  -2.05000   0.00551   0.00416\n",
      "Z:         1  -1.95000   0.01831   0.00933\n",
      "Z:         1  -1.85000   0.02262   0.01872\n",
      "Z:         1  -1.75000   0.04167   0.03400\n",
      "Z:         1  -1.65000   0.05596   0.05658\n",
      "Z:         1  -1.55000   0.09391   0.08721\n",
      "Z:         1  -1.45000   0.12516   0.12565\n",
      "Z:         1  -1.35000   0.15746   0.17059\n",
      "Z:         1  -1.25000   0.21461   0.21981\n",
      "Z:         1  -1.15000   0.25569   0.27052\n",
      "Z:         1  -1.05000   0.31254   0.31973\n",
      "Z:         1  -0.95000   0.34855   0.36469\n",
      "Z:         1  -0.85000   0.40645   0.40316\n",
      "Z:         1  -0.75000   0.44202   0.43360\n",
      "Z:         1  -0.65000   0.46047   0.45519\n",
      "Z:         1  -0.55000   0.44306   0.46779\n",
      "Z:         1  -0.45000   0.47193   0.47182\n",
      "Z:         1  -0.35000   0.44976   0.46811\n",
      "Z:         1  -0.25000   0.45616   0.45775\n",
      "Z:         1  -0.15000   0.44797   0.44196\n",
      "Z:         1  -0.05000   0.43353   0.42194\n",
      "Z:         1   0.05000   0.38650   0.39889\n",
      "Z:         1   0.15000   0.38487   0.37383\n",
      "Z:         1   0.25000   0.33665   0.34769\n",
      "Z:         1   0.35000   0.32355   0.32121\n",
      "Z:         1   0.45000   0.28366   0.29500\n",
      "Z:         1   0.55000   0.27518   0.26953\n",
      "Z:         1   0.65000   0.23708   0.24513\n",
      "Z:         1   0.75000   0.22294   0.22205\n",
      "Z:         1   0.85000   0.19467   0.20043\n",
      "Z:         1   0.95000   0.18246   0.18036\n",
      "Z:         1   1.05000   0.16862   0.16185\n",
      "Z:         1   1.15000   0.15255   0.14489\n",
      "Z:         1   1.25000   0.13067   0.12944\n",
      "Z:         1   1.35000   0.12100   0.11542\n",
      "Z:         1   1.45000   0.09986   0.10275\n",
      "Z:         1   1.55000   0.09153   0.09133\n",
      "Z:         1   1.65000   0.08037   0.08109\n",
      "Z:         1   1.75000   0.07635   0.07191\n",
      "Z:         1   1.85000   0.06102   0.06371\n",
      "Z:         1   1.95000   0.05670   0.05639\n",
      "Z:         1   2.05000   0.05358   0.04988\n",
      "Z:         1   2.15000   0.05239   0.04409\n",
      "Z:         1   2.25000   0.04182   0.03895\n",
      "Z:         1   2.35000   0.03453   0.03439\n",
      "Z:         1   2.45000   0.02932   0.03035\n",
      "Z:         1   2.55000   0.02932   0.02678\n",
      "Z:         1   2.65000   0.02247   0.02361\n",
      "Z:         1   2.75000   0.02188   0.02082\n",
      "Z:         1   2.85000   0.02054   0.01835\n",
      "Z:         1   2.95000   0.01682   0.01617\n",
      "Z:         1   3.05000   0.01459   0.01424\n",
      "Z:         1   3.15000   0.01176   0.01255\n",
      "Z:         1   3.25000   0.01176   0.01105\n",
      "Z:         1   3.35000   0.00878   0.00973\n",
      "Z:         1   3.45000   0.00893   0.00857\n",
      "Z:         1   3.55000   0.00655   0.00754\n",
      "Z:         1   3.65000   0.00640   0.00664\n",
      "Z:         1   3.75000   0.00506   0.00584\n",
      "Z:         1   3.85000   0.00566   0.00514\n",
      "Z:         1   3.95000   0.00521   0.00453\n",
      "Z:         1   4.05000   0.00536   0.00398\n",
      "Z:         1   4.15000   0.00327   0.00350\n",
      "Z:         1   4.25000   0.00313   0.00308\n",
      "Z:         1   4.35000   0.00283   0.00271\n",
      "Z:         1   4.45000   0.00253   0.00239\n",
      "Z:         1   4.55000   0.00268   0.00210\n",
      "Z:         1   4.65000   0.00179   0.00185\n",
      "Z:         1   4.75000   0.00208   0.00163\n",
      "Z:         1   4.85000   0.00253   0.00143\n",
      "Z:         1   4.95000   0.00179   0.00126\n",
      "Z:         1   5.05000   0.00134   0.00111\n",
      "Z:         1   5.15000   0.00179   0.00097\n",
      "Z:         1   5.25000   0.00164   0.00086\n",
      "Z:         1   5.35000   0.00030   0.00075\n",
      "Z:         1   5.45000   0.00134   0.00066\n",
      "Z:         1   5.55000   0.00060   0.00058\n",
      "Z:         1   5.65000   0.00119   0.00051\n",
      "Z:         1   5.75000   0.00060   0.00045\n",
      "Z:         1   5.85000   0.00000   0.00040\n",
      "Z:         1   5.95000   0.00045   0.00035\n",
      "Z:         1   6.05000   0.00045   0.00031\n",
      "Z:         1   6.15000   0.00015   0.00027\n",
      "Z:         1   6.25000   0.00074   0.00024\n",
      "Z:         1   6.35000   0.00045   0.00021\n",
      "Z:         1   6.45000   0.00000   0.00018\n",
      "Z:         1   6.55000   0.00015   0.00016\n",
      "Z:         1   6.65000   0.00015   0.00014\n",
      "Z:         1   6.75000   0.00015   0.00013\n",
      "Z:         1   6.85000   0.00030   0.00011\n",
      "Z:         1   6.95000   0.00000   0.00010\n",
      "Z:         1   7.05000   0.00000   0.00009\n",
      "Z:         1   7.15000   0.00000   0.00007\n",
      "Z:         1   7.25000   0.00000   0.00007\n",
      "Z:         1   7.35000   0.00000   0.00006\n",
      "Z:         1   7.45000   0.00000   0.00005\n",
      "Z:         1   7.55000   0.00015   0.00004\n",
      "Z:         1   7.65000   0.00030   0.00004\n",
      "Z:         1   7.75000   0.00015   0.00003\n",
      "Z:         1   7.85000   0.00000   0.00003\n",
      "Z:         1   7.95000   0.00000   0.00003\n",
      "Z:         1   8.05000   0.00015   0.00002\n",
      "\n",
      "HITS FOUND IN ITERATION:     1\n",
      "\n",
      "\n",
      "Dynamically allocated memory at     amaxprofile [B,KiB,MiB]:      1156905    1129.790     1.103\n",
      "> 3d5tA                      1    3132   29250     321     335   44.27     0.0           2   312     4   325     3   316\n",
      "> 1sovA                      1    3257    4700     316     335   26.59    0.74E-02       3   160    81   256    76   248\n",
      "> 6k12A                      1    3397    4650     297     335   27.89    0.88E-02       4   181    94   304    81   270\n",
      "> 6k13A                      1    3424    5100     332     335   28.08    0.99E-03       5   189    94   304   103   305\n",
      "> 6or9A                      1    3835    5300     306     335   25.00    0.32E-03       6   252    47   323    37   292\n",
      "> 1guzA                      1    4067    7100     306     335   25.27    0.28E-07       7   265    13   301     8   280\n",
      "> 3gviA                      1    5566    5350     314     335   29.44    0.25E-03       8   175    13   209    10   206\n",
      "> 1gv1B                      1    5627    4900     290     335   23.44    0.23E-02       9   191    99   301    80   271\n",
      "> 1gv0A                      1    5647    5350     301     335   25.53    0.24E-03      10   274    13   323     8   289\n",
      "> 4bguA                      1    7704    6500     303     335   26.48    0.62E-06      11   281     7   314     3   289\n",
      "> 4bgvA                      1    7705    7650     323     335   31.82    0.18E-08      12   168    13   191    10   185\n",
      "> 1a5zA                      1    8650    6900     312     335   28.14    0.82E-07      13   164    75   242    63   229\n",
      "> 4e0bA                      1   10200    5700     305     335   27.53    0.40E-04      14   282     4   314     2   288\n",
      "> 4ln1A                      1   10437    4850     317     335   24.50    0.34E-02      15   183    78   270    79   278\n",
      "> 3wswA                      1   10921    5800     316     335   24.51    0.25E-04      16   187    78   270    70   273\n",
      "> 6itlA                      1   11933    6350     312     335   26.63    0.14E-05      17   196     7   209     3   201\n",
      "> 6itkA                      1   11962   26850     318     335   40.52     0.0          18   305     7   325     5   310\n",
      "> 4cl3A                      1   12409    6700     309     335   32.16    0.23E-06      19   166    13   191    10   180\n",
      "> 3fi9A                      1   12918    8500     321     335   27.85    0.22E-10      20   231     9   260    12   248\n",
      "> 5mdhA                      1   14863   32900     333     335   44.41     0.0          21   328     2   332     1   331\n",
      "> 7mdhA                      1   14882   22600     351     335   33.74     0.0          22   325     6   334    14   339\n",
      "> 5kkcB                      1   15279    4850     320     335   26.02    0.34E-02      23   191   101   301    98   293\n",
      "> 5zi2A                      1   16147    5400     339     335   31.97    0.21E-03      24   108    87   195    76   197\n",
      "> 1bdmA                      1   17070   30700     318     335   44.52     0.0          25   309     1   325     1   310\n",
      "> 4h7pA                      1   19011   26600     313     335   40.85     0.0          26   304     4   324     3   308\n",
      "> 4h7pB                      1   19012   26350     306     335   42.33     0.0          27   298     4   324     2   301\n",
      "> 3hhpA                      1   19442    5950     312     335   27.24    0.11E-04      28   289     7   320     3   303\n",
      "> 4tvoA                      1   20363   26750     328     335   39.94     0.0          29   312     7   324     6   318\n",
      "> 4tvoB                      1   20364   25250     317     335   40.13     0.0          30   303     7   324     5   308\n",
      "> 2hjrA                      1   21755    6200     313     335   26.00    0.31E-05      31   288    13   333    10   309\n",
      "> 1t2dA                      1   21927    5200     315     335   24.70    0.55E-03      32   238     5   256     4   250\n",
      "> 4plgA                      1   22444    6950     324     335   27.67    0.66E-07      33   236     1   256     1   253\n",
      "> 4pltB                      1   22452    8400     319     335   31.48    0.37E-10      34   240    13   271    11   280\n",
      "> 4plvA                      1   22453    6950     305     335   31.25    0.61E-07      35   231    13   271    10   265\n",
      "> 4plzA                      1   22454    4850     302     335   26.11    0.32E-02      36   155    97   256    80   236\n",
      "> 3d0oA                      1   22775    4850     309     335   24.00    0.33E-02      37   212    79   296    71   295\n",
      "> 1y6jA                      1   23859    5750     289     335   32.73    0.29E-04      38   109    77   191    58   167\n",
      "> 1ez4B                      1   24804    6200     318     335   30.68    0.31E-05      39   159    69   239    55   230\n",
      "> 5hs4A                      1   28807    4950     301     335   26.63    0.19E-02      40   167     8   191     4   172\n",
      "> 5nueA                      1   29100   28000     331     335   38.30     0.0          41   327     3   332     3   331\n",
      "> 1mldA                      1   31072    6100     313     335   25.93    0.52E-05      42   183     5   198     1   189\n",
      "> 6j9sB                      1   31764    5450     315     335   26.86    0.15E-03      43   157    71   239    59   233\n",
      "> 6iheA                      1   32500    5000     303     335   29.93    0.15E-02      44   137    80   225    70   206\n",
      "> 6r8gA                      1   34789    6350     311     335   25.74    0.14E-05      45   232    13   257     8   244\n",
      "> 2v65A                      1   35640    4950     326     335   23.76    0.21E-02      46   197    94   300    97   298\n",
      "> 1v6aA                      1   35647    5100     332     335   25.25    0.99E-03      47   197    94   300   103   304\n",
      "> 6balA                      1   36021    6350     312     335   25.55    0.14E-05      48   269     4   295     1   274\n",
      "> 4q3nA                      1   38966    5850     314     335   23.33    0.19E-04      49   232    78   320    70   309\n",
      "> 4uulA                      1   40092   87350     334     335   99.70     0.0          50   332     2   333     1   332\n",
      "> 4uupA                      1   40095   53150     335     335   60.42     0.0          51   331     3   333     1   331\n",
      "> 4uuoA                      1   40100   39200     328     335   49.69     0.0          52   318     3   327     1   318\n",
      "> 2a92A                      1   40298    5450     316     335   25.97    0.15E-03      53   174     8   191     6   186\n",
      "> 6bb1E                      1   40487    5150     310     335   26.63    0.70E-03      54   182    97   295    94   277\n",
      "> 1oc4A                      1   40691    5450     315     335   27.87    0.15E-03      55   174     5   191     4   186\n",
      "> 1smkA                      1   40965    8750     313     335   33.85    0.75E-11      56   188     7   202     4   198\n",
      "> 6vdhA                      1   42414    5900     308     335   32.73    0.14E-04      57   159    13   191    12   176\n",
      "> 4jcoA                      1   43467    5700     303     335   25.96    0.39E-04      58   274     7   320     3   287\n",
      "> 1lldA                      1   44348    6650     313     335   30.67    0.30E-06      59   216    13   242     9   233\n",
      "> 1hyeA                      1   45128    6800     307     335   29.28    0.13E-06      60   173     7   191     3   183\n",
      "> 4wleA                      1   45463    6250     314     335   25.93    0.24E-05      61   186     5   198     2   190\n",
      "> 3tl2A                      1   45519    6550     312     335   28.29    0.50E-06      62   149    39   191    34   185\n",
      "> 3p7mA                      1   46506    6900     318     335   30.77    0.84E-07      63   185    13   207    11   205\n",
      "> 4nd2A                      1   46802    6850     317     335   26.04    0.11E-06      64   270     1   301     1   288\n",
      "> 4nd5A                      1   46815    4850     302     335   25.72    0.32E-02      65   258     1   301     1   276\n",
      "> 5w8jA                      1   47976    5600     329     335   26.21    0.73E-04      66   196    85   295    91   296\n",
      "> 1b8pA                      1   48421   29400     327     335   42.50     0.0          67   316     7   331     6   325\n",
      "> 2ldxA                      1   48665    5400     331     335   25.83    0.21E-03      68   227    66   306    67   306\n",
      "> 9ldtA                      1   48689    6000     331     335   25.94    0.93E-05      69   207    85   301    93   304\n",
      "> 1ldnA                      1   48709    5000     316     335   25.76    0.16E-02      70   214    78   298    73   301\n",
      "> 5ldhA                      1   48727    5800     333     335   25.59    0.26E-04      71   207    85   300    94   304\n",
      "> 6cepA                      1   49739    5750     331     335   25.12    0.34E-04      72   207    85   300    94   304\n",
      "> 6cepD                      1   49740    5250     317     335   25.50    0.43E-03      73   196    96   300    90   289\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "> 1civA                      1   51585   23950     374     335   34.66     0.0          74   325     6   334    33   358\n",
      "> 5nqbA                      1   52324    5650     331     335   25.94    0.57E-04      75   207    85   301    93   304\n",
      "> 6um4A                      1   52654   25750     326     335   37.93     0.0          76   317     7   331     8   326\n",
      "> 4m1qA                      1   57644    5300     312     335   23.64    0.32E-03      77   162    78   242    69   233\n",
      "> 1pzgA                      1   58602    6350     327     335   30.00    0.15E-05      78   114    74   191    71   190\n",
      "> 2x0jA                      1   58672    5700     294     335   27.98    0.38E-04      79   218    78   320    68   285\n",
      "> 5ujkA                      1   59017    6150     315     335   31.58    0.40E-05      80   169    13   191    10   180\n",
      "> 4ajjA                      1   59997    5750     330     335   25.47    0.34E-04      81   207    85   301    92   303\n",
      "> 3nepX                      1   65237    4650     305     335   24.61    0.91E-02      82   244    13   274     8   263\n",
      "> 6pblA                      1   65962   26900     327     335   39.17     0.0          83   313     7   325     5   318\n",
      "> 1y7tA                      1   66077   33200     327     335   45.14     0.0          84   318     1   325     1   319\n",
      "\n",
      "profile_iteration__> Weight Matrix           [sec]:     0.000000\n",
      "profile_iteration__> Score Collection        [sec]:    22.840000\n",
      "profile_iteration__> Score Collection/Seq    [sec]:     0.000340\n",
      "profile_iteration__> DP Steps per sec    [ali/sec]:  2957.394531\n",
      "profile_iteration__> Statistics              [sec]:     0.050000\n",
      "profile_iteration__> Add to profile          [sec]:     0.160000\n",
      "profile_iteration__> Overall performance    [/seq]:     0.000344\n",
      "profile_iteration__> Overall performance [seq/sec]:  2908.744629\n",
      "openf___224_> Open           build_profile.prf\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1121045    1094.771     1.069\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1122381    1096.075     1.070\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1123625    1097.290     1.072\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1124261    1097.911     1.072\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1124981    1098.614     1.073\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1125733    1099.349     1.074\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1126737    1100.329     1.075\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1127793    1101.360     1.076\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1128489    1102.040     1.076\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1129249    1102.782     1.077\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1130341    1103.849     1.078\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1131461    1104.942     1.079\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1132129    1105.595     1.080\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1132781    1106.231     1.080\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1133905    1107.329     1.081\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1134633    1108.040     1.082\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1135377    1108.767     1.083\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1136157    1109.528     1.084\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1137373    1110.716     1.085\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1138033    1111.360     1.085\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1138953    1112.259     1.086\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1140261    1113.536     1.087\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1141557    1114.802     1.089\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1142317    1115.544     1.089\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1142745    1115.962     1.090\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1143977    1117.165     1.091\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1145189    1118.349     1.092\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1146377    1119.509     1.093\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1147529    1120.634     1.094\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1148773    1121.849     1.096\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1149981    1123.028     1.097\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1151129    1124.149     1.098\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1152077    1125.075     1.099\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1153017    1125.993     1.100\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1153973    1126.927     1.101\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1154893    1127.825     1.101\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1155509    1128.427     1.102\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1156353    1129.251     1.103\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1156785    1129.673     1.103\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1157417    1130.290     1.104\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1158081    1130.938     1.104\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1159385    1132.212     1.106\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1160113    1132.923     1.106\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1160737    1133.532     1.107\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1161281    1134.063     1.107\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1162205    1134.966     1.108\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1162989    1135.731     1.109\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1163773    1136.497     1.110\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1164845    1137.544     1.111\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1165769    1138.446     1.112\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1167093    1139.739     1.113\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1168413    1141.028     1.114\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1169681    1142.267     1.115\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1170373    1142.942     1.116\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1171097    1143.649     1.117\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1171789    1144.325     1.118\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1172537    1145.056     1.118\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1173169    1145.673     1.119\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1174261    1146.739     1.120\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1175121    1147.579     1.121\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1175809    1148.251     1.121\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1176549    1148.974     1.122\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1177141    1149.552     1.123\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1177877    1150.271     1.123\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1178953    1151.321     1.124\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1179981    1152.325     1.125\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1180761    1153.087     1.126\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1182021    1154.317     1.127\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1182925    1155.200     1.128\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1183749    1156.005     1.129\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1184601    1156.837     1.130\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1185425    1157.642     1.131\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1186249    1158.446     1.131\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1187029    1159.208     1.132\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1188325    1160.474     1.133\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1189149    1161.278     1.134\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1190413    1162.513     1.135\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1191057    1163.142     1.136\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1191509    1163.583     1.136\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1192377    1164.431     1.137\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1193049    1165.087     1.138\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1193873    1165.892     1.139\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1194845    1166.841     1.139\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1196093    1168.060     1.141\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1197361    1169.298     1.142\n",
      "openf___224_> Open           build_profile.ali\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "\n",
    "log.verbose()\n",
    "env = environ()\n",
    "\n",
    "#-- Prepare the input files\n",
    "\n",
    "#-- Read in the sequence database\n",
    "sdb = sequence_db(env)\n",
    "sdb.read(seq_database_file='pdb_95.pir', seq_database_format='PIR',\n",
    "         chains_list='ALL', minmax_db_seq_len=(30, 4000), clean_sequences=True)\n",
    "\n",
    "#-- Write the sequence database in binary form\n",
    "sdb.write(seq_database_file='pdb_95.bin', seq_database_format='BINARY',\n",
    "          chains_list='ALL')\n",
    "\n",
    "#-- Now, read in the binary database\n",
    "sdb.read(seq_database_file='pdb_95.bin', seq_database_format='BINARY',\n",
    "         chains_list='ALL')\n",
    "\n",
    "#-- Read the target sequence/alignment in .ali format\n",
    "aln = alignment(env)\n",
    "aln.append(file='TvLDH.ali', alignment_format='PIR', align_codes='ALL')\n",
    "\n",
    "#-- Convert the input sequence/alignment into\n",
    "#   profile format\n",
    "prf = aln.to_profile()\n",
    "\n",
    "#-- Scan sequence database to pick up homologous sequences\n",
    "# *** Feel free to make changes in gap penalties, aln evalue and others if you are aware of the significance\n",
    "\n",
    "prf.build(sdb, matrix_offset=-450, rr_file='${LIB}/blosum62.sim.mat',\n",
    "          gap_penalties_1d=(-500, -50), n_prof_iterations=1,\n",
    "          check_profile=False, max_aln_evalue=0.01)\n",
    "\n",
    "#-- Write out the profile in text format\n",
    "prf.write(file='build_profile.prf', profile_format='TEXT')\n",
    "\n",
    "#-- Convert the profile back to alignment format\n",
    "aln = prf.to_alignment()\n",
    "\n",
    "#-- Write out the alignment file\n",
    "aln.write(file='build_profile.ali', alignment_format='PIR')\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Selecting a template"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Number of sequences:     84\r\n",
      "# Length of profile  :    335\r\n",
      "# N_PROF_ITERATIONS  :      1\r\n",
      "# GAP_PENALTIES_1D   :   -500.0   -50.0\r\n",
      "# MATRIX_OFFSET      : -450.0\r\n",
      "# RR_FILE            : ${LIB}/blosum62.sim.mat\r\n",
      "    1 TvLDH                                    S     0   335     1   335     0     0     0    0.    0.0     MSEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKAAFKDIDCAFLVASMPLKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPENFSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKIGHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVVEGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQGG\r\n",
      "    2 3d5tA                                    X     1   321     4   325     3   316   312   44.    0.0     ---AKRVAVTGAAGQIAYSLLFRIANGDLLGDQPVILQLLDLPQAQAAVKGVVMELDDCAFPLLAGVVITDDPKVAFKDADVALLVGARP--RSMERKDLLSANAEIFTVQGAALNEVASRDVKVLVVGNPANTNAYIAMKSAPDLPKKNFTAMLRLDHNRALSQLAAKSGKPVASIEKLAVWGNHSPTMYPDFRFA--TAEGESLKLIN--DDVWNRDTFIPTVGKRGAAIIEARGLSSAASAANAAIDHVRDWVLGT-NGKWVTMGI--PSDGSYGIPEDIIYGVPV-ICENGEYKRVEGLEIDAFSREKMDGTLAELLEERD----------\r\n",
      "    3 1sovA                                    X     1   316    81   256    76   248   160   27.   0.74E-02 --------------------------------------------------------------------------------DVVIITAGLTKVPS--RNDLLPFNAKIIREVAQGVKKYC-PLAFVIVVTNP--LDCMVKCFHEASGLPKNMGMANVLDSARFRRFIADQLEISPRDIQATVI-GTHGDHMLPLARYVTFIKKGKEAKLAEIVERT-------KKAG---GEIVRLLGQGSAYAPALSAITMAQAFL-------------------------------------------------------------------------------\r\n",
      "    4 6k12A                                    X     1   297    94   304    81   270   181   28.   0.88E-02 ---------------------------------------------------------------------------------------------GESRLNLVQRNVNIFKFIIPNVVKYS-PDCILLIVSNPVDILTYVAWKLSGFPLNRVIGSGCNLDSARFRYLVSEMIGIHPSNFHGCIL-GEHGDSSVPILSGLNIAGM-SIKNCKDV----------HKKVTESAYEIIKLKGYTS--------------WAIGLSVGDLLRKVHPVSTKGQFGIDNEVFLSVPCVLGRNG---ISEVFK-------------------------------\r\n",
      "    5 6k13A                                    X     1   332    94   304   103   305   189   28.   0.99E-03 ---------------------------------------------------------------------------------------------GESRLNLVQRNVNIFKFIIPNVVKYS-PDCILLIVSNPVDILTYVAWKLSGFPLNRVIGSGCNLDSARFRYLVSEMIGIHPSNFHGCIL-GEHGDSSVPILSGLNIA--GMSIKNLHT-DIDTVFKDVHKKVTESAYEIIKLKGYTS--------------WAIGLSVGDLLRKVHPVSTKGQFGIDNEVFLSVPCVLGRNG---ISEVFK-------------------------------\r\n",
      "    6 6or9A                                    X     1   306    47   323    37   292   252   25.   0.32E-03 ----------------------------------------------DKLKGELMDLQGSAFMRNVKIQASTDYSISAGSKICV-VTAGVRQREGESRLDLVQRNTDVLKIIIPQLVKHS-PDTILIIASNPVDILTYVSWKLSGLPKHRVIGSGTNLDSARFRYLLSEKLGIATTSCHGYII-GEHGDSSVSGVNIAGVRLSDLNQKINWKETHTMVVKS--------AYEVIKLKGYTSWAIGLSLS-QLARAILSNANSVHAVSTYLK----GEHDINDEVFLSLPCVLGRSG---------VCDVIRQPLTQTERSQLHQ------------\r\n",
      "    7 1guzA                                    X     1   306    13   301     8   280   265   25.   0.28E-07 ------------AGNVGATTAFRLAEKQLARE----LVLLDVVEGIPQGKALDM--------YESGPVGLFDTKADTANSDIVIITAGLPRKPGMTREDLLMKNAGIVKEVTDNIMKHSKNPI-IIVVSNPLDIMTHVAWVRSGLPKERVIGMAGVLDAARFRSFIAMELGVSMQDINACVL-GGHGDAMVPVVKYTTVA----GIPISDLLPAETI-DKLVERTRNGGAEIVEHLKQGSAYAPASSVVEMVESIVLDRKRVLPCAVGL---EGQ-YGIDKTFV-GVPVKLGRNGVEQIYE----------------------------------\r\n",
      "    8 3gviA                                    X     1   314    13   209    10   206   175   29.   0.25E-03 ------------SGMIGGTLAHLAGLKEL-GD----VVLFDIAEGTPQGKGLDIAESSPVDGFDAKFTGAND-YAAIEGADVVIVTAGVPRKD-----DLLGINLKVMEQVGAGIKKYA-PEAFVICITNP---------LDAMVWALQKFSGLGVLDSARFRYFLSEEFNVSVEDV-TVFVLGGHGDSMVPDLVKMGWTSQDKLDKII------------------------------------------------------------------------------------------------------------------------------\r\n",
      "    9 1gv1B                                    X     1   290    99   301    80   271   191   23.   0.23E-02 --------------------------------------------------------------------------------------------------DLLMKNAGIVKEVTDNIMKHSKNPI-IIVVSNPLDIMTHVAWVRSGLPKERVIGMAGVLDAARFRSFIAMELGVSMQDINACVL-GGHGDAMVPVVKYTTVA----GIPISDLLPAETI-DKLVERTRNGGAEIVEHLKQGSAYAPASSVVEMVESIVLDRKRVLPCAVGL---EGQ-YGIDKTFV-GVPVKLGRNGVEQIYE----------------------------------\r\n",
      "   10 1gv0A                                    X     1   301    13   323     8   289   274   26.   0.24E-03 ------------AGNVGATTAFRLAEKQLARE----LVLLDVVEGIPQGKALDM--------YESGPVGLFDTKADTANSDIVVITAGLPRKPGMT----LSMNAGIVREVTGRIMEHSKNPI-IVVVSNPLDIMTHVAWQKSGLPKERVIGMAGVLDSARFRSFIAMELGVSMQDV-TACVLGGHGDAMVPVVKYTTVA----GIPVADLISAERIAE-LVERTRTGGAEIVNHLKQGSAYSPATSVVEMVESIVLDRK--RVLTCAVSL--DGQYGI-DGTFVGVPVKLGKNGVEHIYE---------IKLDQSDLDLLQK------------\r\n",
      "   11 4bguA                                    X     1   303     7   314     3   289   281   26.   0.62E-06 ------VSVIGAAGTVGAAAGYNLALRDVCDE----LVFVDIPKMEDKTVGQAADTNH-GIAYDSNTVVTQGGYEDTAGSDVVVITAGIPRQPGQTRIDLAGDNAPIMDDIGSSLAEYNDDFVSITTSNPVDLLNRHLYETGDRD-RHKVIGFGGRLDSARFRYVLSQRFDVPVKNV-DATILGEHGDAQV-----PVFSK-------VRVDGNDPAFSADEKELQESAMDVIERKGATQWGPAT--GVAHMEAVLHDT--GEVLP-GSLVLDGE-FGYE-DTAFGVPVKLGSNGIEEVVE-WDLDDYEADLMD---------------------\r\n",
      "   12 4bgvA                                    X     1   323    13   191    10   185   168   32.   0.18E-08 ------------AGAVGATVAQTLAIRQT-GD----IYIFDIVDGLAEGKALDI-LEGA--PHIKGFCTADESKAEMKGSDVIVVTAGLARKPGMSRDDLLLKNIGIMKSVGEAIKKYS-PESKIVVVTNPADI-MAYAIYKASGISPERIIGLGSLDSTRFRTFLAQELNVSFEDVNAFVI-GGHGDDMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   13 1a5zA                                    X     1   312    75   242    63   229   164   28.   0.82E-07 --------------------------------------------------------------------------ADLKGSDVVIVAAGVPQKPGETRLQLLGRNARVMKEIARNVSKYAPDSI-VIVVTNPVDV-LTYFFLKESGMDPRKFGSGTVLDTARLRTLIAQHCGFSPRSVH-VYVIGEHGDSEV-PVWSGAMIGGIPLQNMCQVCQDSKILENFAEKTKRAAYEIIERKGATHYA---------------------------------------------------------------------------------------------\r\n",
      "   14 4e0bA                                    X     1   305     4   314     2   288   282   28.   0.40E-04 ---AMKVAVIGAAGGIGQALALLLKNRLPAGSD---LALYDIAPVTPGVAADLSHIP--THVSIKGY-AGEDPTPALEGADVVLISA------GVARADLFNVNAGIVKSLAERIAV-VCPNACIGIITNPVNTTVPIALKKAGVYDKRKLFGVTTLDVIRSETFVAELKGQDPGEVRVPVIGGHSGVTILPLLSQV----EG-----VEFSDEEIAALT--KRIQNAGTEVVEAKGGSATLSMGQAACRFGLALVKALQGEEVIEYA--YVEGNG---EHASFFAQPVKLGKEGVEEILPYGELSDFEKAALD---------------------\r\n",
      "   15 4ln1A                                    X     1   317    78   270    79   278   183   24.   0.34E-02 -----------------------------------------------------------------------------KDADLVVITAGLP----QTRLDLVEKNAKIFKQIVRSIMDSGFDGI-FLIATNPVDILTYVTWKESGLPKERVIGSGTTLDSARFRYMLGEYFDIGPHNIHAYII-GEHGDT---ELPVWSHVSVG-IQKLQTLLEKDNTYDKIFINVRDAAYHIIERKGATSLLRVTKASVLTVSAYLEGQYGQKDVYIGVP-----------------------------------------------------------------\r\n",
      "   16 3wswA                                    X     1   316    78   270    70   273   187   25.   0.25E-04 -----------------------------------------------------------------------------QDANLVILTAGVNQKPGQTRLDLVKTNATITRQIVKEVMASGFDGIFV-VASNPVDILTYLTWQESGLPASRVVGTGTTLDTTRFRKEIALKLAVDPRSVHGYIL-GEHGDSEVAAWSHTTV--GGK--PIMEYVEKDHRLEVLADKVKNAAYEIIDRKKATKAILNNEQAVLPVSAYLNGEYGEEDIFTGVP-----------------------------------------------------------------\r\n",
      "   17 6itlA                                    X     1   312     7   209     3   201   196   27.   0.14E-05 ------VAVLGAAGGIGQALALLL---KLQLPAGSSLSLYDVAPV---TPGVAKDLSHIPTDVVVEGFAGTDPSEALKGADIVLISAGVARKPGMTRADLFGVNAGIIRSLTEKVAEQC-PKACVGIITNPVNAMVAIALKKAGVYDKRKLFGITTLDILRAETFIAELKGLDPTRVTIPVIGGHSGVTILPLLSQVQNVEWSSEEEII------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   18 6itkA                                    X     1   318     7   325     5   310   305   41.    0.0     ------VTVTGAAGQISYSLLWRIANGEVFGDTPVELKLLEIPQALGGAEGVAMELLDSAFPLLRNITITADANEAFDGANAAFLVGAKP-----ERADLLANNGKIFGPQGKAINDNAADDIRVLVVGNPANTNALIASAAAPDVPASRFNAMMRLDHNRAISQLATKLGRGSAEFNNIVVWGNHSATQFPDITYATVGGE----KVTDLVDHDWYVEEFIPRVANRGAEIIEVRGKSSAASAASSAIDHMRDWVQGT---EAWS-SAAIPSTGAYGIPEGIFVGLP-TVSRNGEWEIVEGLEISDFQRARIDANAQELQAERE----------\r\n",
      "   19 4cl3A                                    X     1   309    13   191    10   180   166   32.   0.23E-06 ------------AGFVGSTTAHWLAAKEL-GD----IVLLDIVEGVPQGKAL--DLYEAS--PIEGFVTGTNNYADTANSDVIVVTSGAPRKPGMSREDLIKVNADI---TRACISQAASPNAVIIMVNNPLDAMTYLAAEVSGFPKERVIGQAGVLDAARYRTFIAMEAGVSVEDVQAMLM-GGHGDEMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   20 3fi9A                                    X     1   321     9   260    12   248   231   28.   0.22E-10 --------IVGAAGMIGSNMAQTAAMMRLTPNLCLY------DPFAVGLEGVAEEIRHCGFEGL-NLTFTSDIKEAL--TDAKYIVSS-----GGTREDLLKGNAEIAAQLGKDIKSYCPDCKHVIIIFNPADITGLVTLIYS-GLKPSQVTTLAGLDSTRLQSELAKHFGIKQSLVTNTRTYGGHGEQM------AVFASTAKVNPLTDLIGTDKLTNEQWAELKQRGANIIKLRGRSSFQSPSYVSIEMIRAAMGGEA---------------------------------------------------------------------------\r\n",
      "   21 5mdhA                                    X     1   333     2   332     1   331   328   44.    0.0     -SEPIRVLVTGAAGQIAYSLLYSIGNGSVFGDQPIILVLLDITPMMGVLDGVLMELQDCALPLLKDVIATDKEEIAFKDLDVAILVGSMPRRDGMERKDLLKANVKIFKCQGAALDKYAKKSVKVIVVGNPANTNCLTASKSAPSIPKENFSCLTRLDHNRAKAQIALKLGVTSDDVKNVIIWGNHSSTQYPDVNHAKVKLQAKEVGVYEVKDDSWLKGEFITTVQQRGAAVIKARKLSSAMSAAKAICDHVRDIWFGTPEGEFVSMGI-ISDGNSYGVPDDLLYSFPVTIDKTWKI--VEGLPINDFSREKMDLTAKELAEEKETAFEFLS---\r\n",
      "   22 7mdhA                                    X     1   351     6   334    14   339   325   34.    0.0     -----NIAVSGAAGMISNHLLFKLASGEVFGDQPIALKLLGSERSFQALEGVAMELEDSLYPLLREVSIGIDPYEVFEDVDWALLIGAKPRGPGMERAALLDINGQIFADQGKALNAVASKNVKVLVVGNPCNTNALICLKNAPDIPAKNFHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDFLNAKI--DGRPVKEV-IKRTKWLEEEFTITVQKRGGALIQKWGRSSAASTAVSIADAIKSLVTPTPEGDWFSTGV-YTTGNPYGIAEDIVFSMPCRSKGDGDYELATDVSNDDFLWERIKKSEAELLAEKKCVAHLTGEG-\r\n",
      "   23 5kkcB                                    X     1   320   101   301    98   293   191   26.   0.34E-02 ----------------------------------------------------------------------------------------------------VQRNVNIFKFIIPNVVKYS-PHCKLLVVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHALSCHGWIL-GEHGDSSVSGMNVAGVSLKTLHPELGTDADKEQ-WKQVHKQVVDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKEDVFLSVPCVLGQNGISDVVK----------------------------------\r\n",
      "   24 5zi2A                                    X     1   339    87   195    76   197   108   32.   0.21E-03 --------------------------------------------------------------------------------------AGVPRKPGLTRDDLFKMNAGIVKSLVTAVGKFA-PNARILVISNPVNSLVPIAVETLKKMKPGNVMGVTNLDLVRAYLMLKNKIGQDKTTMHRTVIGGHSGETIIPIIT--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   25 1bdmA                                    X     1   318     1   325     1   310   309   45.    0.0     MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGDQPVILQLLEIPQAMKALEGVVMELEDCAFPLLAGLEATDDPDVAFKDADYALLVGAAP---------RLQVNGKIFTEQGRALAEVAKKDVKVLVVGNPANTNALIAYKNAPGLNPRNFTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSIMFPDL----FHAEVDGRPALELVDMEWYEKVFIPTVAQRGAAIIQARGASSAASAANAAIEHIRDWALGTPEGDWVSMA--VPSQGEYGIPEGIVYSFPVTA-KDGAYRVVEGLEINEFARKRMEITAQELLDEME----------\r\n",
      "   26 4h7pA                                    X     1   313     4   324     3   308   304   41.    0.0     ---AVKVAVTGAAGQIGYALVPLIARGALLGTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKVVVTADPRVAFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLKSAQKLNPRHVTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVIGTTPAREAIKD----------FVQVVRGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDE-NPYGVPSGLIFSFPCT------CHAGEWTVVSGKLKQRLASTIAELQEER-----------\r\n",
      "   27 4h7pB                                    X     1   306     4   324     2   301   298   42.    0.0     ---AVKVAVTGAAGQIGYALVPLIARGALLGTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKVVVTADPRVAFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLKSAQKLNPRHVTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVI---GTTPDFVQV-------------VRGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDE-NPYGVPSGLIFSFPCT------CHAGEWTVVSGKLKQRLASTIAELQEER-----------\r\n",
      "   28 3hhpA                                    X     1   312     7   320     3   303   289   27.   0.11E-04 ------VAVLGAAGGIGQALALLLKTQLPSGSE---LSLYDIAPVTPGVAVDLSHIPTAV--KIKGF-SGEDATPALEGADVVLISAGVARKPGMDRSDLFNVNAGIVKNLVQQVAKTC-PKACIGIITNPVNTTVAIAVLKKAGVYDKNLFGVTTLDIIRSNTFVAELKGKQPGEVEVPVIGGHSGVTILPLLSQVSFTE----QEVADLT----------KRIQNAGTEVVEAKGGGSATLSMGQAAARFGLSLVRALQGEQGVVECAYVEGDGQYAR---FFSQPLLLGKNGSIGTLSAFEQNA-LEGMLDTLKKDI---------------\r\n",
      "   29 4tvoA                                    X     1   328     7   324     6   318   312   40.    0.0     ------VAVTGAAGQIGYSLLFRLASGSLLGDRPIELRLLEIEPALQALEGVVMELDDCAFPLLSGVEIGSDPQKIFDGVSLALLVGARPRGAGMERSDLLEANGAIFTAQGKALNAVAADDVRVGVTGNPANTNALIAMTNAPDIPRERFSALTRLDHNRAISQLAAKTGAAVTDIKKMTIWGNHSATQYPDLFHAEVA--GKNAAEV-VNDQAWIEDEFIPTVAKRGAAIIDARGASSAASAASATIDAARDWLLGTPADDWVSMAV-VSDGS-YGVPEGLISSFPVTT-KGGNWTIVSGLEIDEFSRGRIDKSTAELADER-----------\r\n",
      "   30 4tvoB                                    X     1   317     7   324     5   308   303   40.    0.0     ------VAVTGAAGQIGYSLLFRLASGSLLGDRPIELRLLEIEPALQALEGVVMELDDCAFPLLSGVEIGSDPQKIFDGVSLALLVGARP---------LLEANGAIFTAQGKALNAVAADDVRVGVTGNPANTNALIAMTNAPDIPRERFSALTRLDHNRAISQLAAKTGAAVTDIKKMTIWGNHSATQYPDLFHAEVA--GKNAAEV-VNDQAWIEDEFIPTVAKRGAAIIDARGASSAASAASATIDAARDWLLGTPADDWVSMAV-VSDGS-YGVPEGLISSFPVTT-KGGNWTIVSGLEIDEFSRGRIDKSTAELADER-----------\r\n",
      "   31 2hjrA                                    X     1   313    13   333    10   309   288   26.   0.31E-05 ------------AGQIGSTIALLLGQKDL-GD----VYMFDIIEGVPQGKAL--DLNHCALIGSPAKIFGENNYEYLQNSDVVIITAGVPRKPNMTRSDLLTVNAKIVGSVAENVGKYC-PNAFVICITNP--LDAMVYYFKEKSGIPANCGMSGVLDSARFRCNLSRALGVKPSDVSAIVV-GGHGDEMIPLTSSVTFVEQGK-------ITHSQI-NEIIKKTAFGGGEIVELLKTGSAYAPAASAVAMAQAYLKDSKSVLVCSTYLT----GQYNVN-NLFVGVPVVIGKNG---------IEDVVIVNLSDDEKSLFSKSVESIQNLVQ--\r\n",
      "   32 1t2dA                                    X     1   315     5   256     4   250   238   25.   0.55E-03 ----AKIVLVG-SGMIGGVMATLIVQKNL-GDVVLFDIVKNMPKALDTSHTNVMAYSNC---KVSGSNTYDDLAGA----DVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHIKKNC-PNAFIIVVTNP--VDVMVQLLHQHSGVPKNIGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMVLLKRYITVGGIPLQEFINNKLISDAELEAIFDRTVNTALEIVNLHA-SPYVAPAAAIIEMAESYL-------------------------------------------------------------------------------\r\n",
      "   33 4plgA                                    X     1   324     1   256     1   253   236   28.   0.66E-07 MTKRKKISLIG-SGMIGGTMAYLCAQKEL-GD----VVLFDVVKNMPQGKALDLS-HSSSIADTNVKVTGTNSYEDIKGSDVVIITAGLTKAPGKSRDDLLPFNAKIMREVGENIKKYC-PNAFVIVITNP--LDVMVKVLHEHSGLPKNCGMAGVLDSSRFRHFIAEKLNVSPRDVQAMVI-GAHGDKMVPLTRYVTFIKKGRTQEEIDEI---------VERTKNAGGEIVNLLGQGSAFAPAASAIEMAEAYL-------------------------------------------------------------------------------\r\n",
      "   34 4pltB                                    X     1   319    13   271    11   280   240   31.   0.37E-10 ------------AGNIGGTLAHLIAQKEL-GD----VVLFDIVEGMPQGKALDISHSS---PIMGSNVKITGTN-NYEDIDVVIITAGIPRKPGKSRDDLLSVNAKIMKDVAENIKKYC-PNAFVIVVTNP--LDVMVYVLHKYSGLPHNCGMAGVLDSSRFRYFLAEKLNVSPNDVQAMVI-GGHGDTMVGGIPLTEFIKQGITQEEID----EIVERT--RNAGGEIVNLLKTGSFAPAASAIEMAESYLKAYLEGQYGVKDLFVGVPV----------------------------------------------------------------\r\n",
      "   35 4plvA                                    X     1   305    13   271    10   265   231   31.   0.61E-07 ------------AGNIGGTLAHLIAQKEL-GD----VVLFDIVEGMPQGKALDISHSS---PIMGSNVKITGTN-NYEDIDVVIITAGI---------DLLSVNAKIMKDVAENIKKYC-PNAFVIVVTNP--LDVMVYVLHKYSGLPHNCGMAGVLDSSRFRYFLAEKLNVSPNDVQAMVI-GGHGDTMVGGIPLTEFIKQGITQEEID----EIVERT--RNAGGEIVNLLKTGSFAPAASAIEMAESYLKAYLEGQYGVKDLFVGVPV----------------------------------------------------------------\r\n",
      "   36 4plzA                                    X     1   302    97   256    80   236   155   26.   0.32E-02 ------------------------------------------------------------------------------------------------RDDLLPLNNKIMIEIGGHIKKNC-PNAFIIVVTNP--VDVMVQLLHQHSGVPKNIGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMVLLKRYITVGGIPLQEFINNKLISDAELEAIFDRTVNTALEIVNLHA-SPYVAPAAAIIEMAESYL-------------------------------------------------------------------------------\r\n",
      "   37 3d0oA                                    X     1   309    79   296    71   295   212   24.   0.33E-02 ------------------------------------------------------------------------------DADLVVICAGAAQKPGETRLDLVSKNLKIFKSVGEVMA--SKFDGIFLVATNPVDILAYATWKFSGLPKERVIGSGTILDSARFRLLLSEAFDVAPRSV-DAQIIGEHGDTELPVWSHANIAGQPLKTLLEQRPEGKAQIEQIFVQTRDAAYDIIQAKGITEAIFRNEDAVLTVSALLEGEYEEEDVYIGVPAVI-NRNGIRN--VVEIPLNDEEQSK---------------------------------------\r\n",
      "   38 1y6jA                                    X     1   289    77   191    58   167   109   33.   0.29E-04 ----------------------------------------------------------------------------YSDVDCDVIVVT----AGATRLDLAKKNVMIAKEVTQNIMKYYNHGV-ILVVSNPVDIITYMIQKWSGLPVGKVIGSGTVLDSIRFRYLLSEKLGVDVKNVHGYII-GEHGDSQL------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   39 1ez4B                                    X     1   318    69   239    55   230   159   31.   0.31E-05 --------------------------------------------------------------------AFTAPKKIYKDADLVVITAGAPQKPGESRLDLVNKNLNI-------LSSIVKPVVDFLVAANPVDILTYATWKFSGFPKERVIGSGTSLDSSRLRVALGKQFNVDPRSV-DAYIMGEHGDSEFAAYSTATI----GTRPVRDVAKEQGVSDDDLAKVRNKAYDIINLKGAT------------------------------------------------------------------------------------------------\r\n",
      "   40 5hs4A                                    X     1   301     8   191     4   172   167   27.   0.19E-02 -------IVLVGSGMIGGVMATLIVQKNL-GDVVMFDVVKNMPKALDTSHSNVMAYSNC---KVTGSNSYDDLKGA----DVVIVTAGF-------RDDLLPLNNKIMIEIGGHIKNLC-PNAFIIVVTNPVDVMVQLLFEHSGVPKNKIIGLGGVLDTSRLKYYISQKLNVCPRDVNALIV-GAHGNKMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   41 5nueA                                    X     1   331     3   332     3   331   327   38.    0.0     --EPVRVLVTGAAGQIGYALVPMIARGIMLGDQPVILHMLDIPPAAEALNGVKMELIDAAFPLLKGVVATTDAVEGCTGVNVAVMVGGFPRKEGMERKDVMSKNVSIYKSQAAALEKHAAPNCKVLVVANPANTNALILKEFAPSIPEKNISCLTRLDHNRALGQISERLSVPVSDVKNVIIWGNHSSSQYPDVNHAKVQTSSGEKPVRELVKDDAWLDEFISTVQQRGAAIIKARKLSSALSAASSACDHIRDWVLGTPEGTFVSMGV-YSDGS-YSVPSGLIYSFPVTC-RNGDWSIVQGLPIDEVSRKKMDLTAEELKEEKDLAYSCLS---\r\n",
      "   42 1mldA                                    X     1   313     5   198     1   189   183   26.   0.52E-05 ----AKVAVLGASGGIGQPLSLLLKNSPLVSRLTLYAHTPGVAADLSHITRATVK----------GYLGPEQLPDCLKGCDVVVIPAGVPRKPGMTRDDLFNTNATIVATLTAACAQHC-PDAMICIISNPVNSTIPIAEVFKKHYNPNKIFGVTTLDIVRANAFVAELKGLDPARVSVPVIGGHAGKTIIPLISQCT-----------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   43 6j9sB                                    X     1   315    71   239    59   233   157   27.   0.15E-03 ----------------------------------------------------------------------TSPKKIYKDADLVVITAGAPQKPGETRLDLVNKNLKILKSIVDPIVDSGFNGI-FLVAANPVDILTYATWKLSGFPKNRVVGSGTSLDTARFRQSIAKMVNVDARSVHAYIM-GEHGDANIGGVTIAEWVKEIKEDKLVKM----------FEDVRNKAYEIIKLKGAT------------------------------------------------------------------------------------------------\r\n",
      "   44 6iheA                                    X     1   303    80   225    70   206   137   30.   0.15E-02 -------------------------------------------------------------------------------MDIVVISAGKPRKPGMSRRDLFVDNAKIMIDLAQKLPS-KNPGAIYLMVANPVDMMASVFMKYSKQF---TISAGDQVETMRMRSFIAKKLKIPVTSV-DGFVGGEHGEDAV--VLWSTVKIKGKPVDEFNI-NKDEVSD-YVKKI--------------------------------------------------------------------------------------------------------------\r\n",
      "   45 6r8gA                                    X     1   311    13   257     8   244   232   26.   0.14E-05 ------------SGQIGAIVGE-LCLLENLGD----LILYDVVPGIPQGKALDLK----HFSTILGVILGTNQIEDIKDADIIVITAGVQRKEGMTREDLIGVNGKIMKSVAESVKLHCSKAF-VICVSNP--LDIMVNVFHKSNLPHEKICGMAILDTSRYCSLIADKLKVSAEDVNAVIL-GGHGDLMVPLQRYTSVNGVPLSEFVKKNMISQNEIQEIIQKTRNMGAEIIKLAKASAAFAPAAAITKMIKSYLY------------------------------------------------------------------------------\r\n",
      "   46 2v65A                                    X     1   326    94   300    97   298   197   24.   0.21E-02 ---------------------------------------------------------------------------------------------GESRLNLVQRNVNIFKFIIPNIVKYS-PNCILMVVSNPVDILTYVAWKLSGFPRHRVIGSGTNLDSARFRHLIGEKLHLHPSSCHAWIV-GEHGDSSVSGVNVAGVSLQGLNPQMGTEGDGEN-WKAIHKEVVDGAYEVIKLKGYTSWAIGMSVAVESIIKNMHKVHPVSTLVQGM-------HGVKDEVFLSVPCVLGNSGLTDVI-----------------------------------\r\n",
      "   47 1v6aA                                    X     1   332    94   300   103   304   197   25.   0.99E-03 ---------------------------------------------------------------------------------------------GESRLNLVQRNVNIFKFIIPNIIKYS-PNCILLVVSNPVDILTYVAWKLSGLPRNRVIGSGTNLDSARFRHLMGEKLGIHPSNCHGWVI-GEHGDSSVSGVNVAGVFLQGLNPDMGTDKDKE-DWKSVHKMVVDSAYEVIKLKGYTSWAIGMSAACQSILKNLRKCHPVSTLVKGM-------HGVNEEVFLSVPCILGNSGLTDVV-----------------------------------\r\n",
      "   48 6balA                                    X     1   312     4   295     1   274   269   26.   0.14E-05 ---AMKVAVLGAAGGIGQALALLL---KLQLPAGTDLSLYDIAPVTPGVAVDVSHIPTAV--NVKGF-SGEDPTPALEGADVVLISAGVARKPGMDRSDLFNINAGIVRGLIEKVAVTC-PKACVGIITNPVNTTVAIALKKAGVYDKRKLFGVTTLDVLRSETFVAELKGLNVSRTSVPVIGGHSGVTILPLLSQVQYAKWNEDE-----------IEPLTKRIQNAGTEVLNAKGGSATLSMAQAAARFARSLVKGLSGETVVECTYVEGDG-----KYARFFSQPVRLGKEG----------------------------------------\r\n",
      "   49 4q3nA                                    X     1   314    78   320    70   309   232   23.   0.19E-04 -----------------------------------------------------------------------------KDAGLVVITAGVNQKEGETRIHLLKRNAEIIKTVVKNIMSSGFNGI-LLVASNPVDVLAYVAWKESGLHHSRVIGSGTSLDTARLRYEISQYIHIDSRNVHAYIL-GEHGDSEFVCWSNANV----GAKPIADVIDSDEILEHIYLKVRDAAYEIISRKKATYYG--IGMALVRITAAIFN---NENRILPISVLNNGVYDCEDQVYIGLPAVLNKDGVHHIVKNEKENSQLNNSANILRKNL---------------\r\n",
      "   50 4uulA                                    X     1   334     2   333     1   332   332  100.    0.0     -SEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKAAFKDIDCAFLVASMPRKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPENFSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKIGHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVVEGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQ--\r\n",
      "   51 4uupA                                    X     1   335     3   333     1   331   331   60.    0.0     --EPIHVLITGAAGQIGYALAFRIAKGDLFGDRKVVLHLLEIPPAMKALEGVCMELQDCAFPTLAGVVATDDPEEAFKDVDVAFLVGSFPRKPGMERADLLEKNAGIFKVQGKALSEYAKPTVKVLVVGNPANTNCLIAMANAPKLGPENFSAMTRLDHNRAIGEIAAKLGVPVDKVHNVVVWGNHSNTQVPDVSHATVDKEGGTKKVSDALPKEYLEGEFVQKIAQRGGAVIEARGASSAASAANAALCHMRDWLFGTKPGDWVSMGIPVPEGNPYGIKPGVIYSFPCTVDKDGKVHIVEGLEINDWVREKMEATEKELIEERETAFKVLAQ--\r\n",
      "   52 4uuoA                                    X     1   328     3   327     1   318   318   50.    0.0     --EPIHILITGAAGQIGYALTFRIAKGDLCGDRKVVLHLLEIPFGMKALEGCVMELQDCAFPNVAGIVWTDKVEEAFKGVDVAFLVGSFPRKDGMDRSDLLAKNGGIFTVQGKALNDYAKPTVKVLVVGNPANTNCLIAQASAPKLQNKNWCAMTRLDHNRMVGALAAKFGVTPEKIHKVCIWGNHSNTQVPDTTHATV----DTVKVADKLPKEYLEGEFAQMIATRGGAVIKMRGASSAASAANAALTCVKDWLYGT---DFVSMAIPVPDNEPYGIKQGTIFSFPVTVSKDGEVHVVEGLELNDWVKGRLEATEKELIGEKETA--------\r\n",
      "   53 2a92A                                    X     1   316     8   191     6   186   174   26.   0.15E-03 -------IVLVGSGMIGGVMATLIVQKNL-GDVVMFDVVKNMPKALDTSHSNVMAYSNC---KVTGSNSYDDLKGA----DVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHIKNLC-PNAFIIVVTNPVDVMVQLLFEHSGVPKNKIIGLGGVLDTSRLKYYISQKLNVCPRDVNALIV-GAHGNKMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   54 6bb1E                                    X     1   310    97   295    94   277   182   27.   0.70E-03 ------------------------------------------------------------------------------------------------RLNLVQRNVNIFKFIIPNVVKYS-PNCKLLIVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVP--VWSGMNVAGVSLKTL------HPDKEVHKQVVESAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPVSTMIKGL-------YGIKDDVFLSVPCILGQNG----------------------------------------\r\n",
      "   55 1oc4A                                    X     1   315     5   191     4   186   174   28.   0.15E-03 ----AKIVLVG-SGMIGGVMATLIVQKNL-GDVVMFDIVKNMPKALDTSHTNVMAYSNCK-------VSGSNTYDDLKDADVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHI-KNNCPNAFIIVVTNP--VDVMVQLLHQHSGVPKNVGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   56 1smkA                                    X     1   313     7   202     4   198   188   34.   0.75E-11 ------VAILGAAGGIGQPLAMLMKMNPLVS----VLHLYDVVNAPG-VTADISHMDTGAV--VRGFLGQQQLEAALTGMDLIIVPAGVPRKPGMTRDDLFKINAGIVKTLCEGIAKCC-PRAIVNLISNPVNSTVPIAFKKAGTYDPKRLLGVTMLDVVRANTFVAEVLGLDPRDVDVPVVGGHAGVTILPLLSQSSFTQE-------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   57 6vdhA                                    X     1   308    13   191    12   176   159   33.   0.14E-04 ------------SGMIGGTMAYLCALKEL-GD----VVLFDVVKNMPQGKALDLS-------HSTS-VADTNVKVSYEDIKGSDVVI---ITAGLSRDDLLPINAKIMKEVGENIKKYC-PNAFVICITNP--LDVMVKVLQEASGLPHNCGMAGVLDSSRFRYFIAEKLNVSPRDVQAMVI-GAHGDNMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   58 4jcoA                                    X     1   303     7   320     3   287   274   26.   0.39E-04 ------VSVVGAAGTVGAAAGYNIALRDI-ADEVVF---VDIPDKEDDTVGQAADTN-----HGIAYDSNTRVRGGYEDTDVVVITAGIPRQPGQTRIDLAGDNAPIMEDIQSSLDEHNDDYISLTTSNPVDLLNRHLYEAGDRS-REQVIGFGGRLDSARFRYVLSEEFDAPVQNVEGTIL-GEHGDAQVVDGTDPEFSGDEKEQLLGDLQE--------------SAMDVIERKGATE-WGPARGVAHMVEAILHDT--GEVLPASVKL-EGE-FG-HEDTAFGVPVRLGSNGVEEIVE------W---DLDDYEQDL---------------\r\n",
      "   59 1lldA                                    X     1   313    13   242     9   233   216   31.   0.30E-06 ------------AGAVGSTLA--FAAAQRGIAREIVLE--DI--AKERVEAEVLDMQGSSFYPTVSIDGSDDPEIC-RDADMVVITAGPRQKPGQSRLELVGATVNILKAIMPNLVKVA-PNAIYMLITNP----VDIATHVAQKLTPENFGSGTNLDSARLRFLIAQQTGVNVKNVHAYIA-GEHGDSEVPLWESATIMSDWTPLPGHDPLDAD-KREEIHQEVKNAAYKIINGKGATNYA---------------------------------------------------------------------------------------------\r\n",
      "   60 1hyeA                                    X     1   307     7   191     3   183   173   29.   0.13E-06 ------VTIIGASGRVGSATALLLAKEPFMKD----LVLIGREHSINKLEGLREDIYDA----LAGYVESDENLRIIDESDVVIITSGVPRKEGMSRMDLAKTNA---KIVGKYAKKIAECDTKIFVITNPVDVMTYKALVDSKFERNQVFGLGTHLDSLRFKVAIAKFFGVHIDEVRTRII-GEHGDSMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   61 4wleA                                    X     1   314     5   198     2   190   186   26.   0.24E-05 ----AKVAVLGASGGIGQPLSLLLKNSPLVSRLTLY-DIAHTPGVAADLSHIETKAA------VKGYLGPEQLPDCLKGCDVVVIPAGVPRKPGMTRDDLFNTNATIVATLTAACAQHC-PEAMICVIANPVNSTIPIAEVFKKHYNPNKIFGVTTLDIVRANTFVAELKGLDPARVNVPVIGGHAGKTIIPLISQCT-----------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   62 3tl2A                                    X     1   312    39   191    34   185   149   28.   0.50E-06 --------------------------------------LVDIPQLENPTKGKALDMLEAS--PVQGFDANTSDYADTADSDVVVITAGIARKPGMSRDDLVATNSKIMKSITRDIAKHS-PNAIIVVLTNPVDAMTYSVFKEAGFPKERVIGQSGVLDTARFRTFIAQELNLSVKDITGFVL-GGHGDDMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   63 3p7mA                                    X     1   318    13   207    11   205   185   31.   0.84E-07 ------------AGNIGGTLAH-LALIKQLGD----VVLFDIAQGMPNGKALDL-LQTCPIEGVDFKVRGTNDYKDLENSDVVIVTAGVPRKPGMSRDDLLGINIKVMQTVGEGI-KHNCPNAFVICITNP--LDIMVNMLQKFSGVPDNVGMAGVLDSARFRTFLADELNVSVQQV-QAYVMGGHGDTMVAGVSLEQLVKEGKLKQ--------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   64 4nd2A                                    X     1   317     1   301     1   288   270   26.   0.11E-06 MIERRKIAVIG-SGQIGGNIAYIVGK-----DNLADVVLFDIAEGIPQGKALDITHSMVMFGSTSKVIGTND-YADISGSDVVIITASIPGRPKDDRSELLFGNARILDSVAEGVKKYC-PNAFVICITNP----LDVMVSHFQKVSPHNCGMAGVLDSSRFRTFIAQHFGVNASDVSANVI-GGHGDGMVGGVPLSSFIKQGLTQEQIDIVCHTRI-----------AWKEVADNLKTGTAAPAAAAVKMAEAYL------KDKKAVVPCSASNHYGVK-GIYMGVPTIIGKNGVEDILE----------------------------------\r\n",
      "   65 4nd5A                                    X     1   302     1   301     1   276   258   26.   0.32E-02 MIERRKIAVIG-SGQIGGNIAYIVGK-----DNLADVVLFDIAEGIPQGKALDITHSMVMFGSTSKVIGTND----YADISGSDVV---------IITALLFGNARILDSVAEGVKKYC-PNAFVICITNP----LDVMVSHFQKVSPHNCGMAGVLDSSRFRTFIAQHFGVNASDVSANVI-GGHGDGMVGGVPLSSFIKQGLTQEQIDIVCHTRI-----------AWKEVADNLKTGTAAPAAAAVKMAEAYL------KDKKAVVPCSASNHYGVK-GIYMGVPTIIGKNGVEDILE----------------------------------\r\n",
      "   66 5w8jA                                    X     1   329    85   295    91   296   196   26.   0.73E-04 ------------------------------------------------------------------------------------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PNCKLLIVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVAGVSLKTLHPDLGTDK-----DKEQ-WKEVHKQVVESAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPVSTMIKGL-------YGIKDDVFLSVPCILGQNG----------------------------------------\r\n",
      "   67 1b8pA                                    X     1   327     7   331     6   325   316   42.    0.0     ------VAVTGAAGQICYSLLFRIANGDMLGDQPVILQLLEIPKAQKALQGVMMEIDDCAFPLLAGMTAHADPMTAFKDADVALLVGARPRGPGMERKDLLEANAQIFTVQGKAIDAVASRNIKVLVVGNPANTNAYIAMKSAPSLPAKNFTAMLRLDHNRALSQIAAKTGKPVSSIEKLFVWGNHSPTMYADYRYAQI--DGASVK--DMINDDWNRDTFLPTVGKRGAAIIDARGVSSAASAANAAIDHIHDWVLGTA-GKWTTMGI--PSDGSYGIPEGVIFGFPVTTE-NGEYKIVQGLSIDAFSQERINVTLNELLEEQN-GVQHL----\r\n",
      "   68 2ldxA                                    X     1   331    66   306    67   306   227   26.   0.21E-03 -----------------------------------------------------------------GSLFLSTPKIVFKDYNVSIITAGARMVSGQTRLDLLQRNVAIMKAIVPGVIQ-NSPDCKIIVVTNPVDILTYVVWKISGFPVGRVIGSGCNLDSARFRYLIGEKLGVNPTSCHGWVL-GEHGDSSVSGVNVAGVTLKSLNPAIGTDKNKQH-WKNVHKQVVEGGYEVLDMKGYTSWASVTDLARSILKN-LKRVHPVTTLVKGF-------HGIKEEVFLSIPCVLGESG---ITDFVKVN-----------------------------\r\n",
      "   69 9ldtA                                    X     1   331    85   301    93   304   207   26.   0.93E-05 ------------------------------------------------------------------------------------ITAGARQQEGESRLNLVQRNVNIFKFIIPNIVKYS-PNCKLLVVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHPLSCHGWIL-GEHGDSSVSGVNVAGVSLKNLHPELGTDADKEH-WKAVHKEVVDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKENVFLSVPCILGQNGISDVVK----------------------------------\r\n",
      "   70 1ldnA                                    X     1   316    78   298    73   301   214   26.   0.16E-02 -----------------------------------------------------------------------------RDADLVVICAGANQKPGETRLDLVDKNIAIFRSIVESVMASGFQGL-FLVATNPVDI-LTYATWKFSGLPHERIGSGTILDTARFRFLLGEYFSVAPQNVHAYII-GEHGDTELPVWSQA-YIGVMPIRKLVESKGEEAQKDRIFVNVRDAAYQIIEKKGATGLARVTRAAILTVSAYLDGLYGERDVYIGVPAVI-NRNGIRE--VIEIELNDDEKNRFH-------------------------------------\r\n",
      "   71 5ldhA                                    X     1   333    85   300    94   304   207   26.   0.26E-04 ------------------------------------------------------------------------------------VTAGVRQQEGESRLNLVQRNVNVFKFIIPQIVKYS-PNCIIIVVSNPVDILTYVAWKLSGLPKHRVIGSGCNLDSARFRYLMGEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVVLQQLNPEMTDNDSEWKEVHKMVVESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI-----------------------------------\r\n",
      "   72 6cepA                                    X     1   331    85   300    94   304   207   25.   0.34E-04 ------------------------------------------------------------------------------------VTAGVRQQEGESRLNLVQRNVNVFKFIIPQIVKYS-PDCIIIVVSNPVDILTYVTWKLSGLPKHRVIGSGCNLDSARFRYLMAEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVSLQELNPEMTDNDSEWKEVHKMVVESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI-----------------------------------\r\n",
      "   73 6cepD                                    X     1   317    96   300    90   289   196   26.   0.43E-03 -----------------------------------------------------------------------------------------------VTAGVVQRNVNVFKFIIPQIVKYS-PDCIIIVVSNPVDILTYVTWKLSGLPKHRVIGSGCNLDSARFRYLMAEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVSLQELNPEMTDNDSEWKEVHKMVVESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI-----------------------------------\r\n",
      "   74 1civA                                    X     1   374     6   334    33   358   325   35.    0.0     -----NVAVSGAAGMISNHLLFKLASGEVFGDQPISLKLLGSERSFAALEGVAMELEDSLYPLLRQVSIGIDPYEIFQDAEWALLIGAKPRGPGMERADLLDINGQIFAEQGKALNAVASPNVKVMVVGNPCNTNALICLKNAPNIPPKNFHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDFLNAKIHGIPVTEV---IRDRKWLEDEFTNMVQTRGGVLIKKWGRSSAASTAVSIVDAIRSLVTPTPEGDWFSTGV-YTNGNPYGIAEDIVFSMPCRSKGDGDYEFVKDVIFDDYLSKKIKKSEDELLAEKKCVAHLTGEG-\r\n",
      "   75 5nqbA                                    X     1   331    85   301    93   304   207   26.   0.57E-04 ------------------------------------------------------------------------------------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PHCKLLVVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHALSCHGWIL-GEHGDSSVSGMNVAGVSLKTLHPELGTDADKEQ-WKQVHKQVVDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMLKGL-------YGIKEDVFLSVPCVLGQNGISDVVK----------------------------------\r\n",
      "   76 6um4A                                    X     1   326     7   331     8   326   317   38.    0.0     ------ITVSGACGQIANSLLFRIANGEMLGENQVKLRLLERPEAMKALEGVKMELDDGAFPLLEEIYLTDSENDAFRGADYAILLGGKPRGPGMERADVMKDNAAIFKAQGEALNKQANGDVLVLIVANPANTNAMITSANAPDIPPENITAMTRLDHDRGLAQVAAKVGCNITDISRFAIWGNHSATQYPDLSFTTI----KGQWGLNVIDEQWITNEFIPNVQQRGAAIIKARGKSSAASAADAAIKHMHDWVLGNS--EWVSMAI--PSRGQYGIPRGIWCSMPVQCFGAGKYGVIEGLPINSFSADRINASVKELIEEKKIVENLL----\r\n",
      "   77 4m1qA                                    X     1   312    78   242    69   233   162   24.   0.32E-03 -----------------------------------------------------------------------------KSADIVVITAGAAQKPGETRLDLVEKNANIFKGIVDQVMGSGFNGI-FIIATNPVDVLAYATWKFSGLPKERVIGSGTILDTARFRFLLSEYFDIDVRNIHGYIM-GEHGDTELPVWSQTRIGSE-PISRYMDKYKPDKDLDEIFVNVRDAAYHIIERKGATHYA---------------------------------------------------------------------------------------------\r\n",
      "   78 1pzgA                                    X     1   327    74   191    71   190   114   30.   0.15E-05 -------------------------------------------------------------------------EAALTGADCVIVTAGLTKVPGKSRNDLLPFNSKIIREIGQNIKKYC-PKTFIIVVTNP--LDCMVKVMEASGVPTNMICGMAMLDSGRFRRYVADALSVSPRDVQATVI-GTHGDCMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   79 2x0jA                                    X     1   294    78   320    68   285   218   28.   0.38E-04 -----------------------------------------------------------------------------KGSEIIVVTAGLARKPGMTRLDLAHKNAGIIKDIAKKIVENA-PESKILVVTNPMDVMTYIMWKESGKPRNEVFGMGNQLDSQRLKERL---YNAGARNIRRAWIIGEHGDSMFVAKSLADFDGEVDWEAV----ENDVRF---------VAAEVIKRKGAT-IFGPAVAIYRMVKAVVEDT--GEIIPTSM-ILQGE-YGIE-NVAVGVPAKLGKNGA--EVADIKLSDEEIEKLRNSAKIL---------------\r\n",
      "   80 5ujkA                                    X     1   315    13   191    10   180   169   32.   0.40E-05 ------------AGQIGGTLAHLAGLKEL-GD----VVLFDIVDGVPQGKALDI-AESAPVDGFDAKYSGASDYSAIAGADVVIVTAGVPRKPGMSRDDLIGINLKVMEAVGAGIKEHA-PDAFVICITNP--LDAMVWALQKFSGLPTNVGMAGVLDSARFRHFLAEEFGVSVEDV-TAFVLGGHGDDMV------------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   81 4ajjA                                    X     1   330    85   301    92   303   207   25.   0.34E-04 ------------------------------------------------------------------------------------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PQCKLLIVSNPVDILTYVAWKISGFPKNRVIGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVSGVNVAGVSLKSLNPQLGTDADKEQWKDV-HKQVVDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKEDVFLSVPCILGQNGISDVVK----------------------------------\r\n",
      "   82 3nepX                                    X     1   305    13   274     8   263   244   25.   0.91E-02 ------------AGNVGATVAECVARQDVAKE----VVMVDIKDGMPQGKALDMR-ESSPIHGFDTRVTGTNDYGPTEDSDVCIITAGL-------RDDLLAKNTEIVGGVTEQFVE-GSPDSTIIVVANPLDVMTYVA--YEASGFPTNMGMAGVLDTGRFRSFIAEELDVSVRDVQALLM-GGHGDTMVPLPRYTTVGGIPVPQLIDDARIEEIVERT--KGAGGEIVDLMGTSAWAAAAEMTEAILKDNKAYCDGEYGLDDLFIGVPVKLG-------------------------------------------------------------\r\n",
      "   83 6pblA                                    X     1   327     7   325     5   318   313   39.    0.0     ------VAVTGAAGQIGYALVFRIASGQMFGNTEVELNLLELEPALPSLEGVAMELDDCAFPLLKRIVCTADLNKAMDGVNWALLVGSVPRKQGMERSDLLQINGGIFTKQGQAINDYASDDVRVFVVGNPCNTNCLIAMNHAKDVPSDRFYAMTTLDELRARTQLAKKAGVDITAVTQMTIWGNHSATQYPDFYNA---KINGTSAAQVINDETWLKETFVSTVQQRGAAVIKARGSSSAASAANAIITGVNHLVTDTPAGESFSM-CRRSKGE-YGVDEGLIFSFPCR-REHGELKVVENLEFNDFGRERFNTTLNELRSERD----------\r\n",
      "   84 1y7tA                                    X     1   327     1   325     1   319   318   45.    0.0     MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGDQPVILQLLEIPQAMKALEGVVMELEDCAFPLLAGLEATDDPKVAFKDADYALLVGAAPRKAGMERRDLLQVNGKIFTEQGRALAEVAKKDVKVLVVGNPANTNALIAYKNAPGLNPRNFTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSTMFPDL----FHAEVDGRPALELVDMEWYEKVFIPTVAQRGAAIIQARGASSAASAANAAIEHIRDWALGTPEGDWVSMA--VPSQGEYGIPEGIVYSFPVTA-KDGAYRVVEGLEINEFARKRMEITAQELLDEME----------\r\n"
     ]
    }
   ],
   "source": [
    "#Six PDB sequences show very significant similarities to the \n",
    "#query sequence with e-values equal to 0\n",
    "!cat build_profile.prf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      ">P1;TvLDH\r\n",
      "sequence:TvLDH:    0: :    0: :::-1.00:-1.00\r\n",
      "MSEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKA\r\n",
      "AFKDIDCAFLVASMPLKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPEN\r\n",
      "FSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKI\r\n",
      "GHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVV\r\n",
      "EGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQGG*\r\n",
      "\r\n",
      ">P1;3d5tA\r\n",
      "structure:3d5tA:    3: :  316: :::-1.00:-1.00\r\n",
      "---AKRVAVTGAAGQIAYSLLFRIANGDLLGDQPVILQLLDLPQAQAAVKGVVMELDDCAFPLLAGVVITDDPKV\r\n",
      "AFKDADVALLVGARP--RSMERKDLLSANAEIFTVQGAALNEVASRDVKVLVVGNPANTNAYIAMKSAPDLPKKN\r\n",
      "FTAMLRLDHNRALSQLAAKSGKPVASIEKLAVWGNHSPTMYPDFRFA--TAEGESLKLIN--DDVWNRDTFIPTV\r\n",
      "GKRGAAIIEARGLSSAASAANAAIDHVRDWVLGT-NGKWVTMGI--PSDGSYGIPEDIIYGVPV-ICENGEYKRV\r\n",
      "EGLEIDAFSREKMDGTLAELLEERD----------*\r\n",
      "\r\n",
      ">P1;1sovA\r\n",
      "structure:1sovA:   76: :  248: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----DVVIITAGLTKVPS--RNDLLPFNAKIIREVAQGVKKYC-PLAFVIVVTNP--LDCMVKCFHEASGLPKN\r",
      "\r\n",
      "MGMANVLDSARFRRFIADQLEISPRDIQATVI-GTHGDHMLPLARYVTFIKKGKEAKLAEIVERT-------KKA\r\n",
      "G---GEIVRLLGQGSAYAPALSAITMAQAFL--------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6k12A\r\n",
      "structure:6k12A:   81: :  270: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "------------------GESRLNLVQRNVNIFKFIIPNVVKYS-PDCILLIVSNPVDILTYVAWKLSGFPLNRV\r\n",
      "IGSGCNLDSARFRYLVSEMIGIHPSNFHGCIL-GEHGDSSVPILSGLNIAGM-SIKNCKDV----------HKKV\r\n",
      "TESAYEIIKLKGYTS--------------WAIGLSVGDLLRKVHPVSTKGQFGIDNEVFLSVPCVLGRNG---IS\r\n",
      "EVFK-------------------------------*\r\n",
      "\r\n",
      ">P1;6k13A\r\n",
      "structure:6k13A:  103: :  305: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "------------------GESRLNLVQRNVNIFKFIIPNVVKYS-PDCILLIVSNPVDILTYVAWKLSGFPLNRV\r\n",
      "IGSGCNLDSARFRYLVSEMIGIHPSNFHGCIL-GEHGDSSVPILSGLNIA--GMSIKNLHT-DIDTVFKDVHKKV\r\n",
      "TESAYEIIKLKGYTS--------------WAIGLSVGDLLRKVHPVSTKGQFGIDNEVFLSVPCVLGRNG---IS\r\n",
      "EVFK-------------------------------*\r\n",
      "\r\n",
      ">P1;6or9A\r\n",
      "structure:6or9A:   37: :  292: :::-1.00:-1.00\r\n",
      "----------------------------------------------DKLKGELMDLQGSAFMRNVKIQASTDYSI\r\n",
      "SAGSKICV-VTAGVRQREGESRLDLVQRNTDVLKIIIPQLVKHS-PDTILIIASNPVDILTYVSWKLSGLPKHRV\r\n",
      "IGSGTNLDSARFRYLLSEKLGIATTSCHGYII-GEHGDSSVSGVNIAGVRLSDLNQKINWKETHTMVVKS-----\r\n",
      "---AYEVIKLKGYTSWAIGLSLS-QLARAILSNANSVHAVSTYLK----GEHDINDEVFLSLPCVLGRSG-----\r\n",
      "----VCDVIRQPLTQTERSQLHQ------------*\r\n",
      "\r\n",
      ">P1;1guzA\r\n",
      "structure:1guzA:    8: :  280: :::-1.00:-1.00\r\n",
      "------------AGNVGATTAFRLAEKQLARE----LVLLDVVEGIPQGKALDM--------YESGPVGLFDTKA\r\n",
      "DTANSDIVIITAGLPRKPGMTREDLLMKNAGIVKEVTDNIMKHSKNPI-IIVVSNPLDIMTHVAWVRSGLPKERV\r\n",
      "IGMAGVLDAARFRSFIAMELGVSMQDINACVL-GGHGDAMVPVVKYTTVA----GIPISDLLPAETI-DKLVERT\r\n",
      "RNGGAEIVEHLKQGSAYAPASSVVEMVESIVLDRKRVLPCAVGL---EGQ-YGIDKTFV-GVPVKLGRNGVEQIY\r\n",
      "E----------------------------------*\r\n",
      "\r\n",
      ">P1;3gviA\r\n",
      "structure:3gviA:   10: :  206: :::-1.00:-1.00\r\n",
      "------------SGMIGGTLAHLAGLKEL-GD----VVLFDIAEGTPQGKGLDIAESSPVDGFDAKFTGAND-YA\r\n",
      "AIEGADVVIVTAGVPRKD-----DLLGINLKVMEQVGAGIKKYA-PEAFVICITNP---------LDAMVWALQK\r\n",
      "FSGLGVLDSARFRYFLSEEFNVSVEDV-TVFVLGGHGDSMVPDLVKMGWTSQDKLDKII----------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1gv1B\r\n",
      "structure:1gv1B:   80: :  271: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------DLLMKNAGIVKEVTDNIMKHSKNPI-IIVVSNPLDIMTHVAWVRSGLPKERV\r\n",
      "IGMAGVLDAARFRSFIAMELGVSMQDINACVL-GGHGDAMVPVVKYTTVA----GIPISDLLPAETI-DKLVERT\r\n",
      "RNGGAEIVEHLKQGSAYAPASSVVEMVESIVLDRKRVLPCAVGL---EGQ-YGIDKTFV-GVPVKLGRNGVEQIY\r\n",
      "E----------------------------------*\r\n",
      "\r\n",
      ">P1;1gv0A\r\n",
      "structure:1gv0A:    8: :  289: :::-1.00:-1.00\r\n",
      "------------AGNVGATTAFRLAEKQLARE----LVLLDVVEGIPQGKALDM--------YESGPVGLFDTKA\r\n",
      "DTANSDIVVITAGLPRKPGMT----LSMNAGIVREVTGRIMEHSKNPI-IVVVSNPLDIMTHVAWQKSGLPKERV\r\n",
      "IGMAGVLDSARFRSFIAMELGVSMQDV-TACVLGGHGDAMVPVVKYTTVA----GIPVADLISAERIAE-LVERT\r\n",
      "RTGGAEIVNHLKQGSAYSPATSVVEMVESIVLDRK--RVLTCAVSL--DGQYGI-DGTFVGVPVKLGKNGVEHIY\r\n",
      "E---------IKLDQSDLDLLQK------------*\r\n",
      "\r\n",
      ">P1;4bguA\r\n",
      "structure:4bguA:    3: :  289: :::-1.00:-1.00\r\n",
      "------VSVIGAAGTVGAAAGYNLALRDVCDE----LVFVDIPKMEDKTVGQAADTNH-GIAYDSNTVVTQGGYE\r\n",
      "DTAGSDVVVITAGIPRQPGQTRIDLAGDNAPIMDDIGSSLAEYNDDFVSITTSNPVDLLNRHLYETGDRD-RHKV\r\n",
      "IGFGGRLDSARFRYVLSQRFDVPVKNV-DATILGEHGDAQV-----PVFSK-------VRVDGNDPAFSADEKEL\r\n",
      "QESAMDVIERKGATQWGPAT--GVAHMEAVLHDT--GEVLP-GSLVLDGE-FGYE-DTAFGVPVKLGSNGIEEVV\r\n",
      "E-WDLDDYEADLMD---------------------*\r\n",
      "\r\n",
      ">P1;4bgvA\r\n",
      "structure:4bgvA:   10: :  185: :::-1.00:-1.00\r\n",
      "------------AGAVGATVAQTLAIRQT-GD----IYIFDIVDGLAEGKALDI-LEGA--PHIKGFCTADESKA\r\n",
      "EMKGSDVIVVTAGLARKPGMSRDDLLLKNIGIMKSVGEAIKKYS-PESKIVVVTNPADI-MAYAIYKASGISPER\r\n",
      "IIGLGSLDSTRFRTFLAQELNVSFEDVNAFVI-GGHGDDMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1a5zA\r\n",
      "structure:1a5zA:   63: :  229: :::-1.00:-1.00\r\n",
      "--------------------------------------------------------------------------A\r\n",
      "DLKGSDVVIVAAGVPQKPGETRLQLLGRNARVMKEIARNVSKYAPDSI-VIVVTNPVDV-LTYFFLKESGMDPRK\r\n",
      "FGSGTVLDTARLRTLIAQHCGFSPRSVH-VYVIGEHGDSEV-PVWSGAMIGGIPLQNMCQVCQDSKILENFAEKT\r\n",
      "KRAAYEIIERKGATHYA----------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4e0bA\r\n",
      "structure:4e0bA:    2: :  288: :::-1.00:-1.00\r\n",
      "---AMKVAVIGAAGGIGQALALLLKNRLPAGSD---LALYDIAPVTPGVAADLSHIP--THVSIKGY-AGEDPTP\r\n",
      "ALEGADVVLISA------GVARADLFNVNAGIVKSLAERIAV-VCPNACIGIITNPVNTTVPIALKKAGVYDKRK\r\n",
      "LFGVTTLDVIRSETFVAELKGQDPGEVRVPVIGGHSGVTILPLLSQV----EG-----VEFSDEEIAALT--KRI\r\n",
      "QNAGTEVVEAKGGSATLSMGQAACRFGLALVKALQGEEVIEYA--YVEGNG---EHASFFAQPVKLGKEGVEEIL\r\n",
      "PYGELSDFEKAALD---------------------*\r\n",
      "\r\n",
      ">P1;4ln1A\r\n",
      "structure:4ln1A:   79: :  278: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--KDADLVVITAGLP----QTRLDLVEKNAKIFKQIVRSIMDSGFDGI-FLIATNPVDILTYVTWKESGLPKERV\r\n",
      "IGSGTTLDSARFRYMLGEYFDIGPHNIHAYII-GEHGDT---ELPVWSHVSVG-IQKLQTLLEKDNTYDKIFINV\r\n",
      "RDAAYHIIERKGATSLLRVTKASVLTVSAYLEGQYGQKDVYIGVP------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3wswA\r\n",
      "structure:3wswA:   70: :  273: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--QDANLVILTAGVNQKPGQTRLDLVKTNATITRQIVKEVMASGFDGIFV-VASNPVDILTYLTWQESGLPASRV\r\n",
      "VGTGTTLDTTRFRKEIALKLAVDPRSVHGYIL-GEHGDSEVAAWSHTTV--GGK--PIMEYVEKDHRLEVLADKV\r\n",
      "KNAAYEIIDRKKATKAILNNEQAVLPVSAYLNGEYGEEDIFTGVP------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6itlA\r\n",
      "structure:6itlA:    3: :  201: :::-1.00:-1.00\r\n",
      "------VAVLGAAGGIGQALALLL---KLQLPAGSSLSLYDVAPV---TPGVAKDLSHIPTDVVVEGFAGTDPSE\r\n",
      "ALKGADIVLISAGVARKPGMTRADLFGVNAGIIRSLTEKVAEQC-PKACVGIITNPVNAMVAIALKKAGVYDKRK\r\n",
      "LFGITTLDILRAETFIAELKGLDPTRVTIPVIGGHSGVTILPLLSQVQNVEWSSEEEII----------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6itkA\r\n",
      "structure:6itkA:    5: :  310: :::-1.00:-1.00\r\n",
      "------VTVTGAAGQISYSLLWRIANGEVFGDTPVELKLLEIPQALGGAEGVAMELLDSAFPLLRNITITADANE\r\n",
      "AFDGANAAFLVGAKP-----ERADLLANNGKIFGPQGKAINDNAADDIRVLVVGNPANTNALIASAAAPDVPASR\r\n",
      "FNAMMRLDHNRAISQLATKLGRGSAEFNNIVVWGNHSATQFPDITYATVGGE----KVTDLVDHDWYVEEFIPRV\r\n",
      "ANRGAEIIEVRGKSSAASAASSAIDHMRDWVQGT---EAWS-SAAIPSTGAYGIPEGIFVGLP-TVSRNGEWEIV\r\n",
      "EGLEISDFQRARIDANAQELQAERE----------*\r\n",
      "\r\n",
      ">P1;4cl3A\r\n",
      "structure:4cl3A:   10: :  180: :::-1.00:-1.00\r\n",
      "------------AGFVGSTTAHWLAAKEL-GD----IVLLDIVEGVPQGKAL--DLYEAS--PIEGFVTGTNNYA\r\n",
      "DTANSDVIVVTSGAPRKPGMSREDLIKVNADI---TRACISQAASPNAVIIMVNNPLDAMTYLAAEVSGFPKERV\r\n",
      "IGQAGVLDAARYRTFIAMEAGVSVEDVQAMLM-GGHGDEMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3fi9A\r\n",
      "structure:3fi9A:   12: :  248: :::-1.00:-1.00\r\n",
      "--------IVGAAGMIGSNMAQTAAMMRLTPNLCLY------DPFAVGLEGVAEEIRHCGFEGL-NLTFTSDIKE\r\n",
      "AL--TDAKYIVSS-----GGTREDLLKGNAEIAAQLGKDIKSYCPDCKHVIIIFNPADITGLVTLIYS-GLKPSQ\r\n",
      "VTTLAGLDSTRLQSELAKHFGIKQSLVTNTRTYGGHGEQM------AVFASTAKVNPLTDLIGTDKLTNEQWAEL\r\n",
      "KQRGANIIKLRGRSSFQSPSYVSIEMIRAAMGGEA----------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;5mdhA\r\n",
      "structure:5mdhA:    1: :  331: :::-1.00:-1.00\r\n",
      "-SEPIRVLVTGAAGQIAYSLLYSIGNGSVFGDQPIILVLLDITPMMGVLDGVLMELQDCALPLLKDVIATDKEEI\r\n",
      "AFKDLDVAILVGSMPRRDGMERKDLLKANVKIFKCQGAALDKYAKKSVKVIVVGNPANTNCLTASKSAPSIPKEN\r\n",
      "FSCLTRLDHNRAKAQIALKLGVTSDDVKNVIIWGNHSSTQYPDVNHAKVKLQAKEVGVYEVKDDSWLKGEFITTV\r\n",
      "QQRGAAVIKARKLSSAMSAAKAICDHVRDIWFGTPEGEFVSMGI-ISDGNSYGVPDDLLYSFPVTIDKTWKI--V\r\n",
      "EGLPINDFSREKMDLTAKELAEEKETAFEFLS---*\r\n",
      "\r\n",
      ">P1;7mdhA\r\n",
      "structure:7mdhA:   14: :  339: :::-1.00:-1.00\r\n",
      "-----NIAVSGAAGMISNHLLFKLASGEVFGDQPIALKLLGSERSFQALEGVAMELEDSLYPLLREVSIGIDPYE\r\n",
      "VFEDVDWALLIGAKPRGPGMERAALLDINGQIFADQGKALNAVASKNVKVLVVGNPCNTNALICLKNAPDIPAKN\r\n",
      "FHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDFLNAKI--DGRPVKEV-IKRTKWLEEEFTITV\r\n",
      "QKRGGALIQKWGRSSAASTAVSIADAIKSLVTPTPEGDWFSTGV-YTTGNPYGIAEDIVFSMPCRSKGDGDYELA\r\n",
      "TDVSNDDFLWERIKKSEAELLAEKKCVAHLTGEG-*\r\n",
      "\r\n",
      ">P1;5kkcB\r\n",
      "structure:5kkcB:   98: :  293: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-------------------------VQRNVNIFKFIIPNVVKYS-PHCKLLVVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHALSCHGWIL-GEHGDSSVSGMNVAGVSLKTLHPELGTDADKEQ-WKQVHKQV\r\n",
      "VDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKEDVFLSVPCVLGQNGISDVV\r\n",
      "K----------------------------------*\r\n",
      "\r\n",
      ">P1;5zi2A\r\n",
      "structure:5zi2A:   76: :  197: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------AGVPRKPGLTRDDLFKMNAGIVKSLVTAVGKFA-PNARILVISNPVNSLVPIAVETLKKMKPGN\r\n",
      "VMGVTNLDLVRAYLMLKNKIGQDKTTMHRTVIGGHSGETIIPIIT------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1bdmA\r\n",
      "structure:1bdmA:    1: :  310: :::-1.00:-1.00\r\n",
      "MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGDQPVILQLLEIPQAMKALEGVVMELEDCAFPLLAGLEATDDPDV\r\n",
      "AFKDADYALLVGAAP---------RLQVNGKIFTEQGRALAEVAKKDVKVLVVGNPANTNALIAYKNAPGLNPRN\r\n",
      "FTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSIMFPDL----FHAEVDGRPALELVDMEWYEKVFIPTV\r\n",
      "AQRGAAIIQARGASSAASAANAAIEHIRDWALGTPEGDWVSMA--VPSQGEYGIPEGIVYSFPVTA-KDGAYRVV\r\n",
      "EGLEINEFARKRMEITAQELLDEME----------*\r\n",
      "\r\n",
      ">P1;4h7pA\r\n",
      "structure:4h7pA:    3: :  308: :::-1.00:-1.00\r\n",
      "---AVKVAVTGAAGQIGYALVPLIARGALLGTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKVVVTADPRV\r\n",
      "AFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLKSAQKLNPRH\r\n",
      "VTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVIGTTPAREAIKD----------FVQVV\r\n",
      "RGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDE-NPYGVPSGLIFSFPCT------CHAG\r\n",
      "EWTVVSGKLKQRLASTIAELQEER-----------*\r\n",
      "\r\n",
      ">P1;4h7pB\r\n",
      "structure:4h7pB:    2: :  301: :::-1.00:-1.00\r\n",
      "---AVKVAVTGAAGQIGYALVPLIARGALLGTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKVVVTADPRV\r\n",
      "AFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLKSAQKLNPRH\r\n",
      "VTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVI---GTTPDFVQV-------------V\r\n",
      "RGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDE-NPYGVPSGLIFSFPCT------CHAG\r\n",
      "EWTVVSGKLKQRLASTIAELQEER-----------*\r\n",
      "\r\n",
      ">P1;3hhpA\r\n",
      "structure:3hhpA:    3: :  303: :::-1.00:-1.00\r\n",
      "------VAVLGAAGGIGQALALLLKTQLPSGSE---LSLYDIAPVTPGVAVDLSHIPTAV--KIKGF-SGEDATP\r\n",
      "ALEGADVVLISAGVARKPGMDRSDLFNVNAGIVKNLVQQVAKTC-PKACIGIITNPVNTTVAIAVLKKAGVYDKN\r\n",
      "LFGVTTLDIIRSNTFVAELKGKQPGEVEVPVIGGHSGVTILPLLSQVSFTE----QEVADLT----------KRI\r\n",
      "QNAGTEVVEAKGGGSATLSMGQAAARFGLSLVRALQGEQGVVECAYVEGDGQYAR---FFSQPLLLGKNGSIGTL\r\n",
      "SAFEQNA-LEGMLDTLKKDI---------------*\r\n",
      "\r\n",
      ">P1;4tvoA\r\n",
      "structure:4tvoA:    6: :  318: :::-1.00:-1.00\r\n",
      "------VAVTGAAGQIGYSLLFRLASGSLLGDRPIELRLLEIEPALQALEGVVMELDDCAFPLLSGVEIGSDPQK\r\n",
      "IFDGVSLALLVGARPRGAGMERSDLLEANGAIFTAQGKALNAVAADDVRVGVTGNPANTNALIAMTNAPDIPRER\r\n",
      "FSALTRLDHNRAISQLAAKTGAAVTDIKKMTIWGNHSATQYPDLFHAEVA--GKNAAEV-VNDQAWIEDEFIPTV\r\n",
      "AKRGAAIIDARGASSAASAASATIDAARDWLLGTPADDWVSMAV-VSDGS-YGVPEGLISSFPVTT-KGGNWTIV\r\n",
      "SGLEIDEFSRGRIDKSTAELADER-----------*\r\n",
      "\r\n",
      ">P1;4tvoB\r\n",
      "structure:4tvoB:    5: :  308: :::-1.00:-1.00\r\n",
      "------VAVTGAAGQIGYSLLFRLASGSLLGDRPIELRLLEIEPALQALEGVVMELDDCAFPLLSGVEIGSDPQK\r\n",
      "IFDGVSLALLVGARP---------LLEANGAIFTAQGKALNAVAADDVRVGVTGNPANTNALIAMTNAPDIPRER\r\n",
      "FSALTRLDHNRAISQLAAKTGAAVTDIKKMTIWGNHSATQYPDLFHAEVA--GKNAAEV-VNDQAWIEDEFIPTV\r\n",
      "AKRGAAIIDARGASSAASAASATIDAARDWLLGTPADDWVSMAV-VSDGS-YGVPEGLISSFPVTT-KGGNWTIV\r\n",
      "SGLEIDEFSRGRIDKSTAELADER-----------*\r\n",
      "\r\n",
      ">P1;2hjrA\r\n",
      "structure:2hjrA:   10: :  309: :::-1.00:-1.00\r\n",
      "------------AGQIGSTIALLLGQKDL-GD----VYMFDIIEGVPQGKAL--DLNHCALIGSPAKIFGENNYE\r\n",
      "YLQNSDVVIITAGVPRKPNMTRSDLLTVNAKIVGSVAENVGKYC-PNAFVICITNP--LDAMVYYFKEKSGIPAN\r\n",
      "CGMSGVLDSARFRCNLSRALGVKPSDVSAIVV-GGHGDEMIPLTSSVTFVEQGK-------ITHSQI-NEIIKKT\r\n",
      "AFGGGEIVELLKTGSAYAPAASAVAMAQAYLKDSKSVLVCSTYLT----GQYNVN-NLFVGVPVVIGKNG-----\r\n",
      "----IEDVVIVNLSDDEKSLFSKSVESIQNLVQ--*\r\n",
      "\r\n",
      ">P1;1t2dA\r\n",
      "structure:1t2dA:    4: :  250: :::-1.00:-1.00\r\n",
      "----AKIVLVG-SGMIGGVMATLIVQKNL-GDVVLFDIVKNMPKALDTSHTNVMAYSNC---KVSGSNTYDDLAG\r\n",
      "A----DVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHIKKNC-PNAFIIVVTNP--VDVMVQLLHQHSGVPKN\r\n",
      "IGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMVLLKRYITVGGIPLQEFINNKLISDAELEAIFDRT\r\n",
      "VNTALEIVNLHA-SPYVAPAAAIIEMAESYL--------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4plgA\r\n",
      "structure:4plgA:    1: :  253: :::-1.00:-1.00\r\n",
      "MTKRKKISLIG-SGMIGGTMAYLCAQKEL-GD----VVLFDVVKNMPQGKALDLS-HSSSIADTNVKVTGTNSYE\r\n",
      "DIKGSDVVIITAGLTKAPGKSRDDLLPFNAKIMREVGENIKKYC-PNAFVIVITNP--LDVMVKVLHEHSGLPKN\r\n",
      "CGMAGVLDSSRFRHFIAEKLNVSPRDVQAMVI-GAHGDKMVPLTRYVTFIKKGRTQEEIDEI---------VERT\r\n",
      "KNAGGEIVNLLGQGSAFAPAASAIEMAEAYL--------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4pltB\r\n",
      "structure:4pltB:   11: :  280: :::-1.00:-1.00\r\n",
      "------------AGNIGGTLAHLIAQKEL-GD----VVLFDIVEGMPQGKALDISHSS---PIMGSNVKITGTN-\r\n",
      "NYEDIDVVIITAGIPRKPGKSRDDLLSVNAKIMKDVAENIKKYC-PNAFVIVVTNP--LDVMVYVLHKYSGLPHN\r\n",
      "CGMAGVLDSSRFRYFLAEKLNVSPNDVQAMVI-GGHGDTMVGGIPLTEFIKQGITQEEID----EIVERT--RNA\r\n",
      "GGEIVNLLKTGSFAPAASAIEMAESYLKAYLEGQYGVKDLFVGVPV-----------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4plvA\r\n",
      "structure:4plvA:   10: :  265: :::-1.00:-1.00\r\n",
      "------------AGNIGGTLAHLIAQKEL-GD----VVLFDIVEGMPQGKALDISHSS---PIMGSNVKITGTN-\r\n",
      "NYEDIDVVIITAGI---------DLLSVNAKIMKDVAENIKKYC-PNAFVIVVTNP--LDVMVYVLHKYSGLPHN\r\n",
      "CGMAGVLDSSRFRYFLAEKLNVSPNDVQAMVI-GGHGDTMVGGIPLTEFIKQGITQEEID----EIVERT--RNA\r\n",
      "GGEIVNLLKTGSFAPAASAIEMAESYLKAYLEGQYGVKDLFVGVPV-----------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4plzA\r\n",
      "structure:4plzA:   80: :  236: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------------------RDDLLPLNNKIMIEIGGHIKKNC-PNAFIIVVTNP--VDVMVQLLHQHSGVPKN\r\n",
      "IGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMVLLKRYITVGGIPLQEFINNKLISDAELEAIFDRT\r\n",
      "VNTALEIVNLHA-SPYVAPAAAIIEMAESYL--------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3d0oA\r\n",
      "structure:3d0oA:   71: :  295: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---DADLVVICAGAAQKPGETRLDLVSKNLKIFKSVGEVMA--SKFDGIFLVATNPVDILAYATWKFSGLPKERV\r\n",
      "IGSGTILDSARFRLLLSEAFDVAPRSV-DAQIIGEHGDTELPVWSHANIAGQPLKTLLEQRPEGKAQIEQIFVQT\r\n",
      "RDAAYDIIQAKGITEAIFRNEDAVLTVSALLEGEYEEEDVYIGVPAVI-NRNGIRN--VVEIPLNDEEQSK----\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1y6jA\r\n",
      "structure:1y6jA:   58: :  167: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-YSDVDCDVIVVT----AGATRLDLAKKNVMIAKEVTQNIMKYYNHGV-ILVVSNPVDIITYMIQKWSGLPVGKV\r\n",
      "IGSGTVLDSIRFRYLLSEKLGVDVKNVHGYII-GEHGDSQL----------------------------------\r",
      "\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1ez4B\r\n",
      "structure:1ez4B:   55: :  230: :::-1.00:-1.00\r\n",
      "--------------------------------------------------------------------AFTAPKK\r\n",
      "IYKDADLVVITAGAPQKPGESRLDLVNKNLNI-------LSSIVKPVVDFLVAANPVDILTYATWKFSGFPKERV\r\n",
      "IGSGTSLDSSRLRVALGKQFNVDPRSV-DAYIMGEHGDSEFAAYSTATI----GTRPVRDVAKEQGVSDDDLAKV\r\n",
      "RNKAYDIINLKGAT-------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;5hs4A\r\n",
      "structure:5hs4A:    4: :  172: :::-1.00:-1.00\r\n",
      "-------IVLVGSGMIGGVMATLIVQKNL-GDVVMFDVVKNMPKALDTSHSNVMAYSNC---KVTGSNSYDDLKG\r\n",
      "A----DVVIVTAGF-------RDDLLPLNNKIMIEIGGHIKNLC-PNAFIIVVTNPVDVMVQLLFEHSGVPKNKI\r\n",
      "IGLGGVLDTSRLKYYISQKLNVCPRDVNALIV-GAHGNKMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;5nueA\r\n",
      "structure:5nueA:    3: :  331: :::-1.00:-1.00\r\n",
      "--EPVRVLVTGAAGQIGYALVPMIARGIMLGDQPVILHMLDIPPAAEALNGVKMELIDAAFPLLKGVVATTDAVE\r\n",
      "GCTGVNVAVMVGGFPRKEGMERKDVMSKNVSIYKSQAAALEKHAAPNCKVLVVANPANTNALILKEFAPSIPEKN\r\n",
      "ISCLTRLDHNRALGQISERLSVPVSDVKNVIIWGNHSSSQYPDVNHAKVQTSSGEKPVRELVKDDAWLDEFISTV\r\n",
      "QQRGAAIIKARKLSSALSAASSACDHIRDWVLGTPEGTFVSMGV-YSDGS-YSVPSGLIYSFPVTC-RNGDWSIV\r\n",
      "QGLPIDEVSRKKMDLTAEELKEEKDLAYSCLS---*\r\n",
      "\r\n",
      ">P1;1mldA\r\n",
      "structure:1mldA:    1: :  189: :::-1.00:-1.00\r\n",
      "----AKVAVLGASGGIGQPLSLLLKNSPLVSRLTLYAHTPGVAADLSHITRATVK----------GYLGPEQLPD\r\n",
      "CLKGCDVVVIPAGVPRKPGMTRDDLFNTNATIVATLTAACAQHC-PDAMICIISNPVNSTIPIAEVFKKHYNPNK\r\n",
      "IFGVTTLDIVRANAFVAELKGLDPARVSVPVIGGHAGKTIIPLISQCT---------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6j9sB\r\n",
      "structure:6j9sB:   59: :  233: :::-1.00:-1.00\r\n",
      "----------------------------------------------------------------------TSPKK\r\n",
      "IYKDADLVVITAGAPQKPGETRLDLVNKNLKILKSIVDPIVDSGFNGI-FLVAANPVDILTYATWKLSGFPKNRV\r\n",
      "VGSGTSLDTARFRQSIAKMVNVDARSVHAYIM-GEHGDANIGGVTIAEWVKEIKEDKLVKM----------FEDV\r\n",
      "RNKAYEIIKLKGAT-------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6iheA\r\n",
      "structure:6iheA:   70: :  206: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "----MDIVVISAGKPRKPGMSRRDLFVDNAKIMIDLAQKLPS-KNPGAIYLMVANPVDMMASVFMKYSKQF---T\r\n",
      "ISAGDQVETMRMRSFIAKKLKIPVTSV-DGFVGGEHGEDAV--VLWSTVKIKGKPVDEFNI-NKDEVSD-YVKKI\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6r8gA\r\n",
      "structure:6r8gA:    8: :  244: :::-1.00:-1.00\r\n",
      "------------SGQIGAIVGE-LCLLENLGD----LILYDVVPGIPQGKALDLK----HFSTILGVILGTNQIE\r\n",
      "DIKDADIIVITAGVQRKEGMTREDLIGVNGKIMKSVAESVKLHCSKAF-VICVSNP--LDIMVNVFHKSNLPHEK\r\n",
      "ICGMAILDTSRYCSLIADKLKVSAEDVNAVIL-GGHGDLMVPLQRYTSVNGVPLSEFVKKNMISQNEIQEIIQKT\r\n",
      "RNMGAEIIKLAKASAAFAPAAAITKMIKSYLY-------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;2v65A\r\n",
      "structure:2v65A:   97: :  298: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "------------------GESRLNLVQRNVNIFKFIIPNIVKYS-PNCILMVVSNPVDILTYVAWKLSGFPRHRV\r\n",
      "IGSGTNLDSARFRHLIGEKLHLHPSSCHAWIV-GEHGDSSVSGVNVAGVSLQGLNPQMGTEGDGEN-WKAIHKEV\r\n",
      "VDGAYEVIKLKGYTSWAIGMSVAVESIIKNMHKVHPVSTLVQGM-------HGVKDEVFLSVPCVLGNSGLTDVI\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1v6aA\r\n",
      "structure:1v6aA:  103: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "------------------GESRLNLVQRNVNIFKFIIPNIIKYS-PNCILLVVSNPVDILTYVAWKLSGLPRNRV\r\n",
      "IGSGTNLDSARFRHLMGEKLGIHPSNCHGWVI-GEHGDSSVSGVNVAGVFLQGLNPDMGTDKDKE-DWKSVHKMV\r\n",
      "VDSAYEVIKLKGYTSWAIGMSAACQSILKNLRKCHPVSTLVKGM-------HGVNEEVFLSVPCILGNSGLTDVV\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6balA\r\n",
      "structure:6balA:    1: :  274: :::-1.00:-1.00\r\n",
      "---AMKVAVLGAAGGIGQALALLL---KLQLPAGTDLSLYDIAPVTPGVAVDVSHIPTAV--NVKGF-SGEDPTP\r\n",
      "ALEGADVVLISAGVARKPGMDRSDLFNINAGIVRGLIEKVAVTC-PKACVGIITNPVNTTVAIALKKAGVYDKRK\r\n",
      "LFGVTTLDVLRSETFVAELKGLNVSRTSVPVIGGHSGVTILPLLSQVQYAKWNEDE-----------IEPLTKRI\r\n",
      "QNAGTEVLNAKGGSATLSMAQAAARFARSLVKGLSGETVVECTYVEGDG-----KYARFFSQPVRLGKEG-----\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4q3nA\r\n",
      "structure:4q3nA:   70: :  309: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--KDAGLVVITAGVNQKEGETRIHLLKRNAEIIKTVVKNIMSSGFNGI-LLVASNPVDVLAYVAWKESGLHHSRV\r\n",
      "IGSGTSLDTARLRYEISQYIHIDSRNVHAYIL-GEHGDSEFVCWSNANV----GAKPIADVIDSDEILEHIYLKV\r\n",
      "RDAAYEIISRKKATYYG--IGMALVRITAAIFN---NENRILPISVLNNGVYDCEDQVYIGLPAVLNKDGVHHIV\r\n",
      "KNEKENSQLNNSANILRKNL---------------*\r\n",
      "\r\n",
      ">P1;4uulA\r\n",
      "structure:4uulA:    1: :  332: :::-1.00:-1.00\r\n",
      "-SEAAHVLITGAAGQIGYILSHWIASGELYGDRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPKA\r\n",
      "AFKDIDCAFLVASMPRKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAKNLKPEN\r\n",
      "FSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFKKI\r\n",
      "GHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIHVV\r\n",
      "EGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQ--*\r\n",
      "\r\n",
      ">P1;4uupA\r\n",
      "structure:4uupA:    1: :  331: :::-1.00:-1.00\r\n",
      "--EPIHVLITGAAGQIGYALAFRIAKGDLFGDRKVVLHLLEIPPAMKALEGVCMELQDCAFPTLAGVVATDDPEE\r\n",
      "AFKDVDVAFLVGSFPRKPGMERADLLEKNAGIFKVQGKALSEYAKPTVKVLVVGNPANTNCLIAMANAPKLGPEN\r\n",
      "FSAMTRLDHNRAIGEIAAKLGVPVDKVHNVVVWGNHSNTQVPDVSHATVDKEGGTKKVSDALPKEYLEGEFVQKI\r\n",
      "AQRGGAVIEARGASSAASAANAALCHMRDWLFGTKPGDWVSMGIPVPEGNPYGIKPGVIYSFPCTVDKDGKVHIV\r\n",
      "EGLEINDWVREKMEATEKELIEERETAFKVLAQ--*\r\n",
      "\r\n",
      ">P1;4uuoA\r\n",
      "structure:4uuoA:    1: :  318: :::-1.00:-1.00\r\n",
      "--EPIHILITGAAGQIGYALTFRIAKGDLCGDRKVVLHLLEIPFGMKALEGCVMELQDCAFPNVAGIVWTDKVEE\r\n",
      "AFKGVDVAFLVGSFPRKDGMDRSDLLAKNGGIFTVQGKALNDYAKPTVKVLVVGNPANTNCLIAQASAPKLQNKN\r\n",
      "WCAMTRLDHNRMVGALAAKFGVTPEKIHKVCIWGNHSNTQVPDTTHATV----DTVKVADKLPKEYLEGEFAQMI\r\n",
      "ATRGGAVIKMRGASSAASAANAALTCVKDWLYGT---DFVSMAIPVPDNEPYGIKQGTIFSFPVTVSKDGEVHVV\r\n",
      "EGLELNDWVKGRLEATEKELIGEKETA--------*\r\n",
      "\r\n",
      ">P1;2a92A\r\n",
      "structure:2a92A:    6: :  186: :::-1.00:-1.00\r\n",
      "-------IVLVGSGMIGGVMATLIVQKNL-GDVVMFDVVKNMPKALDTSHSNVMAYSNC---KVTGSNSYDDLKG\r\n",
      "A----DVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHIKNLC-PNAFIIVVTNPVDVMVQLLFEHSGVPKNKI\r\n",
      "IGLGGVLDTSRLKYYISQKLNVCPRDVNALIV-GAHGNKMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6bb1E\r\n",
      "structure:6bb1E:   94: :  277: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------------------RLNLVQRNVNIFKFIIPNVVKYS-PNCKLLIVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVP--VWSGMNVAGVSLKTL------HPDKEVHKQV\r\n",
      "VESAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPVSTMIKGL-------YGIKDDVFLSVPCILGQNG-----\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1oc4A\r\n",
      "structure:1oc4A:    4: :  186: :::-1.00:-1.00\r\n",
      "----AKIVLVG-SGMIGGVMATLIVQKNL-GDVVMFDIVKNMPKALDTSHTNVMAYSNCK-------VSGSNTYD\r\n",
      "DLKDADVVIVTAGFTKAPGKSRDDLLPLNNKIMIEIGGHI-KNNCPNAFIIVVTNP--VDVMVQLLHQHSGVPKN\r\n",
      "VGLGGVLDTSRLKYYISQKLNVCPRDVNAHIV-GAHGNKMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1smkA\r\n",
      "structure:1smkA:    4: :  198: :::-1.00:-1.00\r\n",
      "------VAILGAAGGIGQPLAMLMKMNPLVS----VLHLYDVVNAPG-VTADISHMDTGAV--VRGFLGQQQLEA\r\n",
      "ALTGMDLIIVPAGVPRKPGMTRDDLFKINAGIVKTLCEGIAKCC-PRAIVNLISNPVNSTVPIAFKKAGTYDPKR\r\n",
      "LLGVTMLDVVRANTFVAEVLGLDPRDVDVPVVGGHAGVTILPLLSQSSFTQE-----------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6vdhA\r\n",
      "structure:6vdhA:   12: :  176: :::-1.00:-1.00\r\n",
      "------------SGMIGGTMAYLCALKEL-GD----VVLFDVVKNMPQGKALDLS-------HSTS-VADTNVKV\r\n",
      "SYEDIKGSDVVI---ITAGLSRDDLLPINAKIMKEVGENIKKYC-PNAFVICITNP--LDVMVKVLQEASGLPHN\r\n",
      "CGMAGVLDSSRFRYFIAEKLNVSPRDVQAMVI-GAHGDNMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4jcoA\r\n",
      "structure:4jcoA:    3: :  287: :::-1.00:-1.00\r\n",
      "------VSVVGAAGTVGAAAGYNIALRDI-ADEVVF---VDIPDKEDDTVGQAADTN-----HGIAYDSNTRVRG\r\n",
      "GYEDTDVVVITAGIPRQPGQTRIDLAGDNAPIMEDIQSSLDEHNDDYISLTTSNPVDLLNRHLYEAGDRS-REQV\r\n",
      "IGFGGRLDSARFRYVLSEEFDAPVQNVEGTIL-GEHGDAQVVDGTDPEFSGDEKEQLLGDLQE------------\r\n",
      "--SAMDVIERKGATE-WGPARGVAHMVEAILHDT--GEVLPASVKL-EGE-FG-HEDTAFGVPVRLGSNGVEEIV\r\n",
      "E------W---DLDDYEQDL---------------*\r\n",
      "\r\n",
      ">P1;1lldA\r\n",
      "structure:1lldA:    9: :  233: :::-1.00:-1.00\r\n",
      "------------AGAVGSTLA--FAAAQRGIAREIVLE--DI--AKERVEAEVLDMQGSSFYPTVSIDGSDDPEI\r\n",
      "C-RDADMVVITAGPRQKPGQSRLELVGATVNILKAIMPNLVKVA-PNAIYMLITNP----VDIATHVAQKLTPEN\r\n",
      "FGSGTNLDSARLRFLIAQQTGVNVKNVHAYIA-GEHGDSEVPLWESATIMSDWTPLPGHDPLDAD-KREEIHQEV\r\n",
      "KNAAYKIINGKGATNYA----------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1hyeA\r\n",
      "structure:1hyeA:    3: :  183: :::-1.00:-1.00\r\n",
      "------VTIIGASGRVGSATALLLAKEPFMKD----LVLIGREHSINKLEGLREDIYDA----LAGYVESDENLR\r\n",
      "IIDESDVVIITSGVPRKEGMSRMDLAKTNA---KIVGKYAKKIAECDTKIFVITNPVDVMTYKALVDSKFERNQV\r\n",
      "FGLGTHLDSLRFKVAIAKFFGVHIDEVRTRII-GEHGDSMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4wleA\r\n",
      "structure:4wleA:    2: :  190: :::-1.00:-1.00\r\n",
      "----AKVAVLGASGGIGQPLSLLLKNSPLVSRLTLY-DIAHTPGVAADLSHIETKAA------VKGYLGPEQLPD\r\n",
      "CLKGCDVVVIPAGVPRKPGMTRDDLFNTNATIVATLTAACAQHC-PEAMICVIANPVNSTIPIAEVFKKHYNPNK\r\n",
      "IFGVTTLDIVRANTFVAELKGLDPARVNVPVIGGHAGKTIIPLISQCT---------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3tl2A\r\n",
      "structure:3tl2A:   34: :  185: :::-1.00:-1.00\r\n",
      "--------------------------------------LVDIPQLENPTKGKALDMLEAS--PVQGFDANTSDYA\r\n",
      "DTADSDVVVITAGIARKPGMSRDDLVATNSKIMKSITRDIAKHS-PNAIIVVLTNPVDAMTYSVFKEAGFPKERV\r\n",
      "IGQSGVLDTARFRTFIAQELNLSVKDITGFVL-GGHGDDMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;3p7mA\r\n",
      "structure:3p7mA:   11: :  205: :::-1.00:-1.00\r\n",
      "------------AGNIGGTLAH-LALIKQLGD----VVLFDIAQGMPNGKALDL-LQTCPIEGVDFKVRGTNDYK\r\n",
      "DLENSDVVIVTAGVPRKPGMSRDDLLGINIKVMQTVGEGI-KHNCPNAFVICITNP--LDIMVNMLQKFSGVPDN\r\n",
      "VGMAGVLDSARFRTFLADELNVSVQQV-QAYVMGGHGDTMVAGVSLEQLVKEGKLKQ------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4nd2A\r\n",
      "structure:4nd2A:    1: :  288: :::-1.00:-1.00\r\n",
      "MIERRKIAVIG-SGQIGGNIAYIVGK-----DNLADVVLFDIAEGIPQGKALDITHSMVMFGSTSKVIGTND-YA\r\n",
      "DISGSDVVIITASIPGRPKDDRSELLFGNARILDSVAEGVKKYC-PNAFVICITNP----LDVMVSHFQKVSPHN\r\n",
      "CGMAGVLDSSRFRTFIAQHFGVNASDVSANVI-GGHGDGMVGGVPLSSFIKQGLTQEQIDIVCHTRI--------\r\n",
      "---AWKEVADNLKTGTAAPAAAAVKMAEAYL------KDKKAVVPCSASNHYGVK-GIYMGVPTIIGKNGVEDIL\r\n",
      "E----------------------------------*\r\n",
      "\r\n",
      ">P1;4nd5A\r\n",
      "structure:4nd5A:    1: :  276: :::-1.00:-1.00\r\n",
      "MIERRKIAVIG-SGQIGGNIAYIVGK-----DNLADVVLFDIAEGIPQGKALDITHSMVMFGSTSKVIGTND---\r\n",
      "-YADISGSDVV---------IITALLFGNARILDSVAEGVKKYC-PNAFVICITNP----LDVMVSHFQKVSPHN\r\n",
      "CGMAGVLDSSRFRTFIAQHFGVNASDVSANVI-GGHGDGMVGGVPLSSFIKQGLTQEQIDIVCHTRI--------\r\n",
      "---AWKEVADNLKTGTAAPAAAAVKMAEAYL------KDKKAVVPCSASNHYGVK-GIYMGVPTIIGKNGVEDIL\r\n",
      "E----------------------------------*\r\n",
      "\r\n",
      ">P1;5w8jA\r\n",
      "structure:5w8jA:   91: :  296: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PNCKLLIVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVAGVSLKTLHPDLGTDK-----DKEQ-WKEVHKQV\r\n",
      "VESAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPVSTMIKGL-------YGIKDDVFLSVPCILGQNG-----\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1b8pA\r\n",
      "structure:1b8pA:    6: :  325: :::-1.00:-1.00\r\n",
      "------VAVTGAAGQICYSLLFRIANGDMLGDQPVILQLLEIPKAQKALQGVMMEIDDCAFPLLAGMTAHADPMT\r\n",
      "AFKDADVALLVGARPRGPGMERKDLLEANAQIFTVQGKAIDAVASRNIKVLVVGNPANTNAYIAMKSAPSLPAKN\r\n",
      "FTAMLRLDHNRALSQIAAKTGKPVSSIEKLFVWGNHSPTMYADYRYAQI--DGASVK--DMINDDWNRDTFLPTV\r\n",
      "GKRGAAIIDARGVSSAASAANAAIDHIHDWVLGTA-GKWTTMGI--PSDGSYGIPEGVIFGFPVTTE-NGEYKIV\r\n",
      "QGLSIDAFSQERINVTLNELLEEQN-GVQHL----*\r\n",
      "\r\n",
      ">P1;2ldxA\r\n",
      "structure:2ldxA:   67: :  306: :::-1.00:-1.00\r\n",
      "-----------------------------------------------------------------GSLFLSTPKI\r\n",
      "VFKDYNVSIITAGARMVSGQTRLDLLQRNVAIMKAIVPGVIQ-NSPDCKIIVVTNPVDILTYVVWKISGFPVGRV\r\n",
      "IGSGCNLDSARFRYLIGEKLGVNPTSCHGWVL-GEHGDSSVSGVNVAGVTLKSLNPAIGTDKNKQH-WKNVHKQV\r\n",
      "VEGGYEVLDMKGYTSWASVTDLARSILKN-LKRVHPVTTLVKGF-------HGIKEEVFLSIPCVLGESG---IT\r\n",
      "DFVKVN-----------------------------*\r\n",
      "\r\n",
      ">P1;9ldtA\r\n",
      "structure:9ldtA:   93: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------ITAGARQQEGESRLNLVQRNVNIFKFIIPNIVKYS-PNCKLLVVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHPLSCHGWIL-GEHGDSSVSGVNVAGVSLKNLHPELGTDADKEH-WKAVHKEV\r\n",
      "VDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKENVFLSVPCILGQNGISDVV\r\n",
      "K----------------------------------*\r\n",
      "\r\n",
      ">P1;1ldnA\r\n",
      "structure:1ldnA:   73: :  301: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--RDADLVVICAGANQKPGETRLDLVDKNIAIFRSIVESVMASGFQGL-FLVATNPVDI-LTYATWKFSGLPHER\r\n",
      "IGSGTILDTARFRFLLGEYFSVAPQNVHAYII-GEHGDTELPVWSQA-YIGVMPIRKLVESKGEEAQKDRIFVNV\r\n",
      "RDAAYQIIEKKGATGLARVTRAAILTVSAYLDGLYGERDVYIGVPAVI-NRNGIRE--VIEIELNDDEKNRFH--\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;5ldhA\r\n",
      "structure:5ldhA:   94: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------VTAGVRQQEGESRLNLVQRNVNVFKFIIPQIVKYS-PNCIIIVVSNPVDILTYVAWKLSGLPKHRV\r\n",
      "IGSGCNLDSARFRYLMGEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVVLQQLNPEMTDNDSEWKEVHKMV\r\n",
      "VESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6cepA\r\n",
      "structure:6cepA:   94: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------VTAGVRQQEGESRLNLVQRNVNVFKFIIPQIVKYS-PDCIIIVVSNPVDILTYVTWKLSGLPKHRV\r\n",
      "IGSGCNLDSARFRYLMAEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVSLQELNPEMTDNDSEWKEVHKMV\r\n",
      "VESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6cepD\r\n",
      "structure:6cepD:   90: :  289: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--------------------VTAGVVQRNVNVFKFIIPQIVKYS-PDCIIIVVSNPVDILTYVTWKLSGLPKHRV\r\n",
      "IGSGCNLDSARFRYLMAEKLGVHPSSCHGWIL-GEHGDSSVAVWSGVNVAGVSLQELNPEMTDNDSEWKEVHKMV\r\n",
      "VESAYEVIKLKGYTNWAIGLSVAIESMLKNLSRIHPVSTMVQGM-------YGIENEVFLSLPCVLNARGLTSVI\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1civA\r\n",
      "structure:1civA:   33: :  358: :::-1.00:-1.00\r\n",
      "-----NVAVSGAAGMISNHLLFKLASGEVFGDQPISLKLLGSERSFAALEGVAMELEDSLYPLLRQVSIGIDPYE\r\n",
      "IFQDAEWALLIGAKPRGPGMERADLLDINGQIFAEQGKALNAVASPNVKVMVVGNPCNTNALICLKNAPNIPPKN\r\n",
      "FHALTRLDENRAKCQLALKAGVFYDKVSNVTIWGNHSTTQVPDFLNAKIHGIPVTEV---IRDRKWLEDEFTNMV\r\n",
      "QTRGGVLIKKWGRSSAASTAVSIVDAIRSLVTPTPEGDWFSTGV-YTNGNPYGIAEDIVFSMPCRSKGDGDYEFV\r\n",
      "KDVIFDDYLSKKIKKSEDELLAEKKCVAHLTGEG-*\r\n",
      "\r\n",
      ">P1;5nqbA\r\n",
      "structure:5nqbA:   93: :  304: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PHCKLLVVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHALSCHGWIL-GEHGDSSVSGMNVAGVSLKTLHPELGTDADKEQ-WKQVHKQV\r\n",
      "VDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMLKGL-------YGIKEDVFLSVPCVLGQNGISDVV\r\n",
      "K----------------------------------*\r\n",
      "\r\n",
      ">P1;6um4A\r\n",
      "structure:6um4A:    8: :  326: :::-1.00:-1.00\r\n",
      "------ITVSGACGQIANSLLFRIANGEMLGENQVKLRLLERPEAMKALEGVKMELDDGAFPLLEEIYLTDSEND\r\n",
      "AFRGADYAILLGGKPRGPGMERADVMKDNAAIFKAQGEALNKQANGDVLVLIVANPANTNAMITSANAPDIPPEN\r\n",
      "ITAMTRLDHDRGLAQVAAKVGCNITDISRFAIWGNHSATQYPDLSFTTI----KGQWGLNVIDEQWITNEFIPNV\r\n",
      "QQRGAAIIKARGKSSAASAADAAIKHMHDWVLGNS--EWVSMAI--PSRGQYGIPRGIWCSMPVQCFGAGKYGVI\r\n",
      "EGLPINSFSADRINASVKELIEEKKIVENLL----*\r\n",
      "\r\n",
      ">P1;4m1qA\r\n",
      "structure:4m1qA:   69: :  233: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--KSADIVVITAGAAQKPGETRLDLVEKNANIFKGIVDQVMGSGFNGI-FIIATNPVDVLAYATWKFSGLPKERV\r\n",
      "IGSGTILDTARFRFLLSEYFDIDVRNIHGYIM-GEHGDTELPVWSQTRIGSE-PISRYMDKYKPDKDLDEIFVNV\r\n",
      "RDAAYHIIERKGATHYA----------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;1pzgA\r\n",
      "structure:1pzgA:   71: :  190: :::-1.00:-1.00\r\n",
      "-------------------------------------------------------------------------EA\r\n",
      "ALTGADCVIVTAGLTKVPGKSRNDLLPFNSKIIREIGQNIKKYC-PKTFIIVVTNP--LDCMVKVMEASGVPTNM\r\n",
      "ICGMAMLDSGRFRRYVADALSVSPRDVQATVI-GTHGDCMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;2x0jA\r\n",
      "structure:2x0jA:   68: :  285: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "--KGSEIIVVTAGLARKPGMTRLDLAHKNAGIIKDIAKKIVENA-PESKILVVTNPMDVMTYIMWKESGKPRNEV\r\n",
      "FGMGNQLDSQRLKERL---YNAGARNIRRAWIIGEHGDSMFVAKSLADFDGEVDWEAV----ENDVRF-------\r\n",
      "--VAAEVIKRKGAT-IFGPAVAIYRMVKAVVEDT--GEIIPTSM-ILQGE-YGIE-NVAVGVPAKLGKNGA--EV\r\n",
      "ADIKLSDEEIEKLRNSAKIL---------------*\r\n",
      "\r\n",
      ">P1;5ujkA\r\n",
      "structure:5ujkA:   10: :  180: :::-1.00:-1.00\r\n",
      "------------AGQIGGTLAHLAGLKEL-GD----VVLFDIVDGVPQGKALDI-AESAPVDGFDAKYSGASDYS\r\n",
      "AIAGADVVIVTAGVPRKPGMSRDDLIGINLKVMEAVGAGIKEHA-PDAFVICITNP--LDAMVWALQKFSGLPTN\r\n",
      "VGMAGVLDSARFRHFLAEEFGVSVEDV-TAFVLGGHGDDMV----------------------------------\r\n",
      "---------------------------------------------------------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;4ajjA\r\n",
      "structure:4ajjA:   92: :  303: :::-1.00:-1.00\r\n",
      "---------------------------------------------------------------------------\r\n",
      "---------ITAGARQQEGESRLNLVQRNVNIFKFIIPNVVKYS-PQCKLLIVSNPVDILTYVAWKISGFPKNRV\r\n",
      "IGSGCNLDSARFRYLMGERLGVHPLSCHGWVL-GEHGDSSVSGVNVAGVSLKSLNPQLGTDADKEQWKDV-HKQV\r\n",
      "VDSAYEVIKLKGYTSWAIGLSVAAESIMKNLRRVHPISTMIKGL-------YGIKEDVFLSVPCILGQNGISDVV\r\n",
      "K----------------------------------*\r\n",
      "\r\n",
      ">P1;3nepX\r\n",
      "structure:3nepX:    8: :  263: :::-1.00:-1.00\r\n",
      "------------AGNVGATVAECVARQDVAKE----VVMVDIKDGMPQGKALDMR-ESSPIHGFDTRVTGTNDYG\r\n",
      "PTEDSDVCIITAGL-------RDDLLAKNTEIVGGVTEQFVE-GSPDSTIIVVANPLDVMTYVA--YEASGFPTN\r\n",
      "MGMAGVLDTGRFRSFIAEELDVSVRDVQALLM-GGHGDTMVPLPRYTTVGGIPVPQLIDDARIEEIVERT--KGA\r\n",
      "GGEIVDLMGTSAWAAAAEMTEAILKDNKAYCDGEYGLDDLFIGVPVKLG--------------------------\r\n",
      "-----------------------------------*\r\n",
      "\r\n",
      ">P1;6pblA\r\n",
      "structure:6pblA:    5: :  318: :::-1.00:-1.00\r\n",
      "------VAVTGAAGQIGYALVFRIASGQMFGNTEVELNLLELEPALPSLEGVAMELDDCAFPLLKRIVCTADLNK\r\n",
      "AMDGVNWALLVGSVPRKQGMERSDLLQINGGIFTKQGQAINDYASDDVRVFVVGNPCNTNCLIAMNHAKDVPSDR\r\n",
      "FYAMTTLDELRARTQLAKKAGVDITAVTQMTIWGNHSATQYPDFYNA---KINGTSAAQVINDETWLKETFVSTV\r\n",
      "QQRGAAVIKARGSSSAASAANAIITGVNHLVTDTPAGESFSM-CRRSKGE-YGVDEGLIFSFPCR-REHGELKVV\r\n",
      "ENLEFNDFGRERFNTTLNELRSERD----------*\r\n",
      "\r\n",
      ">P1;1y7tA\r\n",
      "structure:1y7tA:    1: :  319: :::-1.00:-1.00\r\n",
      "MKAPVRVAVTGAAGQIGYSLLFRIAAGEMLGDQPVILQLLEIPQAMKALEGVVMELEDCAFPLLAGLEATDDPKV\r\n",
      "AFKDADYALLVGAAPRKAGMERRDLLQVNGKIFTEQGRALAEVAKKDVKVLVVGNPANTNALIAYKNAPGLNPRN\r\n",
      "FTAMTRLDHNRAKAQLAKKTGTGVDRIRRMTVWGNHSSTMFPDL----FHAEVDGRPALELVDMEWYEKVFIPTV\r\n",
      "AQRGAAIIQARGASSAASAANAAIEHIRDWALGTPEGDWVSMA--VPSQGEYGIPEGIVYSFPVTA-KDGAYRVV\r\n",
      "EGLEINEFARKRMEITAQELLDEME----------*\r\n"
     ]
    }
   ],
   "source": [
    "!cat build_profile.ali"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100  825k    0  825k    0     0  62044      0 --:--:--  0:00:13 --:--:-- 91810:12 --:--:-- 85902\n"
     ]
    }
   ],
   "source": [
    "#Select the most appropriate template for our query sequence over the six similar PDB structures\n",
    "#Download the pdb files either manually from RCSB or using following command in Linux/MacOSX (wget/curl)\n",
    "\n",
    "#Change PDBIDs in the following link and after -o option (all in small letters)\n",
    "!curl https://files.rcsb.org/download/4h7p.pdb -o 4h7p.pdb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1265579    1235.917     1.207\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1266107    1236.433     1.207\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1314707    1283.894     1.254\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1328007    1296.882     1.266\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           3d5t.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1339202    1307.814     1.277\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1339278    1307.889     1.277\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1339449    1308.056     1.277\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1340299    1308.886     1.278\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1341557    1310.114     1.279\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1343461    1311.974     1.281\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1343501    1312.013     1.281\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1343591    1312.101     1.281\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1346447    1314.890     1.284\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1346507    1314.948     1.284\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1346642    1315.080     1.284\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1350926    1319.264     1.288\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1351014    1319.350     1.288\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1351212    1319.543     1.289\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1357638    1325.818     1.295\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1357770    1325.947     1.295\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1358067    1326.237     1.295\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1367689    1335.634     1.304\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1367889    1335.829     1.305\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1368339    1336.269     1.305\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382789    1350.380     1.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1383089    1350.673     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383764    1351.332     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1405422    1372.482     1.340\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1405870    1372.920     1.341\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1406878    1373.904     1.342\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1439382    1405.646     1.373\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1671254    1632.084     1.594\n",
      "read_mo_297_> Segments, residues, atoms:        1      321     2376\n",
      "read_mo_298_> Segment:        1     3 A   325 A     2376\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1680534    1641.146     1.603\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1861803    1818.167     1.776\n",
      "openf___224_> Open           6itk.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1872814    1828.920     1.786\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1872890    1828.994     1.786\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1873061    1829.161     1.786\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1873911    1829.991     1.787\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1875169    1831.220     1.788\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1877073    1833.079     1.790\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1877113    1833.118     1.790\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1877203    1833.206     1.790\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1880059    1835.995     1.793\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1880119    1836.054     1.793\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1880254    1836.186     1.793\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1884538    1840.369     1.797\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1884626    1840.455     1.797\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1884824    1840.648     1.798\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1891250    1846.924     1.804\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1891382    1847.053     1.804\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1891679    1847.343     1.804\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1901301    1856.739     1.813\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1901501    1856.935     1.813\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1901951    1857.374     1.814\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1916401    1871.485     1.828\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1916701    1871.778     1.828\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1917376    1872.438     1.829\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1939034    1893.588     1.849\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1939482    1894.025     1.850\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1940490    1895.010     1.851\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1972994    1926.752     1.882\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2204866    2153.189     2.103\n",
      "read_mo_297_> Segments, residues, atoms:        1      318     2387\n",
      "read_mo_298_> Segment:        1     8 A   330 A     2387\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1865134    1821.420     1.779\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2045838    1997.889     1.951\n",
      "openf___224_> Open           5mdh.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2056849    2008.642     1.962\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2056925    2008.716     1.962\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2057096    2008.883     1.962\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2057946    2009.713     1.963\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2059204    2010.941     1.964\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2061108    2012.801     1.966\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2061148    2012.840     1.966\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2061238    2012.928     1.966\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2064094    2015.717     1.968\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2064154    2015.775     1.969\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2064289    2015.907     1.969\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2068573    2020.091     1.973\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2068661    2020.177     1.973\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2068859    2020.370     1.973\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2075285    2026.646     1.979\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2075417    2026.774     1.979\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2075714    2027.064     1.980\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2085336    2036.461     1.989\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2085536    2036.656     1.989\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2085986    2037.096     1.989\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2100436    2051.207     2.003\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2100736    2051.500     2.003\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2101411    2052.159     2.004\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2123069    2073.310     2.025\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2123517    2073.747     2.025\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2124525    2074.731     2.026\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2157029    2106.474     2.057\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2388901    2332.911     2.278\n",
      "read_mo_297_> Segments, residues, atoms:        1      333     2550\n",
      "read_mo_298_> Segment:        1     1 A   333 A     2550\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2049733    2001.692     1.955\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2240674    2188.158     2.137\n",
      "openf___224_> Open           1bdm.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2251685    2198.911     2.147\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2251761    2198.985     2.147\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2251932    2199.152     2.148\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2252782    2199.982     2.148\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2254040    2201.211     2.150\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2255944    2203.070     2.151\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2255984    2203.109     2.151\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2256074    2203.197     2.152\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2258930    2205.986     2.154\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2258990    2206.045     2.154\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2259125    2206.177     2.154\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2263409    2210.360     2.159\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2263497    2210.446     2.159\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2263695    2210.640     2.159\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2270121    2216.915     2.165\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2270253    2217.044     2.165\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2270550    2217.334     2.165\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2280172    2226.730     2.175\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2280372    2226.926     2.175\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2280822    2227.365     2.175\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2295272    2241.477     2.189\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2295572    2241.770     2.189\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2296247    2242.429     2.190\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2317905    2263.579     2.211\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2318353    2264.017     2.211\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2319361    2265.001     2.212\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2351865    2296.743     2.243\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2583737    2523.181     2.464\n",
      "read_mo_297_> Segments, residues, atoms:        1      318     2406\n",
      "read_mo_298_> Segment:        1     0 A   332 A     2406\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2244113    2191.517     2.140\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2425463    2368.616     2.313\n",
      "openf___224_> Open           7mdh.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2436474    2379.369     2.324\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2436550    2379.443     2.324\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2436721    2379.610     2.324\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2437571    2380.440     2.325\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2438829    2381.669     2.326\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2440733    2383.528     2.328\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2440773    2383.567     2.328\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2440863    2383.655     2.328\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2443719    2386.444     2.331\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2443779    2386.503     2.331\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2443914    2386.635     2.331\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2448198    2390.818     2.335\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2448286    2390.904     2.335\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2448484    2391.098     2.335\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2454910    2397.373     2.341\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2455042    2397.502     2.341\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2455339    2397.792     2.342\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2464961    2407.188     2.351\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2465161    2407.384     2.351\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2465611    2407.823     2.351\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2480061    2421.935     2.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2480361    2422.228     2.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2481036    2422.887     2.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2502694    2444.037     2.387\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2503142    2444.475     2.387\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2504150    2445.459     2.388\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2536654    2477.201     2.419\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2537326    2477.857     2.420\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2538838    2479.334     2.421\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2794902    2729.396     2.665\n",
      "read_mo_297_> Segments, residues, atoms:        1      351     2688\n",
      "read_mo_298_> Segment:        1    23 A   385 A     2688\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2456448    2398.875     2.343\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2657715    2595.425     2.535\n",
      "openf___224_> Open           4h7p.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2668726    2606.178     2.545\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2668802    2606.252     2.545\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2668973    2606.419     2.545\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2669823    2607.249     2.546\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2671081    2608.478     2.547\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2672985    2610.337     2.549\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2673025    2610.376     2.549\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2673115    2610.464     2.549\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2675971    2613.253     2.552\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2676031    2613.312     2.552\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2676166    2613.443     2.552\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2680450    2617.627     2.556\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2680538    2617.713     2.556\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2680736    2617.906     2.557\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2687162    2624.182     2.563\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2687294    2624.311     2.563\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2687591    2624.601     2.563\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2697213    2633.997     2.572\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2697413    2634.192     2.572\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2697863    2634.632     2.573\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2712313    2648.743     2.587\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2712613    2649.036     2.587\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2713288    2649.695     2.588\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2734946    2670.846     2.608\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      2735394    2671.283     2.609\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2736402    2672.268     2.610\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      2768906    2704.010     2.641\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      3000778    2930.447     2.862\n",
      "read_mo_297_> Segments, residues, atoms:        1      313     2244\n",
      "read_mo_298_> Segment:        1     1 A   324 A     2244\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2634940    2573.184     2.513\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2809217    2743.376     2.679\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2809513    2743.665     2.679\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2809587    2743.737     2.679\n",
      "\n",
      "Multiple dynamic programming alignment (MALIGN):\n",
      "  Residue-residue metric  : $(LIB)/as1.sim.mat\n",
      "  ALIGN_BLOCK             :            1\n",
      "  Gap introduction penalty:    -900.0000\n",
      "  Gap extension penalty   :     -50.0000\n",
      "  Length of alignment     :          356\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2813233    2747.298     2.683\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2936777    2867.946     2.801\n",
      "malign3_328_> Initial framework positions:      321\n",
      "malign3_330_> Framework, Cycles, RMS_frw(i-1,i):        1        3        0.0000\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2938354    2869.486     2.802\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2944947    2875.925     2.809\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2945279    2876.249     2.809\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2946667    2877.604     2.810\n",
      "malign3_332_> Framework:        1\n",
      "# Sequence alignment of the structurally conserved regions\n",
      "# [average distance and standard deviation are with respect\n",
      "#  to the framework (i.e., average structure)]\n",
      "#\n",
      "#  N av ds st dv   3d5tA  6itkA  5mdhA  1bdmA  7mdhA  4h7pA  \n",
      "# ===========================================================\n",
      "   1 1.129 0.565   P   4  S   8  E   2  A   2  K  45  S   2  \n",
      "   2 0.601 0.252   A   5  T   9  P   3  P   3  L  46  A   3  \n",
      "   3 0.298 0.155   K   6  K  10  I   4  V   4  V  47  V   4  \n",
      "   4 0.265 0.126   R   7  K  11  R   5  R   5  N  48  K   5  \n",
      "   5 0.197 0.035   V   8  V  12  V   6  V   6  I  49  V   6  \n",
      "   6 0.283 0.086   A   9  T  13  L   7  A   7  A  50  A   7  \n",
      "   7 0.351 0.078 * V  10  V  14  V   8  V   8  V  51  V   8  \n",
      "   8 0.259 0.117   T  11  T  15  T   9  T   9  S  52  T   9  \n",
      "   9 0.322 0.147 * G  12  G  16  G  10  G  10  G  53  G  10  \n",
      "  10 0.398 0.092 * A  13  A  17  A  11  A  11  A  54  A  11  \n",
      "  11 0.370 0.072 * A  14  A  18  A  12  A  12  A  55  A  12  \n",
      "  12 0.288 0.125 * G  15  G  19  G  13  G  13  G  56  G  13  \n",
      "  13 0.274 0.113   Q  16  Q  20  Q  14  Q  14  M  57  Q  14  \n",
      "  14 0.230 0.125 * I  17  I  21  I  15  I  15  I  58  I  15  \n",
      "  15 0.268 0.115   A  18  S  22  A  16  G  16  S  59  G  16  \n",
      "  16 0.214 0.082   Y  19  Y  23  Y  17  Y  17  N  60  Y  17  \n",
      "  17 0.356 0.232   S  20  S  24  S  18  S  18  H  61  A  18  \n",
      "  18 0.309 0.103 * L  21  L  25  L  19  L  19  L  62  L  19  \n",
      "  19 0.274 0.119   L  22  L  26  L  20  L  20  L  63  V  20  \n",
      "  20 0.407 0.144   F  23  W  27  Y  21  F  21  F  64  P  21  \n",
      "  21 0.427 0.139   R  24  R  28  S  22  R  22  K  65  L  22  \n",
      "  22 0.339 0.128   I  25  I  29  I  23  I  23  L  66  I  23  \n",
      "  23 0.278 0.061   A  26  A  30  G  24  A  24  A  67  A  24  \n",
      "  24 0.385 0.140   N  27  N  31  N  25  A  25  S  68  R  25  \n",
      "  25 0.449 0.143 * G  28  G  32  G  26  G  26  G  69  G  26  \n",
      "  26 0.836 0.252   D  29  E  33  S  27  E  27  E  70  A  27  \n",
      "  27 0.833 0.276   L  30  V  34  V  28  M  28  V  71  L  28  \n",
      "  28 0.622 0.296   L  31  F  35  F  29  L  29  F  72  L  29  \n",
      "  29 0.666 0.250 * G  32  G  36  G  30  G  30  G  73  G  30  \n",
      "  30 0.546 0.205   K  33  T  37  K  31  K  31  Q  74  P  31  \n",
      "  31 0.554 0.147   D  34  D  38  D  32  D  32  D  75  T  32  \n",
      "  32 0.387 0.154   Q  35  T  39  Q  33  Q  33  Q  76  T  33  \n",
      "  33 0.412 0.118 * P  36  P  40  P  34  P  34  P  77  P  34  \n",
      "  34 0.310 0.073   V  37  V  41  I  35  V  35  I  78  V  35  \n",
      "  35 0.389 0.209   I  38  E  42  I  36  I  36  A  79  E  36  \n",
      "  36 0.250 0.123 * L  39  L  43  L  37  L  37  L  80  L  37  \n",
      "  37 0.270 0.065   Q  40  K  44  V  38  Q  38  K  81  R  38  \n",
      "  38 0.394 0.140 * L  41  L  45  L  39  L  39  L  82  L  39  \n",
      "  39 0.360 0.133 * L  42  L  46  L  40  L  40  L  83  L  40  \n",
      "  40 0.413 0.143   D  43  E  47  D  41  E  41  G  84  D  41  \n",
      "  41 0.544 0.218   L  44  I  48  I  42  I  42  S  85  I  42  \n",
      "  42 0.599 0.326   P  45  P  49  T  43  P  43  E  86  E  43  \n",
      "  43 0.860 0.563   Q  46  Q  50  P  44  Q  44  R  87  P  44  \n",
      "  44 0.559 0.272   A  47  A  51  M  45  A  45  S  88  A  45  \n",
      "  45 0.591 0.306   Q  48  L  52  M  46  M  46  F  89  L  46  \n",
      "  46 0.626 0.305   A  49  G  53  G  47  K  47  Q  90  K  47  \n",
      "  47 0.543 0.286   A  50  G  54  V  48  A  48  A  91  A  48  \n",
      "  48 0.531 0.384   V  51  A  55  L  49  L  49  L  92  L  49  \n",
      "  49 0.586 0.430   K  52  E  56  D  50  E  50  E  93  A  50  \n",
      "  50 0.532 0.371 * G  53  G  57  G  51  G  51  G  94  G  51  \n",
      "  51 0.445 0.253 * V  54  V  58  V  52  V  52  V  95  V  52  \n",
      "  52 0.586 0.358   V  55  A  59  L  53  V  53  A  96  E  53  \n",
      "  53 0.636 0.478   M  56  M  60  M  54  M  54  M  97  A  54  \n",
      "  54 0.465 0.325 * E  57  E  61  E  55  E  55  E  98  E  55  \n",
      "  55 0.479 0.268 * L  58  L  62  L  56  L  56  L  99  L  56  \n",
      "  56 0.707 0.487   D  59  L  63  Q  57  E  57  E 100  E  57  \n",
      "  57 0.714 0.541 * D  60  D  64  D  58  D  58  D 101  D  58  \n",
      "  58 0.602 0.468   C  61  S  65  C  59  C  59  S 102  C  59  \n",
      "  59 0.794 0.645   A  62  A  66  A  60  A  60  L 103  A  60  \n",
      "  60 0.338 0.067   F  63  F  67  L  61  F  61  Y 104  F  61  \n",
      "  61 0.530 0.218 * P  64  P  68  P  62  P  62  P 105  P  62  \n",
      "  62 0.273 0.126 * L  65  L  69  L  63  L  63  L 106  L  63  \n",
      "  63 0.256 0.113 * L  66  L  70  L  64  L  64  L 107  L  64  \n",
      "  64 0.374 0.167   A  67  R  71  K  65  A  65  R 108  D  65  \n",
      "  65 0.403 0.115   G  68  N  72  D  66  G  66  E 109  K  66  \n",
      "  66 0.362 0.052   V  69  I  73  V  67  L  67  V 110  V  67  \n",
      "  67 0.362 0.141   V  70  T  74  I  68  E  68  S 111  V  68  \n",
      "  68 0.504 0.176   I  71  I  75  A  69  A  69  I 112  V  69  \n",
      "  69 0.467 0.154   T  72  T  76  T  70  T  70  G 113  T  70  \n",
      "  70 0.463 0.216   D  73  A  77  D  71  D  71  I 114  A  71  \n",
      "  71 0.535 0.311   D  74  D  78  K  72  D  72  D 115  D  72  \n",
      "  72 0.568 0.426   P  75  A  79  E  73  P  73  P 116  P  73  \n",
      "  73 0.718 0.485   K  76  N  80  E  74  D  74  Y 117  R  74  \n",
      "  74 0.512 0.310   V  77  E  81  I  75  V  75  E 118  V  75  \n",
      "  75 0.442 0.209   A  78  A  82  A  76  A  76  V 119  A  76  \n",
      "  76 0.403 0.273 * F  79  F  83  F  77  F  77  F 120  F  77  \n",
      "  77 0.372 0.163   K  80  D  84  K  78  K  78  E 121  D  78  \n",
      "  78 0.347 0.224   D  81  G  85  D  79  D  79  D 122  G  79  \n",
      "  79 0.317 0.134   A  82  A  86  L  80  A  80  V 123  V  80  \n",
      "  80 0.217 0.101   D  83  N  87  D  81  D  81  D 124  A  81  \n",
      "  81 0.236 0.094   V  84  A  88  V  82  Y  82  W 125  I  82  \n",
      "  82 0.325 0.194 * A  85  A  89  A  83  A  83  A 126  A  83  \n",
      "  83 0.328 0.121   L  86  F  90  I  84  L  84  L 127  I  84  \n",
      "  84 0.262 0.148   L  87  L  91  L  85  L  85  L 128  M  85  \n",
      "  85 0.280 0.099   V  88  V  92  V  86  V  86  I 129  C  86  \n",
      "  86 0.349 0.084 * G  89  G  93  G  87  G  87  G 130  G  87  \n",
      "  87 0.525 0.236   A  90  A  94  S  88  A  88  A 131  A  88  \n",
      "# ===========================================================\n",
      "  88 1.167 0.574   P  92  P  96  P  90  R  91  P 133  P  90  \n",
      "# ===========================================================\n",
      "  89 1.162 0.371   A 105  N 109  A 103  V 103  I 146  M 103  \n",
      "  90 0.772 0.301 * N 106  N 110  N 104  N 104  N 147  N 104  \n",
      "  91 1.044 0.319   A 107  G 111  V 105  G 105  G 148  A 105  \n",
      "  92 0.901 0.233   E 108  K 112  K 106  K 106  Q 149  R 106  \n",
      "  93 0.451 0.144 * I 109  I 113  I 107  I 107  I 150  I 107  \n",
      "  94 0.438 0.153 * F 110  F 114  F 108  F 108  F 151  F 108  \n",
      "  95 0.524 0.171   T 111  G 115  K 109  T 109  A 152  K 109  \n",
      "  96 0.464 0.151   V 112  P 116  C 110  E 110  D 153  E 110  \n",
      "  97 0.322 0.107 * Q 113  Q 117  Q 111  Q 111  Q 154  Q 111  \n",
      "  98 0.317 0.123 * G 114  G 118  G 112  G 112  G 155  G 112  \n",
      "  99 0.409 0.157   A 115  K 119  A 113  R 113  K 156  E 113  \n",
      " 100 0.378 0.133 * A 116  A 120  A 114  A 114  A 157  A 114  \n",
      " 101 0.332 0.107   L 117  I 121  L 115  L 115  L 158  I 115  \n",
      " 102 0.409 0.155   N 118  N 122  D 116  A 116  N 159  A 116  \n",
      " 103 0.462 0.175   E 119  D 123  K 117  E 117  A 160  A 117  \n",
      " 104 0.440 0.212   V 120  N 124  Y 118  V 118  V 161  V 118  \n",
      " 105 0.351 0.150 * A 121  A 125  A 119  A 119  A 162  A 119  \n",
      " 106 0.525 0.323   S 122  A 126  K 120  K 120  S 163  A 120  \n",
      " 107 0.738 0.538   R 123  D 127  K 121  K 121  K 164  S 121  \n",
      " 108 0.629 0.485   D 124  D 128  S 122  D 122  N 165  D 122  \n",
      " 109 0.228 0.119   V 125  I 129  V 123  V 123  V 166  C 123  \n",
      " 110 0.224 0.068   K 126  R 130  K 124  K 124  K 167  R 124  \n",
      " 111 0.256 0.112 * V 127  V 131  V 125  V 125  V 168  V 125  \n",
      " 112 0.234 0.148   L 128  L 132  I 126  L 126  L 169  V 126  \n",
      " 113 0.158 0.056 * V 129  V 133  V 127  V 127  V 170  V 127  \n",
      " 114 0.294 0.117 * V 130  V 134  V 128  V 128  V 171  V 128  \n",
      " 115 0.384 0.282 * G 131  G 135  G 129  G 129  G 172  G 129  \n",
      " 116 0.388 0.246 * N 132  N 136  N 130  N 130  N 173  N 130  \n",
      " 117 0.374 0.241 * P 133  P 137  P 131  P 131  P 174  P 131  \n",
      " 118 0.330 0.175   A 134  A 138  A 132  A 132  C 175  A 132  \n",
      " 119 0.285 0.172 * N 135  N 139  N 133  N 133  N 176  N 133  \n",
      " 120 0.291 0.146 * T 136  T 140  T 134  T 134  T 177  T 134  \n",
      " 121 0.315 0.163 * N 137  N 141  N 135  N 135  N 178  N 135  \n",
      " 122 0.331 0.100   A 138  A 142  C 136  A 136  A 179  A 136  \n",
      " 123 0.330 0.118   Y 139  L 143  L 137  L 137  L 180  L 137  \n",
      " 124 0.370 0.159   I 140  I 144  T 138  I 138  I 181  I 138  \n",
      " 125 0.324 0.104   A 141  A 145  A 139  A 139  C 182  L 139  \n",
      " 126 0.363 0.145   M 142  S 146  S 140  Y 140  L 183  L 140  \n",
      " 127 0.341 0.169   K 143  A 147  K 141  K 141  K 184  K 141  \n",
      " 128 0.446 0.245   S 144  A 148  S 142  N 142  N 185  S 142  \n",
      " 129 0.503 0.278 * A 145  A 149  A 143  A 143  A 186  A 143  \n",
      "# ===========================================================\n",
      " 130 0.837 0.557   P 146  P 150  P 144  P 144  P 187  G 145  \n",
      " 131 0.449 0.251   D 147  D 151  S 145  G 145  D 188  K 146  \n",
      " 132 0.501 0.297   L 148  V 152  I 146  L 146  I 189  L 147  \n",
      " 133 0.436 0.130   P 149  P 153  P 147  N 147  P 190  N 148  \n",
      " 134 0.449 0.191   K 150  A 154  K 148  P 148  A 191  P 149  \n",
      " 135 0.475 0.186   K 151  S 155  E 149  R 149  K 192  R 150  \n",
      " 136 0.324 0.089   N 152  R 156  N 150  N 150  N 193  H 151  \n",
      " 137 0.292 0.086   F 153  F 157  F 151  F 151  F 194  V 152  \n",
      " 138 0.281 0.080   T 154  N 158  S 152  T 152  H 195  T 153  \n",
      " 139 0.263 0.163   A 155  A 159  C 153  A 153  A 196  A 154  \n",
      " 140 0.166 0.062   M 156  M 160  L 154  M 154  L 197  M 155  \n",
      " 141 0.238 0.137   L 157  M 161  T 155  T 155  T 198  T 156  \n",
      " 142 0.184 0.057 * R 158  R 162  R 156  R 156  R 199  R 157  \n",
      " 143 0.248 0.084 * L 159  L 163  L 157  L 157  L 200  L 158  \n",
      " 144 0.261 0.122 * D 160  D 164  D 158  D 158  D 201  D 159  \n",
      " 145 0.207 0.086   H 161  H 165  H 159  H 159  E 202  H 160  \n",
      " 146 0.184 0.038 * N 162  N 166  N 160  N 160  N 203  N 161  \n",
      " 147 0.253 0.114 * R 163  R 167  R 161  R 161  R 204  R 162  \n",
      " 148 0.247 0.124 * A 164  A 168  A 162  A 162  A 205  A 163  \n",
      " 149 0.234 0.115   L 165  I 169  K 163  K 163  K 206  L 164  \n",
      " 150 0.346 0.090   S 166  S 170  A 164  A 164  C 207  S 165  \n",
      " 151 0.420 0.080   Q 167  Q 171  Q 165  Q 165  Q 208  L 166  \n",
      " 152 0.348 0.198   L 168  L 172  I 166  L 166  L 209  L 167  \n",
      " 153 0.446 0.212 * A 169  A 173  A 167  A 167  A 210  A 168  \n",
      " 154 0.548 0.205   A 170  T 174  L 168  K 168  L 211  R 169  \n",
      " 155 0.558 0.258 * K 171  K 175  K 169  K 169  K 212  K 170  \n",
      " 156 0.664 0.279   S 172  L 176  L 170  T 170  A 213  A 171  \n",
      " 157 0.805 0.354 * G 173  G 177  G 171  G 171  G 214  G 172  \n",
      " 158 0.786 0.253   K 174  R 178  V 172  T 172  V 215  V 173  \n",
      " 159 0.864 0.262   P 175  G 179  T 173  G 173  F 216  P 174  \n",
      " 160 0.667 0.150   V 176  S 180  S 174  V 174  Y 217  V 175  \n",
      " 161 0.697 0.192   A 177  A 181  D 175  D 175  D 218  S 176  \n",
      " 162 0.620 0.266   S 178  E 182  D 176  R 176  K 219  Q 177  \n",
      " 163 0.690 0.204   I 179  F 183  V 177  I 177  V 220  V 178  \n",
      " 164 1.002 0.347   E 180  N 184  K 178  R 178  S 221  R 179  \n",
      " 165 0.780 0.329   K 181  N 185  N 179  R 179  N 222  N 180  \n",
      " 166 0.755 0.209   L 182  I 186  V 180  M 180  V 223  V 181  \n",
      " 167 0.513 0.227   A 183  V 187  I 181  T 181  T 224  I 182  \n",
      " 168 0.476 0.196   V 184  V 188  I 182  V 182  I 225  I 183  \n",
      " 169 0.283 0.100 * W 185  W 189  W 183  W 183  W 226  W 184  \n",
      " 170 0.426 0.066 * G 186  G 190  G 184  G 184  G 227  G 185  \n",
      " 171 0.642 0.412 * N 187  N 191  N 185  N 185  N 228  N 186  \n",
      " 172 0.563 0.292 * H 188  H 192  H 186  H 186  H 229  H 187  \n",
      " 173 0.527 0.144 * S 189  S 193  S 187  S 187  S 230  S 188  \n",
      " 174 0.671 0.243   P 190  A 194  S 188  S 188  T 231  S 189  \n",
      " 175 0.703 0.266   T 191  T 195  T 189  I 189  T 232  T 190  \n",
      " 176 0.458 0.141   M 192  Q 196  Q 190  M 190  Q 233  Q 191  \n",
      " 177 0.451 0.134   Y 193  F 197  Y 191  F 191  V 234  V 192  \n",
      " 178 0.406 0.201 * P 194  P 198  P 192  P 192  P 235  P 193  \n",
      " 179 0.433 0.179 * D 195  D 199  D 193  D 193  D 236  D 194  \n",
      " 180 0.706 0.257   F 196  I 200  V 194  L 194  F 237  T 195  \n",
      " 181 0.599 0.252   R 197  T 201  N 195  F 195  L 238  D 196  \n",
      " 182 0.719 0.420   F 198  Y 202  H 196  H 196  N 239  S 197  \n",
      " 183 0.711 0.422 * A 199  A 203  A 197  A 197  A 240  A 198  \n",
      " 184 0.703 0.426   T 200  T 204  K 198  E 198  K 241  V 199  \n",
      " 185 0.769 0.301   A 201  V 205  V 199  V 199  I 242  I 200  \n",
      " 186 1.283 0.465   E 202  G 206  K 200  D 200  D 243  G 201  \n",
      "# ===========================================================\n",
      " 187 1.495 0.434   G 203  G 207  E 205  G 205  G 244  T 202  \n",
      " 188 0.918 0.484   E 204  E 208  V 206  R 206  R 245  T 203  \n",
      " 189 0.722 0.375   S 205  K 209  G 207  P 207  P 246  P 204  \n",
      " 190 0.605 0.298   L 206  V 210  V 208  A 208  V 247  A 205  \n",
      " 191 0.680 0.459   L 207  T 211  Y 209  L 209  K 248  R 206  \n",
      " 192 0.762 0.475   K 208  D 212  E 210  E 210  E 249  E 207  \n",
      " 193 0.945 0.476   L 209  L 213  A 211  L 211  V 250  A 208  \n",
      " 194 1.017 0.613   I 210  V 214  V 212  V 212  I 251  I 209  \n",
      "# ===========================================================\n",
      " 195 0.716 0.444   T 219  E 222  E 221  V 221  E 260  D 217  \n",
      " 196 0.477 0.234 * F 220  F 223  F 222  F 222  F 261  F 218  \n",
      " 197 0.435 0.282   I 221  I 224  I 223  I 223  T 262  V 219  \n",
      " 198 0.453 0.280   P 222  P 225  T 224  P 224  I 263  Q 220  \n",
      " 199 0.442 0.281   T 223  R 226  T 225  T 225  T 264  V 221  \n",
      " 200 0.455 0.295 * V 224  V 227  V 226  V 226  V 265  V 222  \n",
      " 201 0.695 0.314   G 225  A 228  Q 227  A 227  Q 266  R 223  \n",
      " 202 0.784 0.480   K 226  N 229  Q 228  Q 228  K 267  G 224  \n",
      " 203 0.656 0.440 * R 227  R 230  R 229  R 229  R 268  R 225  \n",
      " 204 0.773 0.527 * G 228  G 231  G 230  G 230  G 269  G 226  \n",
      " 205 1.004 0.676   A 229  A 232  A 231  A 231  G 270  A 227  \n",
      " 206 0.905 0.715   A 230  E 233  A 232  A 232  A 271  E 228  \n",
      " 207 0.811 0.561   I 231  I 234  V 233  I 233  L 272  I 229  \n",
      " 208 0.946 0.693 * I 232  I 235  I 234  I 234  I 273  I 230  \n",
      "# ===========================================================\n",
      " 209 0.773 0.295   E 233  E 236  A 236  Q 235  Q 274  Q 231  \n",
      "# ===========================================================\n",
      " 210 1.116 0.718   R 235  R 238  R 237  R 237  W 276  R 233  \n",
      " 211 1.121 0.889   G 236  G 239  K 238  G 238  G 277  G 234  \n",
      " 212 0.805 0.600   L 237  K 240  L 239  A 239  R 278  L 235  \n",
      " 213 0.942 0.543 * S 238  S 241  S 240  S 240  S 279  S 236  \n",
      " 214 0.666 0.418 * S 239  S 242  S 241  S 241  S 280  S 237  \n",
      " 215 0.379 0.153 * A 240  A 243  A 242  A 242  A 281  A 238  \n",
      " 216 0.339 0.176   A 241  A 244  M 243  A 243  A 282  M 239  \n",
      " 217 0.370 0.209 * S 242  S 245  S 244  S 244  S 283  S 240  \n",
      " 218 0.327 0.151   A 243  A 246  A 245  A 245  T 284  A 241  \n",
      " 219 0.276 0.154 * A 244  A 247  A 246  A 246  A 285  A 242  \n",
      " 220 0.261 0.113   N 245  S 248  K 247  N 247  V 286  K 243  \n",
      " 221 0.262 0.121   A 246  S 249  A 248  A 248  S 287  A 244  \n",
      " 222 0.299 0.119   A 247  A 250  I 249  A 249  I 288  A 245  \n",
      " 223 0.364 0.065   I 248  I 251  C 250  I 250  A 289  V 246  \n",
      " 224 0.294 0.073   D 249  D 252  D 251  E 251  D 290  D 247  \n",
      " 225 0.266 0.087   H 250  H 253  H 252  H 252  A 291  H 248  \n",
      " 226 0.371 0.165   V 251  M 254  V 253  I 253  I 292  V 249  \n",
      " 227 0.435 0.282   R 252  R 255  R 254  R 254  K 293  H 250  \n",
      " 228 0.391 0.204   D 253  D 256  D 255  D 255  S 294  D 251  \n",
      " 229 0.475 0.138   W 254  W 257  I 256  W 256  L 295  W 252  \n",
      " 230 0.697 0.402   V 255  V 258  W 257  A 257  V 296  I 253  \n",
      " 231 0.670 0.528   L 256  Q 259  F 258  L 258  T 297  H 254  \n",
      " 232 0.891 0.618   G 257  G 260  G 259  G 259  P 298  G 255  \n",
      " 233 0.589 0.298 * T 258  T 261  T 260  T 260  T 299  T 256  \n",
      "# ===========================================================\n",
      " 234 1.592 0.540   G 260  E 262  E 262  E 262  E 301  E 258  \n",
      "# ===========================================================\n",
      " 235 0.953 0.476   K 261  A 263  E 264  D 264  D 303  V 260  \n",
      " 236 0.559 0.258   W 262  W 264  F 265  W 265  W 304  Y 261  \n",
      " 237 0.437 0.170   V 263  S 265  V 266  V 266  F 305  V 262  \n",
      " 238 0.230 0.068   T 264  S 266  S 267  S 267  S 306  S 263  \n",
      " 239 0.368 0.204   M 265  A 267  M 268  M 268  T 307  M 264  \n",
      " 240 0.314 0.141   G 266  A 268  G 269  A 269  G 308  G 265  \n",
      " 241 0.290 0.151   I 267  I 269  I 270  V 270  V 309  V 266  \n",
      " 242 0.353 0.116   P 268  P 270  I 271  P 271  Y 310  Y 267  \n",
      " 243 0.333 0.131   S 269  S 271  S 272  S 272  T 311  S 268  \n",
      " 244 1.102 0.305   D 270  T 272  D 273  Q 273  T 312  D 269  \n",
      "# ===========================================================\n",
      " 245 1.268 0.532   S 272  A 274  N 275  E 275  N 314  N 271  \n",
      "# ===========================================================\n",
      " 246 0.649 0.378 * Y 273  Y 275  Y 277  Y 277  Y 316  Y 273  \n",
      " 247 1.091 0.543 * G 274  G 276  G 278  G 278  G 317  G 274  \n",
      " 248 0.549 0.244   I 275  I 277  V 279  I 279  I 318  V 275  \n",
      " 249 0.456 0.246   P 276  P 278  P 280  P 280  A 319  P 276  \n",
      " 250 0.461 0.216   E 277  E 279  D 281  E 281  E 320  S 277  \n",
      " 251 0.892 0.362   D 278  G 280  D 282  G 282  D 321  G 278  \n",
      " 252 0.652 0.242   I 279  I 281  L 283  I 283  I 322  L 279  \n",
      " 253 0.376 0.163   I 280  F 282  L 284  V 284  V 323  I 280  \n",
      " 254 0.360 0.189   Y 281  V 283  Y 285  Y 285  F 324  F 281  \n",
      " 255 0.369 0.206   G 282  G 284  S 286  S 286  S 325  S 282  \n",
      " 256 0.317 0.110   V 283  L 285  F 287  F 287  M 326  F 283  \n",
      " 257 0.403 0.260 * P 284  P 286  P 288  P 288  P 327  P 284  \n",
      " 258 0.427 0.135   V 285  T 287  V 289  V 289  C 328  C 285  \n",
      " 259 0.476 0.248   I 286  V 288  T 290  T 290  R 329  T 286  \n",
      " 260 0.542 0.238   C 287  S 289  I 291  A 291  S 330  C 287  \n",
      " 261 0.818 0.138   E 288  R 290  K 292  K 292  K 331  H 288  \n",
      "# ===========================================================\n",
      " 262 1.243 0.399   N 289  N 291  D 293  D 293  D 333  A 289  \n",
      " 263 0.767 0.480   G 290  G 292  K 294  G 294  G 334  G 290  \n",
      " 264 0.805 0.348   E 291  E 293  T 295  A 295  D 335  E 291  \n",
      " 265 0.693 0.188   Y 292  W 294  W 296  Y 296  Y 336  W 292  \n",
      " 266 0.583 0.207   K 293  E 295  K 297  R 297  E 337  T 293  \n",
      " 267 0.443 0.139   R 294  I 296  I 298  V 298  L 338  V 294  \n",
      " 268 0.422 0.141   V 295  V 297  V 299  V 299  A 339  V 295  \n",
      " 269 0.861 0.366   E 296  E 298  E 300  E 300  T 340  S 296  \n",
      "# ===========================================================\n",
      " 270 1.014 0.563   L 298  L 300  L 302  L 302  V 342  G 297  \n",
      "# ===========================================================\n",
      " 271 0.890 0.488   I 300  I 302  I 304  I 304  N 344  L 299  \n",
      "# ===========================================================\n",
      " 272 1.016 0.613   R 305  R 307  R 309  R 309  W 349  K 305  \n",
      " 273 0.954 0.740   E 306  A 308  E 310  K 310  E 350  Q 306  \n",
      " 274 0.902 0.706   K 307  R 309  K 311  R 311  R 351  R 307  \n",
      " 275 0.587 0.250   M 308  I 310  M 312  M 312  I 352  L 308  \n",
      " 276 0.675 0.251   D 309  D 311  D 313  E 313  K 353  A 309  \n",
      " 277 0.596 0.222   G 310  A 312  L 314  I 314  K 354  S 310  \n",
      " 278 0.511 0.315   T 311  N 313  T 315  T 315  S 355  T 311  \n",
      " 279 0.543 0.294   L 312  A 314  A 316  A 316  E 356  I 312  \n",
      " 280 0.618 0.266   A 313  Q 315  K 317  Q 317  A 357  A 313  \n",
      " 281 0.629 0.263 * E 314  E 316  E 318  E 318  E 358  E 314  \n",
      " 282 0.445 0.208 * L 315  L 317  L 319  L 319  L 359  L 315  \n",
      " 283 0.535 0.204   L 316  Q 318  A 320  L 320  L 360  Q 316  \n",
      " 284 0.701 0.205   E 317  A 319  E 321  D 321  A 361  E 317  \n",
      " 285 0.635 0.156 * E 318  E 320  E 322  E 322  E 362  E 318  \n",
      " 286 0.570 0.143   R 319  R 321  K 323  M 323  K 363  R 319  \n",
      " 287 0.743 0.235   D 320  E 322  E 324  E 324  K 364  A 320  \n",
      " 288 0.804 0.291   G 321  A 323  T 325  Q 325  C 365  Q 321  \n",
      " 289 0.895 0.315   V 322  V 324  A 326  V 326  V 366  A 322  \n",
      " 290 1.027 0.304   A 323  R 325  F 327  K 327  A 367  G 323  \n",
      "# ===========================================================\n",
      "malign3_330_> Framework, Cycles, RMS_frw(i-1,i):        2        2        0.0000\n",
      "malign3_332_> Framework:        2\n",
      "# Sequence alignment of the structurally conserved regions\n",
      "# [average distance and standard deviation are with respect\n",
      "#  to the framework (i.e., average structure)]\n",
      "#\n",
      "#  N av ds st dv   3d5tA  6itkA  5mdhA  1bdmA  7mdhA  4h7pA  \n",
      "# ===========================================================\n",
      "   1 1.133 0.571   P   4  S   8  E   2  A   2  K  45  S   2  \n",
      "   2 0.593 0.256   A   5  T   9  P   3  P   3  L  46  A   3  \n",
      "   3 0.284 0.134   K   6  K  10  I   4  V   4  V  47  V   4  \n",
      "   4 0.245 0.114   R   7  K  11  R   5  R   5  N  48  K   5  \n",
      "   5 0.195 0.028   V   8  V  12  V   6  V   6  I  49  V   6  \n",
      "   6 0.281 0.075   A   9  T  13  L   7  A   7  A  50  A   7  \n",
      "   7 0.342 0.057 * V  10  V  14  V   8  V   8  V  51  V   8  \n",
      "   8 0.253 0.114   T  11  T  15  T   9  T   9  S  52  T   9  \n",
      "   9 0.332 0.147 * G  12  G  16  G  10  G  10  G  53  G  10  \n",
      "  10 0.382 0.088 * A  13  A  17  A  11  A  11  A  54  A  11  \n",
      "  11 0.367 0.066 * A  14  A  18  A  12  A  12  A  55  A  12  \n",
      "  12 0.280 0.124 * G  15  G  19  G  13  G  13  G  56  G  13  \n",
      "  13 0.292 0.128   Q  16  Q  20  Q  14  Q  14  M  57  Q  14  \n",
      "  14 0.257 0.171 * I  17  I  21  I  15  I  15  I  58  I  15  \n",
      "  15 0.285 0.127   A  18  S  22  A  16  G  16  S  59  G  16  \n",
      "  16 0.219 0.098   Y  19  Y  23  Y  17  Y  17  N  60  Y  17  \n",
      "  17 0.373 0.254   S  20  S  24  S  18  S  18  H  61  A  18  \n",
      "  18 0.330 0.112 * L  21  L  25  L  19  L  19  L  62  L  19  \n",
      "  19 0.271 0.116   L  22  L  26  L  20  L  20  L  63  V  20  \n",
      "  20 0.391 0.131   F  23  W  27  Y  21  F  21  F  64  P  21  \n",
      "  21 0.413 0.124   R  24  R  28  S  22  R  22  K  65  L  22  \n",
      "  22 0.342 0.124   I  25  I  29  I  23  I  23  L  66  I  23  \n",
      "  23 0.263 0.069   A  26  A  30  G  24  A  24  A  67  A  24  \n",
      "  24 0.371 0.147   N  27  N  31  N  25  A  25  S  68  R  25  \n",
      "  25 0.439 0.127 * G  28  G  32  G  26  G  26  G  69  G  26  \n",
      "  26 0.846 0.246   D  29  E  33  S  27  E  27  E  70  A  27  \n",
      "  27 0.846 0.260   L  30  V  34  V  28  M  28  V  71  L  28  \n",
      "  28 0.624 0.265   L  31  F  35  F  29  L  29  F  72  L  29  \n",
      "  29 0.649 0.215 * G  32  G  36  G  30  G  30  G  73  G  30  \n",
      "  30 0.545 0.202   K  33  T  37  K  31  K  31  Q  74  P  31  \n",
      "  31 0.585 0.126   D  34  D  38  D  32  D  32  D  75  T  32  \n",
      "  32 0.375 0.143   Q  35  T  39  Q  33  Q  33  Q  76  T  33  \n",
      "  33 0.396 0.129 * P  36  P  40  P  34  P  34  P  77  P  34  \n",
      "  34 0.302 0.073   V  37  V  41  I  35  V  35  I  78  V  35  \n",
      "  35 0.384 0.201   I  38  E  42  I  36  I  36  A  79  E  36  \n",
      "  36 0.249 0.112 * L  39  L  43  L  37  L  37  L  80  L  37  \n",
      "  37 0.255 0.053   Q  40  K  44  V  38  Q  38  K  81  R  38  \n",
      "  38 0.366 0.159 * L  41  L  45  L  39  L  39  L  82  L  39  \n",
      "  39 0.328 0.137 * L  42  L  46  L  40  L  40  L  83  L  40  \n",
      "  40 0.422 0.142   D  43  E  47  D  41  E  41  G  84  D  41  \n",
      "  41 0.557 0.200   L  44  I  48  I  42  I  42  S  85  I  42  \n",
      "  42 0.608 0.321   P  45  P  49  T  43  P  43  E  86  E  43  \n",
      "  43 0.838 0.564   Q  46  Q  50  P  44  Q  44  R  87  P  44  \n",
      "  44 0.545 0.254   A  47  A  51  M  45  A  45  S  88  A  45  \n",
      "  45 0.564 0.287   Q  48  L  52  M  46  M  46  F  89  L  46  \n",
      "  46 0.593 0.280   A  49  G  53  G  47  K  47  Q  90  K  47  \n",
      "  47 0.517 0.246   A  50  G  54  V  48  A  48  A  91  A  48  \n",
      "  48 0.497 0.352   V  51  A  55  L  49  L  49  L  92  L  49  \n",
      "  49 0.549 0.395   K  52  E  56  D  50  E  50  E  93  A  50  \n",
      "  50 0.490 0.333 * G  53  G  57  G  51  G  51  G  94  G  51  \n",
      "  51 0.418 0.229 * V  54  V  58  V  52  V  52  V  95  V  52  \n",
      "  52 0.563 0.331   V  55  A  59  L  53  V  53  A  96  E  53  \n",
      "  53 0.599 0.437   M  56  M  60  M  54  M  54  M  97  A  54  \n",
      "  54 0.427 0.279 * E  57  E  61  E  55  E  55  E  98  E  55  \n",
      "  55 0.453 0.247 * L  58  L  62  L  56  L  56  L  99  L  56  \n",
      "  56 0.676 0.461   D  59  L  63  Q  57  E  57  E 100  E  57  \n",
      "  57 0.671 0.507 * D  60  D  64  D  58  D  58  D 101  D  58  \n",
      "  58 0.556 0.455   C  61  S  65  C  59  C  59  S 102  C  59  \n",
      "  59 0.751 0.625   A  62  A  66  A  60  A  60  L 103  A  60  \n",
      "  60 0.309 0.071   F  63  F  67  L  61  F  61  Y 104  F  61  \n",
      "  61 0.556 0.223 * P  64  P  68  P  62  P  62  P 105  P  62  \n",
      "  62 0.266 0.103 * L  65  L  69  L  63  L  63  L 106  L  63  \n",
      "  63 0.257 0.096 * L  66  L  70  L  64  L  64  L 107  L  64  \n",
      "  64 0.349 0.182   A  67  R  71  K  65  A  65  R 108  D  65  \n",
      "  65 0.411 0.110   G  68  N  72  D  66  G  66  E 109  K  66  \n",
      "  66 0.341 0.052   V  69  I  73  V  67  L  67  V 110  V  67  \n",
      "  67 0.341 0.135   V  70  T  74  I  68  E  68  S 111  V  68  \n",
      "  68 0.474 0.153   I  71  I  75  A  69  A  69  I 112  V  69  \n",
      "  69 0.450 0.124   T  72  T  76  T  70  T  70  G 113  T  70  \n",
      "  70 0.455 0.177   D  73  A  77  D  71  D  71  I 114  A  71  \n",
      "  71 0.540 0.290   D  74  D  78  K  72  D  72  D 115  D  72  \n",
      "  72 0.549 0.414   P  75  A  79  E  73  P  73  P 116  P  73  \n",
      "  73 0.703 0.478   K  76  N  80  E  74  D  74  Y 117  R  74  \n",
      "  74 0.514 0.310   V  77  E  81  I  75  V  75  E 118  V  75  \n",
      "  75 0.400 0.199   A  78  A  82  A  76  A  76  V 119  A  76  \n",
      "  76 0.374 0.264 * F  79  F  83  F  77  F  77  F 120  F  77  \n",
      "  77 0.346 0.138   K  80  D  84  K  78  K  78  E 121  D  78  \n",
      "  78 0.315 0.218   D  81  G  85  D  79  D  79  D 122  G  79  \n",
      "  79 0.289 0.125   A  82  A  86  L  80  A  80  V 123  V  80  \n",
      "  80 0.246 0.093   D  83  N  87  D  81  D  81  D 124  A  81  \n",
      "  81 0.242 0.083   V  84  A  88  V  82  Y  82  W 125  I  82  \n",
      "  82 0.327 0.174 * A  85  A  89  A  83  A  83  A 126  A  83  \n",
      "  83 0.313 0.132   L  86  F  90  I  84  L  84  L 127  I  84  \n",
      "  84 0.249 0.167   L  87  L  91  L  85  L  85  L 128  M  85  \n",
      "  85 0.278 0.100   V  88  V  92  V  86  V  86  I 129  C  86  \n",
      "  86 0.313 0.089 * G  89  G  93  G  87  G  87  G 130  G  87  \n",
      "  87 0.497 0.244   A  90  A  94  S  88  A  88  A 131  A  88  \n",
      "# ===========================================================\n",
      "  88 1.118 0.587   P  92  P  96  P  90  R  91  P 133  P  90  \n",
      "# ===========================================================\n",
      "  89 0.860 0.437   S 104  A 108  K 102  L 101  D 145  E 102  \n",
      "# ===========================================================\n",
      "  90 1.117 0.349   A 105  N 109  A 103  V 103  I 146  M 103  \n",
      "  91 0.739 0.302 * N 106  N 110  N 104  N 104  N 147  N 104  \n",
      "  92 1.041 0.342   A 107  G 111  V 105  G 105  G 148  A 105  \n",
      "  93 0.875 0.258   E 108  K 112  K 106  K 106  Q 149  R 106  \n",
      "  94 0.414 0.089 * I 109  I 113  I 107  I 107  I 150  I 107  \n",
      "  95 0.402 0.156 * F 110  F 114  F 108  F 108  F 151  F 108  \n",
      "  96 0.500 0.193   T 111  G 115  K 109  T 109  A 152  K 109  \n",
      "  97 0.433 0.142   V 112  P 116  C 110  E 110  D 153  E 110  \n",
      "  98 0.287 0.050 * Q 113  Q 117  Q 111  Q 111  Q 154  Q 111  \n",
      "  99 0.281 0.090 * G 114  G 118  G 112  G 112  G 155  G 112  \n",
      " 100 0.337 0.138   A 115  K 119  A 113  R 113  K 156  E 113  \n",
      " 101 0.330 0.116 * A 116  A 120  A 114  A 114  A 157  A 114  \n",
      " 102 0.276 0.091   L 117  I 121  L 115  L 115  L 158  I 115  \n",
      " 103 0.341 0.139   N 118  N 122  D 116  A 116  N 159  A 116  \n",
      " 104 0.399 0.169   E 119  D 123  K 117  E 117  A 160  A 117  \n",
      " 105 0.396 0.179   V 120  N 124  Y 118  V 118  V 161  V 118  \n",
      " 106 0.339 0.117 * A 121  A 125  A 119  A 119  A 162  A 119  \n",
      " 107 0.532 0.290   S 122  A 126  K 120  K 120  S 163  A 120  \n",
      " 108 0.744 0.538   R 123  D 127  K 121  K 121  K 164  S 121  \n",
      " 109 0.661 0.488   D 124  D 128  S 122  D 122  N 165  D 122  \n",
      " 110 0.251 0.107   V 125  I 129  V 123  V 123  V 166  C 123  \n",
      " 111 0.244 0.075   K 126  R 130  K 124  K 124  K 167  R 124  \n",
      " 112 0.254 0.107 * V 127  V 131  V 125  V 125  V 168  V 125  \n",
      " 113 0.230 0.156   L 128  L 132  I 126  L 126  L 169  V 126  \n",
      " 114 0.180 0.049 * V 129  V 133  V 127  V 127  V 170  V 127  \n",
      " 115 0.292 0.095 * V 130  V 134  V 128  V 128  V 171  V 128  \n",
      " 116 0.352 0.225 * G 131  G 135  G 129  G 129  G 172  G 129  \n",
      " 117 0.340 0.158 * N 132  N 136  N 130  N 130  N 173  N 130  \n",
      " 118 0.329 0.146 * P 133  P 137  P 131  P 131  P 174  P 131  \n",
      " 119 0.294 0.091   A 134  A 138  A 132  A 132  C 175  A 132  \n",
      " 120 0.237 0.088 * N 135  N 139  N 133  N 133  N 176  N 133  \n",
      " 121 0.235 0.068 * T 136  T 140  T 134  T 134  T 177  T 134  \n",
      " 122 0.275 0.084 * N 137  N 141  N 135  N 135  N 178  N 135  \n",
      " 123 0.284 0.097   A 138  A 142  C 136  A 136  A 179  A 136  \n",
      " 124 0.291 0.114   Y 139  L 143  L 137  L 137  L 180  L 137  \n",
      " 125 0.343 0.129   I 140  I 144  T 138  I 138  I 181  I 138  \n",
      " 126 0.288 0.098   A 141  A 145  A 139  A 139  C 182  L 139  \n",
      " 127 0.311 0.059   M 142  S 146  S 140  Y 140  L 183  L 140  \n",
      " 128 0.305 0.074   K 143  A 147  K 141  K 141  K 184  K 141  \n",
      " 129 0.419 0.196   S 144  A 148  S 142  N 142  N 185  S 142  \n",
      " 130 0.494 0.254 * A 145  A 149  A 143  A 143  A 186  A 143  \n",
      "# ===========================================================\n",
      " 131 0.828 0.619   P 146  P 150  P 144  P 144  P 187  G 145  \n",
      " 132 0.377 0.246   D 147  D 151  S 145  G 145  D 188  K 146  \n",
      " 133 0.472 0.331   L 148  V 152  I 146  L 146  I 189  L 147  \n",
      " 134 0.399 0.120   P 149  P 153  P 147  N 147  P 190  N 148  \n",
      " 135 0.476 0.203   K 150  A 154  K 148  P 148  A 191  P 149  \n",
      " 136 0.474 0.184   K 151  S 155  E 149  R 149  K 192  R 150  \n",
      " 137 0.343 0.113   N 152  R 156  N 150  N 150  N 193  H 151  \n",
      " 138 0.323 0.106   F 153  F 157  F 151  F 151  F 194  V 152  \n",
      " 139 0.317 0.116   T 154  N 158  S 152  T 152  H 195  T 153  \n",
      " 140 0.284 0.189   A 155  A 159  C 153  A 153  A 196  A 154  \n",
      " 141 0.166 0.056   M 156  M 160  L 154  M 154  L 197  M 155  \n",
      " 142 0.259 0.170   L 157  M 161  T 155  T 155  T 198  T 156  \n",
      " 143 0.192 0.062 * R 158  R 162  R 156  R 156  R 199  R 157  \n",
      " 144 0.256 0.108 * L 159  L 163  L 157  L 157  L 200  L 158  \n",
      " 145 0.287 0.130 * D 160  D 164  D 158  D 158  D 201  D 159  \n",
      " 146 0.204 0.121   H 161  H 165  H 159  H 159  E 202  H 160  \n",
      " 147 0.182 0.097 * N 162  N 166  N 160  N 160  N 203  N 161  \n",
      " 148 0.264 0.103 * R 163  R 167  R 161  R 161  R 204  R 162  \n",
      " 149 0.195 0.056 * A 164  A 168  A 162  A 162  A 205  A 163  \n",
      " 150 0.169 0.095   L 165  I 169  K 163  K 163  K 206  L 164  \n",
      " 151 0.310 0.125   S 166  S 170  A 164  A 164  C 207  S 165  \n",
      " 152 0.355 0.108   Q 167  Q 171  Q 165  Q 165  Q 208  L 166  \n",
      " 153 0.308 0.098   L 168  L 172  I 166  L 166  L 209  L 167  \n",
      " 154 0.422 0.166 * A 169  A 173  A 167  A 167  A 210  A 168  \n",
      " 155 0.520 0.152   A 170  T 174  L 168  K 168  L 211  R 169  \n",
      " 156 0.511 0.158 * K 171  K 175  K 169  K 169  K 212  K 170  \n",
      " 157 0.641 0.284   S 172  L 176  L 170  T 170  A 213  A 171  \n",
      " 158 0.787 0.343 * G 173  G 177  G 171  G 171  G 214  G 172  \n",
      " 159 0.783 0.293   K 174  R 178  V 172  T 172  V 215  V 173  \n",
      " 160 0.843 0.226   P 175  G 179  T 173  G 173  F 216  P 174  \n",
      " 161 0.650 0.127   V 176  S 180  S 174  V 174  Y 217  V 175  \n",
      " 162 0.669 0.183   A 177  A 181  D 175  D 175  D 218  S 176  \n",
      " 163 0.629 0.237   S 178  E 182  D 176  R 176  K 219  Q 177  \n",
      " 164 0.683 0.258   I 179  F 183  V 177  I 177  V 220  V 178  \n",
      " 165 0.992 0.390   E 180  N 184  K 178  R 178  S 221  R 179  \n",
      " 166 0.780 0.370   K 181  N 185  N 179  R 179  N 222  N 180  \n",
      " 167 0.740 0.234   L 182  I 186  V 180  M 180  V 223  V 181  \n",
      " 168 0.487 0.245   A 183  V 187  I 181  T 181  T 224  I 182  \n",
      " 169 0.418 0.227   V 184  V 188  I 182  V 182  I 225  I 183  \n",
      " 170 0.223 0.084 * W 185  W 189  W 183  W 183  W 226  W 184  \n",
      " 171 0.337 0.077 * G 186  G 190  G 184  G 184  G 227  G 185  \n",
      " 172 0.569 0.391 * N 187  N 191  N 185  N 185  N 228  N 186  \n",
      " 173 0.511 0.255 * H 188  H 192  H 186  H 186  H 229  H 187  \n",
      " 174 0.448 0.132 * S 189  S 193  S 187  S 187  S 230  S 188  \n",
      " 175 0.569 0.219   P 190  A 194  S 188  S 188  T 231  S 189  \n",
      " 176 0.602 0.224   T 191  T 195  T 189  I 189  T 232  T 190  \n",
      " 177 0.379 0.083   M 192  Q 196  Q 190  M 190  Q 233  Q 191  \n",
      " 178 0.365 0.105   Y 193  F 197  Y 191  F 191  V 234  V 192  \n",
      " 179 0.321 0.221 * P 194  P 198  P 192  P 192  P 235  P 193  \n",
      " 180 0.357 0.203 * D 195  D 199  D 193  D 193  D 236  D 194  \n",
      " 181 0.682 0.228   F 196  I 200  V 194  L 194  F 237  T 195  \n",
      " 182 0.569 0.269   R 197  T 201  N 195  F 195  L 238  D 196  \n",
      " 183 0.682 0.392   F 198  Y 202  H 196  H 196  N 239  S 197  \n",
      " 184 0.671 0.382 * A 199  A 203  A 197  A 197  A 240  A 198  \n",
      " 185 0.681 0.380   T 200  T 204  K 198  E 198  K 241  V 199  \n",
      " 186 0.759 0.217   A 201  V 205  V 199  V 199  I 242  I 200  \n",
      " 187 1.290 0.449   E 202  G 206  K 200  D 200  D 243  G 201  \n",
      "# ===========================================================\n",
      " 188 1.506 0.397   G 203  G 207  E 205  G 205  G 244  T 202  \n",
      " 189 0.921 0.442   E 204  E 208  V 206  R 206  R 245  T 203  \n",
      " 190 0.724 0.297   S 205  K 209  G 207  P 207  P 246  P 204  \n",
      " 191 0.614 0.275   L 206  V 210  V 208  A 208  V 247  A 205  \n",
      " 192 0.764 0.460   L 207  T 211  Y 209  L 209  K 248  R 206  \n",
      " 193 0.855 0.499   K 208  D 212  E 210  E 210  E 249  E 207  \n",
      " 194 1.027 0.505   L 209  L 213  A 211  L 211  V 250  A 208  \n",
      " 195 1.144 0.626   I 210  V 214  V 212  V 212  I 251  I 209  \n",
      "# ===========================================================\n",
      " 196 0.600 0.379   T 219  E 222  E 221  V 221  E 260  D 217  \n",
      " 197 0.377 0.192 * F 220  F 223  F 222  F 222  F 261  F 218  \n",
      " 198 0.355 0.240   I 221  I 224  I 223  I 223  T 262  V 219  \n",
      " 199 0.511 0.196   P 222  P 225  T 224  P 224  I 263  Q 220  \n",
      " 200 0.500 0.224   T 223  R 226  T 225  T 225  T 264  V 221  \n",
      " 201 0.439 0.254 * V 224  V 227  V 226  V 226  V 265  V 222  \n",
      " 202 0.698 0.306   G 225  A 228  Q 227  A 227  Q 266  R 223  \n",
      " 203 0.845 0.477   K 226  N 229  Q 228  Q 228  K 267  G 224  \n",
      " 204 0.712 0.450 * R 227  R 230  R 229  R 229  R 268  R 225  \n",
      " 205 0.818 0.555 * G 228  G 231  G 230  G 230  G 269  G 226  \n",
      " 206 1.050 0.719   A 229  A 232  A 231  A 231  G 270  A 227  \n",
      " 207 0.973 0.736   A 230  E 233  A 232  A 232  A 271  E 228  \n",
      " 208 0.859 0.609   I 231  I 234  V 233  I 233  L 272  I 229  \n",
      " 209 0.995 0.757 * I 232  I 235  I 234  I 234  I 273  I 230  \n",
      "# ===========================================================\n",
      " 210 0.846 0.401   E 233  E 236  A 236  Q 235  Q 274  Q 231  \n",
      "# ===========================================================\n",
      " 211 1.178 0.785   R 235  R 238  R 237  R 237  W 276  R 233  \n",
      " 212 1.186 0.977   G 236  G 239  K 238  G 238  G 277  G 234  \n",
      " 213 0.843 0.651   L 237  K 240  L 239  A 239  R 278  L 235  \n",
      " 214 0.956 0.600 * S 238  S 241  S 240  S 240  S 279  S 236  \n",
      " 215 0.695 0.471 * S 239  S 242  S 241  S 241  S 280  S 237  \n",
      " 216 0.403 0.191 * A 240  A 243  A 242  A 242  A 281  A 238  \n",
      " 217 0.366 0.210   A 241  A 244  M 243  A 243  A 282  M 239  \n",
      " 218 0.389 0.261 * S 242  S 245  S 244  S 244  S 283  S 240  \n",
      " 219 0.368 0.212   A 243  A 246  A 245  A 245  T 284  A 241  \n",
      " 220 0.308 0.197 * A 244  A 247  A 246  A 246  A 285  A 242  \n",
      " 221 0.281 0.157   N 245  S 248  K 247  N 247  V 286  K 243  \n",
      " 222 0.286 0.155   A 246  S 249  A 248  A 248  S 287  A 244  \n",
      " 223 0.332 0.146   A 247  A 250  I 249  A 249  I 288  A 245  \n",
      " 224 0.390 0.080   I 248  I 251  C 250  I 250  A 289  V 246  \n",
      " 225 0.306 0.078   D 249  D 252  D 251  E 251  D 290  D 247  \n",
      " 226 0.291 0.094   H 250  H 253  H 252  H 252  A 291  H 248  \n",
      " 227 0.401 0.156   V 251  M 254  V 253  I 253  I 292  V 249  \n",
      " 228 0.452 0.272   R 252  R 255  R 254  R 254  K 293  H 250  \n",
      " 229 0.368 0.218   D 253  D 256  D 255  D 255  S 294  D 251  \n",
      " 230 0.476 0.134   W 254  W 257  I 256  W 256  L 295  W 252  \n",
      " 231 0.718 0.375   V 255  V 258  W 257  A 257  V 296  I 253  \n",
      " 232 0.652 0.524   L 256  Q 259  F 258  L 258  T 297  H 254  \n",
      " 233 0.871 0.609   G 257  G 260  G 259  G 259  P 298  G 255  \n",
      " 234 0.556 0.329 * T 258  T 261  T 260  T 260  T 299  T 256  \n",
      "# ===========================================================\n",
      " 235 1.611 0.548   G 260  E 262  E 262  E 262  E 301  E 258  \n",
      "# ===========================================================\n",
      " 236 0.931 0.474   K 261  A 263  E 264  D 264  D 303  V 260  \n",
      " 237 0.503 0.263   W 262  W 264  F 265  W 265  W 304  Y 261  \n",
      " 238 0.400 0.187   V 263  S 265  V 266  V 266  F 305  V 262  \n",
      " 239 0.211 0.065   T 264  S 266  S 267  S 267  S 306  S 263  \n",
      " 240 0.350 0.209   M 265  A 267  M 268  M 268  T 307  M 264  \n",
      " 241 0.272 0.146   G 266  A 268  G 269  A 269  G 308  G 265  \n",
      " 242 0.279 0.137   I 267  I 269  I 270  V 270  V 309  V 266  \n",
      " 243 0.281 0.108   P 268  P 270  I 271  P 271  Y 310  Y 267  \n",
      " 244 0.286 0.086   S 269  S 271  S 272  S 272  T 311  S 268  \n",
      " 245 1.000 0.348   D 270  T 272  D 273  Q 273  T 312  D 269  \n",
      "# ===========================================================\n",
      " 246 1.181 0.516   S 272  A 274  N 275  E 275  N 314  N 271  \n",
      "# ===========================================================\n",
      " 247 0.620 0.345 * Y 273  Y 275  Y 277  Y 277  Y 316  Y 273  \n",
      " 248 1.050 0.499 * G 274  G 276  G 278  G 278  G 317  G 274  \n",
      " 249 0.548 0.268   I 275  I 277  V 279  I 279  I 318  V 275  \n",
      " 250 0.535 0.293   P 276  P 278  P 280  P 280  A 319  P 276  \n",
      " 251 0.522 0.257   E 277  E 279  D 281  E 281  E 320  S 277  \n",
      " 252 0.842 0.403   D 278  G 280  D 282  G 282  D 321  G 278  \n",
      " 253 0.601 0.227   I 279  I 281  L 283  I 283  I 322  L 279  \n",
      " 254 0.307 0.129   I 280  F 282  L 284  V 284  V 323  I 280  \n",
      " 255 0.292 0.169   Y 281  V 283  Y 285  Y 285  F 324  F 281  \n",
      " 256 0.296 0.186   G 282  G 284  S 286  S 286  S 325  S 282  \n",
      " 257 0.256 0.098   V 283  L 285  F 287  F 287  M 326  F 283  \n",
      " 258 0.364 0.267 * P 284  P 286  P 288  P 288  P 327  P 284  \n",
      " 259 0.386 0.112   V 285  T 287  V 289  V 289  C 328  C 285  \n",
      " 260 0.437 0.228   I 286  V 288  T 290  T 290  R 329  T 286  \n",
      " 261 0.505 0.203   C 287  S 289  I 291  A 291  S 330  C 287  \n",
      " 262 0.768 0.106   E 288  R 290  K 292  K 292  K 331  H 288  \n",
      "# ===========================================================\n",
      " 263 1.210 0.426   N 289  N 291  D 293  D 293  D 333  A 289  \n",
      " 264 0.766 0.488   G 290  G 292  K 294  G 294  G 334  G 290  \n",
      " 265 0.789 0.354   E 291  E 293  T 295  A 295  D 335  E 291  \n",
      " 266 0.642 0.189   Y 292  W 294  W 296  Y 296  Y 336  W 292  \n",
      " 267 0.558 0.167   K 293  E 295  K 297  R 297  E 337  T 293  \n",
      " 268 0.419 0.151   R 294  I 296  I 298  V 298  L 338  V 294  \n",
      " 269 0.415 0.139   V 295  V 297  V 299  V 299  A 339  V 295  \n",
      " 270 0.912 0.357   E 296  E 298  E 300  E 300  T 340  S 296  \n",
      "# ===========================================================\n",
      " 271 1.030 0.570   L 298  L 300  L 302  L 302  V 342  G 297  \n",
      "# ===========================================================\n",
      " 272 0.960 0.454   I 300  I 302  I 304  I 304  N 344  L 299  \n",
      "# ===========================================================\n",
      " 273 0.995 0.573   R 305  R 307  R 309  R 309  W 349  K 305  \n",
      " 274 0.911 0.723   E 306  A 308  E 310  K 310  E 350  Q 306  \n",
      " 275 0.847 0.721   K 307  R 309  K 311  R 311  R 351  R 307  \n",
      " 276 0.547 0.250   M 308  I 310  M 312  M 312  I 352  L 308  \n",
      " 277 0.624 0.290   D 309  D 311  D 313  E 313  K 353  A 309  \n",
      " 278 0.526 0.231   G 310  A 312  L 314  I 314  K 354  S 310  \n",
      " 279 0.501 0.248   T 311  N 313  T 315  T 315  S 355  T 311  \n",
      " 280 0.506 0.266   L 312  A 314  A 316  A 316  E 356  I 312  \n",
      " 281 0.578 0.254   A 313  Q 315  K 317  Q 317  A 357  A 313  \n",
      " 282 0.576 0.252 * E 314  E 316  E 318  E 318  E 358  E 314  \n",
      " 283 0.391 0.177 * L 315  L 317  L 319  L 319  L 359  L 315  \n",
      " 284 0.432 0.205   L 316  Q 318  A 320  L 320  L 360  Q 316  \n",
      " 285 0.592 0.226   E 317  A 319  E 321  D 321  A 361  E 317  \n",
      " 286 0.550 0.223 * E 318  E 320  E 322  E 322  E 362  E 318  \n",
      " 287 0.481 0.170   R 319  R 321  K 323  M 323  K 363  R 319  \n",
      " 288 0.607 0.251   D 320  E 322  E 324  E 324  K 364  A 320  \n",
      " 289 0.678 0.316   G 321  A 323  T 325  Q 325  C 365  Q 321  \n",
      " 290 0.802 0.304   V 322  V 324  A 326  V 326  V 366  A 322  \n",
      " 291 0.905 0.310   A 323  R 325  F 327  K 327  A 367  G 323  \n",
      "# ===========================================================\n",
      "malign3_330_> Framework, Cycles, RMS_frw(i-1,i):        3        2        0.0000\n",
      "malign3_332_> Framework:        3\n",
      "# Sequence alignment of the structurally conserved regions\n",
      "# [average distance and standard deviation are with respect\n",
      "#  to the framework (i.e., average structure)]\n",
      "#\n",
      "#  N av ds st dv   3d5tA  6itkA  5mdhA  1bdmA  7mdhA  4h7pA  \n",
      "# ===========================================================\n",
      "   1 1.132 0.571   P   4  S   8  E   2  A   2  K  45  S   2  \n",
      "   2 0.592 0.256   A   5  T   9  P   3  P   3  L  46  A   3  \n",
      "   3 0.283 0.134   K   6  K  10  I   4  V   4  V  47  V   4  \n",
      "   4 0.245 0.113   R   7  K  11  R   5  R   5  N  48  K   5  \n",
      "   5 0.196 0.028   V   8  V  12  V   6  V   6  I  49  V   6  \n",
      "   6 0.280 0.075   A   9  T  13  L   7  A   7  A  50  A   7  \n",
      "   7 0.341 0.057 * V  10  V  14  V   8  V   8  V  51  V   8  \n",
      "   8 0.253 0.114   T  11  T  15  T   9  T   9  S  52  T   9  \n",
      "   9 0.332 0.147 * G  12  G  16  G  10  G  10  G  53  G  10  \n",
      "  10 0.380 0.088 * A  13  A  17  A  11  A  11  A  54  A  11  \n",
      "  11 0.366 0.066 * A  14  A  18  A  12  A  12  A  55  A  12  \n",
      "  12 0.279 0.124 * G  15  G  19  G  13  G  13  G  56  G  13  \n",
      "  13 0.293 0.128   Q  16  Q  20  Q  14  Q  14  M  57  Q  14  \n",
      "  14 0.258 0.171 * I  17  I  21  I  15  I  15  I  58  I  15  \n",
      "  15 0.285 0.128   A  18  S  22  A  16  G  16  S  59  G  16  \n",
      "  16 0.219 0.099   Y  19  Y  23  Y  17  Y  17  N  60  Y  17  \n",
      "  17 0.373 0.255   S  20  S  24  S  18  S  18  H  61  A  18  \n",
      "  18 0.329 0.111 * L  21  L  25  L  19  L  19  L  62  L  19  \n",
      "  19 0.269 0.115   L  22  L  26  L  20  L  20  L  63  V  20  \n",
      "  20 0.390 0.130   F  23  W  27  Y  21  F  21  F  64  P  21  \n",
      "  21 0.413 0.124   R  24  R  28  S  22  R  22  K  65  L  22  \n",
      "  22 0.342 0.124   I  25  I  29  I  23  I  23  L  66  I  23  \n",
      "  23 0.262 0.069   A  26  A  30  G  24  A  24  A  67  A  24  \n",
      "  24 0.370 0.147   N  27  N  31  N  25  A  25  S  68  R  25  \n",
      "  25 0.438 0.127 * G  28  G  32  G  26  G  26  G  69  G  26  \n",
      "  26 0.845 0.246   D  29  E  33  S  27  E  27  E  70  A  27  \n",
      "  27 0.845 0.260   L  30  V  34  V  28  M  28  V  71  L  28  \n",
      "  28 0.624 0.265   L  31  F  35  F  29  L  29  F  72  L  29  \n",
      "  29 0.648 0.214 * G  32  G  36  G  30  G  30  G  73  G  30  \n",
      "  30 0.544 0.202   K  33  T  37  K  31  K  31  Q  74  P  31  \n",
      "  31 0.586 0.125   D  34  D  38  D  32  D  32  D  75  T  32  \n",
      "  32 0.375 0.143   Q  35  T  39  Q  33  Q  33  Q  76  T  33  \n",
      "  33 0.396 0.128 * P  36  P  40  P  34  P  34  P  77  P  34  \n",
      "  34 0.301 0.072   V  37  V  41  I  35  V  35  I  78  V  35  \n",
      "  35 0.385 0.201   I  38  E  42  I  36  I  36  A  79  E  36  \n",
      "  36 0.250 0.112 * L  39  L  43  L  37  L  37  L  80  L  37  \n",
      "  37 0.254 0.054   Q  40  K  44  V  38  Q  38  K  81  R  38  \n",
      "  38 0.365 0.160 * L  41  L  45  L  39  L  39  L  82  L  39  \n",
      "  39 0.327 0.137 * L  42  L  46  L  40  L  40  L  83  L  40  \n",
      "  40 0.422 0.142   D  43  E  47  D  41  E  41  G  84  D  41  \n",
      "  41 0.558 0.201   L  44  I  48  I  42  I  42  S  85  I  42  \n",
      "  42 0.608 0.322   P  45  P  49  T  43  P  43  E  86  E  43  \n",
      "  43 0.836 0.564   Q  46  Q  50  P  44  Q  44  R  87  P  44  \n",
      "  44 0.544 0.253   A  47  A  51  M  45  A  45  S  88  A  45  \n",
      "  45 0.563 0.287   Q  48  L  52  M  46  M  46  F  89  L  46  \n",
      "  46 0.592 0.278   A  49  G  53  G  47  K  47  Q  90  K  47  \n",
      "  47 0.516 0.244   A  50  G  54  V  48  A  48  A  91  A  48  \n",
      "  48 0.495 0.349   V  51  A  55  L  49  L  49  L  92  L  49  \n",
      "  49 0.548 0.394   K  52  E  56  D  50  E  50  E  93  A  50  \n",
      "  50 0.489 0.330 * G  53  G  57  G  51  G  51  G  94  G  51  \n",
      "  51 0.416 0.228 * V  54  V  58  V  52  V  52  V  95  V  52  \n",
      "  52 0.562 0.329   V  55  A  59  L  53  V  53  A  96  E  53  \n",
      "  53 0.597 0.435   M  56  M  60  M  54  M  54  M  97  A  54  \n",
      "  54 0.425 0.277 * E  57  E  61  E  55  E  55  E  98  E  55  \n",
      "  55 0.452 0.245 * L  58  L  62  L  56  L  56  L  99  L  56  \n",
      "  56 0.675 0.460   D  59  L  63  Q  57  E  57  E 100  E  57  \n",
      "  57 0.670 0.506 * D  60  D  64  D  58  D  58  D 101  D  58  \n",
      "  58 0.555 0.454   C  61  S  65  C  59  C  59  S 102  C  59  \n",
      "  59 0.750 0.625   A  62  A  66  A  60  A  60  L 103  A  60  \n",
      "  60 0.309 0.071   F  63  F  67  L  61  F  61  Y 104  F  61  \n",
      "  61 0.557 0.223 * P  64  P  68  P  62  P  62  P 105  P  62  \n",
      "  62 0.266 0.102 * L  65  L  69  L  63  L  63  L 106  L  63  \n",
      "  63 0.258 0.096 * L  66  L  70  L  64  L  64  L 107  L  64  \n",
      "  64 0.349 0.183   A  67  R  71  K  65  A  65  R 108  D  65  \n",
      "  65 0.411 0.110   G  68  N  72  D  66  G  66  E 109  K  66  \n",
      "  66 0.341 0.053   V  69  I  73  V  67  L  67  V 110  V  67  \n",
      "  67 0.341 0.135   V  70  T  74  I  68  E  68  S 111  V  68  \n",
      "  68 0.472 0.151   I  71  I  75  A  69  A  69  I 112  V  69  \n",
      "  69 0.449 0.123   T  72  T  76  T  70  T  70  G 113  T  70  \n",
      "  70 0.455 0.175   D  73  A  77  D  71  D  71  I 114  A  71  \n",
      "  71 0.540 0.288   D  74  D  78  K  72  D  72  D 115  D  72  \n",
      "  72 0.549 0.413   P  75  A  79  E  73  P  73  P 116  P  73  \n",
      "  73 0.704 0.478   K  76  N  80  E  74  D  74  Y 117  R  74  \n",
      "  74 0.515 0.310   V  77  E  81  I  75  V  75  E 118  V  75  \n",
      "  75 0.400 0.197   A  78  A  82  A  76  A  76  V 119  A  76  \n",
      "  76 0.374 0.263 * F  79  F  83  F  77  F  77  F 120  F  77  \n",
      "  77 0.345 0.137   K  80  D  84  K  78  K  78  E 121  D  78  \n",
      "  78 0.315 0.216   D  81  G  85  D  79  D  79  D 122  G  79  \n",
      "  79 0.290 0.125   A  82  A  86  L  80  A  80  V 123  V  80  \n",
      "  80 0.248 0.092   D  83  N  87  D  81  D  81  D 124  A  81  \n",
      "  81 0.242 0.083   V  84  A  88  V  82  Y  82  W 125  I  82  \n",
      "  82 0.327 0.174 * A  85  A  89  A  83  A  83  A 126  A  83  \n",
      "  83 0.312 0.132   L  86  F  90  I  84  L  84  L 127  I  84  \n",
      "  84 0.249 0.167   L  87  L  91  L  85  L  85  L 128  M  85  \n",
      "  85 0.278 0.099   V  88  V  92  V  86  V  86  I 129  C  86  \n",
      "  86 0.312 0.089 * G  89  G  93  G  87  G  87  G 130  G  87  \n",
      "  87 0.494 0.244   A  90  A  94  S  88  A  88  A 131  A  88  \n",
      "# ===========================================================\n",
      "  88 1.115 0.585   P  92  P  96  P  90  R  91  P 133  P  90  \n",
      "# ===========================================================\n",
      "  89 0.852 0.432   S 104  A 108  K 102  L 101  D 145  E 102  \n",
      "# ===========================================================\n",
      "  90 1.112 0.350   A 105  N 109  A 103  V 103  I 146  M 103  \n",
      "  91 0.735 0.303 * N 106  N 110  N 104  N 104  N 147  N 104  \n",
      "  92 1.038 0.343   A 107  G 111  V 105  G 105  G 148  A 105  \n",
      "  93 0.873 0.259   E 108  K 112  K 106  K 106  Q 149  R 106  \n",
      "  94 0.411 0.088 * I 109  I 113  I 107  I 107  I 150  I 107  \n",
      "  95 0.399 0.156 * F 110  F 114  F 108  F 108  F 151  F 108  \n",
      "  96 0.498 0.194   T 111  G 115  K 109  T 109  A 152  K 109  \n",
      "  97 0.432 0.141   V 112  P 116  C 110  E 110  D 153  E 110  \n",
      "  98 0.284 0.048 * Q 113  Q 117  Q 111  Q 111  Q 154  Q 111  \n",
      "  99 0.279 0.089 * G 114  G 118  G 112  G 112  G 155  G 112  \n",
      " 100 0.335 0.136   A 115  K 119  A 113  R 113  K 156  E 113  \n",
      " 101 0.328 0.115 * A 116  A 120  A 114  A 114  A 157  A 114  \n",
      " 102 0.275 0.090   L 117  I 121  L 115  L 115  L 158  I 115  \n",
      " 103 0.340 0.137   N 118  N 122  D 116  A 116  N 159  A 116  \n",
      " 104 0.398 0.167   E 119  D 123  K 117  E 117  A 160  A 117  \n",
      " 105 0.394 0.176   V 120  N 124  Y 118  V 118  V 161  V 118  \n",
      " 106 0.340 0.115 * A 121  A 125  A 119  A 119  A 162  A 119  \n",
      " 107 0.533 0.288   S 122  A 126  K 120  K 120  S 163  A 120  \n",
      " 108 0.745 0.536   R 123  D 127  K 121  K 121  K 164  S 121  \n",
      " 109 0.661 0.490   D 124  D 128  S 122  D 122  N 165  D 122  \n",
      " 110 0.252 0.108   V 125  I 129  V 123  V 123  V 166  C 123  \n",
      " 111 0.245 0.075   K 126  R 130  K 124  K 124  K 167  R 124  \n",
      " 112 0.254 0.108 * V 127  V 131  V 125  V 125  V 168  V 125  \n",
      " 113 0.230 0.155   L 128  L 132  I 126  L 126  L 169  V 126  \n",
      " 114 0.181 0.050 * V 129  V 133  V 127  V 127  V 170  V 127  \n",
      " 115 0.293 0.095 * V 130  V 134  V 128  V 128  V 171  V 128  \n",
      " 116 0.352 0.222 * G 131  G 135  G 129  G 129  G 172  G 129  \n",
      " 117 0.340 0.156 * N 132  N 136  N 130  N 130  N 173  N 130  \n",
      " 118 0.328 0.143 * P 133  P 137  P 131  P 131  P 174  P 131  \n",
      " 119 0.293 0.089   A 134  A 138  A 132  A 132  C 175  A 132  \n",
      " 120 0.237 0.085 * N 135  N 139  N 133  N 133  N 176  N 133  \n",
      " 121 0.235 0.065 * T 136  T 140  T 134  T 134  T 177  T 134  \n",
      " 122 0.273 0.081 * N 137  N 141  N 135  N 135  N 178  N 135  \n",
      " 123 0.282 0.096   A 138  A 142  C 136  A 136  A 179  A 136  \n",
      " 124 0.290 0.115   Y 139  L 143  L 137  L 137  L 180  L 137  \n",
      " 125 0.343 0.128   I 140  I 144  T 138  I 138  I 181  I 138  \n",
      " 126 0.286 0.098   A 141  A 145  A 139  A 139  C 182  L 139  \n",
      " 127 0.309 0.057   M 142  S 146  S 140  Y 140  L 183  L 140  \n",
      " 128 0.304 0.071   K 143  A 147  K 141  K 141  K 184  K 141  \n",
      " 129 0.419 0.192   S 144  A 148  S 142  N 142  N 185  S 142  \n",
      " 130 0.493 0.253 * A 145  A 149  A 143  A 143  A 186  A 143  \n",
      "# ===========================================================\n",
      " 131 0.827 0.622   P 146  P 150  P 144  P 144  P 187  G 145  \n",
      " 132 0.374 0.246   D 147  D 151  S 145  G 145  D 188  K 146  \n",
      " 133 0.471 0.329   L 148  V 152  I 146  L 146  I 189  L 147  \n",
      " 134 0.400 0.119   P 149  P 153  P 147  N 147  P 190  N 148  \n",
      " 135 0.479 0.202   K 150  A 154  K 148  P 148  A 191  P 149  \n",
      " 136 0.474 0.185   K 151  S 155  E 149  R 149  K 192  R 150  \n",
      " 137 0.345 0.113   N 152  R 156  N 150  N 150  N 193  H 151  \n",
      " 138 0.325 0.107   F 153  F 157  F 151  F 151  F 194  V 152  \n",
      " 139 0.318 0.118   T 154  N 158  S 152  T 152  H 195  T 153  \n",
      " 140 0.285 0.190   A 155  A 159  C 153  A 153  A 196  A 154  \n",
      " 141 0.166 0.056   M 156  M 160  L 154  M 154  L 197  M 155  \n",
      " 142 0.260 0.171   L 157  M 161  T 155  T 155  T 198  T 156  \n",
      " 143 0.193 0.062 * R 158  R 162  R 156  R 156  R 199  R 157  \n",
      " 144 0.256 0.109 * L 159  L 163  L 157  L 157  L 200  L 158  \n",
      " 145 0.288 0.131 * D 160  D 164  D 158  D 158  D 201  D 159  \n",
      " 146 0.204 0.122   H 161  H 165  H 159  H 159  E 202  H 160  \n",
      " 147 0.183 0.098 * N 162  N 166  N 160  N 160  N 203  N 161  \n",
      " 148 0.265 0.105 * R 163  R 167  R 161  R 161  R 204  R 162  \n",
      " 149 0.194 0.056 * A 164  A 168  A 162  A 162  A 205  A 163  \n",
      " 150 0.167 0.095   L 165  I 169  K 163  K 163  K 206  L 164  \n",
      " 151 0.310 0.126   S 166  S 170  A 164  A 164  C 207  S 165  \n",
      " 152 0.354 0.106   Q 167  Q 171  Q 165  Q 165  Q 208  L 166  \n",
      " 153 0.305 0.096   L 168  L 172  I 166  L 166  L 209  L 167  \n",
      " 154 0.421 0.166 * A 169  A 173  A 167  A 167  A 210  A 168  \n",
      " 155 0.520 0.153   A 170  T 174  L 168  K 168  L 211  R 169  \n",
      " 156 0.509 0.157 * K 171  K 175  K 169  K 169  K 212  K 170  \n",
      " 157 0.639 0.284   S 172  L 176  L 170  T 170  A 213  A 171  \n",
      " 158 0.786 0.344 * G 173  G 177  G 171  G 171  G 214  G 172  \n",
      " 159 0.782 0.295   K 174  R 178  V 172  T 172  V 215  V 173  \n",
      " 160 0.842 0.226   P 175  G 179  T 173  G 173  F 216  P 174  \n",
      " 161 0.650 0.128   V 176  S 180  S 174  V 174  Y 217  V 175  \n",
      " 162 0.667 0.184   A 177  A 181  D 175  D 175  D 218  S 176  \n",
      " 163 0.628 0.238   S 178  E 182  D 176  R 176  K 219  Q 177  \n",
      " 164 0.682 0.260   I 179  F 183  V 177  I 177  V 220  V 178  \n",
      " 165 0.991 0.392   E 180  N 184  K 178  R 178  S 221  R 179  \n",
      " 166 0.780 0.372   K 181  N 185  N 179  R 179  N 222  N 180  \n",
      " 167 0.740 0.235   L 182  I 186  V 180  M 180  V 223  V 181  \n",
      " 168 0.487 0.246   A 183  V 187  I 181  T 181  T 224  I 182  \n",
      " 169 0.418 0.228   V 184  V 188  I 182  V 182  I 225  I 183  \n",
      " 170 0.223 0.084 * W 185  W 189  W 183  W 183  W 226  W 184  \n",
      " 171 0.337 0.079 * G 186  G 190  G 184  G 184  G 227  G 185  \n",
      " 172 0.568 0.390 * N 187  N 191  N 185  N 185  N 228  N 186  \n",
      " 173 0.511 0.254 * H 188  H 192  H 186  H 186  H 229  H 187  \n",
      " 174 0.449 0.133 * S 189  S 193  S 187  S 187  S 230  S 188  \n",
      " 175 0.570 0.219   P 190  A 194  S 188  S 188  T 231  S 189  \n",
      " 176 0.600 0.223   T 191  T 195  T 189  I 189  T 232  T 190  \n",
      " 177 0.378 0.081   M 192  Q 196  Q 190  M 190  Q 233  Q 191  \n",
      " 178 0.365 0.104   Y 193  F 197  Y 191  F 191  V 234  V 192  \n",
      " 179 0.320 0.221 * P 194  P 198  P 192  P 192  P 235  P 193  \n",
      " 180 0.356 0.201 * D 195  D 199  D 193  D 193  D 236  D 194  \n",
      " 181 0.681 0.228   F 196  I 200  V 194  L 194  F 237  T 195  \n",
      " 182 0.569 0.269   R 197  T 201  N 195  F 195  L 238  D 196  \n",
      " 183 0.682 0.390   F 198  Y 202  H 196  H 196  N 239  S 197  \n",
      " 184 0.670 0.381 * A 199  A 203  A 197  A 197  A 240  A 198  \n",
      " 185 0.681 0.378   T 200  T 204  K 198  E 198  K 241  V 199  \n",
      " 186 0.761 0.214   A 201  V 205  V 199  V 199  I 242  I 200  \n",
      " 187 1.292 0.448   E 202  G 206  K 200  D 200  D 243  G 201  \n",
      "# ===========================================================\n",
      " 188 1.507 0.396   G 203  G 207  E 205  G 205  G 244  T 202  \n",
      " 189 0.924 0.440   E 204  E 208  V 206  R 206  R 245  T 203  \n",
      " 190 0.726 0.297   S 205  K 209  G 207  P 207  P 246  P 204  \n",
      " 191 0.615 0.274   L 206  V 210  V 208  A 208  V 247  A 205  \n",
      " 192 0.767 0.458   L 207  T 211  Y 209  L 209  K 248  R 206  \n",
      " 193 0.858 0.497   K 208  D 212  E 210  E 210  E 249  E 207  \n",
      " 194 1.030 0.504   L 209  L 213  A 211  L 211  V 250  A 208  \n",
      " 195 1.146 0.625   I 210  V 214  V 212  V 212  I 251  I 209  \n",
      "# ===========================================================\n",
      " 196 0.600 0.375   T 219  E 222  E 221  V 221  E 260  D 217  \n",
      " 197 0.376 0.189 * F 220  F 223  F 222  F 222  F 261  F 218  \n",
      " 198 0.354 0.239   I 221  I 224  I 223  I 223  T 262  V 219  \n",
      " 199 0.513 0.194   P 222  P 225  T 224  P 224  I 263  Q 220  \n",
      " 200 0.501 0.226   T 223  R 226  T 225  T 225  T 264  V 221  \n",
      " 201 0.440 0.256 * V 224  V 227  V 226  V 226  V 265  V 222  \n",
      " 202 0.699 0.308   G 225  A 228  Q 227  A 227  Q 266  R 223  \n",
      " 203 0.847 0.482   K 226  N 229  Q 228  Q 228  K 267  G 224  \n",
      " 204 0.714 0.454 * R 227  R 230  R 229  R 229  R 268  R 225  \n",
      " 205 0.820 0.560 * G 228  G 231  G 230  G 230  G 269  G 226  \n",
      " 206 1.054 0.724   A 229  A 232  A 231  A 231  G 270  A 227  \n",
      " 207 0.976 0.742   A 230  E 233  A 232  A 232  A 271  E 228  \n",
      " 208 0.863 0.613   I 231  I 234  V 233  I 233  L 272  I 229  \n",
      " 209 1.000 0.762 * I 232  I 235  I 234  I 234  I 273  I 230  \n",
      "# ===========================================================\n",
      " 210 0.848 0.406   E 233  E 236  A 236  Q 235  Q 274  Q 231  \n",
      "# ===========================================================\n",
      " 211 1.183 0.790   R 235  R 238  R 237  R 237  W 276  R 233  \n",
      " 212 1.192 0.982   G 236  G 239  K 238  G 238  G 277  G 234  \n",
      " 213 0.847 0.655   L 237  K 240  L 239  A 239  R 278  L 235  \n",
      " 214 0.959 0.602 * S 238  S 241  S 240  S 240  S 279  S 236  \n",
      " 215 0.698 0.473 * S 239  S 242  S 241  S 241  S 280  S 237  \n",
      " 216 0.404 0.193 * A 240  A 243  A 242  A 242  A 281  A 238  \n",
      " 217 0.368 0.211   A 241  A 244  M 243  A 243  A 282  M 239  \n",
      " 218 0.390 0.263 * S 242  S 245  S 244  S 244  S 283  S 240  \n",
      " 219 0.370 0.214   A 243  A 246  A 245  A 245  T 284  A 241  \n",
      " 220 0.309 0.199 * A 244  A 247  A 246  A 246  A 285  A 242  \n",
      " 221 0.282 0.158   N 245  S 248  K 247  N 247  V 286  K 243  \n",
      " 222 0.286 0.156   A 246  S 249  A 248  A 248  S 287  A 244  \n",
      " 223 0.333 0.147   A 247  A 250  I 249  A 249  I 288  A 245  \n",
      " 224 0.390 0.080   I 248  I 251  C 250  I 250  A 289  V 246  \n",
      " 225 0.307 0.079   D 249  D 252  D 251  E 251  D 290  D 247  \n",
      " 226 0.291 0.095   H 250  H 253  H 252  H 252  A 291  H 248  \n",
      " 227 0.401 0.155   V 251  M 254  V 253  I 253  I 292  V 249  \n",
      " 228 0.452 0.271   R 252  R 255  R 254  R 254  K 293  H 250  \n",
      " 229 0.368 0.219   D 253  D 256  D 255  D 255  S 294  D 251  \n",
      " 230 0.475 0.134   W 254  W 257  I 256  W 256  L 295  W 252  \n",
      " 231 0.718 0.374   V 255  V 258  W 257  A 257  V 296  I 253  \n",
      " 232 0.652 0.523   L 256  Q 259  F 258  L 258  T 297  H 254  \n",
      " 233 0.870 0.608   G 257  G 260  G 259  G 259  P 298  G 255  \n",
      " 234 0.556 0.330 * T 258  T 261  T 260  T 260  T 299  T 256  \n",
      "# ===========================================================\n",
      " 235 1.610 0.548   G 260  E 262  E 262  E 262  E 301  E 258  \n",
      "# ===========================================================\n",
      " 236 0.931 0.474   K 261  A 263  E 264  D 264  D 303  V 260  \n",
      " 237 0.502 0.263   W 262  W 264  F 265  W 265  W 304  Y 261  \n",
      " 238 0.399 0.188   V 263  S 265  V 266  V 266  F 305  V 262  \n",
      " 239 0.211 0.066   T 264  S 266  S 267  S 267  S 306  S 263  \n",
      " 240 0.351 0.209   M 265  A 267  M 268  M 268  T 307  M 264  \n",
      " 241 0.273 0.146   G 266  A 268  G 269  A 269  G 308  G 265  \n",
      " 242 0.281 0.137   I 267  I 269  I 270  V 270  V 309  V 266  \n",
      " 243 0.281 0.107   P 268  P 270  I 271  P 271  Y 310  Y 267  \n",
      " 244 0.285 0.085   S 269  S 271  S 272  S 272  T 311  S 268  \n",
      " 245 1.000 0.351   D 270  T 272  D 273  Q 273  T 312  D 269  \n",
      "# ===========================================================\n",
      " 246 1.181 0.515   S 272  A 274  N 275  E 275  N 314  N 271  \n",
      "# ===========================================================\n",
      " 247 0.620 0.343 * Y 273  Y 275  Y 277  Y 277  Y 316  Y 273  \n",
      " 248 1.048 0.498 * G 274  G 276  G 278  G 278  G 317  G 274  \n",
      " 249 0.548 0.267   I 275  I 277  V 279  I 279  I 318  V 275  \n",
      " 250 0.537 0.291   P 276  P 278  P 280  P 280  A 319  P 276  \n",
      " 251 0.523 0.256   E 277  E 279  D 281  E 281  E 320  S 277  \n",
      " 252 0.839 0.403   D 278  G 280  D 282  G 282  D 321  G 278  \n",
      " 253 0.600 0.225   I 279  I 281  L 283  I 283  I 322  L 279  \n",
      " 254 0.307 0.128   I 280  F 282  L 284  V 284  V 323  I 280  \n",
      " 255 0.293 0.167   Y 281  V 283  Y 285  Y 285  F 324  F 281  \n",
      " 256 0.297 0.185   G 282  G 284  S 286  S 286  S 325  S 282  \n",
      " 257 0.256 0.100   V 283  L 285  F 287  F 287  M 326  F 283  \n",
      " 258 0.363 0.267 * P 284  P 286  P 288  P 288  P 327  P 284  \n",
      " 259 0.387 0.113   V 285  T 287  V 289  V 289  C 328  C 285  \n",
      " 260 0.437 0.228   I 286  V 288  T 290  T 290  R 329  T 286  \n",
      " 261 0.504 0.202   C 287  S 289  I 291  A 291  S 330  C 287  \n",
      " 262 0.766 0.105   E 288  R 290  K 292  K 292  K 331  H 288  \n",
      "# ===========================================================\n",
      " 263 1.209 0.427   N 289  N 291  D 293  D 293  D 333  A 289  \n",
      " 264 0.767 0.488   G 290  G 292  K 294  G 294  G 334  G 290  \n",
      " 265 0.789 0.355   E 291  E 293  T 295  A 295  D 335  E 291  \n",
      " 266 0.641 0.188   Y 292  W 294  W 296  Y 296  Y 336  W 292  \n",
      " 267 0.558 0.166   K 293  E 295  K 297  R 297  E 337  T 293  \n",
      " 268 0.419 0.150   R 294  I 296  I 298  V 298  L 338  V 294  \n",
      " 269 0.416 0.138   V 295  V 297  V 299  V 299  A 339  V 295  \n",
      " 270 0.913 0.357   E 296  E 298  E 300  E 300  T 340  S 296  \n",
      "# ===========================================================\n",
      " 271 1.030 0.570   L 298  L 300  L 302  L 302  V 342  G 297  \n",
      "# ===========================================================\n",
      " 272 0.960 0.452   I 300  I 302  I 304  I 304  N 344  L 299  \n",
      "# ===========================================================\n",
      " 273 0.996 0.573   R 305  R 307  R 309  R 309  W 349  K 305  \n",
      " 274 0.911 0.724   E 306  A 308  E 310  K 310  E 350  Q 306  \n",
      " 275 0.847 0.721   K 307  R 309  K 311  R 311  R 351  R 307  \n",
      " 276 0.547 0.251   M 308  I 310  M 312  M 312  I 352  L 308  \n",
      " 277 0.623 0.292   D 309  D 311  D 313  E 313  K 353  A 309  \n",
      " 278 0.526 0.233   G 310  A 312  L 314  I 314  K 354  S 310  \n",
      " 279 0.503 0.247   T 311  N 313  T 315  T 315  S 355  T 311  \n",
      " 280 0.507 0.266   L 312  A 314  A 316  A 316  E 356  I 312  \n",
      " 281 0.578 0.254   A 313  Q 315  K 317  Q 317  A 357  A 313  \n",
      " 282 0.575 0.253 * E 314  E 316  E 318  E 318  E 358  E 314  \n",
      " 283 0.391 0.176 * L 315  L 317  L 319  L 319  L 359  L 315  \n",
      " 284 0.429 0.206   L 316  Q 318  A 320  L 320  L 360  Q 316  \n",
      " 285 0.589 0.228   E 317  A 319  E 321  D 321  A 361  E 317  \n",
      " 286 0.548 0.225 * E 318  E 320  E 322  E 322  E 362  E 318  \n",
      " 287 0.478 0.170   R 319  R 321  K 323  M 323  K 363  R 319  \n",
      " 288 0.602 0.249   D 320  E 322  E 324  E 324  K 364  A 320  \n",
      " 289 0.672 0.316   G 321  A 323  T 325  Q 325  C 365  Q 321  \n",
      " 290 0.797 0.301   V 322  V 324  A 326  V 326  V 366  A 322  \n",
      " 291 0.901 0.309   A 323  R 325  F 327  K 327  A 367  G 323  \n",
      "# ===========================================================\n",
      "\n",
      "Least-squares dynamic programming alignment:\n",
      "     FIT_ATOMS atoms for alignment     : CA\n",
      "     Max dist from frw for equivalence :    3.5000\n",
      "     Gap introduction penalty          :    0.0000\n",
      "     Gap extension penalty             :    1.7500\n",
      "     Numb of residues in framework     :       291\n",
      "compare_343_> >> BEGINNING OF COMMAND\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5        1  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       78  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       91  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       94  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       96  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      104  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      170  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      204  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      295  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      313  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6        1  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       78  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       91  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       94  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       96  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      104  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      170  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      204  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      295  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      313  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7        1  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       78  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       91  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       94  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       96  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      104  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      170  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      204  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      295  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      313  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8        1  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       78  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       91  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       96  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      170  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      204  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       90  ALA\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       91  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       93  LEU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        2       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        2       93  LEU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        3       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        4       91  PRO\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        4       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       92  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5        1  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      103  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      110  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      211  ASP\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      214  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      225  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      252  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      282  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      294  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      295  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      306  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6        1  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      103  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      110  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      211  ASP\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      214  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      225  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      252  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      282  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      294  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      295  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      306  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7        1  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      103  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      110  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      214  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      225  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      252  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      294  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      295  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      306  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      294  LYS\n",
      "fndatmi_285W> Only      317 residues out of      318 contain atoms of type  CA\n",
      "              (This is usually caused by non-standard residues, such\n",
      "              as ligands, or by PDB files with missing atoms.)\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "COMPARISON OF SEVERAL 3D STRUCTURES:\n",
      "\n",
      "\n",
      "  #  ALGNMT CODE      ATOM FILE                          \n",
      "\n",
      "  1  3d5tA\n",
      "  2  6itkA\n",
      "  3  5mdhA\n",
      "  4  1bdmA\n",
      "  5  7mdhA\n",
      "  6  4h7pA\n",
      "\n",
      "\n",
      "\n",
      "Variability at a given position is calculated as: \n",
      "VAR = 1/Nij * sum_ij (feat_i - feat_j)\n",
      "sum runs over all pairs of proteins with residues present.\n",
      "\n",
      "\n",
      "\n",
      ">> Least-squares superposition (FIT)           :       T\n",
      "\n",
      "   Atom types for superposition/RMS (FIT_ATOMS): CA\n",
      "   Atom type for position average/variability (VARATOM): CA\n",
      "\n",
      "\n",
      "   Position comparison (FIT_ATOMS): \n",
      "\n",
      "       Cutoff for RMS calculation:     3.5000\n",
      "\n",
      "       Upper = RMS, Lower = numb equiv positions\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000   0.859   1.136   0.810   1.165   1.027\n",
      "6itkA        314   0.000   0.991   0.887   1.170   0.950\n",
      "5mdhA        312     308   0.000   1.099   1.182   1.030\n",
      "1bdmA        309     309     308   0.000   0.917   0.913\n",
      "7mdhA        317     313     307     311   0.000   1.101\n",
      "4h7pA        303     297     299     295     306   0.000\n",
      "\n",
      "\n",
      "\n",
      "   Distance comparison (FIT_ATOMS): \n",
      "\n",
      "       Cutoff for rms calculation:     3.5000\n",
      "\n",
      "       Upper = Distance RMS, Lower = numb equiv distances\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000   0.622   0.810   0.625   0.911   0.759\n",
      "6itkA      49137   0.000   0.749   0.715   0.919   0.744\n",
      "5mdhA      48412   47524   0.000   0.802   0.917   0.779\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1bdmA      47576   47522   47701   0.000   0.706   0.670\n",
      "7mdhA      50311   48905   48769   48195   0.000   0.836\n",
      "4h7pA      45647   44179   44518   43354   47180   0.000\n",
      "\n",
      ">> Sequence comparison: \n",
      "\n",
      "       Diag=numb res, Upper=numb equiv res, Lower = % seq ID\n",
      "\n",
      "\n",
      "            3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA\n",
      "3d5tA         321     176     166     199     147     158\n",
      "6itkA          55     318     146     172     145     145\n",
      "5mdhA          52      46     333     169     142     144\n",
      "1bdmA          63      54      53     318     152     157\n",
      "7mdhA          46      46      43      48     351     136\n",
      "4h7pA          50      46      46      50      43     313\n",
      "\n",
      "\n",
      ">> Dihedral angle comparison: Alph\n",
      "\n",
      "\n",
      "       Cutoff for rms comparison:    60.0000\n",
      "\n",
      "       Upper = RMS Alph, Lower = numb of equiv angles\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000  12.193  13.025  13.697  15.605  12.323\n",
      "6itkA        310   0.000  12.182  11.343  15.326  11.401\n",
      "5mdhA        303     297   0.000  11.652  14.121  10.590\n",
      "1bdmA        302     299     298   0.000  13.876  11.335\n",
      "7mdhA        305     304     306     298   0.000  13.695\n",
      "4h7pA        290     284     290     283     299   0.000\n",
      "\n",
      "\n",
      ">> Dihedral angle comparison: Phi \n",
      "\n",
      "\n",
      "       Cutoff for rms comparison:    60.0000\n",
      "\n",
      "       Upper = RMS Phi , Lower = numb of equiv angles\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000  15.595  17.337  16.596  17.733  15.327\n",
      "6itkA        306   0.000  15.606  13.439  15.624  11.788\n",
      "5mdhA        298     298   0.000  13.774  15.685  13.567\n",
      "1bdmA        299     298     298   0.000  15.310  12.372\n",
      "7mdhA        303     302     299     300   0.000  15.592\n",
      "4h7pA        290     285     290     283     296   0.000\n",
      "\n",
      "\n",
      ">> Dihedral angle comparison: Psi \n",
      "\n",
      "\n",
      "       Cutoff for rms comparison:    15.0000\n",
      "\n",
      "       Upper = RMS Psi , Lower = numb of equiv angles\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000   7.667   7.876   7.046   7.225   7.646\n",
      "6itkA        231   0.000   7.079   7.833   7.397   6.453\n",
      "5mdhA        201     220   0.000   7.610   8.100   7.159\n",
      "1bdmA        231     241     226   0.000   7.988   7.191\n",
      "7mdhA        201     220     203     226   0.000   7.198\n",
      "4h7pA        232     250     229     242     221   0.000\n",
      "\n",
      "\n",
      ">> Dihedral angle comparison: Omeg\n",
      "\n",
      "\n",
      "       Cutoff for rms comparison:    60.0000\n",
      "\n",
      "       Upper = RMS Omeg, Lower = numb of equiv angles\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000   7.825   7.084   6.855   6.569   7.501\n",
      "6itkA        313   0.000   6.776   6.457   6.696   6.461\n",
      "5mdhA        311     308   0.000   5.304   4.291   5.867\n",
      "1bdmA        308     307     309   0.000   4.299   5.983\n",
      "7mdhA        317     313     314     309   0.000   6.028\n",
      "4h7pA        301     296     298     294     307   0.000\n",
      "\n",
      "\n",
      ">> Dihedral angle comparison: chi1\n",
      "\n",
      "\n",
      "       Cutoff for rms comparison:    60.0000\n",
      "\n",
      "       Upper = RMS chi1, Lower = numb of equiv angles\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000  19.203  19.321  18.032  19.384  15.846\n",
      "6itkA        170   0.000  17.550  17.038  13.376  11.951\n",
      "5mdhA        156     153   0.000  18.592  16.554  14.052\n",
      "1bdmA        170     150     168   0.000  19.028  14.457\n",
      "7mdhA        155     139     165     164   0.000  15.929\n",
      "4h7pA        135     134     147     147     149   0.000\n",
      "\n",
      "\n",
      ">> Dihedral angle comparison: chi2\n",
      "\n",
      "\n",
      "       Cutoff for rms comparison:    60.0000\n",
      "\n",
      "       Upper = RMS chi2, Lower = numb of equiv angles\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000  23.915  22.192  21.313  23.858  17.979\n",
      "6itkA        105   0.000  18.771  22.437  22.166  15.609\n",
      "5mdhA        101      89   0.000  23.733  19.460  17.822\n",
      "1bdmA        117     107     103   0.000  24.366  15.570\n",
      "7mdhA         98     100      92     106   0.000  19.039\n",
      "4h7pA         80      77      78      80      83   0.000\n",
      "\n",
      "\n",
      ">> Dihedral angle comparison: chi3\n",
      "\n",
      "\n",
      "       Cutoff for rms comparison:    60.0000\n",
      "\n",
      "       Upper = RMS chi3, Lower = numb of equiv angles\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000  28.948  28.888  25.842  27.667  17.797\n",
      "6itkA         33   0.000  25.981  23.959  28.386  23.725\n",
      "5mdhA         34      27   0.000  25.513  27.377  23.125\n",
      "1bdmA         36      33      35   0.000  30.593  24.317\n",
      "7mdhA         25      33      32      39   0.000  28.573\n",
      "4h7pA         24      23      25      29      21   0.000\n",
      "\n",
      "\n",
      ">> Dihedral angle comparison: chi4\n",
      "\n",
      "\n",
      "       Cutoff for rms comparison:    60.0000\n",
      "\n",
      "       Upper = RMS chi4, Lower = numb of equiv angles\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000  21.769  31.561  25.177  21.665  26.187\n",
      "6itkA         10   0.000  25.457  26.694  23.759  29.356\n",
      "5mdhA         11       9   0.000  29.482  18.541  12.618\n",
      "1bdmA         10       8       9   0.000  24.989  28.248\n",
      "7mdhA         10       8       9      10   0.000  12.948\n",
      "4h7pA         12       6       6       7       6   0.000\n",
      "\n",
      "\n",
      ">> Dihedral angle comparison: chi5\n",
      "\n",
      "\n",
      "       Cutoff for rms comparison:    60.0000\n",
      "\n",
      "       Upper = RMS chi5, Lower = numb of equiv angles\n",
      "\n",
      "        3d5tA   6itkA   5mdhA   1bdmA   7mdhA   4h7pA   \n",
      "3d5tA      0.000   0.000   0.000   0.000   0.000   0.000\n",
      "6itkA          0   0.000   0.000   0.000   0.000   0.000\n",
      "5mdhA          0       0   0.000   0.000   0.000   0.000\n",
      "1bdmA          0       0       0   0.000   0.000   0.000\n",
      "7mdhA          0       0       0       0   0.000   0.000\n",
      "4h7pA          0       0       0       0       0   0.000\n",
      "\n",
      "\n",
      "\n",
      ">> Ramachandran mainchain class substitution\n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              A      P      B      L      E      U\n",
      "      A    2556     25     18     25      0      3\n",
      "      P      22    507    111      6      1      6\n",
      "      B      18    110    900      6      2      1\n",
      "      L      31     12     10    192      4      0\n",
      "      E       1      3      3      1     36      0\n",
      "      U       4     10      0      0      1      3\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      A   2556    71  2627   55.4    1.5   57.0\n",
      "      P    507   146   653   11.0    3.2   14.2\n",
      "      B    900   137  1037   19.5    3.0   22.5\n",
      "      L    192    57   249    4.2    1.2    5.4\n",
      "      E     36     8    44    0.8    0.2    1.0\n",
      "   --------------------------------------------\n",
      "   SUMS   4191   419  4610   90.9    9.1  100.0\n",
      "   --------------------------------------------\n",
      "      U      3    15    18    0.1    0.3    0.4\n",
      "\n",
      "\n",
      "\n",
      ">> Dihedral class substitution: Alph\n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              1      2      3      4\n",
      "      1     151     15     65      2\n",
      "      2      19   2589     39     11\n",
      "      3      70     47   1589      7\n",
      "      4       3      7     11      3\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      1    151    82   233    3.3    1.8    5.1\n",
      "      2   2589    69  2658   56.2    1.5   57.7\n",
      "      3   1589   124  1713   34.5    2.7   37.2\n",
      "   --------------------------------------------\n",
      "   SUMS   4329   275  4604   94.0    6.0  100.0\n",
      "   --------------------------------------------\n",
      "      4      3    21    24    0.1    0.5    0.5\n",
      "\n",
      "\n",
      "\n",
      ">> Dihedral class substitution: Phi \n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              1      2      3      4\n",
      "      1    2552     24    221      9\n",
      "      2      31    223     27      0\n",
      "      3     252     22   1251      0\n",
      "      4      12      1      0      3\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      1   2552   254  2806   55.3    5.5   60.8\n",
      "      2    223    58   281    4.8    1.3    6.1\n",
      "      3   1251   274  1525   27.1    5.9   33.1\n",
      "   --------------------------------------------\n",
      "   SUMS   4026   586  4612   87.3   12.7  100.0\n",
      "   --------------------------------------------\n",
      "      4      3    13    16    0.1    0.3    0.3\n",
      "\n",
      "\n",
      "\n",
      ">> Dihedral class substitution: Psi \n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              1      2      3      4\n",
      "      1    2227    128     41      0\n",
      "      2     113    468     65      3\n",
      "      3      19     62   1497      1\n",
      "      4       2      2      0      0\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      1   2227   169  2396   48.2    3.7   51.8\n",
      "      2    468   181   649   10.1    3.9   14.0\n",
      "      3   1497    82  1579   32.4    1.8   34.1\n",
      "   --------------------------------------------\n",
      "   SUMS   4192   432  4624   90.7    9.3  100.0\n",
      "   --------------------------------------------\n",
      "      4      0     4     4    0.0    0.1    0.1\n",
      "\n",
      "\n",
      "\n",
      ">> Dihedral class substitution: Omeg\n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              1      2      3      4\n",
      "      1    4590      6      0      6\n",
      "      2       5     15      0      1\n",
      "      3       0      0      0      0\n",
      "      4       5      0      0      0\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      1   4590    12  4602   99.3    0.3   99.5\n",
      "      2     15     6    21    0.3    0.1    0.5\n",
      "      3      0     0     0    0.0    0.0    0.0\n",
      "   --------------------------------------------\n",
      "   SUMS   4605    18  4623   99.6    0.4  100.0\n",
      "   --------------------------------------------\n",
      "      4      0     5     5    0.0    0.1    0.1\n",
      "\n",
      "\n",
      "\n",
      ">> Dihedral class substitution: chi1\n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              1      2      3      4\n",
      "      1    1510    364    110    204\n",
      "      2     302    602     62    125\n",
      "      3      95     85    109     45\n",
      "      4     175    142     49    649\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      1   1510   678  2188   41.8   18.8   60.6\n",
      "      2    602   489  1091   16.7   13.5   30.2\n",
      "      3    109   225   334    3.0    6.2    9.2\n",
      "   --------------------------------------------\n",
      "   SUMS   2221  1392  3613   61.5   38.5  100.0\n",
      "   --------------------------------------------\n",
      "      4    649   366  1015   14.0    7.9   21.9\n",
      "\n",
      "\n",
      "\n",
      ">> Dihedral class substitution: chi2\n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              1      2      3      4\n",
      "      1    1080    238     42    402\n",
      "      2     263    327     28    161\n",
      "      3      52     30     59     51\n",
      "      4     357    152     25   1361\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      1   1080   682  1762   39.5   25.0   64.5\n",
      "      2    327   452   779   12.0   16.5   28.5\n",
      "      3     59   133   192    2.2    4.9    7.0\n",
      "   --------------------------------------------\n",
      "   SUMS   1466  1267  2733   53.6   46.4  100.0\n",
      "   --------------------------------------------\n",
      "      4   1361   534  1895   29.4   11.5   40.9\n",
      "\n",
      "\n",
      "\n",
      ">> Dihedral class substitution: chi3\n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              1      2      3      4\n",
      "      1     354     86     29    279\n",
      "      2      83     80     24    115\n",
      "      3      30     17     23     37\n",
      "      4     300     85     34   3052\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      1    354   394   748   30.6   34.1   64.6\n",
      "      2     80   222   302    6.9   19.2   26.1\n",
      "      3     23    84   107    2.0    7.3    9.2\n",
      "   --------------------------------------------\n",
      "   SUMS    457   700  1157   39.5   60.5  100.0\n",
      "   --------------------------------------------\n",
      "      4   3052   419  3471   65.9    9.1   75.0\n",
      "\n",
      "\n",
      "\n",
      ">> Dihedral class substitution: chi4\n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              1      2      3      4\n",
      "      1      93     23     18    140\n",
      "      2      13     24     11     33\n",
      "      3      23      3     13     55\n",
      "      4     139     28     56   3956\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      1     93   181   274   20.7   40.3   61.0\n",
      "      2     24    57    81    5.3   12.7   18.0\n",
      "      3     13    81    94    2.9   18.0   20.9\n",
      "   --------------------------------------------\n",
      "   SUMS    130   319   449   29.0   71.0  100.0\n",
      "   --------------------------------------------\n",
      "      4   3956   223  4179   85.5    4.8   90.3\n",
      "\n",
      "\n",
      "\n",
      ">> Dihedral class substitution: chi5\n",
      "   prot 1 --> prot 2; class on vert --> class on horiz\n",
      "\n",
      "              1      2      3      4\n",
      "      1       0      0      0      1\n",
      "      2       0      0      0      3\n",
      "      3       0      0      0      3\n",
      "      4       4      2      2   4613\n",
      "\n",
      "\n",
      "   Summary (% does not count undefined):\n",
      "\n",
      "   CODE   # ID # NID # TOT   % ID  % NID  % TOT\n",
      "      1      0     1     1    0.0   14.3   14.3\n",
      "      2      0     3     3    0.0   42.9   42.9\n",
      "      3      0     3     3    0.0   42.9   42.9\n",
      "   --------------------------------------------\n",
      "   SUMS      0     7     7    0.0  100.0  100.0\n",
      "   --------------------------------------------\n",
      "      4   4613     8  4621   99.7    0.2   99.8\n",
      "\n",
      "\n",
      "\n",
      ">> Detailed comparison:\n",
      "\n",
      "\n",
      "       Structure :  4\n",
      "       Structure :  5\n",
      "       Structure :  6\n",
      "       ALN      ... alignment position\n",
      "       RESID... ... residue numbers 1/2/3\n",
      "       AAA      ... amino acid types 1/2/3\n",
      "       MNC      ... mainchain classes 1/2/3\n",
      "       ALP...   ... dihedral angle classes (alpha,phi,psi,...)\n",
      "\n",
      "ALN RESIDUE_NUMBERS AAA MNC ALP PHI PSI OMG CH1 CH2 CH3 CH4 CH5 \n",
      "---------------------------------------------------------------\n",
      "  1  ----  23  ---- -D- -U- -4- -4- -3- -1- -1- -1- -4- -4- -4-\n",
      "  2  ----  24  ---- -C- -B- -3- -3- -3- -1- -3- -2- -2- -1- -1-\n",
      "  3  ----  25  ---- -F- -B- -1- -3- -3- -1- -2- -1- -4- -4- -4-\n",
      "  4  ----  26  ---- -G- -E- -1- -2- -3- -1- -4- -4- -4- -4- -4-\n",
      "  5  ----  27  ---- -V- -A- -2- -1- -1- -1- -2- -4- -4- -4- -4-\n",
      "  6  ----  28  ---- -F- -A- -2- -1- -1- -1- -1- -1- -4- -4- -4-\n",
      "  7  ----  29  ---- -C- -B- -2- -3- -2- -1- -1- -1- -2- -2- -3-\n",
      "  8  ----  30  ---- -T- -A- -2- -3- -1- -2- -3- -4- -4- -4- -4-\n",
      "  9    0   43  ---- MW- UE- 42- 42- 33- 11- 11- 23- 34- 44- 44-\n",
      " 10    1   44    1  KKM PPU 314 114 332 111 114 114 134 114 444\n",
      " 11    2   45    2  AKS PPP 333 111 333 111 423 414 414 424 444\n",
      " 12    3   46    3  PLA PPP 333 111 233 111 124 224 144 444 444\n",
      " 13    4   47    4  VVV BBP 333 331 333 111 221 444 444 444 444\n",
      " 14    5   48    5  RNK BBB 333 333 333 111 111 121 141 242 444\n",
      " 15    6   49    6  VIV BBB 333 333 333 111 111 414 444 444 444\n",
      " 16    7   50    7  AAA BBB 333 333 333 111 444 444 444 444 444\n",
      " 17    8   51    8  VVV BBB 333 333 333 111 111 444 444 444 444\n",
      " 18    9   52    9  TST PPP 222 333 333 111 212 444 444 444 444\n",
      " 19   10   53   10  GGG LLL 333 222 222 111 444 444 444 444 444\n",
      " 20   11   54   11  AAA AAA 222 111 111 111 444 444 444 444 444\n",
      " 21   12   55   12  AAA AAA 222 333 222 111 444 444 444 444 444\n",
      " 22   13   56   13  GGG PPP 133 111 333 111 444 444 444 444 444\n",
      " 23   14   57   14  QMQ AAA 222 111 111 111 111 112 121 444 444\n",
      " 24   15   58   15  III AAA 222 111 111 111 111 111 444 444 444\n",
      " 25   16   59   16  GSG AAA 222 111 111 111 434 444 444 444 444\n",
      " 26   17   60   17  YNY AAA 222 111 111 111 222 222 444 444 444\n",
      " 27   18   61   18  SHA AAA 222 331 111 111 214 414 444 444 444\n",
      " 28   19   62   19  LLL AAA 222 311 111 111 222 222 444 444 444\n",
      " 29   20   63   20  LLV AAA 222 111 111 111 112 114 444 444 444\n",
      " 30   21   64   21  FFP AAA 222 111 111 111 112 221 442 444 444\n",
      " 31   22   65   22  RKL AAA 222 111 111 111 111 111 314 124 444\n",
      " 32   23   66   23  ILI AAA 222 111 111 111 111 111 444 444 444\n",
      " 33   24   67   24  AAA AAA 222 111 111 111 444 444 444 444 444\n",
      " 34   25   68   25  ASR AAA 333 111 111 111 413 441 442 443 444\n",
      " 35   26   69   26  GGG LLL 333 222 222 111 444 444 444 444 444\n",
      " 36   27   70   27  EEA AAA 222 111 111 111 134 114 114 444 444\n",
      " 37   28   71   28  MVL AAA 222 313 111 111 212 142 244 444 444\n",
      " 38   29   72   29  LFL AAA 133 333 222 111 111 111 444 444 444\n",
      " 39   30   73   30  GGG EEE 111 222 333 111 444 444 444 444 444\n",
      " 40   31   74   31  KQP AAA 222 111 111 111 112 321 112 144 444\n",
      " 41   32   75   32  DDT AAA 222 333 221 111 332 114 444 444 444\n",
      " 42   33   76   33  QQT BBB 333 333 333 111 221 334 114 444 444\n",
      " 43   34   77   34  PPP PPP 333 111 222 111 321 212 121 444 444\n",
      " 44   35   78   35  VIV BBB 333 333 333 111 222 414 444 444 444\n",
      " 45   36   79   36  IAE BBB 333 333 333 111 142 141 441 444 444\n",
      " 46   37   80   37  LLL BBB 333 333 333 111 111 111 444 444 444\n",
      " 47   38   81   38  QKR BBB 333 333 333 111 111 121 213 433 444\n",
      " 48   39   82   39  LLL BBB 333 333 333 111 111 111 444 444 444\n",
      " 49   40   83   40  LLL BBB 333 333 333 111 222 222 444 444 444\n",
      " 50   41   84   41  EGD BBB 311 333 333 111 142 241 144 444 444\n",
      " 51   42   85   42  ISI PPP 333 111 333 111 212 141 444 444 444\n",
      " 52   43   86   43  PEE AAA 222 111 111 111 132 222 111 444 444\n",
      " 53   44   87   44  QRP AAA 222 111 111 111 121 232 211 414 444\n",
      " 54   45   88   45  ASA AAA 222 333 222 111 424 444 444 444 444\n",
      " 55   46   89   46  MFL AAA 222 111 111 111 121 221 144 444 444\n",
      " 56   47   90   47  KQK AAA 222 111 111 111 222 111 313 341 444\n",
      " 57   48   91   48  AAA AAA 222 111 111 111 444 444 444 444 444\n",
      " 58   49   92   49  LLL AAA 222 111 111 111 211 211 444 444 444\n",
      " 59   50   93   50  EEA AAA 222 111 111 111 114 214 114 444 444\n",
      " 60   51   94   51  GGG AAA 222 111 111 111 444 444 444 444 444\n",
      " 61   52   95   52  VVV AAA 222 111 111 111 111 444 444 444 444\n",
      " 62   53   96   53  VAE AAA 222 111 111 111 142 441 441 444 444\n",
      " 63   54   97   54  MMA AAA 222 111 111 111 114 114 124 444 444\n",
      " 64   55   98   55  EEE AAA 222 111 111 111 111 111 111 444 444\n",
      " 65   56   99   56  LLL AAA 222 111 111 111 111 111 444 444 444\n",
      " 66   57  100   57  EEE AAA 222 111 111 111 111 111 111 444 444\n",
      " 67   58  101   58  DDD AAA 222 111 211 111 111 111 444 444 444\n",
      " 68   59  102   59  CSC AAA 333 331 111 111 121 444 444 444 444\n",
      " 69   60  103   60  ALA LLL 333 222 222 111 414 414 444 444 444\n",
      " 70   61  104   61  FYF BPB 333 111 333 111 111 111 444 444 444\n",
      " 71   62  105   62  PPP AAA 222 111 111 111 322 111 222 444 444\n",
      " 72   63  106   63  LLL AAA 222 313 211 111 111 111 444 444 444\n",
      " 73   64  107   64  LLL BBB 333 113 333 111 222 222 444 444 444\n",
      " 74   65  108   65  ARD AAA 121 111 111 111 411 421 424 414 444\n",
      " 75   66  109   66  GEK BBB 333 333 333 111 412 411 411 441 444\n",
      " 76   67  110   67  LVV BBB 333 333 333 111 211 144 444 444 444\n",
      " 77   68  111   68  ESV BBB 333 333 333 111 211 144 144 444 444\n",
      " 78   69  112   69  AIV BBB 333 333 333 111 422 414 444 444 444\n",
      " 79   70  113   70  TGT BBB 333 333 333 111 343 444 444 444 444\n",
      " 80   71  114   71  DIA AAA 222 333 222 111 324 114 444 444 444\n",
      " 81   72  115   72  DDD BBB 333 333 333 111 222 111 444 444 444\n",
      " 82   73  116   73  PPP AAA 222 111 111 111 123 212 121 444 444\n",
      " 83   74  117   74  DYR AAA 222 111 111 111 111 121 442 441 444\n",
      " 84   75  118   75  VEV AAA 222 131 111 111 121 434 414 444 444\n",
      " 85   76  119   76  AVA AAA 222 111 111 111 414 444 444 444 444\n",
      " 86   77  120   77  FFF AAA 333 311 222 111 111 111 444 444 444\n",
      " 87   78  121   78  KED PPP 111 111 333 111 232 111 114 344 444\n",
      " 88   79  122   79  DDG LLL 333 222 222 111 114 114 444 444 444\n",
      " 89   80  123   80  AVV PPP 333 111 333 111 422 444 444 444 444\n",
      " 90   81  124   81  DDA AAA 222 333 111 111 114 114 444 444 444\n",
      " 91   82  125   82  YWI BBB 333 333 333 111 111 212 444 444 444\n",
      " 92   83  126   83  AAA BBB 333 333 333 111 444 444 444 444 444\n",
      " 93   84  127   84  LLI BBB 333 333 333 111 111 211 444 444 444\n",
      " 94   85  128   85  LLM BBB 333 333 333 111 111 112 441 444 444\n",
      " 95   86  129   86  VIC AAA 222 333 111 111 111 414 444 444 444\n",
      " 96   87  130   87  GGG PPP 333 111 333 111 444 444 444 444 444\n",
      " 97   88  131   88  AAA BBB 131 333 333 111 444 444 444 444 444\n",
      " 98   89  ----  89  A-F P-P 4-3 1-1 3-3 1-1 4-2 4-1 4-4 4-4 4-4\n",
      " 99   90  132  ---- PK- PP- 43- 11- 23- 41- 21- 13- 11- 41- 44-\n",
      "100   91  133   90  RPP UPP 413 411 422 411 422 411 421 444 444\n",
      "101  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "102  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "103  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "104  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "105  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "106  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "107  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "108  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "109  ---- 134   91  -RR -PP -33 -11 -33 -11 -22 -11 -11 -12 -44\n",
      "110  ---- 135   92  -GK -PB -11 -13 -33 -11 -44 -44 -44 -44 -44\n",
      "111  ---- 136   93  -PA -PP -12 -11 -23 -11 -24 -14 -24 -44 -44\n",
      "112  ---- 137   94  -GG -LL -33 -22 -22 -11 -44 -44 -44 -44 -44\n",
      "113  ---- 138   95  -MM -BP -33 -31 -33 -11 -21 -11 -12 -44 -44\n",
      "114  ---- 139   96  -EE -PP -33 -11 -33 -11 -33 -21 -11 -44 -44\n",
      "115  ---- 140   97  -RR -AA -22 -11 -11 -11 -21 -31 -13 -13 -44\n",
      "116  ---- 141   98  -AK -AA -22 -11 -11 -11 -44 -44 -44 -44 -44\n",
      "117  ---- 142   99  -AD -AA -22 -11 -11 -11 -41 -41 -44 -44 -44\n",
      "118  ---- 143  100  -LL -AA -22 -11 -11 -11 -22 -22 -44 -44 -44\n",
      "119  ---- 144  101  -LL -AA -22 -11 -11 -11 -12 -11 -44 -44 -44\n",
      "120  101  145  102  LDE UAA 422 411 111 111 121 211 441 444 444\n",
      "121  102  ---- ---- Q-- A-- 2-- 1-- 1-- 1-- 1-- 2-- 1-- 4-- 4--\n",
      "122  103  146  103  VIM AAA 222 111 111 111 314 414 444 444 444\n",
      "123  104  147  104  NNN AAA 222 111 111 111 111 211 444 444 444\n",
      "124  105  148  105  GGA AAA 222 111 111 111 444 444 444 444 444\n",
      "125  106  149  106  KQR AAA 222 111 111 111 124 214 214 344 444\n",
      "126  107  150  107  III AAA 222 111 111 111 111 111 444 444 444\n",
      "127  108  151  108  FFF AAA 222 111 111 111 111 221 444 444 444\n",
      "128  109  152  109  TAK AAA 222 111 111 111 241 441 441 441 444\n",
      "129  110  153  110  EDE AAA 222 111 111 111 224 114 141 444 444\n",
      "130  111  154  111  QQQ AAA 222 111 111 111 111 111 221 444 444\n",
      "131  112  155  112  GGG AAA 222 111 111 111 444 444 444 444 444\n",
      "132  113  156  113  RKE AAA 222 111 111 111 111 111 211 314 444\n",
      "133  114  157  114  AAA AAA 222 111 111 111 444 444 444 444 444\n",
      "134  115  158  115  LLI AAA 222 111 111 111 111 111 444 444 444\n",
      "135  116  159  116  ANA AAA 222 111 111 111 424 424 444 444 444\n",
      "136  117  160  117  EAA AAA 222 111 111 111 244 144 144 444 444\n",
      "137  118  161  118  VVV AAA 222 333 111 111 222 444 444 444 444\n",
      "138  119  162  119  AAA PPP 111 311 333 111 444 444 444 444 444\n",
      "139  120  163  120  KSA PPP 333 111 333 111 134 244 144 344 444\n",
      "140  121  164  121  KKS AAA 222 111 111 111 111 324 214 114 444\n",
      "141  122  165  122  DND AAA 333 333 222 111 223 111 444 444 444\n",
      "142  123  166  123  VVC PPP 333 111 333 111 332 444 444 444 444\n",
      "143  124  167  124  KKR BBB 333 333 333 111 111 221 111 213 444\n",
      "144  125  168  125  VVV BBB 333 333 333 111 111 444 444 444 444\n",
      "145  126  169  126  LLV BBB 333 333 333 111 213 224 444 444 444\n",
      "146  127  170  127  VVV BBB 333 333 333 111 111 444 444 444 444\n",
      "147  128  171  128  VVV AAA 222 333 111 111 111 444 444 444 444\n",
      "148  129  172  129  GGG PPP 111 111 333 111 444 444 444 444 444\n",
      "149  130  173  130  NNN PPP 222 113 333 222 111 111 444 444 444\n",
      "150  131  174  131  PPP PPP 333 111 222 111 212 121 212 444 444\n",
      "151  132  175  132  ACA AAA 222 111 111 111 414 444 444 444 444\n",
      "152  133  176  133  NNN AAA 222 111 111 111 111 122 444 444 444\n",
      "153  134  177  134  TTT AAA 222 111 111 111 111 444 444 444 444\n",
      "154  135  178  135  NNN AAA 222 111 111 111 111 111 444 444 444\n",
      "155  136  179  136  AAA AAA 222 111 111 111 444 444 444 444 444\n",
      "156  137  180  137  LLL AAA 222 111 111 111 222 222 444 444 444\n",
      "157  138  181  138  III AAA 222 111 111 111 111 111 444 444 444\n",
      "158  139  182  139  ACL AAA 222 111 111 111 422 441 444 444 444\n",
      "159  140  183  140  YLL AAA 222 111 111 111 332 232 444 444 444\n",
      "160  141  184  141  KKK AAA 222 111 111 111 111 322 111 311 444\n",
      "161  142  185  142  NNS AAA 222 331 221 111 112 114 444 444 444\n",
      "162  143  186  143  AAA BBA 333 331 221 111 444 444 444 444 444\n",
      "163  ---- ---- 144  --Q --L --1 --2 --2 --1 --1 --1 --1 --4 --4\n",
      "164  144  187  145  PPG PAL 222 112 212 111 224 114 224 444 444\n",
      "165  145  188  146  GDK LAA 222 213 221 111 414 414 444 444 444\n",
      "166  146  189  147  LIL BBB 333 333 333 111 111 111 444 444 444\n",
      "167  147  190  148  NPN PPP 333 111 323 111 212 121 414 444 444\n",
      "168  148  191  149  PAP AAA 222 111 111 111 242 141 242 444 444\n",
      "169  149  192  150  RKR AAA 222 111 111 111 121 211 213 123 444\n",
      "170  150  193  151  NNH AAA 222 113 222 111 111 113 444 444 444\n",
      "171  151  194  152  FFV BBB 333 333 333 111 111 224 444 444 444\n",
      "172  152  195  153  THT BBB 333 333 333 111 212 414 444 444 444\n",
      "173  153  196  154  AAA BBB 333 333 333 111 444 444 444 444 444\n",
      "174  154  197  155  MLM BPB 333 313 333 111 212 111 242 444 444\n",
      "175  155  198  156  TTT AAA 222 333 222 111 222 444 444 444 444\n",
      "176  156  199  157  RRR AAA 222 111 111 111 222 333 111 222 444\n",
      "177  157  200  158  LLL AAA 222 111 111 111 222 222 444 444 444\n",
      "178  158  201  159  DDD AAA 222 111 111 111 111 112 444 444 444\n",
      "179  159  202  160  HEH AAA 222 111 111 111 222 212 414 444 444\n",
      "180  160  203  161  NNN AAA 222 111 111 111 111 111 444 444 444\n",
      "181  161  204  162  RRR AAA 222 111 111 111 111 111 111 111 444\n",
      "182  162  205  163  AAA AAA 222 111 111 111 444 444 444 444 444\n",
      "183  163  206  164  KKL AAA 222 111 111 111 111 111 334 114 444\n",
      "184  164  207  165  ACS AAA 222 111 111 111 423 444 444 444 444\n",
      "185  165  208  166  QQL AAA 222 111 111 111 112 222 224 444 444\n",
      "186  166  209  167  LLL AAA 222 111 111 111 212 212 444 444 444\n",
      "187  167  210  168  AAA AAA 222 111 111 111 444 444 444 444 444\n",
      "188  168  211  169  KLR AAA 222 111 111 111 221 121 242 142 444\n",
      "189  169  212  170  KKK AAA 222 111 111 111 221 331 111 212 444\n",
      "190  170  213  171  TAA AAA 333 131 221 111 244 444 444 444 444\n",
      "191  171  214  172  GGG LLL 323 222 222 111 444 444 444 444 444\n",
      "192  172  215  173  TVV PBB 331 333 333 111 212 444 444 444 444\n",
      "193  173  216  174  GFP PPP 333 111 332 111 421 422 441 444 444\n",
      "194  174  217  175  VYV AAA 222 111 111 111 333 424 444 444 444\n",
      "195  175  218  176  DDS AAA 222 131 121 111 331 114 444 444 444\n",
      "196  176  219  177  RKQ AAA 222 333 221 111 111 212 331 114 444\n",
      "197  177  220  178  IVV PBB 333 333 333 111 111 244 444 444 444\n",
      "198  178  221  179  RSR BBB 222 333 333 111 314 144 244 144 444\n",
      "199  179  222  180  RNN LLL 333 222 222 111 111 121 144 144 444\n",
      "200  180  223  181  MVV PBP 333 131 333 111 212 344 244 444 444\n",
      "201  181  224  182  TTI BBB 333 333 333 111 111 441 444 444 444\n",
      "202  182  225  183  VII BBB 333 333 333 111 111 422 444 444 444\n",
      "203  183  226  184  WWW BPB 211 333 333 111 111 111 444 444 444\n",
      "204  184  227  185  GGG EEE 122 222 333 111 444 444 444 444 444\n",
      "205  185  228  186  NNN BPP 133 333 333 111 322 111 444 444 444\n",
      "206  186  229  187  HHH BBB 222 111 333 111 222 211 444 444 444\n",
      "207  187  230  188  SSS BBB 311 333 333 111 111 444 444 444 444\n",
      "208  188  231  189  STS AAA 222 111 121 111 122 444 444 444 444\n",
      "209  189  232  190  ITT AAA 333 333 212 111 222 144 444 444 444\n",
      "210  190  233  191  MQQ PPP 333 111 333 111 333 311 311 444 444\n",
      "211  191  234  192  FVV BBB 333 333 333 111 211 244 444 444 444\n",
      "212  192  235  193  PPP BPP 333 111 222 111 213 122 211 444 444\n",
      "213  193  236  194  DDD BBB 333 313 333 111 222 111 444 444 444\n",
      "214  194  237  195  LFT AAA 222 333 212 111 132 224 444 444 444\n",
      "215  195  238  196  FLD AAA 222 111 111 111 121 111 444 444 444\n",
      "216  196  239  197  HNS AAA 222 333 222 111 112 114 444 444 444\n",
      "217  197  240  198  AAA PPP 333 111 333 111 444 444 444 444 444\n",
      "218  198  241  199  EKV BBB 333 333 333 111 112 114 114 414 444\n",
      "219  199  242  200  VII BBB 111 333 333 111 111 411 444 444 444\n",
      "220  200  243  201  DDG LLE 111 222 223 111 224 114 444 444 444\n",
      "221  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "222  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "223  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "224  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "225  205  244  202  GGT LLA 222 223 222 111 442 444 444 444 444\n",
      "226  206  245  203  RRT PBB 333 333 333 111 111 214 224 114 444\n",
      "227  207  246  204  PPP PPP 333 111 222 111 111 222 211 444 444\n",
      "228  208  247  205  AVA AAA 222 111 111 111 434 444 444 444 444\n",
      "229  209  248  206  LKR AAA 222 111 211 111 114 214 434 434 444\n",
      "230  210  249  207  EEE AAA 222 131 111 111 111 111 111 444 444\n",
      "231  211  250  208  LVA AAA 222 333 112 111 114 144 444 444 444\n",
      "232  212  251  209  VII BBB 333 313 333 111 311 411 444 444 444\n",
      "233  ---- ---- 210  --K --A --1 --3 --2 --1 --4 --4 --4 --4 --4\n",
      "234  ---- 252  ---- -K- -A- -2- -3- -1- -1- -1- -3- -1- -2- -4-\n",
      "235  214  253  ---- DR- PB- 33- 31- 33- 11- 32- 11- 41- 41- 44-\n",
      "236  215  254  ---- MT- AA- 22- 11- 11- 11- 21- 34- 34- 44- 44-\n",
      "237  216  255  ---- EK- AA- 22- 11- 11- 11- 21- 12- 11- 41- 44-\n",
      "238  217  256  ---- WW- AA- 22- 11- 11- 11- 22- 11- 44- 44- 44-\n",
      "239  218  257  ---- YL- AA- 22- 11- 11- 11- 21- 21- 44- 44- 44-\n",
      "240  219  258  ---- EE- AA- 22- 11- 11- 11- 21- 11- 11- 44- 44-\n",
      "241  220  259  ---- KE- AA- 22- 33- 11- 11- 11- 12- 11- 14- 44-\n",
      "242  221  260  217  VED AAL 223 312 111 111 114 412 414 444 444\n",
      "243  222  261  218  FFF AAA 222 111 111 111 222 221 444 444 444\n",
      "244  223  262  219  ITV AAA 222 111 111 111 131 144 444 444 444\n",
      "245  224  263  220  PIQ AAA 222 111 111 111 314 214 144 444 444\n",
      "246  225  264  221  TTV AAA 222 111 111 111 121 444 444 444 444\n",
      "247  226  265  222  VVV AAA 222 111 111 111 111 444 444 444 444\n",
      "248  227  266  223  AQR AAA 222 111 111 111 421 411 421 441 444\n",
      "249  228  267  224  QKG AAA 222 331 222 111 214 224 124 414 444\n",
      "250  229  268  225  RRR AAA 222 111 111 111 222 333 111 111 444\n",
      "251  230  269  226  GGG AAA 222 111 111 111 444 444 444 444 444\n",
      "252  231  270  227  AGA AAA 222 111 111 111 444 444 444 444 444\n",
      "253  232  271  228  AAE AAA 222 111 111 111 442 441 441 444 444\n",
      "254  233  272  229  ILI AAA 222 111 111 111 111 112 444 444 444\n",
      "255  234  273  230  III AAA 222 111 111 111 111 121 444 444 444\n",
      "256  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "257  235  274  231  QQQ AAA 222 111 111 111 134 134 114 444 444\n",
      "258  236  275  232  AKL AAA 222 131 111 111 411 421 414 434 444\n",
      "259  237  276  233  RWR AAA 333 313 111 111 121 121 343 142 444\n",
      "260  238  277  234  GGG LLL 222 222 212 111 444 444 444 444 444\n",
      "261  239  278  235  ARL BBB 313 333 333 111 421 411 424 414 444\n",
      "262  240  279  236  SSS PPP 131 111 333 111 132 444 444 444 444\n",
      "263  241  280  237  SSS BAP 333 131 223 111 112 444 444 444 444\n",
      "264  242  281  238  AAA AAA 222 113 111 111 444 444 444 444 444\n",
      "265  243  282  239  AAM AAA 222 111 111 111 442 441 441 444 444\n",
      "266  244  283  240  SSS AAA 222 111 111 111 111 444 444 444 444\n",
      "267  245  284  241  ATA AAA 222 111 111 111 424 444 444 444 444\n",
      "268  246  285  242  AAA AAA 222 111 111 111 444 444 444 444 444\n",
      "269  247  286  243  NVK AAA 222 111 111 111 212 242 441 441 444\n",
      "270  248  287  244  ASA AAA 222 111 111 111 414 444 444 444 444\n",
      "271  249  288  245  AIA AAA 222 111 111 111 414 414 444 444 444\n",
      "272  250  289  246  IAV AAA 222 111 111 111 141 144 444 444 444\n",
      "273  251  290  247  EDD AAA 222 111 111 111 111 111 144 444 444\n",
      "274  252  291  248  HAH AAA 222 111 111 111 242 142 444 444 444\n",
      "275  253  292  249  IIV AAA 222 111 111 111 111 214 444 444 444\n",
      "276  254  293  250  RKH AAA 222 111 111 111 212 111 324 114 444\n",
      "277  255  294  251  DSD AAA 222 111 111 111 121 141 444 444 444\n",
      "278  256  295  252  WLW AAA 222 131 111 111 212 313 444 444 444\n",
      "279  257  296  253  AVI AAA 222 131 121 111 421 442 444 444 444\n",
      "280  258  297  254  LTH ABA 333 333 131 111 111 141 444 444 444\n",
      "281  259  298  255  GPG BPE 131 312 323 111 414 424 414 444 444\n",
      "282  260  299  256  TTT PPP 333 111 333 111 212 444 444 444 444\n",
      "283  261  300  257  PPP PPP 111 111 222 111 312 221 112 444 444\n",
      "284  262  301  258  EEE PPP 222 111 333 111 114 224 111 444 444\n",
      "285  263  302  259  GGG LLL 222 222 222 111 444 444 444 444 444\n",
      "286  264  303  260  DDV BPB 333 333 333 111 221 114 444 444 444\n",
      "287  265  304  261  WWY BBB 333 333 333 111 333 112 444 444 444\n",
      "288  266  305  262  VFV BBB 333 333 333 111 232 424 444 444 444\n",
      "289  267  306  263  SSS BPB 333 311 333 111 222 444 444 444 444\n",
      "290  268  307  264  MTM BBB 333 333 333 111 131 141 242 444 444\n",
      "291  269  308  265  AGG BBB 333 311 333 111 444 444 444 444 444\n",
      "292  270  309  266  VVV BBB 111 333 333 111 222 444 444 444 444\n",
      "293  271  310  267  PYY PPP 333 111 233 111 122 222 144 444 444\n",
      "294  272  311  268  STS PAA 323 131 321 111 321 444 444 444 444\n",
      "295  273  312  269  QTD AAP 122 311 213 111 111 141 244 444 444\n",
      "296  ---- 313  270  -GE -AL -33 -12 -22 -11 -41 -41 -41 -44 -44\n",
      "297  274  ---- ---- G-- L-- 3-- 2-- 2-- 1-- 4-- 4-- 4-- 4-- 4--\n",
      "298  275  314  271  ENN PPP 233 111 333 111 133 322 144 444 444\n",
      "299  ---- 315  272  -PP -AA -22 -11 -11 -11 -32 -21 -12 -44 -44\n",
      "300  277  316  273  YYY LAA 133 213 222 111 111 111 444 444 444\n",
      "301  278  317  274  GGG LLL 333 222 222 111 444 444 444 444 444\n",
      "302  279  318  275  IIV PPP 313 331 333 111 131 324 444 444 444\n",
      "303  280  319  276  PAP PPP 333 111 232 111 341 142 241 444 444\n",
      "304  281  320  277  EES PPP 222 131 333 111 112 114 114 444 444\n",
      "305  282  321  278  GDG LLL 222 222 122 111 424 414 444 444 444\n",
      "306  283  322  279  IIL PBB 333 133 333 111 221 111 444 444 444\n",
      "307  284  323  280  VVI BBB 333 133 333 111 111 442 444 444 444\n",
      "308  285  324  281  YFF BBB 333 333 333 111 222 211 444 444 444\n",
      "309  286  325  282  SSS BBP 333 111 333 111 333 444 444 444 444\n",
      "310  287  326  283  FMF BBB 313 333 333 111 111 111 414 444 444\n",
      "311  288  327  284  PPP PPP 333 111 222 111 222 111 222 444 444\n",
      "312  289  328  285  VCC BBB 333 333 333 111 211 444 444 444 444\n",
      "313  290  329  286  TRT BBP 333 333 333 111 232 414 414 424 444\n",
      "314  291  330  287  ASC BBB 333 333 333 111 431 444 444 444 444\n",
      "315  292  331  288  KKH BAB 132 333 323 111 214 114 314 314 444\n",
      "316  ---- 332  ---- -G- -L- -2- -2- -2- -1- -4- -4- -4- -4- -4-\n",
      "317  293  333  289  DDA LAL 131 232 222 111 134 114 444 444 444\n",
      "318  294  334  290  GGG LLL 222 222 222 111 444 444 444 444 444\n",
      "319  295  335  291  ADE BPB 333 333 333 111 421 412 441 444 444\n",
      "320  296  336  292  YYW BBP 333 331 333 111 333 212 444 444 444\n",
      "321  297  337  293  RET BBB 333 333 333 111 231 314 314 344 444\n",
      "322  298  338  294  VLV PPP 333 111 333 111 111 434 444 444 444\n",
      "323  299  339  295  VAV BBP 333 131 333 111 141 444 444 444 444\n",
      "324  300  340  296  ETS PAP 222 331 313 111 121 144 144 444 444\n",
      "325  ---- 341  ---- -D- -A- -2- -1- -2- -1- -3- -1- -4- -4- -4-\n",
      "326  301  ---- ---- G-- L-- 3-- 2-- 2-- 1-- 4-- 4-- 4-- 4-- 4--\n",
      "327  302  342  297  LVG PBL 332 132 332 111 114 144 444 444 444\n",
      "328  ---- ---- 298  --K --A --1 --3 --2 --1 --1 --2 --1 --2 --4\n",
      "329  303  343  ---- ES- BB- 33- 13- 33- 11- 12- 14- 14- 44- 44-\n",
      "330  304  344  299  INL BBP 333 333 333 112 121 211 444 444 444\n",
      "331  305  345  ---- ND- PB- 33- 13- 33- 11- 33- 11- 44- 44- 44-\n",
      "332  306  346  ---- ED- AA- 22- 11- 11- 11- 21- 11- 14- 44- 44-\n",
      "333  307  347  ---- FF- AA- 22- 11- 11- 11- 22- 12- 44- 44- 44-\n",
      "334  308  348  ---- AL- AA- 22- 11- 11- 11- 42- 42- 44- 44- 44-\n",
      "335  309  349  305  RWK AAA 222 113 111 111 324 124 344 144 444\n",
      "336  310  350  306  KEQ AAA 222 111 111 111 214 324 214 344 444\n",
      "337  311  351  307  RRR AAA 222 111 111 111 111 111 313 331 444\n",
      "338  312  352  308  MIL AAA 222 113 111 111 111 111 244 444 444\n",
      "339  313  353  309  EKA AAA 222 111 111 111 114 224 124 414 444\n",
      "340  314  354  310  IKS AAA 222 111 111 111 112 114 414 434 444\n",
      "341  315  355  311  TST AAA 222 111 111 111 333 444 444 444 444\n",
      "342  316  356  312  AEI AAA 222 111 111 111 421 411 414 444 444\n",
      "343  317  357  313  QAA AAA 222 111 111 111 144 244 244 444 444\n",
      "344  318  358  314  EEE AAA 222 111 111 111 222 311 111 444 444\n",
      "345  319  359  315  LLL AAA 222 111 111 111 111 111 444 444 444\n",
      "346  320  360  316  LLQ AAA 222 111 111 111 111 121 441 444 444\n",
      "347  321  361  317  DAE AAA 222 111 111 111 144 144 441 444 444\n",
      "348  322  362  318  EEE AAA 222 111 111 111 111 111 111 444 444\n",
      "349  323  363  319  MKR AAA 222 111 111 111 222 111 313 411 444\n",
      "350  324  364  320  EKA AAA 222 111 111 111 214 114 114 414 444\n",
      "351  325  365  321  QCQ AAA 222 111 111 111 221 121 221 414 414\n",
      "352  326  366  322  VVA AAA 233 131 122 111 124 444 444 444 444\n",
      "353  327  367  323  KAG AAL 224 112 112 111 144 144 144 144 444\n",
      "354  ---- ---- 324  --L --U --4 --3 --4 --4 --1 --1 --4 --4 --4\n",
      "355  328  ---- ---- A-- A-- 2-- 1-- 1-- 1-- 4-- 4-- 4-- 4-- 4--\n",
      "356  329  ---- ---- L-- A-- 3-- 1-- 2-- 1-- 1-- 1-- 4-- 4-- 4--\n",
      "357  330  368  ---- GH- LA- 32- 21- 21- 11- 43- 43- 44- 44- 44-\n",
      "358  331  369  ---- LL- AA- 42- 33- 11- 11- 11- 11- 44- 44- 44-\n",
      "359  332  370  ---- IT- UA- 43- 33- 41- 41- 22- 14- 44- 44- 44-\n",
      "360  ---- 371  ---- -G- -L- -3- -2- -2- -1- -4- -4- -4- -4- -4-\n",
      "361  ---- 372  ---- -E- -A- -3- -3- -2- -1- -1- -3- -1- -4- -4-\n",
      "362  ---- 373  ---- -G- -E- -2- -2- -3- -1- -4- -4- -4- -4- -4-\n",
      "363  ---- 374  ---- -N- -P- -3- -3- -3- -1- -1- -2- -4- -4- -4-\n",
      "364  ---- 375  ---- -A- -B- -3- -3- -3- -1- -4- -4- -4- -4- -4-\n",
      "365  ---- 376  ---- -Y- -B- -3- -3- -3- -1- -2- -2- -4- -4- -4-\n",
      "366  ---- 377  ---- -C- -B- -3- -3- -3- -1- -1- -1- -2- -2- -2-\n",
      "367  ---- 378  ---- -D- -A- -3- -3- -2- -1- -2- -1- -4- -4- -4-\n",
      "368  ---- 379  ---- -V- -B- -3- -3- -3- -1- -2- -4- -4- -4- -4-\n",
      "369  ---- 380  ---- -P- -P- -3- -1- -2- -1- -2- -1- -2- -4- -4-\n",
      "370  ---- 381  ---- -E- -A- -2- -1- -1- -1- -1- -2- -1- -4- -4-\n",
      "371  ---- 382  ---- -D- -B- -2- -1- -3- -1- -1- -1- -4- -4- -4-\n",
      "372  ---- 383  ---- -T- -A- -3- -3- -1- -1- -3- -4- -4- -4- -4-\n",
      "373  ---- 384  ---- -M- -P- -4- -1- -3- -1- -2- -3- -2- -4- -4-\n",
      "374  ---- 385  ---- -L- -U- -4- -1- -4- -4- -1- -1- -4- -4- -4-\n",
      "375  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "376  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "377  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "378  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "379  ---- ---- ---- --- --- --- --- --- --- --- --- --- --- ---\n",
      "compare_344_> << END OF COMMAND\n",
      "\n",
      "Sequence identity comparison (ID_TABLE):\n",
      "\n",
      "   Diagonal       ... number of residues;\n",
      "   Upper triangle ... number of identical residues;\n",
      "   Lower triangle ... % sequence identity, id/min(length).\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "         3d5tA @26itkA @25mdhA @21bdmA @17mdhA @24h7pA @1\n",
      "3d5tA @2      321     176     166     199     147     158\n",
      "6itkA @2       55     318     146     172     145     145\n",
      "5mdhA @2       52      46     333     169     142     144\n",
      "1bdmA @1       63      54      53     318     152     157\n",
      "7mdhA @2       46      46      43      48     351     136\n",
      "4h7pA @1       50      46      46      50      43     313\n",
      "openf___224_> Open           family.mat\n",
      "\n",
      "\n",
      "Weighted pair-group average clustering based on a distance matrix:\n",
      "\n",
      "\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2949729    2880.595     2.813\n",
      "                                                               .--- 3d5tA @2.5    37.0000\n",
      "                                                               |\n",
      "                                       .--------------------------- 1bdmA @1.8    45.5000\n",
      "                                       |\n",
      "                       .------------------------------------------- 6itkA @2.0    50.7500\n",
      "                       |\n",
      "                 .------------------------------------------------- 5mdhA @2.4    53.0000\n",
      "                 |\n",
      "        .---------------------------------------------------------- 4h7pA @1.3    56.1250\n",
      "        |\n",
      "      .------------------------------------------------------------ 7mdhA @2.4\n",
      "\n",
      "      +----+----+----+----+----+----+----+----+----+----+----+----+\n",
      "    56.8900   53.4475   50.0050   46.5625   43.1200   39.6775   36.2350\n",
      "         55.1688   51.7263   48.2838   44.8413   41.3988   37.9563\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "\n",
    "env = environ()\n",
    "aln = alignment(env)\n",
    "#update the following PDB ID and chain from the selected template sturctures\n",
    "for (pdb, chain) in (('3d5t', 'A'), ('6itk', 'A'), ('5mdh', 'A'),\n",
    "                     ('1bdm', 'A'), ('7mdh', 'A'), ('4h7p', 'A')):\n",
    "    m = model(env, file=pdb, model_segment=('FIRST:'+chain, 'LAST:'+chain))\n",
    "    aln.append_model(m, atom_files=pdb, align_codes=pdb+chain)\n",
    "aln.malign()\n",
    "aln.malign3d()\n",
    "aln.compare_structures()\n",
    "aln.id_table(matrix_file='family.mat')\n",
    "env.dendrogram(matrix_file='family.mat', cluster_cut=-1.0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Aligning target sequence with a template"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      4047732    3952.863     3.860\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      4048260    3953.379     3.861\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      4096860    4000.840     3.907\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      4110160    4013.828     3.920\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           4h7p.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3739077    3651.442     3.566\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3739153    3651.517     3.566\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3739324    3651.684     3.566\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3740174    3652.514     3.567\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3741432    3653.742     3.568\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3743336    3655.602     3.570\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3743376    3655.641     3.570\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3743466    3655.729     3.570\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3746322    3658.518     3.573\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3746382    3658.576     3.573\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3746517    3658.708     3.573\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3750801    3662.892     3.577\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3750889    3662.978     3.577\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3751087    3663.171     3.577\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3757513    3669.446     3.583\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3757645    3669.575     3.584\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3757942    3669.865     3.584\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3767564    3679.262     3.593\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3767764    3679.457     3.593\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3768214    3679.896     3.594\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3782664    3694.008     3.607\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3782964    3694.301     3.608\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3783639    3694.960     3.608\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3805297    3716.110     3.629\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3805745    3716.548     3.629\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3806753    3717.532     3.630\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      3839257    3749.274     3.661\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      4071129    3975.712     3.883\n",
      "read_mo_297_> Segments, residues, atoms:        1      313     2244\n",
      "read_mo_298_> Segment:        1     1 A   324 A     2244\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      3483120    3401.484     3.322\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      3657397    3571.677     3.488\n",
      "openf___224_> Open           TvLDH.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      3758426    3670.338     3.584\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      3854830    3764.482     3.676\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      3856166    3765.787     3.678\n",
      "\n",
      "Read the alignment from file       : TvLDH.ali\n",
      "\n",
      "Total number of alignment positions:   335\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1       4h7p     313      1    4h7p.pdb MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  2      TvLDH     335      1       TvLDH\n",
      "align2d_276_> 'align_block' changed to 1.\n",
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "openf___224_> Open           $(LIB)/as1.sim.mat\n",
      "rdrrwgh_268_> Number of residue types:       20\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :       999999\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          337\n",
      "  Score                    :  201403.6406\n",
      "openf___224_> Open           TvLDH-4h7pA.ali\n",
      "openf___224_> Open           TvLDH-4h7pA.pap\n"
     ]
    }
   ],
   "source": [
    "# based on the distance matrix, we picket 4h7p as template\n",
    "# you could try more templates\n",
    "from modeller import *\n",
    "\n",
    "env = environ()\n",
    "aln = alignment(env)\n",
    "\n",
    "#update the following with the best selected template (less resolution value and better similarity/identity)\n",
    "mdl = model(env, file='4h7p', model_segment=('FIRST:A','LAST:A'))\n",
    "#define the pdb file of the template\n",
    "aln.append_model(mdl, align_codes='4h7p', atom_files='4h7p.pdb')\n",
    "aln.append(file='TvLDH.ali', align_codes='TvLDH')\n",
    "aln.align2d()\n",
    "aln.write(file='TvLDH-4h7pA.ali', alignment_format='PIR')\n",
    "aln.write(file='TvLDH-4h7pA.pap', alignment_format='PAP')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      ">P1;4h7p\r\n",
      "structureX:4h7p.pdb:   1 :A:+313 :A:MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED  YES:MOL_ID  1; ORGANISM_SCIENTIFIC  LEISHMANIA MAJOR; ORGANISM_TAXID  347515; STRAIN  FRIEDLIN; GENE  CMDH, LMJF_28_2860; EXPRESSION_SYSTEM  ESCHERICHIA COLI; EXPRESSION_SYSTEM_TAXID  469008; EXPRESSION_SYSTEM_STRAIN  BL21(DE3); EXPRESSION_SYSTEM_VECTOR_TYPE  PLASMID; EXPRESSION_SYSTEM_PLASMID  AVA0421: 1.30: 0.15\r\n",
      "MS-AVKVAVTGAAGQIGYALVPLIARGALLGPTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKVVVTADPR\r\n",
      "VAFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTNALILLKSAQGKLNP\r\n",
      "RHVTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVI---G-TTPAREAI------KDFVQ\r\n",
      "VVRGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVYSDE-NPYGVPSGLIFSFPCTCH-AGEWT\r\n",
      "VVSG-K----LKQRLASTIAELQEERAQA------GL*\r\n",
      "\r\n",
      ">P1;TvLDH\r\n",
      "sequence:TvLDH:     : :     : ::: 0.00: 0.00\r\n",
      "MSEAAHVLITGAAGQIGYILSHWIASGELYG-DRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGFVATTDPK\r\n",
      "AAFKDIDCAFLVASMPLKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTNCEIAMLHAK-NLKP\r\n",
      "ENFSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKEGKTQKVVDVLDHDYVFDTFFK\r\n",
      "KIGHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIPVPEGNPYGIKPGVVFSFPCNVDKEGKIH\r\n",
      "VVEGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQGG*\r\n"
     ]
    }
   ],
   "source": [
    "#Alignment between target and selected template\n",
    "!cat TvLDH-4h7pA.ali"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " _aln.pos         10        20        30        40        50        60\r\n",
      "4h7p      MS-AVKVAVTGAAGQIGYALVPLIARGALLGPTTPVELRLLDIEPALKALAGVEAELEDCAFPLLDKV \r\n",
      "TvLDH     MSEAAHVLITGAAGQIGYILSHWIASGELYG-DRQVYLHLLDIPPAMNRLTALTMELEDCAFPHLAGF \r\n",
      " _consrvd ** *  *  ********* *   ** * * *    * * **** **   *     ******** *\r\n",
      "\r\n",
      " _aln.p   70        80        90       100       110       120       130\r\n",
      "4h7p      VVTADPRVAFDGVAIAIMCGAFPRKAGMERKDLLEMNARIFKEQGEAIAAVAASDCRVVVVGNPANTN \r\n",
      "TvLDH     VATTDPKAAFKDIDCAFLVASMPLKPGQVRADLISSNSVIFKNTGEYLSKWAKPSVKVLVIGNPDNTN \r\n",
      " _consrvd * * **  **     *      * * *  * **   *  ***  **     *     * * *** ***\r\n",
      "\r\n",
      " _aln.pos  140       150       160       170       180       190       200\r\n",
      "4h7p      ALILLKSAQGKLNPRHVTAMTRLDHNRALSLLARKAGVPVSQVRNVIIWGNHSSTQVPDTDSAVI--- \r\n",
      "TvLDH     CEIAMLHAK-NLKPENFSSLSMLDQNRAYYEVASKLGVDVKDVHDIIVWGNHGESMVADLTQATFTKE \r\n",
      " _consrvd   *    *   * *        ** ***    * * ** *  *   * ****    * *   *\r\n",
      "\r\n",
      " _aln.pos    210       220       230       240       250       260       270\r\n",
      "4h7p      G-TTPAREAI------KDFVQVVRGRGAEIIQLRGLSSAMSAAKAAVDHVHDWIHGTPEGVYVSMGVY \r\n",
      "TvLDH     GKTQKVVDVLDHDYVFDTFFKKIGHRAWDILEHRGFTSAASPTKAAIQHMKAWLFGTAPGEVLSMGIP \r\n",
      " _consrvd * *               *      *   *   **  ** *  ***  *   *  **  *   ***\r\n",
      "\r\n",
      " _aln.pos      280       290       300       310       320       330\r\n",
      "4h7p      SDE-NPYGVPSGLIFSFPCTCH-AGEWTVVSG-K----LKQRLASTIAELQEERAQA------GL \r\n",
      "TvLDH     VPEGNPYGIKPGVVFSFPCNVDKEGKIHVVEGFKVNDWLREKLDFTEKDLFHEKEIALNHLAQGG \r\n",
      " _consrvd   * ****   *  *****     *   ** * *    *   *  *   *  *   *      *\r\n"
     ]
    }
   ],
   "source": [
    "#Alignment between target and selected template\n",
    "!cat TvLDH-4h7pA.pap"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Model building "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     23570051   23017.629    22.478\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     23570579   23018.145    22.479\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     23619179   23065.605    22.525\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     23632479   23078.594    22.538\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           TvLDH-4h7pA.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23644092   23089.934    22.549\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23645542   23091.350    22.550\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23648442   23094.182    22.553\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23654242   23099.846    22.558\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23779544   23222.211    22.678\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23780792   23223.430    22.679\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23782128   23224.734    22.680\n",
      "\n",
      "Read the alignment from file       : TvLDH-4h7pA.ali\n",
      "\n",
      "Total number of alignment positions:   337\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1       4h7p     313      1    4h7p.pdb MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  2      TvLDH     335      1       TvLDH\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           4h7p.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23877753   23318.117    22.772\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23877753   23318.117    22.772\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23877924   23318.285    22.772\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23878774   23319.115    22.773\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23880032   23320.344    22.774\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23881936   23322.203    22.776\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23881936   23322.203    22.776\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23882026   23322.291    22.776\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23884882   23325.080    22.778\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23884882   23325.080    22.778\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23885017   23325.211    22.779\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23889301   23329.395    22.783\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23889301   23329.395    22.783\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23889499   23329.590    22.783\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23895925   23335.863    22.789\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23895925   23335.863    22.789\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23896222   23336.154    22.789\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23905844   23345.551    22.798\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23905844   23345.551    22.798\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23906294   23345.990    22.799\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23920744   23360.102    22.813\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23920744   23360.102    22.813\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23921419   23360.762    22.813\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23943077   23381.910    22.834\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23943077   23381.910    22.834\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23944085   23382.895    22.835\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23976589   23414.637    22.866\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     23976373   23414.426    22.866\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    221     1  210   217      D     T   14.330\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24115625   23550.414    22.998\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24117225   23551.977    23.000\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24121205   23555.863    23.004\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24123205   23557.816    23.006\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24126205   23560.746    23.009\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24130705   23565.141    23.013\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24136277   23570.582    23.018\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24139077   23573.316    23.021\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24143277   23577.418    23.025\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24149577   23583.570    23.031\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24152349   23586.277    23.033\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     24153749   23587.645    23.035\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     24206187   23638.855    23.085\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     24463241   23889.883    23.330\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     24502777   23928.492    23.368\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2604   2670      0         0      1136\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     24615233   24038.312    23.475\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2604   2670   3623      4313      1136\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     24725713   24146.203    23.580\n",
      "patch_______> segment topology patched using RTF:   335  ;  GLY  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2605   2671   3625      4314      1137\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2605   2671   3625      4314      1137\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1935      670\n",
      "The following 1 residues contain 6-membered rings with poor geometries\n",
      "after transfer from templates. Rebuilding rings from internal coordinates:\n",
      "   <Residue 257 (type PHE)>\n",
      "openf___224_> Open           TvLDH.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24833028   24251.004    23.683\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24841220   24259.004    23.690\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24857604   24275.004    23.706\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24890372   24307.004    23.737\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24923140   24339.004    23.769\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24988676   24403.004    23.831\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25021444   24435.004    23.862\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25054212   24467.004    23.894\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25119748   24531.004    23.956\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25250820   24659.004    24.081\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25316356   24723.004    24.144\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25381892   24787.004    24.206\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25512964   24915.004    24.331\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25775108   25171.004    24.581\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25906180   25299.004    24.706\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     2671    3625    4132    1121\n",
      "              Total number of restraints before, now         :        0    11549\n",
      "make_re_422_> Number of previous, current restraints         :        0    11549\n",
      "make_re_423_> Number of previous, current selected restraints:        0    11549\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     26097434   25485.775    24.888\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.mdt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5        1  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      103  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      110  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      211  ASP\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      214  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      225  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      252  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      282  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      294  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      295  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        5      306  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6        1  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      103  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      110  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      211  ASP\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      214  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      225  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      252  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      282  HIS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      294  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      295  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        6      306  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7        1  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      103  MET\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      110  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      214  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      225  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      252  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      294  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      295  GLN\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7      306  GLU\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       92  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8       98  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      106  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      146  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      179  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      206  ARG\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      210  LYS\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        8      294  LYS\n",
      "getdata_643_> Protein accepted:  4h7p\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    11549    11882\n",
      "make_re_423_> Number of previous, current selected restraints:    11549    11882\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     26097450   25485.791    24.888\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  4h7p\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      350\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :    11882    12216\n",
      "make_re_423_> Number of previous, current selected restraints:    11882    11866\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     26097450   25485.791    24.888\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  4h7p\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     26228522   25613.791    25.013\n",
      "make_re_422_> Number of previous, current restraints         :    12216    12495\n",
      "make_re_423_> Number of previous, current selected restraints:    11866    12145\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     26228522   25613.791    25.013\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  4h7p\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    12495    12709\n",
      "make_re_423_> Number of previous, current selected restraints:    12145    12359\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     26228522   25613.791    25.013\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  4h7p\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    12709    12793\n",
      "make_re_423_> Number of previous, current selected restraints:    12359    12443\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     26228522   25613.791    25.013\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  4h7p\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :    12793    12825\n",
      "make_re_423_> Number of previous, current selected restraints:    12443    12475\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     26978726   26346.412    25.729\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27240870   26602.412    25.979\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     27503014   26858.412    26.229\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28027302   27370.412    26.729\n",
      "make_re_422_> Number of previous, current restraints         :    12825    19748\n",
      "make_re_423_> Number of previous, current selected restraints:    12475    19398\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28289446   27626.412    26.979\n",
      "make_re_422_> Number of previous, current restraints         :    19748    26784\n",
      "make_re_423_> Number of previous, current selected restraints:    19398    26434\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    26784    29819\n",
      "make_re_423_> Number of previous, current selected restraints:    26434    29469\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "make_re_422_> Number of previous, current restraints         :    29819    30983\n",
      "make_re_423_> Number of previous, current selected restraints:    29469    30633\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    30983    30983\n",
      "make_re_423_> Number of previous, current selected restraints:    30633    30633\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    30983    30983\n",
      "make_re_423_> Number of previous, current selected restraints:    30633    30633\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    30983    30983\n",
      "make_re_423_> Number of previous, current selected restraints:    30633    30633\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    30983    30983\n",
      "make_re_423_> Number of previous, current selected restraints:    30633    30633\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    30983    30983\n",
      "make_re_423_> Number of previous, current selected restraints:    30633    30633\n",
      "rmdupl__427_>     2039 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    30983    28594\n",
      "openf___224_> Open           TvLDH.rsr\n",
      "openf___224_> Open           TvLDH.rsr\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28287068   27624.090    26.977\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     29073500   28392.090    27.727\n",
      "rdcsr2__307_> Number of restraints read    :    28594\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    28594    28594\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           TvLDH.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     29073524   28392.113    27.727\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    14472\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16215\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    17214\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    19883\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21676\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    24352\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25479\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    26682\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     29428624   28738.891    28.065\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    27946\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "iupac_m_486_> OE1/2 will be swapped:     -175.1559        3     3\n",
      "iupac_m_487_> NH1/2 swapped:     -179.4447       33    33\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.2473       36    36\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      -90.8777       41    41\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       97.0292       67    67\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      136.5205       81    81\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.6847      137   137\n",
      "iupac_m_485_> OD1/2 will be swapped:      139.2170      158   158\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.1809      163   163\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.3186      165   165\n",
      "iupac_m_485_> OD1/2 will be swapped:      131.0052      176   176\n",
      "iupac_m_485_> OD1/2 will be swapped:      118.4963      179   179\n",
      "iupac_m_486_> OE1/2 will be swapped:     -174.1823      202   202\n",
      "iupac_m_485_> OD1/2 will be swapped:     -162.5617      219   219\n",
      "iupac_m_485_> OD1/2 will be swapped:      -96.0788      231   231\n",
      "iupac_m_487_> NH1/2 swapped:      179.9115      236   236\n",
      "iupac_m_486_> OE1/2 will be swapped:      166.1054      263   263\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.0731      285   285\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.5009\n",
      "iupac_m_485_> OD1/2 will be swapped:     -177.5850      292   292\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -92.3627      303   303\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.4084      310   310\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.3906      315   315\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -164.5381      325   325\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLY:OXT  GLY\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     29783916   29085.855    28.404\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     29908956   29207.965    28.523\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     30159036   29452.184    28.762\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     30659196   29940.621    29.239\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     31659516   30917.496    30.193\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     33660156   32871.246    32.101\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     37661436   36778.746    35.917\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   530731\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -37831.7461\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -37831.746094\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /Applications/miniconda3/lib/modeller-9.25/modlib/surf5.de\n",
      "Pair library             : /Applications/miniconda3/lib/modeller-9.25/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.173000\n",
      "Sequence length          :          335\n",
      "Compactness              :     0.319460\n",
      "Native energy (pair)     :  -255.263596\n",
      "Native energy (surface)  :   -12.806508\n",
      "Native energy (combined) :    -7.490397\n",
      "Z score (pair)           :    -8.366871\n",
      "Z score (surface)        :    -6.482712\n",
      "Z score (combined)       :   -10.188781\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5855\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1900.1821\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      18.481       1.000\n",
      " 2 Bond angle potential               :    3625       0      4   1.971   1.971      277.27       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     42  46.818  46.818      584.98       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      0   1.169   1.169      33.686       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5855       0      0   0.002   0.002      3.7067       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6923       1      6   0.260   0.260      164.28       1.000\n",
      "10 Distance restraints 2 (N-O)        :    7036       3     12   0.374   0.374      266.85       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       0      2   4.442   4.442      76.863       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      2  74.065  74.065      62.550       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      4  83.536  83.536      83.306       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      1  78.865  78.865      55.221       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  93.336  93.336      26.592       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3035       0      0   0.436   0.436      73.726       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      26     40  30.782  50.102      66.272       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1164       0      2   0.745   0.745      106.39       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990001\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   33205.2812\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22336 219D 170L N   O    1682 1304   10.21    6.73    3.48    7.31    6.73    3.48    7.31\n",
      "    2  24038 309L 179D N   O    2383 1372   12.81   10.31    2.50    5.53   10.31    2.50    5.53\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9163   3E   4A C   N      22   24 -169.50 -134.00   35.63    1.30  -68.20  101.31    8.63\n",
      "    1          4A   4A N   CA     24   25  143.99  147.00                  145.30\n",
      "    2   9191  31G  32D C   N     225  227  -88.19  -96.50   69.50    2.93  -63.30   88.76    9.73\n",
      "    2         32D  32D N   CA    227  228   45.20  114.20                  -40.00\n",
      "    3   9304 144K 145N C   N    1092 1094 -150.13 -119.90  106.97    5.46  -63.20  115.14   13.15\n",
      "    3        145N 145N N   CA   1094 1095   34.39  137.00                  -41.10\n",
      "    4   9359 199F 200T C   N    1524 1526 -164.74 -124.80   76.37    2.37  -63.20  149.20   25.18\n",
      "    4        200T 200T N   CA   1526 1527 -151.41  143.50                  -42.10\n",
      "    5   9364 204K 205T C   N    1562 1564 -158.71 -124.80   33.92    1.82  -63.20 -160.31   31.77\n",
      "    5        205T 205T N   CA   1564 1565  142.53  143.50                  -42.10\n",
      "    6   9372 212L 213D C   N    1624 1626 -118.88  -63.30   80.18   13.28  -63.30   80.18   13.28\n",
      "    6        213D 213D N   CA   1626 1627  -97.79  -40.00                  -40.00\n",
      "    7   9373 213D 214H C   N    1632 1634  -68.04  -67.60   18.70    1.43  -63.20  159.07   20.19\n",
      "    7        214H 214H N   CA   1634 1635  158.70  140.00                  -42.30\n",
      "    8   9374 214H 215D C   N    1642 1644   77.92   54.50  108.17    9.85  -63.30 -136.30   24.41\n",
      "    8        215D 215D N   CA   1644 1645  146.51   40.90                  -40.00\n",
      "    9   9375 215D 216Y C   N    1650 1652  -65.68  -98.40   32.76    1.25  -63.50  173.31   27.39\n",
      "    9        216Y 216Y N   CA   1652 1653  129.90  128.40                  -43.40\n",
      "   10   9377 217V 218F C   N    1669 1671  -97.35  -71.40   48.82    4.34  -63.20  147.65   18.35\n",
      "   10        218F 218F N   CA   1671 1672   99.35  140.70                  -44.30\n",
      "   11   9379 219D 220T C   N    1688 1690  -70.59  -78.10   72.11    2.84   55.90  132.24   12.60\n",
      "   11        220T 220T N   CA   1690 1691   78.08  149.80                   39.50\n",
      "   12   9432 272P 273E C   N    2098 2100   74.29   54.60   26.31    1.72  -63.60  152.55   26.17\n",
      "   12        273E 273E N   CA   2100 2101   24.95   42.40                  -40.30\n",
      "   13   9452 292D 293K C   N    2248 2250   61.56   56.60   42.29    2.66  -62.90  129.95   22.48\n",
      "   13        293K 293K N   CA   2250 2251   -3.40   38.60                  -40.80\n",
      "   14   9453 293K 294E C   N    2257 2259  -71.96  -63.60   18.60    2.16   54.60  142.77   18.71\n",
      "   14        294E 294E N   CA   2259 2260  -23.69  -40.30                   42.40\n",
      "   15   9461 301E 302G C   N    2320 2322  -59.69  -62.40   16.33    2.58   82.20  145.82   10.80\n",
      "   15        302G 302G N   CA   2322 2323  -25.10  -41.20                    8.50\n",
      "   16   9464 304K 305V C   N    2344 2346  -60.88  -62.40    1.60    0.27 -125.40 -173.69   10.46\n",
      "   16        305V 305V N   CA   2346 2347  -41.92  -42.40                  143.30\n",
      "   17   9465 305V 306N C   N    2351 2353   82.03  -71.20  165.66   13.19  -63.20 -175.89   21.45\n",
      "   17        306N 306N N   CA   2353 2354 -154.26  142.80                  -41.10\n",
      "   18   9467 307D 308W C   N    2367 2369 -148.61 -124.90   38.13    2.19  -63.00  179.47   19.75\n",
      "   18        308W 308W N   CA   2369 2370  113.54  143.40                  -44.20\n",
      "   19   9468 308W 309L C   N    2381 2383 -119.95 -108.50  155.46    8.52  -70.70  171.37   14.89\n",
      "   19        309L 309L N   CA   2383 2384  -22.54  132.50                  141.60\n",
      "   20   9487 327A 328L C   N    2547 2549  -44.20  -70.70   29.80    2.18  -63.50  170.26   24.79\n",
      "   20        328L 328L N   CA   2549 2550  127.96  141.60                  -41.20\n",
      "   21   9488 328L 329N C   N    2555 2557 -135.66 -119.90   54.47    2.08  -63.20  148.63   22.63\n",
      "   21        329N 329N N   CA   2557 2558 -170.86  137.00                  -41.10\n",
      "   22   9489 329N 330H C   N    2563 2565  -91.33  -63.20   81.50   12.03  -63.20   81.50   12.03\n",
      "   22        330H 330H N   CA   2565 2566 -118.79  -42.30                  -42.30\n",
      "   23   9490 330H 331L C   N    2573 2575  -63.35  -70.70    8.02    0.61  -63.50  179.60   24.95\n",
      "   23        331L 331L N   CA   2575 2576  138.39  141.60                  -41.20\n",
      "   24   9491 331L 332A C   N    2581 2583   71.95  -68.20  140.37   12.16  -62.50 -146.51   32.40\n",
      "   24        332A 332A N   CA   2583 2584  153.26  145.30                  -40.90\n",
      "   25   9492 332A 333Q C   N    2586 2588   59.19  -73.00  135.35   10.15  -63.80 -166.07   24.96\n",
      "   25        333Q 333Q N   CA   2588 2589  169.76  140.70                  -40.30\n",
      "   26   9493 333Q 334G C   N    2595 2597   97.98   78.70   63.92    1.10   82.20  125.44    6.87\n",
      "   26        334G 334G N   CA   2597 2598  132.95 -166.10                    8.50\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   15   32  115  234  284  257  339  411  455  416\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "openf___224_> Open           TvLDH.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     37368688   36492.859    35.638\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    14472\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16215\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    17214\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    19883\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21676\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    24352\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25479\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    26682\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    27946\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "iupac_m_486_> OE1/2 will be swapped:       96.8119        3     3\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -95.7645       30    30\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.8727       36    36\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      109.1433       41    41\n",
      "iupac_m_487_> NH1/2 swapped:      179.7593       48    48\n",
      "iupac_m_486_> OE1/2 will be swapped:     -118.3783       57    57\n",
      "iupac_m_485_> OD1/2 will be swapped:      109.7781       79    79\n",
      "iupac_m_486_> OE1/2 will be swapped:     -111.1902      113   113\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.2042      151   151\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      125.7425      158   158\n",
      "iupac_m_487_> NH1/2 swapped:     -176.8822      161   161\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.8233      165   165\n",
      "iupac_m_485_> OD1/2 will be swapped:      172.2781      176   176\n",
      "iupac_m_486_> OE1/2 will be swapped:     -145.5787      188   188\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       97.4158      199   199\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -129.8781      202   202\n",
      "iupac_m_485_> OD1/2 will be swapped:      161.3226      213   213\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.0558      216   216\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -178.7631      228   228\n",
      "iupac_m_487_> NH1/2 swapped:     -179.2517      236   236\n",
      "iupac_m_486_> OE1/2 will be swapped:      153.8417      263   263\n",
      "iupac_m_485_> OD1/2 will be swapped:      144.6108      292   292\n",
      "iupac_m_486_> OE1/2 will be swapped:      169.9760      294   294\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       92.7712      303   303\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      138.9842      307   307\n",
      "iupac_m_486_> OE1/2 will be swapped:       96.3623      311   311\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLY:OXT  GLY\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   531568\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -37554.0781\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -37554.078125\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /Applications/miniconda3/lib/modeller-9.25/modlib/surf5.de\n",
      "Pair library             : /Applications/miniconda3/lib/modeller-9.25/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.173000\n",
      "Sequence length          :          335\n",
      "Compactness              :     0.328916\n",
      "Native energy (pair)     :  -273.076866\n",
      "Native energy (surface)  :   -14.439221\n",
      "Native energy (combined) :    -7.956273\n",
      "Z score (pair)           :    -8.404028\n",
      "Z score (surface)        :    -6.836980\n",
      "Z score (combined)       :   -10.459860\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5916\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1849.7994\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      20.361       1.000\n",
      " 2 Bond angle potential               :    3625       0      4   2.006   2.006      284.84       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     51  47.470  47.470      596.26       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      1   1.178   1.178      34.915       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5916       0      0   0.003   0.003      5.3663       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6923       1      5   0.253   0.253      149.96       1.000\n",
      "10 Distance restraints 2 (N-O)        :    7036       3      9   0.354   0.354      254.05       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       0      1   4.602   4.602      82.564       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      1  75.111  75.111      65.397       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      2  76.056  76.056      82.050       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      0  75.067  75.067      59.240       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  90.289  90.289      23.207       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3035       0      0   0.387   0.387      51.435       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      29     41  28.222  51.756      70.391       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1164       0      0   0.651   0.651      69.764       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990002\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   33426.9844\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22336 219D 170L N   O    1682 1304    9.99    6.73    3.26    6.85    6.73    3.26    6.85\n",
      "    2  24038 309L 179D N   O    2383 1372   12.77   10.31    2.46    5.45   10.31    2.46    5.45\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9161   1M   2S C   N       7    9  -71.66  -64.10   45.12    2.90  -72.40  142.93    8.38\n",
      "    1          2S   2S N   CA      9   10    9.48  -35.00                  152.40\n",
      "    2   9191  31G  32D C   N     225  227  -78.96  -96.50   59.14    2.49  -63.30   98.97   11.35\n",
      "    2         32D  32D N   CA    227  228   57.72  114.20                  -40.00\n",
      "    3   9252  92K  93P C   N     696  698  -54.41  -58.70   18.01    1.18  -64.50  165.12   12.86\n",
      "    3         93P  93P N   CA    698  699  -47.99  -30.50                  147.20\n",
      "    4   9253  93P  94G C   N     703  705  -70.65  -62.40    9.94    1.48   82.20  159.11   11.95\n",
      "    4         94G  94G N   CA    705  706  -35.66  -41.20                    8.50\n",
      "    5   9304 144K 145N C   N    1092 1094 -157.65  -63.20  113.37   13.62  -63.20  113.37   13.62\n",
      "    5        145N 145N N   CA   1094 1095   21.60  -41.10                  -41.10\n",
      "    6   9359 199F 200T C   N    1524 1526 -145.25 -124.80   85.31    4.26  -63.20  131.51   14.03\n",
      "    6        200T 200T N   CA   1526 1527   60.68  143.50                  -42.10\n",
      "    7   9360 200T 201K C   N    1531 1533   54.99   56.60    4.88    0.47  -62.90  139.61   24.34\n",
      "    7        201K 201K N   CA   1533 1534   34.00   38.60                  -40.80\n",
      "    8   9361 201K 202E C   N    1540 1542  112.95 -117.80  133.97    5.81  -63.60 -133.54   38.80\n",
      "    8        202E 202E N   CA   1542 1543  101.53  136.80                  -40.30\n",
      "    9   9362 202E 203G C   N    1549 1551 -115.73 -167.20   71.34    1.55   78.70  179.26    6.86\n",
      "    9        203G 203G N   CA   1551 1552  125.20  174.60                 -166.10\n",
      "   10   9364 204K 205T C   N    1562 1564 -158.94 -124.80   34.26    1.74  -63.20 -163.55   31.35\n",
      "   10        205T 205T N   CA   1564 1565  146.37  143.50                  -42.10\n",
      "   11   9372 212L 213D C   N    1624 1626 -164.72  -96.50   77.42    3.15  -63.30  155.28   17.09\n",
      "   11        213D 213D N   CA   1626 1627   77.59  114.20                  -40.00\n",
      "   12   9373 213D 214H C   N    1632 1634 -132.15 -125.60   44.38    1.56  -63.20  151.59   23.43\n",
      "   12        214H 214H N   CA   1634 1635 -177.31  138.80                  -42.30\n",
      "   13   9375 215D 216Y C   N    1650 1652 -117.22 -124.30   15.24    0.60  -63.50  173.81   25.09\n",
      "   13        216Y 216Y N   CA   1652 1653  121.90  135.40                  -43.40\n",
      "   14   9377 217V 218F C   N    1669 1671 -118.83 -124.20    6.08    0.16  -63.20 -176.13   29.23\n",
      "   14        218F 218F N   CA   1671 1672  140.45  143.30                  -44.30\n",
      "   15   9378 218F 219D C   N    1680 1682 -105.11  -63.30   58.89    9.80  -63.30   58.89    9.80\n",
      "   15        219D 219D N   CA   1682 1683  -81.47  -40.00                  -40.00\n",
      "   16   9379 219D 220T C   N    1688 1690  -49.40  -78.10   78.27    2.61   55.90  111.77   10.29\n",
      "   16        220T 220T N   CA   1690 1691   76.98  149.80                   39.50\n",
      "   17   9432 272P 273E C   N    2098 2100   74.70   54.60   25.30    1.79  -63.60  153.82   26.40\n",
      "   17        273E 273E N   CA   2100 2101   27.03   42.40                  -40.30\n",
      "   18   9453 293K 294E C   N    2257 2259  -51.18  -63.60   16.91    2.88   54.60  127.52   16.65\n",
      "   18        294E 294E N   CA   2259 2260  -28.82  -40.30                   42.40\n",
      "   19   9462 302G 303F C   N    2324 2326   33.21  -71.40  116.98    6.64  -63.20  163.98   28.63\n",
      "   19        303F 303F N   CA   2326 2327   88.35  140.70                  -44.30\n",
      "   20   9465 305V 306N C   N    2351 2353  -73.00  -71.20    5.77    0.32  -63.20  170.90   21.91\n",
      "   20        306N 306N N   CA   2353 2354  148.28  142.80                  -41.10\n",
      "   21   9466 306N 307D C   N    2359 2361   51.10   54.50    5.33    0.29  -63.30  142.53   24.17\n",
      "   21        307D 307D N   CA   2361 2362   45.01   40.90                  -40.00\n",
      "   22   9467 307D 308W C   N    2367 2369   99.07 -124.90  136.96    6.43  -63.00 -123.89   40.11\n",
      "   22        308W 308W N   CA   2369 2370  127.51  143.40                  -44.20\n",
      "   23   9468 308W 309L C   N    2381 2383 -103.22 -108.50  160.40    8.61  -70.70  172.51   14.34\n",
      "   23        309L 309L N   CA   2383 2384  -27.82  132.50                  141.60\n",
      "   24   9487 327A 328L C   N    2547 2549  -62.38  -70.70    9.93    0.69  -63.50  177.40   24.70\n",
      "   24        328L 328L N   CA   2549 2550  136.19  141.60                  -41.20\n",
      "   25   9488 328L 329N C   N    2555 2557 -155.24 -119.90   68.10    2.21  -63.20  154.18   24.27\n",
      "   25        329N 329N N   CA   2557 2558 -164.79  137.00                  -41.10\n",
      "   26   9490 330H 331L C   N    2573 2575  -79.61  -70.70   23.59    1.50  -63.50  156.19   22.64\n",
      "   26        331L 331L N   CA   2575 2576  163.44  141.60                  -41.20\n",
      "   27   9491 331L 332A C   N    2581 2583  -56.82  -68.20   34.23    2.37  -62.50  154.02   25.52\n",
      "   27        332A 332A N   CA   2583 2584  113.02  145.30                  -40.90\n",
      "   28   9492 332A 333Q C   N    2586 2588 -171.67 -121.10   77.94    2.63  -63.80  161.86   28.95\n",
      "   28        333Q 333Q N   CA   2588 2589 -160.99  139.70                  -40.30\n",
      "   29   9493 333Q 334G C   N    2595 2597   89.48   78.70   46.15    0.89   82.20  140.71    7.20\n",
      "   29        334G 334G N   CA   2597 2598  149.02 -166.10                    8.50\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   21   33  125  235  298  270  325  413  453  428\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "openf___224_> Open           TvLDH.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     37368688   36492.859    35.638\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    14472\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16215\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    17214\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    19883\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21676\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    24352\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25479\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    26682\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    27946\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "iupac_m_486_> OE1/2 will be swapped:      174.6857       28    28\n",
      "iupac_m_485_> OD1/2 will be swapped:      171.0874       32    32\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.8363       36    36\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -179.1677       48    48\n",
      "iupac_m_485_> OD1/2 will be swapped:     -128.6361       58    58\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.8487       67    67\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -115.4225       79    79\n",
      "iupac_m_487_> NH1/2 swapped:      177.2020       97    97\n",
      "iupac_m_485_> OD1/2 will be swapped:     -163.5495       99    99\n",
      "iupac_m_485_> OD1/2 will be swapped:     -179.5888      132   132\n",
      "iupac_m_486_> OE1/2 will be swapped:      132.1889      137   137\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.2305      149   149\n",
      "iupac_m_485_> OD1/2 will be swapped:      138.4257      158   158\n",
      "iupac_m_486_> OE1/2 will be swapped:      172.2850      165   165\n",
      "iupac_m_485_> OD1/2 will be swapped:      147.7248      173   173\n",
      "iupac_m_485_> OD1/2 will be swapped:      116.1501      176   176\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.5235      199   199\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -173.2430      202   202\n",
      "iupac_m_485_> OD1/2 will be swapped:      125.4037      215   215\n",
      "iupac_m_485_> OD1/2 will be swapped:       97.8518      219   219\n",
      "iupac_m_486_> OE1/2 will be swapped:      178.4730      234   234\n",
      "iupac_m_487_> NH1/2 swapped:     -179.9477      236   236\n",
      "iupac_m_486_> OE1/2 will be swapped:      139.4400      263   263\n",
      "iupac_m_486_> OE1/2 will be swapped:     -120.7608      301   301\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.4234      325   325\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLY:OXT  GLY\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   530170\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -37380.3633\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -37380.363281\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /Applications/miniconda3/lib/modeller-9.25/modlib/surf5.de\n",
      "Pair library             : /Applications/miniconda3/lib/modeller-9.25/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.173000\n",
      "Sequence length          :          335\n",
      "Compactness              :     0.321603\n",
      "Native energy (pair)     :  -265.580156\n",
      "Native energy (surface)  :   -15.459450\n",
      "Native energy (combined) :    -8.104230\n",
      "Z score (pair)           :    -8.293067\n",
      "Z score (surface)        :    -7.123631\n",
      "Z score (combined)       :   -10.403211\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5973\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1777.7605\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      18.992       1.000\n",
      " 2 Bond angle potential               :    3625       0      6   1.992   1.992      282.85       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     38  46.321  46.321      570.95       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      0   1.214   1.214      36.333       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5973       0      0   0.003   0.003      5.0021       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6923       1      7   0.264   0.264      158.05       1.000\n",
      "10 Distance restraints 2 (N-O)        :    7036       2     13   0.373   0.373      269.96       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       0      3   4.692   4.692      85.608       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      2  74.650  74.650      72.390       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      1  75.056  75.056      69.503       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      0  77.091  77.091      47.950       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  90.439  90.439      21.131       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3035       0      0   0.345   0.345      46.878       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      26     33  26.081  48.482      25.794       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1164       0      0   0.611   0.611      66.370       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990003\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   32008.1777\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22336 219D 170L N   O    1682 1304   10.12    6.73    3.39    7.11    6.73    3.39    7.11\n",
      "    2  24038 309L 179D N   O    2383 1372   12.70   10.31    2.39    5.29   10.31    2.39    5.29\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9163   3E   4A C   N      22   24 -177.95 -134.00   43.98    1.39  -68.20  109.80    9.19\n",
      "    1          4A   4A N   CA     24   25  148.66  147.00                  145.30\n",
      "    2   9191  31G  32D C   N     225  227  -76.97  -96.50   51.49    2.17  -63.30  107.43   12.49\n",
      "    2         32D  32D N   CA    227  228   66.56  114.20                  -40.00\n",
      "    3   9304 144K 145N C   N    1092 1094 -148.92 -119.90  108.67    5.52  -63.20  112.84   12.92\n",
      "    3        145N 145N N   CA   1094 1095   32.28  137.00                  -41.10\n",
      "    4   9359 199F 200T C   N    1524 1526 -160.98 -124.80   89.55    2.91  -63.20  134.59   22.93\n",
      "    4        200T 200T N   CA   1526 1527 -134.59  143.50                  -42.10\n",
      "    5   9361 201K 202E C   N    1540 1542 -119.85 -117.80   23.87    1.16  -63.60  168.77   25.68\n",
      "    5        202E 202E N   CA   1542 1543  160.58  136.80                  -40.30\n",
      "    6   9362 202E 203G C   N    1549 1551  -85.02 -167.20  159.12    4.57   78.70 -134.16   12.16\n",
      "    6        203G 203G N   CA   1551 1552   38.34  174.60                 -166.10\n",
      "    7   9363 203G 204K C   N    1553 1555  -64.55  -70.20    9.55    0.58  -62.90  173.51   22.44\n",
      "    7        204K 204K N   CA   1555 1556  132.70  140.40                  -40.80\n",
      "    8   9364 204K 205T C   N    1562 1564  -56.15 -124.80   82.11    3.01  -63.20  140.73   18.51\n",
      "    8        205T 205T N   CA   1564 1565   98.45  143.50                  -42.10\n",
      "    9   9372 212L 213D C   N    1624 1626 -105.28  -96.50   19.80    0.82  -63.30  142.77   15.58\n",
      "    9        213D 213D N   CA   1626 1627   96.46  114.20                  -40.00\n",
      "   10   9373 213D 214H C   N    1632 1634 -126.10 -125.60   24.42    0.92  -63.20  166.80   24.97\n",
      "   10        214H 214H N   CA   1634 1635  163.21  138.80                  -42.30\n",
      "   11   9375 215D 216Y C   N    1650 1652 -127.54 -124.30   36.22    1.78  -63.50  158.62   28.34\n",
      "   11        216Y 216Y N   CA   1652 1653  171.48  135.40                  -43.40\n",
      "   12   9378 218F 219D C   N    1680 1682 -132.08  -96.50   50.99    2.13  -63.30 -177.28   27.00\n",
      "   12        219D 219D N   CA   1682 1683  150.72  114.20                  -40.00\n",
      "   13   9379 219D 220T C   N    1688 1690 -100.42 -124.80   61.14    1.99   55.90  163.50   15.56\n",
      "   13        220T 220T N   CA   1690 1691   87.43  143.50                   39.50\n",
      "   14   9431 271V 272P C   N    2091 2093  -63.83  -58.70    6.27    0.52  -64.50  174.10   12.96\n",
      "   14        272P 272P N   CA   2093 2094  -26.90  -30.50                  147.20\n",
      "   15   9432 272P 273E C   N    2098 2100  -94.64 -117.80   35.59    1.20  -63.60  153.26   18.90\n",
      "   15        273E 273E N   CA   2100 2101  109.78  136.80                  -40.30\n",
      "   16   9453 293K 294E C   N    2257 2259  -53.73  -63.60   19.41    3.13   54.60  126.84   16.61\n",
      "   16        294E 294E N   CA   2259 2260  -23.60  -40.30                   42.40\n",
      "   17   9462 302G 303F C   N    2324 2326  -36.49  -71.40   54.31    3.24  -63.20  145.86   22.05\n",
      "   17        303F 303F N   CA   2326 2327   99.09  140.70                  -44.30\n",
      "   18   9464 304K 305V C   N    2344 2346  -66.32  -62.40    5.17    0.87 -125.40 -179.15   10.11\n",
      "   18        305V 305V N   CA   2346 2347  -45.77  -42.40                  143.30\n",
      "   19   9465 305V 306N C   N    2351 2353 -162.94 -119.90   98.64    3.39  -63.20  136.47   22.15\n",
      "   19        306N 306N N   CA   2353 2354 -134.25  137.00                  -41.10\n",
      "   20   9467 307D 308W C   N    2367 2369 -148.15 -124.90   25.60    1.37  -63.00 -163.68   21.55\n",
      "   20        308W 308W N   CA   2369 2370  132.70  143.40                  -44.20\n",
      "   21   9468 308W 309L C   N    2381 2383  -86.16 -108.50  121.92    6.33  -70.70  129.88   10.44\n",
      "   21        309L 309L N   CA   2383 2384   12.64  132.50                  141.60\n",
      "   22   9487 327A 328L C   N    2547 2549  -65.62  -70.70    8.32    0.52  -63.50  176.23   24.35\n",
      "   22        328L 328L N   CA   2549 2550  135.02  141.60                  -41.20\n",
      "   23   9488 328L 329N C   N    2555 2557 -131.50 -119.90   39.90    1.52  -63.20  159.13   23.75\n",
      "   23        329N 329N N   CA   2557 2558  175.17  137.00                  -41.10\n",
      "   24   9490 330H 331L C   N    2573 2575  -85.28  -70.70   70.86    4.84  -63.50  110.03   16.60\n",
      "   24        331L 331L N   CA   2575 2576 -149.06  141.60                  -41.20\n",
      "   25   9492 332A 333Q C   N    2586 2588  -57.39  -73.00   25.73    1.97  -63.80  158.67   23.00\n",
      "   25        333Q 333Q N   CA   2588 2589  161.16  140.70                  -40.30\n",
      "   26   9493 333Q 334G C   N    2595 2597  122.06   78.70   72.64    1.27   82.20  133.22    8.37\n",
      "   26        334G 334G N   CA   2597 2598  135.62 -166.10                    8.50\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   15   41  142  199  321  264  318  408  478  428\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "openf___224_> Open           TvLDH.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     37368688   36492.859    35.638\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    14472\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16215\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    17214\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    19883\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21676\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    24352\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25479\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    26682\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    27946\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.9492        3     3\n",
      "iupac_m_486_> OE1/2 will be swapped:      107.9309       55    55\n",
      "iupac_m_486_> OE1/2 will be swapped:      108.0409       57    57\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.6047       67    67\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -173.3420      132   132\n",
      "iupac_m_486_> OE1/2 will be swapped:      166.3814      137   137\n",
      "iupac_m_486_> OE1/2 will be swapped:     -144.6398      149   149\n",
      "iupac_m_485_> OD1/2 will be swapped:      139.2547      158   158\n",
      "iupac_m_487_> NH1/2 swapped:     -172.9116      161   161\n",
      "iupac_m_486_> OE1/2 will be swapped:      124.1904      202   202\n",
      "iupac_m_485_> OD1/2 will be swapped:      133.2833      213   213\n",
      "iupac_m_485_> OD1/2 will be swapped:     -139.6320      215   215\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.2418      221   221\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -165.2980      231   231\n",
      "iupac_m_486_> OE1/2 will be swapped:      171.8593      234   234\n",
      "iupac_m_487_> NH1/2 swapped:      178.5302      236   236\n",
      "iupac_m_486_> OE1/2 will be swapped:      170.8372      317   317\n",
      "iupac_m_485_> OD1/2 will be swapped:      177.4999      319   319\n",
      "iupac_m_486_> OE1/2 will be swapped:      177.6239      325   325\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLY:OXT  GLY\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   530154\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -37620.8477\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -37620.847656\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /Applications/miniconda3/lib/modeller-9.25/modlib/surf5.de\n",
      "Pair library             : /Applications/miniconda3/lib/modeller-9.25/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.173000\n",
      "Sequence length          :          335\n",
      "Compactness              :     0.304368\n",
      "Native energy (pair)     :  -275.373224\n",
      "Native energy (surface)  :   -14.975202\n",
      "Native energy (combined) :    -8.274359\n",
      "Z score (pair)           :    -8.699492\n",
      "Z score (surface)        :    -6.843819\n",
      "Z score (combined)       :   -10.475361\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5936\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1793.6669\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      20.071       1.000\n",
      " 2 Bond angle potential               :    3625       0      5   1.947   1.947      269.82       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     44  46.743  46.743      573.50       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      0   1.146   1.146      33.104       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5936       0      0   0.002   0.002      4.0786       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6923       1      6   0.251   0.251      162.71       1.000\n",
      "10 Distance restraints 2 (N-O)        :    7036       1     13   0.355   0.355      280.62       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       0      2   4.509   4.509      78.500       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      1  69.943  69.943      53.946       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      0  75.256  75.256      79.722       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      0  73.611  73.611      54.564       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  81.400  81.400      18.552       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3035       0      0   0.389   0.389      43.530       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      24     32  25.877  46.974      31.276       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1164       0      2   0.787   0.787      89.670       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990004\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990004\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   31970.9160\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22336 219D 170L N   O    1682 1304    9.91    6.73    3.18    6.67    6.73    3.18    6.67\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9162   2S   3E C   N      13   15  -56.04  -69.30   21.73    1.23  -63.60  165.75   22.52\n",
      "    1          3E   3E N   CA     15   16  125.28  142.50                  -40.30\n",
      "    2   9191  31G  32D C   N     225  227  -91.99  -96.50   57.81    2.43  -63.30  100.74   11.02\n",
      "    2         32D  32D N   CA    227  228   56.57  114.20                  -40.00\n",
      "    3   9249  89M  90P C   N     672  674  -56.22  -58.70   10.97    0.73  -64.50  171.82   13.26\n",
      "    3         90P  90P N   CA    674  675  -41.18  -30.50                  147.20\n",
      "    4   9250  90P  91L C   N     679  681   87.11  -70.70  159.93   13.63  -70.70  159.93   13.63\n",
      "    4         91L  91L N   CA    681  682  115.65  141.60                  141.60\n",
      "    5   9304 144K 145N C   N    1092 1094 -154.99  -63.20  114.01   13.42  -63.20  114.01   13.42\n",
      "    5        145N 145N N   CA   1094 1095   26.52  -41.10                  -41.10\n",
      "    6   9359 199F 200T C   N    1524 1526 -164.59 -124.80   61.54    1.93  -63.20  162.87   27.11\n",
      "    6        200T 200T N   CA   1526 1527 -169.56  143.50                  -42.10\n",
      "    7   9363 203G 204K C   N    1553 1555  -61.68  -70.20   44.67    3.03  -62.90  137.35   17.90\n",
      "    7        204K 204K N   CA   1555 1556   96.55  140.40                  -40.80\n",
      "    8   9364 204K 205T C   N    1562 1564 -167.14 -124.80  120.65    4.04  -63.20  120.74   20.80\n",
      "    8        205T 205T N   CA   1564 1565 -103.52  143.50                  -42.10\n",
      "    9   9372 212L 213D C   N    1624 1626 -112.05  -96.50   34.65    1.44  -63.30  132.53   14.20\n",
      "    9        213D 213D N   CA   1626 1627   83.24  114.20                  -40.00\n",
      "   10   9373 213D 214H C   N    1632 1634 -122.95 -125.60   48.56    1.90  -63.20  143.45   21.83\n",
      "   10        214H 214H N   CA   1634 1635 -172.72  138.80                  -42.30\n",
      "   11   9375 215D 216Y C   N    1650 1652  -99.88  -98.40    6.43    0.67  -63.50 -178.26   27.06\n",
      "   11        216Y 216Y N   CA   1652 1653  134.66  128.40                  -43.40\n",
      "   12   9377 217V 218F C   N    1669 1671 -121.82 -124.20    2.94    0.08  -63.20 -176.28   29.40\n",
      "   12        218F 218F N   CA   1671 1672  141.58  143.30                  -44.30\n",
      "   13   9378 218F 219D C   N    1680 1682 -168.18  -96.50  113.87    4.78  -63.30  157.37   25.88\n",
      "   13        219D 219D N   CA   1682 1683 -157.32  114.20                  -40.00\n",
      "   14   9431 271V 272P C   N    2091 2093  -62.35  -58.70    8.21    0.48  -64.50  170.36   12.60\n",
      "   14        272P 272P N   CA   2093 2094  -23.15  -30.50                  147.20\n",
      "   15   9432 272P 273E C   N    2098 2100  -80.61  -69.30   55.68    4.62  -63.60  129.41   16.38\n",
      "   15        273E 273E N   CA   2100 2101   87.99  142.50                  -40.30\n",
      "   16   9453 293K 294E C   N    2257 2259  -52.45  -63.60   17.61    2.94   54.60  127.40   16.66\n",
      "   16        294E 294E N   CA   2259 2260  -26.68  -40.30                   42.40\n",
      "   17   9466 306N 307D C   N    2359 2361  -91.30  -70.90   62.84    2.24  -63.30  113.77   15.84\n",
      "   17        307D 307D N   CA   2361 2362 -150.27  150.30                  -40.00\n",
      "   18   9467 307D 308W C   N    2367 2369 -100.20 -124.90   24.76    1.05  -63.00  177.99   24.68\n",
      "   18        308W 308W N   CA   2369 2370  141.75  143.40                  -44.20\n",
      "   19   9487 327A 328L C   N    2547 2549  -71.39  -70.70   13.54    1.00  -63.50  163.87   23.21\n",
      "   19        328L 328L N   CA   2549 2550  155.12  141.60                  -41.20\n",
      "   20   9488 328L 329N C   N    2555 2557 -138.64 -119.90   50.59    1.83  -63.20  154.57   23.54\n",
      "   20        329N 329N N   CA   2557 2558 -176.01  137.00                  -41.10\n",
      "   21   9490 330H 331L C   N    2573 2575  -70.59  -70.70   18.20    1.38  -63.50  159.16   22.51\n",
      "   21        331L 331L N   CA   2575 2576  159.80  141.60                  -41.20\n",
      "   22   9491 331L 332A C   N    2581 2583  -59.62  -68.20   16.30    1.05  -62.50  172.36   28.39\n",
      "   22        332A 332A N   CA   2583 2584  131.44  145.30                  -40.90\n",
      "   23   9492 332A 333Q C   N    2586 2588  171.17 -121.10   93.72    3.02  -63.80  170.03   30.71\n",
      "   23        333Q 333Q N   CA   2588 2589 -155.52  139.70                  -40.30\n",
      "   24   9493 333Q 334G C   N    2595 2597   83.17   78.70   24.77    0.53   82.20  161.04    7.87\n",
      "   24        334G 334G N   CA   2597 2598  169.54 -166.10                    8.50\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   28   27  151  216  291  265  322  460  427  415\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990004.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "openf___224_> Open           TvLDH.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     37368688   36492.859    35.638\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     2605     2605        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    14472\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    16215\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    17214\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    18511\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    19883\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    21676\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    24352\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    25479\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    26682\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    27946\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     2605    28594\n",
      "iupac_m_486_> OE1/2 will be swapped:      134.0706        3     3\n",
      "iupac_m_487_> NH1/2 swapped:     -177.4750       33    33\n",
      "iupac_m_487_> NH1/2 swapped:     -178.1775       48    48\n",
      "iupac_m_485_> OD1/2 will be swapped:     -122.6552       58    58\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      109.8556       67    67\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -149.5151       79    79\n",
      "iupac_m_485_> OD1/2 will be swapped:      128.6406       81    81\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      107.2658      114   114\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:     -169.0163      132   132\n",
      "iupac_m_486_> OE1/2 will be swapped:      163.9004      137   137\n",
      "iupac_m_486_> OE1/2 will be swapped:     -146.2907      149   149\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      101.3717      151   151\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      129.8636      158   158\n",
      "iupac_m_487_> NH1/2 swapped:     -177.3663      161   161\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       93.4016      163   163\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      169.1013      176   176\n",
      "iupac_m_486_> OE1/2 will be swapped:     -170.8882      188   188\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      100.7414      199   199\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      153.4982      213   213\n",
      "iupac_m_485_> OD1/2 will be swapped:      166.4714      215   215\n",
      "iupac_m_487_> NH1/2 swapped:     -178.7722      228   228\n",
      "iupac_m_485_> OD1/2 will be swapped:      134.0089      231   231\n",
      "iupac_m_486_> OE1/2 will be swapped:     -169.5440      234   234\n",
      "iupac_m_487_> NH1/2 swapped:      177.0780      236   236\n",
      "iupac_m_486_> OE1/2 will be swapped:      105.0183      273   273\n",
      "iupac_m_485_> OD1/2 will be swapped:      158.8912      292   292\n",
      "iupac_m_486_> OE1/2 will be swapped:      121.5459      294   294\n",
      "iupac_m_485_> OD1/2 will be swapped:      172.3545      307   307\n",
      "iupac_m_487_> NH1/2 swapped:     -179.0026      310   310\n",
      "iupac_m_486_> OE1/2 will be swapped:      163.5353      311   311\n",
      "iupac_m_485_> OD1/2 will be swapped:      155.0902      314   314\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLY:OXT  GLY\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   535411\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -37535.8008\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -37535.800781\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /Applications/miniconda3/lib/modeller-9.25/modlib/surf5.de\n",
      "Pair library             : /Applications/miniconda3/lib/modeller-9.25/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    42.173000\n",
      "Sequence length          :          335\n",
      "Compactness              :     0.373468\n",
      "Native energy (pair)     :  -276.230166\n",
      "Native energy (surface)  :   -14.442289\n",
      "Native energy (combined) :    -8.133649\n",
      "Z score (pair)           :    -8.408126\n",
      "Z score (surface)        :    -6.830316\n",
      "Z score (combined)       :   -10.273971\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    28594   28594\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     6021\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1818.7632\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    2671       0      0   0.005   0.005      19.294       1.000\n",
      " 2 Bond angle potential               :    3625       0      6   1.975   1.975      280.11       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1743       0     46  46.923  46.923      584.90       1.000\n",
      " 4 Stereochemical improper torsion pot:    1121       0      0   1.191   1.191      34.924       1.000\n",
      " 5 Soft-sphere overlap restraints     :    6021       0      0   0.002   0.002      4.0752       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    6923       1      7   0.249   0.249      163.41       1.000\n",
      "10 Distance restraints 2 (N-O)        :    7036       2     10   0.343   0.343      240.71       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     334       1      4  10.588  10.588      92.800       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     279       0      2  74.330  74.330      71.349       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     214       0      1  79.737  79.737      78.976       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      84       0      0  78.032  78.032      52.812       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      32       0      0  84.818  84.818      23.556       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3035       0      0   0.407   0.407      51.680       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     333      19     34  27.055  43.604      52.805       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1164       0      2   0.633   0.633      67.361       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.V99990005\n",
      "# Heavy relative violation of each residue is written to: TvLDH.V99990005\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   32319.8047\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22336 219D 170L N   O    1682 1304   10.15    6.73    3.42    7.18    6.73    3.42    7.18\n",
      "    2  22337 219D 171G N   O    1682 1308   13.56   10.25    3.31    4.59   10.25    3.31    4.59\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9802 309L 309L CA  C    2384 2389   -7.05 -179.18  172.13   40.00 -179.18  172.13   40.00\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9162   2S   3E C   N      13   15  -53.87  -69.30   19.94    1.12  -63.60  170.44   23.26\n",
      "    1          3E   3E N   CA     15   16  129.86  142.50                  -40.30\n",
      "    2   9191  31G  32D C   N     225  227  -74.10  -96.50   66.32    2.79  -63.30   92.42   10.79\n",
      "    2         32D  32D N   CA    227  228   51.79  114.20                  -40.00\n",
      "    3   9304 144K 145N C   N    1092 1094 -156.37 -119.90  104.22    5.42  -63.20  123.12   14.08\n",
      "    3        145N 145N N   CA   1094 1095   39.37  137.00                  -41.10\n",
      "    4   9359 199F 200T C   N    1524 1526 -161.55 -124.80   50.62    1.66  -63.20  170.76   28.05\n",
      "    4        200T 200T N   CA   1526 1527  178.31  143.50                  -42.10\n",
      "    5   9361 201K 202E C   N    1540 1542 -120.90 -117.80   75.84    3.86  -63.60  116.41   13.45\n",
      "    5        202E 202E N   CA   1542 1543   61.03  136.80                  -40.30\n",
      "    6   9363 203G 204K C   N    1553 1555  -72.28  -70.20   54.83    4.02  -62.90  126.75   15.97\n",
      "    6        204K 204K N   CA   1555 1556   85.61  140.40                  -40.80\n",
      "    7   9364 204K 205T C   N    1562 1564 -176.53  -63.20  120.09   20.37  -63.20  120.09   20.37\n",
      "    7        205T 205T N   CA   1564 1565  -81.79  -42.10                  -42.10\n",
      "    8   9370 210D 211V C   N    1609 1611 -111.11 -125.40   49.58    2.06  -62.40  146.56   16.35\n",
      "    8        211V 211V N   CA   1611 1612   95.83  143.30                  -42.40\n",
      "    9   9372 212L 213D C   N    1624 1626 -155.63  -96.50   59.25    2.42  -63.30  176.55   18.78\n",
      "    9        213D 213D N   CA   1626 1627  110.49  114.20                  -40.00\n",
      "   10   9373 213D 214H C   N    1632 1634 -162.86 -125.60   37.27    1.30  -63.20 -155.41   32.02\n",
      "   10        214H 214H N   CA   1634 1635  139.03  138.80                  -42.30\n",
      "   11   9377 217V 218F C   N    1669 1671 -140.29 -124.20   75.27    3.11  -63.20  125.37   22.03\n",
      "   11        218F 218F N   CA   1671 1672 -143.17  143.30                  -44.30\n",
      "   12   9378 218F 219D C   N    1680 1682  -60.94  -63.30   92.41   11.27  -63.30   92.41   11.27\n",
      "   12        219D 219D N   CA   1682 1683 -132.38  -40.00                  -40.00\n",
      "   13   9432 272P 273E C   N    2098 2100   66.41   54.60   93.79    8.16  -63.60 -141.39   36.14\n",
      "   13        273E 273E N   CA   2100 2101  135.44   42.40                  -40.30\n",
      "   14   9452 292D 293K C   N    2248 2250   61.30   56.60   10.82    0.54  -62.90  142.40   24.87\n",
      "   14        293K 293K N   CA   2250 2251   28.85   38.60                  -40.80\n",
      "   15   9465 305V 306N C   N    2351 2353 -152.25 -119.90  101.40    3.80  -63.20  123.66   20.03\n",
      "   15        306N 306N N   CA   2353 2354 -126.90  137.00                  -41.10\n",
      "   16   9487 327A 328L C   N    2547 2549 -178.95 -108.50   71.03    3.28  -63.50 -148.46   36.43\n",
      "   16        328L 328L N   CA   2549 2550  141.54  132.50                  -41.20\n",
      "   17   9488 328L 329N C   N    2555 2557 -118.84  -63.20   87.03   13.85  -63.20   87.03   13.85\n",
      "   17        329N 329N N   CA   2557 2558 -108.02  -41.10                  -41.10\n",
      "   18   9489 329N 330H C   N    2563 2565  -78.09  -67.60   65.03    4.68  -63.20  114.49   15.27\n",
      "   18        330H 330H N   CA   2565 2566 -155.82  140.00                  -42.30\n",
      "   19   9492 332A 333Q C   N    2586 2588  134.16 -121.10  105.99    3.67  -63.80 -129.62   41.44\n",
      "   19        333Q 333Q N   CA   2588 2589  155.94  139.70                  -40.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   31   36  133  233  288  277  356  424  451  426\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           TvLDH.B99990005.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      335     2605     2605\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "TvLDH.B99990001.pdb           1900.18213   -37831.74609        1.00000\n",
      "TvLDH.B99990002.pdb           1849.79944   -37554.07812        1.00000\n",
      "TvLDH.B99990003.pdb           1777.76050   -37380.36328        1.00000\n",
      "TvLDH.B99990004.pdb           1793.66687   -37620.84766        1.00000\n",
      "TvLDH.B99990005.pdb           1818.76318   -37535.80078        1.00000\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "from modeller.automodel import *\n",
    "#from modeller import soap_protein_od\n",
    "\n",
    "env = environ()\n",
    "a = automodel(env, alnfile='TvLDH-4h7pA.ali',\n",
    "#use the correct code used for template as well as for the target sequence\n",
    "              knowns='4h7p', sequence='TvLDH',\n",
    "              assess_methods=(assess.DOPE,\n",
    "                              #soap_protein_od.Scorer(),\n",
    "                              assess.GA341))\n",
    "a.starting_model = 1\n",
    "a.ending_model = 5\n",
    "a.make()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Model evaluation "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22876049   22339.891    21.816\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22876577   22340.406    21.817\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22925177   22387.867    21.863\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     22938477   22400.855    21.876\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23077729   22536.844    22.009\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23079329   22538.406    22.010\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23083309   22542.293    22.014\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23085309   22544.246    22.016\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23088309   22547.176    22.019\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23092809   22551.570    22.023\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23098381   22557.012    22.028\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23101181   22559.746    22.031\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23105381   22563.848    22.035\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23111681   22570.000    22.041\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23114453   22572.707    22.044\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     23115853   22574.074    22.045\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           TvLDH.B99990001.pdb\n",
      "readlinef__W> File: TvLDH.B99990001.pdb, Line: 6\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23126864   22584.828    22.055\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23126940   22584.902    22.056\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23127111   22585.070    22.056\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23127961   22585.898    22.057\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23129219   22587.129    22.058\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23131123   22588.988    22.060\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23131163   22589.027    22.060\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23131253   22589.113    22.060\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23134109   22591.902    22.062\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23134169   22591.961    22.062\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23134304   22592.094    22.063\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23138588   22596.277    22.067\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23138676   22596.363    22.067\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23138874   22596.557    22.067\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23145300   22602.832    22.073\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23145432   22602.961    22.073\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23145729   22603.250    22.073\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23155351   22612.648    22.083\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23155551   22612.844    22.083\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23156001   22613.281    22.083\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23170451   22627.395    22.097\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23170751   22627.688    22.097\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23171426   22628.346    22.098\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23193084   22649.496    22.119\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     23193532   22649.934    22.119\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23194540   22650.918    22.120\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     23227044   22682.660    22.151\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     23458916   22909.098    22.372\n",
      "read_mo_297_> Segments, residues, atoms:        1      335     2605\n",
      "read_mo_298_> Segment:        1     1   335     2605\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23469449   22919.383    22.382\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     23662886   23108.287    22.567\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     23666343   23111.664    22.570\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     23859780   23300.566    22.754\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     23899316   23339.176    22.792\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2604   2670      0         0      1136\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     24011772   23448.996    22.899\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2604   2670   3623      4313      1136\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     24122252   23556.887    23.005\n",
      "patch_______> segment topology patched using RTF:   335  ;  GLY  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2605   2671   3625      4314      1137\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      335   2605   2671   3625      4314      1137\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     2605        0\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  GLY:OXT  GLY\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     24128689   23563.172    23.011\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24483813   23909.973    23.350\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24546333   23971.027    23.409\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24671373   24093.137    23.528\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     24921453   24337.355    23.767\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     25421613   24825.793    24.244\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     26421933   25802.668    25.198\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     28422573   27756.418    27.106\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     32423853   31663.918    30.922\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      335\n",
      "Number of all, selected real atoms                :     2605    2605\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   530733\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -37831.5938\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "31 Non-bonded restraints              :  530399       0    645   6.873   6.873     -37832.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           TvLDH.profile\n",
      "# Energy of each residue is written to: TvLDH.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:     -11.6175\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -37831.593750\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "-37831.59375"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from modeller import *\n",
    "from modeller.scripts import complete_pdb\n",
    "\n",
    "log.verbose()    # request verbose output\n",
    "env = environ()\n",
    "env.libs.topology.read(file='$(LIB)/top_heav.lib') # read topology\n",
    "env.libs.parameters.read(file='$(LIB)/par.lib') # read parameters\n",
    "\n",
    "# read model file with better DOPE Score\n",
    "mdl = complete_pdb(env, 'TvLDH.B99990001.pdb')\n",
    "\n",
    "# Assess with DOPE:\n",
    "s = selection(mdl)   # all atom selection\n",
    "s.assess_dope(output='ENERGY_PROFILE NO_REPORT', file='TvLDH.profile',\n",
    "              normalize_profile=True, smoothing_window=15)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Template Evaluation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1374644    1342.426     1.311\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1375172    1342.941     1.311\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1423772    1390.402     1.358\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1437072    1403.391     1.370\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1576508    1539.559     1.503\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1578108    1541.121     1.505\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1582088    1545.008     1.509\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1584088    1546.961     1.511\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1587088    1549.891     1.514\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1591588    1554.285     1.518\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1597160    1559.727     1.523\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1599960    1562.461     1.526\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1604160    1566.562     1.530\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1610460    1572.715     1.536\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1613232    1575.422     1.538\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1614632    1576.789     1.540\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "io_data____W> Setting io.atom_files_directory to a colon-delimited string is\n",
      "              deprecated, as it is not robust on Windows systems. Set it to\n",
      "              a list of directories instead. For example:\n",
      "              env.io.atom_files_directory = ['./', '../atom_files']\n",
      "openf___224_> Open           4h7p.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1625643    1587.542     1.550\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1625719    1587.616     1.550\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1625890    1587.783     1.551\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1626740    1588.613     1.551\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1627998    1589.842     1.553\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1629902    1591.701     1.554\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1629942    1591.740     1.554\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1630032    1591.828     1.555\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1632888    1594.617     1.557\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1632948    1594.676     1.557\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1633083    1594.808     1.557\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1637367    1598.991     1.562\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1637455    1599.077     1.562\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1637653    1599.271     1.562\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1644079    1605.546     1.568\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1644211    1605.675     1.568\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1644508    1605.965     1.568\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1654130    1615.361     1.578\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1654330    1615.557     1.578\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1654780    1615.996     1.578\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1669230    1630.107     1.592\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1669530    1630.400     1.592\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1670205    1631.060     1.593\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1691863    1652.210     1.613\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1692311    1652.647     1.614\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1693319    1653.632     1.615\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1725823    1685.374     1.646\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1957695    1911.812     1.867\n",
      "read_mo_297_> Segments, residues, atoms:        1      313     2244\n",
      "read_mo_298_> Segment:        1     1 A   324 A     2244\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1967590    1921.475     1.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2141867    2091.667     2.043\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      2145126    2094.850     2.046\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      2319403    2265.042     2.212\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2353211    2298.058     2.244\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      313   2319   2358      0         0       934\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2452023    2394.554     2.338\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      313   2319   2358   3206      3773       934\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      2562519    2502.460     2.444\n",
      "patch_______> segment topology patched using RTF:   313  ;  LEU  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      313   2320   2359   3208      3775       935\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1      313   2320   2359   3208      3775       935\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     2244       76\n",
      "openf___224_> Open           ${LIB}/atmcls-mf.lib\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2201231    2149.640     2.099\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2201295    2149.702     2.099\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2201353    2149.759     2.099\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2201417    2149.821     2.099\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2201483    2149.886     2.099\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2201547    2149.948     2.100\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2201703    2150.101     2.100\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2201831    2150.226     2.100\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2202125    2150.513     2.100\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2202317    2150.700     2.100\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2202821    2151.192     2.101\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2203077    2151.442     2.101\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2204073    2152.415     2.102\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2204457    2152.790     2.102\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2205033    2153.353     2.103\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2207067    2155.339     2.105\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2207963    2156.214     2.106\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2212415    2160.562     2.110\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2213759    2161.874     2.111\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2223377    2171.267     2.120\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2225361    2173.204     2.122\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2246007    2193.366     2.142\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2249015    2196.304     2.145\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2294981    2241.192     2.189\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2299461    2245.567     2.193\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2400681    2344.415     2.289\n",
      "openf___224_> Open           ${LIB}/dist-mf.lib\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2401077    2344.802     2.290\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2401873    2345.579     2.291\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2402669    2346.356     2.291\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2403857    2347.517     2.292\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2404257    2347.907     2.293\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2404857    2348.493     2.293\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2405757    2349.372     2.294\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2407105    2350.688     2.296\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2407305    2350.884     2.296\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2409329    2352.860     2.298\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2409629    2353.153     2.298\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2412665    2356.118     2.301\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2413113    2356.556     2.301\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2417665    2361.001     2.306\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2418337    2361.657     2.306\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2425165    2368.325     2.313\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2426173    2369.310     2.314\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2436417    2379.313     2.324\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2437017    2379.899     2.324\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2437417    2380.290     2.325\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2438929    2381.767     2.326\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2454293    2396.771     2.341\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2455193    2397.649     2.341\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2455793    2398.235     2.342\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2458061    2400.450     2.344\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2481109    2422.958     2.366\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2482453    2424.271     2.367\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2483353    2425.149     2.368\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2486757    2428.474     2.372\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2521329    2462.235     2.405\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2523345    2464.204     2.406\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2524693    2465.521     2.408\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2529797    2470.505     2.413\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2581653    2521.146     2.462\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2584677    2524.099     2.465\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2586701    2526.075     2.467\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2594357    2533.552     2.474\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2672141    2609.513     2.548\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2676677    2613.942     2.553\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2679713    2616.907     2.556\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2691197    2628.122     2.567\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2807873    2742.063     2.678\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2814677    2748.708     2.684\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2819229    2753.153     2.689\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      2836457    2769.978     2.705\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3011473    2940.892     2.872\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3021685    2950.864     2.882\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3028513    2957.532     2.888\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3054353    2982.767     2.913\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3316877    3239.138     3.163\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3332189    3254.091     3.178\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3342433    3264.095     3.188\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3381193    3301.946     3.225\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3774977    3686.501     3.600\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3797945    3708.931     3.622\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3813309    3723.935     3.637\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      3871449    3780.712     3.692\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      4462125    4357.544     4.255\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      4496577    4391.188     4.288\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      4519625    4413.696     4.310\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      4606837    4498.864     4.393\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      5492853    5364.114     5.238\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      5544537    5414.587     5.288\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      5579109    5448.349     5.321\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      5709925    5576.099     5.445\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:      7038949    6873.974     6.713\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                                0       0       0       0   12561\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:      7707233    7526.595     7.350\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8055517    7866.716     7.682\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8111197    7921.091     7.735\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8222557    8029.841     7.842\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8445277    8247.341     8.054\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      8890717    8682.341     8.479\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:      9781597    9552.341     9.328\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     11563357   11292.341    11.028\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     15126877   14772.341    14.426\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      313\n",
      "Number of all, selected real atoms                :     2320    2320\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   460648\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -37543.5781\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "31 Non-bonded restraints              :  460352       0    323   6.942   6.942     -37544.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           4h7pA.profile\n",
      "# Energy of each residue is written to: 4h7pA.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:     -12.8942\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -37543.578125\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "-37543.578125"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from modeller import *\n",
    "from modeller.scripts import complete_pdb\n",
    "\n",
    "log.verbose()    # request verbose output\n",
    "env = environ()\n",
    "env.libs.topology.read(file='$(LIB)/top_heav.lib') # read topology\n",
    "env.libs.parameters.read(file='$(LIB)/par.lib') # read parameters\n",
    "\n",
    "# directories for input atom files\n",
    "env.io.atom_files_directory = './:../atom_files'\n",
    "\n",
    "# read model file\n",
    "mdl = complete_pdb(env, '4h7p.pdb', model_segment=('FIRST:A', 'LAST:A'))\n",
    "\n",
    "s = selection(mdl)\n",
    "s.assess_dope(output='ENERGY_PROFILE NO_REPORT', file='4h7pA.profile',\n",
    "              normalize_profile=True, smoothing_window=15)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting package metadata (current_repodata.json): done\n",
      "Solving environment: done\n",
      "\n",
      "## Package Plan ##\n",
      "\n",
      "  environment location: /Applications/miniconda3\n",
      "\n",
      "  added / updated specs:\n",
      "    - numpy\n",
      "\n",
      "\n",
      "The following packages will be downloaded:\n",
      "\n",
      "    package                    |            build\n",
      "    ---------------------------|-----------------\n",
      "    libblas-3.9.0              |       3_openblas          11 KB  conda-forge\n",
      "    libcblas-3.9.0             |       3_openblas          11 KB  conda-forge\n",
      "    liblapack-3.9.0            |       3_openblas          11 KB  conda-forge\n",
      "    numpy-1.19.4               |   py37h9ebeaaa_1         5.1 MB  conda-forge\n",
      "    ------------------------------------------------------------\n",
      "                                           Total:         5.1 MB\n",
      "\n",
      "The following packages will be REMOVED:\n",
      "\n",
      "  blas-2.2-openblas\n",
      "  liblapacke-3.9.0-2_openblas\n",
      "  numpy-base-1.19.2-py37h68fea81_0\n",
      "\n",
      "The following packages will be UPDATED:\n",
      "\n",
      "  libblas                                  3.9.0-2_openblas --> 3.9.0-3_openblas\n",
      "  libcblas                                 3.9.0-2_openblas --> 3.9.0-3_openblas\n",
      "  liblapack                                3.9.0-2_openblas --> 3.9.0-3_openblas\n",
      "  numpy              pkgs/main::numpy-1.19.2-py37h63973fd_0 --> conda-forge::numpy-1.19.4-py37h9ebeaaa_1\n",
      "\n",
      "\n",
      "\n",
      "Downloading and Extracting Packages\n",
      "libcblas-3.9.0       | 11 KB     | ##################################### | 100% \n",
      "numpy-1.19.4         | 5.1 MB    | ##################################### | 100% \n",
      "libblas-3.9.0        | 11 KB     | ##################################### | 100% \n",
      "liblapack-3.9.0      | 11 KB     | ##################################### | 100% \n",
      "Preparing transaction: done\n",
      "Verifying transaction: done\n",
      "Executing transaction: done\n",
      "\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "#keep numpy and matplotlib updated for the following script to work\n",
    "conda update numpy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Matplotlib is building the font cache; this may take a moment.\n"
     ]
    }
   ],
   "source": [
    "import pylab\n",
    "import modeller"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     14237075   13903.394    13.578\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     14237603   13903.909    13.578\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     14286203   13951.370    13.624\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     14299503   13964.358    13.637\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v25}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           TvLDH-4h7pA.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     14301771   13966.573    13.639\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     14303221   13967.989    13.641\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     14306121   13970.821    13.643\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     14311921   13976.485    13.649\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     14437223   14098.851    13.768\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     14438471   14100.069    13.770\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     14439807   14101.374    13.771\n",
      "\n",
      "Read the alignment from file       : TvLDH-4h7pA.ali\n",
      "\n",
      "Total number of alignment positions:   337\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1       4h7p     313      1    4h7p.pdb MOL_ID  1; MOLECULE  MALATE DEHYDROGENASE; CHAIN  A, B; EC  1.1.1.37; ENGINEERED\n",
      "  2      TvLDH     335      1       TvLDH\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAnMAAAFzCAYAAABVWI+TAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/Il7ecAAAACXBIWXMAAAsTAAALEwEAmpwYAACY/ElEQVR4nO2dd3wc5Z3/38+qF1vNsuUuWzLNYAy2AWPccKUESAgk3JFAuCPlQnLJHceRS35pl0J6LgnJHXcJcOmEcpAEXLABY5ptjDEuGEu2bMuWZFmS1fs+vz+end2VtGW270rf9+u1r92dmZ15dnZ25jPfqrTWCIIgCIIgCKmJI9EDEARBEARBEMJHxJwgCIIgCEIKI2JOEARBEAQhhRExJwiCIAiCkMKImBMEQRAEQUhhRMwJgiAIgiCkMOmJHkA8mTBhgi4vL0/0MARBEARBEILy5ptvntFalwZbbkyJufLycnbt2pXoYQiCIAiCIARFKXXMznLiZhUEQRAEQUhhRMwJgiAIgiCkMCLmBEEQBEEQUpgxFTPni/7+fmpra+np6Un0UFKS7Oxspk2bRkZGRqKHIgiCIAhjkjEv5mpraxk3bhzl5eUopRI9nJRCa01TUxO1tbXMmjUr0cMRBEEQhDHJmHez9vT0UFJSIkIuDJRSlJSUiFVTEARBEBLImBdzgAi5CJB9JwiCIAiJRcRcEqCU4iMf+Yj7/cDAAKWlpVx//fUhrae8vJwzZ85EvIwgCIIgCKmDiLkkIC8vj3379tHd3Q3A5s2bmTp1aoJHJQiCIAhCKpAQMaeUKlZKbVZKHXY9F/lZbr1S6pBSqkopdb/X9FuUUvuVUk6l1ML4jTx2XHPNNfz1r38F4Pe//z233Xabe15zczM33XQT8+bN44orrmDv3r0ANDU1sXbtWi655BI+8YlPoLV2f+Y3v/kNl112GfPnz+cTn/gEg4OD8f1CgiAIgiDEhURZ5u4Htmit5wBbXO+HoJRKAx4ErgEuAG5TSl3gmr0P+ACwLaqjUio2Dxt8+MMf5g9/+AM9PT3s3buXyy+/3D3vK1/5Cpdccgl79+7lW9/6Fh/96EcB+NrXvsZVV13FW2+9xQ033MDx48cBOHjwIH/84x955ZVX2LNnD2lpafz2t7+N6q4SBEEQBCE5SFRpkhuBFa7XjwIvAv86bJnLgCqt9REApdQfXJ87oLU+6JoWj7HGhXnz5lFTU8Pvf/97rr322iHztm/fzhNPPAHA1VdfTVNTE62trWzbto0nn3wSgOuuu46iImPg3LJlC2+++SaLFi0CoLu7m4kTJ8bx2wiCIAjCKOWNN+Cxx2DpUrjppkSPBkicmJukta4D0FrXKaV8KY2pwAmv97XA5T6WC4hS6uPAxwFmzJgReGEvN2UiuOGGG7j33nt58cUXaWpqck/XPsZlCVlfglZrzR133MG3v/3t2A1WEARBEMYiO3fCD38I3d1JI+Zi5mZVSj2vlNrn43Gj3VX4mBay2tJaP6S1Xqi1XlhaWhrqx+PKXXfdxZe//GUuuuiiIdOXLVvmdpO++OKLTJgwgfHjxw+Z/txzz9HS0gLAqlWrePzxxzl9+jRgYu6OHTsWx28iCIIgCKOU+nrzXFaW2HF4ETPLnNZ6tb95SqkGpdRkl1VuMnDax2K1wHSv99OAU1EeZlIxbdo0/vEf/3HE9K9+9at87GMfY968eeTm5vLoo48CJpbutttu49JLL2X58uVuy+MFF1zAN77xDdauXYvT6SQjI4MHH3yQmTNnxvX7CIIgCMKoIwnFnPLlwov5RpX6HtCktX7AlaVarLW+b9gy6cB7wCrgJLAT+But9X6vZV4E7tVa77Kz3YULF+pdu4YuevDgQc4///xIvs6YR/ahIAiCMGa4/nr461/h6afhhhtiuiml1Jta66BVOxKVzfoAsEYpdRhY43qPUmqKUupZAK31AHAPsBE4CDxmCTml1PuVUrXAYuCvSqmNCfgOgiAIgiCMNZLQMpeQBAitdRPG4jZ8+ingWq/3zwLP+ljuKeCpWI5REARBEARhBJaYmzQpsePwQjpACIIgCIIg2MHphIYG81rEnCAIgiAIQorR3AwDA1BYCNnZiR6NGxFzgiAIgiAIdkjCeDkQMScIgiAIgmAPy8UqYk7wpqmpifnz5zN//nzKysqYOnWq+31fX19Ut7VixQqGl2YZzo9//GO6urqiul1BEARBGBUkqWUuUe28BBclJSXs2bMHMMWB8/PzuffeexM2nh//+Mfcfvvt5ObmJmwMgiAIgpCUJKmYE8tcEvLmm2+yfPlyFixYwLp166irqwOMZe3zn/88y5Yt4/zzz2fnzp184AMfYM6cOXzpS18CoKamhvPOO4877riDefPm8cEPftCnpe1Tn/oUCxcuZO7cuXzlK18B4Cc/+QmnTp1i5cqVrFy5EoBNmzaxePFiLr30Um655RY6OjritBcEQRAEIclIUjEnljkv1Nd8tYONHP0V+102tNZ85jOf4emnn6a0tJQ//vGPfPGLX+RXv/oVAJmZmWzbto3/+I//4MYbb+TNN9+kuLiYiooKPv/5zwNw6NAhfvnLX7JkyRLuuusufv7zn4+w9n3zm9+kuLiYwcFBVq1axd69e/nsZz/LD3/4Q1544QUmTJjAmTNn+MY3vsHzzz9PXl4e3/nOd/jhD3/Il7/85ejtHEEQBEFIFUTMCXbo7e1l3759rFmzBoDBwUEmT57snn+Dq3XIRRddxNy5c93zZs+ezYkTJygsLGT69OksWbIEgNtvv52f/OQnI8TcY489xkMPPcTAwAB1dXUcOHCAefPmDVnm9ddf58CBA+519fX1sXjx4th8cUEQBEFIdkTMJT+hWNBiNgatmTt3Lq+99prP+VlZWQA4HA73a+v9wMAAAEoNtTAOf3/06FG+//3vs3PnToqKirjzzjvp6enxOZY1a9bw+9//PqLvJAiCIAijgiQVcxIzl2RkZWXR2NjoFnP9/f3s378/pHUcP37c/fnf//73XHXVVUPmt7W1kZeXR0FBAQ0NDTz33HPueePGjaO9vR2AK664gldeeYWqqioAurq6eO+998L+boIgCIKQ0iRhKy8QMZd0OBwOHn/8cf71X/+Viy++mPnz5/Pqq6+GtI7zzz+fRx99lHnz5tHc3MynPvWpIfMvvvhiLrnkEubOnctdd93ldqMCfPzjH+eaa65h5cqVlJaW8sgjj3Dbbbcxb948rrjiCt59992ofE9BEARBSCn6++HMGXA4oLQ00aMZgtI68a7FeLFw4UI9vM7awYMHOf/88xM0ouhTU1PD9ddfz759++K2zdG2DwVBEARhBCdPwrRpxipnWehijFLqTa31wmDLiWVOEARBEAQhGEna/QFEzI06ysvL42qVEwRBEIQxQZImP4CIOUEQBEEQhOCImEtuxlLcYLSRfScIgiCMCUTMJS/Z2dk0NTWJKAkDrTVNTU1kZ2cneiiCIAiCEFuSWMyN+aLB06ZNo7a2lsbGxkQPJSXJzs5m2rRpiR6GIAiCIMQWEXPJS0ZGBrNmzUr0MARBEARBSGaSWMyNeTerIAiCIAhCUETMCYIgCIIgpDBJ2soLRMwJgiAIgiAEpqsL2tshMxMKCxM9mhGImBMEQRAEQQiEd/cHpRI7Fh+ImBMEQRAEQQhEEsfLgYg5QRAEQRCEwIiYEwRBEARBSGGSOPkBRMwJgiAIgiAEpqnJPJeWJnYcfhAxJwiCIAiCEIjmZvNcXJzYcfhBxJwgCIIgCEIgRMwJgiAIgiCkMCLmBEEQBEEQUhgRc4IgCIIgCCmMiDlBEARBEIQURsScIAiCIAhCiqK1R8wVFSV2LH4QMScIgiAIguCPzk7o74fcXMjOTvRofCJiThAEQRAEwR9JbpUDEXOCIAiCIAj+SfJ4ORAxJwiCIAiC4B8Rc4IgCIIgCCmMiDlBEARBEIQURsScIAiCIAhCCiNiThAEQRAEIYURMScIgiAIgpDCiJgTBEEQBEFIYVpazLOIuaEopYqVUpuVUoddzz4r8Sml1iulDimlqpRS93tN/55S6l2l1F6l1FNKqcK4DV4QBEEQhLGDWOb8cj+wRWs9B9jiej8EpVQa8CBwDXABcJtS6gLX7M3AhVrrecB7wBfiMmpBEARBEMYW0gHCLzcCj7pePwrc5GOZy4AqrfURrXUf8AfX59Bab9JaD7iWex2YFtvhCoIgCIIwJhHLnF8maa3rAFzPE30sMxU44fW+1jVtOHcBz/nbkFLq40qpXUqpXY2NjREMWRAEQRCEMYXWcOaMeZ3EYi49VitWSj0PlPmY9UW7q/AxTQ/bxheBAeC3/laitX4IeAhg4cKF2t9ygiAIgiAIQ2hvh54eyM2F/PxEj8YvMRNzWuvV/uYppRqUUpO11nVKqcnAaR+L1QLTvd5PA055reMO4HpgldZaRJogCIIgCNGlvt48l5WB8mVjSg4S5WZ9BrjD9foO4Gkfy+wE5iilZimlMoEPuz6HUmo98K/ADVrrrjiMVxAEQRCEsYa3mEtiEiXmHgDWKKUOA2tc71FKTVFKPQvgSnC4B9gIHAQe01rvd33+Z8A4YLNSao9S6j/j/QUEQRAEQRjlpIiYi5mbNRBa6yZglY/pp4Brvd4/CzzrY7nKmA5QEARBEAQhRcScdIAQBEEQBEHwhYg5QRAEQRCEFKahwTyLmBMEQRAEQUhBxDInCIIgCIKQwoiYG4N8+tMwcya89lqiRyIIgiAIQqSImBuDNDfD8eNw+HCiRyIIgiAIQiQ4nZ6YuYm+uo4mDyLmokmlq2JKVVVixyEIgiAIQmQ0NcHgoOnJmpWV6NEERMRcNBExJwiCIAijgxRxsYKIuegiYk4QBEEQRgcpUpYERMxFl4oK8yxiThAEQRBSG7HMjVEmTYK8PGhpMckQgiAIgiCkJqdOmWcRc2MMpTyu1urqxI5FEARBEITwsa7js2Yldhw2EDEXbSRuThAEQRBSH+s6bl3XkxgRc9FGxJwgCIIgpD4i5sYwIuYEQRAEIbXp7YUTJyAtzXR2SnJEzEUbEXOCIAiCkNocPQpaQ3k5ZGQkejRBETEXbWbMMM+1tYkdhyAIgiAI4ZFCLlYQMRd9Jk0yz/X1RtULgiAIgpBaWGLOqh+b5IiYizZ5eTBuHPT1QWtrokcjCIIgCEKoiGVOcBcYtKpHC4IgCIKQOqSYmEtP9ABGJWVlcPiwEXPnnZfo0QiCMAp54sAT/GH/H9zv5xTP4RtXfwOHknt0QYgYq2CwiLkxjFjmBEGIIQPOAe7+89209LQMmb5m9hpWzlqZoFEJwiihvx9qakxXpxTo/gDiZo0N3kkQgiAIUWbnyZ209LQws2Amj33wMT54wQcB2Fi9McEjE4RRwPHjMDAA06dDdnaiR2MLEXOxQCxzgiDEEEu0ve+c93HL3Fv4+KUfHzJdEIQISLFMVhAxFxtEzAmCEEM2VG0AYH3legCWzlxKTnoOe+r3UN8h5x1BiIgUS34AiZmLDSLmBGEoWsPOndDSAnPmwOzZiR5RytLUeYYdJ3eQqTJYUTUAtVvJvvJKVpSv4Lmq59hUvYmPXvzR8Fbe1QWvvgqDg55pSsFll0FhYVTGLwhJj4g5ARAxJwjDefppeP/7zevcXDh5UsRBmGx+/AE0mqXV/eR95SYz8bOfZf3frOe5qud4/sjz4Yu5T34Sfv3rkdNXrIAXXgh3yIKQWqSgmBM3aywQMScIQ9m50/O6qwsOHkzcWFKcje/8HwDrO8rg8svNxF27WDB5AQDvnnk3/JVbv9NVV8HateaRkQEvvQRnzkQwakFIIVKsLAmImIsNEyea58bGoe4KQRirWHe6/t4LttBOJxscRwFY9/mfweOPmxlVVVQWmwtPVXOY+3ZwEI4cMa+few42bjSPZcuMm/z55yMdviAkP4ODHjEnCRBjnIwMmDABnE65mxUE8Ii3VauGvhdCYu/2x6nPczKl08GFV74fpkwxpRNOn2biYDb5mfm09LTQ3N0c+spra00bwrIyyM/3TF9vkizYsCE6X0IQkpmTJ83/YPJk054zRRAxFyvE1SoIBq094s0SBiLmwmLDyw8DsE5XoBwOcDjc1gN15Ehk1jl/cULWb7Zxo/ktBWE0k4JlSUDEXOwQMScIhjNnoK0NCgo8MV4i5kKias8WLv58Dl/vcpUkOfc6z0xLfEXqavUn5ubOhalTzbls797Q15usfOtbMGOGqfD/298mbBj9g/3c+qdb+feX/j1hY4gbLS2weLEpxjt9OsycCf/xH4ke1VBSMPkBbIg5pVSuUur/KaX+2/V+jlLq+tgPLcWRLhCCYPA+Oc6ZM3SaYIunN/2MvYU9dGXCxC7F2vd9zjPTW8wVxUDMKQXLl5vXu3eHvt5k5ac/hRMnTNum7343YcN46dhL/OnAn/jeq99Dj3bL55NPwuuvG5d+ba3ptPCLXyR6VEM5ftw8l5cndBihYscy9zDQCyx2va8FvhGzEY0WxDInCAZvt8WkSSYOpbnZPARbVHUcA+DrvUuo+eIZCifN9Mz0YZmrbqkOfSOBMvjOOce9jVFBR4c5N6elQU6OsTieOpWQoWysMl072vvaaexqTMgY4sZGV4eSb38bDh82r48cSa5EQeuaPXlyYscRInbEXIXW+rtAP4DWuhtQMR3VaEDEnCAYvEWCUp5YlOowBMcYpaqvAYAFky8lZ3zx0Jle+zMmblbvaaNFzFnH3jnneJJyNm1KyFA2VHsSS8LORE4FBgZg82bz+tZbzTE1ZYppan/iRGLH5k2D+a+5r+Epgh0x16eUygE0gFKqAmOpEwIhYk4QDMNFgvUsYs42VY4WACrLLhg500toVRQbYReyKPBOUvEV+G1NGy1izvuYXLfOvE5Atm5tWy37Tu/zDGs0i7kdO+Ds2aEdYJLxJsG6ZluhUimCnQ4QXwE2ANOVUr8FlgB3xnJQowIRc4IPtNb89p3fsmzmMmYUzEj0cOKDPzEXzRO41vDLX5qyAt5MmgQf/7jJ+kxRegd6OZ7ZjcMJ5dMvGrnA9OmmHNLJk0xJKyQ7PZvTnadp621jfNZ4exupq4Pubigp8d2Zw/s309pYWFMZ72PSytbdvNm4+9LS4jaMTdVDrYEhibnmZnPMd3WZIs+WhTFe7NkDzzwzNMM5LQ1uu833DYHlYrXEM5j9v22b+T1Wr47pcG1jXbNTzDIXUMwppRxAEfAB4AqMe/UftdZSPC0YIuYEHzx58Ek+8tRHWFuxlo23b0z0cOJDPMTc5s1w992+502eDDfeGL1txZmaszU4FZSfhcwp00cukJ5uMjLfew/H0RrmFM/hndPvsO/0Pq6cfqW9jbzzjnm2ElSGU1JispFbW012cmlpWN8lafA+JisrjfioroZduzwZ13FgQ5WxBl429TJ2nNwRmpj70Y/gG67w9exsU6Teuz5grLn1Vk/cmzcvvui7wLRl+bTEMySfZc7p9LhZU8wyF/B2VWvtBO7RWjdprf+qtf6LCDmbWGLOOjAEAfjr4b8C8GLNi3T0dSR4NHGgpQWamkw/Vus/EYsT+LPPmuc1a+DLXzaPa68dOi9FqW4yF8zKZvxfYLz26bKZy4CRVp+AWPFiVtbqcJQaXe7x4RX+E+BqHXAOsPmIiSG7Z9E9QIiWuf37Pa97eoyIihdVVUbIFRR4/m9f+pKxgG/bZhJMvDlzxrSKy8w0fX4tkk3MtbSYGL7CQiOQUwg7vofNSql7lVLTlVLF1iPmI0t1iovNHXNLC/RKiKFgXKzWnXjfYB8v1ryY2AHFA++LpuWai8UJ3LoIf+Ur8LWvmce//7tnXgqXfKiqNbXdKjszISvL90Je+3R9pbF8bKwOwfJrucC8rSYBtpHyDLcWJ6DLxc6TOznbc5bK4krWVKwBQsxCtr7Dda6ag/GM+fO2snn/3y6/3IihF14Yuvzzz5v/4LJlQ7sqJNsxlaIuVrAn5u4CPg1sA950PXbFclCjAofDcxct1jkBeOf0O9R11LnfW8JuVOMrQ3LqVCNKGhqgvT3ybdTUwKFDQ4sSA8yfb9yBx4/DuxE0n08wVXXGAlPpLPS/kJfVbEX5CjIcGew4uYOmrqbgGzhxwlh58vPhygBu2WS78IZLd7f5zunppmgtwMqVJu5wx464lcyx/v/rK9YzKW8SeRl5NHc322vF5p2w8ulPu1aYIDHnjT8Lp/XeO14OPJbRI0eMizPRjGYxp7We5eMxOx6DS3kkbk7wwqonNbd0rnkfiuUkVfFVu8zh8GSzReKy6+kxCQ9Ws/nVq80F2ns71sXj8cfNsnYfyXBhcVHVbNysFekT/S/klW2an5nP0plLcWonzx/xEbs0HMsqt2qVcYP5I5pirq/P7OeWFt/zY1l37OhR81xe7jle8vNh6VLzu/uK94oBVkmS9ZXrUUp5agQ22/hPeCesrFkDRUXmv/T662a/NtkQ8eHS2+uxvK1dO3SeJe6ee87zX6qt9W/5HT/e3HB1d8Pbbyfegj6axZxSKkMp9Vml1OOuxz1KqYx4DC7lkS4QghfWyfvflv4bRdlFVDVXcbTlaIJHFWP81S6LRvzV9u0wbRr8y7+Y975chNa0L3/ZLGv3cX3yNLl5r90UDK7M85H8YDFMaK2rMCLW1g2DFS833GoynGiVJ+nvhwsuMPu5pMRkRHrz+ONGZD31VGTb8Ye/Y9L6/htjf5PV1NXEzpM7yUzLZEX5CjOcUGoEet8kpacbQQemVda0aTBhAvzmNzEYOeZ/19UF8+aZOnHeLFxoQoyOHvX8l6ZPN9fAqVNNa7jhWL/DpZfCxz4WmzHbZTSLOeAXwALg567HAtc0IRhimRNcdPR18PKxl3EoB+sq1jG/bD4wyutKQXAxF4kwyMoyF5MpU2DRIvjAB0Yu8773GdehtZydh8NhLAunT4c/tihR21ZL9cBp8vrgnGI/maZgrEwOh3Ep9/ayeJpp2LO/cb//z1jsc9U5W7w48HLRsswdPeoRI1qP7Iv6y1+a51/9KrLt+MPfMWl9//029lmEbD6yGY1m6Yyl5GWaGLKQxNzw73DPPZ4ivCUlZlqsxJw/FyuY0iRf+IIRcd7/qWnT4P77fZe0uftuMx88RYUTRYrWmAN7deYWaa0v9nq/VSn1dqwGNKoQMSe4eOHoC/Q7+7li2hWU5JZQlm+OjfqOUX5sxFLMLV06sq7ccMaPh1deCW2969cb68ymTXD77eGPLwpYGalXH4XM+VP9L5iZaeK/jh6FmhrmTDXCL6gwGBwM3MbLm7Iyk5Xc1GSKv/qqR2cH6zefPdvESm3ebLoDpKcbd5uVlbl1q3Hp+Uv6CBd/xZHj2DfYHS9X6RFEbjHXEoaYW7rUUybk9GkjRl56yezPnJyojRvwXS/Om3vvNQ+7fOxj8NGPmnGeOgWdnUOTJOLJKLfMDbq6PgCglJoNRBTQ4MqI3ayUOux6LvKz3Hql1CGlVJVS6n6v6f+ulNqrlNqjlNqklJri6/MJR8Sc4MJyd1nurzEh5qz+l5mZxsXiTTIH0ycgs9Ef7ot+FcEvMF771HZA/cmTJn5t0qTgNcqiVZ7E+s3XrjXra2kxZSvAlLXo6TGvu7qMSy/a+BOvVt/gpib/sXxRQGvtPh94i7mKohC6dwTq1jFxonFZ9vQYQRdNTp40NQnz8mDJkuitNy3NE0d75Ej01hsqo1zM/QvwglLqRaXUS8BW4J8j3O79wBat9Rxgi+v9EJRSacCDwDXABcBtSimrl833tNbztNbzgb8AX45wPLFBxJzgYvid+JgQc9ZFc/bskVX1k1nMWRaHTZsSmgjhXYdsXShirrrafkB9oH6sgbYRye/mq/uCJZwtq09u7tDp0cTfd45T3+C9DXup76hn6rip7mQoILQEiGDWVGu/Rjv+z1rf1VdH32KaDOeE0SzmtNZbgDnAZ12Pc7XWLwT+VFBuBB51vX4UuMnHMpcBVVrrI1rrPuAPrs+htW7zWi4PV9/YpEPEnIC5065uqaYou4hFUxYBXmKucxQfG4EuODNmGLdaba1xBdll/35Tq+rLMbx/O+88M77GRnjrrdhtJwjuOmTtmVS0EDyOZ9jF0OrTGrB2WbKIuV/8Aq65Bh51XRaspJZoi7m+PlPKRinTNWM4cRAU1o3duop1qDfeMN1JrrmGqY9vJCsti4bOBtp7/ZTsOXDAxIZacY7BxNyjj5r96utx000mgzSkwQeIl4uUZBBzVhmxFBRzQWPmlFKfBn6rtd7rel+klPo7rfXPI9juJK11HYDWuk4p5Svnfipwwut9LeAuIqWU+ibwUaAVWBlg/B8HPg4wY0ace2FKFwgB2F23G4ClM5eS5jAWqjFhmaupMc++Lprp6SZov6rKuFV8Zbn54uBBePllU4ohVihlynQ8/DC89hosWBC7bQXAbc19zxXVMj1ANiuMyDatLLIRUG83Xs4i2mKuvNzE3jU2eoTCpElGzH3720a0dHREr03VsWPG2jpzpm/LUhy6XLx+8nUAVs1eBfd+xZ1N7Ni2jYrvzubAmYNUt1S7k6SG8OCDnizfGTNM1qovrrjCXH/q6wML4owM+NOf7A18YMBTtiVY5nM4JLrDSE+POQ7T0vzv1yTGTgLE3VrrB603WusWpdTdmMxWvyilngd8ydsv2hybr07Obguc1vqLwBeVUl8A7gG+4mslWuuHgIcAFi5cGF8LnrdlbjQ0pxbCwopZKsvz/B3GhJizLNLDyxdYVFaaC3t1tX0xFyhWKJqcf755TmDrKquUzbpDg3DJJabkQyCGCS1b2ZGhWuYiLU8yMOAR+bNnm6D3t94yIt3iootMTFZFhZleXQ0XX+xzdSET7PvGwTpk/R7n55ebmDaljLisqaHSWcgB1zI+xZyV5PCNb8Bdd/m/pmRkmDhEq+fucFpa4G//1ogzK/kkGDt3ms9ZvWyjTbRK34SLFas3a5a9/ZFk2BmxQymltDbV/FyxbAEqSxq01qv9zVNKNSilJruscpMBXzUAagHvW9FpwCkfy/0O+Ct+xFxCyc83sR9dXebucty4RI9ISACWmCvO8VyMLTHX0DGKrbbB4k/CuXCGakkKlwRfWNx1yHQaK2oG4Z9tuLWsAPKjR2FgIDZiLlKxc+KEqTM3daony7K83DyGY4m5qqroizl/YiTGYs6pne6YuIr9dSZbd+FC04Hie9+jsr4PMgP8Zta4PvhBmDw58MasOm/++OpXjTjcsSNw5w+LWLpYIfFu1lD/C0mGnQSIjcBjSqlVSqmrgd8DkQYyPAPc4Xp9B/C0j2V2AnOUUrOUUpnAh12fQynlXXDpBiA5e/UoJXFzgk8xV5JTQppKo6m7ib7BvkQNLbYEq9kUzsk7XifcBF9Y3HXIGrLI78OeWysnx1y8Bwbg+PHgYs67JZRdS8u0acY9WV8/spm6HUL5/WLxGwTbfoxFfF17Hd0D3ZTmljJ+8zYzcf16t0Cq2GfsFT5/s74+4yZ2OHyL31AJNWs7WEmSSJk507g4XbUS4068rP4xwo6Y+1dMxumnMD1atwD3RbjdB4A1SqnDwBrXe5RSU5RSzwJorQcw7tONwEHgMa21Vc3xAaXUPqXUXmAt8I8Rjid2SBeIMY8vMZfmSGNingkVPd2Z+OK0sWCwvo4ts+BJ5372n/ZRiDWZxZx3z8hYtpby4tjZYzx58EmePPgkj75tEgHW7e0yFv1gBX0tvPbp1PFTAwfU19cbr0Fxsf0YRO9WbKGWkDhwwBPvlSgxF8yyG6lYDYIl0iqLK4e2uFqyBPLyqNxfN2S5IdTUmHi/GTOik0kaSsZrU5Ox4GVmwooVkW/bF1atRK09LdfiyWi3zGmtnVrr/9RafxC4G3hNax3R2U1r3aS1XqW1nuN6bnZNP6W1vtZruWe11udorSu01t/0mn6z1vpCV3mS92mtg1QOTSBimRvz+BJzMPrj5h4bd4zVd8DNu/6Fhf+9kMbOxqELhHqx7u422a/p6eaCFkvy8owbq6/PbDPGOLWTpQ8v5ebHbubmx272ZDxWE7xnqjdeQeQO5WB2kRFeR1p8CC8rTi3Ui1c41qumJtOl4xeu5kFz5gRe3ntc0YxbDHbB9har70bf4WNlFldmlsGhQ1BQAJdfbsTZ1VdT6SoJePDMQfTwPqXRFhvLl5vjaudOOHMm8LKbNxuRtXRp9JJRfJFIi3i8QjhihJ3erC8qpcYrpYqBPcDDSqkfxnxkowURc2Mef2JuUr6x2o5KMdffz5v5HmtQz0CPu5uBm/JyE4pw7JgRTcGId4ByHC8su+t2c6LtBEXZRbz/vPfz/vPez9frz+eiBkJzaw0b87TxJmaqrqNu5LJbtphnO/FS3lhxWHU+1umPzZuNFXDaNLjzTrjjjqAfifr+Hxz0HEOWYPOFtT+s/RNF3Ja5hn4zYfVqz7G8bh2zWqCsP5vTnadHtmKLttjIyzNlfrQO3kbL24oYSxKZ0TraLXNAgauu2weAh7XWCwC/yQ3CMETMjXnGpGWusZEq19e1+oSOaPqelWUsbE6nJ8MxEPGOaYljEoRlibt17q08+aEnefIDf+T//e6kSekPRcwNG3PAYyzcgPZwzmmWGPjsZ03JFzt1vGbONELnxInQahH6w0q+mDIlcLuoGHYAcYs5lzt1yL5fvx4FrDtsClVbx4TnwzE4/u18V60982MVL2eRKMtcsPqDKYAdMZfuyji9FdNtQQgFEXNjHr9iLm8Ui7n6ereYu+eyewAj5px6WEeFUO7E4+0GiaOVYER7pzfegLY2OPfc0C4uwy6GfsVcQwPs3g3Z2cY6EwqhntO8xUAowjE93Qg6iE4MlV0xtGqVcbdu3w7tfor3hjsEl5ireMXl4vYWRxUVUFnJugPGSu1XzEXz+Le2v3Gj/24ne/ea33rKFLjwwuht2xeJyiK36g9Onx79zhZxwo6Y+zomCaFKa73T1Zv1cGyHNYoQMTem0VqPScucs76OI66Y+uvPuZ7p46dzuvM0e+r3DF0wlDvxeLtB4mQlONtzltdOvEa6I52rZ11tJoZrCfFuR+V0+j/GLLfa8uWhN2IP9ZwWiRiI5m9g9/gpKjJFdwcG4IVImx150Fp7LHO1XXDBBSMLQa9bx5ojoDS8fPxlOvs6Qx9/KMyda8rENDSY38kX3i7WWNdKTZRlLsVdrGAvAeJPrkSDf3C9P6K1vjn2QxsljJUuEKdPmxNDcbGpCdXUlOgRBUdr+NCHzJh9Pa6/PuLenN0D3fQO9pKVlkVOxtCLpnWh9RnPlOLUnTxEdwZMHMhmfNZ4t8VphLUhQWKuu7+bxb9cTPF3ioc8Kn5SwXtN7w3dzuEY3Lu++ab5v7zwAls2/ReDepAlx2H85HJz7H33u2a5UF2g48aZDPreXqit9S/mIqkZFqqY8xamoYqBaP4GoRw/MXC1NnY10t7XTqHOorgb3/t+/XomdMGiU4q+wT5evKLMcz56z3VcBor3CxWlgn/XWNeX82b2bDOmmhrjEo8XY0HMCREyVixzzz9vSg+0tJg7PKsEQTJz+DA89pgZs6/HX/9q4mwiwJ9VDjzB6SdaI9tGMlJ12riRKpT53paYGxE3lyAxt+vULl6vfZ2WnpYhjyMtR3h0j6s/qFXLK8JjwCc//an5v/zoRxx40bRTWnx0wHPs9febC1s4ZSAuusg8v/aabzHndLpbSIUVAxXqDWokwfNe3yViQnHTW67nKPbmdVvlzqb5j4W8+mooL2ddlclk3VDW4TkmrPm5uVEbE97j8FWipKPDuJsdDpOsEWuys4fUSowbr75qnq3jLQURMRdrrDpzDQ0RW3mSGutCW1honmMQPBx1rJPXBz9oLIneDyujLUJzfyAxZ6tCf4pSddbEOFVmm1Zeq2atIk2l8eqJV2ntafUsaFfMRblgqrXPb7ngFprua6LpviaeuPUJwNNGi6Ii0xaptdX0bYwWWnuOva1bqa81wnfKxz479Bg8dCh0FyjA2rXmecMGJuX5yJh+6y3Tg3LGDDjvvNDX7107c3j5jOG0t0cmBqzvsnlz5JaaUG4GrNIpUXT3DXGx5uT4jlXMzYXDh1n/0+cA2LiuYugxYfVGjSarV/uPEXzhBbPfL788tv2QvYl3RuvgYGQ3N0mCiLlYk5VlBM7AADQ3J3o0scM66X3qU+bZ6vmXzFiC8/rrR7pYzz3XzIvwZN7Sbe6ofYm5iXkTycvIo6WnxS36RgtVPab0Y+U4E8BekF3A4umLGXAOsPXoVs+Cw1pQ+cUqmBqlAGXrwnpB6QUU5xRTnFPMNZXXkJ2eze663aaQs8Mx9GYsWlgxZACdndSrLgDKLr5y6DEYbvkVL7dZmUvMNXR6jd/bbRZODFR2tjmn9fd7LEb+sMTAZZcF7y3ri1mz4JxzTDLIG2+E/nkLp9MjDuxkg06ebATXmTNw9mz42/XCauNV2YyxuGZn+14wPZ3Lzl9NYXYhh89WU02L55iIRcyad4zg1q1D58XTxWoR77i5N9801+ZZs0a3m1UpNUkp9Uul1HOu9xcopf4u9kMbRYwFV6v1x1u71tzVtrZGdvKNNT09nuBm6+7fmyidUAJZ5pRSo9Y6V+U0RUgrS891T1tf4cPVmptrAuP7+wO7M6Mc01LVMrQZPUBORg4rylcAeGrixeK/a10gHeb0W++qwVo2bkp01n/hhWaf1tdTdPgEGY4MzvacpWfAZV2MRlsmu/slGvXJQulU4I+6OlPeZMIEU6g3GEpF3ULkPuaaCbrv0x3prJ5tLJkjQhNigTWe4R6VeJUk8SbeYi7Sm5skwY5l7hFMNqt1pnkP+FyMxjM6GQtizvuuN4Z1mqLGyy+bk/vFF/tuWB0HMQcRuFq1hv/6L7jvPvP413+NTlxRlKhON+6aymmeGJR1leaCsKFqw9Dq9nYumlEuSzKkrZIX6yo8YwRi89+1RMnf/71ZtSXm8m3UXrODUu6Lr2PTZndx6oaOBnOT9eqrpgfmqlXhb8NOm0Kt4TnjLoyKmIvkfBLOzUC0xVyTSeKoaMHW/rBufn6x6xf8+0v/Tltvm3veEwee4L7N97kf9z9/f2SDs8bz1FOec8pnP2uKLBcXw8KFka0/FOJRnsT7ez7qipGNp/UxBtix40/QWj+mlPoCmJ6pSqn4NCscLYx2MdfWZrJZs7JMmvvq1SbA2woqTUaCWQy8SzxEgF0xZ7lgbPPaa/DJTw6d9oc/eApfJpgjuaZR9uxZC9zTLp18KcU5xRxrPcbJ9pPuBBAqK2HbNnPy9hdXFUXLnNbavb8rioa63NbMXgPA9uPbzYRo91b2jiH72tfgsceozz8LRFHMgTmuH37YuFpvK6O2rZb6jnpm7njTxAgtXWrPQuUPO+e0U6eM+7ywMDIxsHy5iV3ctcsE5IfTTsoqSh1KJmiULURVZ0w2amX2FOM6DsL6yvU4lIN9p/ex7/Q+0hxp/NvSf6O+o55bH791SM3GNJXGA6sfCH9wCxaYY72hAb73vaHzrrnGiP94EWvLXGMj3HLL0J7L2dmwcmVsthcn7Ii5TqVUCaABlFJXAK2BPyIMYbSLOW+rnMPhqZ2UzOVJgrkPvO8OtQ5bINm2zLWEeOJ69lnzvHo1rFljTsDHj5t+mxdcENZYo0V3axOt2ZA5ACWTPWLJoRyUF5bT3N3MqfZTQ8UcBD55R1HMNXU30drbyvis8UzInTBk3qwiU6C3rqMOrTUq2v9dK4Zs8WIoK6Nj45/pem4puRm55GdGseelV1B72ceMBa6+oz56bZns7BernMj550cmBnJzjQg7dMhYiubNC30d1rmotNT+Z6IoKpq7m2nuayWvDyZVzLN1Ppk6firP/s2zPHnwSR7a/RDPVT3Hvy39NzZVb8KpnVw86WL+5qK/AUAR4Q1cWpo5pwxPsMjIgNtui2zdoeJ9Iz04GH0huXmzWe9FF8Htt5tpV1xhyvqkMHbE3D8BzwAVSqlXgFLggzEd1WhjtIu54RdaK9A5WRM+TpyA/ftNS58lS3wvU1ho4mvOnDHxNlPCi2eKmZvVuij/8z+bC/O+ffDrXxuRmmAx13D8AACTetJQjqGRHD5LZYQi5qLQysjbxaqGXVRzM3IZnzWett42zvacpSja/91h8Wr1lWb9ZfllI8YSEcXFJung9dcpazUWnPqOuujFQNnZL9GMc6ysNGKuqio8MWedi0JJwoiiu887+UFVzrH9uXWV67hi2hX8as+veO3Ea5ztOesOAbjrkrv47OWfjXhsbi691DwSTX6+Ob7q6+HkSZN1HU2s/8Cdd8I//VN0151A7BQN3g0sB64EPgHM1Vr7KRUt+ETEXHJhpaGvWgWZmf6Xi8KdeXNPYDFnuflCEnOnTxuXU1aWp7xBEsUp1p88BEDZwMhsPauFWUOHV3ZlsP08MOBp5xSFgqn+4uXcY3QJzobOhugW/fYRQ2aJ2qi6WC1c2yg7bqxS9TX7jPW2tBQuuSSydds5p0UzzjHS/2I4Yi6Kljn3MddMyPujILuAxdMWM6gH2Vy92Z2cY8V3jkpiVZ7E6YxOAlASYieb9aPA3wALgEuB21zTBLuM9i4Qw8Vcfr4pq9DZaSrRJxt2rRPW93n1VfNdwiCYZW7q+KlkpWVxuvP0kADngHi3YrIKiK5ZY1w327ZBV1dYY40W9Q3mBFzGSLehT8vcsBZUIwjQIL27v5tBpyf2xamdVDVXUdVcNWS6xdmes+w8uROAyqLAYq6+oz66N2JVVUaUegWUW/vBqgcXVVxibtK+GrOtw64CuOvWuTNpwyYRljkI/+LuQ8wNOgfp7u/2/5lp08zNXl1d2P9/i+oWr7IkYewPq+j2t7d/m6buJsoLyzmnZGTcXUt3C6+dSJ5EqLCJRdxcV5dxJZ8+bX7bBHswoo2df/Qir8dS4KvADTEc0+hjtFvmht+BK+U5aQarQxVvtPbEhQSLG7K+zxe+YAKWwygca4m5omzfBTcdykFFsREztpMgfMU9lZaaIObeXnjppZDHGU3qm44BUJYx8jv7FHPjx5vx9/SYoPnhWG2MhrlYj7Ycpfi7xXxuw+fc0z7x508w56dzmPPTOXz0/4bec55sO8m0H07jJzt+YlZX7NtlO6TQbjT/u9bvtmaNOw4oppa5hQuhuJiyY8YyV1fvujBGwyJh7ZeTJ/0vEwsxF0XL3G1P3MbMH880NQV9kZbmsQQfORLedl1YlrmKCMXcW/VGkK+vWD/CLX+y7SQTvjeB6353HQPOJK/xGYxoi7neXlM79H3vM+9TvAyJL+y4WT/j9bgbuAQI4JsSRjDaxZyveKZkdbW2tJgioOPHB3fZ3XqruSBmZRmREcaJxbpQTMyb6HcZd0Zriw0xF8hNkCSu1vo2I8jKckYGm1tlMuo7h/0XAlleXnzRPA/LiHz5+Mv0DPTwv3v/l/7BfvoH+3nswGPu+U8dfGqI5eXP7/2Zzv5OCrMLWTJ9CdfNuc7n+P1a5oJ1OwiGjwKsMRVzaWmwZg0zXOlqNf2N5oWvuoqhcs45xmp18KCJKx2O1kkt5rTWbKzeSGNXI39976+x266LKle/38qzKqwOJvPL5nP7vNuZUzyHS8ou4VOLPjVimanjpzK7aDYtPS1u63PKEu3yJNu3Q22t8WRccgn8wz9EZ71JRDi29i7AfgSnYALpHQ5z0otn8+B40NVl7s7T04cGqiarmLMEta/acsM5/3zYudNTjyvEE4tTO92xYQHFXFEISRB79hg3wfTpZnzeRKO4ahSo7zICtmzcyH3st/F7oIumnyr01v5q623jjZNv8Hrt67T1tnHehPO4pOwSuge6efn4y57VuALHv7/m+2y/a7tbWAYcY36+ce12d49sdRQKvb2eItVeIjymYg5g/Xrj2gOqijR6waUw0f+xaJvcXBOvqbXH7e/N6dOmjEhRUXidH4Yzc6YRpydOhNdabZiYO9N1xh3W4G7f5otoibkzJrO3Mmda4DhdPziUg1+//9e895n32P2J3cyb5DsJxIqji0uh4VgSbcucdQ655x7YvTvymNEkxE7M3J+VUs+4Hn8BDgFPx35oo4i0NE9K/Gk/Jv1UxXI/zJo1tP1Qsou5SSHEKIV5YmnqamJQD1KcU0xWuv8WVCFltAaqVn755aZ22KFDnoSBBFDfb1zrZcUjs9BCFnP19UbA+uhl6b2/NlRtcIu1dRXr3G4pa1rfYB9bjm4x8ysDuxlHjDEalvXt282Nz7Ai1TEXc2vXUtwNRd3QmQkN666K3roDWYKj3LGDjAxj0dI6vGN7mJjzPnY2V2/2GV8JRMVC1N7bTkPPGbIGYOrk4PXlImH4cZ+yeMfRRmoRh8R0sogzdixz3wd+4Hp8G1imtb4/pqMajYxWV6u/k3ayi7myEC6eYRYQtnuhtmK3bIm5QJlY6emeorsJtM7VYyxYkyaNdGN7CyWfXSCGXzStzGMfvSy93dIbqze6rRHrK9ePsFC8euJVOvo6mFs611Pfzg9+xVwkCUx+LiYxF3NTpsC8eR7r3BVRFBPWd9m4cWTiSpQ7dgDhF/IeHDSdL5RyF0r2PnZaelrYecqPWzIKFiJrWxXN4AihLEk4rChfQWZaJjtO7qCpK4nrfAajqAhKSkziSaSJgydPmtJNgUpRjQLsxMy95PV4RWtdG4+BjTqSVcy9+qrp5fiDH8Bf/mIqw//oR/Y/70/MFbmC30eDmAvzhG43U9GWZe7ZZ02F8ldeCdyKybKWfP3rxpK1bBn8/OchjTtS6tOMG6xs6rkj5o3LHEdOeg5d/V109HV4ZvjbxwHuqK395VAOdp3axZt1b5Kdns3ymctZPH0x4zLHcaDxAMdbj7stFZblIhAjxFw0ukD4cRXHXMy5tmmJueopI8vFhM3cuabjS0ODuUj+93975kXbMue9rlCF1dmz5rmw0J144n3sAHzkqY/wsac/Rv/gsDCYKIi5SMqShEp+Zj5XzbgKjeb5I88H/0AM+P07v2f5I8tZ9vAyv49P//XTwVcU6b7XGu6913PuuPpqE/88SvEr5pRS7UqpNn+PeA5yVJCsYm7DBlNAt6bGxLhs325EnV383YGPJstchGIu2IV6RsEM0h3pnGw/6b9Uwre/bRIBBgeN9a2w0Pdy115rLFh1dab/7Msvw7/8S3hxRmGgnU7qc4zLatKMkan/SqnA5UmsjhtgvqtlmRsmgpq7m2nubiYvI48bz73RPf3aOdeSk5FDZlomq2YbwbuxymO1s1Oba0idOfAErO8Ns7xmb6+xDKSlDbEMOLXTnSATk9IkFrfcQmWLcclXtdZEb71KwQdd9eNffx2++EXPvJ0uS9fwuM5ICPfi7iOT1RJYt8+73f3+kT2PsPnIsPg/71i9MMsseRcMjrWYA09P14CxgDFCa829m+9l27FtvHz8Zb+PtxveDr4y69jZGWYyx/79xkixf795f/PN4a0nRfAr5rTW47TW44EfA/cDU4FpwL8C34jL6EYTySrmvMtcWLXKtm83ws4OY8HNWl5uEliOHw/phG5XzKU70plVaNpIHWnxUwLB2s9PPQVPPOF/ZVOmwLvvmvIkL71kail1dZnfNA60NZ6gNx3y+yC/2Pf39inmiouNQO3o8MSV7t5t2jCVl4/oZem+QBZX8psP/IaXP/YyL3/sZX79/l+7l7Euao++/Sh76veQk57D0plLg36H0txSFIrTnadNLJVlBQ3XdW25iSZNGhL83tLdQr+zn6LsooAxlRGzcCGVX/wBEEbbuGB897vGup+TY3petraaZBGrPI4/C3I4xEDM/f0lf8++T+3j7kvvBnzEmkUaq+e1rYoW4iLmrJjQjVUbh4YyxIF9p/dxqv0UZfllvHTnS34fP7v2Z8FXFmnIiGUNv+EGcy756Oguj2snZm6d1vrnWut2rXWb1voXwOiWuLEgGQsHnzlj7noyM01MUkmJaQHU1+cpBxEMf22WklXMWfs/FDGXmWnu0J1OT8NuO5tyWXbsuNACulo7O40Izcw0dZKGFc4dwcyZHherVVcpTuVK6l2tvMp6MvwuM8LyBeYmYvjFOkCyh3cXh9yMXK6acRVXzbiK3Ixc9zLWRe2VE68AsHLWSrLTg7sZM9IyKMktwamdnOk6Y4ozZ2WZrhuNjUE/PwI/NxBxcbG6qKi8DAijbVwwMjNNn1mrzE91tbEGd3eb1lDRyJy1iIGYqyyuZO7Eudw5/07AT+JAhO6+qmZXJmszUelgEoyLJl7E5PzJ1HXU8c7pd2K+PW+8wxmWzVzm9zG/bH7wla1da/73L71kjqeQB+P6LT/8YZO9Osrqyg3HjpgbVEr9rVIqTSnlUEr9LeAn9UfwSzJa5jZvNnecy5Z5BIJ3UHMwenuNtcrhGFk7KVmLBodjmYOwTuihXKwDijnLlT17duhNp0P5PaNA/SlTT6tsMMfvMkEzWq3vGyDZI1hLLoDywnLOLfHE7YXS/mjIGPPyApfhCEYQMeevREo0sfbT4abDsbHWeP92fuIDI2bWLHNBrqkJrcTTMDHX0t1CU3cTuRm57t/5sqmXUZBVwOHmwyOt45GKuUbT3q4ys8xYMGOMUiphWa3uJKSKKPz2ViH0np7QC6F3dpqbCqWMx2kMYEfM/Q1wK9DgetzimiaEQjKKOV8Xy1AKz9bUGGvVjBkjA0uT1TIXqZgLIZNuhJirrYVf/xp++9sRhVYD9miNJJh8yRIjRvbtM9uPMfWnzYWwzDHO7zK2ypO0tMBrr5kM3auvHrEOy10YSMzB0IQHO8kPfsdo/UfCsXD6sQbH0zI3MW8i+Zn5tPa2uruSRBXv3y5WvS+zs019xcFBcxNpl2Fizt1aq7jS3UUh3ZHOmgpz0d9YNezGJ4LyJN393dR21ZM+CDMmxbYsiTfWjUu8xNzJtpP879v/y8vHX8ahHKyevTo6Kw7nf/fmm/CtbxkP06JFps7rGMBONmuN1vpGrfUErXWp1vomrXVNHMY2ukg2Med0+r6DXrTIxC5VVQW/+AcSGcko5gYGjJtMKU/dP7uEcUIfcbH+wAdM3Mbtt8MnPjFkWbdlzldMUyRiLjMz8pivEDjVbC6ykzL9F4q19sep9mGtu6zvd+AAbNlijtElS0y3jmFYMXOWCPaHJeBmFc5iTrH9shB+xdzWrbbX4SaYmzUv9mJOKRVaPcNQsX67rVvN7zdunHG/xmo7oQgrPzXmhh87lgDaWjPsN47AMmdZ+WadhfSK+NXaX1OxBoVi+/Ht9A7Evj/2zY/dzB3/dwd9g30smrKIktyS6Kw41ELoDQ1w5ZVGzHl/fgwQKJv1PtfzT5VSPxn+iN8QRwnRKG8QTfbuNQf+1KmmxIBFejrMcZ10RpuYa2w0rrLS0qEFju0QqZv11CkTn2hZMDdsGJJMEfBCG2mZhzi6Wo90GDE3K3eq32XKC8vNssPdWVam59atphQL+D0Z23GzAqytWMt3Vn+HR256ZEQvy0BYAsst5s4/3wjjkyftJwdZ+ClUHU/LHITYNi7klbt+B6vv8erVJnkg2oRjJRtumfNKnvHmkjLTFeDQmUNDPx+BmItnWRJvinOKmVk4k35nPzVna2K6rfqOet44+QZZaVncOf9O/mP9f0Rv5VdcYWoDvvuuvXjlTZuMRW72bNOy6zOfid5YkpxAlrmDruddwJs+HkIoFBaai0F7u/HnJxrvLNbhFzm7yRqBCoMWFJj1nj1r3CLJQLguVgj5hN432EdTdxNpKo2SnBJPiY3Vq+Gii0ZkmJYXluNQDo63HqdvsG/oyvwlmdjFEkSbNxvrZAyp6jPHTGUAi5lf4VpZab5jS4txRYNPV117bzsNnQ1kpWUxdbx/0Qimjth9S+5j2cxlAZcbzgjLXCRN1/1Z5jrjLOZCaRsXKsOPzVhV2o+GZc6Pi94q3l3dUj00rtCK1Tt2LOR2jG4rYJzFHMRYvHuxqdqc266edTUP3/gwl0+7PHorT08PzbNgeZs+8xl48MEx42KFwKVJ/ux6ftR6AL8GnnK9FkJBqeTKaA3U3sSuSziQxSgtzVMLzSramWgiEXPWhfzoUVuCyKofVppXSpojbWgckY+4xKz0LGYUzMCpnSPvpCOtpj97trG2nj0LO3aEtw6bVCuT8FJZ5r++WMC6etbx2NdnLFkXXzxyG1ZF/eIKd9HXaOMWc51e/4FwLTRJkM0KIbaNC5Xp04da4pJZzPmx6hZmFzIhdwJd/V1D4zmtWL2BgdBi9fCKz0uEmIulePcilILcYWHX1ep0+q1LORaw05v1d0qp8UqpPOAAcEgp9S+xH9ooJFni5trbjVXI4fDU8vHGrks4mMUo2VytkYi5nByYNs32CX3IhXp48Vs/JyefF9ueHlOwNC3NlBwJlzi4WgecAxzN6gJg9oyRIswiYF0975PwunXmGB2GXRdrJLjLp3R43XiluJgLqW1cqKSnGwsWwHnnjcxujxYxFHPe03xajUPdLsPKkoRrWQ+TmIp3F4POQbdlLpRscbs0dTUxuMZ1jXr++cCW0d27TWLZzJlw7sjuM6MdO7e1F2it24CbgGeBGcBHYjmoUYtdMfenP5kA4sxMuPXW6I9j61YjSq64wtN2K9RxDgx4imj6q500msQchJTROuRCvWuX2QezZ5t1WBmm77xjYrBc+MxoPXLExPmVl7stHz9+/cfkfjOXrG9k8c8b/9ne2EPJUn7mGeMmz8zk2fPTKfiCIvPLisyvOCj+TjFbjmzx+bHjrccZcMC0VsiZWh5wE34vNCtXeiw8fqw77otxUezF3BALTTgXdK39HneWUBwVljnw7J9YNjO3BNGRI/bDN7zEXEdfB/Ud9WSlZfns0es3qzzMjNaq066yJI4Sc06PI/EQc7vrdtPU3UR5YTnnlEQ3W/fDj3+Y0u+VsiOt3sSstreb8+anPuVZ6Ne/hvx8c6283OXe9RU6NAawI+YylFIZGDH3tNa6H4hvWenRgl0x94tfmCDr/n4j7E6ciO44gpUOsDPO48eNoJs6FXJzfS+TbGLujTfM8zlhnnRCuJhbTa5LckqG7m+lTBLEypVmmpelzOfJ9xVT8JZ58wDTLuc/3vgPuge66Rvs4xe7fkHPQE/wsa9YYU54O3eOKIsygv/8T2hrg/5+/mv+IG3Z0J8G/Q5NS08LT737lM+PuetpNRO0WKzfC01+Ptx9t+lcce21Pj/rzmQtjp2lw6r95rflmF06Okx8ZE7OkIt5/2A/Z7rO4FAOJuTGJ65nyrgpZKdn09jVSFtvDDoy3norTJ4Md90V/XVb5OWZ80pfn/0allah55IS97Ezu2i2Txd9NC1zfYN9HOuoxeGE8pnzbX8uWsTUEuvi3TPvkpmWyfqK9SElGNmhNLcUjTZu3E9+0ljp+/vhf/7HnJ/AxMV1dprpTqe5Fo3yTg/+sCPm/guoAfKAbUqpmYD0Zg0HOzFzVn9Uh8MUKoXousa0hueeM6/9xRXYEXN2MiyTScz19ZlyF2Aqi4dDCBdz62JZkFXguwSMD1erzwvJsNjGw82HqTlbQ3FOMRdNvIjugW5ePvZy8LF7F761Mg590d3t7v7Rd7SKrfPyAThy9Eae+KNZxF9AdVWt6V1a2ZU1pG2VLwLW1XvwQdNP0U//Wbs15iKhOKeYdEc6LT0tntIO4VjmvGvMeV3sGrsa0Wgm5k00MZVxwKEc7v1uiZqocscdJmvbdeMRM0I5r/T2GtGXng4lJZ6EBD83An5LBIXx29ecrcGJZkYrZK29xvbnosXsIuMxOXr2KAPO2CQ+feTij9B8XzNfXfHVqK/bXfi4egN89rMm5GTJEmNE2LrVtPrbscOca5qbzW/d1mZKk4xB7NSZ+4nWeqrW+lptOAasjMPYRh92RNILL5i7jMsvh9tuM9Oi2Yrp8GGT4l1SYqprhztOO0H5ySTmXn3VCOW5c00wcziEcEJ3izmyjEUwI8NjjQOfGaYjss/6+z3Ca52n3yKYkhvXzjGWK6vqelDsFOC0WjHNn8+rnKCjr4O5pXOZteYWznMZ9Pzd6Ved2me+x2BB0KEErKsXhHjEzDmUg0l5xjrnbjtmNV2vrTUXFjsE6/6QF/vuD97Ew/UWc0I5r3j3xXU4PAkJflz07v/gcLEbRtHwqjOmG0pFCwkJyM/NyGXquKkMOAc40Rpl744XeZl5MelisqJ8BZlpmew8udN4OjIyht4EP/+8uTldutSEC2Vmht4hZxRhJwFiklLql0qp51zvLwDuiPnIRiN2RJK3Fce6+D7/fPRKSliWoLVr/R/43gkQ/lr/pJplLhpV6UMQc629rQCMP95gzP9XXTU0ZqaiwlOGY+dOwOtOusV1J/366+ZO0yugfEO1K3OsYn3oLXu8T4T+flevkjVDstTWrGF2CyhtLA79gyMDkastkZUe/MQerqjo7u+mtq2WdEc6MwpmhPTZUBkRNxdO0/UkqTFnMebE3LD9H+xGwHv/DClPYlnlq6ttx+pV7dtm1tmba8IGEkAq/955mXksnbEUjWbzEVcbPe/Y32AepjGGHTfrI8BGYIrr/XvA52I0ntFNIDF35Ah8/vPw+OPm/fr1JjvsnHOgtdUT7xUpgUqSWOTnm0dPjyc2YTh2xJyVXJEMYi4a/SK9T+hOZ8BF3Za5d10XfV/7e5ir1bqT7nf2c+Lb98P997s/+1bdW3zm2c/wwtEXAGOZu3L6leRn5rO/cb+9O++5c02MY329KRrtC6/9NETMTZxI9sULmNZmslaPt47M6K1yFQyuzAtu+QxYVy8A7or6hbNId4RY+DlEopIEkSSZrBapenH/86E/88SBJ8ybcCxzrv0fTMyV5JRQkFVAa28rTd1Nnhl5eSYesK9vSNJSIKoPmnjXyskXJiwgPxa/9/7T+/nY0x/jV2/9Kmrr9MeIG9ZLLzW142pqTDw5iJhzYUfMTdBaPwY4AbTWA0CSVIFNMQKV/PjSl+DHP4bTp81ylgs0lCzEYPT0GDcuBI8bCyQ8tYa33zavAyUTJItlrr0d9uwxZvilS8Nfz7hx5rfp7Q16QrfE3Pg9rtrbvk44Pn5bKyNsz29+YFzDAO97H59+9tP8bOfP6B7oZuGUhUweN5nMtEyunmX6llrlAQKiVGBX6/HjphVTfj51F5XzdsPb5KTncNWMq8z8NWtMcgMjLw6dfZ0c7m9Aaago8pPd7EVWehZTx03FqZ0huYDi4WK1iIqYe8+42pg2NHMyUWLOHasYhns7UbT2tHLzYzdzy59uMfstHMucTTGnlPL8B+v3DJ1pnev2DJvuh6omV1mSC66ytXwsiIWY21O/h0f2PBKXvq9rZpt+uduPuwqsOxxw3XXmdVeXCX3w7mA0hrEj5jqVUiW4MliVUlcArTEd1WjFn/tycNDj3nrgAeNWtVygofamC8T27e54KCZPtj/W4bz3nqmGPmECXHih/3VYJ127WWexwju+Lzs7snXZjJ1xu1nrz5oLia+g8BUrjOtuxw4TzAusLDdxdZsqgJtvhiefpHnxfN44+QYZjgwevPZBHvvgY+5VrK/wChK2Q6CbA+sYW7WKTcdfNOOZtZLsdNc+mzvXVLJn5MXhxZoX6WeQy05Cfpk992dpnumPG0rjd399NWNBVMScVV9wxYohk8UyZ58tR7fQ7+xHo81NS5hirru/mxNtJ0h3pDOz0H/NRvd/cPgNkhXzusnGjRNQ5TgLJImYi6J4j+cN1XkTzkOhhoZ2/OAH8N//bRKlNmwYk2VIfGFHzP0T8AxQoZR6BfhfYOw0PIsmeXnGutPXN7QrgncdsvvuGyqQli83pSx27fKk2IeLHRerRSDLnLWetWt9FnV1kyyWuUh7m3pj82LudrP24ilJMpz8fGMp9MowXe8qvLmhEvS//Au8//08f3QLTu3kqhlX8Q+L/oFZRbPcq1hXaZbfXL3ZXsba6tXmN3vlFWOx9MY7Xs4rNs/7u/uzzFl36euqsF3HrzjHHB+hiDl3AHscLXNDCgeHUp7k6FE4dMjU7LNqYLlIlJibXjCdDEcGp9pP0dXfFddth4uV9AOuZJ9Qwje8xNzRsybkYWbBzIAueus/NcLyZCeByMVAfy9H84z4mH1hBN6ACImFeI9HNrmF1RlnUA9yrPWYmVhSAn//96b36nnnxXwMqUJAMaeUSgOWux5XAp8A5mqt/QTcCEHxJZIC9UnNzfWUlNi8ObJthxI3FkjM2U0mGI1izubF3O1m7SXw/h5mKbu0czwlXVBTBIdnjTezArTLmV00mznFc2jtbWXHSRutuoqKjLDo7/e43MG8dx1fg2tWs7l688htBhJzlviLsZiLp1XAyjQNu6WX9T9ZvdqUxvAiUWIu3ZHuvhkY0X0jCdFaD7E6b6zayGBRoXljx+LvJebsHjtLpi8hLyOPd06/w8k2r3CKRYvM/6e6Oujvf+LdHfSnwdQOB7kFiesPalmwj7QcwakDx/naJZ7/Qe/tpJI1OREEFHNa60HgRq31gNZ6v9Z6n6tosBAuvkRSMItZNOLmamtN7a78fHt1ePyJuZ4edx2yoHF3ySbmotFOx+bFvLX7LOASc2vW+F/Q+25/927Sfv0b1ro8uI++8xt21+32WL38tMsJO6v1D38wLXB27zav29rgnHN4M7uZpu4mZhXOGnrCLimhsjcPgAMN77C7bje763bz3OHnqGquorDPwaJT2Bdz2ckt5ny6WUNpuu59kzYMq9xJvMUcpM7Fsb23nb+89xeOtx6nNLeUmQUzaepu4k/qAK1ZhGyZs3vsZKVn+Y5FTUvznPOChL1UHTLxrhX98e36MJxxWeOYmDeRnoEeTrWfiso63UW74xDq4L2dZD9eE40dN+srSqmfKaWWKqUutR4xH9loxbrQ1daa55YW33XIvLEu+Js2Bc2i9ItVr2zVqqAFXX2O0+LVV03c3SWXBL9oe7tD/JXCiAeJcLN2mgtNwdxLjVvAHxddZOIX6+tN0ss3v2msW8C3tn+LBQ8toK6jjrL8MuZN8l2MNWwx9/vfm20uWOCpmj4si3VIVXelqCgx3/9I2zEWPLSABQ8t4NrfmXp3a2rSSHcyogyHP0K1zPUN9nGs9RgO5aC8sNzWZyLBElp17XWeiVbT9cFBI+j84XR6ilT7uElLlGUO4teAPRK01lz8nxdzwx9uAIzr0zrObzv6Pc6/B/pbgnQygSFizhIhdm4ErBunEbGoNl2tVcf3mG1lBO6EEg+iKd5be1pp7GokJz2HyeOCxF1HiVS5+Ug0dnL7LTPO172maeDq6A9nDHDJJSal+uWX4SMfMSLL6TSuVH+9+y64wGTD1daaLNJLLgl9uwcOmOdFi+yPE8w4tfa4f0NZT2amsQR2dJj4rPHjQxtztIiVmPPeL15orWkbNPFI4+ZfPmL+EJSCb30LfvYzt1B///RJPD5bUdtlLkQO5eDTiz7tt13O8pnLyUrLYtepXZzpOhO8PdTChabl0ltvDZ0+bhz8wz+w8RXTjsmXWzd/1rnc//LbbFw102N5BXL6nPzzS2+b0ielpYG378IScy099hJkas7W4NROygvLyUrPsvWZSLD24wixWVlpMn+rqvwfU7W15pgvKxtRpLqrv4u23jYy0zJNh5A4kwoXx7qOOo6ePUpmWiaLpizic5d/juz0bA40HuCN2tepG9dPQ1cjI7ureuHdF3fSJKresG/VtY59KxbVHWNnibmtW01We5bv47C66TBkQmXBLJ/z40llcSWvnniVquYqVpSviGhdVsxqRXGFz3ZosSAVjtdkIKiY01pLt4dosm4d/Nu/mTs7re3FsSll5v/P/5jlwxFzlqCZM8fe8pdcYi7Kx46ZIG4r0DTU9RQXGzHX3JwYMdfdbcqIpKfDjCgUmS0qMt+pudlTRmYYXf1dDOIkux8y59gI0L3zTvNwMQ6TcWSXvMw8ls5cyvNHnmdz9WZuu+i2wB9wOOCXv/Q5q6W7hdf/+DrpjnR3Vt8QKiv59mPw7SV3wCe+5pn+ne/AybfhLj/JHj4I1TIX71idguwCFIrW3tahF/TKSnMxD2SdDfA/sRIqyvLLot7P0g7x6NkZKdbYLp18Kdvv2u6evu1j27j0wXm8deYd6vubA4u5YX1xQzl+KoorqCyupKq5ip0nd7J4+mIzY8oUk5m+d69JIrrat02jquukEXOTE182I5qW2Hj/B723lczHazIQkrRWSv0lGhtVShUrpTYrpQ67nov8LLdeKXVIKVWllLrfx/x7lVJaKZW4CNNQmT/fNCE/ccJYuewmJVh3hOGWKAnVOuVweOJDvF0Koa4n0XFzR1xB3rNmjQhCD5sgrtYhmazRsAbaIOQSJX54/sjz7szZcVk+LMX+vnsYRZmLcszfPlQxF69YHYdyuMd4tuesZ4YdV3uA/0kiXayQGhfHQKKhrGAqAPWDbYHDTrwKBvc5+6k5W4NCMavQnrXMcrWOaJdnw9VapYy1ubLCpickhkTz93b/Ln7aocUCqzPOkZYjDDqlxK0/QrWTTo3Sdu8Htmit5wBbXO+H4MqkfRC4BrgAuM3VSsyaPx1YA4wsRZ/MeIukH/zANKb2V4fMm9WrTQDuK6/478rgD63DSwLwVeMu1PUkWsxF08VqYVPMje8lOkkXNrDcQhurNkaUtWZduIaUJPHGV5299nZzXDoc5ji1SaiWuVBinqKFzzHayWgO8D9JtJjz7r7RO9CbkDEEI5BoKHPFatXn6ZHldbzxipc7dvYYTu1kesF02y56v7GoQRLSnIMDVOea/VoxN3FlSSxiIubi+B/My8xjcv5k0xmnLXY9ZlOdYKVJHEqpW70mveV34dC4EXjU9fpR4CYfy1wGVGmtj2it+4A/uD5n8SPgPlzFjFMK62TwyCPmeVgdsleOv8KVv7ySBQ8t4BvbvmEmFhbCFVeYHq1bt4a2vbo6426cMMHUvLKLJTpffNF8fnDQY+maHbzKP5AwMae15p5n72HB3k+z4OOwYMEud8C+9Vj68FLeqgvjkLYu0IcP+5zd2mECswt6cPdUjTUXlF7A1HFTaehsYG9DeJWDtNYBy6AAnu++e7fpN/vmm6bESX+/KXlS5NPI7pOQ3axxrG9l4XOM0bLM5SVGzGWmZTKzYCYaTc3ZmoSMIRgBLXNWlnE+gc8rYWSyemM1et9xcodp9G6xZImpGfrOOz47wZyqeoueDCjtUowvDegIjguWW31/436W/GoJr9e+HvI6GjoaWPvrtTxx8Ikh64wXqWBNTjTBSpM4gXu83t8Vpe1O0lrXudZZB/hK+ZkKeMvwWtc0lFI3ACe11m8H25BS6uNKqV1KqV2NkRbdjRbr1plgcyvD85Zbhsz+zivf4bXa19hdt5uvvfQ1TzFYK9t1h416Yt6Ea52aONH0wuvpgW3bjGu4v9/EjeTl2VtHgsTc/sb9PLjzQXY7T7J7CuxOb3SX0rAe249v58dv/Dj0lVvtY3bu9Dm77bgReeNVtt8A6WijlAo9q3UY9R31nGw/SVF2kd/MWSZPNi7r/n5jjfvhD8Pue5vsMXPgZ4yzXG66Y8f8Z2knsZsVkv/iGG0xZxWctetiBcjPzOeyqZeh0bxZ96ZnRlZWwG4QO141PUMv6E1QwtcwinOKOW/CeQw4B3j1xKt8/9Xvh7yOp959is1HNtPW20ZuRi7zy+ZHf6ABsGojHjsbIIN8jGPHzbrZFZs23RXrVqyUKg72IaXU80qpfT4eNwb7rLUKH9O0UioX+CLwZTsr0Vo/pLVeqLVeWGozyy7mTJhgTva7dhnrjtVrDlN+YetRY3nLTs9mwDng6V157rnmOUgrqRFE4mr0drWGs54EiTmravxNVRns+i/Ytf4pdt29y/148tYn3cuF7Ja8+mpjSX3pJRNgPYzWEy4x5yvmLIa4Xa3DY3xsYl1Az51wrv/AfKVMRvXTT5v3mzaFLeaKsj0xczpI6ZoB5wBHW0wF/9k2er9GC59ibtw4czPT0+PbzRckrEHEXGC01gE7fbiLOYcg5qykk8n5oZXTOLfEnHMtF7+bAK7WDQf/DMCaksTHy1nsvHsnG/7WjPX5I8/b6xbjhXWcfOayz3D0H48Gz5iPMtbvZtVnFEZiR8zdBXwa2Aa86XrsCvYhrfVqrfWFPh5PAw1KqckArufTPlZRC3jn9E8DTgEVwCzgbaVUjWv6bqVU4s6M4TBxoqnvNUwYvXL8FTr7O7lo4kVcPtWUtXCfcEPtC2nh3Zs0VLyDfVNIzFmJALe83c+C3AoWXH4TC6YscD9uOu+m8N2SpaXmt+vtNRbLYbTV1QBQkGvf5RgNVs1ahUM52H58O+29AWKJ/GDb8jVuHLzvfabJ9Zkzpm1VcbHZJyGQk5FDTnoO/c5+Ovs7Ay57ovUE/c5+po6bSm5GbkjbiQS/hY0DdUjxDmsoLBwx2+ookUgxl8yFWM90naGtt43C7EK3mPYmHMtcuALar+i1xNzmzSb8xIV2Otmozc3c+ivvCGlbsSQ/M591les4t+RcWntbeaP2jZA+b33/q2ZcxcS8+NfO81nAWxhCUDGntZ7l4xHprfEzgHWk3wE87WOZncAcpdQspVQm8GHgGa31O1rriVrrcq11OUb0Xaq1HhW/sne1/xEnkuE1zuwSSQeExYvNxfvgQU/7pyQXc519nWw7tg2lYU01Pi1GSilPYdBw3JIB7szbTptCy+PHxdcSXJRTxBXTrmDAOeC27oZCSNmiSg0thrt2rUnQCRG7rlb32OIcq+N3fIHEXJCbJ+/SJIkiFg3Yo4X3TYUvC3HIYm7SpLAFtFv0Dt9PFRXm0dIyJNzi3Z3Pcjx/kNIuxSUrPhzStuJBuOe8RIQ4eOO2xoqY80vQWg0ut+Y/ATO01h9XSs0BztVaR1Km5AHgMaXU32GyUW9xbWsK8D9a62u11gNKqXuAjUAa8Cut9f4ItpkSuLMJK9ez85Q5SbjFXEmJqdXW1mYsInbdxgEsav2D/Txx8Alauk0qfWZaJjeddxMlua6uBRkZJkPxqafgscfMtFBEoRUQb6ePYpR46dhL9A32sag1j9KuTr9t0tZXrudXe37F7975nbt46yWTL+GKaVcE38j69fCNb/gUc60tdZAH44vjUyF9yLAq1vPqiVf5+a6f0z3QzS0X3EKaw57ICjnBYP16eOghz+swKM4p5mT7SZq7m5lR4L8OoNvtFseSCBCGmNuxAx5+2Lz28z8RN2tggt1UWPutIQ/7lrl9UbbMgTnmH3wQvv9901kH2LD/l1AKawdm4kiLUimkKLK+cj0/2fETHjvwGLOKZnHz+TdTkB04Kc6pnZ5iwXEqCzScaFnmOvo6eOLAE3T1jwyPAVgwZQGXTb3M7+f7B/vZVL2JvQ17+cLSL0Q0lmhj52h7GONatTpB1AJ/AsIWc1rrJmCVj+mngGu93j8LPBtkXeXhjiPZONV+ircb3iY3I5erZlzlrozvvitUygiy3bvN3b8dMecdv+NDzP3pwJ/42yf/dsi07Se28/CND3smXHONEXMWVuyeHSzL3BkbrXeixAtHjQVx7TudAdukrZ69mjSVxjun3+Efnv0HwMQp1v1zHYXZhYE3cvnlRlgfOmTcapM9wq2t3ZXNOjEKRYpD5Jo51/DlF7/MpupNbKrexIBzgNvn3W7rsyHffa9aZfbvwEDwPr1+CNUyF2/LgN/xWcWivcVcd7c51qw4Sh//E621+4I0Kd9e27NYYMUd1pytGVoQOQnY32ju2f391uOzxpNNOh1ZA3Q0niTf34q8xdzr4e1zyxJsNaof0vXgmmuMmHviCfMANt4OlAbIBk8wy8uXk52ezXtN7/F3z/wdexv28uP1Pw74mbr2OnoGeijNLQ0q/GJFtMTcj177EV9+0X+4/f9b9v8CirlBPcgtf7qF7oFu7px/Z9xamtnBzj+4Qmv9IaXUbQBa626ViLLlYwCrqfPK8pVkpWf5jmuxxFxVlSlVEowzZ4wlr6DAZ49QK15s8bTFnDvhXB7Z8wh/fe+vQ09cH/mIydxraoJzzgleE88b66J32ldYZGx4r/k9AObXAUuXmpZiPijKKeK3H/gtL9a8CMCmI5s40nKELUe2cPMFNwfeSHo6XHih6VX73nseMTc4SJurwOz4yeWRf5kQWTB5AT9e92P+cvgvPH/kef7y3l9siTmtdeiCafx4cxHr7R0iZkMhZcWcL8vckSNGyBUXm3Zpn/zkiPW19rbSO9jLuMxxcY39G05ORg7Txk+jtq2W463H45pUEozNRzYDsHSG7xptSinKMoqo6W+k4eR7vsWc0+kuGqwnTgzbGjo+azwT8yZyuvM0p9pPMW28V6mRa66Bb37TZPkD3Qzw0qSHgUHWfviLIW0nXuRm5PL7m3/Pnw78id+98zv+8t5fgoq5RLtYIXpibu9pc71bV7HOZ2bzoimBk1ay07NZOWslzx5+lk3Vm7hjfvLERdoRc31KqRxc9dyUUhVAclaaTHG8Xazg564w1CQIb6ucDw3unaX04Qs/zJYjWzjRdoK36t5iwRRXQHt2tnEphkOg2KIY4T75NAN3+HaxWnzowg/xoQs/BMAD2x/gC1u+wIaqDcHFHJh9+uqrZh8vX26mnThBa4aJZxw/Lv6NSZRS/OMV/8h151zHnJ/OYVP1Jgadg0FdrVbQeUFWASU5I0W/X973vojGO6rEnPVfu/xy+N73fK4vGVysFpXFldS21VLVXJU0Yu5052l21+0mOz2bZTOX+V2uLL+MmpZG6huq8en4a2kxFuPCQtpUHz0DPeRl5JGf6deO55fK4kpOd56mqrlqqJhzOExrRhcvVW2g57f/w6WTL2ViUeLry/njpvNu4n3nvI8NVRuobqmmqrkq4P8qGcRcYXYhmWmZtPe109XfFfaNkPVdvr7y6wEtcIFYV7GOZw8/y8bqjUkl5uxks34F2ABMV0r9FtOx4b6YjmoMMugcdFvmrCBV666wZ6CHU+2nzIKRiDlfs4cFG0da3mIEhYWQmWmsgz7KeEQbp3a6ywhUNBNSLJc7OLh6Q9BSGWYDPjoBVFXR5iotlyiXBJjfc3bRbFp6Wtyxl4HwTjCIp+HdjpgbErOTzAkQNrK9k0rMRbFnZ7SwzoEryleQk5Hjd7my4pkA1LfW+m7pFYVMVgu78YXugtv+uqckEWmONNbMXgMET4ZIBjGnlPLESnaEV54kLO+DD6xrpHWjnCzYyWbdDHwAuBP4PbBQa/1ibIc19th1ahfN3c3MLpo95EALmNFqhwCZrL4O7kgLz45AKc+FryH2NYJOtp2kd7CXSR0wrmQyXHSR7c9eXHYxk/ImUdtWy8EzB4N/wNdv4SXmxmcltmiodVGxau4FIlBdr1hi1ZqzEnB8car9lDtmJ977NKiY8z6mU0zMWcLYrpg73no85PpkoeKdzR+ISYWu/qxZA6Yl4nCiKOascJf3mt4bUpNywDlA/2C/+zHcs5Ls2LlxH3QOcrjZlFpJVPKDRaSu1tOdp+no66Aou8hnyRu7zCmew6zCWTR1Nw0tJp1g7PZmXY5JWFgJJL7Z3CjEuyemt2UkYjEXoExCQ2cDnf2dFOcUuxuKr5q1ijSVxqsnXnX3GI2YOLpah7hYh7VJC4ZDOVhXGULqvh8x15ptXloZsonCLcyrg3+XRDTQBo9Yaupu8rtMIi0D1v+ipadlaHHp0WCZc+1P62IdiOt/dz0zfzyTnSeDW3nDxamdtgWRtf/qxuH7XOhdliRKlrnvvfo9pvxgCqfaT/GlrV8i498zyPxGpvvx7pl3GZ813l42fBKwtsIkLW09utXdo/etureY9P1JPLrnUXbX7aboO0XuFl6JtMxB5GIuWueRaHTbiQVBxZxS6ufAJ4F3gH3AJ5RSD8Z6YGONQ02HAEb48Ue4QsrKIDfXJCOcPRt8xQEuML4O7oLsAi6adBGDepCDjTasU3aIo5hzW5iaMb1DQySkOky+6v5VVXHWEnMJdLMCrJy1kgxHxsjekj5IlGAqzTMZ2Y1d/lvtWW7zRFxM0h3pjM8aj1M7h97cTHQVTm1o8Lj5UkzMWS2ZXjn+SlB3UXlhORDF8AsfvFX3Fme6zjCzYKa784I/po839eSPFxBYzJWVubsGhLvPV5avZEbBDBSKhs4G/rT/T/znrv8EIE2lke5IJ92RTmZaJp9e9Gky0jLC2k68mTJuCvMmzaOrv4vtx7cD8Mu3fsnpztM8uPNBfv32r2nva8ehHFw86WIuLrs4oeONtNZcNL0P6yrWkabSwnb5xgI7lrnlwDqt9cNa64cxpUNWxHRUYxDrAB2e6jzCFaKUx2Vqp61XiGIOjBl5yDYjxVcZhxjhjv1qBubMCfnza2avQaHYdmyb31pEboqLTR29jg5Ptm5VFc2uUJ+QEgliQH5mPlfNuAqndvL8kecDLpsoMWfnbjvRMTs+Xa1ZWeb3Hxw0N1Z9fXD8uAmKLy/3u65kEnMVRRXuuMpdpwI39YmHJcLbxRosbtPtsSgmqJiLdJ9PHjeZY587xq9u/BUAP3z9hzR1NzGrcBb9/6/f/ej9Ui/fWvWtsLaRKKxQDGvfW8+7Tu3i8YOPA/DiHS+y55N7yE7PTswgXSSLZQ7M/6HpviYevC557Fp2xNwhwLtg1nQgxP5HQjD8nXB8Bt/adbW2tJgLTW6uxzrmhb/inFEvKJooN2sY7ctK80pZOGUhvYO9vFTzUvAPWNuorgank+5jVXRnmOLLiSw9YWE3oSVRHRZsiblQixlHGb9xfd7HdU2NsdDNmGESfvwQqZUomiilRlzM/bGifAWZaZm2rLzhYoUD2Ik5i6eYs7DcksdbjwP2RGeyY4WVbKzeSFVzldt6pdHUttUyLnNc0riNoyXmohH7l5WelXDPy3DsiLkS4KBS6kWl1IvAAaBUKfWMUuqZmI5uDBFMzFW3VHsyLO2KOctyV1ERsCzJ8Itk1Fv9xFPMNRp3dWVnVti1z0JytXpntNbV0aJM7ElRdlFSnOi9LSr+MnRbulto6m4iJz0n5EbkkeLtOvE3vqS0zMFQi7PNvsXJZJkD+3GVlpVXo9114KJJa08rr514jXRHOlfPujro8pPHTSbHkUVjHrQee2/kAj7EnHWshYvllrRIlUSHQCyZvoS8jDzeOf0Ov3rLWB4z0zw3I6tnr04at7FbzHUm3jKXjNgRc18GrsGUKPkKxs3678APXA/BxU/f+Cm3/OkWDjQeCOlzvQO9NHc3k6bSRrjminOKKcouoqOvg9OdLleeXTEXQlkSb0Jtwr39+Ha+98r3hgaIexMPMXfsGPrfvuD5ToWzQ0p+8CaUxAH3vv3xj+HjH3e7WCPJloomF028iMn5k6nrqONDj3+IJw8+OWIZ71iSeAvQvMw8xmWOo2+wj7OuYsveRKucQCQEzWj9xjc8dRhtirlIhUW08I6rDFbrz50dHYO4uS1HtzCoB1k8bbEti4dDOZjtKvpa3eyjV3UMLHPg2Qd2RWeyk5We5f4eP3r9RwB89rLPuucnk2BNJjdrMmKnNMlLgR7xGGSq8ELNCzx+4HHeaXgnpM9ZIm1i3kSfxV39ZrQGi5nbs8c8+2gr1DvQ6y6/cU7JOT63ZwWeB0JrzR3/dwf3PX+f/xIY8RBzX/gC1f/1AJ3OHko7oWhmCC3HhnH5tMspyCrgvab3ONpyNPDC8+eb57fegmefTToxp5TihnNvAEzrtjv+7w76BvuGLJPok5zVYslyQXpjlRMozC5M2D61brDOdA1rSWf9r7Ztg9deM6+t48EHg87BIf/1ZCCUuEo7Vt5wcddoC0E8VJaa/V+V0z2yw8zJk+Z58uSoirmbzrsJMBarcVnjIl5fMmCdH3oGeshwZPD5xZ/nnJJzyEzL5JrKaxI8Og+W1+Bk28mQP3u89TgtPS0UZBUkzX8v2iRPQ75RQLixZsFONpXFlew8tZOq5iqWzFhi3zK30SWurh55B7n9+Ha6+ruYN2keE3KHdiqYPG4yOek5NHY10trTGvBOuaq5iiMtRwBzQr5mjo8/f6zF3OAgbNrERpfH8+qjhBUvZ5HuSGdNxRoeP/A4G6s38smFn/S/8E03wTPPuBt+t/TthVM/TBoxB/DdNd9l+czl/L8X/h/VLdW8euJVVpSvcM9PZLYomOO+qrmK+o56zptw3pB5iRaaEEBs3nsvnH++6ckKpr3Zddf5Xc+ZrjM4tZMJuROSxnUFcO+V9/KJBZ9wF5H1x4UTL2TKOFOaY2/D3qhlN2qtwxNz1k1nEeZcaLm9m5tNvHBeHoMTSqIqoBdPX8yrd706qqw7d86/kwm5E2jtaeX80vOZMm4Kmz+ymZbuFqYXTE/08NxML5hOuiOdk+0n6e7vDlhUejiWoeHqWVcnRfhLLLBbZ06wgeWetNxWdgkWFD3C7Tl1qgmyrquDzk7fK62rM5a5nBzTn3QYgaqVO5TDHQgf7Lt4x5X5dUtaJ9mGhpHukGjw5pvQ1MQG1/l1fRURiTkIIW4uLc20tLrjDrjjDpoXXggkj2UOTPHi2y66jRvPvREY+Z0SnWAQyH2SqGLG3vgdX14e3Hqr+7fn/e8PmPyQbPFyFtfOuZYPXfghd009fyil3P+LaLpa3z3zLifaTjAxb6K7XIod/CZBeNXWbOppZlAPUpxTTFZ6VlTGu3j6YndJndFAuiOdm867iTvm3+EujTWjYEbCS5EMJ92R7u6nahkQ7BJKck2q4lfMKaX8llpXSs3wN28sE0vLHHglJKSlwWxXL0V/rtZNpi0OK1ea3qrDsE7GVjaT320G+S7eJ3W/bsm8PNPsvrcXWlsDri8sNm6kNw22uvomr60mamJu69GtI9ySgbDijpJJzFn4y2yNZpZXOJTl+RdziSpm7E20mnwnq5gLhViUKLHWtbZirek/bZMhYs77POgVKzwa9rngIdSuJQD9g/3uEIJgnUVSmUD/nBetF0qpLcPm/V8sBpPqxFzM+StP4nQa69TLL3sef/yjmb9u5MF7su0k75x+h7yMPJZMX+J7mzb6NvYM9PBCzQsA7vX4vWOPhat1YAB27ID/+z+2z4CuTJhXD1PaiVjMTS+YztzSubT3tfPaiddsfy6ZxdzSmUvJSc9hT/0enj38LC8fe5mXj73MoTOuDOAktMwlg5tVxJyH1bNX41AOth/fztajW3n52MvsO70vonW6rSYh9jT1a5nzEnNWUddU3ueCh3D6Cb9e+zptvW2cN+E8ZhbOjNXQEk4gMeftWB5+ZRqdTucImTZ+GplpmdR11NHZ58f96QO7Ym5IQoIlVg4dgv/6L1i4EJYt8zyee87M99FofstRo81Xzlrp1/VgbdPqTOEL77i7O+ffCQS4Y7fEXF2d3/WFzNe+BpdfDrt3s3GOOSTXVWNcXdOmRbz6kEqUuEhmMZednu2Olbvud9ex7JFlLHtkGY1djWSmZTJtfOT7LBzGnJjLS11hUZxTzGVTL6Pf2c+q/13FskeWcdEvLrLV/9cX3f3dbDu2DYVy13Gzy/Tx08lQ6ZwaD51Hvc5TXv2oR4OAFjyEYzAZC1Y5CJwAof289vVeANIcacwums27Z96luqV6SE2iQAQrV1CaV0qGI4OWnhZP4Ofll5uZW7d62glddBEUeCUrXHmlzy4IVpuuRVMW+R3T5dPM+rce3YrW2mfQqHfc3XC3pHetIgAmuJIsmgOXPwiJx02FchYuZMNVJ8FZx/orboebFhtXdISsr1zPD1//IRurN/Lt1d+29ZnmnuQVcwBfWvYl+gb76B3sHTL9A+d9wGcmdTywI+biXczYm+G18MINoB4twuLrK77Ot7d/m35nPw0dDRxuPswTB5/wG7IRiJeOvUTPQA8LJi8IOQ4tzZHG7IJyDp2t4siZKi6yZgxxs5rOFqksoAUP3nVX7WJVbQh0vRsNBBJzE5VS/4Sxwlmvcb0fPdGfUaayuJJ3z7xLVXNVyGLO30neoRxMyp9EbVstDZ0Npk/imjWmjtq2bSapwOGAF180LYaCYCfg/eJJF1OWX0ZtWy0HGg8wd+LcEct4N8WeXjCdC0ov4EDjAV478RrLy5cPXdgaV7TE3LFj8O67MG4cJzf+iXd+Osu4jb/4PxClQGfLLflW/VvUd9Tbuggns2UO4MrpV/L8RwOXoIg3/sRcc3czLT0t5GXkJbQuW05GDgVZBbT2ttLS0xL2b5tM3R8iYU3FGtZUmMzX3XW7WfDQAne5klCFbjhZrN5UTDiHQ2erqEpv46LmZnOe8UqAqD/wFyD197lgCMcylwzW/XgQyM3638A4IN/rtfX+f2I/tNQkHJ++nTv2ERe8khJYtMj0hOzvh8susyXkvMcW6OBWyuP28OVmrG2rZd/pfUZAzTDxcgFbA0VbzFllV1avZuOxrUBgt3E4eLslN1VvsvWZZBdzyYg/Med9nCa6nEA0XK1uC3x+chQMjgbzy+YzMW8iJ9pO8O6Zd0P+fKRizm2psZIg2ttN1nxWFkyd6u4WMJr2+VimvLAch3JwrPWYrcS0ZCg6Hi/8ijmt9de01l8DHrReez/iOMaUIpw7h7DEHAyNh/MRG+eLUA7uQBXfvev2WC7VgJ0Toi3mNri2sX69x0IYYgC1HULN3hMxFzpW/a/GrkYGnYPu6Ymuf+eNJQaiIeZGk5XIoRwBb/oCUXO2hkNNhxifNZ7Lp14e1vZHJEF4tzB0OEblPh/LZKVnMaNgBk7tpOZsTdDlG7saae9rT2jR8XgRqDTJ9UqpRmCvUqpWKXVlHMeVsoQq5jr6Oujs7yQ7PZvxWX6rwfgu3+Cdqeoja9UXjV2NdPR1UJRdFPTgXlOxBoXipWMvuRM6TneeZunDS7l3873A0Dtq72zJERe9aIq5/n543rgKB9esZnP15hFjiRbWOjdVb/LfrswLEXOhk5GWwYTcCTi1k8auRvf0ZLqjjqZlbrQJC7dF3k77Oy+sG8JI+n8OEXP/+I9w/fWuGWb6aN3nY5lQrrHeN4SJtu7HmkBu1m8BS7XWU4CbAXsR4GMcu8V2LU60ngBg6ripAQ82nxeTyy6D884zVegX2QvuDOUCOSF3AgunLKRvsI+XjpnObc8efpbtx7dztucsuRm57lYwYNySi6aacYxoaRZNMff668adct55HBk/SEtPC1PHTY1JkPyc4jlMzp9MU3dT0DvB/sF+2nrbcChHQGEujGR2kamb+FbdW+5piS5m7I11M2WVugiV3oFeWnpaSHekjzqhv7Zirbnpq3mJrv4u25/bfmI7EJlFfYiYa2z0tPFabmJ2RcyNPkLpHZ5MN4SxJpCYG9BavwugtX4DEy8nBKG8sJwdf7+Dtz7xVvCFsZ+tZ52MhlxM0tNNT9Ddu21nb4Z6cA93M1qfv2fRPdR+vnZEOYup46aacQ5vfRRNMeflYrXGc+6E8HuxBkIp5e5dG6xXrdUovjC7MKTipwLuVlLeLv1EFzP2JlLLnPV/mJQ3adQdG6V5pVw6+VJ6B3vZdmyb7c89etOj7Lx7J+8///1hb3tm4UzSVBrHCxW9+/eapKijR+Gf/onegV6au5tJU2nu/rpC6hOKZS6ZziGxJtBZZaJS6p+sh4/3gg/SHeksmrrI9t233Qr37otJ57CLSXa2zw4PwbZn9+Ae3jXA+vxlUy/z2f7H70UvxmIulh0C7J48WnpaAHGxhoOvmn7JdFft9/9nk9FuIQqnM4RDOVg4ZeGI3tChkJmWyczCmWg0RydmwLnnQnk5wJCerIkquyNEn1DKkySTdT/W2MlmtR7D3wtRwO4FK1qFS0O9QF429TIKsgp4r+k9jrQcCfr5oGKupSWMUXtx+rSxRGZnw7JlcendaVfMSbxc+Fwx7QrGZ43nUNMhas7W0N7bzunO02SlZTF1/NREDy/i/5+Iudjhz+02WkrBCEMJxzI3FsSc3zpzkrEaH+yKkWhk00HoB3e6I501FWt4/MDjbKjaELmYi9QyZ/WcXb4ccnLi8mcd0RvXDyLmwicjLYPVs1fz5MEn2Vi10V20uqK4IinckiLmAnP51MvdYvzrL32diXkT+eTCT8Zl25XFlWw+snlEGMRo3+djFSu+9mjLUQacA6Q7/JfLHUtiLuBZUil1jVJqm1LqjFKqUSn1klLq2ngNbiwQjmVO6/AbcIRzcFsByr9753e09rYyLnOcX9eI34tefr6J8evshN5eH5+0iZeLFeLzZ/XZTs0HTV1NgIi5cHG7Wqs3uPt9JstJWMRcYDLSMtwlSr7y4lf45svfjNu2/VlqTrWfAkbvPh+r5GbkMnXcVPqd/e4EQl+0dLfQ3N2c8KLj8cKvpFVK3Q18ArgP2OWavBB4QCk1TWv9UBzGN6rpH+yn5mwNCsWsolkBl83PzCcvI4/O/k7aetsoyC4IuLwvrIr6+Zn57tpedrDa9Lxy4hUgcJq334ueUsY6d/q0cbWWhXGCdTo9xYLXr2fQOciRliOA524tFlhunOqWapza6ddS5B3kLoSOJea2HNlCdrqJA10+c3mgj8SN0rxSFIrGzsag1gBfjIWG799d/V3OKT6HvsG+sM5P4eLPcm7dfM0qDHxuFVKPyuJKTrafpKq5yu+10/J6VRRXjPqyJBDYMvd5YK3WeqvWus312Apc45onRMix1mMM6kGmjZ/mvngFIlLrQLg1d6aNn8bcUk87r0DWEnfW7fBsVojc1frWW3DmDMyYAeeey4m2E/Q7+5kybgp5mXnhrdMG47LGMTFvIj0DPe67fV+MdutLrJlZOJPzJ5xPe187j+1/DIhN7cBwSHekMyF3AhpNY2dj8A8Mw92JYBQL/VlFs/jmqm/yvbXf40vLvhS37fqzzI2l4Pexhp24ubHkYoXAYk5prUdcdbXWTTEcz5gi1IMtUjEXSZq290U10HhLckpIU2mc6TpD/2D/0JmRijlvF6tScU07t3PyEDEXOdZx5tROpo2fxvkTzk/wiDxE8v+TYyN2zC6ajUJRc7ZmyDknmTqICNHFTq25eFQ6SCYCibk2pdTFwye6prXHbkiji86+TrcrcDiJEnPhnNwsF1iwz6c50ijNKwU8pQHcRFPMEd+TtYi5+OB9nK2vWJ9U7pGAVucgyLERO7LTs5k2fhoDzgGOtx4HxlZPzrGInfIkY+33DyTm/hl4Rin1VaXU+1ztvb4GPA1InTkbvHzsZYq/W8yd/3enz/mHmw4DcRRzEbgdrFZddj7vd5xFrrp04Yi51lZ47TVTHPnqq4H4/lmtuzvrN/OFXLAjZ9nMZe6Qg2RxsVqIZS55GX5xb+hsoLO/k+KcYp/1MIXURtysI/Er5rTW24HLXcvcCdzlen2Fa54QhHmT5jHoHOS12tdo7WkdMX/bcVMt/ZKyS2ytL5GWuez0bO6/6n6unnU1i6YEbh0Wk/IkW7bA4CBceSUUmODqeMbEzCiYAcDJ9pN+l5ELduTkZORw35X3sbJ85agRc9393XT1d5GZlkl+Zn4shjbmGe52G2sX8rGGd9tMfz2zx9oxEDAlS2tdD3xZKVXqeh965O8YpiC7gMXTF7P9+Ha2Ht06pG1NfUc9e+r3kJOew9KZS22tLxI3D0R+cH95+ZdtLRcTMTfMxQrx/bMGu5D3DfbR1N2EQzmkdVCEfG1lcpa4DFfMWfUHS3JKksptPJoYbqmxQjDGQhunscj4rPFMzJvI6c7TnGo/NaKtZHtvOw2dDUlTdDwe+LXMKcNXlVKNwLvAIVetOXtXdAHw1GgbXhl9U7UpfruifIWtTFaIzDJnVdTPTs9myrgpIX8+FKym5FHrAqH1kJIkZpKO6wk72L6X1kGjn0jFnNQfjB3DxdxYs8qMRQK5Wi13++yi2UlRdDweBPqWnwOWAJdprUu01sUYt+sSpZSUJrGJd29T72K/lrgLxZUUiZiL58Eddcvcu+/C8eNQWgrz5wNQ11FH90A3pbmlcalpFWzfj4U6YmMdq6yIiLnkY4SYk7Iko56AYm4MZjIHuqp/FLhNa33UmqC1PgLc7pon2OCSyZdQmlvKsdZj3PTHm/jT/j8x6Bx0W+a8s/eCEe7FBBLkkhzelDxcMWe5WNetA4c5ZN1lSYrj40aZkDsBh3LQ1N00suQKEi83FhDLXPJinQcONx/mg499kOePPA+MrYv5WMNdzN1HZ56xaJkNJOYytNZnhk90xc1lxG5IowuHcnDjuTcC8MyhZ7j7z3ez4+QOmrqbKC8s55ySc2yvy+racLrzNIPOwZDGEc+aO9Y4RxRXLXHFkjWGGHq5Y4d5XrnSPSnef9Y0R9qQ/T8cEXOjHxFzyUt+Zj7nlJzDgHOAJw4+wZmuM2SnZydVnUIhugTqmb27fjcA55acG9cxJZJACRB9Yc4ThvGj9T/i+nOu5/MbP8/Rs0f5+ravA6HX0cpKz6I4p5jm7maauptCaskVT/FjuT1be4dl8FotvBpCTOCod108Z850T0pEQciy/DLqO+qp76gfEVTrFnN5IuZGK0U5RWQ4MmjtbaVnoMd2rKuIufjw/Eee542Tb7jfnzfhPClLMorx52YddA6yuXozAKtmr4r7uBJFIDF3sVKqzcd0Bdg7iwmAuWu88bwb2XJ0Cz/d8VN3vJzV8zQUyvLLaO5upr6jPmnF3Pis8QC09Q47fCa6xnv6tOmz6rAZu2eJOa9+rokwowdyc4tlbvTjUA4m5U+itq2Who4GZhbODP4hPGKuKFuERSyZXjCd6QXTEz0MIU54izmttdswsvPUTlp6WqgoqhA3K4DWOk1rPd7HY5zWWtysYeCd7JDuSOfqWVeHvI5wXT1xtcxlGcvcCDGXlWUKBw8MhBY3lyRiLtC+d/fezB+9vTeF8P5/YpkThOhTnFNMUXYRHX0dQ0Jf3MaSEOLRRwNjI2c3SVg+czlZaVkALJm+xG3BCoVwLiZd/V2cbD9JuiM9Lneu1vdq7WkdksELeARZvc3x9/TA2bOQkeHuIKG1dmfnJo2YE8vcmCAsMdcjYk4QYoF1/v/Dvj+4BV04lSJGAyLm4kheZp67QHC4dw1+a7gFwOoNO6twFumOgHWio0JWehaZaZn0O/vpHewdOjNUMWfF102a5HbL1nXU0dbbRkFWQVwvkIEu5CdaTwxZRhidWP+/U+2nbH9GLHOCEBvmlMwB4HMbP8eKR1bQ1NXEzlM7yXBksHLWyiCfHl2ImIszD6x6gLsvvZtPLfpUWJ8PxzKQiJo7fl2toYo5a7lJHvflliNbAFg8fXFcK+r7K7lytOUox1qPUZBVMKZiNMYiVpzc0bNHgyzpQcScIMSGz1/xeW449wbyM/M5eOYgP9vxM5zaydKZS8dc6zwRc3FmwZQFPPS+hyjMLgzr8+G09EpEfJm3q3UI4VrmvOLlNlabbhDxjonwJ6St8aypWBMXy6eQOOw0+B5OS7fpeCJiThCiy8IpC3n6w0+7y39999XvAmMvXg4SJOaUUsVKqc1KqcOuZ59pXkqp9UqpQ0qpKqXU/V7Tv6qUOqmU2uN6XBu/0SeWcCxziRRzUbPMuT7n1E63eIp3TIS/fT9WA27HIuGIObHMCUJssa4FXf1dQ96PJRJlmbsf2KK1ngNscb0fglIqDXgQuAa4ALhNKXWB1yI/0lrPdz2ejcegk4GwxFwCWttYteaiLeZ21+3mTNcZZhbMjHtBSLdVtMNjFe0b7GPLUeP2FTE3+nFXnW+pHpnc44P+wX7a+9pJU2lhJTwJghCctRVr3a8n50/mookXJXA0iSFRYu5G4FHX60eBm3wscxlQpbU+orXuA/7g+tyYxip9kSqWOb+Fg8MUcxurPC7WeMbLgYkDzErLor2vnc6+TgBeO/EaHX0dzC2dKzWuxgBFOUWU5JTQ1d9l6z/Y0tPi/ly8j1dBGCtMzJvIpZMvBUz91rH4X0uUmJukta4DcD37qn47FTjh9b7WNc3iHqXUXqXUr/y5aQGUUh9XSu1SSu1qDLWNVBJSklNCmkqjubuZ3oHeoMs7tdOdaTmjYEash+cmqJvVbheIYWLuzbo3AVhevjziMYaKUsrdA9Iax1hNgx/LhOJqlYLBghAf/v6Sv8ehHNx58Z2JHkpCiJmYU0o9r5Ta5+Nh17rmS1pbfo1fABXAfKAO+IG/lWitH9JaL9RaLywtLQ3lKyQlwXqEDqepq4lBPUhRdpHt9kPRIOrZrK7PWfXlQulpG03WzjbmfMtCuKFa4uXGGuGIOYmXE4TY8smFn6T3S70JudFPBmIm5rTWq7XWF/p4PA00KKUmA7iefamSWsDbbzUNOOVad4PWelBr7QT+G+OSHTOEEjdnZb3Gu/6Z32zWkhJTL+7MGejvD74iLzGntXZfQK3YpXhjWeA2VG+gvqOePfV7yEnPcdcPFEY/IuYEIflQSo3pagKJcrM+A9zhen0H8LSPZXYCc5RSs5RSmcCHXZ+zBKDF+4F9MRxr0hGKmEtUZwK/bta0tKE9WgOh9ZA6c/Ud9XT1d1GSU5KwBtrLZi4jOz2b3XW7+c3e3wCwctbKuFo9hcRi3UhYiUWBEDEnCEI8SJSYewBYo5Q6DKxxvUcpNUUp9SyA1noAuAfYCBwEHtNa73d9/rtKqXeUUnuBlcDn4/0FEkkqiDnLzToiAQLsu1rb26G7G3JzIT8/IYkcw8nJyGH5TGPG/+bL3wTExTrWEMucIAjJRkJsklrrJmCVj+mngGu93j8LjCg7orX+SEwHmOSkgpjza5kD+2Ku2sTHMWUKKJUUYg7gujnXsbF6I2d7zgJwTeU1CR2PEF+sJBirTV4gmrqaABFzgiDElrHrYE5hQukCkdJibtMm87xsGZCYEiu+uHvB3Qw4B2jtbeXCiRe6+wMKY4MJuRPIcGRwtucsPQM9AV3s1n90Ut4kv8sIgiBEioi5FCQVLHNW0eCI3KwbTcYo64wb08pkTbSYy07P5vOLx5RnX/DCoRxMzJvIyfaTNHQ0uPu1+iJR/z9BEMYW0ps1BQlHzMXbMhCxZa69HbZvN5mvq1cDyWOZEwS7/0ERc4IgxAMRcylIKljmbIm5QIWDX3jBlC65/HIoLh5SlkTEnJBoRMwJgpBMiJhLQSwrWzLXmXNnsw6vMwf2LHPDXKyNXY209rZSkFVASU5JNIcqCCFjR8xprT2W8XyJmRMEIXaImEtBxmeNJzs9m87+Tjr6Ovwu1z/Yz5muMziUgwm5E+I4QhiXNQ4wlrkRDcmDiTmt4bnnzOv1pkjvSzUvAXDJ5EvGZN89Ibmwk4TU0tNCv7Of8Vnjyc3IjdfQBEEYg4iYS0GUUrYsA1a7r4l5E0lzpMVlbBaZaZlkp2czqAfpHugeOnOSy0rhT8xVVcHRo1BcDAsXAl49UCukB6qQeOz8/8TFKghCvBAxl6KkwsXEr6u1oACyskySQ2fnyA9aLtY1ayAtDa21uweqNLQXkgE7/7+GjsSEOAiCMPYQMZeipIKY85sEoVTgJIgNRrhZLtb9jfs51X6Ksvwy5k2aF6vhCoJtUuH/JwjC2EHEXIpSlpf8F5OwypP09ppMVnAnP1gu1nUV6yReTkgKQhJzeSLmBEGILSLmUpQp46YAUNtW63cZa97k/MlxGdNwrMLBLT0tI2f6E3Pbt0NXF8ybB5PNuHed2gXAivIVsRqqIISEt5gbkeDjItE3U4IgjB1EzKUoVn9IqyuCL6paEluXzepH2dIdgpgb5mIFz0VxRsGMqI9REMIhPzOf3Ixcuge6/WaU13dKWRJBEOKDiLkUxRJoViFdX1Q3J7b9VXG2S8wFsswNj5kLIObEwiEkE8FcrXLcCoIQL0TMpSjeYs6fm8cSehVFFXEblzeWZa65u3nkTF+Wudpa2LcP8vJgyRL3ZLkoCsmIiDlBEJKF9EQPQAiP4pxiCrMLOdtzltOdp0e4clp7WmnsaiQnPYfJ4xITMxeymNu0yTxffTVkZgLQ3d9Na28rGY4MirKLYjlcQQgJX2Lu3TPvsu/0PgBOtJ4YspwgCEKsEDGXwlQWV7Lr1C6qmqtGiDkrlq6iuAKHSowB1paYO3XKM82Hi9W7HZlksgrJhJVYdKz1GAAdfR1c9t+X0d7X7l4mTaVRmluakPEJgjB2EDdrCmO5Wn0lQSRDU/qAYq683DwfPWqeBwZg82bzWuLlhBRg0ZRFALxQY0rpvHD0Bdr72pmcP5mbz7+Zm8+/mZ9f93My0jISOUxBEMYAYplLYSqL/CdBuMVcUZKKuUmTTGxcUxO0tMC778LZs1BZCbNnuxeTRuVCsrK2Yi1gRFzPQI+7HuInF36SLy//ciKHJgjCGEMscylMoIzWpLfMKWWEG0B1tU8XK3i1RJLCq0KSMXncZOaXzad7oJuXj73sbjm3rmJdgkcmCMJYQ8RcCmPVmgsk5qxlEkFRjklY8CnmACpcY6uq8ivmxM0qJDPrK8zx+uDOBznScoTinGIWTlmY4FEJgjDWEDGXwlhWt8PNh0eUJ0k2y5zP8imWZe6NN2DnTpPBumLFkEVEzAnJzPpKI+aePvQ0YFyvaY60RA5JEIQxiIi5FGZS3iRKc0s523OWd8+8655+tOUodR11FGQVMH389ISNLyc9h6y0LHoHe+ke6B65gCXmHnkEtIalS00cnRdWFX0Rc0Iysnj6Ys6fcD4A6Y50PjrvowkekSAIYxERcymMUsodhL2xeqN7uvV69ezVCbUSKKUCx81ZYu7sWfM8zMUKYpkTkpvMtEze+dQ7tPxrC233t3HNnGsSPSRBEMYgIuZSHMvNY2XSeb+25iUSW2LOYt3IwHERc0Kyk+ZIozC7kJyMnEQPRRCEMYqIuRTHssy9dOwluvu76RvsY+vRrUByZNUFFHNTp0JWlnk9ZQpceOGQ2VprKU0iCIIgCEGQOnMpzsS8iSyYvIA3697k7j/fTbojnfa+di4ovYDpBYmLl7MIKOYcDlNT7uBB42Id1uGhrbeNnoEe8jLyyM/Mj8dwBUEQBCHlEDE3CrhuznW8Wfcmv33nt0OmJQMBxRzAxRcbMfe+942Ydbj5MABTx0+N2fgEQRAEIdURMTcKuPfKeynLL6OzvxMwWaS3z7s9waMyWGKupbvF9wI//CF8+MNwww0jZm2q3gTA8pnLYzY+QRAEQUh1RMyNAsZljeNTiz6V6GH4JKhlbvJkuPFGn7OsrNxkiP0TBEEQhGRFEiCEmBJUzPmhrbeNV0+8SppKY9XsVbEYmiAIgiCMCkTMCTGlKNvV0qsnNDG39ehWBpwDXDHtCgqzC2MwMkEQBEEYHYiYE2KKZZlr6moK6XNWvFwy1MoTBEEQhGRGxJwQU6aNnwbAsdZjIX1uf+N+AC6fennUxyQIgiAIowkRc0JMmVU0C4Xi2Nlj9A/22/5cVXMVAHNK5sRqaIIgCIIwKhAxJ8SU7PRsphdMZ1AP2rbOdfZ1cqr9FBmODKaPT3zhY0EQBEFIZkTMCTGnstj0YLWsbcE40nIEMFa9NEdazMYlCIIgCKMBEXNCzKksCk3MVbdUm8+5RKAgCIIgCP4RMSfEnIriCsC+mLOWs0SgIAiCIAj+ETEnxJxQ3axuMSeWOUEQBEEIiog5IeaImBMEQRCE2CFiTog5FUXGzXr07FEGnYNBlxcxJwiCIAj2ETEnxJy8zDwm50+mb7CP2rbagMv2DvRyvPU4aSqNmYUz4zRCQRAEQUhdRMwJccFKgrAyVf3xXtN7aDQzC2eSmZYZj6EJgiAIQkojYk6IC1Zbr7r2uoDLbT6yGYAl05fEfEyCIAiCMBoQMSfEhbK8MgDqO+oDLrehagMA6yvXx3xMgiAIgjAaSIiYU0oVK6U2K6UOu56L/Cy3Xil1SClVpZS6f9i8z7jm7VdKfTc+IxfCZVL+JCCwmOvq72LbsW0oFGtmr4nX0ARBEAQhpUmUZe5+YIvWeg6wxfV+CEqpNOBB4BrgAuA2pdQFrnkrgRuBeVrrucD34zVwITzK8l2WuU7/Yu6lmpfoHexlwZQFlOaVxmtogiAIgpDSJErM3Qg86nr9KHCTj2UuA6q01ke01n3AH1yfA/gU8IDWuhdAa306tsMVIsUt5gJY5jZWbwRgfYW4WAVBEATBLokSc5O01nUArueJPpaZCpzwel/rmgZwDrBUKfWGUuolpdQifxtSSn1cKbVLKbWrsbExSsMXQsUScw0dDX6X2VO/B4CrZlwVjyEJgiAIwqggPVYrVko9D5T5mPVFu6vwMU27ntOBIuAKYBHwmFJqttZaj/iA1g8BDwEsXLhwxHwhPtixzFnFgueUzInLmARBEARhNBAzMae1Xu1vnlKqQSk1WWtdp5SaDPhyk9YC073eTwNOec170iXediilnMAEQExvSUppbikO5eBM1xn6B/vJSMsYMr+7v5uT7SdJd6Qzo2BGgkYpCIIgCKlHotyszwB3uF7fATztY5mdwByl1CylVCbwYdfnAP4PuBpAKXUOkAmcieWAhchIc6RRmluKRtPYNVJzH2k5AsCswlmkO2J2jyEIgiAIo45EibkHgDVKqcPAGtd7lFJTlFLPAmitB4B7gI3AQeAxrfV+1+d/BcxWSu3DJEbc4cvFKiQXgVyt0o9VEARBEMIjISYQrXUTsMrH9FPAtV7vnwWe9bFcH3B7LMcoRJ+y/DLebnhbxJwgCIIgRBHpACHEDbHMCYIgCEL0ETEnxI2AYq7FiLmKooq4jkkQBEEQUh0Rc0LcCFRrTixzgiAIghAeIuaEuOGvpVffYB/HW4/jUA7KC8sTMDJBEARBSF1EzAlxY1LeJADq2uuGTK85W4NTO5lRMIOs9KxEDE0QBEEQUhYRc0LcmFk4E4CjZ48OmS4uVkEQBEEIHxFzQtyYUTCDdEc6tW21dPd3u6e7xVyRiDlBEARBCBURc0LcSHekM6twFuDp+ABimRMEQRCESBAxJ8QVS7BZAs77dUWxlCURBEEQhFARMSfEFUvMVbdUu6eJZU4QBEEQwkfEnBBXrKLAloAbcA64EyJmF81O2LgEQRAEIVURMSfEleFu1hOtJxhwDjB13FRyM3ITOTRBEARBSElEzAlxZbiYExerIAiCIESGiDkhrpQXluNQDo61HqNvsE/EnCAIgiBEiIg5Ia5kpWcxo2AGTu2k5myNiDlBEARBiBARc0LcsYTbvtP72N+4f8g0QRAEQRBCQ8ScEHeWzlgKwF/f+yvbjm0DYMn0JYkckiAIgiCkLCLmhLizrmIdAP+793/pHuhmftl8Jo+bnOBRCYIgCEJqImJOiDsLpyykOKeYAecA4BF3giAIgiCEjog5Ie6kOdJYM3uN+/36yvUJHI0gCIIgpDYi5oSEYAm4/Mx8rpx+ZYJHIwiCIAipS3qiByCMTW4890YWTF7AtXOuJTMtM9HDEQRBEISURcSckBCKcorY9fFdiR6GIAiCIKQ84mYVBEEQBEFIYUTMCYIgCIIgpDAi5gRBEARBEFIYEXOCIAiCIAgpjIg5QRAEQRCEFEbEnCAIgiAIQgojYk4QBEEQBCGFETEnCIIgCIKQwoiYEwRBEARBSGFEzAmCIAiCIKQwIuYEQRAEQRBSGBFzgiAIgiAIKYyIOUEQBEEQhBRGaa0TPYa4oZRqBI7FcBMTgDMxXP9oQvZVaMj+so/sq9CQ/WUf2VehIfvLPv721UytdWmwD48pMRdrlFK7tNYLEz2OVED2VWjI/rKP7KvQkP1lH9lXoSH7yz6R7itxswqCIAiCIKQwIuYEQRAEQRBSGBFz0eWhRA8ghZB9FRqyv+wj+yo0ZH/ZR/ZVaMj+sk9E+0pi5gRBEARBEFIYscwJgiAIgiCkMCLmooBSar1S6pBSqkopdX+ix5OMKKVqlFLvKKX2KKV2uaYVK6U2K6UOu56LEj3ORKCU+pVS6rRSap/XNL/7Rin1BdexdkgptS4xo04cfvbXV5VSJ13H1x6l1LVe88bs/lJKTVdKvaCUOqiU2q+U+kfXdDm+hhFgX8mx5QOlVLZSaodS6m3X/vqaa7ocW8MIsK+id2xpreURwQNIA6qB2UAm8DZwQaLHlWwPoAaYMGzad4H7Xa/vB76T6HEmaN8sAy4F9gXbN8AFrmMsC5jlOvbSEv0dkmB/fRW418eyY3p/AZOBS12vxwHvufaJHF/295UcW773lwLyXa8zgDeAK+TYCmlfRe3YEstc5FwGVGmtj2it+4A/ADcmeEypwo3Ao67XjwI3JW4oiUNrvQ1oHjbZ3765EfiD1rpXa30UqMIcg2MGP/vLH2N6f2mt67TWu12v24GDwFTk+BpBgH3ljzG7rwC0ocP1NsP10MixNYIA+8ofIe8rEXORMxU44fW+lsAngLGKBjYppd5USn3cNW2S1roOzIkUmJiw0SUf/vaNHG/+uUcptdflhrVcO7K/XCilyoFLMFYBOb4CMGxfgRxbPlFKpSml9gCngc1aazm2/OBnX0GUji0Rc5GjfEyTFOGRLNFaXwpcA3xaKbUs0QNKUeR4880vgApgPlAH/MA1XfYXoJTKB54APqe1bgu0qI9pY2p/+dhXcmz5QWs9qLWeD0wDLlNKXRhg8TG9v/zsq6gdWyLmIqcWmO71fhpwKkFjSVq01qdcz6eBpzAm4wal1GQA1/PpxI0w6fC3b+R484HWusF1snQC/43HJTHm95dSKgMjTn6rtX7SNVmOLx/42ldybAVHa30WeBFYjxxbAfHeV9E8tkTMRc5OYI5SapZSKhP4MPBMgseUVCil8pRS46zXwFpgH2Y/3eFa7A7g6cSMMCnxt2+eAT6slMpSSs0C5gA7EjC+pMK6eLh4P+b4gjG+v5RSCvglcFBr/UOvWXJ8DcPfvpJjyzdKqVKlVKHrdQ6wGngXObZG4G9fRfPYSo/6qMcYWusBpdQ9wEZMZuuvtNb7EzysZGMS8JQ5V5IO/E5rvUEptRN4TCn1d8Bx4JYEjjFhKKV+D6wAJiilaoGvAA/gY99orfcrpR4DDgADwKe11oMJGXiC8LO/Viil5mNcETXAJ0D2F7AE+AjwjiteB+DfkOPLF/721W1ybPlkMvCoUioNYxh6TGv9F6XUa8ixNRx/++rX0Tq2pAOEIAiCIAhCCiNuVkEQBEEQhBRGxJwgCIIgCEIKI2JOEARBEAQhhRExJwiCIAiCkMKImBMEQRAEQUhhRMwJgpBUKKXer5TSSqnzvKaVK6X2uV4vVEr9JHEj9I1S6ial1AUx3sarrudypdTfeE1Pyn0iCEJ8EDEnCEKycRuwHVOAewRa611a68/Gd0i2uAmIqZjTWl/pelkO/I3X9GTdJ4IgxAERc4IgJA2uvphLgL/Dj5hTSq1QSv3F9bpUKbVZKbVbKfVfSqljSqkJLsvVQaXUfyul9iulNrkqr6OUelEp9SOl1DbXMouUUk8qpQ4rpb7htZ3blVI7lFJ7XOtOc03vUEp9Uyn1tlLqdaXUJKXUlcANwPdcy1cMG/MjSqn/VEq9rJR6Tyl1vWt6tlLqYaXUO0qpt5RSK13T53pte69Sao61bdcqHwCWuuZ/ftg+KVZK/Z/rc68rpea5pn9VmWbeLyqljiilRPwJwihBxJwgCMnETcAGrfV7QLNS6tIgy38F2Kq1vhTT83eG17w5wINa67nAWeBmr3l9WutlwH9i2g19GrgQuFMpVaKUOh/4ELDE1Rx7EPhb12fzgNe11hcD24C7tdavYlrw/IvWer7WutrHWMuB5cB1wH8qpbJd20VrfRHGIvmoa/ongf9wbXshplejN/cDL7u29aNh874GvKW1nofpYPC/XvPOA9ZhekB+RZlepIIgpDjSzksQhGTiNuDHrtd/cL3fHWD5qzA9DXG1iGvxmndUa73H9fpNjJiysPonvwPs11rXASiljmAaXF8FLAB2utrQ5eBpGN4H/MVrvWtsfrfHXA21D7u2c55rOz91jf9dpdQx4BzgNeCLSqlpwJNa68M2t4FrnTe71rnVJU4LXPP+qrXuBXqVUqcxrfaGC0VBEFIMEXOCICQFSqkS4GrgQqWUxvQ61kqp+wJ9LMC8Xq/XgxhBNnyec9hyTsx5UQGPaq2/4GO9/drTB3EQ++fR4b0TNX7Gr7X+nVLqDYwVb6NS6u+11lttbsfXOq1tD98ncg0QhFGAuFkFQUgWPgj8r9Z6pta6XGs9HTiKsTT5YztwK4BSai1QFKWxbAE+qJSa6Fp3sVJqZpDPtAPjAsy/RSnlcMXTzQYOYdy0f+vaxjkYN/EhpdRs4IjW+icYK+K8ELblvc4VwBmtdVuQsQuCkMKImBMEIVm4DRP35s0TeGVt+uBrwFql1G7gGqAOI3QiQmt9APgSsEkptRfYDEwO8rE/AP/iSmSo8DH/EPAS8BzwSa11D/BzIE0p9Q7wR+BOlxv0Q8A+pdQejDv2f4etay8w4ErC+PyweV8FFrrG/QBwh53vLAhC6qI83gJBEITUQimVBQxqrQeUUouBX7iSBpIKpdQjwF+01o8neiyCIIw+JF5CEIRUZgbwmFLKgUlMuDvB4xEEQYg7YpkTBEEQBEFIYSRmThAEQRAEIYURMScIgiAIgpDCiJgTBEEQBEFIYUTMCYIgCIIgpDAi5gRBEARBEFIYEXOCIAiCIAgpzP8HaJF4Mp0Twt4AAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 720x432 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "\n",
    "\n",
    "def r_enumerate(seq):\n",
    "    \"\"\"Enumerate a sequence in reverse order\"\"\"\n",
    "    # Note that we don't use reversed() since Python 2.3 doesn't have it\n",
    "    num = len(seq) - 1\n",
    "    while num >= 0:\n",
    "        yield num, seq[num]\n",
    "        num -= 1\n",
    "\n",
    "def get_profile(profile_file, seq):\n",
    "    \"\"\"Read `profile_file` into a Python array, and add gaps corresponding to\n",
    "       the alignment sequence `seq`.\"\"\"\n",
    "    # Read all non-comment and non-blank lines from the file:\n",
    "    f = open(profile_file)\n",
    "    vals = []\n",
    "    for line in f:\n",
    "        if not line.startswith('#') and len(line) > 10:\n",
    "            spl = line.split()\n",
    "            vals.append(float(spl[-1]))\n",
    "    # Insert gaps into the profile corresponding to those in seq:\n",
    "    for n, res in r_enumerate(seq.residues):\n",
    "        for gap in range(res.get_leading_gaps()):\n",
    "            vals.insert(n, None)\n",
    "    # Add a gap at position '0', so that we effectively count from 1:\n",
    "    vals.insert(0, None)\n",
    "    return vals\n",
    "\n",
    "e = modeller.environ()\n",
    "a = modeller.alignment(e, file='TvLDH-4h7pA.ali')\n",
    "\n",
    "template = get_profile('4h7pA.profile', a['4h7p'])\n",
    "model = get_profile('TvLDH.profile', a['TvLDH'])\n",
    "\n",
    "# Plot the template and model profiles in the same plot for comparison:\n",
    "pylab.figure(1, figsize=(10,6))\n",
    "pylab.xlabel('Alignment position')\n",
    "pylab.ylabel('DOPE per-residue score')\n",
    "pylab.plot(model, color='red', linewidth=2, label='Model')\n",
    "pylab.plot(template, color='green', linewidth=2, label='Template')\n",
    "pylab.legend()\n",
    "pylab.savefig('dope_profile.png', dpi=65)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# This notebook is compiled and verfied by Girinath G. Pillai, @giribio\n",
    "\n",
    "### Kindly cite this notebook in your publications if you have used it for calculations.\n",
    "\n",
    "### Cite as : NyroMDNotebook v1, Girinath G. Pillai : https://github.com/giribio/MDNotebooks"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
